<html><head><title>avillach-lab.hms.harvard.edu Combined Content</title></head><body>
<!-- Source: avillach-lab.hms.harvard.edu -->
<h1>Avillach Lab | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu'>avillach-lab.hms.harvard.edu</a></div>
<main aria-label="Avillach Lab" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base hwp-sr-only">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Avillach Lab</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="field field--name-field-hwp-additional-content field--type-entity-reference-revisions field--label-hidden">
<div class="paragraphs-item__first paragraphs-item">
<div class="hwp-text-block hwp-bg-light-core hwp-py-32">
<div class="hwp-container-px hwp-container">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><h2>Welcome to Avillach Lab at the <a href="https://cbmi.med.harvard.edu">Department of Biomedical Informatics</a></h2><p><strong>The real value in biomedical research lies not at the scale of any single source of data but in the ability to integrate and interrogate multiple complementary datasets simultaneously.</strong></p><p dir="ltr">The Avillach Lab, led by <a data-entity-substitution="canonical" data-entity-type="node" data-entity-uuid="d7b96fd9-1e1b-403e-a097-7e636b911581" href="/people/paul-avillach-md-phd" title="Paul Avillach, MD, PhD">Paul Avillach, M.D., Ph.D.</a>,  focuses on translational bioinformatics, specifically in integrating multiple heterogeneous sources of clinical and genomic data in a meaningful way. We are passionate about combining data across different scales and resolutions to enable new perspectives for essential biomedical questions. Our research focuses on developing novel methods and techniques for the integration of biological data, clinical cohorts, and Electronic Health Records to encompass biological observations.</p><p dir="ltr">We leverage the PIC-SURE (Patient-centered Information Commons: Standardized Unification of Research Elements) platform to manage a wide variety of arbitrarily large datasets with very little computing in a secured way by searching and extracting clinical and genomic variables of interest. Exposing the data not only at the file level but also at the variable and variant level directly accelerates creation of statistically ready dataframes. PIC-SURE has been deployed in a variety of environments, such as the NIH NHLBI BioData Catalyst® Ecosystem, the Undiagnosed Disease Network, and Boston Children’s Hospital (see <a data-entity-substitution="canonical" data-entity-type="node" data-entity-uuid="e41c2647-f3b6-4a46-a5e8-f8504d568a91" href="/research" title="Research &amp; Collaborations">research</a>).</p><figure class="hwp-media hwp-media--full-width" role="group">
<div>
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <img alt="A person's data consists of many &quot;layers&quot; or data types" height="500" loading="lazy" src="/sites/g/files/omnuum4001/files/avillach_lab/files/home2.png" width="1000"/>
</div>
</div>
<figcaption><strong>Figure depicting various, interconnected “layers” of clinical and genomic data.</strong></figcaption>
</figure>
<p>PIC-SURE contains both a graphical user interface (UI) and an application programming interface (API). With the UI, researchers can test their hypotheses by running multi-criteria queries at both the file- and variable-levels and applying distinct clinical and genomic variable filters in the same search. With the API, users can filter and directly retrieve the data in an analysis-ready format using either R or Python. </p><p> </p></div>
</div></div>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/courses-0 -->
<h1>Courses | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/courses-0'>avillach-lab.hms.harvard.edu/courses-0</a></div>
<main aria-label="Courses" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Courses</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p><a href="http://www.medcatalog.harvard.edu/coursedetails.aspx?cl=&amp;id=21648"><strong>AISC610.0  Computationally-Enabled Medicine</strong></a></p><p><a href="https://dbmi.hms.harvard.edu/education/courses"><strong>BMI 705 - Precision Medicine II: Integrating Clinical and Genomic Data</strong></a></p></div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/login -->
<h1>Log in | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/login'>avillach-lab.hms.harvard.edu/login</a></div>
<main aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="hwp-login-section js-hwp-login-section hide">
<div class="hwp-login-section__inner">
<img alt="Harward Key Logo" class="hwp-login-section__logo" src="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/images/svg/harvard-key.svg"/>
<h1 class="h2 hwp-login-section__title"> Log In</h1>
<a class="hwp-button hwp-button--large hwp-login-section__harvard-key-button hwp-button--has-bg hwp-button--strong-primary analytics-cta" href="/caslogin">
<span class="hwp-button__text">Log in with HarvardKey</span>
<span aria-hidden="true" class="material-icon">arrow_circle_right</span>
</a>
<div class="hwp-login-section__add-links-wrapper">
<a class="hwp-login-section__support-link" href="mailto:ithelp@harvard.edu">Contact support</a>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/about -->
<h1>Contact us | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/about'>avillach-lab.hms.harvard.edu/about</a></div>
<main aria-label="Contact us" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Contact us</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="field field--name-field-hwp-additional-content field--type-entity-reference-revisions field--label-hidden">
<div class="paragraphs-item__first paragraphs-item">
<div class="hwp-text-block hwp-bg-light-core hwp-py-32">
<div class="hwp-container-px hwp-container">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><div class="align-left hwp-media hwp-media--medium">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="150" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_1_1__720x720_scale/public/avillach_lab/files/countway.jpg?itok=pNmXJC8V 1x, /sites/g/files/omnuum4001/files/styles/hwp_1_1__1440x1440_scale/public/avillach_lab/files/countway.jpg?itok=3WmOajQa 2x" type="image/jpeg" width="200"/>
<img alt="countway.jpg" height="150" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_1_1__720x720_scale/public/avillach_lab/files/countway.jpg?itok=pNmXJC8V" width="200"/>
</picture>
</div>
</div>
<p><strong>Avillach Lab</strong><br/>Department of Biomedical Informatics<br/>10 Shattuck St, 4th Floor<br/>Boston, MA 02115<br/><a href="mailto:paul_avillach@hms.harvard.edu">paul_avillach@hms.harvard.edu</a></p></div>
</div></div>
</div>
<div class="paragraphs-item">
<div class="hwp-text-block hwp-bg-light-core hwp-py-32">
<div class="hwp-container-px hwp-container">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p><strong>To the Longwood Medical Area:</strong></p><p>* If you're coming by the T, Green Lines D ("Longwood" stop) and E ("Longwood Medical Area" stop) are 2-3 blocks away.<br/>* From the Harvard main campus, take the LMA (Longwood Medical Area) M2 shuttle (click <a href="http://www.masco.org/transit/ptsM2.htm">here</a> for schedule). A Harvard ID is required. You can catch the bus by Lamont Library; it takes 15-25 minutes depending on traffic.<br/>* From MIT, you can take the Crosstown bus (CT2) (click <a href="http://www.mbta.com/traveling_t/schedules_buses_findbus.asp?routenumber=CT2&amp;imageField2.x=60&amp;imageField2.y=14">here</a> for route and schedule).</p></div>
</div></div>
</div>
<div class="paragraphs-item">
<div class="hwp-text-block hwp-bg-light-core hwp-py-32">
<div class="hwp-container-px hwp-container">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p><strong>Entry to the Building:</strong></p><p>* <strong>Countway</strong>: A Harvard, MIT, or a hospital ID is needed. (If you do not have it, please speak to the guard and explain that you are visitor. A call to my office may be needed). The Library is located next to the Gordon Hall in the main Quad (you can see the top of the library between the buildings):</p><div class="hwp-media hwp-media--medium">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="270" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_1_1__720x720_scale/public/avillach_lab/files/rsch_sci_env_quad_0.jpg?itok=UNR3Oav6 1x, /sites/g/files/omnuum4001/files/styles/hwp_1_1__1440x1440_scale/public/avillach_lab/files/rsch_sci_env_quad_0.jpg?itok=SBwxDP92 2x" type="image/jpeg" width="508"/>
<img alt="rsch_sci_env_quad_0.jpg" height="270" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_1_1__720x720_scale/public/avillach_lab/files/rsch_sci_env_quad_0.jpg?itok=UNR3Oav6" width="508"/>
</picture>
</div>
</div>
</div>
</div></div>
</div>
<div class="paragraphs-item__last paragraphs-item">
<div class="hwp-text-block hwp-bg-light-core hwp-py-32">
<div class="hwp-container-px hwp-container">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p><strong>Map of the Longwood Medical Area (Click </strong><a href="https://prod-avillachlab.drupalsites.harvard.edu/sites/g/files/omnuum4001/files/avillach_lab/files/longwoodmap.pdf"><strong>here</strong></a><strong> for the pdf map)</strong></p><div class="hwp-media hwp-media--medium">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="562" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_1_1__720x720_scale/public/avillach_lab/files/longwoodmap.gif?itok=rWIfPa1h 1x, /sites/g/files/omnuum4001/files/styles/hwp_1_1__1440x1440_scale/public/avillach_lab/files/longwoodmap.gif?itok=Is4PFdii 2x" type="image/gif" width="720"/>
<img alt="longwoodmap.gif" height="562" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_1_1__720x720_scale/public/avillach_lab/files/longwoodmap.gif?itok=rWIfPa1h" width="720"/>
</picture>
</div>
</div>
</div>
</div></div>
</div>
</div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="hwp-attachments" data-component-name="attachments">
<hr class="hwp-text-button-light-secondary hwp-my-32"/>
<span>Attachments</span>
<ul aria-label="Attachments" class="lg:hwp-grid hwp-attachments__items">
<li>
<a class="hwp-icon-link hwp-icon-link--x-large hwp-icon-link--has-bg hwp-icon-link--dark-primary analytics-cta" href="/sites/g/files/omnuum4001/files/avillach_lab/files/longwoodmap.pdf">
<span class="hwp-icon-link__icon">
<span aria-hidden="true" class="material-icon">picture_as_pdf</span>
</span>
<span class="hwp-icon-link__text">longwoodmap.pdf</span>
</a>
</li>
</ul>
<hr class="hwp-text-button-light-secondary hwp-my-24"/>
</div>
<div class="page-tags">
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/pic-sure -->
<h1>PIC-SURE | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/pic-sure'>avillach-lab.hms.harvard.edu/pic-sure</a></div>
<main aria-label="PIC-SURE" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>PIC-SURE</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><h2>Introduction to PIC-SURE</h2><p dir="ltr"><span>The Patient-centered Information Commons: Standard Unification of Research Elements (PIC-SURE) platform integrates different layers of clinical and genomic data from diverse data sources, providing a multifaceted approach to biomedical research. </span></p><p dir="ltr"><span>The PIC-SURE platform was built on </span><a href="https://www.i2b2.org/"><span>Informatics for Integrating Biology &amp; the Bedside (i2b2)</span></a><span>, a data model created for electronic health record data. PIC-SURE is developed with an open-source Apache 2.0 license and deployed in Federal Information Security Modernization Act (FISMA) Moderate ATO and HI-TRUST environments.</span></p><p dir="ltr"> </p><h2><span>Features of PIC-SURE</span></h2><h3><span>User Interface and Application Programming Interface</span></h3><p><span>The PIC-SURE platform provides both an intuitive graphical user interface (UI) and an application programming interface (API) to meet different use cases and levels of experience with data manipulation. The PIC-SURE UI allows for an investigator to search for variables of interest and to conduct feasibility queries. In this way, cohorts are built in real-time and results can be retrieved for analysis. </span></p><h3><span>Query Builder</span></h3><p><span>The Query Builder in the PIC-SURE UI allows users to interactively build queries and returns the counts of participants that meet the criteria. This tool functions similarly to a search engine, providing the ability to search a term of interest and returning variables that contain the search term. After determining a variable of interest, users can refine their query to filter participants with certain values and build a cohort. Both single and multiple criteria queries are supported.</span></p><h3><span>GitHub</span></h3><p dir="ltr"><span>PIC-SURE is developed under an Apache 2.0 license; all code is open source and accessible on the </span><a href="https://github.com/hms-dbmi/pic-sure-all-in-one"><span>PIC-SURE All-in-one GitHub repository</span></a><span>.   </span></p><p dir="ltr"><span>For users interested in the PIC-SURE API, several Jupyter notebooks (R and Python) and R Markdown files contain coding examples for key PIC-SURE use cases. These examples provide a starting point for users to get started with their own analyses and are publicly available on the </span><a href="https://github.com/hms-dbmi/Access-to-Data-using-PIC-SURE-API"><span>Access to Data Using PIC-SURE API GitHub Repository</span></a><span>.  </span></p><p dir="ltr"> </p><h2><span>Training Materials</span></h2><h3><span>Documentation </span></h3><p><span>Written documentation for PIC-SURE can be found on the </span><a href="https://pic-sure.gitbook.io/pic-sure"><span>PIC-SURE Gitbook documentation</span></a><span>.   </span></p><h3><span>Videos </span></h3><p>The <a href="https://www.youtube.com/channel/UCIfahDUjW4A4B1KyIek4YJA">PIC-SURE YouTube channel</a> has video demonstrations of PIC-SURE features such as searching, filtering, and building cohorts. Researchers can find videos specific to certain PIC-SURE environments by <a href="https://www.youtube.com/channel/UCIfahDUjW4A4B1KyIek4YJA/playlists">viewing the playlists</a>.</p><p> </p><h2>PIC-SURE Instances</h2><h3>Demo PIC-SURE</h3><p>The demonstration site of PIC-SURE allows users to access the <a href="https://www.cdc.gov/nchs/nhanes/index.html">National Health and Nutrition Examination Survey (NHANES) dataset</a>. You can submit a request to access the site by emailing <a href="mailto:avillach_lab_developers@googlegroups.com">avillach_lab_developers@googlegroups.com</a>. </p><h3>NHLBI BioData Catalyst®</h3><p>The National Heart, Lung, and Blood Institute BioData Catalyst® (BDC) is a cloud-based ecosystem that offers researchers data, analytic tools, applications, and workflows in secure workspaces. </p><p><a href="https://picsure.biodatacatalyst.nhlbi.nih.gov/">BDC Powered by PIC-SURE</a> is available to search hosted heart, lung, blood, and sleep data publicly. </p><h3>Genomic Information Commons </h3><p>The Genomic Information Commons (GIC) is a continuously updating, queryable, federated genomic data commons. <span>Inspired by a common vision of accelerated genomic research discovery, collaboration, and improved clinical outcomes, leaders at Boston Children’s Hospital (BCH), Harvard Medical School, Cincinnati Children’s Hospital Medical Center (CCHMC), the Children’s Hospital of Philadelphia (CHOP), St. Louis Children’s Hospital (SLCH)/Washington University at St. Louis (WUSTL), and the University of Pittsburgh Medical Center (UPMC) have come together to create the GIC.</span></p><p><span>The GIC is available to those in the GIC consortium. You can learn more at the </span><a href="https://www.genomicinformationcommons.org/"><span>GIC website</span></a><span>. </span></p><h3>AIM-AHEAD</h3><p>The National Institute of Health's AIM-AHEAD program has established mutually beneficial, coordinated, and trusted partnerships to enhance the participation and representation of researchers and communities currently under-resourced in the development of artificial intelligence and machine learning models and to improve the capabilities of this emerging technology, beginning with electronic health records (EHR) and extending to other data sets to address health inconsistencies. </p><p>To learn more, please refer to the <a href="https://www.aim-ahead.net/">AIM-AHEAD website</a>. </p></div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/publications -->
<h1>Publications | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/publications'>avillach-lab.hms.harvard.edu/publications</a></div>
<main aria-label="Publications" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="search-page search-page--no-filters hwp-listing-page hwp-bibcite_reference">
<div class="hwp-page-title hwp-bg-dark-base" lang="en">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Publications
    </h1>
</div>
</div>
<div class="hwp-section-heading" data-component-name="section-heading">
<div class="hwp-section-heading__container hwp-container-px hwp-container">
<div class="hwp-section-heading__description hwp-text-block">
</div>
</div>
</div>
<div class="search-page__columns hwp-container hwp-container-px">
<div class="search-page__results">
<div class="search-page__search-form">
<form accept-charset="UTF-8" action="/publications" class="hwp-search-form-listing" data-drupal-selector="hwp-search-form-listing" id="hwp-search-form-listing" method="post">
<div class="hwp-form-type-search hwp-form-item js-form-item form-item form-type-search js-form-type-search form-item-search js-form-item-search form-no-label">
<label class="visually-hidden" for="edit-search">Search Within Results</label>
<input class="form-search" data-drupal-selector="edit-search" id="edit-search" maxlength="128" name="search" size="60" type="search" value=""/>
</div>
<button class="hwp-button hwp-button--small button js-form-submit form-submit" data-drupal-selector="edit-submit" data-twig-suggestion="header_site_search_submit" id="edit-submit" name="op" type="submit" value="Search Within Results">
<span class="hwp-button__text">Search Within Results</span>
<span aria-hidden="true" class="material-icon">search</span>
</button>
<input autocomplete="off" data-drupal-selector="form-imypagg73lxskojfuiqljy0prljdf1zrgbqht0wzqqg" name="form_build_id" type="hidden" value="form-IMyPAgg73LxsKOjFuiQLJy0PRLJdF1zrgBQht0WzQQg"/>
<input data-drupal-selector="edit-hwp-search-form-listing" name="form_id" type="hidden" value="hwp_search_form_listing"/>
</form>
</div>
<div class="views-element-container">
<div class="js-view-dom-id-ddcb69bd5855ccc9e781df57ec03f7ba1cb5ed7530fbfada62e7b414010061bf">
<div class="search-page__search-header">
<div class="hwp-hidden lg:hwp-block search-page__results-summary">
        100 results 
      </div>
<button aria-expanded="false" class="search-page__filters-trigger" data-filters-trigger="">
<span>Show filters</span>
<span aria-hidden="true" class="material-icon">filter_alt</span>
</button>
<form accept-charset="UTF-8" action="/node/1774776" class="views-exposed-form bef-exposed-form" data-bef-auto-submit="" data-bef-auto-submit-delay="500" data-bef-auto-submit-full-form="" data-bef-auto-submit-minimum-length="3" data-drupal-selector="views-exposed-form-hwp-search-search-page" id="views-exposed-form-hwp-search-search-page" method="get">
<input data-drupal-selector="edit-search" name="search" type="hidden" value=""/>
<div class="js-form-item form-item hwp-form-item js-form-type-select hwp-form-type-select form-item-sort-bef-combine js-form-item-sort-bef-combine">
<label for="edit-sort-bef-combine">Sort by</label>
<select class="form-select" data-drupal-selector="edit-sort-bef-combine" id="edit-sort-bef-combine" name="sort_bef_combine"><option selected="selected" value="hwp_bibcite_reference_year--desc">Year of Publication</option><option value="title--asc">Alphabetical A-Z</option></select>
<span aria-hidden="true" class="material-icon">sort</span>
</div>
<div class="form-actions js-form-wrapper form-wrapper" data-drupal-selector="edit-actions" id="edit-actions"><input class="js-hide button js-form-submit form-submit" data-bef-auto-submit-click="" data-drupal-selector="edit-submit-hwp-search" id="edit-submit-hwp-search" type="submit" value="Apply"/>
</div>
</form>
</div>
<div class="lg:hwp-hidden search-page__results-summary">
    100 results 
  </div>
<div class="search-page__result-items search-page__result-items--">
<div class="hwp-citations-list__downloads">
<div class="button-dropdown" data-component-name="button-dropdown">
<button class="hwp-text-link hwp-text-link--icon-right" id="citations_download">
<span class="hwp-text-link__text">    Download 100 citations  </span>
<span aria-hidden="true" class="material-icon">download</span>
</button>
<ul aria-labelledby="citations_download" class="button-dropdown__items" role="menu">
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=bibtex">BibTeX</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=endnote8">EndNote X3 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=endnote7">EndNote 7 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=tagged">Endnote tagged</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=marc">Marc</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=pubmed_id">PubMedId</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=ris">RIS</a>
</li>
</ul>
</div>
</div>
<div id="">
<div class="hwp-page-list__items">
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">2025</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Alkuraya I, Almansa AS, Eleonu A, Avillach P, Poduri A, Srivastava S. <a href="/publication/use-computational-phenotypes-predicting-genetic-subgroups-cerebral-palsy">Use of Computational Phenotypes for Predicting Genetic Subgroups of Cerebral Palsy.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2025; PMID: 39990554</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Alkuraya I, Almansa AS, Eleonu A, Avillach P, Poduri A, Srivastava S. <a href="/publication/use-computational-phenotypes-predicting-genetic-subgroups-cerebral-palsy">Use of Computational Phenotypes for Predicting Genetic Subgroups of Cerebral Palsy.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2025; PMID: 39990554</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-983783844" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-983783844">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>INTRODUCTION: Emerging evidence suggests that 20-30% of cases of cerebral palsy (CP) may have a genetic cause. Our group previously identified subsets of patients with CP or CP-masquerading conditions who warrant genetic testing, including those with regression or progressive neurological symptoms (CP masqueraders) and those without any known risk factors for CP (cryptogenic CP). Recognition of these subgroups in clinical settings remains challenging.</p><p>METHODS: To address this challenge, we developed and evaluated a computational phenotyping approach using ICD- 9/ICD-10 billing codes to automatically identify patients with unexplained CP or CP-masquerading conditions who may benefit from genetic testing. We applied this computational phenotyping approach to a cohort of 250 participants from the Boston Children's Hospital CP Sequencing Study, aimed at identifying genetic causes in CP and CP-masquerading conditions.</p><p>RESULTS: Manual review served as the gold standard, identifying 8% as CP masqueraders, 42% as cryptogenic CP, and 50% as non-cryptogenic CP. Computational phenotyping based on ICD-9/10 codes achieved a sensitivity of 95%, specificity of 72%, positive predictive value of 77%, and negative predictive value of 94% in identifying cases warranting genetic testing.</p><p>CONCLUSIONS: Our findings demonstrate the feasibility of using computational phenotyping to identify patients with CP or CP- masquerading conditions who warrant genetic testing. Further studies are needed to evaluate the effectiveness and real-world application of this tool in larger healthcare systems. Nonetheless, the computational phenotyping approach holds promise as a possible clinical decision support that could be integrated into electronic health record systems, enhancing clinical workflows and facilitating actionable genetic diagnoses.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yang D, Zhou D, Cai S, Gan Z, Pencina M, Avillach P, Cai T, Hong C. <a href="/publication/robust-automated-harmonization-heterogeneous-data-through-ensemble-machine-learning">Robust Automated Harmonization of Heterogeneous Data Through Ensemble Machine Learning: Algorithm Development and Validation Study.</a>.</div><div class="csl-right-inline">JMIR medical informatics. 2025;13:e54133. PMID: 39844378</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yang D, Zhou D, Cai S, Gan Z, Pencina M, Avillach P, Cai T, Hong C. <a href="/publication/robust-automated-harmonization-heterogeneous-data-through-ensemble-machine-learning">Robust Automated Harmonization of Heterogeneous Data Through Ensemble Machine Learning: Algorithm Development and Validation Study.</a>.</div><div class="csl-right-inline">JMIR medical informatics. 2025;13:e54133. PMID: 39844378</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-1907437921" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-1907437921">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Cohort studies contain rich clinical data across large and diverse patient populations and are a common source of observational data for clinical research. Because large scale cohort studies are both time and resource intensive, one alternative is to harmonize data from existing cohorts through multicohort studies. However, given differences in variable encoding, accurate variable harmonization is difficult.</p><p>OBJECTIVE: We propose SONAR (Semantic and Distribution-Based Harmonization) as a method for harmonizing variables across cohort studies to facilitate multicohort studies.</p><p>METHODS: SONAR used semantic learning from variable descriptions and distribution learning from study participant data. Our method learned an embedding vector for each variable and used pairwise cosine similarity to score the similarity between variables. This approach was built off 3 National Institutes of Health cohorts, including the Cardiovascular Health Study, the Multi-Ethnic Study of Atherosclerosis, and the Women's Health Initiative. We also used gold standard labels to further refine the embeddings in a supervised manner.</p><p>RESULTS: The method was evaluated using manually curated gold standard labels from the 3 National Institutes of Health cohorts. We evaluated both the intracohort and intercohort variable harmonization performance. The supervised SONAR method outperformed existing benchmark methods for almost all intracohort and intercohort comparisons using area under the curve and top-k accuracy metrics. Notably, SONAR was able to significantly improve harmonization of concepts that were difficult for existing semantic methods to harmonize.</p><p>CONCLUSIONS: SONAR achieves accurate variable harmonization within and between cohort studies by harnessing the complementary strengths of semantic learning and variable distribution learning.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Menezes MCS, Hoffmann AF, Tan ALM, Nalbandyan M, Omenn GS, Mazzotti DR, Hernández-Arango A, Visweswaran S, Venkatesh S, Mandl KD, Bourgeois FT, Lee JWK, Makmur A, Hanauer DA, Semanik MG, Kerivan LT, Hill T, Forero J, Restrepo C, Vigna M, Ceriana P, Abu-El-Rub N, Avillach P, Bellazzi R, Callaci T, Gutiérrez-Sacristán A, Malovini A, Mathew JP, Morris M, Murthy VL, Buonocore TM, Parimbelli E, Patel LP, Saez C, Samayamuthu MJ, Thompson JA, Tibollo V, Xia Z, Kohane IS, Records C for CC of C 19 by EH. <a href="/publication/potential-generative-pre-trained-transformer-4-gpt-4-analyse-medical-notes-three">The potential of Generative Pre-trained Transformer 4 (GPT-4) to analyse medical notes in three different languages: a retrospective model-evaluation study.</a>.</div><div class="csl-right-inline">The Lancet. Digital health. 2025;7(1):e35-e43. PMID: 39722251</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Menezes MCS, Hoffmann AF, Tan ALM, Nalbandyan M, Omenn GS, Mazzotti DR, Hernández-Arango A, Visweswaran S, Venkatesh S, Mandl KD, Bourgeois FT, Lee JWK, Makmur A, Hanauer DA, Semanik MG, Kerivan LT, Hill T, Forero J, Restrepo C, Vigna M, Ceriana P, Abu-El-Rub N, Avillach P, Bellazzi R, Callaci T, Gutiérrez-Sacristán A, Malovini A, Mathew JP, Morris M, Murthy VL, Buonocore TM, Parimbelli E, Patel LP, Saez C, Samayamuthu MJ, Thompson JA, Tibollo V, Xia Z, Kohane IS, Records C for CC of C 19 by EH. <a href="/publication/potential-generative-pre-trained-transformer-4-gpt-4-analyse-medical-notes-three">The potential of Generative Pre-trained Transformer 4 (GPT-4) to analyse medical notes in three different languages: a retrospective model-evaluation study.</a>.</div><div class="csl-right-inline">The Lancet. Digital health. 2025;7(1):e35-e43. PMID: 39722251</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-303159569" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-303159569">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Patient notes contain substantial information but are difficult for computers to analyse due to their unstructured format. Large-language models (LLMs), such as Generative Pre-trained Transformer 4 (GPT-4), have changed our ability to process text, but we do not know how effectively they handle medical notes. We aimed to assess the ability of GPT-4 to answer predefined questions after reading medical notes in three different languages.</p><p>METHODS: For this retrospective model-evaluation study, we included eight university hospitals from four countries (ie, the USA, Colombia, Singapore, and Italy). Each site submitted seven de-identified medical notes related to seven separate patients to the coordinating centre between June 1, 2023, and Feb 28, 2024. Medical notes were written between Feb 1, 2020, and June 1, 2023. One site provided medical notes in Spanish, one site provided notes in Italian, and the remaining six sites provided notes in English. We included admission notes, progress notes, and consultation notes. No discharge summaries were included in this study. We advised participating sites to choose medical notes that, at time of hospital admission, were for patients who were male or female, aged 18-65 years, had a diagnosis of obesity, had a diagnosis of COVID-19, and had submitted an admission note. Adherence to these criteria was optional and participating sites randomly chose which medical notes to submit. When entering information into GPT-4, we prepended each medical note with an instruction prompt and a list of 14 questions that had been chosen a priori. Each medical note was individually given to GPT-4 in its original language and in separate sessions; the questions were always given in English. At each site, two physicians independently validated responses by GPT-4 and responded to all 14 questions. Each pair of physicians evaluated responses from GPT-4 to the seven medical notes from their own site only. Physicians were not masked to responses from GPT-4 before providing their own answers, but were masked to responses from the other physician.</p><p>FINDINGS: We collected 56 medical notes, of which 42 (75%) were in English, seven (13%) were in Italian, and seven (13%) were in Spanish. For each medical note, GPT-4 responded to 14 questions, resulting in 784 responses. In 622 (79%, 95% CI 76-82) of 784 responses, both physicians agreed with GPT-4. In 82 (11%, 8-13) responses, only one physician agreed with GPT-4. In the remaining 80 (10%, 8-13) responses, neither physician agreed with GPT-4. Both physicians agreed with GPT-4 more often for medical notes written in Spanish (86 [88%, 95% CI 79-93] of 98 responses) and Italian (82 [84%, 75-90] of 98 responses) than in English (454 [77%, 74-80] of 588 responses).</p><p>INTERPRETATION: The results of our model-evaluation study suggest that GPT-4 is accurate when analysing medical notes in three different languages. In the future, research should explore how LLMs can be integrated into clinical workflows to maximise their use in health care.</p><p>FUNDING: None.</p></div>
</div>
</div>
</div>
</div>
</div>
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">2024</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gutiérrez-Sacristán A, Makwana S, Dionne A, Mahanta S, Dyer KJ, Serrano F, Watrin C, Pages P, Mousavi S, Degala A, Lyons J, Pillion D, Zachariasse JM, Shekerdemian LS, Truong DT, Newburger JW, Avillach P. <a href="/publication/development-and-validation-open-source-pipeline-automatic-population-case-report-forms">Development and validation of an open-source pipeline for automatic population of case report forms from electronic health records: a pediatric multi-center prospective study.</a>.</div><div class="csl-right-inline">EBioMedicine. 2024;108:105337. PMID: 39288532</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gutiérrez-Sacristán A, Makwana S, Dionne A, Mahanta S, Dyer KJ, Serrano F, Watrin C, Pages P, Mousavi S, Degala A, Lyons J, Pillion D, Zachariasse JM, Shekerdemian LS, Truong DT, Newburger JW, Avillach P. <a href="/publication/development-and-validation-open-source-pipeline-automatic-population-case-report-forms">Development and validation of an open-source pipeline for automatic population of case report forms from electronic health records: a pediatric multi-center prospective study.</a>.</div><div class="csl-right-inline">EBioMedicine. 2024;108:105337. PMID: 39288532</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-613921253" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-613921253">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Clinical trials and registry studies are essential for advancing research and developing novel treatments. However, these studies rely on manual entry of thousands of variables for each patient. Repurposing real-world data can significantly simplify the data collection, reduce transcription errors, and make the data entry process more efficient, consistent, and cost-effective.</p><p>METHODS: We developed an open-source computational pipeline to collect laboratory and medication information from the electronic health record (EHR) data and populate case report forms. The pipeline was developed and validated with data from two independent pediatric hospitals in the US as part of the Long-terM OUtcomes after Multisystem Inflammatory Syndrome In Children (MUSIC) study. Our pipeline allowed the completion of two of the most time-consuming forms. We compared automatically extracted results with manually entered values in one hospital and applied the pipeline to a second hospital, where the output served as the primary data source for case report forms.</p><p>FINDINGS: We extracted and populated 51,845 laboratory and 4913 medication values for 159 patients in two hospitals participating in a prospective pediatric study. We evaluated pipeline performance against data for 104 patients manually entered by clinicians in one of the hospitals. The highest concordance was found during patient hospitalization, with 91.59% of the automatically extracted laboratory and medication values corresponding with the manually entered values. In addition to the successfully populated values, we identified an additional 13,396 laboratory and 567 medication values of interest for the study.</p><p>INTERPRETATION: The automatic data entry of laboratory and medication values during admission is feasible and has a high concordance with the manually entered data. By implementing this proof of concept, we demonstrate the quality of automatic data extraction and highlight the potential of secondary use of EHR data to advance medical science by improving data entry efficiency and expediting clinical research.</p><p>FUNDING: NIH Grant 1OT3HL147154-01, U24HL135691, UG1HL135685.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Wack M, Hassen-Khodja C, McDuffie MT, Bliss G, Horn EJ, Kothari C, McLarney B, Davis R, Hanson K, O’Boyle M, Betancur C, Avillach P. <a href="/publication/phenome-wide-profiling-identifies-genotype-phenotype-associations-phelan-mcdermid">Phenome-wide profiling identifies genotype-phenotype associations in Phelan-McDermid syndrome using family-sourced data from an international registry.</a>.</div><div class="csl-right-inline">Molecular autism. 2024;15(1):40. PMID: 39350236</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Wack M, Hassen-Khodja C, McDuffie MT, Bliss G, Horn EJ, Kothari C, McLarney B, Davis R, Hanson K, O’Boyle M, Betancur C, Avillach P. <a href="/publication/phenome-wide-profiling-identifies-genotype-phenotype-associations-phelan-mcdermid">Phenome-wide profiling identifies genotype-phenotype associations in Phelan-McDermid syndrome using family-sourced data from an international registry.</a>.</div><div class="csl-right-inline">Molecular autism. 2024;15(1):40. PMID: 39350236</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-1645799944" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-1645799944">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Phelan-McDermid syndrome (PMS) is a rare neurodevelopmental disorder caused by 22q13 deletions that include the SHANK3 gene or pathogenic sequence variants in SHANK3. It is characterized by global developmental delay, intellectual disability, speech impairment, autism spectrum disorder, and hypotonia; other variable features include epilepsy, brain and renal malformations, and mild dysmorphic features. Here, we conducted genotype-phenotype correlation analyses using the PMS International Registry, a family-driven registry that compiles clinical data in the form of family-reported outcomes and family-sourced genetic test results.</p><p>METHODS: Data from the registry were harmonized and integrated into the i2b2/tranSMART clinical and genomics data warehouse. We gathered information from 401 individuals with 22q13 deletions including SHANK3 (n = 350, ranging in size from 10 kb to 9.1 Mb) or pathogenic or likely pathogenic SHANK3 sequence variants (n = 51), and used regression models with deletion size as a potential predictor of clinical outcomes for 328 phenotypes.</p><p>RESULTS: Our results showed that increased deletion size was significantly associated with delay in gross and fine motor acquisitions, a spectrum of conditions related to poor muscle tone, renal malformations, mild dysmorphic features (e.g., large fleshy hands, sacral dimple, dysplastic toenails, supernumerary teeth), lymphedema, congenital heart defects, and more frequent neuroimaging abnormalities and infections. These findings indicate that genes upstream of SHANK3 also contribute to some of the manifestations of PMS in individuals with larger deletions. We also showed that self-help skills, verbal ability and a range of psychiatric diagnoses (e.g., autism, ADHD, anxiety disorder) were more common among individuals with smaller deletions and SHANK3 variants.</p><p>LIMITATIONS: Some participants were tested with targeted 22q microarrays rather than genome-wide arrays, and karyotypes were unavailable in many cases, thus precluding the analysis of the effect of other copy number variants or chromosomal rearrangements on the phenotype.</p><p>CONCLUSIONS: This is the largest reported case series of individuals with PMS. Overall, we demonstrate the feasibility of using data from a family-sourced registry to conduct genotype-phenotype analyses in rare genetic disorders. We replicate and strengthen previous findings, and reveal novel associations between larger 22q13 deletions and congenital heart defects, neuroimaging abnormalities and recurrent infections.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hernández-Agosto J, Santiago-Quiñones DI, García-Carmona LM, Gutiérrez-Sacristán A. <a href="/publication/substance-use-disorders-and-disparities-hispanic-and-non-hispanic-older-adults">Substance use disorders and disparities in hispanic and non-hispanic older adults.</a>.</div><div class="csl-right-inline">Drug and alcohol dependence. 2024;261:111357. PMID: 38896947</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hernández-Agosto J, Santiago-Quiñones DI, García-Carmona LM, Gutiérrez-Sacristán A. <a href="/publication/substance-use-disorders-and-disparities-hispanic-and-non-hispanic-older-adults">Substance use disorders and disparities in hispanic and non-hispanic older adults.</a>.</div><div class="csl-right-inline">Drug and alcohol dependence. 2024;261:111357. PMID: 38896947</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2110352893" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-2110352893">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: The global longevity revolution increased the older adult population, posing unique health and economic challenges with implications for healthcare, especially substance use disorders (SUD).</p><p>METHODS: This was a retrospective cohort study of United States older adults, Hispanic and non-Hispanic, who got at least one mental and/or behavioral disorder diagnosis between 2017 and 2021 at age 65 or older. SUD prevalence, prescription frequency changes over time, and comorbidities associated with each medication were compared.</p><p>RESULTS: Electronic health records for 356,133 older adults (110,236 Hispanics and 245,897 non-Hispanics) were analyzed. Notably, 79 % of Hispanics fell below the 100 % federal poverty level, compared to 60 % of non-Hispanics (P&lt;.001). Non-Hispanics also had significantly more average encounters (P=.003) and diagnoses (P&lt;.001). Regression analysis on alcohol-related disorders indicated that the odd ratios of being male (OR=2.93, P&lt;.000), and having low income (OR=1.62, P&lt;.000), increase the odds for this SUD, while being Hispanic and primarily speaking Spanish decreases the odds for all SUDs considered in this study.</p><p>CONCLUSIONS: This cohort study revealed significant disparities related to social determinants of health between Hispanic and non-Hispanic older adults and emphasizes the need for continuous surveillance of older adults as with SUDs. Differences in comorbidity patterns imply distinct risk factors within each population, influenced by demographic-specific elements. Recognizing these variations is essential for tailoring culturally sensitive prevention, intervention, and treatment strategies to each population's unique needs.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Gutierrez A, Network UD, Kobren SN, Avillach P. <a href="/publication/varppud-variant-post-prioritization-developed-undiagnosed-genetic-disorders">VarPPUD: Variant post prioritization developed for undiagnosed genetic disorders.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2024; PMID: 38699371</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Gutierrez A, Network UD, Kobren SN, Avillach P. <a href="/publication/varppud-variant-post-prioritization-developed-undiagnosed-genetic-disorders">VarPPUD: Variant post prioritization developed for undiagnosed genetic disorders.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2024; PMID: 38699371</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-551639828" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-551639828">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>Rare and ultra-rare genetic conditions are estimated to impact nearly 1 in 17 people worldwide, yet accurately pinpointing the diagnostic variants underlying each of these conditions remains a formidable challenge. Because comprehensive,  functional assessment of all possible genetic variants is infeasible, clinicians instead consider  variant pathogenicity predictions to distinguish plausibly disease-causing from benign variants across the genome. However, in the most difficult undiagnosed cases, such as those accepted to the Undiagnosed Diseases Network (UDN), existing pathogenicity predictions cannot reliably discern true etiological variant(s) from other deleterious candidate variants that were prioritized through N-of-1 efforts. Pinpointing the disease-causing variant from a pool of plausible candidates remains a largely manual effort requiring extensive clinical workups, functional and experimental assays, and eventual identification of genotype- and phenotype-matched individuals. Here, we introduce VarPPUD, a tool trained on prioritized variants from UDN cases, that leverages gene-, amino acid-, and nucleotide-level features to discern pathogenic variants from other deleterious variants that are unlikely to be confirmed as disease relevant. VarPPUD achieves a cross-validated accuracy of 79.3% and precision of 77.5% on a held-out subset of uniquely challenging UDN cases, respectively representing an average 18.6% and 23.4% improvement over nine traditional pathogenicity prediction approaches on this task. We validate VarPPUD's ability to discriminate likely from unlikely pathogenic variants on synthetic, GAN-generated candidate variants as well. Finally, we show how VarPPUD can be probed to evaluate each input feature's importance and contribution toward prediction-an essential step toward understanding the distinct characteristics of newly-uncovered disease-causing variants.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hutch MR, Son J, Le TT, Hong C, Wang X, Abad ZSH, Morris M, Gutiérrez-Sacristán A, Klann JG, Spiridou A, Batugo A, Bellazzi R, Benoit V, Bonzel CL, Bryant WA, Chiudinelli L, Cho K, Das P, González TG, Hanauer DA, Henderson DW, Ho YL, Loh NHW, Makoudjou A, Makwana S, Malovini A, Moal B, Mowery DL, Neuraz A, Samayamuthu MJ, Vidorreta FJS, Schriver ER, Schubert P, Talbert J, Tan ALM, Tan BWL, Tan BWQ, Tibollo V, Tippman P, Verdy G, Yuan W, Avillach P, Gehlenborg N, Omenn GS, EHR C for CC of C 19 by, Visweswaran S, Cai T, Luo Y, Xia Z. <a href="/publication/neurological-diagnoses-hospitalized-covid-19-patients-associated-adverse-outcomes">Neurological diagnoses in hospitalized COVID-19 patients associated with adverse outcomes: A multinational cohort study.</a>.</div><div class="csl-right-inline">PLOS digital health. 2024;3(4):e0000484. PMID: 38620037</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hutch MR, Son J, Le TT, Hong C, Wang X, Abad ZSH, Morris M, Gutiérrez-Sacristán A, Klann JG, Spiridou A, Batugo A, Bellazzi R, Benoit V, Bonzel CL, Bryant WA, Chiudinelli L, Cho K, Das P, González TG, Hanauer DA, Henderson DW, Ho YL, Loh NHW, Makoudjou A, Makwana S, Malovini A, Moal B, Mowery DL, Neuraz A, Samayamuthu MJ, Vidorreta FJS, Schriver ER, Schubert P, Talbert J, Tan ALM, Tan BWL, Tan BWQ, Tibollo V, Tippman P, Verdy G, Yuan W, Avillach P, Gehlenborg N, Omenn GS, EHR C for CC of C 19 by, Visweswaran S, Cai T, Luo Y, Xia Z. <a href="/publication/neurological-diagnoses-hospitalized-covid-19-patients-associated-adverse-outcomes">Neurological diagnoses in hospitalized COVID-19 patients associated with adverse outcomes: A multinational cohort study.</a>.</div><div class="csl-right-inline">PLOS digital health. 2024;3(4):e0000484. PMID: 38620037</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2100453899" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-2100453899">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>Few studies examining the patient outcomes of concurrent neurological manifestations during acute COVID-19 leveraged multinational cohorts of adults and children or distinguished between central and peripheral nervous system (CNS vs. PNS) involvement. Using a federated multinational network in which local clinicians and informatics experts curated the electronic health records data, we evaluated the risk of prolonged hospitalization and mortality in hospitalized COVID-19 patients from 21 healthcare systems across 7 countries. For adults, we used a federated learning approach whereby we ran Cox proportional hazard models locally at each healthcare system and performed a meta-analysis on the aggregated results to estimate the overall risk of adverse outcomes across our geographically diverse populations. For children, we reported descriptive statistics separately due to their low frequency of neurological involvement and poor outcomes. Among the 106,229 hospitalized COVID-19 patients (104,031 patients ≥18 years; 2,198 patients &lt;18 years, January 2020-October 2021), 15,101 (14%) had at least one CNS diagnosis, while 2,788 (3%) had at least one PNS diagnosis. After controlling for demographics and pre-existing conditions, adults with CNS involvement had longer hospital stay (11 versus 6 days) and greater risk of (Hazard Ratio = 1.78) and faster time to death (12 versus 24 days) than patients with no neurological condition (NNC) during acute COVID-19 hospitalization. Adults with PNS involvement also had longer hospital stay but lower risk of mortality than the NNC group. Although children had a low frequency of neurological involvement during COVID-19 hospitalization, a substantially higher proportion of children with CNS involvement died compared to those with NNC (6% vs 1%). Overall, patients with concurrent CNS manifestation during acute COVID-19 hospitalization faced greater risks for adverse clinical outcomes than patients without any neurological diagnosis. Our global informatics framework using a federated approach (versus a centralized data collection approach) has utility for clinical discovery beyond COVID-19.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Huang M, Gutiérrez-Sacristán A, Janiak E, Young K, Starosta A, Blanton K, Azhir A, Goldfarb CN, Kuperwasser F, Schaefer KM, Stoddard RE, Vatsa R, Merz-Herrala AA, Bartz D. <a href="/publication/contraceptive-content-shared-social-media-analysis-twitter">Contraceptive content shared on social media: an analysis of Twitter.</a>.</div><div class="csl-right-inline">Contraception and reproductive medicine. 2024;9(1):5. PMID: 38321582</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Huang M, Gutiérrez-Sacristán A, Janiak E, Young K, Starosta A, Blanton K, Azhir A, Goldfarb CN, Kuperwasser F, Schaefer KM, Stoddard RE, Vatsa R, Merz-Herrala AA, Bartz D. <a href="/publication/contraceptive-content-shared-social-media-analysis-twitter">Contraceptive content shared on social media: an analysis of Twitter.</a>.</div><div class="csl-right-inline">Contraception and reproductive medicine. 2024;9(1):5. PMID: 38321582</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-1649283050" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-1649283050">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Information on social media may affect peoples' contraceptive decision making. We performed an exploratory analysis of contraceptive content on Twitter (recently renamed X), a popular social media platform.</p><p>METHODS: We selected a random subset of 1% of publicly available, English-language tweets related to reversible, prescription contraceptive methods posted between January 2014 and December 2019. We oversampled tweets for the contraceptive patch to ensure at least 200 tweets per method. To create the codebook, we identified common themes specific to tweet content topics, tweet sources, and tweets soliciting information or providing advice. All posts were coded by two team members, and differences were adjudicated by a third reviewer. Descriptive analyses were reported with accompanying qualitative findings.</p><p>RESULTS: During the study period, 457,369 tweets about reversible contraceptive methods were published, with a random sample of 4,434 tweets used for final analysis. Tweets most frequently discussed contraceptive method decision-making (26.7%) and side effects (20.5%), particularly for long-acting reversible contraceptive methods and the depot medroxyprogesterone acetate shot. Tweets about logistics of use or adherence were common for short-acting reversible contraceptives. Tweets were frequently posted by contraceptive consumers (50.6%). A small proportion of tweets explicitly requested information (6.2%) or provided advice (4.2%).</p><p>CONCLUSIONS: Clinicians should be aware that individuals are exposed to information through Twitter that may affect contraceptive perceptions and decision making, particularly regarding long-acting reversible contraceptives. Social media is a valuable source for studying contraceptive beliefs missing in traditional health research and may be used by professionals to disseminate accurate contraceptive information.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gargano MA, Matentzoglu N, Coleman B, Addo-Lartey EB, Anagnostopoulos A V, Anderton J, Avillach P, Bagley AM, Bakštein E, Balhoff JP, Baynam G, Bello SM, Berk M, Bertram H, Bishop S, Blau H, Bodenstein DF, Botas P, Boztug K, Čady J, Callahan TJ, Cameron R, Carbon SJ, Castellanos F, Caufield H, Chan LE, Chute CG, Cruz-Rojo J, Dahan-Oliel N, Davids JR, de Dieuleveult M, de Souza V, de Vries BBA, de Vries E, DePaulo R, Derfalvi B, Dhombres F, Diaz-Byrd C, Dingemans AJM, Donadille B, Duyzend M, Elfeky R, Essaid S, Fabrizzi C, Fico G, Firth H V, Freudenberg-Hua Y, Fullerton JM, Gabriel DL, Gilmour K, Giordano J, Goes FS, Moses RG, Green I, Griese M, Groza T, Gu W, Guthrie J, Gyori B, Hamosh A, Hanauer M, Hanušová K, He YO, Hegde H, Helbig I, Holasová K, Hoyt CT, Huang S, Hurwitz E, Jacobsen JOB, Jiang X, Joseph L, Keramatian K, King B, Knoflach K, Koolen DA, Kraus ML, Kroll C, Kusters M, Ladewig MS, Lagorce D, Lai MC, Lapunzina P, Laraway B, Lewis-Smith D, Li X, Lucano C, Majd M, Marazita ML, Martinez-Glez V, McHenry TH, McInnis MG, McMurry JA, Mihulová M, Millett CE, Mitchell PB, Moslerová V, Narutomi K, Nematollahi S, Nevado J, Nierenberg AA, Čajbiková NN, Nurnberger JI, Ogishima S, Olson D, Ortiz A, Pachajoa H, de Nanclares GP, Peters A, Putman T, Rapp CK, Rath A, Reese J, Rekerle L, Roberts AM, Roy S, Sanders SJ, Schuetz C, Schulte EC, Schulze TG, Schwarz M, Scott K, Seelow D, Seitz B, Shen Y, Similuk MN, Simon ES, Singh B, Smedley D, Smith CL, Smolinsky JT, Sperry S, Stafford E, Stefancsik R, Steinhaus R, Strawbridge R, Sundaramurthi JC, Talapova P, Castano JAT, Tesner P, Thomas RH, Thurm A, Turnovec M, van Gijn ME, Vasilevsky NA, Vlčková M, Walden A, Wang K, Wapner R, Ware JS, Wiafe AA, Wiafe SA, Wiggins LD, Williams AE, Wu C, Wyrwoll MJ, Xiong H, Yalin N, Yamamoto Y, Yatham LN, Yocum AK, Young AH, Yüksel Z, Zandi PP, Zankl A, Zarante I, Zvolský M, Toro S, Carmody LC, Harris NL, Munoz-Torres MC, Danis D, Mungall CJ, Köhler S, Haendel MA, Robinson PN. <a href="/publication/human-phenotype-ontology-2024-phenotypes-around-world">The Human Phenotype Ontology in 2024: phenotypes around the world.</a>.</div><div class="csl-right-inline">Nucleic acids research. 2024;52(D1):D1333-D1346. PMID: 37953324</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gargano MA, Matentzoglu N, Coleman B, Addo-Lartey EB, Anagnostopoulos A V, Anderton J, Avillach P, Bagley AM, Bakštein E, Balhoff JP, Baynam G, Bello SM, Berk M, Bertram H, Bishop S, Blau H, Bodenstein DF, Botas P, Boztug K, Čady J, Callahan TJ, Cameron R, Carbon SJ, Castellanos F, Caufield H, Chan LE, Chute CG, Cruz-Rojo J, Dahan-Oliel N, Davids JR, de Dieuleveult M, de Souza V, de Vries BBA, de Vries E, DePaulo R, Derfalvi B, Dhombres F, Diaz-Byrd C, Dingemans AJM, Donadille B, Duyzend M, Elfeky R, Essaid S, Fabrizzi C, Fico G, Firth H V, Freudenberg-Hua Y, Fullerton JM, Gabriel DL, Gilmour K, Giordano J, Goes FS, Moses RG, Green I, Griese M, Groza T, Gu W, Guthrie J, Gyori B, Hamosh A, Hanauer M, Hanušová K, He YO, Hegde H, Helbig I, Holasová K, Hoyt CT, Huang S, Hurwitz E, Jacobsen JOB, Jiang X, Joseph L, Keramatian K, King B, Knoflach K, Koolen DA, Kraus ML, Kroll C, Kusters M, Ladewig MS, Lagorce D, Lai MC, Lapunzina P, Laraway B, Lewis-Smith D, Li X, Lucano C, Majd M, Marazita ML, Martinez-Glez V, McHenry TH, McInnis MG, McMurry JA, Mihulová M, Millett CE, Mitchell PB, Moslerová V, Narutomi K, Nematollahi S, Nevado J, Nierenberg AA, Čajbiková NN, Nurnberger JI, Ogishima S, Olson D, Ortiz A, Pachajoa H, de Nanclares GP, Peters A, Putman T, Rapp CK, Rath A, Reese J, Rekerle L, Roberts AM, Roy S, Sanders SJ, Schuetz C, Schulte EC, Schulze TG, Schwarz M, Scott K, Seelow D, Seitz B, Shen Y, Similuk MN, Simon ES, Singh B, Smedley D, Smith CL, Smolinsky JT, Sperry S, Stafford E, Stefancsik R, Steinhaus R, Strawbridge R, Sundaramurthi JC, Talapova P, Castano JAT, Tesner P, Thomas RH, Thurm A, Turnovec M, van Gijn ME, Vasilevsky NA, Vlčková M, Walden A, Wang K, Wapner R, Ware JS, Wiafe AA, Wiafe SA, Wiggins LD, Williams AE, Wu C, Wyrwoll MJ, Xiong H, Yalin N, Yamamoto Y, Yatham LN, Yocum AK, Young AH, Yüksel Z, Zandi PP, Zankl A, Zarante I, Zvolský M, Toro S, Carmody LC, Harris NL, Munoz-Torres MC, Danis D, Mungall CJ, Köhler S, Haendel MA, Robinson PN. <a href="/publication/human-phenotype-ontology-2024-phenotypes-around-world">The Human Phenotype Ontology in 2024: phenotypes around the world.</a>.</div><div class="csl-right-inline">Nucleic acids research. 2024;52(D1):D1333-D1346. PMID: 37953324</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2011850567" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-2011850567">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>The Human Phenotype Ontology (HPO) is a widely used resource that comprehensively organizes and defines the phenotypic features of human disease, enabling computational inference and supporting genomic and phenotypic analyses through semantic similarity and machine learning algorithms. The HPO has widespread applications in clinical diagnostics and translational research, including genomic diagnostics, gene-disease discovery, and cohort analytics. In recent years, groups around the world have developed translations of the HPO from English to other languages, and the HPO browser has been internationalized, allowing users to view HPO term labels and in many cases synonyms and definitions in ten languages in addition to English. Since our last report, a total of 2239 new HPO terms and 49235 new HPO annotations were developed, many in collaboration with external groups in the fields of psychiatry, arthrogryposis, immunology and cardiology. The Medical Action Ontology (MAxO) is a new effort to model treatments and other measures taken for clinical management. Finally, the HPO consortium is contributing to efforts to integrate the HPO and the GA4GH Phenopacket Schema into electronic health records (EHRs) with the goal of more standardized and computable integration of rare disease data in EHRs.</p></div>
</div>
</div>
</div>
</div>
</div>
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">2023</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Sperotto F, Gutiérrez-Sacristán A, Makwana S, Li X, Rofeberg VN, Cai T, Bourgeois FT, Omenn GS, Hanauer DA, Saez C, Bonzel CL, Bucholz E, Dionne A, Elias MD, García-Barrio N, González TG, Issitt RW, Kernan KF, Laird-Gion J, Maidlow SE, Mandl KD, Ahooyi TM, Moraleda C, Morris M, Moshal KL, Pedrera-Jiménez M, Shah MA, South AM, Spiridou A, Taylor DM, Verdy G, Visweswaran S, Wang X, Xia Z, Zachariasse JM, EHR C for CC of C 19 by, Newburger JW, Avillach P. <a href="/publication/clinical-phenotypes-and-outcomes-children-multisystem-inflammatory-syndrome-across-0">Clinical phenotypes and outcomes in children with multisystem inflammatory syndrome across SARS-CoV-2 variant eras: a multinational study from the 4CE consortium.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102212. PMID: 37745025</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Sperotto F, Gutiérrez-Sacristán A, Makwana S, Li X, Rofeberg VN, Cai T, Bourgeois FT, Omenn GS, Hanauer DA, Saez C, Bonzel CL, Bucholz E, Dionne A, Elias MD, García-Barrio N, González TG, Issitt RW, Kernan KF, Laird-Gion J, Maidlow SE, Mandl KD, Ahooyi TM, Moraleda C, Morris M, Moshal KL, Pedrera-Jiménez M, Shah MA, South AM, Spiridou A, Taylor DM, Verdy G, Visweswaran S, Wang X, Xia Z, Zachariasse JM, EHR C for CC of C 19 by, Newburger JW, Avillach P. <a href="/publication/clinical-phenotypes-and-outcomes-children-multisystem-inflammatory-syndrome-across-0">Clinical phenotypes and outcomes in children with multisystem inflammatory syndrome across SARS-CoV-2 variant eras: a multinational study from the 4CE consortium.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102212. PMID: 37745025</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-87103762" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-87103762">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Multisystem inflammatory syndrome in children (MIS-C) is a severe complication of SARS-CoV-2 infection. It remains unclear how MIS-C phenotypes vary across SARS-CoV-2 variants. We aimed to investigate clinical characteristics and outcomes of MIS-C across SARS-CoV-2 eras.</p><p>METHODS: We performed a multicentre observational retrospective study including seven paediatric hospitals in four countries (France, Spain, U.K., and U.S.). All consecutive confirmed patients with MIS-C hospitalised between February 1st, 2020, and May 31st, 2022, were included. Electronic Health Records (EHR) data were used to calculate pooled risk differences (RD) and effect sizes (ES) at site level, using  as reference. Meta-analysis was used to pool data across sites.</p><p>FINDINGS: Of 598 patients with MIS-C (61% male, 39% female; mean age 9.7 years [SD 4.5]), 383 (64%) were admitted in the  era, 111 (19%) in the  era, and 104 (17%) in the  era. Compared with patients admitted in the  era, those admitted in the  era were younger (ES -1.18 years [95% CI -2.05, -0.32]), had fewer respiratory symptoms (RD -0.15 [95% CI -0.33, -0.04]), less frequent non-cardiogenic shock or systemic inflammatory response syndrome (SIRS) (RD -0.35 [95% CI -0.64, -0.07]), lower lymphocyte count (ES -0.16 × 10/uL [95% CI -0.30, -0.01]), lower C-reactive protein (ES -28.5 mg/L [95% CI -46.3, -10.7]), and lower troponin (ES -0.14 ng/mL [95% CI -0.26, -0.03]). Patients admitted in the  versus  eras were younger (ES -1.6 years [95% CI -2.5, -0.8]), had less frequent SIRS (RD -0.18 [95% CI -0.30, -0.05]), lower lymphocyte count (ES -0.39 × 10/uL [95% CI -0.52, -0.25]), lower troponin (ES -0.16 ng/mL [95% CI -0.30, -0.01]) and less frequently received anticoagulation therapy (RD -0.19 [95% CI -0.37, -0.04]). Length of hospitalization was shorter in the  versus  eras (-1.3 days [95% CI -2.3, -0.4]).</p><p>INTERPRETATION: Our study suggested that MIS-C clinical phenotypes varied across SARS-CoV-2 eras, with patients in  and  eras being younger and less sick. EHR data can be effectively leveraged to identify rare complications of pandemic diseases and their variation over time.</p><p>FUNDING: None.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Dagliati A, Strasser ZH, Abad ZSH, Klann JG, Wagholikar KB, Mesa R, Visweswaran S, Morris M, Luo Y, Henderson DW, Samayamuthu MJ, Tan BWQ, Verdy G, Omenn GS, Xia Z, Bellazzi R, EHR C for CC of C 19 by, Murphy SN, Holmes JH, Estiri H, EHR C for CC of C 19 by. <a href="/publication/characterization-long-covid-temporal-sub-phenotypes-distributed-representation-0">Characterization of long COVID temporal sub-phenotypes by distributed representation learning from electronic health record data: a cohort study.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102210. PMID: 37745021</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Dagliati A, Strasser ZH, Abad ZSH, Klann JG, Wagholikar KB, Mesa R, Visweswaran S, Morris M, Luo Y, Henderson DW, Samayamuthu MJ, Tan BWQ, Verdy G, Omenn GS, Xia Z, Bellazzi R, EHR C for CC of C 19 by, Murphy SN, Holmes JH, Estiri H, EHR C for CC of C 19 by. <a href="/publication/characterization-long-covid-temporal-sub-phenotypes-distributed-representation-0">Characterization of long COVID temporal sub-phenotypes by distributed representation learning from electronic health record data: a cohort study.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102210. PMID: 37745021</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-250295374" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-250295374">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Characterizing Post-Acute Sequelae of COVID (SARS-CoV-2 Infection), or  has been challenging due to the multitude of sub-phenotypes, temporal attributes, and definitions. Scalable characterization of PASC sub-phenotypes can enhance screening capacities, disease management, and treatment planning.</p><p>METHODS: We conducted a retrospective multi-centre observational cohort study, leveraging longitudinal electronic health record (EHR) data of 30,422 patients from three healthcare systems in the Consortium for the Clinical Characterization of COVID-19 by EHR (4CE). From the total cohort, we applied a deductive approach on 12,424 individuals with follow-up data and developed a distributed representation learning process for providing augmented definitions for PASC sub-phenotypes.</p><p>FINDINGS: Our framework characterized seven PASC sub-phenotypes. We estimated that on average 15.7% of the hospitalized COVID-19 patients were likely to suffer from at least one PASC symptom and almost 5.98%, on average, had multiple symptoms. Joint pain and dyspnea had the highest prevalence, with an average prevalence of 5.45% and 4.53%, respectively.</p><p>INTERPRETATION: We provided a scalable framework to every participating healthcare system for estimating PASC sub-phenotypes prevalence and temporal attributes, thus developing a unified model that characterizes augmented sub-phenotypes across the different systems.</p><p>FUNDING: Authors are supported by National Institute of Allergy and Infectious Diseases, National Institute on Aging, National Center for Advancing Translational Sciences, National Medical Research Council, National Institute of Neurological Disorders and Stroke, European Union, National Institutes of Health, National Center for Advancing Translational Sciences.</p></div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<nav aria-label="Pagination" class="hwp-pager hwp-pager--theme-light hwp-bg-light-core" role="navigation">
<ul class="hwp-pager__items js-pager__items">
<li>
<div class="hwp-pager__item hwp-pager__item--previous hwp-pager__item--not-active" href="" rel="prev" title="Go to previous page">
<span class="hwp-visually-hidden">Previous page</span>
<span aria-hidden="true" class="material-icon material-icon--chevron-left">chevron_left</span>
</div>
</li>
<li class="hwp-pager__item hwp-pager__item--is-active">
<a aria-current="page" href="?page=0" title="Current page">1</a>
</li>
<li class="hwp-pager__item">
<a href="?page=1" title="Go to page 2">2</a>
</li>
<li class="hwp-pager__item">
<a href="?page=2" title="Go to page 3">3</a>
</li>
<li class="hwp-pager__item hwp-pager__item--last hwp-pager__item--ellipsis">
<a href="?page=8" title="Go to last page">
<span class="hwp-visually-hidden">Last page</span>
<span aria-hidden="true">9</span>
</a>
</li>
<li>
<a class="hwp-pager__item hwp-pager__item--next" href="?page=1" rel="prev" title="Go to next page">
<span class="hwp-visually-hidden">Next page</span>
<span aria-hidden="true" class="material-icon material-icon--chevron-right">chevron_right</span>
</a>
</li>
</ul>
</nav>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/research -->
<h1>Research & Collaborations | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/research'>avillach-lab.hms.harvard.edu/research</a></div>
<main aria-label="Research &amp; Collaborations" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Research &amp; Collaborations</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="field field--name-field-hwp-additional-content field--type-entity-reference-revisions field--label-hidden">
<div class="paragraphs-item__first paragraphs-item">
<div class="hwp-page-list hwp-page-list--list hwp-page-list--grid-3 hwp-bg-light-core">
<div class="field field--name-field-hwp-section-heading field--type-entity-reference-revisions field--label-hidden">
<div class="hwp-section-heading" data-component-name="section-heading">
<div class="hwp-section-heading__container hwp-container-px hwp-container">
<h2 class="hwp-section-heading__title text-medium">
      Current Projects
    </h2>
<div class="hwp-section-heading__description hwp-text-block">
<p>Ongoing collaborations and research projects of the Avillach Lab. </p>
</div>
</div>
</div>
</div>
<div class="hwp-page-list__container hwp-container-px hwp-container">
<div class="hwp-flex hwp-flex-col hwp-gap-32">
<div id="hwp-node-list-results">
<div class="hwp-page-list__items">
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/nhlbi-biodatacatalyst"> <h3 class="page-card__heading hwp-order-2">NHLBI BioData Catalyst</h3>
</a>
</div>
</article>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="paragraphs-item__last paragraphs-item">
<div class="hwp-page-list hwp-page-list--list hwp-page-list--grid-3 hwp-bg-light-core">
<div class="field field--name-field-hwp-section-heading field--type-entity-reference-revisions field--label-hidden">
<div class="hwp-section-heading" data-component-name="section-heading">
<div class="hwp-section-heading__container hwp-container-px hwp-container">
<h2 class="hwp-section-heading__title text-medium">
      Past Projects
    </h2>
</div>
</div>
</div>
<div class="hwp-page-list__container hwp-container-px hwp-container">
<div class="hwp-flex hwp-flex-col hwp-gap-32">
<div id="hwp-node-list-results--2">
<div class="hwp-page-list__items">
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/2016-2-presentationvideo-nih-bd2k-pic-sure-supplements-1-ncats-grdr-2-pittsburg-ccd-3"> <h3 class="page-card__heading hwp-order-2">2016-02 PRESENTATION/VIDEO - NIH BD2K PIC-SURE Supplements 1) NCATS GRDR, 2) Pittsburg CCD &amp; 3) Count Everything</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/autism-pilot"> <h3 class="page-card__heading hwp-order-2">Autism Pilot</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/bch-precision-link-genomics-research-and-innovation-network-grin"> <h3 class="page-card__heading hwp-order-2">BCH Precision Link - Genomics Research and Innovation Network (GRIN)</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/center-causal-modeling-and-discovery-biomedical-knowledge-big-data-ccd"> <h3 class="page-card__heading hwp-order-2">Center for Causal Modeling and Discovery of Biomedical Knowledge from Big Data (CCD)</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/center-causal-modeling-and-discovery-biomedical-knowledge-big-data-ccd-phase-ii"> <h3 class="page-card__heading hwp-order-2">Center for Causal Modeling and Discovery of Biomedical Knowledge from Big Data (CCD) - Phase II</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/count-everything"> <h3 class="page-card__heading hwp-order-2">Count Everything</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/data-fusion-self-scaling-open-source-registry-advancing-pediatric-pulmonary-vascular"> <h3 class="page-card__heading hwp-order-2">Data Fusion - A Self-Scaling, Open Source Registry Advancing Pediatric Pulmonary Vascular Disease Research</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/delineating-autism-subtypes-phenotype-wide-scan-across-genome-wide-genotypes-patient"> <h3 class="page-card__heading hwp-order-2">Delineating autism subtypes by phenotype-wide scan across genome-wide genotypes in a patient-centric information commons</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/electronic-health-records-clinical-research-ehr4cr"> <h3 class="page-card__heading hwp-order-2">Electronic Health Records for Clinical Research (EHR4CR)</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/emif-european-medical-information-framework-0"> <h3 class="page-card__heading hwp-order-2">EMIF: European Medical Information Framework</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/evaluating-scalability-packaged-open-source-biomedical-informatics-applications-across-1"> <h3 class="page-card__heading hwp-order-2">Evaluating the scalability of packaged open source biomedical informatics applications across cloud based providers</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/evaluating-scalability-packaged-open-source-biomedical-informatics-applications-across-2"> <h3 class="page-card__heading hwp-order-2">Evaluating the scalability of packaged open source biomedical informatics applications across cloud based providers - Phase II</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/exploring-and-understanding-adverse-drug-reactions-integrative-mining-clinical-records"> <h3 class="page-card__heading hwp-order-2">Exploring and Understanding Adverse Drug Reactions by Integrative Mining of Clinical Records and Biomedical Knowledge</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/i2b2-clinical-data-warehouse-cdw-implementation-h%C3%B4pital-europ%C3%A9en-georges-pompidou-aphp"> <h3 class="page-card__heading hwp-order-2">I2b2 Clinical Data Warehouse (CDW) implementation at Hôpital Européen Georges Pompidou - APHP, Paris, France</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/i2b2transmart-pilot-study-cloud-using-redshift-phase-i"> <h3 class="page-card__heading hwp-order-2">i2b2/tranSMART pilot study on the cloud using Redshift - Phase I</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/i2b2transmart-pilot-study-cloud-using-redshift-phase-ii"> <h3 class="page-card__heading hwp-order-2">i2b2/tranSMART pilot study on the cloud using Redshift - Phase II</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/interagency-personnel-agreement-veterans-administration"> <h3 class="page-card__heading hwp-order-2">Interagency Personnel Agreement with Veteran's Administration</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/neuropsychiatric-genome-scale-and-rdoc-individualized-domains-n-grid-0"> <h3 class="page-card__heading hwp-order-2">Neuropsychiatric Genome-Scale and RDoC Individualized Domains (N-GRID)</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/nhlbi-cure-sickle-cell-initiative"> <h3 class="page-card__heading hwp-order-2">NHLBI Cure Sickle Cell Initiative </h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/nhlbinih-data-fusion-self-scaling-open-source-registry-advancing-pediatric-pulmonary-vascular"> <h3 class="page-card__heading hwp-order-2">NHLBI/NIH Data Fusion - A Self-Scaling, Open Source Registry Advancing Pediatric Pulmonary Vascular Disease Research</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/nihncats-global-rare-diseases-patient-registry-data-repository-grdr%C2%AE"> <h3 class="page-card__heading hwp-order-2">NIH/NCATS Global Rare Diseases Patient Registry Data Repository (GRDR®)</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/nihncats-global-rare-diseases-patient-registry-i2b2transmart-data-repository"> <h3 class="page-card__heading hwp-order-2">NIH/NCATS Global Rare Diseases Patient Registry i2b2/tranSMART Data Repository</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/nihncats-global-rare-diseases-patient-registry-i2b2transmart-data-repository-0"> <h3 class="page-card__heading hwp-order-2">NIH/NCATS Global Rare Diseases Patient Registry i2b2/tranSMART Data Repository</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/patient-centered-information-commons-0"> <h3 class="page-card__heading hwp-order-2">Patient-Centered Information Commons</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/pcori-cdrn-scalable-collaborative-infrastructure-learning-healthcare-system-scilhs"> <h3 class="page-card__heading hwp-order-2">PCORI CDRN: Scalable Collaborative Infrastructure for a Learning Healthcare System (SCILHS) - Phase II</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/pcori-pprn-phelan-mcdermid-syndrome-data-network-pmsdn-phase-ii"> <h3 class="page-card__heading hwp-order-2">PCORI PPRN - Phelan McDermid Syndrome Data Network (PMS_DN) Phase II</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/pcori-pprn-phelan-mcdermid-syndrome-data-network"> <h3 class="page-card__heading hwp-order-2">PCORI PPRN - Phelan-McDermid Syndrome Data Network</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/phelan-mcdermid-syndrome-data-network-pmsdn-phase-i"> <h3 class="page-card__heading hwp-order-2">Phelan-McDermid Syndrome Data Network (PMS_DN) - Phase I</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/randomized-control-study-impact-computerized-prescription-entry-drug-prescription"> <h3 class="page-card__heading hwp-order-2">Randomized control study on the impact of computerized prescription entry on drug prescription by undergraduate medical students</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/scalable-collaborative-infrastructure-learning-healthcare-system-scilhs-phase-ii"> <h3 class="page-card__heading hwp-order-2">Scalable Collaborative Infrastructure for a Learning Healthcare System (SCILHS) - Phase II</h3>
</a>
</div>
</article>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/people/paul-avillach-md-phd -->
<h1>Paul Avillach, MD, PhD | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/people/paul-avillach-md-phd'>avillach-lab.hms.harvard.edu/people/paul-avillach-md-phd</a></div>
<main aria-label="Paul Avillach, MD, PhD" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="page-content-top" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Paul Avillach, MD, PhD
    </h1>
<div class="hwp-page-header__subtitle">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Assistant Professor at Harvard Medical School</div>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 lg:hwp-pt-64 lg:hwp-flex lg:hwp-gap-24">
<div class="lg:hwp-w-[19.375rem]">
<div class="hwp-person-card" data-component-name="person-card">
<div class="hwp-person-card__image">
<div class="field field--name-field-hwp-person-headshot field--type-entity-reference field--label-hidden"><div class="hwp-media hwp-media--4-5-small-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/paul_avillach_h.jpg?itok=DT9lYtlP 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/paul_avillach_h.jpg?itok=ggQZ85Au 2x" type="image/jpeg" width="480"/>
<img alt="Dr. Paul Avillach" height="600" loading="lazy" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/paul_avillach_h.jpg?itok=DT9lYtlP" width="480"/>
</picture>
</div>
</div>
</div>
</div>
<div class="hwp-person-card__information hwp-bg-light-base hwp-flex-1"> <div class="hwp-flex hwp-gap-8 hwp-items-center">
<span aria-hidden="true" class="material-icon">email</span>
<a class="hwp-break-all" href="mailto:paul_avillach@hms.harvard.edu">paul_avillach@hms.harvard.edu</a>
</div>
</div>
</div>
</div>
<div class="hwp-flex-1 lg:hwp-px-32">
<div class="hwp-person-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>Dr. Avillach is Assistant Professor in Pediatrics &amp; Biomedical Informatics, Harvard Medical School and Boston Children’s Hospital. He received his MD in Public Health from University of Bordeaux, France and PhD in Biomedical Informatics from University of Marseilles, France. His work focuses on translational bioinformatics, combining clinical and genomic data across different scales and resolutions to address essential biomedical questions.</p><p><a href="/"><strong>http://avillach-lab.hms.harvard.edu</strong></a><strong>  </strong><br/>Department of Biomedical Informatics | Harvard Medical School  <a href="https://dbmi.hms.harvard.edu/">https://dbmi.hms.harvard.edu</a>  <br/>Children’s Hospital Informatics Program  |  Boston Children’s Hospital  <a href="http://chip.org/">http://chip.org</a>  </p></div>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 md:hwp-pt-64">
<div class="horizontal-rule hwp-flex hwp-w-full hwp-justify-center">
<hr class="hwp-border-t-3 hwp-w-1/4 hwp-text-accent-dark-regular hwp-my-24 md:hwp-my-32 xl:hwp-my-64 hwp-pt-1 hwp-rounded-5"/>
</div>
</div>
<div class="hwp-container hwp-container-px">
</div>
<div class="hwp-taxonomy-group hwp-bg-light-core">
<div class="hwp-container hwp-container-px">
<ul class="hwp-taxonomy-group__grid">
<li class="hwp-taxonomy-group-card hwp-bg-light-base">
<h2 class="hwp-taxonomy-group-card__title text-x-small">People</h2>
<a href="/people/pi" hreflang="en">Principal Investigator</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/posters -->
<h1>Posters | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/posters'>avillach-lab.hms.harvard.edu/posters</a></div>
<main aria-label="Posters" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Posters</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="field field--name-field-hwp-additional-content field--type-entity-reference-revisions field--label-hidden">
<div class="paragraphs-item__first paragraphs-item">
<div class="hwp-tabs hwp-tabs--container- hwp-pb-32 md:hwp-pb-64 hwp-bg-light-core hwp-pt-32 md:hwp-pt-64">
<div class="field field--name-field-hwp-section-heading field--type-entity-reference-revisions field--label-hidden">
</div>
<div class="hwp-container-px hwp-container">
<div class="hwp-tabs__buttons js-hwp-tabs__buttons" data-primary-color="light-base" data-secondary-color="light-core" role="tablist">
<button aria-controls="tabpanel-98e72437-69cb-4df1-b6c9-05248c34526f-0" aria-selected="true" class="hwp-tabs__button hwp-bg-light-base" id="tab-98e72437-69cb-4df1-b6c9-05248c34526f-0" role="tab" type="button">2022</button>
<button aria-controls="tabpanel-98e72437-69cb-4df1-b6c9-05248c34526f-1" aria-selected="false" class="hwp-tabs__button hwp-bg-light-core" id="tab-98e72437-69cb-4df1-b6c9-05248c34526f-1" role="tab" tabindex="-1" type="button">2023</button>
<button aria-controls="tabpanel-98e72437-69cb-4df1-b6c9-05248c34526f-2" aria-selected="false" class="hwp-tabs__button hwp-bg-light-core" id="tab-98e72437-69cb-4df1-b6c9-05248c34526f-2" role="tab" tabindex="-1" type="button">2024</button>
</div>
<div aria-labelledby="98e72437-69cb-4df1-b6c9-05248c34526f-0" class="hwp-tabs__panel hwp-bg-light-base" id="tabpanel-98e72437-69cb-4df1-b6c9-05248c34526f-0" role="tabpanel" tabindex="0">
<div class="hwp-tabs__panel-intro">
<h2 class="h2 hwp-tabs__panel-title">2022</h2>
<div class="hwp-text-block">
</div>
</div>
<div class="hwp-tabs__panel-widgets">
<div class="field field--name-field-hwp-items field--type-entity-reference-revisions field--label-hidden">
<div>
<div class="hwp-accordions js-hwp-accordions hwp-accordions-theme-light-core hwp-bg-light-core hwp-pt-32 md:hwp-pt-64">
<div class="hwp-container hwp-container-px">
<div class="hwp-flex hwp-flex-col hwp-gap-32">
<div class="hwp-accordions__controls js-hwp-accordions__controls__trigger hwp-flex hwp-py-16">
<button class="expand-all hwp-mr-16" data-live-message="All sections have been opened.">
            Open all sections
          </button>
<button class="collapse-all" data-live-message="All sections have been closed.">
            Close all sections
          </button>
</div>
<div aria-live="polite" class="hwp-sr-only hwp-accordions__announce"></div>
<div class="hwp-accordions__items">
<div class="hwp-accordion hwp-bg-light-base" data-component-name="accordion">
<h3 class="h4">
<button aria-controls="hwp-accordion-36929429" aria-expanded="false" class="hwp-accordion__button js-hwp-accordion__trigger hwp-accordion__button-auto-close" type="button">
<span class="title">
          Medication Use in the Management of Comorbidities Among Individuals With Autism Spectrum Disorder From a Large Nationwide Insurance Database
        </span>
<span aria-hidden="true" class="material-icon !hwp-block">expand_more</span>
</button>
</h3>
<div class="hwp-accordion__content hwp-text-block" hidden="" id="hwp-accordion-36929429">
<div class="hwp-media hwp-media--full-width">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <img alt="Poster for Medication Use in the Management of Comorbidities Among Individuals With Autism Spectrum Disorder From a Large Nationwide Insurance Database" height="2579" loading="lazy" src="/sites/g/files/omnuum4001/files/2025-03/medication_ASD.png" width="3429"/>
</div>
</div>
</div>
</div>
<div class="hwp-accordion hwp-bg-light-base" data-component-name="accordion">
<h3 class="h4">
<button aria-controls="hwp-accordion-621070423" aria-expanded="false" class="hwp-accordion__button js-hwp-accordion__trigger hwp-accordion__button-auto-close" type="button">
<span class="title">
          Service Workbench: an Amazon Web Services platform enables easy access to cloud-based computing for teaching and research
        </span>
<span aria-hidden="true" class="material-icon !hwp-block">expand_more</span>
</button>
</h3>
<div class="hwp-accordion__content hwp-text-block" hidden="" id="hwp-accordion-621070423">
<div class="hwp-media hwp-media--full-width">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <img alt="Service Workbench: an Amazon Web Services platform enables easy access to cloud-based computing for teaching and research" height="2592" loading="lazy" src="/sites/g/files/omnuum4001/files/2025-03/SWB%20poster%20for%20science%20day%202022_0.png" width="3456"/>
</div>
</div>
</div>
</div>
<div class="hwp-accordion hwp-bg-light-base" data-component-name="accordion">
<h3 class="h4">
<button aria-controls="hwp-accordion-1669678527" aria-expanded="false" class="hwp-accordion__button js-hwp-accordion__trigger hwp-accordion__button-auto-close" type="button">
<span class="title">
          Enabling research with BioData Catalyst Powered by PIC-SURE: Faceted search at the variable and variant level across 65 high-value datasets
        </span>
<span aria-hidden="true" class="material-icon !hwp-block">expand_more</span>
</button>
</h3>
<div class="hwp-accordion__content hwp-text-block" hidden="" id="hwp-accordion-1669678527">
<div class="hwp-media hwp-media--full-width">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <img alt="Enabling research with BioData Catalyst Powered by PIC-SURE: Faceted search at the variable and variant level across 65 high-value datasets" height="2579" loading="lazy" src="/sites/g/files/omnuum4001/files/2025-03/PICSURE_BDC_new.png" width="3429"/>
</div>
</div>
</div>
</div>
<div class="hwp-accordion hwp-bg-light-base" data-component-name="accordion">
<h3 class="h4">
<button aria-controls="hwp-accordion-1044536649" aria-expanded="false" class="hwp-accordion__button js-hwp-accordion__trigger hwp-accordion__button-auto-close" type="button">
<span class="title">
          Populating electronic case report forms from Electronic Health Records in a nationwide multicentric study
        </span>
<span aria-hidden="true" class="material-icon !hwp-block">expand_more</span>
</button>
</h3>
<div class="hwp-accordion__content hwp-text-block" hidden="" id="hwp-accordion-1044536649">
<div class="hwp-media hwp-media--full-width">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <img alt="Populating electronic case report forms from Electronic Health Records in a nationwide multicentric study" height="2579" loading="lazy" src="/sites/g/files/omnuum4001/files/2025-03/populating_ecrf_from_ehr.png" width="3429"/>
</div>
</div>
</div>
</div>
<div class="hwp-accordion hwp-bg-light-base" data-component-name="accordion">
<h3 class="h4">
<button aria-controls="hwp-accordion-486867416" aria-expanded="false" class="hwp-accordion__button js-hwp-accordion__trigger hwp-accordion__button-auto-close" type="button">
<span class="title">
          VarPPUD: Variant pathogenicity predictor developed for undiagnosed populations
        </span>
<span aria-hidden="true" class="material-icon !hwp-block">expand_more</span>
</button>
</h3>
<div class="hwp-accordion__content hwp-text-block" hidden="" id="hwp-accordion-486867416">
<div class="hwp-media hwp-media--full-width">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <img alt="VarPPUD: Variant pathogenicity predictor developed for undiagnosed populations" height="3456" loading="lazy" src="/sites/g/files/omnuum4001/files/2025-03/VarPPUD_variant_predictor.png" width="2592"/>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<div aria-labelledby="98e72437-69cb-4df1-b6c9-05248c34526f-1" class="hwp-tabs__panel hwp-bg-light-base hwp-hidden" id="tabpanel-98e72437-69cb-4df1-b6c9-05248c34526f-1" role="tabpanel" tabindex="0">
<div class="hwp-tabs__panel-intro">
<h2 class="h2 hwp-tabs__panel-title">2023</h2>
<div class="hwp-text-block">
</div>
</div>
<div class="hwp-tabs__panel-widgets">
<div class="field field--name-field-hwp-items field--type-entity-reference-revisions field--label-hidden">
<div>
<div class="hwp-accordions js-hwp-accordions hwp-accordions-theme-light-core hwp-bg-light-core hwp-pt-32 md:hwp-pt-64">
<div class="hwp-container hwp-container-px">
<div class="hwp-flex hwp-flex-col hwp-gap-32">
<div class="hwp-accordions__controls js-hwp-accordions__controls__trigger hwp-flex hwp-py-16">
<button class="expand-all hwp-mr-16" data-live-message="All sections have been opened.">
            Open all sections
          </button>
<button class="collapse-all" data-live-message="All sections have been closed.">
            Close all sections
          </button>
</div>
<div aria-live="polite" class="hwp-sr-only hwp-accordions__announce"></div>
<div class="hwp-accordions__items">
<div class="hwp-accordion hwp-bg-light-base" data-component-name="accordion">
<h3 class="h4">
<button aria-controls="hwp-accordion-270238122" aria-expanded="false" class="hwp-accordion__button js-hwp-accordion__trigger hwp-accordion__button-auto-close" type="button">
<span class="title">
          Hospitalizations Associated With Mental Health Conditions Among Adolescents in the US and France During the COVID-19 Pandemic
        </span>
<span aria-hidden="true" class="material-icon !hwp-block">expand_more</span>
</button>
</h3>
<div class="hwp-accordion__content hwp-text-block" hidden="" id="hwp-accordion-270238122">
<div class="hwp-media hwp-media--full-width">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <img alt="Hospitalizations Associated With Mental Health Conditions Among Adolescents in the US and France During the COVID-19 Pandemic" height="2880" loading="lazy" src="/sites/g/files/omnuum4001/files/2025-03/covid_mental_2023_0.png" width="2160"/>
</div>
</div>
</div>
</div>
<div class="hwp-accordion hwp-bg-light-base" data-component-name="accordion">
<h3 class="h4">
<button aria-controls="hwp-accordion-1752932541" aria-expanded="false" class="hwp-accordion__button js-hwp-accordion__trigger hwp-accordion__button-auto-close" type="button">
<span class="title">
          Clinical phenotypes and outcomes in children with multisystem inflammatory syndrome across SARS-CoV-2 variant eras: a multinational study from the 4CE Consortium
        </span>
<span aria-hidden="true" class="material-icon !hwp-block">expand_more</span>
</button>
</h3>
<div class="hwp-accordion__content hwp-text-block" hidden="" id="hwp-accordion-1752932541">
<div class="hwp-media hwp-media--full-width">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <img alt="Clinical phenotypes and outcomes in children with multisystem inflammatory syndrome across SARS-CoV-2 variant eras: a multinational study from the 4CE Consortium" height="2880" loading="lazy" src="/sites/g/files/omnuum4001/files/2025-03/misc_2023_0.png" width="2160"/>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<div aria-labelledby="98e72437-69cb-4df1-b6c9-05248c34526f-2" class="hwp-tabs__panel hwp-bg-light-base hwp-hidden" id="tabpanel-98e72437-69cb-4df1-b6c9-05248c34526f-2" role="tabpanel" tabindex="0">
<div class="hwp-tabs__panel-intro">
<h2 class="h2 hwp-tabs__panel-title">2024</h2>
<div class="hwp-text-block">
</div>
</div>
<div class="hwp-tabs__panel-widgets">
<div class="field field--name-field-hwp-items field--type-entity-reference-revisions field--label-hidden">
<div>
<div class="hwp-accordions js-hwp-accordions hwp-accordions-theme-light-core hwp-bg-light-core hwp-pt-32 md:hwp-pt-64">
<div class="hwp-container hwp-container-px">
<div class="hwp-flex hwp-flex-col hwp-gap-32">
<div class="hwp-accordions__controls js-hwp-accordions__controls__trigger hwp-flex hwp-py-16">
<button class="expand-all hwp-mr-16" data-live-message="All sections have been opened.">
            Open all sections
          </button>
<button class="collapse-all" data-live-message="All sections have been closed.">
            Close all sections
          </button>
</div>
<div aria-live="polite" class="hwp-sr-only hwp-accordions__announce"></div>
<div class="hwp-accordions__items">
<div class="hwp-accordion hwp-bg-light-base" data-component-name="accordion">
<h3 class="h4">
<button aria-controls="hwp-accordion-1528264216" aria-expanded="false" class="hwp-accordion__button js-hwp-accordion__trigger hwp-accordion__button-auto-close" type="button">
<span class="title">
          Impact of COVID-19 Non-Pharmaceutical Interventions on Bacterial Infections in Children: a multicenter international study
        </span>
<span aria-hidden="true" class="material-icon !hwp-block">expand_more</span>
</button>
</h3>
<div class="hwp-accordion__content hwp-text-block" hidden="" id="hwp-accordion-1528264216">
<div class="hwp-media hwp-media--full-width">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <img alt="Impact of COVID-19 Non-Pharmaceutical Interventions on Bacterial Infections in Children: a multicenter international study" height="2880" loading="lazy" src="/sites/g/files/omnuum4001/files/2025-03/bacterial24_0.png" width="2160"/>
</div>
</div>
</div>
</div>
<div class="hwp-accordion hwp-bg-light-base" data-component-name="accordion">
<h3 class="h4">
<button aria-controls="hwp-accordion-378291849" aria-expanded="false" class="hwp-accordion__button js-hwp-accordion__trigger hwp-accordion__button-auto-close" type="button">
<span class="title">
          Development and validation of an open-source pipeline for automatic population of case report forms from electronic health records: a pediatric multi-center prospective study
        </span>
<span aria-hidden="true" class="material-icon !hwp-block">expand_more</span>
</button>
</h3>
<div class="hwp-accordion__content hwp-text-block" hidden="" id="hwp-accordion-378291849">
<div class="hwp-media hwp-media--full-width">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <img alt="Development and validation of an open-source pipeline for automatic population of case report forms from electronic health records: a pediatric multi-center prospective study" height="2880" loading="lazy" src="/sites/g/files/omnuum4001/files/2025-03/music24_0.png" width="2160"/>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu -->
<h1>Avillach Lab | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu'>avillach-lab.hms.harvard.edu</a></div>
<main aria-label="Avillach Lab" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base hwp-sr-only">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Avillach Lab</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="field field--name-field-hwp-additional-content field--type-entity-reference-revisions field--label-hidden">
<div class="paragraphs-item__first paragraphs-item">
<div class="hwp-text-block hwp-bg-light-core hwp-py-32">
<div class="hwp-container-px hwp-container">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><h2>Welcome to Avillach Lab at the <a href="https://cbmi.med.harvard.edu">Department of Biomedical Informatics</a></h2><p><strong>The real value in biomedical research lies not at the scale of any single source of data but in the ability to integrate and interrogate multiple complementary datasets simultaneously.</strong></p><p dir="ltr">The Avillach Lab, led by <a data-entity-substitution="canonical" data-entity-type="node" data-entity-uuid="d7b96fd9-1e1b-403e-a097-7e636b911581" href="/people/paul-avillach-md-phd" title="Paul Avillach, MD, PhD">Paul Avillach, M.D., Ph.D.</a>,  focuses on translational bioinformatics, specifically in integrating multiple heterogeneous sources of clinical and genomic data in a meaningful way. We are passionate about combining data across different scales and resolutions to enable new perspectives for essential biomedical questions. Our research focuses on developing novel methods and techniques for the integration of biological data, clinical cohorts, and Electronic Health Records to encompass biological observations.</p><p dir="ltr">We leverage the PIC-SURE (Patient-centered Information Commons: Standardized Unification of Research Elements) platform to manage a wide variety of arbitrarily large datasets with very little computing in a secured way by searching and extracting clinical and genomic variables of interest. Exposing the data not only at the file level but also at the variable and variant level directly accelerates creation of statistically ready dataframes. PIC-SURE has been deployed in a variety of environments, such as the NIH NHLBI BioData Catalyst® Ecosystem, the Undiagnosed Disease Network, and Boston Children’s Hospital (see <a data-entity-substitution="canonical" data-entity-type="node" data-entity-uuid="e41c2647-f3b6-4a46-a5e8-f8504d568a91" href="/research" title="Research &amp; Collaborations">research</a>).</p><figure class="hwp-media hwp-media--full-width" role="group">
<div>
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <img alt="A person's data consists of many &quot;layers&quot; or data types" height="500" loading="lazy" src="/sites/g/files/omnuum4001/files/avillach_lab/files/home2.png" width="1000"/>
</div>
</div>
<figcaption><strong>Figure depicting various, interconnected “layers” of clinical and genomic data.</strong></figcaption>
</figure>
<p>PIC-SURE contains both a graphical user interface (UI) and an application programming interface (API). With the UI, researchers can test their hypotheses by running multi-criteria queries at both the file- and variable-levels and applying distinct clinical and genomic variable filters in the same search. With the API, users can filter and directly retrieve the data in an analysis-ready format using either R or Python. </p><p> </p></div>
</div></div>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/news-events -->
<h1>News & Events | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/news-events'>avillach-lab.hms.harvard.edu/news-events</a></div>
<main aria-label="News &amp; Events" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>News &amp; Events</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="field field--name-field-hwp-additional-content field--type-entity-reference-revisions field--label-hidden">
<div class="paragraphs-item__first paragraphs-item">
<div class="hwp-page-list hwp-page-list--list hwp-page-list--grid-3 hwp-bg-light-core">
<div class="field field--name-field-hwp-section-heading field--type-entity-reference-revisions field--label-hidden">
<div class="hwp-section-heading" data-component-name="section-heading">
<div class="hwp-section-heading__container hwp-container-px hwp-container">
<h2 class="hwp-section-heading__title text-medium">
      News &amp; Events
    </h2>
</div>
</div>
</div>
<div class="hwp-page-list__container hwp-container-px hwp-container">
<div class="hwp-flex hwp-flex-col hwp-gap-32">
<div id="hwp-node-list-results">
<div class="hwp-page-list__items">
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/more-precise-approach-fighting-cancer"> <h3 class="page-card__heading hwp-order-2">A More Precise Approach to Fighting Cancer</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/datathonhackathon-september-14-15-2017"> <h3 class="page-card__heading hwp-order-2">Datathon/Hackathon September 14 - 15, 2017</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/2016-10-splunk-revolution-awards"> <h3 class="page-card__heading hwp-order-2">2016-10 Splunk Revolution Awards</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/2016-2-presentationvideo-nih-bd2k-pic-sure-supplements-1-ncats-grdr-2-pittsburg-ccd-3"> <h3 class="page-card__heading hwp-order-2">2016-02 PRESENTATION/VIDEO - NIH BD2K PIC-SURE Supplements 1) NCATS GRDR, 2) Pittsburg CCD &amp; 3) Count Everything</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/2015-11-keynote-address"> <h3 class="page-card__heading hwp-order-2">2015-11 KEYNOTE - tranSMART Foundation Annual Meeting -  Creating Trust in Translational Research Platforms</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/2015-09-press-release-dbmi-and-ncats-collaborate-global-rare-diseases-data-repository"> <h3 class="page-card__heading hwp-order-2">2015-09 PRESS RELEASE - DBMI and NCATS to Collaborate on Global Rare Diseases Data Repository</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/presentation-technological-tools-open-science-and-collaboration-neuroscience"> <h3 class="page-card__heading hwp-order-2">2015-05 PRESENTATION - Technological tools for Open Science and Collaboration in neuroscience</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/presentation-creating-patient-centric-information-commons-autism-research"> <h3 class="page-card__heading hwp-order-2">2015-05 PRESENTATION - INSAR - Creating a patient centric information commons for autism research</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/presentation-functional-genomics-predictive-medicine"> <h3 class="page-card__heading hwp-order-2">2015-05 PRESENTATION - Functional Genomics &amp; Predictive Medicine</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/toward-precision-medicine-building-patient-centric-information-commons-common-and-rare-0"> <h3 class="page-card__heading hwp-order-2">2015-03 PRESENTATION - "Toward Precision Medicine: Building a Patient -centric Information Commons on Common and Rare Diseases with i2b2/tranSMART” Application to Autism and Phelan McDermid Syndrome</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/toward-precision-medicine-building-patient-centric-information-commons-common-and-rare"> <h3 class="page-card__heading hwp-order-2">2014-11 KEYNOTE - VIDEO - Toward Precision Medicine: Building a patient centric information commons on common and rare diseases with tranSMART</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/press-release-transmart-informatics-platform-be-used-1m-phelan-mcdermid-syndrome"> <h3 class="page-card__heading hwp-order-2">2014-07 PRESS RELEASE - TranSmart Informatics Platform to be Used in $1M Phelan-McDermid Syndrome Research Project</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/phenome-wide-association-studies-quantitative-trait-application-tpmt-enzymatic-activity"> <h3 class="page-card__heading hwp-order-2">2014-04 PRESENTATION - Phenome-wide association studies on a quantitative trait: Application to TPMT enzymatic activity </h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/phelan-mcdermid-syndrome-foundation-team-approved-funding-award-become-part-new"> <h3 class="page-card__heading hwp-order-2">2014-02 PRESS RELEASE - Phelan-McDermid Syndrome Foundation team approved for funding award to become part of new National Clinical Research Network</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/creating-clinical-and-omics-information-commons-using-i2b2-and-transmart"> <h3 class="page-card__heading hwp-order-2">2013-12 KEYNOTE - Institut Curie - Paris France - Creating clinical and 'omics information commons using i2b2 and tranSMART </h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/about-30-people-autism-take-multiple-medications-manage-comorbid-conditions-0"> <h3 class="page-card__heading hwp-order-2">About 30% Of People With Autism Take Multiple Medications To Manage Comorbid Conditions</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/numbers-polypharmacy-outpatient-autism-care-pandemic-behaviors"> <h3 class="page-card__heading hwp-order-2">By the Numbers: Polypharmacy, outpatient autism care, pandemic behaviors</h3>
</a>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-page page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/polypharmacy-shifting-prescriptions-common-autism-comorbidities"> <h3 class="page-card__heading hwp-order-2">Polypharmacy, shifting prescriptions common for autism comorbidities</h3>
</a>
</div>
</article>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/people-display -->
<h1>People | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/people-display'>avillach-lab.hms.harvard.edu/people-display</a></div>
<main aria-label="People" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>People</span>
</h1>
</div>
</div>
<div class="field field--name-field-hwp-content-top field--type-entity-reference-revisions field--label-hidden">
<div class="paragraphs-item__first paragraphs-item">
<div class="hwp-hero hwp-hero--split-horizontal hwp-hero--medium hwp-hero--static" data-component-name="hwp-hero">
<div class="hwp-hero__slides js-hwp-hero-slider-trigger">
<div class="hwp-hero__slide">
<div class="hwp-hero__slides-wrapper">
<div class="hwp-hero__content hwp-bg-light-core hwp-hero__content--left">
<h2 class="hwp-hero__title hwp-order-2 h1" id="">
    Avillach Lab Members
  </h2>
<div class="hwp-hero__description hwp-order-3 hwp-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>Members of the Avillach Lab, including the current team and alumni. </p></div>
</div>
<div class="hwp-hero__buttons hwp-order-4">
</div>
</div>
<div class="hwp-hero__media hwp-bg-light-core">
<div class="hwp-media hwp-media--28-10-medium-hero">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="685" media="all and (min-width: 1440px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_28_10__1920x685/public/avillach_lab/files/img_6093.jpg?h=4c50a2ad&amp;itok=gJ59gfB1 1x, /sites/g/files/omnuum4001/files/styles/hwp_28_10__3840x1370/public/avillach_lab/files/img_6093.jpg?h=4c50a2ad&amp;itok=PCw1Lu-r 2x" type="image/jpeg" width="1920"/>
<source height="510" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_28_10__1440x510/public/avillach_lab/files/img_6093.jpg?h=4c50a2ad&amp;itok=C9wkGUdG 1x, /sites/g/files/omnuum4001/files/styles/hwp_28_10__2880x1030/public/avillach_lab/files/img_6093.jpg?h=4c50a2ad&amp;itok=MroHCWeq 2x" type="image/jpeg" width="1440"/>
<source height="365" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_28_10__1024x365/public/avillach_lab/files/img_6093.jpg?h=4c50a2ad&amp;itok=WMoaL6FD 1x, /sites/g/files/omnuum4001/files/styles/hwp_28_10__2048x730/public/avillach_lab/files/img_6093.jpg?h=4c50a2ad&amp;itok=LvG_lVPH 2x" type="image/jpeg" width="1024"/>
<source height="275" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_28_10__768x275/public/avillach_lab/files/img_6093.jpg?h=4c50a2ad&amp;itok=H2Q0b2Tz 1x, /sites/g/files/omnuum4001/files/styles/hwp_28_10__1536x550/public/avillach_lab/files/img_6093.jpg?h=4c50a2ad&amp;itok=6btoKLOJ 2x" type="image/jpeg" width="768"/>
<source height="270" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_16_9__480x270/public/avillach_lab/files/img_6093.jpg?h=a64ac712&amp;itok=we5aTRPY 1x, /sites/g/files/omnuum4001/files/styles/hwp_16_9__960x540/public/avillach_lab/files/img_6093.jpg?h=a64ac712&amp;itok=-dPyTglP 2x" type="image/jpeg" width="480"/>
<img alt="Avillach Lab group photo Sept. 2023" height="685" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_28_10__1920x685/public/avillach_lab/files/img_6093.jpg?h=4c50a2ad&amp;itok=gJ59gfB1" width="1920"/>
</picture>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="field field--name-field-hwp-additional-content field--type-entity-reference-revisions field--label-hidden">
<div class="paragraphs-item__first paragraphs-item">
<div class="hwp-page-list hwp-page-list--list hwp-page-list--grid-4 hwp-bg-light-core">
<div class="field field--name-field-hwp-section-heading field--type-entity-reference-revisions field--label-hidden">
<div class="hwp-section-heading" data-component-name="section-heading">
<div class="hwp-section-heading__container hwp-container-px hwp-container">
<h2 class="hwp-section-heading__title text-medium">
      Principal Investigator
    </h2>
</div>
</div>
</div>
<div class="hwp-page-list__container hwp-container-px hwp-container">
<div class="hwp-flex hwp-flex-col hwp-gap-32">
<div id="hwp-node-list-results">
<div class="hwp-page-list__items">
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/paul-avillach-md-phd"> <h3 class="page-card__heading hwp-order-2">Paul Avillach, MD, PhD</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:paul_avillach@hms.harvard.edu">paul_avillach@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Assistant Professor at Harvard Medical School</div>
</div>
</div>
<div class="page-card__description hwp-order-3 hwp-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden">  Dr. Avillach is Assistant Professor in Pediatrics &amp; Biomedical Informatics, Harvard Medical School and Boston Children’s Hospital. He received his MD in Public Health from University of Bordeaux, France and PhD in Biomedical Informatics from University of...

</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/paul_avillach_h.jpg?itok=q5VtKch5 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/paul_avillach_h.jpg?itok=ZU8ONmSp 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/paul_avillach_h.jpg?itok=DT9lYtlP 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/paul_avillach_h.jpg?itok=ggQZ85Au 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/paul_avillach_h.jpg?itok=q5VtKch5 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/paul_avillach_h.jpg?itok=ZU8ONmSp 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/paul_avillach_h.jpg?itok=DT9lYtlP 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/paul_avillach_h.jpg?itok=ggQZ85Au 2x" type="image/jpeg" width="480"/>
<img alt="Dr. Paul Avillach" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/paul_avillach_h.jpg?itok=q5VtKch5" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="paragraphs-item">
<div class="hwp-page-list hwp-page-list--list hwp-page-list--grid-4 hwp-bg-light-core">
<div class="field field--name-field-hwp-section-heading field--type-entity-reference-revisions field--label-hidden">
<div class="hwp-section-heading" data-component-name="section-heading">
<div class="hwp-section-heading__container hwp-container-px hwp-container">
<h2 class="hwp-section-heading__title text-medium">
      Project Manager
    </h2>
</div>
</div>
</div>
<div class="hwp-page-list__container hwp-container-px hwp-container">
<div class="hwp-flex hwp-flex-col hwp-gap-32">
<div id="hwp-node-list-results--2">
<div class="hwp-page-list__items">
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/jessica-carpenter-lyons"> <h3 class="page-card__heading hwp-order-2">Jessica  Carpenter Lyons</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:jessica_lyons@hms.harvard.edu">jessica_lyons@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior Project Manager</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=xv1SCPBa 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=pKzi9Bp- 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=37BK6FiK 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=2CGF93KE 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=xv1SCPBa 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=pKzi9Bp- 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=37BK6FiK 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=2CGF93KE 2x" type="image/jpeg" width="480"/>
<img alt="Jessica Lyons" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=xv1SCPBa" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="paragraphs-item">
<div class="hwp-page-list hwp-page-list--list hwp-page-list--grid-4 hwp-bg-light-core">
<div class="field field--name-field-hwp-section-heading field--type-entity-reference-revisions field--label-hidden">
<div class="hwp-section-heading" data-component-name="section-heading">
<div class="hwp-section-heading__container hwp-container-px hwp-container">
<h2 class="hwp-section-heading__title text-medium">
      Instructor
    </h2>
</div>
</div>
</div>
<div class="hwp-page-list__container hwp-container-px hwp-container">
<div class="hwp-flex hwp-flex-col hwp-gap-32">
<div id="hwp-node-list-results--3">
<div class="hwp-page-list__items">
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/alba-gutierrez-sacristan"> <h3 class="page-card__heading hwp-order-2">Alba Gutiérrez-Sacristán, PhD</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:alba_gutierrez@hms.harvard.edu">alba_gutierrez@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Instructor in Biomedical Informatics</div>
</div>
</div>
<div class="page-card__description hwp-order-3 hwp-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden">  Ph.D. in Biomedicine. Working on ASD.

</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/foto2.png?itok=JGkVWHr6 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/foto2.png?itok=22xo_F3P 2x" type="image/png" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/foto2.png?itok=IjflggkL 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/foto2.png?itok=SHyvXr-I 2x" type="image/png" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/foto2.png?itok=JGkVWHr6 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/foto2.png?itok=22xo_F3P 2x" type="image/png" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/foto2.png?itok=IjflggkL 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/foto2.png?itok=SHyvXr-I 2x" type="image/png" width="480"/>
<img alt="Alba Gutierrez, MSc" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/foto2.png?itok=JGkVWHr6" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="paragraphs-item">
<div class="hwp-page-list hwp-page-list--list hwp-page-list--grid-4 hwp-bg-light-core">
<div class="field field--name-field-hwp-section-heading field--type-entity-reference-revisions field--label-hidden">
<div class="hwp-section-heading" data-component-name="section-heading">
<div class="hwp-section-heading__container hwp-container-px hwp-container">
<h2 class="hwp-section-heading__title text-medium">
      Visiting Scholars
    </h2>
</div>
</div>
</div>
<div class="hwp-page-list__container hwp-container-px hwp-container">
<div class="hwp-flex hwp-flex-col hwp-gap-32">
<div id="hwp-node-list-results--4">
<div class="hwp-page-list__items">
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/joany-zachariasse"> <h3 class="page-card__heading hwp-order-2">Joany Zachariasse </h3>
</a>
<div class="page-card__subheading hwp-order-2"></div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/thumbnail_1h8a1890.jpeg?itok=sZKZlbmy 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/thumbnail_1h8a1890.jpeg?itok=ZU0j-Siw 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/thumbnail_1h8a1890.jpeg?itok=swyOZDv0 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/thumbnail_1h8a1890.jpeg?itok=QC0Obcel 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/thumbnail_1h8a1890.jpeg?itok=sZKZlbmy 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/thumbnail_1h8a1890.jpeg?itok=ZU0j-Siw 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/thumbnail_1h8a1890.jpeg?itok=swyOZDv0 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/thumbnail_1h8a1890.jpeg?itok=QC0Obcel 2x" type="image/jpeg" width="480"/>
<img alt="Joany Zachariasse" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/thumbnail_1h8a1890.jpeg?itok=sZKZlbmy" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/louis-hayot"> <h3 class="page-card__heading hwp-order-2">Louis Hayot</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Visiting scholar</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/2025-03/IMG_4579.jpg?itok=umdE2E0M 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/2025-03/IMG_4579.jpg?itok=ybNSAb6V 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/2025-03/IMG_4579.jpg?itok=IOSEO2gj 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/2025-03/IMG_4579.jpg?itok=9VG-_kjc 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/2025-03/IMG_4579.jpg?itok=umdE2E0M 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/2025-03/IMG_4579.jpg?itok=ybNSAb6V 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/2025-03/IMG_4579.jpg?itok=IOSEO2gj 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/2025-03/IMG_4579.jpg?itok=9VG-_kjc 2x" type="image/jpeg" width="480"/>
<img alt="Louis Hayot" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/2025-03/IMG_4579.jpg?itok=umdE2E0M" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="paragraphs-item">
<div class="hwp-page-list hwp-page-list--list hwp-page-list--grid-4 hwp-bg-light-core">
<div class="field field--name-field-hwp-section-heading field--type-entity-reference-revisions field--label-hidden">
<div class="hwp-section-heading" data-component-name="section-heading">
<div class="hwp-section-heading__container hwp-container-px hwp-container">
<h2 class="hwp-section-heading__title text-medium">
      Program, Product &amp; Research Engagement Team
    </h2>
</div>
</div>
</div>
<div class="hwp-page-list__container hwp-container-px hwp-container">
<div class="hwp-flex hwp-flex-col hwp-gap-32">
<div id="hwp-node-list-results--5">
<div class="hwp-page-list__items">
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/emily-hughes"> <h3 class="page-card__heading hwp-order-2">Emily Hughes, MS</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Bioinformatics Systems Product Manager</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/2025-03/0083_Emily_Hughes_001_tiny.jpeg?itok=en7jScTQ 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/2025-03/0083_Emily_Hughes_001_tiny.jpeg?itok=N0dPCQ8L 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/2025-03/0083_Emily_Hughes_001_tiny.jpeg?itok=UGXT2lnD 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/2025-03/0083_Emily_Hughes_001_tiny.jpeg?itok=olmXQqAl 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/2025-03/0083_Emily_Hughes_001_tiny.jpeg?itok=en7jScTQ 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/2025-03/0083_Emily_Hughes_001_tiny.jpeg?itok=N0dPCQ8L 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/2025-03/0083_Emily_Hughes_001_tiny.jpeg?itok=UGXT2lnD 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/2025-03/0083_Emily_Hughes_001_tiny.jpeg?itok=olmXQqAl 2x" type="image/jpeg" width="480"/>
<img alt="Emily Hughes" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/2025-03/0083_Emily_Hughes_001_tiny.jpeg?itok=en7jScTQ" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/kimberly-huynh-mph"> <h3 class="page-card__heading hwp-order-2">Kimberly Huynh, MPH</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:Kimberly_Huynh@hms.harvard.edu">Kimberly_Huynh@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior Program Manager</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/2025-03/IMG_3980.jpg?itok=yD1Jw7cz 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/2025-03/IMG_3980.jpg?itok=_TKxPhTc 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/2025-03/IMG_3980.jpg?itok=BnOcF0Gt 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/2025-03/IMG_3980.jpg?itok=WSiHu6YS 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/2025-03/IMG_3980.jpg?itok=yD1Jw7cz 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/2025-03/IMG_3980.jpg?itok=_TKxPhTc 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/2025-03/IMG_3980.jpg?itok=BnOcF0Gt 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/2025-03/IMG_3980.jpg?itok=WSiHu6YS 2x" type="image/jpeg" width="480"/>
<img alt="Kimberly Huynh, MPH" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/2025-03/IMG_3980.jpg?itok=yD1Jw7cz" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="paragraphs-item">
<div class="hwp-page-list hwp-page-list--list hwp-page-list--grid-4 hwp-bg-light-core">
<div class="field field--name-field-hwp-section-heading field--type-entity-reference-revisions field--label-hidden">
<div class="hwp-section-heading" data-component-name="section-heading">
<div class="hwp-section-heading__container hwp-container-px hwp-container">
<h2 class="hwp-section-heading__title text-medium">
      Software Engineers and Programmers
    </h2>
</div>
</div>
</div>
<div class="hwp-page-list__container hwp-container-px hwp-container">
<div class="hwp-flex hwp-flex-col hwp-gap-32">
<div id="hwp-node-list-results--6">
<div class="hwp-page-list__items">
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/danielle-pillion"> <h3 class="page-card__heading hwp-order-2">Danielle Pillion</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Software Engineering Manager</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/dpillion_profile_image.jpeg?itok=kC_GtNcA 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/dpillion_profile_image.jpeg?itok=83WCEWJM 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/dpillion_profile_image.jpeg?itok=-FK2Dnv0 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/dpillion_profile_image.jpeg?itok=I3Tuserr 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/dpillion_profile_image.jpeg?itok=kC_GtNcA 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/dpillion_profile_image.jpeg?itok=83WCEWJM 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/dpillion_profile_image.jpeg?itok=-FK2Dnv0 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/dpillion_profile_image.jpeg?itok=I3Tuserr 2x" type="image/jpeg" width="480"/>
<img alt="DaniellePillion" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/dpillion_profile_image.jpeg?itok=kC_GtNcA" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/thomas-n-desain"> <h3 class="page-card__heading hwp-order-2">Thomas N. DeSain</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior Full Stack Software Engineer </div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/img_0106.jpg?itok=QpYf2Mt1 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/img_0106.jpg?itok=Udf-eCyS 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/img_0106.jpg?itok=CxD8NA3e 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/img_0106.jpg?itok=FPE3cBPz 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/img_0106.jpg?itok=QpYf2Mt1 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/img_0106.jpg?itok=Udf-eCyS 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/img_0106.jpg?itok=CxD8NA3e 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/img_0106.jpg?itok=FPE3cBPz 2x" type="image/jpeg" width="480"/>
<img alt="Thomas N. DeSain" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/img_0106.jpg?itok=QpYf2Mt1" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/ryan-amari"> <h3 class="page-card__heading hwp-order-2">Ryan Amari</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior Full Stack Software Engineer</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/screen_shot_2021-09-13_at_3.12.45_pm.png?itok=72YrFmhk 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/screen_shot_2021-09-13_at_3.12.45_pm.png?itok=Np1nK8v9 2x" type="image/png" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/screen_shot_2021-09-13_at_3.12.45_pm.png?itok=XzRQcUMD 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/screen_shot_2021-09-13_at_3.12.45_pm.png?itok=bLz3IIXd 2x" type="image/png" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/screen_shot_2021-09-13_at_3.12.45_pm.png?itok=72YrFmhk 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/screen_shot_2021-09-13_at_3.12.45_pm.png?itok=Np1nK8v9 2x" type="image/png" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/screen_shot_2021-09-13_at_3.12.45_pm.png?itok=XzRQcUMD 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/screen_shot_2021-09-13_at_3.12.45_pm.png?itok=bLz3IIXd 2x" type="image/png" width="480"/>
<img alt="ryan amari" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/screen_shot_2021-09-13_at_3.12.45_pm.png?itok=72YrFmhk" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/james-peck"> <h3 class="page-card__heading hwp-order-2">James Peck</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Full Stack Software Engineer </div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/jamespeck.jpg?itok=qJS8OQWP 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/jamespeck.jpg?itok=nGiEqHD_ 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/jamespeck.jpg?itok=nuZiAzoi 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/jamespeck.jpg?itok=Cr_-nsQS 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/jamespeck.jpg?itok=qJS8OQWP 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/jamespeck.jpg?itok=nGiEqHD_ 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/jamespeck.jpg?itok=nuZiAzoi 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/jamespeck.jpg?itok=Cr_-nsQS 2x" type="image/jpeg" width="480"/>
<img alt="James Peck" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/jamespeck.jpg?itok=qJS8OQWP" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/sharra-neely"> <h3 class="page-card__heading hwp-order-2">Sharra Neely</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Full Stack Software Engineer </div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/headshot.jpg?itok=BIRCcahN 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/headshot.jpg?itok=PBleYpVp 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/headshot.jpg?itok=w4cJj3_k 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/headshot.jpg?itok=sDhlylsP 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/headshot.jpg?itok=BIRCcahN 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/headshot.jpg?itok=PBleYpVp 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/headshot.jpg?itok=w4cJj3_k 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/headshot.jpg?itok=sDhlylsP 2x" type="image/jpeg" width="480"/>
<img alt="Sharra Neely" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/headshot.jpg?itok=BIRCcahN" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/samantha-pliatt"> <h3 class="page-card__heading hwp-order-2">Samantha Piatt</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Research Software Developer II</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/samanthapiatt.jpg?itok=nF_2aqda 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/samanthapiatt.jpg?itok=r9N49qdx 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/samanthapiatt.jpg?itok=s6_tkhLW 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/samanthapiatt.jpg?itok=HHFU-p4k 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/samanthapiatt.jpg?itok=nF_2aqda 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/samanthapiatt.jpg?itok=r9N49qdx 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/samanthapiatt.jpg?itok=s6_tkhLW 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/samanthapiatt.jpg?itok=HHFU-p4k 2x" type="image/jpeg" width="480"/>
<img alt="Samantha Piatt" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/samanthapiatt.jpg?itok=nF_2aqda" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/luke-sikina"> <h3 class="page-card__heading hwp-order-2">Luke Sikina</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior Full Stack Software Engineer </div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/2025-03/Option%209.jpg?itok=lFlHY01Q 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/2025-03/Option%209.jpg?itok=eucl-b6V 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/2025-03/Option%209.jpg?itok=Mk8TT1F9 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/2025-03/Option%209.jpg?itok=mNbkYRmW 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/2025-03/Option%209.jpg?itok=lFlHY01Q 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/2025-03/Option%209.jpg?itok=eucl-b6V 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/2025-03/Option%209.jpg?itok=Mk8TT1F9 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/2025-03/Option%209.jpg?itok=mNbkYRmW 2x" type="image/jpeg" width="480"/>
<img alt="Luke Sikina" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/2025-03/Option%209.jpg?itok=lFlHY01Q" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/george-colon"> <h3 class="page-card__heading hwp-order-2">George Colon</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Full Stack Software Engineer </div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/george_colon_headshot.jpg?itok=zCnMSzV6 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/george_colon_headshot.jpg?itok=yEw67Lpk 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/george_colon_headshot.jpg?itok=fvwG64cS 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/george_colon_headshot.jpg?itok=r1KNsJ1y 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/george_colon_headshot.jpg?itok=zCnMSzV6 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/george_colon_headshot.jpg?itok=yEw67Lpk 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/george_colon_headshot.jpg?itok=fvwG64cS 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/george_colon_headshot.jpg?itok=r1KNsJ1y 2x" type="image/jpeg" width="480"/>
<img alt="George Colon" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/george_colon_headshot.jpg?itok=zCnMSzV6" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/michael-corace"> <h3 class="page-card__heading hwp-order-2">Michael Corace</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Bioinformatics Data Scientist</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/image_6.png?itok=H-GcTsO9 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/image_6.png?itok=jb_b6YWf 2x" type="image/png" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/image_6.png?itok=o2sjKVN_ 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/image_6.png?itok=SEIUsgBN 2x" type="image/png" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/image_6.png?itok=H-GcTsO9 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/image_6.png?itok=jb_b6YWf 2x" type="image/png" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/image_6.png?itok=o2sjKVN_ 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/image_6.png?itok=SEIUsgBN 2x" type="image/png" width="480"/>
<img alt="Michael Corace" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/image_6.png?itok=H-GcTsO9" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="paragraphs-item">
<div class="hwp-page-list hwp-page-list--list hwp-page-list--grid-4 hwp-bg-light-core">
<div class="field field--name-field-hwp-section-heading field--type-entity-reference-revisions field--label-hidden">
<div class="hwp-section-heading" data-component-name="section-heading">
<div class="hwp-section-heading__container hwp-container-px hwp-container">
<h2 class="hwp-section-heading__title text-medium">
      Alumni
    </h2>
</div>
</div>
</div>
<div class="hwp-page-list__container hwp-container-px hwp-container">
<div class="hwp-flex hwp-flex-col hwp-gap-32">
<div id="hwp-node-list-results--7">
<div class="hwp-page-list__items">
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/andre-rosa"> <h3 class="page-card__heading hwp-order-2">Andre Rosa</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:andre_rosa@hms.harvard.edu">andre_rosa@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior Java Web Developer</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/headshot-cleaned.jpg?itok=2PjFEe9k 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/headshot-cleaned.jpg?itok=dgS0x1JY 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/headshot-cleaned.jpg?itok=vaCHFmDU 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/headshot-cleaned.jpg?itok=8Je78izZ 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/headshot-cleaned.jpg?itok=2PjFEe9k 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/headshot-cleaned.jpg?itok=dgS0x1JY 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/headshot-cleaned.jpg?itok=vaCHFmDU 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/headshot-cleaned.jpg?itok=8Je78izZ 2x" type="image/jpeg" width="480"/>
<img alt="Andre Rosa" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/headshot-cleaned.jpg?itok=2PjFEe9k" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/aleksander-nikitin"> <h3 class="page-card__heading hwp-order-2">Aleksandr Nikitin</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:aleksandr.nikitin@childrens.harvard.edu">aleksandr.nikitin@childrens.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Application Development Specialist</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/20160617_163651_0.jpg?itok=vVoWJYEK 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/20160617_163651_0.jpg?itok=1cFvC6e7 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/20160617_163651_0.jpg?itok=aU-L7w4d 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/20160617_163651_0.jpg?itok=nhp7T8Bg 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/20160617_163651_0.jpg?itok=vVoWJYEK 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/20160617_163651_0.jpg?itok=1cFvC6e7 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/20160617_163651_0.jpg?itok=aU-L7w4d 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/20160617_163651_0.jpg?itok=nhp7T8Bg 2x" type="image/jpeg" width="480"/>
<img alt="Aleksandr Nikitin" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/20160617_163651_0.jpg?itok=vVoWJYEK" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/%C3%A1lvaro-hern%C3%A1ndez"> <h3 class="page-card__heading hwp-order-2">Álvaro Hernández</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Visiting Graduate Student</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/img_3772_2.jpeg?itok=NF2WBudw 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/img_3772_2.jpeg?itok=VkkTkyYs 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/img_3772_2.jpeg?itok=L7ACdIij 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/img_3772_2.jpeg?itok=wOOLg59m 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/img_3772_2.jpeg?itok=NF2WBudw 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/img_3772_2.jpeg?itok=VkkTkyYs 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/img_3772_2.jpeg?itok=L7ACdIij 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/img_3772_2.jpeg?itok=wOOLg59m 2x" type="image/jpeg" width="480"/>
<img alt="Álvaro Hernández" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/img_3772_2.jpeg?itok=NF2WBudw" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/ambia-ahmed"> <h3 class="page-card__heading hwp-order-2">Ambia Ahmed</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior Quality Assurance Engineer</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/1690403079698.jpg?itok=TiJBwf0a 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/1690403079698.jpg?itok=kaONz3Vc 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/1690403079698.jpg?itok=QlGUEd2y 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/1690403079698.jpg?itok=p0UPH6Nn 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/1690403079698.jpg?itok=TiJBwf0a 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/1690403079698.jpg?itok=kaONz3Vc 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/1690403079698.jpg?itok=QlGUEd2y 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/1690403079698.jpg?itok=p0UPH6Nn 2x" type="image/jpeg" width="480"/>
<img alt="Ambia Ahmed" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/1690403079698.jpg?itok=TiJBwf0a" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/andrew-j-guidetti"> <h3 class="page-card__heading hwp-order-2">Andrew J Guidetti</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:andrew_guidetti@hms.harvard.edu">andrew_guidetti@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior DevOps Engineer</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/2016-02-04_23.53.59.jpg?itok=0RLimr8m 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/2016-02-04_23.53.59.jpg?itok=mni77eGS 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/2016-02-04_23.53.59.jpg?itok=SJ3smFdZ 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/2016-02-04_23.53.59.jpg?itok=bhxb6rLd 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/2016-02-04_23.53.59.jpg?itok=0RLimr8m 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/2016-02-04_23.53.59.jpg?itok=mni77eGS 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/2016-02-04_23.53.59.jpg?itok=SJ3smFdZ 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/2016-02-04_23.53.59.jpg?itok=bhxb6rLd 2x" type="image/jpeg" width="480"/>
<img alt="Andrew J. Guidetti" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/2016-02-04_23.53.59.jpg?itok=0RLimr8m" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/andrzej-cuper"> <h3 class="page-card__heading hwp-order-2">Andrzej Cuper</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>ETL Developer</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/image_1.jpg?itok=dFY2Sppr 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/image_1.jpg?itok=YUX-u_ij 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/image_1.jpg?itok=GM2onAWE 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/image_1.jpg?itok=J-ZjSGLz 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/image_1.jpg?itok=dFY2Sppr 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/image_1.jpg?itok=YUX-u_ij 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/image_1.jpg?itok=GM2onAWE 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/image_1.jpg?itok=J-ZjSGLz 2x" type="image/jpeg" width="480"/>
<img alt="Andrzej Cuper" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/image_1.jpg?itok=dFY2Sppr" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/anil-degala"> <h3 class="page-card__heading hwp-order-2">Anil Degala</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Director of Data Engineering, Data Integrations and Data Architecture</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/anild.jpeg?itok=GBFx98L3 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/anild.jpeg?itok=-ZkAyA-2 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/anild.jpeg?itok=ocY4GtNS 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/anild.jpeg?itok=yBh80BHO 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/anild.jpeg?itok=GBFx98L3 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/anild.jpeg?itok=-ZkAyA-2 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/anild.jpeg?itok=ocY4GtNS 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/anild.jpeg?itok=yBh80BHO 2x" type="image/jpeg" width="480"/>
<img alt="Anil Degala" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/anild.jpeg?itok=GBFx98L3" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/antoine-neuraz-md-msc"> <h3 class="page-card__heading hwp-order-2">Antoine Neuraz, MD, MSc</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:antoine_neuraz@hms.harvard.edu">antoine_neuraz@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Research associate at Harvard Medical School</div>
</div>
</div>
<div class="page-card__description hwp-order-3 hwp-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden">  I work on large scale genotype-phenotype associations studies (GWAS, PheWAS) in the field of autism spectrum disorders.

</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/carre_0.jpg?itok=OpxsEfxx 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/carre_0.jpg?itok=g6gj3mos 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/carre_0.jpg?itok=FJTaARiR 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/carre_0.jpg?itok=V-7xXsAA 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/carre_0.jpg?itok=OpxsEfxx 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/carre_0.jpg?itok=g6gj3mos 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/carre_0.jpg?itok=FJTaARiR 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/carre_0.jpg?itok=V-7xXsAA 2x" type="image/jpeg" width="480"/>
<img alt="Antoine Neurraz" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/carre_0.jpg?itok=OpxsEfxx" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/antoine-tisseaux"> <h3 class="page-card__heading hwp-order-2">Antoine Tisseaux</h3>
</a>
<div class="page-card__subheading hwp-order-2"></div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/antoine_tisseaux_headshot.jpg?itok=F83V8r-6 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/antoine_tisseaux_headshot.jpg?itok=bqg0AJe9 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/antoine_tisseaux_headshot.jpg?itok=jbOe9kEk 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/antoine_tisseaux_headshot.jpg?itok=NmKk7bWL 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/antoine_tisseaux_headshot.jpg?itok=F83V8r-6 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/antoine_tisseaux_headshot.jpg?itok=bqg0AJe9 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/antoine_tisseaux_headshot.jpg?itok=jbOe9kEk 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/antoine_tisseaux_headshot.jpg?itok=NmKk7bWL 2x" type="image/jpeg" width="480"/>
<img alt="Antoine Tisseaux" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/antoine_tisseaux_headshot.jpg?itok=F83V8r-6" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/antoine-tran-md-msc"> <h3 class="page-card__heading hwp-order-2">Antoine Tran, MD, MSc</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:antoine_tran@hms.harvard.edu">antoine_tran@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Research Fellow in Biomedical Informatics</div>
</div>
</div>
<div class="page-card__description hwp-order-3 hwp-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden">  MD in Pediatrics, specializing in Epidemiology, from the University Children Hospital, CHU-Lenval, Nice (France). Working on Global Rare Disease Registry Project

</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/at.png?itok=8-ld0c4V 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/at.png?itok=m1rij1Y7 2x" type="image/png" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/at.png?itok=2ChMlef9 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/at.png?itok=sTivDYah 2x" type="image/png" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/at.png?itok=8-ld0c4V 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/at.png?itok=m1rij1Y7 2x" type="image/png" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/at.png?itok=2ChMlef9 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/at.png?itok=sTivDYah 2x" type="image/png" width="480"/>
<img alt="Antoine Tran" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/at.png?itok=8-ld0c4V" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/arabdha-biswas-0"> <h3 class="page-card__heading hwp-order-2">Arabdha Biswas</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Summer Institute in Biomedical Informatics Intern</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/dsc0334_1_01.jpg?itok=ZHKGImFH 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/dsc0334_1_01.jpg?itok=lEcitwV1 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/dsc0334_1_01.jpg?itok=CZje8kIN 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/dsc0334_1_01.jpg?itok=HWdrN0ju 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/dsc0334_1_01.jpg?itok=ZHKGImFH 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/dsc0334_1_01.jpg?itok=lEcitwV1 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/dsc0334_1_01.jpg?itok=CZje8kIN 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/dsc0334_1_01.jpg?itok=HWdrN0ju 2x" type="image/jpeg" width="480"/>
<img alt="Arabdha Biswas" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/dsc0334_1_01.jpg?itok=ZHKGImFH" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/arnaud-serret-larmande"> <h3 class="page-card__heading hwp-order-2">Arnaud Serret-Larmande</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:Arnaud_Serret-Larmande@hms.harvard.edu">Arnaud_Serret-Larmande@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Research Fellow in Biomedical Informatics</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/lespiderman.jpeg?itok=cMNURX4J 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/lespiderman.jpeg?itok=gOgxvVKm 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/lespiderman.jpeg?itok=3xQnSDMX 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/lespiderman.jpeg?itok=yPl_ikKt 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/lespiderman.jpeg?itok=cMNURX4J 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/lespiderman.jpeg?itok=gOgxvVKm 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/lespiderman.jpeg?itok=3xQnSDMX 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/lespiderman.jpeg?itok=yPl_ikKt 2x" type="image/jpeg" width="480"/>
<img alt="Arnaud Serret-Larmande" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/lespiderman.jpeg?itok=cMNURX4J" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/atul-sanagar"> <h3 class="page-card__heading hwp-order-2">Atul Sanagar</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Software Developer</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/atul_profile.jpg?itok=HWkVCWb6 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/atul_profile.jpg?itok=2CPURjnd 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/atul_profile.jpg?itok=ejCjU36z 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/atul_profile.jpg?itok=Nn4SFIFm 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/atul_profile.jpg?itok=HWkVCWb6 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/atul_profile.jpg?itok=2CPURjnd 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/atul_profile.jpg?itok=ejCjU36z 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/atul_profile.jpg?itok=Nn4SFIFm 2x" type="image/jpeg" width="480"/>
<img alt="atul_photo" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/atul_profile.jpg?itok=HWkVCWb6" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/bertrand-moal"> <h3 class="page-card__heading hwp-order-2">Bertrand Moal</h3>
</a>
<div class="page-card__subheading hwp-order-2"></div>
<div class="page-card__description hwp-order-3 hwp-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden">  Bertrand Moal holds an MD in public health and epidemiology from the University of Bordeaux and a PhD in biomechanics from Arts et Métiers ParisTech.

</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/moal.jpg?itok=k1lRA0CG 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/moal.jpg?itok=GexPsVbK 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/moal.jpg?itok=M-OedEKS 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/moal.jpg?itok=45A0xJf_ 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/moal.jpg?itok=k1lRA0CG 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/moal.jpg?itok=GexPsVbK 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/moal.jpg?itok=M-OedEKS 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/moal.jpg?itok=45A0xJf_ 2x" type="image/jpeg" width="480"/>
<img alt="Moal" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/moal.jpg?itok=k1lRA0CG" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/bria-morgan"> <h3 class="page-card__heading hwp-order-2">Bria Morgan</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:bria_morgan@hms.harvard.edu">bria_morgan@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Software Developer</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/img_0087.cropped.jpg?itok=TWJTqWRn 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/img_0087.cropped.jpg?itok=6C0Lh6lG 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/img_0087.cropped.jpg?itok=UE2VuqRI 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/img_0087.cropped.jpg?itok=IC1TNXJ6 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/img_0087.cropped.jpg?itok=TWJTqWRn 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/img_0087.cropped.jpg?itok=6C0Lh6lG 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/img_0087.cropped.jpg?itok=UE2VuqRI 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/img_0087.cropped.jpg?itok=IC1TNXJ6 2x" type="image/jpeg" width="480"/>
<img alt="Bria Morgan" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/img_0087.cropped.jpg?itok=TWJTqWRn" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/brigitta-gough"> <h3 class="page-card__heading hwp-order-2">Brigitta Gough</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:brigitta_gough@hms.harvard.edu">brigitta_gough@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Research and Administrative Assistant</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/brigitta_photo.jpg?itok=e9WhTamJ 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/brigitta_photo.jpg?itok=COQv903z 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/brigitta_photo.jpg?itok=8UTRwhvp 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/brigitta_photo.jpg?itok=VpN4Lvip 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/brigitta_photo.jpg?itok=e9WhTamJ 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/brigitta_photo.jpg?itok=COQv903z 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/brigitta_photo.jpg?itok=8UTRwhvp 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/brigitta_photo.jpg?itok=VpN4Lvip 2x" type="image/jpeg" width="480"/>
<img alt="Brigitta Gough" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/brigitta_photo.jpg?itok=e9WhTamJ" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/burt-beckwith"> <h3 class="page-card__heading hwp-order-2">Burt Beckwith</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior Full Stack Web Developer</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/burt.beckwith.jpg?itok=S2zrKVIY 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/burt.beckwith.jpg?itok=aQQezQCs 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/burt.beckwith.jpg?itok=FkggTWre 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/burt.beckwith.jpg?itok=zZtLDdvU 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/burt.beckwith.jpg?itok=S2zrKVIY 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/burt.beckwith.jpg?itok=aQQezQCs 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/burt.beckwith.jpg?itok=FkggTWre 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/burt.beckwith.jpg?itok=zZtLDdvU 2x" type="image/jpeg" width="480"/>
<img alt="burt_beckwith" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/burt.beckwith.jpg?itok=S2zrKVIY" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/carlos-de-niz"> <h3 class="page-card__heading hwp-order-2">Carlos De Niz</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:Carlos_DeNiz@hms.harvard.edu">Carlos_DeNiz@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div> Research Fellow in Biomedical Informatics</div>
<div>MSc and PhD in Electrical Engineering from Texas Tech University</div>
</div>
</div>
<div class="page-card__description hwp-order-3 hwp-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden">  Working on analysis of variant data of patients with Autism

</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/carlos_hms_small.jpg?itok=aTP71TZ- 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/carlos_hms_small.jpg?itok=D5vWnT75 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/carlos_hms_small.jpg?itok=0JniV0Ze 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/carlos_hms_small.jpg?itok=bs4b14rN 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/carlos_hms_small.jpg?itok=aTP71TZ- 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/carlos_hms_small.jpg?itok=D5vWnT75 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/carlos_hms_small.jpg?itok=0JniV0Ze 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/carlos_hms_small.jpg?itok=bs4b14rN 2x" type="image/jpeg" width="480"/>
<img alt="N/A" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/carlos_hms_small.jpg?itok=aTP71TZ-" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/carlos-s%C3%A1ez"> <h3 class="page-card__heading hwp-order-2">Carlos Sáez</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:carlos_saez@hms.harvard.edu">carlos_saez@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Visiting Research Fellow at Harvard Medical School</div>
<div>Postdoctoral Researcher at Universitat Politècnica de València, Spain</div>
</div>
</div>
<div class="page-card__description hwp-order-3 hwp-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden">  PhD in Technologies for Health and Well-Being, MsC in Artificial Intelligence, Pattern Recognition and Digital Imaging. He is researcher at the Biomedical Data Science Lab (ITACA Institute) of the Universitat Politècnica de València (UPV) since 2006. His...

</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/img_0275_s2.jpg?itok=E9GsKD67 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/img_0275_s2.jpg?itok=GBw4fp76 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/img_0275_s2.jpg?itok=kYAk_ZrW 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/img_0275_s2.jpg?itok=xysADbTM 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/img_0275_s2.jpg?itok=E9GsKD67 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/img_0275_s2.jpg?itok=GBw4fp76 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/img_0275_s2.jpg?itok=kYAk_ZrW 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/img_0275_s2.jpg?itok=xysADbTM 2x" type="image/jpeg" width="480"/>
<img alt="Carlos Sáez" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/img_0275_s2.jpg?itok=E9GsKD67" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/cartik-k-saravanamuthu"> <h3 class="page-card__heading hwp-order-2">Cartik K Saravanamuthu, MS, MS, PhD</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:cartik_saravanamuthu@hms.harvard.edu">cartik_saravanamuthu@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Data Scientist in Biomedical Informatics</div>
</div>
</div>
<div class="page-card__description hwp-order-3 hwp-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden">  MS in Biomedical Engineering. MS, PhD in Computer Engineering. Projects include: Phelan McDermid Syndrome Data Network (PMS_DN) PTEN Hamartoma Tumor Syndrome (PHTS) Cure Sickle Cell Disease (CureSCD) National Institute of Health DataSTAGE

</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/cartik020315-2.jpeg?itok=W5qHgffn 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/cartik020315-2.jpeg?itok=HSDcbCFu 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/cartik020315-2.jpeg?itok=ojPowVsC 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/cartik020315-2.jpeg?itok=NcE91O7Z 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/cartik020315-2.jpeg?itok=W5qHgffn 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/cartik020315-2.jpeg?itok=HSDcbCFu 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/cartik020315-2.jpeg?itok=ojPowVsC 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/cartik020315-2.jpeg?itok=NcE91O7Z 2x" type="image/jpeg" width="480"/>
<img alt="Cartik K. Saravanamuthu" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/cartik020315-2.jpeg?itok=W5qHgffn" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/cassandra-perry"> <h3 class="page-card__heading hwp-order-2">Cassandra Perry, MS, CGC</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:Cassandra_Perry@hms.harvard.edu">Cassandra_Perry@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Project Manager at Harvard Medical School (2015 - 17)</div>
</div>
</div>
<div class="page-card__description hwp-order-3 hwp-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden">  Cassandra Perry holds a Master of Science degree in genetic counseling and is a Certified Genetic Counselor.

</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/new_profile_photo.jpg?itok=Td8gsc82 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/new_profile_photo.jpg?itok=cxVZJ7PC 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/new_profile_photo.jpg?itok=DtnSQhxt 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/new_profile_photo.jpg?itok=6zYLaRqm 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/new_profile_photo.jpg?itok=Td8gsc82 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/new_profile_photo.jpg?itok=cxVZJ7PC 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/new_profile_photo.jpg?itok=DtnSQhxt 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/new_profile_photo.jpg?itok=6zYLaRqm 2x" type="image/jpeg" width="480"/>
<img alt="Cassandra Perry" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/new_profile_photo.jpg?itok=Td8gsc82" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/claire-hassen-khodja"> <h3 class="page-card__heading hwp-order-2">Claire Hassen-Khodja, MD, MSc</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:claire_hassen-khodja@hms.harvard.edu">claire_hassen-khodja@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Visiting Research Associate at Harvard Medical School</div>
</div>
</div>
<div class="page-card__description hwp-order-3 hwp-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden">  Student Intern at Center For Biomedical Informatics MD Resident in Public Health, in France, specializing in biostatistics Research project at Avillach Lab: Phenotypic Identification of patients with Phelan-McDermid Syndrome in the Simons Simplex...

</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/0b4156f.jpg?itok=1uwZoaQB 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/0b4156f.jpg?itok=wVpiUiEl 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/0b4156f.jpg?itok=LIkIyyVJ 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/0b4156f.jpg?itok=CouCPlCW 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/0b4156f.jpg?itok=1uwZoaQB 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/0b4156f.jpg?itok=wVpiUiEl 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/0b4156f.jpg?itok=LIkIyyVJ 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/0b4156f.jpg?itok=CouCPlCW 2x" type="image/jpeg" width="480"/>
<img alt="Claire Hassen-Khodja" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/0b4156f.jpg?itok=1uwZoaQB" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/emmanuelle-sylvestre"> <h3 class="page-card__heading hwp-order-2">Emmanuelle Sylvestre, MD, MSc</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:Emmanuelle_Sylvestre@hms.harvard.edu">Emmanuelle_Sylvestre@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Visiting Research Associate at Harvard Medical School</div>
</div>
</div>
<div class="page-card__description hwp-order-3 hwp-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden">  MD resident in Public health, specializing in Biomedical informatics, working on SHRINE.

</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/es.jpg?itok=ubs_h_6w 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/es.jpg?itok=4E0wgw5F 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/es.jpg?itok=VvvsgRnd 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/es.jpg?itok=-d85M_oE 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/es.jpg?itok=ubs_h_6w 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/es.jpg?itok=4E0wgw5F 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/es.jpg?itok=VvvsgRnd 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/es.jpg?itok=-d85M_oE 2x" type="image/jpeg" width="480"/>
<img alt="Emmanuelle Syvelstre" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/es.jpg?itok=ubs_h_6w" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/ephi-sachs"> <h3 class="page-card__heading hwp-order-2">Ephi Sachs, MD</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:ephi79@gmail.com">ephi79@gmail.com</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Visiting Research Associate at Harvard Medical School (2013 - 15)</div>
</div>
</div>
<div class="page-card__description hwp-order-3 hwp-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden">  Medical Intern at Wolfson Medical Center, Holon, Israel Software Engineer at MedCPU

</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/59436_10150273296475319_2948834_n.jpg?itok=H_LSCm6x 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/59436_10150273296475319_2948834_n.jpg?itok=iBQKjS7O 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/59436_10150273296475319_2948834_n.jpg?itok=egqGmWy3 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/59436_10150273296475319_2948834_n.jpg?itok=3CrDYeMU 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/59436_10150273296475319_2948834_n.jpg?itok=H_LSCm6x 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/59436_10150273296475319_2948834_n.jpg?itok=iBQKjS7O 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/59436_10150273296475319_2948834_n.jpg?itok=egqGmWy3 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/59436_10150273296475319_2948834_n.jpg?itok=3CrDYeMU 2x" type="image/jpeg" width="480"/>
<img alt="Ephi Sachs, MD" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/59436_10150273296475319_2948834_n.jpg?itok=H_LSCm6x" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/gabor-korodi"> <h3 class="page-card__heading hwp-order-2">Gabor Korodi</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:gabor_korodi@hms.harvard.edu">gabor_korodi@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Sr. Software Developer</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/gabekorodi.jpeg?itok=2Y2P9v05 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/gabekorodi.jpeg?itok=zBXZUfS6 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/gabekorodi.jpeg?itok=Lg84MeqN 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/gabekorodi.jpeg?itok=qHBk8BVq 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/gabekorodi.jpeg?itok=2Y2P9v05 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/gabekorodi.jpeg?itok=zBXZUfS6 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/gabekorodi.jpeg?itok=Lg84MeqN 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/gabekorodi.jpeg?itok=qHBk8BVq 2x" type="image/jpeg" width="480"/>
<img alt="Gabor Korodi" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/gabekorodi.jpeg?itok=2Y2P9v05" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/haishuai-wang"> <h3 class="page-card__heading hwp-order-2">Haishuai Wang</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:Haishuai_Wang@hms.harvard.edu">Haishuai_Wang@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Research Fellow in Biomedical Informatics</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/screen_shot_2021-09-13_at_3.18.17_pm.png?itok=X-CkoL4f 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/screen_shot_2021-09-13_at_3.18.17_pm.png?itok=vFYCZvbd 2x" type="image/png" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/screen_shot_2021-09-13_at_3.18.17_pm.png?itok=bKc2sAys 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/screen_shot_2021-09-13_at_3.18.17_pm.png?itok=sge1PaUZ 2x" type="image/png" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/screen_shot_2021-09-13_at_3.18.17_pm.png?itok=X-CkoL4f 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/screen_shot_2021-09-13_at_3.18.17_pm.png?itok=vFYCZvbd 2x" type="image/png" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/screen_shot_2021-09-13_at_3.18.17_pm.png?itok=bKc2sAys 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/screen_shot_2021-09-13_at_3.18.17_pm.png?itok=sge1PaUZ 2x" type="image/png" width="480"/>
<img alt="haishuai wang" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/screen_shot_2021-09-13_at_3.18.17_pm.png?itok=X-CkoL4f" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/in%C3%A8s-krissaane"> <h3 class="page-card__heading hwp-order-2">Inès Krissaane</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:ines_krissaane@hms.harvard.edu">ines_krissaane@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Visiting MSc Student from Sorbonne Université</div>
</div>
</div>
<div class="page-card__description hwp-order-3 hwp-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden">  Working towards MSc in Data Science and Statistics

</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/photo.jpg?itok=DURI0bfh 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/photo.jpg?itok=v_jw0g1T 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/photo.jpg?itok=KGXTv3ky 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/photo.jpg?itok=HX-Wf6TA 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/photo.jpg?itok=DURI0bfh 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/photo.jpg?itok=v_jw0g1T 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/photo.jpg?itok=KGXTv3ky 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/photo.jpg?itok=HX-Wf6TA 2x" type="image/jpeg" width="480"/>
<img alt="Ines Krissaane" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/photo.jpg?itok=DURI0bfh" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/jason-stedman"> <h3 class="page-card__heading hwp-order-2">Jason Stedman</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:Jason_Stedman@hms.harvard.edu">Jason_Stedman@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Director of Software Engineering</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/jasonstedman.jpg?itok=ef9nGIxx 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/jasonstedman.jpg?itok=L-XeZ247 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/jasonstedman.jpg?itok=-OuRvCN5 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/jasonstedman.jpg?itok=oxRM9S6a 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/jasonstedman.jpg?itok=ef9nGIxx 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/jasonstedman.jpg?itok=L-XeZ247 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/jasonstedman.jpg?itok=-OuRvCN5 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/jasonstedman.jpg?itok=oxRM9S6a 2x" type="image/jpeg" width="480"/>
<img alt="Jason Stedman" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/jasonstedman.jpg?itok=ef9nGIxx" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/jaspreet-khanna"> <h3 class="page-card__heading hwp-order-2">Jaspreet Khanna</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior Data Analyst</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/screen_shot_2016-11-30_at_12.54.31_pm_01.png?itok=VU71O_5i 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/screen_shot_2016-11-30_at_12.54.31_pm_01.png?itok=-j6n43-- 2x" type="image/png" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/screen_shot_2016-11-30_at_12.54.31_pm_01.png?itok=LGwnDyyh 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/screen_shot_2016-11-30_at_12.54.31_pm_01.png?itok=pqPi8ihj 2x" type="image/png" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/screen_shot_2016-11-30_at_12.54.31_pm_01.png?itok=VU71O_5i 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/screen_shot_2016-11-30_at_12.54.31_pm_01.png?itok=-j6n43-- 2x" type="image/png" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/screen_shot_2016-11-30_at_12.54.31_pm_01.png?itok=LGwnDyyh 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/screen_shot_2016-11-30_at_12.54.31_pm_01.png?itok=pqPi8ihj 2x" type="image/png" width="480"/>
<img alt="jk3" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/screen_shot_2016-11-30_at_12.54.31_pm_01.png?itok=VU71O_5i" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/jeremy-r-easton-marks"> <h3 class="page-card__heading hwp-order-2">Jeremy R Easton-Marks </h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:Jeremy_Easton-Marks@hms.harvard.edu">Jeremy_Easton-Marks@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Web Application Developer (2015 - 17)</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/unnamed.jpg?itok=9A3lxxLY 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/unnamed.jpg?itok=OJKvk49n 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/unnamed.jpg?itok=DgVtfocP 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/unnamed.jpg?itok=_ws2PqVb 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/unnamed.jpg?itok=9A3lxxLY 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/unnamed.jpg?itok=OJKvk49n 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/unnamed.jpg?itok=DgVtfocP 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/unnamed.jpg?itok=_ws2PqVb 2x" type="image/jpeg" width="480"/>
<img alt="Jeremy R. Easton-Marks" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/unnamed.jpg?itok=9A3lxxLY" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/melinda-rushing"> <h3 class="page-card__heading hwp-order-2">Melinda Rushing</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Visiting Research Fellow at Harvard Medical School</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/2025-03/IMG_2930.jpeg?itok=7yueuy1N 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/2025-03/IMG_2930.jpeg?itok=3dk2kx0X 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/2025-03/IMG_2930.jpeg?itok=hTAN6LoT 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/2025-03/IMG_2930.jpeg?itok=9ViUdhMW 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/2025-03/IMG_2930.jpeg?itok=7yueuy1N 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/2025-03/IMG_2930.jpeg?itok=3dk2kx0X 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/2025-03/IMG_2930.jpeg?itok=hTAN6LoT 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/2025-03/IMG_2930.jpeg?itok=9ViUdhMW 2x" type="image/jpeg" width="480"/>
<img alt="Melinda Rushing" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/2025-03/IMG_2930.jpeg?itok=7yueuy1N" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/rachel-payne"> <h3 class="page-card__heading hwp-order-2">Rachel  Payne </h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Application Support Specialist II</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/fullsizerender.jpg?itok=REOXqpPr 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/fullsizerender.jpg?itok=md53JcAW 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/fullsizerender.jpg?itok=Vdrpmmgt 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/fullsizerender.jpg?itok=AHtZ-h0r 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/fullsizerender.jpg?itok=REOXqpPr 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/fullsizerender.jpg?itok=md53JcAW 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/fullsizerender.jpg?itok=Vdrpmmgt 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/fullsizerender.jpg?itok=AHtZ-h0r 2x" type="image/jpeg" width="480"/>
<img alt="Rachel Payne" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/fullsizerender.jpg?itok=REOXqpPr" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/simran-makwana"> <h3 class="page-card__heading hwp-order-2">Simran Makwana, MS</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Bioinformatics Research Engagement Manager</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/simran_headshot.jpg?itok=w5ab4S7t 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/simran_headshot.jpg?itok=dLAjyrv2 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/simran_headshot.jpg?itok=PfshZHXt 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/simran_headshot.jpg?itok=1o2Dt3Wi 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/simran_headshot.jpg?itok=w5ab4S7t 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/simran_headshot.jpg?itok=dLAjyrv2 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/simran_headshot.jpg?itok=PfshZHXt 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/simran_headshot.jpg?itok=1o2Dt3Wi 2x" type="image/jpeg" width="480"/>
<img alt="makwana" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/simran_headshot.jpg?itok=w5ab4S7t" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="paragraphs-item__last paragraphs-item">
<div class="hwp-images hwp-py-24 lg:hwp-py-64 hwp-bg-light-core">
<div class="hwp-container hwp-container-px hwp-images__items">
<div class="hwp-image-layout hwp-image-layout--three-equal hwp-gap-16 xl:hwp-gap-24">
<div class="hwp-image-layout__region">
<div class="hwp-image hwp-image--caption-stacked sm:hwp-h-full">
<div class="hwp-image-item-heading">
</div>
<div class="hwp-flex hwp-flex-col hwp-gap-32">
<figure class="hwp-image__figure hwp-relative hwp-overflow-hidden sm:hwp-mb-[1px]">
<div class="hwp-image__media hwp-overflow-hidden sm:hwp-h-full">
<div class="field field--name-field-hwp-image field--type-entity-reference field--label-hidden"><div class="hwp-media hwp-media--ult">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <img alt="AvillachLabMembers_LabOutingAugust2018" height="1889" loading="lazy" src="/sites/g/files/omnuum4001/files/avillach_lab/files/avillachlab_august2018.png" width="3959"/>
</div>
</div>
</div>
</div>
</figure>
</div>
</div>
</div>
<div class="hwp-image-layout__region">
<div class="hwp-image hwp-image--caption-stacked sm:hwp-h-full">
<div class="hwp-image-item-heading">
</div>
<div class="hwp-flex hwp-flex-col hwp-gap-32">
<figure class="hwp-image__figure hwp-relative hwp-overflow-hidden sm:hwp-mb-[1px]">
<div class="hwp-image__media hwp-overflow-hidden sm:hwp-h-full">
<div class="field field--name-field-hwp-image field--type-entity-reference field--label-hidden"><div class="hwp-media hwp-media--ult">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <img alt="AvillachLab-PartyDay-06292017" height="1198" loading="lazy" src="/sites/g/files/omnuum4001/files/avillach_lab/files/img-20170629-wa0018.jpg" width="1600"/>
</div>
</div>
</div>
</div>
</figure>
</div>
</div>
</div>
<div class="hwp-image-layout__region">
<div class="hwp-image hwp-image--caption-stacked sm:hwp-h-full">
<div class="hwp-image-item-heading">
</div>
<div class="hwp-flex hwp-flex-col hwp-gap-32">
<figure class="hwp-image__figure hwp-relative hwp-overflow-hidden sm:hwp-mb-[1px]">
<div class="hwp-image__media hwp-overflow-hidden sm:hwp-h-full">
<div class="field field--name-field-hwp-image field--type-entity-reference field--label-hidden"><div class="hwp-media hwp-media--ult">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <img alt="Group picture of Avillach Lab members" height="429" loading="lazy" src="/sites/g/files/omnuum4001/files/avillach_lab/files/avillachlab_01.jpg" width="800"/>
</div>
</div>
</div>
</div>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/caslogin -->
<h1>HarvardKey - Sign In</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/caslogin'>avillach-lab.hms.harvard.edu/caslogin</a></div>
<div class="content">
<div class="applogin-banner">
<div class="applogin-background"></div>
<div class="applogin-container">
<h1>
<span class="applogin-app-title">
                    Connecting to</span>
<div class="applogin-app-logo">
<img alt="HarvardKey" class="logo harvard_casprod1_1" src="https://ok3static.oktacdn.com/assets/img/logos/default.6770228fb0dab49a1695ef440a5279bb.png"/></div>
</h1>
<p>Sign in with your account to access HarvardKey</p>
</div>
</div>
<style nonce="PqEU9ICB-VdUEjQJgf-Cow" type="text/css">
    .noscript-msg {
        background-color: #fff;
        border-color: #ddd #ddd #d8d8d8;
        box-shadow:0 2px 0 rgba(175, 175, 175, 0.12);
        text-align: center;
        width: 398px;
        min-width: 300px;
        margin: 200px auto;
        border-radius: 3px;
        border-width: 1px;
        border-style: solid;
    }

    .noscript-content {
        padding: 42px;
    }

    .noscript-content h2 {
        padding-bottom: 20px;
    }

    .noscript-content h1 {
        padding-bottom: 25px;
    }

    .noscript-content a {
        background: transparent;
        box-shadow: none;
        display: table-cell;
        vertical-align: middle;
        width: 314px;
        height: 50px;
        line-height: 36px;
        color: #fff;
        background: linear-gradient(#007dc1, #0073b2), #007dc1;
        border: 1px solid;
        border-color: #004b75;
        border-bottom-color: #00456a;
        box-shadow: rgba(0, 0, 0, 0.15) 0 1px 0, rgba(255, 255, 255, 0.1) 0 1px 0 0 inset;
        -webkit-border-radius: 3px;
        border-radius: 3px;
    }

    .noscript-content a:hover {
        background: #007dc1;
        cursor: hand;
        text-decoration: none;
    }
 </style><noscript>
<div class="noscript-msg" id="noscript-msg">
<div class="noscript-content">
<h2>Javascript is required</h2>
<h1>Javascript is disabled on your browser.&amp;nbspPlease enable Javascript and refresh this page.</h1>
<a class="tb--button" href=".">Refresh</a>
</div>
</div>
</noscript>
<div id="signin-container"></div>
<div class="auth-container main-container hide" id="okta-sign-in">
<div class="unsupported-message hide" id="unsupported-onedrive">
<h2 class="o-form-head">Your OneDrive version is not supported</h2>
<p>Upgrade now by installing the OneDrive for Business Next Generation Sync Client to login to Okta</p>
<a class="button button-primary tb--button" href="https://support.okta.com/help/articles/Knowledge_Article/Upgrading-to-OneDrive-for-Business-Next-Generation-Sync-Client" target="_blank">
          Learn how to upgrade</a>
</div>
<div class="unsupported-message hide" id="unsupported-cookie">
<h2 class="o-form-head">Cookies are required</h2>
<p>Cookies are disabled on your browser. Please enable Cookies and refresh this page.</p>
<a class="button button-primary tb--button" href="." target="_blank">
              Refresh</a>
</div>
</div>
</div><hr>
<!-- Source: avillach-lab.hms.harvard.edu/sites/g/files/omnuum4001/files/avillach/lab/files/longwoodmap.pdf -->
<!-- Source: avillach-lab.hms.harvard.edu/publications -->
<h1>Publications | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/publications'>avillach-lab.hms.harvard.edu/publications</a></div>
<main aria-label="Publications" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="search-page search-page--no-filters hwp-listing-page hwp-bibcite_reference">
<div class="hwp-page-title hwp-bg-dark-base" lang="en">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Publications
    </h1>
</div>
</div>
<div class="hwp-section-heading" data-component-name="section-heading">
<div class="hwp-section-heading__container hwp-container-px hwp-container">
<div class="hwp-section-heading__description hwp-text-block">
</div>
</div>
</div>
<div class="search-page__columns hwp-container hwp-container-px">
<div class="search-page__results">
<div class="search-page__search-form">
<form accept-charset="UTF-8" action="/publications" class="hwp-search-form-listing" data-drupal-selector="hwp-search-form-listing" id="hwp-search-form-listing" method="post">
<div class="hwp-form-type-search hwp-form-item js-form-item form-item form-type-search js-form-type-search form-item-search js-form-item-search form-no-label">
<label class="visually-hidden" for="edit-search">Search Within Results</label>
<input class="form-search" data-drupal-selector="edit-search" id="edit-search" maxlength="128" name="search" size="60" type="search" value=""/>
</div>
<button class="hwp-button hwp-button--small button js-form-submit form-submit" data-drupal-selector="edit-submit" data-twig-suggestion="header_site_search_submit" id="edit-submit" name="op" type="submit" value="Search Within Results">
<span class="hwp-button__text">Search Within Results</span>
<span aria-hidden="true" class="material-icon">search</span>
</button>
<input autocomplete="off" data-drupal-selector="form-imypagg73lxskojfuiqljy0prljdf1zrgbqht0wzqqg" name="form_build_id" type="hidden" value="form-IMyPAgg73LxsKOjFuiQLJy0PRLJdF1zrgBQht0WzQQg"/>
<input data-drupal-selector="edit-hwp-search-form-listing" name="form_id" type="hidden" value="hwp_search_form_listing"/>
</form>
</div>
<div class="views-element-container">
<div class="js-view-dom-id-ddcb69bd5855ccc9e781df57ec03f7ba1cb5ed7530fbfada62e7b414010061bf">
<div class="search-page__search-header">
<div class="hwp-hidden lg:hwp-block search-page__results-summary">
        100 results 
      </div>
<button aria-expanded="false" class="search-page__filters-trigger" data-filters-trigger="">
<span>Show filters</span>
<span aria-hidden="true" class="material-icon">filter_alt</span>
</button>
<form accept-charset="UTF-8" action="/node/1774776" class="views-exposed-form bef-exposed-form" data-bef-auto-submit="" data-bef-auto-submit-delay="500" data-bef-auto-submit-full-form="" data-bef-auto-submit-minimum-length="3" data-drupal-selector="views-exposed-form-hwp-search-search-page" id="views-exposed-form-hwp-search-search-page" method="get">
<input data-drupal-selector="edit-search" name="search" type="hidden" value=""/>
<div class="js-form-item form-item hwp-form-item js-form-type-select hwp-form-type-select form-item-sort-bef-combine js-form-item-sort-bef-combine">
<label for="edit-sort-bef-combine">Sort by</label>
<select class="form-select" data-drupal-selector="edit-sort-bef-combine" id="edit-sort-bef-combine" name="sort_bef_combine"><option selected="selected" value="hwp_bibcite_reference_year--desc">Year of Publication</option><option value="title--asc">Alphabetical A-Z</option></select>
<span aria-hidden="true" class="material-icon">sort</span>
</div>
<div class="form-actions js-form-wrapper form-wrapper" data-drupal-selector="edit-actions" id="edit-actions"><input class="js-hide button js-form-submit form-submit" data-bef-auto-submit-click="" data-drupal-selector="edit-submit-hwp-search" id="edit-submit-hwp-search" type="submit" value="Apply"/>
</div>
</form>
</div>
<div class="lg:hwp-hidden search-page__results-summary">
    100 results 
  </div>
<div class="search-page__result-items search-page__result-items--">
<div class="hwp-citations-list__downloads">
<div class="button-dropdown" data-component-name="button-dropdown">
<button class="hwp-text-link hwp-text-link--icon-right" id="citations_download">
<span class="hwp-text-link__text">    Download 100 citations  </span>
<span aria-hidden="true" class="material-icon">download</span>
</button>
<ul aria-labelledby="citations_download" class="button-dropdown__items" role="menu">
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=bibtex">BibTeX</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=endnote8">EndNote X3 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=endnote7">EndNote 7 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=tagged">Endnote tagged</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=marc">Marc</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=pubmed_id">PubMedId</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=ris">RIS</a>
</li>
</ul>
</div>
</div>
<div id="">
<div class="hwp-page-list__items">
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">2025</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Alkuraya I, Almansa AS, Eleonu A, Avillach P, Poduri A, Srivastava S. <a href="/publication/use-computational-phenotypes-predicting-genetic-subgroups-cerebral-palsy">Use of Computational Phenotypes for Predicting Genetic Subgroups of Cerebral Palsy.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2025; PMID: 39990554</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Alkuraya I, Almansa AS, Eleonu A, Avillach P, Poduri A, Srivastava S. <a href="/publication/use-computational-phenotypes-predicting-genetic-subgroups-cerebral-palsy">Use of Computational Phenotypes for Predicting Genetic Subgroups of Cerebral Palsy.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2025; PMID: 39990554</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-983783844" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-983783844">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>INTRODUCTION: Emerging evidence suggests that 20-30% of cases of cerebral palsy (CP) may have a genetic cause. Our group previously identified subsets of patients with CP or CP-masquerading conditions who warrant genetic testing, including those with regression or progressive neurological symptoms (CP masqueraders) and those without any known risk factors for CP (cryptogenic CP). Recognition of these subgroups in clinical settings remains challenging.</p><p>METHODS: To address this challenge, we developed and evaluated a computational phenotyping approach using ICD- 9/ICD-10 billing codes to automatically identify patients with unexplained CP or CP-masquerading conditions who may benefit from genetic testing. We applied this computational phenotyping approach to a cohort of 250 participants from the Boston Children's Hospital CP Sequencing Study, aimed at identifying genetic causes in CP and CP-masquerading conditions.</p><p>RESULTS: Manual review served as the gold standard, identifying 8% as CP masqueraders, 42% as cryptogenic CP, and 50% as non-cryptogenic CP. Computational phenotyping based on ICD-9/10 codes achieved a sensitivity of 95%, specificity of 72%, positive predictive value of 77%, and negative predictive value of 94% in identifying cases warranting genetic testing.</p><p>CONCLUSIONS: Our findings demonstrate the feasibility of using computational phenotyping to identify patients with CP or CP- masquerading conditions who warrant genetic testing. Further studies are needed to evaluate the effectiveness and real-world application of this tool in larger healthcare systems. Nonetheless, the computational phenotyping approach holds promise as a possible clinical decision support that could be integrated into electronic health record systems, enhancing clinical workflows and facilitating actionable genetic diagnoses.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yang D, Zhou D, Cai S, Gan Z, Pencina M, Avillach P, Cai T, Hong C. <a href="/publication/robust-automated-harmonization-heterogeneous-data-through-ensemble-machine-learning">Robust Automated Harmonization of Heterogeneous Data Through Ensemble Machine Learning: Algorithm Development and Validation Study.</a>.</div><div class="csl-right-inline">JMIR medical informatics. 2025;13:e54133. PMID: 39844378</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yang D, Zhou D, Cai S, Gan Z, Pencina M, Avillach P, Cai T, Hong C. <a href="/publication/robust-automated-harmonization-heterogeneous-data-through-ensemble-machine-learning">Robust Automated Harmonization of Heterogeneous Data Through Ensemble Machine Learning: Algorithm Development and Validation Study.</a>.</div><div class="csl-right-inline">JMIR medical informatics. 2025;13:e54133. PMID: 39844378</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-1907437921" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-1907437921">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Cohort studies contain rich clinical data across large and diverse patient populations and are a common source of observational data for clinical research. Because large scale cohort studies are both time and resource intensive, one alternative is to harmonize data from existing cohorts through multicohort studies. However, given differences in variable encoding, accurate variable harmonization is difficult.</p><p>OBJECTIVE: We propose SONAR (Semantic and Distribution-Based Harmonization) as a method for harmonizing variables across cohort studies to facilitate multicohort studies.</p><p>METHODS: SONAR used semantic learning from variable descriptions and distribution learning from study participant data. Our method learned an embedding vector for each variable and used pairwise cosine similarity to score the similarity between variables. This approach was built off 3 National Institutes of Health cohorts, including the Cardiovascular Health Study, the Multi-Ethnic Study of Atherosclerosis, and the Women's Health Initiative. We also used gold standard labels to further refine the embeddings in a supervised manner.</p><p>RESULTS: The method was evaluated using manually curated gold standard labels from the 3 National Institutes of Health cohorts. We evaluated both the intracohort and intercohort variable harmonization performance. The supervised SONAR method outperformed existing benchmark methods for almost all intracohort and intercohort comparisons using area under the curve and top-k accuracy metrics. Notably, SONAR was able to significantly improve harmonization of concepts that were difficult for existing semantic methods to harmonize.</p><p>CONCLUSIONS: SONAR achieves accurate variable harmonization within and between cohort studies by harnessing the complementary strengths of semantic learning and variable distribution learning.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Menezes MCS, Hoffmann AF, Tan ALM, Nalbandyan M, Omenn GS, Mazzotti DR, Hernández-Arango A, Visweswaran S, Venkatesh S, Mandl KD, Bourgeois FT, Lee JWK, Makmur A, Hanauer DA, Semanik MG, Kerivan LT, Hill T, Forero J, Restrepo C, Vigna M, Ceriana P, Abu-El-Rub N, Avillach P, Bellazzi R, Callaci T, Gutiérrez-Sacristán A, Malovini A, Mathew JP, Morris M, Murthy VL, Buonocore TM, Parimbelli E, Patel LP, Saez C, Samayamuthu MJ, Thompson JA, Tibollo V, Xia Z, Kohane IS, Records C for CC of C 19 by EH. <a href="/publication/potential-generative-pre-trained-transformer-4-gpt-4-analyse-medical-notes-three">The potential of Generative Pre-trained Transformer 4 (GPT-4) to analyse medical notes in three different languages: a retrospective model-evaluation study.</a>.</div><div class="csl-right-inline">The Lancet. Digital health. 2025;7(1):e35-e43. PMID: 39722251</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Menezes MCS, Hoffmann AF, Tan ALM, Nalbandyan M, Omenn GS, Mazzotti DR, Hernández-Arango A, Visweswaran S, Venkatesh S, Mandl KD, Bourgeois FT, Lee JWK, Makmur A, Hanauer DA, Semanik MG, Kerivan LT, Hill T, Forero J, Restrepo C, Vigna M, Ceriana P, Abu-El-Rub N, Avillach P, Bellazzi R, Callaci T, Gutiérrez-Sacristán A, Malovini A, Mathew JP, Morris M, Murthy VL, Buonocore TM, Parimbelli E, Patel LP, Saez C, Samayamuthu MJ, Thompson JA, Tibollo V, Xia Z, Kohane IS, Records C for CC of C 19 by EH. <a href="/publication/potential-generative-pre-trained-transformer-4-gpt-4-analyse-medical-notes-three">The potential of Generative Pre-trained Transformer 4 (GPT-4) to analyse medical notes in three different languages: a retrospective model-evaluation study.</a>.</div><div class="csl-right-inline">The Lancet. Digital health. 2025;7(1):e35-e43. PMID: 39722251</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-303159569" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-303159569">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Patient notes contain substantial information but are difficult for computers to analyse due to their unstructured format. Large-language models (LLMs), such as Generative Pre-trained Transformer 4 (GPT-4), have changed our ability to process text, but we do not know how effectively they handle medical notes. We aimed to assess the ability of GPT-4 to answer predefined questions after reading medical notes in three different languages.</p><p>METHODS: For this retrospective model-evaluation study, we included eight university hospitals from four countries (ie, the USA, Colombia, Singapore, and Italy). Each site submitted seven de-identified medical notes related to seven separate patients to the coordinating centre between June 1, 2023, and Feb 28, 2024. Medical notes were written between Feb 1, 2020, and June 1, 2023. One site provided medical notes in Spanish, one site provided notes in Italian, and the remaining six sites provided notes in English. We included admission notes, progress notes, and consultation notes. No discharge summaries were included in this study. We advised participating sites to choose medical notes that, at time of hospital admission, were for patients who were male or female, aged 18-65 years, had a diagnosis of obesity, had a diagnosis of COVID-19, and had submitted an admission note. Adherence to these criteria was optional and participating sites randomly chose which medical notes to submit. When entering information into GPT-4, we prepended each medical note with an instruction prompt and a list of 14 questions that had been chosen a priori. Each medical note was individually given to GPT-4 in its original language and in separate sessions; the questions were always given in English. At each site, two physicians independently validated responses by GPT-4 and responded to all 14 questions. Each pair of physicians evaluated responses from GPT-4 to the seven medical notes from their own site only. Physicians were not masked to responses from GPT-4 before providing their own answers, but were masked to responses from the other physician.</p><p>FINDINGS: We collected 56 medical notes, of which 42 (75%) were in English, seven (13%) were in Italian, and seven (13%) were in Spanish. For each medical note, GPT-4 responded to 14 questions, resulting in 784 responses. In 622 (79%, 95% CI 76-82) of 784 responses, both physicians agreed with GPT-4. In 82 (11%, 8-13) responses, only one physician agreed with GPT-4. In the remaining 80 (10%, 8-13) responses, neither physician agreed with GPT-4. Both physicians agreed with GPT-4 more often for medical notes written in Spanish (86 [88%, 95% CI 79-93] of 98 responses) and Italian (82 [84%, 75-90] of 98 responses) than in English (454 [77%, 74-80] of 588 responses).</p><p>INTERPRETATION: The results of our model-evaluation study suggest that GPT-4 is accurate when analysing medical notes in three different languages. In the future, research should explore how LLMs can be integrated into clinical workflows to maximise their use in health care.</p><p>FUNDING: None.</p></div>
</div>
</div>
</div>
</div>
</div>
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">2024</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gutiérrez-Sacristán A, Makwana S, Dionne A, Mahanta S, Dyer KJ, Serrano F, Watrin C, Pages P, Mousavi S, Degala A, Lyons J, Pillion D, Zachariasse JM, Shekerdemian LS, Truong DT, Newburger JW, Avillach P. <a href="/publication/development-and-validation-open-source-pipeline-automatic-population-case-report-forms">Development and validation of an open-source pipeline for automatic population of case report forms from electronic health records: a pediatric multi-center prospective study.</a>.</div><div class="csl-right-inline">EBioMedicine. 2024;108:105337. PMID: 39288532</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gutiérrez-Sacristán A, Makwana S, Dionne A, Mahanta S, Dyer KJ, Serrano F, Watrin C, Pages P, Mousavi S, Degala A, Lyons J, Pillion D, Zachariasse JM, Shekerdemian LS, Truong DT, Newburger JW, Avillach P. <a href="/publication/development-and-validation-open-source-pipeline-automatic-population-case-report-forms">Development and validation of an open-source pipeline for automatic population of case report forms from electronic health records: a pediatric multi-center prospective study.</a>.</div><div class="csl-right-inline">EBioMedicine. 2024;108:105337. PMID: 39288532</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-613921253" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-613921253">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Clinical trials and registry studies are essential for advancing research and developing novel treatments. However, these studies rely on manual entry of thousands of variables for each patient. Repurposing real-world data can significantly simplify the data collection, reduce transcription errors, and make the data entry process more efficient, consistent, and cost-effective.</p><p>METHODS: We developed an open-source computational pipeline to collect laboratory and medication information from the electronic health record (EHR) data and populate case report forms. The pipeline was developed and validated with data from two independent pediatric hospitals in the US as part of the Long-terM OUtcomes after Multisystem Inflammatory Syndrome In Children (MUSIC) study. Our pipeline allowed the completion of two of the most time-consuming forms. We compared automatically extracted results with manually entered values in one hospital and applied the pipeline to a second hospital, where the output served as the primary data source for case report forms.</p><p>FINDINGS: We extracted and populated 51,845 laboratory and 4913 medication values for 159 patients in two hospitals participating in a prospective pediatric study. We evaluated pipeline performance against data for 104 patients manually entered by clinicians in one of the hospitals. The highest concordance was found during patient hospitalization, with 91.59% of the automatically extracted laboratory and medication values corresponding with the manually entered values. In addition to the successfully populated values, we identified an additional 13,396 laboratory and 567 medication values of interest for the study.</p><p>INTERPRETATION: The automatic data entry of laboratory and medication values during admission is feasible and has a high concordance with the manually entered data. By implementing this proof of concept, we demonstrate the quality of automatic data extraction and highlight the potential of secondary use of EHR data to advance medical science by improving data entry efficiency and expediting clinical research.</p><p>FUNDING: NIH Grant 1OT3HL147154-01, U24HL135691, UG1HL135685.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Wack M, Hassen-Khodja C, McDuffie MT, Bliss G, Horn EJ, Kothari C, McLarney B, Davis R, Hanson K, O’Boyle M, Betancur C, Avillach P. <a href="/publication/phenome-wide-profiling-identifies-genotype-phenotype-associations-phelan-mcdermid">Phenome-wide profiling identifies genotype-phenotype associations in Phelan-McDermid syndrome using family-sourced data from an international registry.</a>.</div><div class="csl-right-inline">Molecular autism. 2024;15(1):40. PMID: 39350236</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Wack M, Hassen-Khodja C, McDuffie MT, Bliss G, Horn EJ, Kothari C, McLarney B, Davis R, Hanson K, O’Boyle M, Betancur C, Avillach P. <a href="/publication/phenome-wide-profiling-identifies-genotype-phenotype-associations-phelan-mcdermid">Phenome-wide profiling identifies genotype-phenotype associations in Phelan-McDermid syndrome using family-sourced data from an international registry.</a>.</div><div class="csl-right-inline">Molecular autism. 2024;15(1):40. PMID: 39350236</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-1645799944" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-1645799944">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Phelan-McDermid syndrome (PMS) is a rare neurodevelopmental disorder caused by 22q13 deletions that include the SHANK3 gene or pathogenic sequence variants in SHANK3. It is characterized by global developmental delay, intellectual disability, speech impairment, autism spectrum disorder, and hypotonia; other variable features include epilepsy, brain and renal malformations, and mild dysmorphic features. Here, we conducted genotype-phenotype correlation analyses using the PMS International Registry, a family-driven registry that compiles clinical data in the form of family-reported outcomes and family-sourced genetic test results.</p><p>METHODS: Data from the registry were harmonized and integrated into the i2b2/tranSMART clinical and genomics data warehouse. We gathered information from 401 individuals with 22q13 deletions including SHANK3 (n = 350, ranging in size from 10 kb to 9.1 Mb) or pathogenic or likely pathogenic SHANK3 sequence variants (n = 51), and used regression models with deletion size as a potential predictor of clinical outcomes for 328 phenotypes.</p><p>RESULTS: Our results showed that increased deletion size was significantly associated with delay in gross and fine motor acquisitions, a spectrum of conditions related to poor muscle tone, renal malformations, mild dysmorphic features (e.g., large fleshy hands, sacral dimple, dysplastic toenails, supernumerary teeth), lymphedema, congenital heart defects, and more frequent neuroimaging abnormalities and infections. These findings indicate that genes upstream of SHANK3 also contribute to some of the manifestations of PMS in individuals with larger deletions. We also showed that self-help skills, verbal ability and a range of psychiatric diagnoses (e.g., autism, ADHD, anxiety disorder) were more common among individuals with smaller deletions and SHANK3 variants.</p><p>LIMITATIONS: Some participants were tested with targeted 22q microarrays rather than genome-wide arrays, and karyotypes were unavailable in many cases, thus precluding the analysis of the effect of other copy number variants or chromosomal rearrangements on the phenotype.</p><p>CONCLUSIONS: This is the largest reported case series of individuals with PMS. Overall, we demonstrate the feasibility of using data from a family-sourced registry to conduct genotype-phenotype analyses in rare genetic disorders. We replicate and strengthen previous findings, and reveal novel associations between larger 22q13 deletions and congenital heart defects, neuroimaging abnormalities and recurrent infections.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hernández-Agosto J, Santiago-Quiñones DI, García-Carmona LM, Gutiérrez-Sacristán A. <a href="/publication/substance-use-disorders-and-disparities-hispanic-and-non-hispanic-older-adults">Substance use disorders and disparities in hispanic and non-hispanic older adults.</a>.</div><div class="csl-right-inline">Drug and alcohol dependence. 2024;261:111357. PMID: 38896947</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hernández-Agosto J, Santiago-Quiñones DI, García-Carmona LM, Gutiérrez-Sacristán A. <a href="/publication/substance-use-disorders-and-disparities-hispanic-and-non-hispanic-older-adults">Substance use disorders and disparities in hispanic and non-hispanic older adults.</a>.</div><div class="csl-right-inline">Drug and alcohol dependence. 2024;261:111357. PMID: 38896947</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2110352893" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-2110352893">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: The global longevity revolution increased the older adult population, posing unique health and economic challenges with implications for healthcare, especially substance use disorders (SUD).</p><p>METHODS: This was a retrospective cohort study of United States older adults, Hispanic and non-Hispanic, who got at least one mental and/or behavioral disorder diagnosis between 2017 and 2021 at age 65 or older. SUD prevalence, prescription frequency changes over time, and comorbidities associated with each medication were compared.</p><p>RESULTS: Electronic health records for 356,133 older adults (110,236 Hispanics and 245,897 non-Hispanics) were analyzed. Notably, 79 % of Hispanics fell below the 100 % federal poverty level, compared to 60 % of non-Hispanics (P&lt;.001). Non-Hispanics also had significantly more average encounters (P=.003) and diagnoses (P&lt;.001). Regression analysis on alcohol-related disorders indicated that the odd ratios of being male (OR=2.93, P&lt;.000), and having low income (OR=1.62, P&lt;.000), increase the odds for this SUD, while being Hispanic and primarily speaking Spanish decreases the odds for all SUDs considered in this study.</p><p>CONCLUSIONS: This cohort study revealed significant disparities related to social determinants of health between Hispanic and non-Hispanic older adults and emphasizes the need for continuous surveillance of older adults as with SUDs. Differences in comorbidity patterns imply distinct risk factors within each population, influenced by demographic-specific elements. Recognizing these variations is essential for tailoring culturally sensitive prevention, intervention, and treatment strategies to each population's unique needs.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Gutierrez A, Network UD, Kobren SN, Avillach P. <a href="/publication/varppud-variant-post-prioritization-developed-undiagnosed-genetic-disorders">VarPPUD: Variant post prioritization developed for undiagnosed genetic disorders.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2024; PMID: 38699371</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Gutierrez A, Network UD, Kobren SN, Avillach P. <a href="/publication/varppud-variant-post-prioritization-developed-undiagnosed-genetic-disorders">VarPPUD: Variant post prioritization developed for undiagnosed genetic disorders.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2024; PMID: 38699371</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-551639828" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-551639828">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>Rare and ultra-rare genetic conditions are estimated to impact nearly 1 in 17 people worldwide, yet accurately pinpointing the diagnostic variants underlying each of these conditions remains a formidable challenge. Because comprehensive,  functional assessment of all possible genetic variants is infeasible, clinicians instead consider  variant pathogenicity predictions to distinguish plausibly disease-causing from benign variants across the genome. However, in the most difficult undiagnosed cases, such as those accepted to the Undiagnosed Diseases Network (UDN), existing pathogenicity predictions cannot reliably discern true etiological variant(s) from other deleterious candidate variants that were prioritized through N-of-1 efforts. Pinpointing the disease-causing variant from a pool of plausible candidates remains a largely manual effort requiring extensive clinical workups, functional and experimental assays, and eventual identification of genotype- and phenotype-matched individuals. Here, we introduce VarPPUD, a tool trained on prioritized variants from UDN cases, that leverages gene-, amino acid-, and nucleotide-level features to discern pathogenic variants from other deleterious variants that are unlikely to be confirmed as disease relevant. VarPPUD achieves a cross-validated accuracy of 79.3% and precision of 77.5% on a held-out subset of uniquely challenging UDN cases, respectively representing an average 18.6% and 23.4% improvement over nine traditional pathogenicity prediction approaches on this task. We validate VarPPUD's ability to discriminate likely from unlikely pathogenic variants on synthetic, GAN-generated candidate variants as well. Finally, we show how VarPPUD can be probed to evaluate each input feature's importance and contribution toward prediction-an essential step toward understanding the distinct characteristics of newly-uncovered disease-causing variants.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hutch MR, Son J, Le TT, Hong C, Wang X, Abad ZSH, Morris M, Gutiérrez-Sacristán A, Klann JG, Spiridou A, Batugo A, Bellazzi R, Benoit V, Bonzel CL, Bryant WA, Chiudinelli L, Cho K, Das P, González TG, Hanauer DA, Henderson DW, Ho YL, Loh NHW, Makoudjou A, Makwana S, Malovini A, Moal B, Mowery DL, Neuraz A, Samayamuthu MJ, Vidorreta FJS, Schriver ER, Schubert P, Talbert J, Tan ALM, Tan BWL, Tan BWQ, Tibollo V, Tippman P, Verdy G, Yuan W, Avillach P, Gehlenborg N, Omenn GS, EHR C for CC of C 19 by, Visweswaran S, Cai T, Luo Y, Xia Z. <a href="/publication/neurological-diagnoses-hospitalized-covid-19-patients-associated-adverse-outcomes">Neurological diagnoses in hospitalized COVID-19 patients associated with adverse outcomes: A multinational cohort study.</a>.</div><div class="csl-right-inline">PLOS digital health. 2024;3(4):e0000484. PMID: 38620037</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hutch MR, Son J, Le TT, Hong C, Wang X, Abad ZSH, Morris M, Gutiérrez-Sacristán A, Klann JG, Spiridou A, Batugo A, Bellazzi R, Benoit V, Bonzel CL, Bryant WA, Chiudinelli L, Cho K, Das P, González TG, Hanauer DA, Henderson DW, Ho YL, Loh NHW, Makoudjou A, Makwana S, Malovini A, Moal B, Mowery DL, Neuraz A, Samayamuthu MJ, Vidorreta FJS, Schriver ER, Schubert P, Talbert J, Tan ALM, Tan BWL, Tan BWQ, Tibollo V, Tippman P, Verdy G, Yuan W, Avillach P, Gehlenborg N, Omenn GS, EHR C for CC of C 19 by, Visweswaran S, Cai T, Luo Y, Xia Z. <a href="/publication/neurological-diagnoses-hospitalized-covid-19-patients-associated-adverse-outcomes">Neurological diagnoses in hospitalized COVID-19 patients associated with adverse outcomes: A multinational cohort study.</a>.</div><div class="csl-right-inline">PLOS digital health. 2024;3(4):e0000484. PMID: 38620037</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2100453899" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-2100453899">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>Few studies examining the patient outcomes of concurrent neurological manifestations during acute COVID-19 leveraged multinational cohorts of adults and children or distinguished between central and peripheral nervous system (CNS vs. PNS) involvement. Using a federated multinational network in which local clinicians and informatics experts curated the electronic health records data, we evaluated the risk of prolonged hospitalization and mortality in hospitalized COVID-19 patients from 21 healthcare systems across 7 countries. For adults, we used a federated learning approach whereby we ran Cox proportional hazard models locally at each healthcare system and performed a meta-analysis on the aggregated results to estimate the overall risk of adverse outcomes across our geographically diverse populations. For children, we reported descriptive statistics separately due to their low frequency of neurological involvement and poor outcomes. Among the 106,229 hospitalized COVID-19 patients (104,031 patients ≥18 years; 2,198 patients &lt;18 years, January 2020-October 2021), 15,101 (14%) had at least one CNS diagnosis, while 2,788 (3%) had at least one PNS diagnosis. After controlling for demographics and pre-existing conditions, adults with CNS involvement had longer hospital stay (11 versus 6 days) and greater risk of (Hazard Ratio = 1.78) and faster time to death (12 versus 24 days) than patients with no neurological condition (NNC) during acute COVID-19 hospitalization. Adults with PNS involvement also had longer hospital stay but lower risk of mortality than the NNC group. Although children had a low frequency of neurological involvement during COVID-19 hospitalization, a substantially higher proportion of children with CNS involvement died compared to those with NNC (6% vs 1%). Overall, patients with concurrent CNS manifestation during acute COVID-19 hospitalization faced greater risks for adverse clinical outcomes than patients without any neurological diagnosis. Our global informatics framework using a federated approach (versus a centralized data collection approach) has utility for clinical discovery beyond COVID-19.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Huang M, Gutiérrez-Sacristán A, Janiak E, Young K, Starosta A, Blanton K, Azhir A, Goldfarb CN, Kuperwasser F, Schaefer KM, Stoddard RE, Vatsa R, Merz-Herrala AA, Bartz D. <a href="/publication/contraceptive-content-shared-social-media-analysis-twitter">Contraceptive content shared on social media: an analysis of Twitter.</a>.</div><div class="csl-right-inline">Contraception and reproductive medicine. 2024;9(1):5. PMID: 38321582</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Huang M, Gutiérrez-Sacristán A, Janiak E, Young K, Starosta A, Blanton K, Azhir A, Goldfarb CN, Kuperwasser F, Schaefer KM, Stoddard RE, Vatsa R, Merz-Herrala AA, Bartz D. <a href="/publication/contraceptive-content-shared-social-media-analysis-twitter">Contraceptive content shared on social media: an analysis of Twitter.</a>.</div><div class="csl-right-inline">Contraception and reproductive medicine. 2024;9(1):5. PMID: 38321582</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-1649283050" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-1649283050">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Information on social media may affect peoples' contraceptive decision making. We performed an exploratory analysis of contraceptive content on Twitter (recently renamed X), a popular social media platform.</p><p>METHODS: We selected a random subset of 1% of publicly available, English-language tweets related to reversible, prescription contraceptive methods posted between January 2014 and December 2019. We oversampled tweets for the contraceptive patch to ensure at least 200 tweets per method. To create the codebook, we identified common themes specific to tweet content topics, tweet sources, and tweets soliciting information or providing advice. All posts were coded by two team members, and differences were adjudicated by a third reviewer. Descriptive analyses were reported with accompanying qualitative findings.</p><p>RESULTS: During the study period, 457,369 tweets about reversible contraceptive methods were published, with a random sample of 4,434 tweets used for final analysis. Tweets most frequently discussed contraceptive method decision-making (26.7%) and side effects (20.5%), particularly for long-acting reversible contraceptive methods and the depot medroxyprogesterone acetate shot. Tweets about logistics of use or adherence were common for short-acting reversible contraceptives. Tweets were frequently posted by contraceptive consumers (50.6%). A small proportion of tweets explicitly requested information (6.2%) or provided advice (4.2%).</p><p>CONCLUSIONS: Clinicians should be aware that individuals are exposed to information through Twitter that may affect contraceptive perceptions and decision making, particularly regarding long-acting reversible contraceptives. Social media is a valuable source for studying contraceptive beliefs missing in traditional health research and may be used by professionals to disseminate accurate contraceptive information.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gargano MA, Matentzoglu N, Coleman B, Addo-Lartey EB, Anagnostopoulos A V, Anderton J, Avillach P, Bagley AM, Bakštein E, Balhoff JP, Baynam G, Bello SM, Berk M, Bertram H, Bishop S, Blau H, Bodenstein DF, Botas P, Boztug K, Čady J, Callahan TJ, Cameron R, Carbon SJ, Castellanos F, Caufield H, Chan LE, Chute CG, Cruz-Rojo J, Dahan-Oliel N, Davids JR, de Dieuleveult M, de Souza V, de Vries BBA, de Vries E, DePaulo R, Derfalvi B, Dhombres F, Diaz-Byrd C, Dingemans AJM, Donadille B, Duyzend M, Elfeky R, Essaid S, Fabrizzi C, Fico G, Firth H V, Freudenberg-Hua Y, Fullerton JM, Gabriel DL, Gilmour K, Giordano J, Goes FS, Moses RG, Green I, Griese M, Groza T, Gu W, Guthrie J, Gyori B, Hamosh A, Hanauer M, Hanušová K, He YO, Hegde H, Helbig I, Holasová K, Hoyt CT, Huang S, Hurwitz E, Jacobsen JOB, Jiang X, Joseph L, Keramatian K, King B, Knoflach K, Koolen DA, Kraus ML, Kroll C, Kusters M, Ladewig MS, Lagorce D, Lai MC, Lapunzina P, Laraway B, Lewis-Smith D, Li X, Lucano C, Majd M, Marazita ML, Martinez-Glez V, McHenry TH, McInnis MG, McMurry JA, Mihulová M, Millett CE, Mitchell PB, Moslerová V, Narutomi K, Nematollahi S, Nevado J, Nierenberg AA, Čajbiková NN, Nurnberger JI, Ogishima S, Olson D, Ortiz A, Pachajoa H, de Nanclares GP, Peters A, Putman T, Rapp CK, Rath A, Reese J, Rekerle L, Roberts AM, Roy S, Sanders SJ, Schuetz C, Schulte EC, Schulze TG, Schwarz M, Scott K, Seelow D, Seitz B, Shen Y, Similuk MN, Simon ES, Singh B, Smedley D, Smith CL, Smolinsky JT, Sperry S, Stafford E, Stefancsik R, Steinhaus R, Strawbridge R, Sundaramurthi JC, Talapova P, Castano JAT, Tesner P, Thomas RH, Thurm A, Turnovec M, van Gijn ME, Vasilevsky NA, Vlčková M, Walden A, Wang K, Wapner R, Ware JS, Wiafe AA, Wiafe SA, Wiggins LD, Williams AE, Wu C, Wyrwoll MJ, Xiong H, Yalin N, Yamamoto Y, Yatham LN, Yocum AK, Young AH, Yüksel Z, Zandi PP, Zankl A, Zarante I, Zvolský M, Toro S, Carmody LC, Harris NL, Munoz-Torres MC, Danis D, Mungall CJ, Köhler S, Haendel MA, Robinson PN. <a href="/publication/human-phenotype-ontology-2024-phenotypes-around-world">The Human Phenotype Ontology in 2024: phenotypes around the world.</a>.</div><div class="csl-right-inline">Nucleic acids research. 2024;52(D1):D1333-D1346. PMID: 37953324</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gargano MA, Matentzoglu N, Coleman B, Addo-Lartey EB, Anagnostopoulos A V, Anderton J, Avillach P, Bagley AM, Bakštein E, Balhoff JP, Baynam G, Bello SM, Berk M, Bertram H, Bishop S, Blau H, Bodenstein DF, Botas P, Boztug K, Čady J, Callahan TJ, Cameron R, Carbon SJ, Castellanos F, Caufield H, Chan LE, Chute CG, Cruz-Rojo J, Dahan-Oliel N, Davids JR, de Dieuleveult M, de Souza V, de Vries BBA, de Vries E, DePaulo R, Derfalvi B, Dhombres F, Diaz-Byrd C, Dingemans AJM, Donadille B, Duyzend M, Elfeky R, Essaid S, Fabrizzi C, Fico G, Firth H V, Freudenberg-Hua Y, Fullerton JM, Gabriel DL, Gilmour K, Giordano J, Goes FS, Moses RG, Green I, Griese M, Groza T, Gu W, Guthrie J, Gyori B, Hamosh A, Hanauer M, Hanušová K, He YO, Hegde H, Helbig I, Holasová K, Hoyt CT, Huang S, Hurwitz E, Jacobsen JOB, Jiang X, Joseph L, Keramatian K, King B, Knoflach K, Koolen DA, Kraus ML, Kroll C, Kusters M, Ladewig MS, Lagorce D, Lai MC, Lapunzina P, Laraway B, Lewis-Smith D, Li X, Lucano C, Majd M, Marazita ML, Martinez-Glez V, McHenry TH, McInnis MG, McMurry JA, Mihulová M, Millett CE, Mitchell PB, Moslerová V, Narutomi K, Nematollahi S, Nevado J, Nierenberg AA, Čajbiková NN, Nurnberger JI, Ogishima S, Olson D, Ortiz A, Pachajoa H, de Nanclares GP, Peters A, Putman T, Rapp CK, Rath A, Reese J, Rekerle L, Roberts AM, Roy S, Sanders SJ, Schuetz C, Schulte EC, Schulze TG, Schwarz M, Scott K, Seelow D, Seitz B, Shen Y, Similuk MN, Simon ES, Singh B, Smedley D, Smith CL, Smolinsky JT, Sperry S, Stafford E, Stefancsik R, Steinhaus R, Strawbridge R, Sundaramurthi JC, Talapova P, Castano JAT, Tesner P, Thomas RH, Thurm A, Turnovec M, van Gijn ME, Vasilevsky NA, Vlčková M, Walden A, Wang K, Wapner R, Ware JS, Wiafe AA, Wiafe SA, Wiggins LD, Williams AE, Wu C, Wyrwoll MJ, Xiong H, Yalin N, Yamamoto Y, Yatham LN, Yocum AK, Young AH, Yüksel Z, Zandi PP, Zankl A, Zarante I, Zvolský M, Toro S, Carmody LC, Harris NL, Munoz-Torres MC, Danis D, Mungall CJ, Köhler S, Haendel MA, Robinson PN. <a href="/publication/human-phenotype-ontology-2024-phenotypes-around-world">The Human Phenotype Ontology in 2024: phenotypes around the world.</a>.</div><div class="csl-right-inline">Nucleic acids research. 2024;52(D1):D1333-D1346. PMID: 37953324</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2011850567" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-2011850567">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>The Human Phenotype Ontology (HPO) is a widely used resource that comprehensively organizes and defines the phenotypic features of human disease, enabling computational inference and supporting genomic and phenotypic analyses through semantic similarity and machine learning algorithms. The HPO has widespread applications in clinical diagnostics and translational research, including genomic diagnostics, gene-disease discovery, and cohort analytics. In recent years, groups around the world have developed translations of the HPO from English to other languages, and the HPO browser has been internationalized, allowing users to view HPO term labels and in many cases synonyms and definitions in ten languages in addition to English. Since our last report, a total of 2239 new HPO terms and 49235 new HPO annotations were developed, many in collaboration with external groups in the fields of psychiatry, arthrogryposis, immunology and cardiology. The Medical Action Ontology (MAxO) is a new effort to model treatments and other measures taken for clinical management. Finally, the HPO consortium is contributing to efforts to integrate the HPO and the GA4GH Phenopacket Schema into electronic health records (EHRs) with the goal of more standardized and computable integration of rare disease data in EHRs.</p></div>
</div>
</div>
</div>
</div>
</div>
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">2023</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Sperotto F, Gutiérrez-Sacristán A, Makwana S, Li X, Rofeberg VN, Cai T, Bourgeois FT, Omenn GS, Hanauer DA, Saez C, Bonzel CL, Bucholz E, Dionne A, Elias MD, García-Barrio N, González TG, Issitt RW, Kernan KF, Laird-Gion J, Maidlow SE, Mandl KD, Ahooyi TM, Moraleda C, Morris M, Moshal KL, Pedrera-Jiménez M, Shah MA, South AM, Spiridou A, Taylor DM, Verdy G, Visweswaran S, Wang X, Xia Z, Zachariasse JM, EHR C for CC of C 19 by, Newburger JW, Avillach P. <a href="/publication/clinical-phenotypes-and-outcomes-children-multisystem-inflammatory-syndrome-across-0">Clinical phenotypes and outcomes in children with multisystem inflammatory syndrome across SARS-CoV-2 variant eras: a multinational study from the 4CE consortium.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102212. PMID: 37745025</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Sperotto F, Gutiérrez-Sacristán A, Makwana S, Li X, Rofeberg VN, Cai T, Bourgeois FT, Omenn GS, Hanauer DA, Saez C, Bonzel CL, Bucholz E, Dionne A, Elias MD, García-Barrio N, González TG, Issitt RW, Kernan KF, Laird-Gion J, Maidlow SE, Mandl KD, Ahooyi TM, Moraleda C, Morris M, Moshal KL, Pedrera-Jiménez M, Shah MA, South AM, Spiridou A, Taylor DM, Verdy G, Visweswaran S, Wang X, Xia Z, Zachariasse JM, EHR C for CC of C 19 by, Newburger JW, Avillach P. <a href="/publication/clinical-phenotypes-and-outcomes-children-multisystem-inflammatory-syndrome-across-0">Clinical phenotypes and outcomes in children with multisystem inflammatory syndrome across SARS-CoV-2 variant eras: a multinational study from the 4CE consortium.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102212. PMID: 37745025</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-87103762" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-87103762">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Multisystem inflammatory syndrome in children (MIS-C) is a severe complication of SARS-CoV-2 infection. It remains unclear how MIS-C phenotypes vary across SARS-CoV-2 variants. We aimed to investigate clinical characteristics and outcomes of MIS-C across SARS-CoV-2 eras.</p><p>METHODS: We performed a multicentre observational retrospective study including seven paediatric hospitals in four countries (France, Spain, U.K., and U.S.). All consecutive confirmed patients with MIS-C hospitalised between February 1st, 2020, and May 31st, 2022, were included. Electronic Health Records (EHR) data were used to calculate pooled risk differences (RD) and effect sizes (ES) at site level, using  as reference. Meta-analysis was used to pool data across sites.</p><p>FINDINGS: Of 598 patients with MIS-C (61% male, 39% female; mean age 9.7 years [SD 4.5]), 383 (64%) were admitted in the  era, 111 (19%) in the  era, and 104 (17%) in the  era. Compared with patients admitted in the  era, those admitted in the  era were younger (ES -1.18 years [95% CI -2.05, -0.32]), had fewer respiratory symptoms (RD -0.15 [95% CI -0.33, -0.04]), less frequent non-cardiogenic shock or systemic inflammatory response syndrome (SIRS) (RD -0.35 [95% CI -0.64, -0.07]), lower lymphocyte count (ES -0.16 × 10/uL [95% CI -0.30, -0.01]), lower C-reactive protein (ES -28.5 mg/L [95% CI -46.3, -10.7]), and lower troponin (ES -0.14 ng/mL [95% CI -0.26, -0.03]). Patients admitted in the  versus  eras were younger (ES -1.6 years [95% CI -2.5, -0.8]), had less frequent SIRS (RD -0.18 [95% CI -0.30, -0.05]), lower lymphocyte count (ES -0.39 × 10/uL [95% CI -0.52, -0.25]), lower troponin (ES -0.16 ng/mL [95% CI -0.30, -0.01]) and less frequently received anticoagulation therapy (RD -0.19 [95% CI -0.37, -0.04]). Length of hospitalization was shorter in the  versus  eras (-1.3 days [95% CI -2.3, -0.4]).</p><p>INTERPRETATION: Our study suggested that MIS-C clinical phenotypes varied across SARS-CoV-2 eras, with patients in  and  eras being younger and less sick. EHR data can be effectively leveraged to identify rare complications of pandemic diseases and their variation over time.</p><p>FUNDING: None.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Dagliati A, Strasser ZH, Abad ZSH, Klann JG, Wagholikar KB, Mesa R, Visweswaran S, Morris M, Luo Y, Henderson DW, Samayamuthu MJ, Tan BWQ, Verdy G, Omenn GS, Xia Z, Bellazzi R, EHR C for CC of C 19 by, Murphy SN, Holmes JH, Estiri H, EHR C for CC of C 19 by. <a href="/publication/characterization-long-covid-temporal-sub-phenotypes-distributed-representation-0">Characterization of long COVID temporal sub-phenotypes by distributed representation learning from electronic health record data: a cohort study.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102210. PMID: 37745021</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Dagliati A, Strasser ZH, Abad ZSH, Klann JG, Wagholikar KB, Mesa R, Visweswaran S, Morris M, Luo Y, Henderson DW, Samayamuthu MJ, Tan BWQ, Verdy G, Omenn GS, Xia Z, Bellazzi R, EHR C for CC of C 19 by, Murphy SN, Holmes JH, Estiri H, EHR C for CC of C 19 by. <a href="/publication/characterization-long-covid-temporal-sub-phenotypes-distributed-representation-0">Characterization of long COVID temporal sub-phenotypes by distributed representation learning from electronic health record data: a cohort study.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102210. PMID: 37745021</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-250295374" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-250295374">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Characterizing Post-Acute Sequelae of COVID (SARS-CoV-2 Infection), or  has been challenging due to the multitude of sub-phenotypes, temporal attributes, and definitions. Scalable characterization of PASC sub-phenotypes can enhance screening capacities, disease management, and treatment planning.</p><p>METHODS: We conducted a retrospective multi-centre observational cohort study, leveraging longitudinal electronic health record (EHR) data of 30,422 patients from three healthcare systems in the Consortium for the Clinical Characterization of COVID-19 by EHR (4CE). From the total cohort, we applied a deductive approach on 12,424 individuals with follow-up data and developed a distributed representation learning process for providing augmented definitions for PASC sub-phenotypes.</p><p>FINDINGS: Our framework characterized seven PASC sub-phenotypes. We estimated that on average 15.7% of the hospitalized COVID-19 patients were likely to suffer from at least one PASC symptom and almost 5.98%, on average, had multiple symptoms. Joint pain and dyspnea had the highest prevalence, with an average prevalence of 5.45% and 4.53%, respectively.</p><p>INTERPRETATION: We provided a scalable framework to every participating healthcare system for estimating PASC sub-phenotypes prevalence and temporal attributes, thus developing a unified model that characterizes augmented sub-phenotypes across the different systems.</p><p>FUNDING: Authors are supported by National Institute of Allergy and Infectious Diseases, National Institute on Aging, National Center for Advancing Translational Sciences, National Medical Research Council, National Institute of Neurological Disorders and Stroke, European Union, National Institutes of Health, National Center for Advancing Translational Sciences.</p></div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<nav aria-label="Pagination" class="hwp-pager hwp-pager--theme-light hwp-bg-light-core" role="navigation">
<ul class="hwp-pager__items js-pager__items">
<li>
<div class="hwp-pager__item hwp-pager__item--previous hwp-pager__item--not-active" href="" rel="prev" title="Go to previous page">
<span class="hwp-visually-hidden">Previous page</span>
<span aria-hidden="true" class="material-icon material-icon--chevron-left">chevron_left</span>
</div>
</li>
<li class="hwp-pager__item hwp-pager__item--is-active">
<a aria-current="page" href="?page=0" title="Current page">1</a>
</li>
<li class="hwp-pager__item">
<a href="?page=1" title="Go to page 2">2</a>
</li>
<li class="hwp-pager__item">
<a href="?page=2" title="Go to page 3">3</a>
</li>
<li class="hwp-pager__item hwp-pager__item--last hwp-pager__item--ellipsis">
<a href="?page=8" title="Go to last page">
<span class="hwp-visually-hidden">Last page</span>
<span aria-hidden="true">9</span>
</a>
</li>
<li>
<a class="hwp-pager__item hwp-pager__item--next" href="?page=1" rel="prev" title="Go to next page">
<span class="hwp-visually-hidden">Next page</span>
<span aria-hidden="true" class="material-icon material-icon--chevron-right">chevron_right</span>
</a>
</li>
</ul>
</nav>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/publication/clinical-phenotypes-and-outcomes-children-multisystem-inflammatory-syndrome-across-0 -->
<h1>Clinical phenotypes and outcomes in children with multisystem inflammatory syndrome across SARS-CoV-2 variant eras: a multinational study from the 4CE consortium. | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/publication/clinical-phenotypes-and-outcomes-children-multisystem-inflammatory-syndrome-across-0'>avillach-lab.hms.harvard.edu/publication/clinical-phenotypes-and-outcomes-children-multisystem-inflammatory-syndrome-across-0</a></div>
<main aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="biblio-details">
<div class="hwp-page-title bibcite-reference" lang="en">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Clinical phenotypes and outcomes in children with multisystem inflammatory syndrome across SARS-CoV-2 variant eras: a multinational study from the 4CE consortium.
    </h1>
</div>
</div>
<div class="md:hwp-pt-64">
<div class="hwp-container hwp-container-px max-lg:hwp-px-0">
<div class="publication-info" data-component-name="publication-info">
<div class="publication-info__image">
</div>
<div class="publication-info__text">
<h4 class="h4 hwp-mb-8">Publication information:</h4>
<div class="publication-info__body">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Sperotto F, Gutiérrez-Sacristán A, Makwana S, Li X, Rofeberg VN, Cai T, Bourgeois FT, Omenn GS, Hanauer DA, Saez C, Bonzel CL, Bucholz E, Dionne A, Elias MD, García-Barrio N, González TG, Issitt RW, Kernan KF, Laird-Gion J, Maidlow SE, Mandl KD, Ahooyi TM, Moraleda C, Morris M, Moshal KL, Pedrera-Jiménez M, Shah MA, South AM, Spiridou A, Taylor DM, Verdy G, Visweswaran S, Wang X, Xia Z, Zachariasse JM, EHR C for CC of C 19 by, Newburger JW, Avillach P. Clinical phenotypes and outcomes in children with multisystem inflammatory syndrome across SARS-CoV-2 variant eras: a multinational study from the 4CE consortium.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102212. PMID: 37745025</div></div></div>
</div>
</div>
<ul class="publication-info__actions">
<li>
<div class="button-dropdown" data-component-name="button-dropdown">
<button class="hwp-text-link hwp-text-link--icon-left" id="publication_citation_download">
<span aria-hidden="true" class="material-icon">download_for_offline</span>
<span class="hwp-text-link__text">Download citation</span>
</button>
<ul aria-labelledby="publication_citation_download" class="button-dropdown__items" role="menu">
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773831&amp;format=bibtex">BibTeX</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773831&amp;format=endnote8">EndNote X3 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773831&amp;format=endnote7">EndNote 7 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773831&amp;format=tagged">Endnote tagged</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773831&amp;format=marc">Marc</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773831&amp;format=pubmed_id">PubMedId</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773831&amp;format=ris">RIS</a>
</li>
</ul>
</div>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 hwp-pb-64 md:hwp-py-64">
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">Abstract</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-trim hwp-bg-light-base js-hwp-citation" data-component-name="citation">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<p><b>BACKGROUND: </b>Multisystem inflammatory syndrome in children (MIS-C) is a severe complication of SARS-CoV-2 infection. It remains unclear how MIS-C phenotypes vary across SARS-CoV-2 variants. We aimed to investigate clinical characteristics and outcomes of MIS-C across SARS-CoV-2 eras.</p><p><b>METHODS: </b>We performed a multicentre observational retrospective study including seven paediatric hospitals in four countries (France, Spain, U.K., and U.S.). All consecutive confirmed patients with MIS-C hospitalised between February 1st, 2020, and May 31st, 2022, were included. Electronic Health Records (EHR) data were used to calculate pooled risk differences (RD) and effect sizes (ES) at site level, using  as reference. Meta-analysis was used to pool data across sites.</p><p><b>FINDINGS: </b>Of 598 patients with MIS-C (61% male, 39% female; mean age 9.7 years [SD 4.5]), 383 (64%) were admitted in the  era, 111 (19%) in the  era, and 104 (17%) in the  era. Compared with patients admitted in the  era, those admitted in the  era were younger (ES -1.18 years [95% CI -2.05, -0.32]), had fewer respiratory symptoms (RD -0.15 [95% CI -0.33, -0.04]), less frequent non-cardiogenic shock or systemic inflammatory response syndrome (SIRS) (RD -0.35 [95% CI -0.64, -0.07]), lower lymphocyte count (ES -0.16 × 10/uL [95% CI -0.30, -0.01]), lower C-reactive protein (ES -28.5 mg/L [95% CI -46.3, -10.7]), and lower troponin (ES -0.14 ng/mL [95% CI -0.26, -0.03]). Patients admitted in the  versus  eras were younger (ES -1.6 years [95% CI -2.5, -0.8]), had less frequent SIRS (RD -0.18 [95% CI -0.30, -0.05]), lower lymphocyte count (ES -0.39 × 10/uL [95% CI -0.52, -0.25]), lower troponin (ES -0.16 ng/mL [95% CI -0.30, -0.01]) and less frequently received anticoagulation therapy (RD -0.19 [95% CI -0.37, -0.04]). Length of hospitalization was shorter in the  versus  eras (-1.3 days [95% CI -2.3, -0.4]).</p><p><b>INTERPRETATION: </b>Our study suggested that MIS-C clinical phenotypes varied across SARS-CoV-2 eras, with patients in  and  eras being younger and less sick. EHR data can be effectively leveraged to identify rare complications of pandemic diseases and their variation over time.</p><p><b>FUNDING: </b>None.</p>
</div>
<div class="hwp-text-block">
<p><b>BACKGROUND: </b>Multisystem inflammatory syndrome in children (MIS-C) is a severe complication of SARS-CoV-2 infection. It remains unclear how MIS-C phenotypes vary across SARS-CoV-2 variants. We aimed to investigate clinical characteristics and outcomes of MIS-C across SARS-CoV-2 eras.</p><p><b>METHODS: </b>We performed a multicentre observational retrospective study including seven paediatric hospitals in four countries (France, Spain, U.K., and U.S.). All consecutive confirmed patients with MIS-C hospitalised between February 1st, 2020, and May 31st, 2022, were included. Electronic Health Records (EHR) data were used to calculate pooled risk differences (RD) and effect sizes (ES) at site level, using  as reference. Meta-analysis was used to pool data across sites.</p><p><b>FINDINGS: </b>Of 598 patients with MIS-C (61% male, 39% female; mean age 9.7 years [SD 4.5]), 383 (64%) were admitted in the  era, 111 (19%) in the  era, and 104 (17%) in the  era. Compared with patients admitted in the  era, those admitted in the  era were younger (ES -1.18 years [95% CI -2.05, -0.32]), had fewer respiratory symptoms (RD -0.15 [95% CI -0.33, -0.04]), less frequent non-cardiogenic shock or systemic inflammatory response syndrome (SIRS) (RD -0.35 [95% CI -0.64, -0.07]), lower lymphocyte count (ES -0.16 × 10/uL [95% CI -0.30, -0.01]), lower C-reactive protein (ES -28.5 mg/L [95% CI -46.3, -10.7]), and lower troponin (ES -0.14 ng/mL [95% CI -0.26, -0.03]). Patients admitted in the  versus  eras were younger (ES -1.6 years [95% CI -2.5, -0.8]), had less frequent SIRS (RD -0.18 [95% CI -0.30, -0.05]), lower lymphocyte count (ES -0.39 × 10/uL [95% CI -0.52, -0.25]), lower troponin (ES -0.16 ng/mL [95% CI -0.30, -0.01]) and less frequently received anticoagulation therapy (RD -0.19 [95% CI -0.37, -0.04]). Length of hospitalization was shorter in the  versus  eras (-1.3 days [95% CI -2.3, -0.4]).</p><p><b>INTERPRETATION: </b>Our study suggested that MIS-C clinical phenotypes varied across SARS-CoV-2 eras, with patients in  and  eras being younger and less sick. EHR data can be effectively leveraged to identify rare complications of pandemic diseases and their variation over time.</p><p><b>FUNDING: </b>None.</p>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-443299305" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
</div>
</div>
</div>
<hr class="hwp-text-button-light-secondary hwp-mt-32"/>
</div>
<div class="hwp-news-footer hwp-py-24 lg:hwp-py-64 hwp-container hwp-container-px">
<div class="page-tags">
</div>
<div class="a2a_kit" data-a2a-title="" data-a2a-url="">
<span class="hwp-mb-16 hwp-block">Share on:</span>
<ul aria-label="Share on:" class="md:hwp-inline-flex md:hwp-flex-wrap">
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_facebook hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-facebook"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Facebook</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_twitter hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-twitter"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Twitter</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_linkedin hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-linkedin"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Linkedin</span>
</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/publication/substance-use-disorders-and-disparities-hispanic-and-non-hispanic-older-adults -->
<h1>Substance use disorders and disparities in hispanic and non-hispanic older adults. | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/publication/substance-use-disorders-and-disparities-hispanic-and-non-hispanic-older-adults'>avillach-lab.hms.harvard.edu/publication/substance-use-disorders-and-disparities-hispanic-and-non-hispanic-older-adults</a></div>
<main aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="biblio-details">
<div class="hwp-page-title bibcite-reference" lang="en">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Substance use disorders and disparities in hispanic and non-hispanic older adults.
    </h1>
</div>
</div>
<div class="md:hwp-pt-64">
<div class="hwp-container hwp-container-px max-lg:hwp-px-0">
<div class="publication-info" data-component-name="publication-info">
<div class="publication-info__image">
</div>
<div class="publication-info__text">
<h4 class="h4 hwp-mb-8">Publication information:</h4>
<div class="publication-info__body">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hernández-Agosto J, Santiago-Quiñones DI, García-Carmona LM, Gutiérrez-Sacristán A. Substance use disorders and disparities in hispanic and non-hispanic older adults.</div><div class="csl-right-inline">Drug and alcohol dependence. 2024;261:111357. PMID: 38896947</div></div></div>
</div>
</div>
<ul class="publication-info__actions">
<li>
<div class="button-dropdown" data-component-name="button-dropdown">
<button class="hwp-text-link hwp-text-link--icon-left" id="publication_citation_download">
<span aria-hidden="true" class="material-icon">download_for_offline</span>
<span class="hwp-text-link__text">Download citation</span>
</button>
<ul aria-labelledby="publication_citation_download" class="button-dropdown__items" role="menu">
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773941&amp;format=bibtex">BibTeX</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773941&amp;format=endnote8">EndNote X3 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773941&amp;format=endnote7">EndNote 7 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773941&amp;format=tagged">Endnote tagged</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773941&amp;format=marc">Marc</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773941&amp;format=pubmed_id">PubMedId</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773941&amp;format=ris">RIS</a>
</li>
</ul>
</div>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 hwp-pb-64 md:hwp-py-64">
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">Abstract</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-trim hwp-bg-light-base js-hwp-citation" data-component-name="citation">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<p><b>BACKGROUND: </b>The global longevity revolution increased the older adult population, posing unique health and economic challenges with implications for healthcare, especially substance use disorders (SUD).</p><p><b>METHODS: </b>This was a retrospective cohort study of United States older adults, Hispanic and non-Hispanic, who got at least one mental and/or behavioral disorder diagnosis between 2017 and 2021 at age 65 or older. SUD prevalence, prescription frequency changes over time, and comorbidities associated with each medication were compared.</p><p><b>RESULTS: </b>Electronic health records for 356,133 older adults (110,236 Hispanics and 245,897 non-Hispanics) were analyzed. Notably, 79 % of Hispanics fell below the 100 % federal poverty level, compared to 60 % of non-Hispanics (P&lt;.001). Non-Hispanics also had significantly more average encounters (P=.003) and diagnoses (P&lt;.001). Regression analysis on alcohol-related disorders indicated that the odd ratios of being male (OR=2.93, P&lt;.000), and having low income (OR=1.62, P&lt;.000), increase the odds for this SUD, while being Hispanic and primarily speaking Spanish decreases the odds for all SUDs considered in this study.</p><p><b>CONCLUSIONS: </b>This cohort study revealed significant disparities related to social determinants of health between Hispanic and non-Hispanic older adults and emphasizes the need for continuous surveillance of older adults as with SUDs. Differences in comorbidity patterns imply distinct risk factors within each population, influenced by demographic-specific elements. Recognizing these variations is essential for tailoring culturally sensitive prevention, intervention, and treatment strategies to each population's unique needs.</p>
</div>
<div class="hwp-text-block">
<p><b>BACKGROUND: </b>The global longevity revolution increased the older adult population, posing unique health and economic challenges with implications for healthcare, especially substance use disorders (SUD).</p><p><b>METHODS: </b>This was a retrospective cohort study of United States older adults, Hispanic and non-Hispanic, who got at least one mental and/or behavioral disorder diagnosis between 2017 and 2021 at age 65 or older. SUD prevalence, prescription frequency changes over time, and comorbidities associated with each medication were compared.</p><p><b>RESULTS: </b>Electronic health records for 356,133 older adults (110,236 Hispanics and 245,897 non-Hispanics) were analyzed. Notably, 79 % of Hispanics fell below the 100 % federal poverty level, compared to 60 % of non-Hispanics (P&lt;.001). Non-Hispanics also had significantly more average encounters (P=.003) and diagnoses (P&lt;.001). Regression analysis on alcohol-related disorders indicated that the odd ratios of being male (OR=2.93, P&lt;.000), and having low income (OR=1.62, P&lt;.000), increase the odds for this SUD, while being Hispanic and primarily speaking Spanish decreases the odds for all SUDs considered in this study.</p><p><b>CONCLUSIONS: </b>This cohort study revealed significant disparities related to social determinants of health between Hispanic and non-Hispanic older adults and emphasizes the need for continuous surveillance of older adults as with SUDs. Differences in comorbidity patterns imply distinct risk factors within each population, influenced by demographic-specific elements. Recognizing these variations is essential for tailoring culturally sensitive prevention, intervention, and treatment strategies to each population's unique needs.</p>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-1787088472" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
</div>
</div>
</div>
<hr class="hwp-text-button-light-secondary hwp-mt-32"/>
</div>
<div class="hwp-news-footer hwp-py-24 lg:hwp-py-64 hwp-container hwp-container-px">
<div class="page-tags">
</div>
<div class="a2a_kit" data-a2a-title="" data-a2a-url="">
<span class="hwp-mb-16 hwp-block">Share on:</span>
<ul aria-label="Share on:" class="md:hwp-inline-flex md:hwp-flex-wrap">
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_facebook hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-facebook"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Facebook</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_twitter hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-twitter"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Twitter</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_linkedin hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-linkedin"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Linkedin</span>
</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/node/1774776/export -->
<!-- Source: avillach-lab.hms.harvard.edu/node/1774776/export -->
<!-- Source: avillach-lab.hms.harvard.edu/publication/development-and-validation-open-source-pipeline-automatic-population-case-report-forms -->
<h1>Development and validation of an open-source pipeline for automatic population of case report forms from electronic health records: a pediatric multi-center prospective study. | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/publication/development-and-validation-open-source-pipeline-automatic-population-case-report-forms'>avillach-lab.hms.harvard.edu/publication/development-and-validation-open-source-pipeline-automatic-population-case-report-forms</a></div>
<main aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="biblio-details">
<div class="hwp-page-title bibcite-reference" lang="en">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Development and validation of an open-source pipeline for automatic population of case report forms from electronic health records: a pediatric multi-center prospective study.
    </h1>
</div>
</div>
<div class="md:hwp-pt-64">
<div class="hwp-container hwp-container-px max-lg:hwp-px-0">
<div class="publication-info" data-component-name="publication-info">
<div class="publication-info__image">
</div>
<div class="publication-info__text">
<h4 class="h4 hwp-mb-8">Publication information:</h4>
<div class="publication-info__body">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gutiérrez-Sacristán A, Makwana S, Dionne A, Mahanta S, Dyer KJ, Serrano F, Watrin C, Pages P, Mousavi S, Degala A, Lyons J, Pillion D, Zachariasse JM, Shekerdemian LS, Truong DT, Newburger JW, Avillach P. Development and validation of an open-source pipeline for automatic population of case report forms from electronic health records: a pediatric multi-center prospective study.</div><div class="csl-right-inline">EBioMedicine. 2024;108:105337. PMID: 39288532</div></div></div>
</div>
</div>
<ul class="publication-info__actions">
<li>
<div class="button-dropdown" data-component-name="button-dropdown">
<button class="hwp-text-link hwp-text-link--icon-left" id="publication_citation_download">
<span aria-hidden="true" class="material-icon">download_for_offline</span>
<span class="hwp-text-link__text">Download citation</span>
</button>
<ul aria-labelledby="publication_citation_download" class="button-dropdown__items" role="menu">
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773906&amp;format=bibtex">BibTeX</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773906&amp;format=endnote8">EndNote X3 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773906&amp;format=endnote7">EndNote 7 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773906&amp;format=tagged">Endnote tagged</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773906&amp;format=marc">Marc</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773906&amp;format=pubmed_id">PubMedId</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773906&amp;format=ris">RIS</a>
</li>
</ul>
</div>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 hwp-pb-64 md:hwp-py-64">
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">Abstract</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-trim hwp-bg-light-base js-hwp-citation" data-component-name="citation">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<p><b>BACKGROUND: </b>Clinical trials and registry studies are essential for advancing research and developing novel treatments. However, these studies rely on manual entry of thousands of variables for each patient. Repurposing real-world data can significantly simplify the data collection, reduce transcription errors, and make the data entry process more efficient, consistent, and cost-effective.</p><p><b>METHODS: </b>We developed an open-source computational pipeline to collect laboratory and medication information from the electronic health record (EHR) data and populate case report forms. The pipeline was developed and validated with data from two independent pediatric hospitals in the US as part of the Long-terM OUtcomes after Multisystem Inflammatory Syndrome In Children (MUSIC) study. Our pipeline allowed the completion of two of the most time-consuming forms. We compared automatically extracted results with manually entered values in one hospital and applied the pipeline to a second hospital, where the output served as the primary data source for case report forms.</p><p><b>FINDINGS: </b>We extracted and populated 51,845 laboratory and 4913 medication values for 159 patients in two hospitals participating in a prospective pediatric study. We evaluated pipeline performance against data for 104 patients manually entered by clinicians in one of the hospitals. The highest concordance was found during patient hospitalization, with 91.59% of the automatically extracted laboratory and medication values corresponding with the manually entered values. In addition to the successfully populated values, we identified an additional 13,396 laboratory and 567 medication values of interest for the study.</p><p><b>INTERPRETATION: </b>The automatic data entry of laboratory and medication values during admission is feasible and has a high concordance with the manually entered data. By implementing this proof of concept, we demonstrate the quality of automatic data extraction and highlight the potential of secondary use of EHR data to advance medical science by improving data entry efficiency and expediting clinical research.</p><p><b>FUNDING: </b>NIH Grant 1OT3HL147154-01, U24HL135691, UG1HL135685.</p>
</div>
<div class="hwp-text-block">
<p><b>BACKGROUND: </b>Clinical trials and registry studies are essential for advancing research and developing novel treatments. However, these studies rely on manual entry of thousands of variables for each patient. Repurposing real-world data can significantly simplify the data collection, reduce transcription errors, and make the data entry process more efficient, consistent, and cost-effective.</p><p><b>METHODS: </b>We developed an open-source computational pipeline to collect laboratory and medication information from the electronic health record (EHR) data and populate case report forms. The pipeline was developed and validated with data from two independent pediatric hospitals in the US as part of the Long-terM OUtcomes after Multisystem Inflammatory Syndrome In Children (MUSIC) study. Our pipeline allowed the completion of two of the most time-consuming forms. We compared automatically extracted results with manually entered values in one hospital and applied the pipeline to a second hospital, where the output served as the primary data source for case report forms.</p><p><b>FINDINGS: </b>We extracted and populated 51,845 laboratory and 4913 medication values for 159 patients in two hospitals participating in a prospective pediatric study. We evaluated pipeline performance against data for 104 patients manually entered by clinicians in one of the hospitals. The highest concordance was found during patient hospitalization, with 91.59% of the automatically extracted laboratory and medication values corresponding with the manually entered values. In addition to the successfully populated values, we identified an additional 13,396 laboratory and 567 medication values of interest for the study.</p><p><b>INTERPRETATION: </b>The automatic data entry of laboratory and medication values during admission is feasible and has a high concordance with the manually entered data. By implementing this proof of concept, we demonstrate the quality of automatic data extraction and highlight the potential of secondary use of EHR data to advance medical science by improving data entry efficiency and expediting clinical research.</p><p><b>FUNDING: </b>NIH Grant 1OT3HL147154-01, U24HL135691, UG1HL135685.</p>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-909846853" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
</div>
</div>
</div>
<hr class="hwp-text-button-light-secondary hwp-mt-32"/>
</div>
<div class="hwp-news-footer hwp-py-24 lg:hwp-py-64 hwp-container hwp-container-px">
<div class="page-tags">
</div>
<div class="a2a_kit" data-a2a-title="" data-a2a-url="">
<span class="hwp-mb-16 hwp-block">Share on:</span>
<ul aria-label="Share on:" class="md:hwp-inline-flex md:hwp-flex-wrap">
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_facebook hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-facebook"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Facebook</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_twitter hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-twitter"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Twitter</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_linkedin hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-linkedin"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Linkedin</span>
</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/publication/contraceptive-content-shared-social-media-analysis-twitter -->
<h1>Contraceptive content shared on social media: an analysis of Twitter. | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/publication/contraceptive-content-shared-social-media-analysis-twitter'>avillach-lab.hms.harvard.edu/publication/contraceptive-content-shared-social-media-analysis-twitter</a></div>
<main aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="biblio-details">
<div class="hwp-page-title bibcite-reference" lang="en">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Contraceptive content shared on social media: an analysis of Twitter.
    </h1>
</div>
</div>
<div class="md:hwp-pt-64">
<div class="hwp-container hwp-container-px max-lg:hwp-px-0">
<div class="publication-info" data-component-name="publication-info">
<div class="publication-info__image">
</div>
<div class="publication-info__text">
<h4 class="h4 hwp-mb-8">Publication information:</h4>
<div class="publication-info__body">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Huang M, Gutiérrez-Sacristán A, Janiak E, Young K, Starosta A, Blanton K, Azhir A, Goldfarb CN, Kuperwasser F, Schaefer KM, Stoddard RE, Vatsa R, Merz-Herrala AA, Bartz D. Contraceptive content shared on social media: an analysis of Twitter.</div><div class="csl-right-inline">Contraception and reproductive medicine. 2024;9(1):5. PMID: 38321582</div></div></div>
</div>
</div>
<ul class="publication-info__actions">
<li>
<div class="button-dropdown" data-component-name="button-dropdown">
<button class="hwp-text-link hwp-text-link--icon-left" id="publication_citation_download">
<span aria-hidden="true" class="material-icon">download_for_offline</span>
<span class="hwp-text-link__text">Download citation</span>
</button>
<ul aria-labelledby="publication_citation_download" class="button-dropdown__items" role="menu">
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773946&amp;format=bibtex">BibTeX</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773946&amp;format=endnote8">EndNote X3 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773946&amp;format=endnote7">EndNote 7 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773946&amp;format=tagged">Endnote tagged</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773946&amp;format=marc">Marc</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773946&amp;format=pubmed_id">PubMedId</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773946&amp;format=ris">RIS</a>
</li>
</ul>
</div>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 hwp-pb-64 md:hwp-py-64">
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">Abstract</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-trim hwp-bg-light-base js-hwp-citation" data-component-name="citation">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<p><b>BACKGROUND: </b>Information on social media may affect peoples' contraceptive decision making. We performed an exploratory analysis of contraceptive content on Twitter (recently renamed X), a popular social media platform.</p><p><b>METHODS: </b>We selected a random subset of 1% of publicly available, English-language tweets related to reversible, prescription contraceptive methods posted between January 2014 and December 2019. We oversampled tweets for the contraceptive patch to ensure at least 200 tweets per method. To create the codebook, we identified common themes specific to tweet content topics, tweet sources, and tweets soliciting information or providing advice. All posts were coded by two team members, and differences were adjudicated by a third reviewer. Descriptive analyses were reported with accompanying qualitative findings.</p><p><b>RESULTS: </b>During the study period, 457,369 tweets about reversible contraceptive methods were published, with a random sample of 4,434 tweets used for final analysis. Tweets most frequently discussed contraceptive method decision-making (26.7%) and side effects (20.5%), particularly for long-acting reversible contraceptive methods and the depot medroxyprogesterone acetate shot. Tweets about logistics of use or adherence were common for short-acting reversible contraceptives. Tweets were frequently posted by contraceptive consumers (50.6%). A small proportion of tweets explicitly requested information (6.2%) or provided advice (4.2%).</p><p><b>CONCLUSIONS: </b>Clinicians should be aware that individuals are exposed to information through Twitter that may affect contraceptive perceptions and decision making, particularly regarding long-acting reversible contraceptives. Social media is a valuable source for studying contraceptive beliefs missing in traditional health research and may be used by professionals to disseminate accurate contraceptive information.</p>
</div>
<div class="hwp-text-block">
<p><b>BACKGROUND: </b>Information on social media may affect peoples' contraceptive decision making. We performed an exploratory analysis of contraceptive content on Twitter (recently renamed X), a popular social media platform.</p><p><b>METHODS: </b>We selected a random subset of 1% of publicly available, English-language tweets related to reversible, prescription contraceptive methods posted between January 2014 and December 2019. We oversampled tweets for the contraceptive patch to ensure at least 200 tweets per method. To create the codebook, we identified common themes specific to tweet content topics, tweet sources, and tweets soliciting information or providing advice. All posts were coded by two team members, and differences were adjudicated by a third reviewer. Descriptive analyses were reported with accompanying qualitative findings.</p><p><b>RESULTS: </b>During the study period, 457,369 tweets about reversible contraceptive methods were published, with a random sample of 4,434 tweets used for final analysis. Tweets most frequently discussed contraceptive method decision-making (26.7%) and side effects (20.5%), particularly for long-acting reversible contraceptive methods and the depot medroxyprogesterone acetate shot. Tweets about logistics of use or adherence were common for short-acting reversible contraceptives. Tweets were frequently posted by contraceptive consumers (50.6%). A small proportion of tweets explicitly requested information (6.2%) or provided advice (4.2%).</p><p><b>CONCLUSIONS: </b>Clinicians should be aware that individuals are exposed to information through Twitter that may affect contraceptive perceptions and decision making, particularly regarding long-acting reversible contraceptives. Social media is a valuable source for studying contraceptive beliefs missing in traditional health research and may be used by professionals to disseminate accurate contraceptive information.</p>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2096064225" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
</div>
</div>
</div>
<hr class="hwp-text-button-light-secondary hwp-mt-32"/>
</div>
<div class="hwp-news-footer hwp-py-24 lg:hwp-py-64 hwp-container hwp-container-px">
<div class="page-tags">
</div>
<div class="a2a_kit" data-a2a-title="" data-a2a-url="">
<span class="hwp-mb-16 hwp-block">Share on:</span>
<ul aria-label="Share on:" class="md:hwp-inline-flex md:hwp-flex-wrap">
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_facebook hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-facebook"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Facebook</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_twitter hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-twitter"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Twitter</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_linkedin hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-linkedin"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Linkedin</span>
</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/node/1774776/export -->
<!-- Source: avillach-lab.hms.harvard.edu/publications -->
<h1>Publications | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/publications'>avillach-lab.hms.harvard.edu/publications</a></div>
<main aria-label="Publications" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="search-page search-page--no-filters hwp-listing-page hwp-bibcite_reference">
<div class="hwp-page-title hwp-bg-dark-base" lang="en">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Publications
    </h1>
</div>
</div>
<div class="hwp-section-heading" data-component-name="section-heading">
<div class="hwp-section-heading__container hwp-container-px hwp-container">
<div class="hwp-section-heading__description hwp-text-block">
</div>
</div>
</div>
<div class="search-page__columns hwp-container hwp-container-px">
<div class="search-page__results">
<div class="search-page__search-form">
<form accept-charset="UTF-8" action="/publications" class="hwp-search-form-listing" data-drupal-selector="hwp-search-form-listing" id="hwp-search-form-listing" method="post">
<div class="hwp-form-type-search hwp-form-item js-form-item form-item form-type-search js-form-type-search form-item-search js-form-item-search form-no-label">
<label class="visually-hidden" for="edit-search">Search Within Results</label>
<input class="form-search" data-drupal-selector="edit-search" id="edit-search" maxlength="128" name="search" size="60" type="search" value=""/>
</div>
<button class="hwp-button hwp-button--small button js-form-submit form-submit" data-drupal-selector="edit-submit" data-twig-suggestion="header_site_search_submit" id="edit-submit" name="op" type="submit" value="Search Within Results">
<span class="hwp-button__text">Search Within Results</span>
<span aria-hidden="true" class="material-icon">search</span>
</button>
<input autocomplete="off" data-drupal-selector="form-imypagg73lxskojfuiqljy0prljdf1zrgbqht0wzqqg" name="form_build_id" type="hidden" value="form-IMyPAgg73LxsKOjFuiQLJy0PRLJdF1zrgBQht0WzQQg"/>
<input data-drupal-selector="edit-hwp-search-form-listing" name="form_id" type="hidden" value="hwp_search_form_listing"/>
</form>
</div>
<div class="views-element-container">
<div class="js-view-dom-id-ddcb69bd5855ccc9e781df57ec03f7ba1cb5ed7530fbfada62e7b414010061bf">
<div class="search-page__search-header">
<div class="hwp-hidden lg:hwp-block search-page__results-summary">
        100 results 
      </div>
<button aria-expanded="false" class="search-page__filters-trigger" data-filters-trigger="">
<span>Show filters</span>
<span aria-hidden="true" class="material-icon">filter_alt</span>
</button>
<form accept-charset="UTF-8" action="/node/1774776" class="views-exposed-form bef-exposed-form" data-bef-auto-submit="" data-bef-auto-submit-delay="500" data-bef-auto-submit-full-form="" data-bef-auto-submit-minimum-length="3" data-drupal-selector="views-exposed-form-hwp-search-search-page" id="views-exposed-form-hwp-search-search-page" method="get">
<input data-drupal-selector="edit-search" name="search" type="hidden" value=""/>
<div class="js-form-item form-item hwp-form-item js-form-type-select hwp-form-type-select form-item-sort-bef-combine js-form-item-sort-bef-combine">
<label for="edit-sort-bef-combine">Sort by</label>
<select class="form-select" data-drupal-selector="edit-sort-bef-combine" id="edit-sort-bef-combine" name="sort_bef_combine"><option selected="selected" value="hwp_bibcite_reference_year--desc">Year of Publication</option><option value="title--asc">Alphabetical A-Z</option></select>
<span aria-hidden="true" class="material-icon">sort</span>
</div>
<div class="form-actions js-form-wrapper form-wrapper" data-drupal-selector="edit-actions" id="edit-actions"><input class="js-hide button js-form-submit form-submit" data-bef-auto-submit-click="" data-drupal-selector="edit-submit-hwp-search" id="edit-submit-hwp-search" type="submit" value="Apply"/>
</div>
</form>
</div>
<div class="lg:hwp-hidden search-page__results-summary">
    100 results 
  </div>
<div class="search-page__result-items search-page__result-items--">
<div class="hwp-citations-list__downloads">
<div class="button-dropdown" data-component-name="button-dropdown">
<button class="hwp-text-link hwp-text-link--icon-right" id="citations_download">
<span class="hwp-text-link__text">    Download 100 citations  </span>
<span aria-hidden="true" class="material-icon">download</span>
</button>
<ul aria-labelledby="citations_download" class="button-dropdown__items" role="menu">
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=bibtex">BibTeX</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=endnote8">EndNote X3 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=endnote7">EndNote 7 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=tagged">Endnote tagged</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=marc">Marc</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=pubmed_id">PubMedId</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=ris">RIS</a>
</li>
</ul>
</div>
</div>
<div id="">
<div class="hwp-page-list__items">
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">2025</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Alkuraya I, Almansa AS, Eleonu A, Avillach P, Poduri A, Srivastava S. <a href="/publication/use-computational-phenotypes-predicting-genetic-subgroups-cerebral-palsy">Use of Computational Phenotypes for Predicting Genetic Subgroups of Cerebral Palsy.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2025; PMID: 39990554</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Alkuraya I, Almansa AS, Eleonu A, Avillach P, Poduri A, Srivastava S. <a href="/publication/use-computational-phenotypes-predicting-genetic-subgroups-cerebral-palsy">Use of Computational Phenotypes for Predicting Genetic Subgroups of Cerebral Palsy.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2025; PMID: 39990554</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-983783844" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-983783844">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>INTRODUCTION: Emerging evidence suggests that 20-30% of cases of cerebral palsy (CP) may have a genetic cause. Our group previously identified subsets of patients with CP or CP-masquerading conditions who warrant genetic testing, including those with regression or progressive neurological symptoms (CP masqueraders) and those without any known risk factors for CP (cryptogenic CP). Recognition of these subgroups in clinical settings remains challenging.</p><p>METHODS: To address this challenge, we developed and evaluated a computational phenotyping approach using ICD- 9/ICD-10 billing codes to automatically identify patients with unexplained CP or CP-masquerading conditions who may benefit from genetic testing. We applied this computational phenotyping approach to a cohort of 250 participants from the Boston Children's Hospital CP Sequencing Study, aimed at identifying genetic causes in CP and CP-masquerading conditions.</p><p>RESULTS: Manual review served as the gold standard, identifying 8% as CP masqueraders, 42% as cryptogenic CP, and 50% as non-cryptogenic CP. Computational phenotyping based on ICD-9/10 codes achieved a sensitivity of 95%, specificity of 72%, positive predictive value of 77%, and negative predictive value of 94% in identifying cases warranting genetic testing.</p><p>CONCLUSIONS: Our findings demonstrate the feasibility of using computational phenotyping to identify patients with CP or CP- masquerading conditions who warrant genetic testing. Further studies are needed to evaluate the effectiveness and real-world application of this tool in larger healthcare systems. Nonetheless, the computational phenotyping approach holds promise as a possible clinical decision support that could be integrated into electronic health record systems, enhancing clinical workflows and facilitating actionable genetic diagnoses.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yang D, Zhou D, Cai S, Gan Z, Pencina M, Avillach P, Cai T, Hong C. <a href="/publication/robust-automated-harmonization-heterogeneous-data-through-ensemble-machine-learning">Robust Automated Harmonization of Heterogeneous Data Through Ensemble Machine Learning: Algorithm Development and Validation Study.</a>.</div><div class="csl-right-inline">JMIR medical informatics. 2025;13:e54133. PMID: 39844378</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yang D, Zhou D, Cai S, Gan Z, Pencina M, Avillach P, Cai T, Hong C. <a href="/publication/robust-automated-harmonization-heterogeneous-data-through-ensemble-machine-learning">Robust Automated Harmonization of Heterogeneous Data Through Ensemble Machine Learning: Algorithm Development and Validation Study.</a>.</div><div class="csl-right-inline">JMIR medical informatics. 2025;13:e54133. PMID: 39844378</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-1907437921" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-1907437921">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Cohort studies contain rich clinical data across large and diverse patient populations and are a common source of observational data for clinical research. Because large scale cohort studies are both time and resource intensive, one alternative is to harmonize data from existing cohorts through multicohort studies. However, given differences in variable encoding, accurate variable harmonization is difficult.</p><p>OBJECTIVE: We propose SONAR (Semantic and Distribution-Based Harmonization) as a method for harmonizing variables across cohort studies to facilitate multicohort studies.</p><p>METHODS: SONAR used semantic learning from variable descriptions and distribution learning from study participant data. Our method learned an embedding vector for each variable and used pairwise cosine similarity to score the similarity between variables. This approach was built off 3 National Institutes of Health cohorts, including the Cardiovascular Health Study, the Multi-Ethnic Study of Atherosclerosis, and the Women's Health Initiative. We also used gold standard labels to further refine the embeddings in a supervised manner.</p><p>RESULTS: The method was evaluated using manually curated gold standard labels from the 3 National Institutes of Health cohorts. We evaluated both the intracohort and intercohort variable harmonization performance. The supervised SONAR method outperformed existing benchmark methods for almost all intracohort and intercohort comparisons using area under the curve and top-k accuracy metrics. Notably, SONAR was able to significantly improve harmonization of concepts that were difficult for existing semantic methods to harmonize.</p><p>CONCLUSIONS: SONAR achieves accurate variable harmonization within and between cohort studies by harnessing the complementary strengths of semantic learning and variable distribution learning.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Menezes MCS, Hoffmann AF, Tan ALM, Nalbandyan M, Omenn GS, Mazzotti DR, Hernández-Arango A, Visweswaran S, Venkatesh S, Mandl KD, Bourgeois FT, Lee JWK, Makmur A, Hanauer DA, Semanik MG, Kerivan LT, Hill T, Forero J, Restrepo C, Vigna M, Ceriana P, Abu-El-Rub N, Avillach P, Bellazzi R, Callaci T, Gutiérrez-Sacristán A, Malovini A, Mathew JP, Morris M, Murthy VL, Buonocore TM, Parimbelli E, Patel LP, Saez C, Samayamuthu MJ, Thompson JA, Tibollo V, Xia Z, Kohane IS, Records C for CC of C 19 by EH. <a href="/publication/potential-generative-pre-trained-transformer-4-gpt-4-analyse-medical-notes-three">The potential of Generative Pre-trained Transformer 4 (GPT-4) to analyse medical notes in three different languages: a retrospective model-evaluation study.</a>.</div><div class="csl-right-inline">The Lancet. Digital health. 2025;7(1):e35-e43. PMID: 39722251</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Menezes MCS, Hoffmann AF, Tan ALM, Nalbandyan M, Omenn GS, Mazzotti DR, Hernández-Arango A, Visweswaran S, Venkatesh S, Mandl KD, Bourgeois FT, Lee JWK, Makmur A, Hanauer DA, Semanik MG, Kerivan LT, Hill T, Forero J, Restrepo C, Vigna M, Ceriana P, Abu-El-Rub N, Avillach P, Bellazzi R, Callaci T, Gutiérrez-Sacristán A, Malovini A, Mathew JP, Morris M, Murthy VL, Buonocore TM, Parimbelli E, Patel LP, Saez C, Samayamuthu MJ, Thompson JA, Tibollo V, Xia Z, Kohane IS, Records C for CC of C 19 by EH. <a href="/publication/potential-generative-pre-trained-transformer-4-gpt-4-analyse-medical-notes-three">The potential of Generative Pre-trained Transformer 4 (GPT-4) to analyse medical notes in three different languages: a retrospective model-evaluation study.</a>.</div><div class="csl-right-inline">The Lancet. Digital health. 2025;7(1):e35-e43. PMID: 39722251</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-303159569" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-303159569">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Patient notes contain substantial information but are difficult for computers to analyse due to their unstructured format. Large-language models (LLMs), such as Generative Pre-trained Transformer 4 (GPT-4), have changed our ability to process text, but we do not know how effectively they handle medical notes. We aimed to assess the ability of GPT-4 to answer predefined questions after reading medical notes in three different languages.</p><p>METHODS: For this retrospective model-evaluation study, we included eight university hospitals from four countries (ie, the USA, Colombia, Singapore, and Italy). Each site submitted seven de-identified medical notes related to seven separate patients to the coordinating centre between June 1, 2023, and Feb 28, 2024. Medical notes were written between Feb 1, 2020, and June 1, 2023. One site provided medical notes in Spanish, one site provided notes in Italian, and the remaining six sites provided notes in English. We included admission notes, progress notes, and consultation notes. No discharge summaries were included in this study. We advised participating sites to choose medical notes that, at time of hospital admission, were for patients who were male or female, aged 18-65 years, had a diagnosis of obesity, had a diagnosis of COVID-19, and had submitted an admission note. Adherence to these criteria was optional and participating sites randomly chose which medical notes to submit. When entering information into GPT-4, we prepended each medical note with an instruction prompt and a list of 14 questions that had been chosen a priori. Each medical note was individually given to GPT-4 in its original language and in separate sessions; the questions were always given in English. At each site, two physicians independently validated responses by GPT-4 and responded to all 14 questions. Each pair of physicians evaluated responses from GPT-4 to the seven medical notes from their own site only. Physicians were not masked to responses from GPT-4 before providing their own answers, but were masked to responses from the other physician.</p><p>FINDINGS: We collected 56 medical notes, of which 42 (75%) were in English, seven (13%) were in Italian, and seven (13%) were in Spanish. For each medical note, GPT-4 responded to 14 questions, resulting in 784 responses. In 622 (79%, 95% CI 76-82) of 784 responses, both physicians agreed with GPT-4. In 82 (11%, 8-13) responses, only one physician agreed with GPT-4. In the remaining 80 (10%, 8-13) responses, neither physician agreed with GPT-4. Both physicians agreed with GPT-4 more often for medical notes written in Spanish (86 [88%, 95% CI 79-93] of 98 responses) and Italian (82 [84%, 75-90] of 98 responses) than in English (454 [77%, 74-80] of 588 responses).</p><p>INTERPRETATION: The results of our model-evaluation study suggest that GPT-4 is accurate when analysing medical notes in three different languages. In the future, research should explore how LLMs can be integrated into clinical workflows to maximise their use in health care.</p><p>FUNDING: None.</p></div>
</div>
</div>
</div>
</div>
</div>
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">2024</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gutiérrez-Sacristán A, Makwana S, Dionne A, Mahanta S, Dyer KJ, Serrano F, Watrin C, Pages P, Mousavi S, Degala A, Lyons J, Pillion D, Zachariasse JM, Shekerdemian LS, Truong DT, Newburger JW, Avillach P. <a href="/publication/development-and-validation-open-source-pipeline-automatic-population-case-report-forms">Development and validation of an open-source pipeline for automatic population of case report forms from electronic health records: a pediatric multi-center prospective study.</a>.</div><div class="csl-right-inline">EBioMedicine. 2024;108:105337. PMID: 39288532</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gutiérrez-Sacristán A, Makwana S, Dionne A, Mahanta S, Dyer KJ, Serrano F, Watrin C, Pages P, Mousavi S, Degala A, Lyons J, Pillion D, Zachariasse JM, Shekerdemian LS, Truong DT, Newburger JW, Avillach P. <a href="/publication/development-and-validation-open-source-pipeline-automatic-population-case-report-forms">Development and validation of an open-source pipeline for automatic population of case report forms from electronic health records: a pediatric multi-center prospective study.</a>.</div><div class="csl-right-inline">EBioMedicine. 2024;108:105337. PMID: 39288532</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-613921253" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-613921253">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Clinical trials and registry studies are essential for advancing research and developing novel treatments. However, these studies rely on manual entry of thousands of variables for each patient. Repurposing real-world data can significantly simplify the data collection, reduce transcription errors, and make the data entry process more efficient, consistent, and cost-effective.</p><p>METHODS: We developed an open-source computational pipeline to collect laboratory and medication information from the electronic health record (EHR) data and populate case report forms. The pipeline was developed and validated with data from two independent pediatric hospitals in the US as part of the Long-terM OUtcomes after Multisystem Inflammatory Syndrome In Children (MUSIC) study. Our pipeline allowed the completion of two of the most time-consuming forms. We compared automatically extracted results with manually entered values in one hospital and applied the pipeline to a second hospital, where the output served as the primary data source for case report forms.</p><p>FINDINGS: We extracted and populated 51,845 laboratory and 4913 medication values for 159 patients in two hospitals participating in a prospective pediatric study. We evaluated pipeline performance against data for 104 patients manually entered by clinicians in one of the hospitals. The highest concordance was found during patient hospitalization, with 91.59% of the automatically extracted laboratory and medication values corresponding with the manually entered values. In addition to the successfully populated values, we identified an additional 13,396 laboratory and 567 medication values of interest for the study.</p><p>INTERPRETATION: The automatic data entry of laboratory and medication values during admission is feasible and has a high concordance with the manually entered data. By implementing this proof of concept, we demonstrate the quality of automatic data extraction and highlight the potential of secondary use of EHR data to advance medical science by improving data entry efficiency and expediting clinical research.</p><p>FUNDING: NIH Grant 1OT3HL147154-01, U24HL135691, UG1HL135685.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Wack M, Hassen-Khodja C, McDuffie MT, Bliss G, Horn EJ, Kothari C, McLarney B, Davis R, Hanson K, O’Boyle M, Betancur C, Avillach P. <a href="/publication/phenome-wide-profiling-identifies-genotype-phenotype-associations-phelan-mcdermid">Phenome-wide profiling identifies genotype-phenotype associations in Phelan-McDermid syndrome using family-sourced data from an international registry.</a>.</div><div class="csl-right-inline">Molecular autism. 2024;15(1):40. PMID: 39350236</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Wack M, Hassen-Khodja C, McDuffie MT, Bliss G, Horn EJ, Kothari C, McLarney B, Davis R, Hanson K, O’Boyle M, Betancur C, Avillach P. <a href="/publication/phenome-wide-profiling-identifies-genotype-phenotype-associations-phelan-mcdermid">Phenome-wide profiling identifies genotype-phenotype associations in Phelan-McDermid syndrome using family-sourced data from an international registry.</a>.</div><div class="csl-right-inline">Molecular autism. 2024;15(1):40. PMID: 39350236</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-1645799944" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-1645799944">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Phelan-McDermid syndrome (PMS) is a rare neurodevelopmental disorder caused by 22q13 deletions that include the SHANK3 gene or pathogenic sequence variants in SHANK3. It is characterized by global developmental delay, intellectual disability, speech impairment, autism spectrum disorder, and hypotonia; other variable features include epilepsy, brain and renal malformations, and mild dysmorphic features. Here, we conducted genotype-phenotype correlation analyses using the PMS International Registry, a family-driven registry that compiles clinical data in the form of family-reported outcomes and family-sourced genetic test results.</p><p>METHODS: Data from the registry were harmonized and integrated into the i2b2/tranSMART clinical and genomics data warehouse. We gathered information from 401 individuals with 22q13 deletions including SHANK3 (n = 350, ranging in size from 10 kb to 9.1 Mb) or pathogenic or likely pathogenic SHANK3 sequence variants (n = 51), and used regression models with deletion size as a potential predictor of clinical outcomes for 328 phenotypes.</p><p>RESULTS: Our results showed that increased deletion size was significantly associated with delay in gross and fine motor acquisitions, a spectrum of conditions related to poor muscle tone, renal malformations, mild dysmorphic features (e.g., large fleshy hands, sacral dimple, dysplastic toenails, supernumerary teeth), lymphedema, congenital heart defects, and more frequent neuroimaging abnormalities and infections. These findings indicate that genes upstream of SHANK3 also contribute to some of the manifestations of PMS in individuals with larger deletions. We also showed that self-help skills, verbal ability and a range of psychiatric diagnoses (e.g., autism, ADHD, anxiety disorder) were more common among individuals with smaller deletions and SHANK3 variants.</p><p>LIMITATIONS: Some participants were tested with targeted 22q microarrays rather than genome-wide arrays, and karyotypes were unavailable in many cases, thus precluding the analysis of the effect of other copy number variants or chromosomal rearrangements on the phenotype.</p><p>CONCLUSIONS: This is the largest reported case series of individuals with PMS. Overall, we demonstrate the feasibility of using data from a family-sourced registry to conduct genotype-phenotype analyses in rare genetic disorders. We replicate and strengthen previous findings, and reveal novel associations between larger 22q13 deletions and congenital heart defects, neuroimaging abnormalities and recurrent infections.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hernández-Agosto J, Santiago-Quiñones DI, García-Carmona LM, Gutiérrez-Sacristán A. <a href="/publication/substance-use-disorders-and-disparities-hispanic-and-non-hispanic-older-adults">Substance use disorders and disparities in hispanic and non-hispanic older adults.</a>.</div><div class="csl-right-inline">Drug and alcohol dependence. 2024;261:111357. PMID: 38896947</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hernández-Agosto J, Santiago-Quiñones DI, García-Carmona LM, Gutiérrez-Sacristán A. <a href="/publication/substance-use-disorders-and-disparities-hispanic-and-non-hispanic-older-adults">Substance use disorders and disparities in hispanic and non-hispanic older adults.</a>.</div><div class="csl-right-inline">Drug and alcohol dependence. 2024;261:111357. PMID: 38896947</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2110352893" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-2110352893">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: The global longevity revolution increased the older adult population, posing unique health and economic challenges with implications for healthcare, especially substance use disorders (SUD).</p><p>METHODS: This was a retrospective cohort study of United States older adults, Hispanic and non-Hispanic, who got at least one mental and/or behavioral disorder diagnosis between 2017 and 2021 at age 65 or older. SUD prevalence, prescription frequency changes over time, and comorbidities associated with each medication were compared.</p><p>RESULTS: Electronic health records for 356,133 older adults (110,236 Hispanics and 245,897 non-Hispanics) were analyzed. Notably, 79 % of Hispanics fell below the 100 % federal poverty level, compared to 60 % of non-Hispanics (P&lt;.001). Non-Hispanics also had significantly more average encounters (P=.003) and diagnoses (P&lt;.001). Regression analysis on alcohol-related disorders indicated that the odd ratios of being male (OR=2.93, P&lt;.000), and having low income (OR=1.62, P&lt;.000), increase the odds for this SUD, while being Hispanic and primarily speaking Spanish decreases the odds for all SUDs considered in this study.</p><p>CONCLUSIONS: This cohort study revealed significant disparities related to social determinants of health between Hispanic and non-Hispanic older adults and emphasizes the need for continuous surveillance of older adults as with SUDs. Differences in comorbidity patterns imply distinct risk factors within each population, influenced by demographic-specific elements. Recognizing these variations is essential for tailoring culturally sensitive prevention, intervention, and treatment strategies to each population's unique needs.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Gutierrez A, Network UD, Kobren SN, Avillach P. <a href="/publication/varppud-variant-post-prioritization-developed-undiagnosed-genetic-disorders">VarPPUD: Variant post prioritization developed for undiagnosed genetic disorders.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2024; PMID: 38699371</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Gutierrez A, Network UD, Kobren SN, Avillach P. <a href="/publication/varppud-variant-post-prioritization-developed-undiagnosed-genetic-disorders">VarPPUD: Variant post prioritization developed for undiagnosed genetic disorders.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2024; PMID: 38699371</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-551639828" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-551639828">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>Rare and ultra-rare genetic conditions are estimated to impact nearly 1 in 17 people worldwide, yet accurately pinpointing the diagnostic variants underlying each of these conditions remains a formidable challenge. Because comprehensive,  functional assessment of all possible genetic variants is infeasible, clinicians instead consider  variant pathogenicity predictions to distinguish plausibly disease-causing from benign variants across the genome. However, in the most difficult undiagnosed cases, such as those accepted to the Undiagnosed Diseases Network (UDN), existing pathogenicity predictions cannot reliably discern true etiological variant(s) from other deleterious candidate variants that were prioritized through N-of-1 efforts. Pinpointing the disease-causing variant from a pool of plausible candidates remains a largely manual effort requiring extensive clinical workups, functional and experimental assays, and eventual identification of genotype- and phenotype-matched individuals. Here, we introduce VarPPUD, a tool trained on prioritized variants from UDN cases, that leverages gene-, amino acid-, and nucleotide-level features to discern pathogenic variants from other deleterious variants that are unlikely to be confirmed as disease relevant. VarPPUD achieves a cross-validated accuracy of 79.3% and precision of 77.5% on a held-out subset of uniquely challenging UDN cases, respectively representing an average 18.6% and 23.4% improvement over nine traditional pathogenicity prediction approaches on this task. We validate VarPPUD's ability to discriminate likely from unlikely pathogenic variants on synthetic, GAN-generated candidate variants as well. Finally, we show how VarPPUD can be probed to evaluate each input feature's importance and contribution toward prediction-an essential step toward understanding the distinct characteristics of newly-uncovered disease-causing variants.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hutch MR, Son J, Le TT, Hong C, Wang X, Abad ZSH, Morris M, Gutiérrez-Sacristán A, Klann JG, Spiridou A, Batugo A, Bellazzi R, Benoit V, Bonzel CL, Bryant WA, Chiudinelli L, Cho K, Das P, González TG, Hanauer DA, Henderson DW, Ho YL, Loh NHW, Makoudjou A, Makwana S, Malovini A, Moal B, Mowery DL, Neuraz A, Samayamuthu MJ, Vidorreta FJS, Schriver ER, Schubert P, Talbert J, Tan ALM, Tan BWL, Tan BWQ, Tibollo V, Tippman P, Verdy G, Yuan W, Avillach P, Gehlenborg N, Omenn GS, EHR C for CC of C 19 by, Visweswaran S, Cai T, Luo Y, Xia Z. <a href="/publication/neurological-diagnoses-hospitalized-covid-19-patients-associated-adverse-outcomes">Neurological diagnoses in hospitalized COVID-19 patients associated with adverse outcomes: A multinational cohort study.</a>.</div><div class="csl-right-inline">PLOS digital health. 2024;3(4):e0000484. PMID: 38620037</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hutch MR, Son J, Le TT, Hong C, Wang X, Abad ZSH, Morris M, Gutiérrez-Sacristán A, Klann JG, Spiridou A, Batugo A, Bellazzi R, Benoit V, Bonzel CL, Bryant WA, Chiudinelli L, Cho K, Das P, González TG, Hanauer DA, Henderson DW, Ho YL, Loh NHW, Makoudjou A, Makwana S, Malovini A, Moal B, Mowery DL, Neuraz A, Samayamuthu MJ, Vidorreta FJS, Schriver ER, Schubert P, Talbert J, Tan ALM, Tan BWL, Tan BWQ, Tibollo V, Tippman P, Verdy G, Yuan W, Avillach P, Gehlenborg N, Omenn GS, EHR C for CC of C 19 by, Visweswaran S, Cai T, Luo Y, Xia Z. <a href="/publication/neurological-diagnoses-hospitalized-covid-19-patients-associated-adverse-outcomes">Neurological diagnoses in hospitalized COVID-19 patients associated with adverse outcomes: A multinational cohort study.</a>.</div><div class="csl-right-inline">PLOS digital health. 2024;3(4):e0000484. PMID: 38620037</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2100453899" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-2100453899">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>Few studies examining the patient outcomes of concurrent neurological manifestations during acute COVID-19 leveraged multinational cohorts of adults and children or distinguished between central and peripheral nervous system (CNS vs. PNS) involvement. Using a federated multinational network in which local clinicians and informatics experts curated the electronic health records data, we evaluated the risk of prolonged hospitalization and mortality in hospitalized COVID-19 patients from 21 healthcare systems across 7 countries. For adults, we used a federated learning approach whereby we ran Cox proportional hazard models locally at each healthcare system and performed a meta-analysis on the aggregated results to estimate the overall risk of adverse outcomes across our geographically diverse populations. For children, we reported descriptive statistics separately due to their low frequency of neurological involvement and poor outcomes. Among the 106,229 hospitalized COVID-19 patients (104,031 patients ≥18 years; 2,198 patients &lt;18 years, January 2020-October 2021), 15,101 (14%) had at least one CNS diagnosis, while 2,788 (3%) had at least one PNS diagnosis. After controlling for demographics and pre-existing conditions, adults with CNS involvement had longer hospital stay (11 versus 6 days) and greater risk of (Hazard Ratio = 1.78) and faster time to death (12 versus 24 days) than patients with no neurological condition (NNC) during acute COVID-19 hospitalization. Adults with PNS involvement also had longer hospital stay but lower risk of mortality than the NNC group. Although children had a low frequency of neurological involvement during COVID-19 hospitalization, a substantially higher proportion of children with CNS involvement died compared to those with NNC (6% vs 1%). Overall, patients with concurrent CNS manifestation during acute COVID-19 hospitalization faced greater risks for adverse clinical outcomes than patients without any neurological diagnosis. Our global informatics framework using a federated approach (versus a centralized data collection approach) has utility for clinical discovery beyond COVID-19.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Huang M, Gutiérrez-Sacristán A, Janiak E, Young K, Starosta A, Blanton K, Azhir A, Goldfarb CN, Kuperwasser F, Schaefer KM, Stoddard RE, Vatsa R, Merz-Herrala AA, Bartz D. <a href="/publication/contraceptive-content-shared-social-media-analysis-twitter">Contraceptive content shared on social media: an analysis of Twitter.</a>.</div><div class="csl-right-inline">Contraception and reproductive medicine. 2024;9(1):5. PMID: 38321582</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Huang M, Gutiérrez-Sacristán A, Janiak E, Young K, Starosta A, Blanton K, Azhir A, Goldfarb CN, Kuperwasser F, Schaefer KM, Stoddard RE, Vatsa R, Merz-Herrala AA, Bartz D. <a href="/publication/contraceptive-content-shared-social-media-analysis-twitter">Contraceptive content shared on social media: an analysis of Twitter.</a>.</div><div class="csl-right-inline">Contraception and reproductive medicine. 2024;9(1):5. PMID: 38321582</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-1649283050" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-1649283050">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Information on social media may affect peoples' contraceptive decision making. We performed an exploratory analysis of contraceptive content on Twitter (recently renamed X), a popular social media platform.</p><p>METHODS: We selected a random subset of 1% of publicly available, English-language tweets related to reversible, prescription contraceptive methods posted between January 2014 and December 2019. We oversampled tweets for the contraceptive patch to ensure at least 200 tweets per method. To create the codebook, we identified common themes specific to tweet content topics, tweet sources, and tweets soliciting information or providing advice. All posts were coded by two team members, and differences were adjudicated by a third reviewer. Descriptive analyses were reported with accompanying qualitative findings.</p><p>RESULTS: During the study period, 457,369 tweets about reversible contraceptive methods were published, with a random sample of 4,434 tweets used for final analysis. Tweets most frequently discussed contraceptive method decision-making (26.7%) and side effects (20.5%), particularly for long-acting reversible contraceptive methods and the depot medroxyprogesterone acetate shot. Tweets about logistics of use or adherence were common for short-acting reversible contraceptives. Tweets were frequently posted by contraceptive consumers (50.6%). A small proportion of tweets explicitly requested information (6.2%) or provided advice (4.2%).</p><p>CONCLUSIONS: Clinicians should be aware that individuals are exposed to information through Twitter that may affect contraceptive perceptions and decision making, particularly regarding long-acting reversible contraceptives. Social media is a valuable source for studying contraceptive beliefs missing in traditional health research and may be used by professionals to disseminate accurate contraceptive information.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gargano MA, Matentzoglu N, Coleman B, Addo-Lartey EB, Anagnostopoulos A V, Anderton J, Avillach P, Bagley AM, Bakštein E, Balhoff JP, Baynam G, Bello SM, Berk M, Bertram H, Bishop S, Blau H, Bodenstein DF, Botas P, Boztug K, Čady J, Callahan TJ, Cameron R, Carbon SJ, Castellanos F, Caufield H, Chan LE, Chute CG, Cruz-Rojo J, Dahan-Oliel N, Davids JR, de Dieuleveult M, de Souza V, de Vries BBA, de Vries E, DePaulo R, Derfalvi B, Dhombres F, Diaz-Byrd C, Dingemans AJM, Donadille B, Duyzend M, Elfeky R, Essaid S, Fabrizzi C, Fico G, Firth H V, Freudenberg-Hua Y, Fullerton JM, Gabriel DL, Gilmour K, Giordano J, Goes FS, Moses RG, Green I, Griese M, Groza T, Gu W, Guthrie J, Gyori B, Hamosh A, Hanauer M, Hanušová K, He YO, Hegde H, Helbig I, Holasová K, Hoyt CT, Huang S, Hurwitz E, Jacobsen JOB, Jiang X, Joseph L, Keramatian K, King B, Knoflach K, Koolen DA, Kraus ML, Kroll C, Kusters M, Ladewig MS, Lagorce D, Lai MC, Lapunzina P, Laraway B, Lewis-Smith D, Li X, Lucano C, Majd M, Marazita ML, Martinez-Glez V, McHenry TH, McInnis MG, McMurry JA, Mihulová M, Millett CE, Mitchell PB, Moslerová V, Narutomi K, Nematollahi S, Nevado J, Nierenberg AA, Čajbiková NN, Nurnberger JI, Ogishima S, Olson D, Ortiz A, Pachajoa H, de Nanclares GP, Peters A, Putman T, Rapp CK, Rath A, Reese J, Rekerle L, Roberts AM, Roy S, Sanders SJ, Schuetz C, Schulte EC, Schulze TG, Schwarz M, Scott K, Seelow D, Seitz B, Shen Y, Similuk MN, Simon ES, Singh B, Smedley D, Smith CL, Smolinsky JT, Sperry S, Stafford E, Stefancsik R, Steinhaus R, Strawbridge R, Sundaramurthi JC, Talapova P, Castano JAT, Tesner P, Thomas RH, Thurm A, Turnovec M, van Gijn ME, Vasilevsky NA, Vlčková M, Walden A, Wang K, Wapner R, Ware JS, Wiafe AA, Wiafe SA, Wiggins LD, Williams AE, Wu C, Wyrwoll MJ, Xiong H, Yalin N, Yamamoto Y, Yatham LN, Yocum AK, Young AH, Yüksel Z, Zandi PP, Zankl A, Zarante I, Zvolský M, Toro S, Carmody LC, Harris NL, Munoz-Torres MC, Danis D, Mungall CJ, Köhler S, Haendel MA, Robinson PN. <a href="/publication/human-phenotype-ontology-2024-phenotypes-around-world">The Human Phenotype Ontology in 2024: phenotypes around the world.</a>.</div><div class="csl-right-inline">Nucleic acids research. 2024;52(D1):D1333-D1346. PMID: 37953324</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gargano MA, Matentzoglu N, Coleman B, Addo-Lartey EB, Anagnostopoulos A V, Anderton J, Avillach P, Bagley AM, Bakštein E, Balhoff JP, Baynam G, Bello SM, Berk M, Bertram H, Bishop S, Blau H, Bodenstein DF, Botas P, Boztug K, Čady J, Callahan TJ, Cameron R, Carbon SJ, Castellanos F, Caufield H, Chan LE, Chute CG, Cruz-Rojo J, Dahan-Oliel N, Davids JR, de Dieuleveult M, de Souza V, de Vries BBA, de Vries E, DePaulo R, Derfalvi B, Dhombres F, Diaz-Byrd C, Dingemans AJM, Donadille B, Duyzend M, Elfeky R, Essaid S, Fabrizzi C, Fico G, Firth H V, Freudenberg-Hua Y, Fullerton JM, Gabriel DL, Gilmour K, Giordano J, Goes FS, Moses RG, Green I, Griese M, Groza T, Gu W, Guthrie J, Gyori B, Hamosh A, Hanauer M, Hanušová K, He YO, Hegde H, Helbig I, Holasová K, Hoyt CT, Huang S, Hurwitz E, Jacobsen JOB, Jiang X, Joseph L, Keramatian K, King B, Knoflach K, Koolen DA, Kraus ML, Kroll C, Kusters M, Ladewig MS, Lagorce D, Lai MC, Lapunzina P, Laraway B, Lewis-Smith D, Li X, Lucano C, Majd M, Marazita ML, Martinez-Glez V, McHenry TH, McInnis MG, McMurry JA, Mihulová M, Millett CE, Mitchell PB, Moslerová V, Narutomi K, Nematollahi S, Nevado J, Nierenberg AA, Čajbiková NN, Nurnberger JI, Ogishima S, Olson D, Ortiz A, Pachajoa H, de Nanclares GP, Peters A, Putman T, Rapp CK, Rath A, Reese J, Rekerle L, Roberts AM, Roy S, Sanders SJ, Schuetz C, Schulte EC, Schulze TG, Schwarz M, Scott K, Seelow D, Seitz B, Shen Y, Similuk MN, Simon ES, Singh B, Smedley D, Smith CL, Smolinsky JT, Sperry S, Stafford E, Stefancsik R, Steinhaus R, Strawbridge R, Sundaramurthi JC, Talapova P, Castano JAT, Tesner P, Thomas RH, Thurm A, Turnovec M, van Gijn ME, Vasilevsky NA, Vlčková M, Walden A, Wang K, Wapner R, Ware JS, Wiafe AA, Wiafe SA, Wiggins LD, Williams AE, Wu C, Wyrwoll MJ, Xiong H, Yalin N, Yamamoto Y, Yatham LN, Yocum AK, Young AH, Yüksel Z, Zandi PP, Zankl A, Zarante I, Zvolský M, Toro S, Carmody LC, Harris NL, Munoz-Torres MC, Danis D, Mungall CJ, Köhler S, Haendel MA, Robinson PN. <a href="/publication/human-phenotype-ontology-2024-phenotypes-around-world">The Human Phenotype Ontology in 2024: phenotypes around the world.</a>.</div><div class="csl-right-inline">Nucleic acids research. 2024;52(D1):D1333-D1346. PMID: 37953324</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2011850567" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-2011850567">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>The Human Phenotype Ontology (HPO) is a widely used resource that comprehensively organizes and defines the phenotypic features of human disease, enabling computational inference and supporting genomic and phenotypic analyses through semantic similarity and machine learning algorithms. The HPO has widespread applications in clinical diagnostics and translational research, including genomic diagnostics, gene-disease discovery, and cohort analytics. In recent years, groups around the world have developed translations of the HPO from English to other languages, and the HPO browser has been internationalized, allowing users to view HPO term labels and in many cases synonyms and definitions in ten languages in addition to English. Since our last report, a total of 2239 new HPO terms and 49235 new HPO annotations were developed, many in collaboration with external groups in the fields of psychiatry, arthrogryposis, immunology and cardiology. The Medical Action Ontology (MAxO) is a new effort to model treatments and other measures taken for clinical management. Finally, the HPO consortium is contributing to efforts to integrate the HPO and the GA4GH Phenopacket Schema into electronic health records (EHRs) with the goal of more standardized and computable integration of rare disease data in EHRs.</p></div>
</div>
</div>
</div>
</div>
</div>
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">2023</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Sperotto F, Gutiérrez-Sacristán A, Makwana S, Li X, Rofeberg VN, Cai T, Bourgeois FT, Omenn GS, Hanauer DA, Saez C, Bonzel CL, Bucholz E, Dionne A, Elias MD, García-Barrio N, González TG, Issitt RW, Kernan KF, Laird-Gion J, Maidlow SE, Mandl KD, Ahooyi TM, Moraleda C, Morris M, Moshal KL, Pedrera-Jiménez M, Shah MA, South AM, Spiridou A, Taylor DM, Verdy G, Visweswaran S, Wang X, Xia Z, Zachariasse JM, EHR C for CC of C 19 by, Newburger JW, Avillach P. <a href="/publication/clinical-phenotypes-and-outcomes-children-multisystem-inflammatory-syndrome-across-0">Clinical phenotypes and outcomes in children with multisystem inflammatory syndrome across SARS-CoV-2 variant eras: a multinational study from the 4CE consortium.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102212. PMID: 37745025</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Sperotto F, Gutiérrez-Sacristán A, Makwana S, Li X, Rofeberg VN, Cai T, Bourgeois FT, Omenn GS, Hanauer DA, Saez C, Bonzel CL, Bucholz E, Dionne A, Elias MD, García-Barrio N, González TG, Issitt RW, Kernan KF, Laird-Gion J, Maidlow SE, Mandl KD, Ahooyi TM, Moraleda C, Morris M, Moshal KL, Pedrera-Jiménez M, Shah MA, South AM, Spiridou A, Taylor DM, Verdy G, Visweswaran S, Wang X, Xia Z, Zachariasse JM, EHR C for CC of C 19 by, Newburger JW, Avillach P. <a href="/publication/clinical-phenotypes-and-outcomes-children-multisystem-inflammatory-syndrome-across-0">Clinical phenotypes and outcomes in children with multisystem inflammatory syndrome across SARS-CoV-2 variant eras: a multinational study from the 4CE consortium.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102212. PMID: 37745025</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-87103762" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-87103762">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Multisystem inflammatory syndrome in children (MIS-C) is a severe complication of SARS-CoV-2 infection. It remains unclear how MIS-C phenotypes vary across SARS-CoV-2 variants. We aimed to investigate clinical characteristics and outcomes of MIS-C across SARS-CoV-2 eras.</p><p>METHODS: We performed a multicentre observational retrospective study including seven paediatric hospitals in four countries (France, Spain, U.K., and U.S.). All consecutive confirmed patients with MIS-C hospitalised between February 1st, 2020, and May 31st, 2022, were included. Electronic Health Records (EHR) data were used to calculate pooled risk differences (RD) and effect sizes (ES) at site level, using  as reference. Meta-analysis was used to pool data across sites.</p><p>FINDINGS: Of 598 patients with MIS-C (61% male, 39% female; mean age 9.7 years [SD 4.5]), 383 (64%) were admitted in the  era, 111 (19%) in the  era, and 104 (17%) in the  era. Compared with patients admitted in the  era, those admitted in the  era were younger (ES -1.18 years [95% CI -2.05, -0.32]), had fewer respiratory symptoms (RD -0.15 [95% CI -0.33, -0.04]), less frequent non-cardiogenic shock or systemic inflammatory response syndrome (SIRS) (RD -0.35 [95% CI -0.64, -0.07]), lower lymphocyte count (ES -0.16 × 10/uL [95% CI -0.30, -0.01]), lower C-reactive protein (ES -28.5 mg/L [95% CI -46.3, -10.7]), and lower troponin (ES -0.14 ng/mL [95% CI -0.26, -0.03]). Patients admitted in the  versus  eras were younger (ES -1.6 years [95% CI -2.5, -0.8]), had less frequent SIRS (RD -0.18 [95% CI -0.30, -0.05]), lower lymphocyte count (ES -0.39 × 10/uL [95% CI -0.52, -0.25]), lower troponin (ES -0.16 ng/mL [95% CI -0.30, -0.01]) and less frequently received anticoagulation therapy (RD -0.19 [95% CI -0.37, -0.04]). Length of hospitalization was shorter in the  versus  eras (-1.3 days [95% CI -2.3, -0.4]).</p><p>INTERPRETATION: Our study suggested that MIS-C clinical phenotypes varied across SARS-CoV-2 eras, with patients in  and  eras being younger and less sick. EHR data can be effectively leveraged to identify rare complications of pandemic diseases and their variation over time.</p><p>FUNDING: None.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Dagliati A, Strasser ZH, Abad ZSH, Klann JG, Wagholikar KB, Mesa R, Visweswaran S, Morris M, Luo Y, Henderson DW, Samayamuthu MJ, Tan BWQ, Verdy G, Omenn GS, Xia Z, Bellazzi R, EHR C for CC of C 19 by, Murphy SN, Holmes JH, Estiri H, EHR C for CC of C 19 by. <a href="/publication/characterization-long-covid-temporal-sub-phenotypes-distributed-representation-0">Characterization of long COVID temporal sub-phenotypes by distributed representation learning from electronic health record data: a cohort study.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102210. PMID: 37745021</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Dagliati A, Strasser ZH, Abad ZSH, Klann JG, Wagholikar KB, Mesa R, Visweswaran S, Morris M, Luo Y, Henderson DW, Samayamuthu MJ, Tan BWQ, Verdy G, Omenn GS, Xia Z, Bellazzi R, EHR C for CC of C 19 by, Murphy SN, Holmes JH, Estiri H, EHR C for CC of C 19 by. <a href="/publication/characterization-long-covid-temporal-sub-phenotypes-distributed-representation-0">Characterization of long COVID temporal sub-phenotypes by distributed representation learning from electronic health record data: a cohort study.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102210. PMID: 37745021</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-250295374" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-250295374">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Characterizing Post-Acute Sequelae of COVID (SARS-CoV-2 Infection), or  has been challenging due to the multitude of sub-phenotypes, temporal attributes, and definitions. Scalable characterization of PASC sub-phenotypes can enhance screening capacities, disease management, and treatment planning.</p><p>METHODS: We conducted a retrospective multi-centre observational cohort study, leveraging longitudinal electronic health record (EHR) data of 30,422 patients from three healthcare systems in the Consortium for the Clinical Characterization of COVID-19 by EHR (4CE). From the total cohort, we applied a deductive approach on 12,424 individuals with follow-up data and developed a distributed representation learning process for providing augmented definitions for PASC sub-phenotypes.</p><p>FINDINGS: Our framework characterized seven PASC sub-phenotypes. We estimated that on average 15.7% of the hospitalized COVID-19 patients were likely to suffer from at least one PASC symptom and almost 5.98%, on average, had multiple symptoms. Joint pain and dyspnea had the highest prevalence, with an average prevalence of 5.45% and 4.53%, respectively.</p><p>INTERPRETATION: We provided a scalable framework to every participating healthcare system for estimating PASC sub-phenotypes prevalence and temporal attributes, thus developing a unified model that characterizes augmented sub-phenotypes across the different systems.</p><p>FUNDING: Authors are supported by National Institute of Allergy and Infectious Diseases, National Institute on Aging, National Center for Advancing Translational Sciences, National Medical Research Council, National Institute of Neurological Disorders and Stroke, European Union, National Institutes of Health, National Center for Advancing Translational Sciences.</p></div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<nav aria-label="Pagination" class="hwp-pager hwp-pager--theme-light hwp-bg-light-core" role="navigation">
<ul class="hwp-pager__items js-pager__items">
<li>
<div class="hwp-pager__item hwp-pager__item--previous hwp-pager__item--not-active" href="" rel="prev" title="Go to previous page">
<span class="hwp-visually-hidden">Previous page</span>
<span aria-hidden="true" class="material-icon material-icon--chevron-left">chevron_left</span>
</div>
</li>
<li class="hwp-pager__item hwp-pager__item--is-active">
<a aria-current="page" href="?page=0" title="Current page">1</a>
</li>
<li class="hwp-pager__item">
<a href="?page=1" title="Go to page 2">2</a>
</li>
<li class="hwp-pager__item">
<a href="?page=2" title="Go to page 3">3</a>
</li>
<li class="hwp-pager__item hwp-pager__item--last hwp-pager__item--ellipsis">
<a href="?page=8" title="Go to last page">
<span class="hwp-visually-hidden">Last page</span>
<span aria-hidden="true">9</span>
</a>
</li>
<li>
<a class="hwp-pager__item hwp-pager__item--next" href="?page=1" rel="prev" title="Go to next page">
<span class="hwp-visually-hidden">Next page</span>
<span aria-hidden="true" class="material-icon material-icon--chevron-right">chevron_right</span>
</a>
</li>
</ul>
</nav>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/publication/phenome-wide-profiling-identifies-genotype-phenotype-associations-phelan-mcdermid -->
<h1>Phenome-wide profiling identifies genotype-phenotype associations in Phelan-McDermid syndrome using family-sourced data from an international registry. | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/publication/phenome-wide-profiling-identifies-genotype-phenotype-associations-phelan-mcdermid'>avillach-lab.hms.harvard.edu/publication/phenome-wide-profiling-identifies-genotype-phenotype-associations-phelan-mcdermid</a></div>
<main aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="biblio-details">
<div class="hwp-page-title bibcite-reference" lang="en">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Phenome-wide profiling identifies genotype-phenotype associations in Phelan-McDermid syndrome using family-sourced data from an international registry.
    </h1>
</div>
</div>
<div class="md:hwp-pt-64">
<div class="hwp-container hwp-container-px max-lg:hwp-px-0">
<div class="publication-info" data-component-name="publication-info">
<div class="publication-info__image">
</div>
<div class="publication-info__text">
<h4 class="h4 hwp-mb-8">Publication information:</h4>
<div class="publication-info__body">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Wack M, Hassen-Khodja C, McDuffie MT, Bliss G, Horn EJ, Kothari C, McLarney B, Davis R, Hanson K, O’Boyle M, Betancur C, Avillach P. Phenome-wide profiling identifies genotype-phenotype associations in Phelan-McDermid syndrome using family-sourced data from an international registry.</div><div class="csl-right-inline">Molecular autism. 2024;15(1):40. PMID: 39350236</div></div></div>
</div>
</div>
<ul class="publication-info__actions">
<li>
<div class="button-dropdown" data-component-name="button-dropdown">
<button class="hwp-text-link hwp-text-link--icon-left" id="publication_citation_download">
<span aria-hidden="true" class="material-icon">download_for_offline</span>
<span class="hwp-text-link__text">Download citation</span>
</button>
<ul aria-labelledby="publication_citation_download" class="button-dropdown__items" role="menu">
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773911&amp;format=bibtex">BibTeX</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773911&amp;format=endnote8">EndNote X3 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773911&amp;format=endnote7">EndNote 7 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773911&amp;format=tagged">Endnote tagged</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773911&amp;format=marc">Marc</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773911&amp;format=pubmed_id">PubMedId</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773911&amp;format=ris">RIS</a>
</li>
</ul>
</div>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 hwp-pb-64 md:hwp-py-64">
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">Abstract</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-trim hwp-bg-light-base js-hwp-citation" data-component-name="citation">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<p><b>BACKGROUND: </b>Phelan-McDermid syndrome (PMS) is a rare neurodevelopmental disorder caused by 22q13 deletions that include the SHANK3 gene or pathogenic sequence variants in SHANK3. It is characterized by global developmental delay, intellectual disability, speech impairment, autism spectrum disorder, and hypotonia; other variable features include epilepsy, brain and renal malformations, and mild dysmorphic features. Here, we conducted genotype-phenotype correlation analyses using the PMS International Registry, a family-driven registry that compiles clinical data in the form of family-reported outcomes and family-sourced genetic test results.</p><p><b>METHODS: </b>Data from the registry were harmonized and integrated into the i2b2/tranSMART clinical and genomics data warehouse. We gathered information from 401 individuals with 22q13 deletions including SHANK3 (n = 350, ranging in size from 10 kb to 9.1 Mb) or pathogenic or likely pathogenic SHANK3 sequence variants (n = 51), and used regression models with deletion size as a potential predictor of clinical outcomes for 328 phenotypes.</p><p><b>RESULTS: </b>Our results showed that increased deletion size was significantly associated with delay in gross and fine motor acquisitions, a spectrum of conditions related to poor muscle tone, renal malformations, mild dysmorphic features (e.g., large fleshy hands, sacral dimple, dysplastic toenails, supernumerary teeth), lymphedema, congenital heart defects, and more frequent neuroimaging abnormalities and infections. These findings indicate that genes upstream of SHANK3 also contribute to some of the manifestations of PMS in individuals with larger deletions. We also showed that self-help skills, verbal ability and a range of psychiatric diagnoses (e.g., autism, ADHD, anxiety disorder) were more common among individuals with smaller deletions and SHANK3 variants.</p><p><b>LIMITATIONS: </b>Some participants were tested with targeted 22q microarrays rather than genome-wide arrays, and karyotypes were unavailable in many cases, thus precluding the analysis of the effect of other copy number variants or chromosomal rearrangements on the phenotype.</p><p><b>CONCLUSIONS: </b>This is the largest reported case series of individuals with PMS. Overall, we demonstrate the feasibility of using data from a family-sourced registry to conduct genotype-phenotype analyses in rare genetic disorders. We replicate and strengthen previous findings, and reveal novel associations between larger 22q13 deletions and congenital heart defects, neuroimaging abnormalities and recurrent infections.</p>
</div>
<div class="hwp-text-block">
<p><b>BACKGROUND: </b>Phelan-McDermid syndrome (PMS) is a rare neurodevelopmental disorder caused by 22q13 deletions that include the SHANK3 gene or pathogenic sequence variants in SHANK3. It is characterized by global developmental delay, intellectual disability, speech impairment, autism spectrum disorder, and hypotonia; other variable features include epilepsy, brain and renal malformations, and mild dysmorphic features. Here, we conducted genotype-phenotype correlation analyses using the PMS International Registry, a family-driven registry that compiles clinical data in the form of family-reported outcomes and family-sourced genetic test results.</p><p><b>METHODS: </b>Data from the registry were harmonized and integrated into the i2b2/tranSMART clinical and genomics data warehouse. We gathered information from 401 individuals with 22q13 deletions including SHANK3 (n = 350, ranging in size from 10 kb to 9.1 Mb) or pathogenic or likely pathogenic SHANK3 sequence variants (n = 51), and used regression models with deletion size as a potential predictor of clinical outcomes for 328 phenotypes.</p><p><b>RESULTS: </b>Our results showed that increased deletion size was significantly associated with delay in gross and fine motor acquisitions, a spectrum of conditions related to poor muscle tone, renal malformations, mild dysmorphic features (e.g., large fleshy hands, sacral dimple, dysplastic toenails, supernumerary teeth), lymphedema, congenital heart defects, and more frequent neuroimaging abnormalities and infections. These findings indicate that genes upstream of SHANK3 also contribute to some of the manifestations of PMS in individuals with larger deletions. We also showed that self-help skills, verbal ability and a range of psychiatric diagnoses (e.g., autism, ADHD, anxiety disorder) were more common among individuals with smaller deletions and SHANK3 variants.</p><p><b>LIMITATIONS: </b>Some participants were tested with targeted 22q microarrays rather than genome-wide arrays, and karyotypes were unavailable in many cases, thus precluding the analysis of the effect of other copy number variants or chromosomal rearrangements on the phenotype.</p><p><b>CONCLUSIONS: </b>This is the largest reported case series of individuals with PMS. Overall, we demonstrate the feasibility of using data from a family-sourced registry to conduct genotype-phenotype analyses in rare genetic disorders. We replicate and strengthen previous findings, and reveal novel associations between larger 22q13 deletions and congenital heart defects, neuroimaging abnormalities and recurrent infections.</p>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-1243396755" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
</div>
</div>
</div>
<hr class="hwp-text-button-light-secondary hwp-mt-32"/>
</div>
<div class="hwp-news-footer hwp-py-24 lg:hwp-py-64 hwp-container hwp-container-px">
<div class="page-tags">
</div>
<div class="a2a_kit" data-a2a-title="" data-a2a-url="">
<span class="hwp-mb-16 hwp-block">Share on:</span>
<ul aria-label="Share on:" class="md:hwp-inline-flex md:hwp-flex-wrap">
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_facebook hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-facebook"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Facebook</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_twitter hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-twitter"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Twitter</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_linkedin hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-linkedin"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Linkedin</span>
</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/publication/human-phenotype-ontology-2024-phenotypes-around-world -->
<h1>The Human Phenotype Ontology in 2024: phenotypes around the world. | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/publication/human-phenotype-ontology-2024-phenotypes-around-world'>avillach-lab.hms.harvard.edu/publication/human-phenotype-ontology-2024-phenotypes-around-world</a></div>
<main aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="biblio-details">
<div class="hwp-page-title bibcite-reference" lang="en">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      The Human Phenotype Ontology in 2024: phenotypes around the world.
    </h1>
</div>
</div>
<div class="md:hwp-pt-64">
<div class="hwp-container hwp-container-px max-lg:hwp-px-0">
<div class="publication-info" data-component-name="publication-info">
<div class="publication-info__image">
</div>
<div class="publication-info__text">
<h4 class="h4 hwp-mb-8">Publication information:</h4>
<div class="publication-info__body">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gargano MA, Matentzoglu N, Coleman B, Addo-Lartey EB, Anagnostopoulos A V, Anderton J, Avillach P, Bagley AM, Bakštein E, Balhoff JP, Baynam G, Bello SM, Berk M, Bertram H, Bishop S, Blau H, Bodenstein DF, Botas P, Boztug K, Čady J, Callahan TJ, Cameron R, Carbon SJ, Castellanos F, Caufield H, Chan LE, Chute CG, Cruz-Rojo J, Dahan-Oliel N, Davids JR, de Dieuleveult M, de Souza V, de Vries BBA, de Vries E, DePaulo R, Derfalvi B, Dhombres F, Diaz-Byrd C, Dingemans AJM, Donadille B, Duyzend M, Elfeky R, Essaid S, Fabrizzi C, Fico G, Firth H V, Freudenberg-Hua Y, Fullerton JM, Gabriel DL, Gilmour K, Giordano J, Goes FS, Moses RG, Green I, Griese M, Groza T, Gu W, Guthrie J, Gyori B, Hamosh A, Hanauer M, Hanušová K, He YO, Hegde H, Helbig I, Holasová K, Hoyt CT, Huang S, Hurwitz E, Jacobsen JOB, Jiang X, Joseph L, Keramatian K, King B, Knoflach K, Koolen DA, Kraus ML, Kroll C, Kusters M, Ladewig MS, Lagorce D, Lai MC, Lapunzina P, Laraway B, Lewis-Smith D, Li X, Lucano C, Majd M, Marazita ML, Martinez-Glez V, McHenry TH, McInnis MG, McMurry JA, Mihulová M, Millett CE, Mitchell PB, Moslerová V, Narutomi K, Nematollahi S, Nevado J, Nierenberg AA, Čajbiková NN, Nurnberger JI, Ogishima S, Olson D, Ortiz A, Pachajoa H, de Nanclares GP, Peters A, Putman T, Rapp CK, Rath A, Reese J, Rekerle L, Roberts AM, Roy S, Sanders SJ, Schuetz C, Schulte EC, Schulze TG, Schwarz M, Scott K, Seelow D, Seitz B, Shen Y, Similuk MN, Simon ES, Singh B, Smedley D, Smith CL, Smolinsky JT, Sperry S, Stafford E, Stefancsik R, Steinhaus R, Strawbridge R, Sundaramurthi JC, Talapova P, Castano JAT, Tesner P, Thomas RH, Thurm A, Turnovec M, van Gijn ME, Vasilevsky NA, Vlčková M, Walden A, Wang K, Wapner R, Ware JS, Wiafe AA, Wiafe SA, Wiggins LD, Williams AE, Wu C, Wyrwoll MJ, Xiong H, Yalin N, Yamamoto Y, Yatham LN, Yocum AK, Young AH, Yüksel Z, Zandi PP, Zankl A, Zarante I, Zvolský M, Toro S, Carmody LC, Harris NL, Munoz-Torres MC, Danis D, Mungall CJ, Köhler S, Haendel MA, Robinson PN. The Human Phenotype Ontology in 2024: phenotypes around the world.</div><div class="csl-right-inline">Nucleic acids research. 2024;52(D1):D1333-D1346. PMID: 37953324</div></div></div>
</div>
</div>
<ul class="publication-info__actions">
<li>
<div class="button-dropdown" data-component-name="button-dropdown">
<button class="hwp-text-link hwp-text-link--icon-left" id="publication_citation_download">
<span aria-hidden="true" class="material-icon">download_for_offline</span>
<span class="hwp-text-link__text">Download citation</span>
</button>
<ul aria-labelledby="publication_citation_download" class="button-dropdown__items" role="menu">
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773751&amp;format=bibtex">BibTeX</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773751&amp;format=endnote8">EndNote X3 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773751&amp;format=endnote7">EndNote 7 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773751&amp;format=tagged">Endnote tagged</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773751&amp;format=marc">Marc</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773751&amp;format=pubmed_id">PubMedId</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773751&amp;format=ris">RIS</a>
</li>
</ul>
</div>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 hwp-pb-64 md:hwp-py-64">
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">Abstract</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-trim hwp-bg-light-base js-hwp-citation" data-component-name="citation">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<p>The Human Phenotype Ontology (HPO) is a widely used resource that comprehensively organizes and defines the phenotypic features of human disease, enabling computational inference and supporting genomic and phenotypic analyses through semantic similarity and machine learning algorithms. The HPO has widespread applications in clinical diagnostics and translational research, including genomic diagnostics, gene-disease discovery, and cohort analytics. In recent years, groups around the world have developed translations of the HPO from English to other languages, and the HPO browser has been internationalized, allowing users to view HPO term labels and in many cases synonyms and definitions in ten languages in addition to English. Since our last report, a total of 2239 new HPO terms and 49235 new HPO annotations were developed, many in collaboration with external groups in the fields of psychiatry, arthrogryposis, immunology and cardiology. The Medical Action Ontology (MAxO) is a new effort to model treatments and other measures taken for clinical management. Finally, the HPO consortium is contributing to efforts to integrate the HPO and the GA4GH Phenopacket Schema into electronic health records (EHRs) with the goal of more standardized and computable integration of rare disease data in EHRs.</p>
</div>
<div class="hwp-text-block">
<p>The Human Phenotype Ontology (HPO) is a widely used resource that comprehensively organizes and defines the phenotypic features of human disease, enabling computational inference and supporting genomic and phenotypic analyses through semantic similarity and machine learning algorithms. The HPO has widespread applications in clinical diagnostics and translational research, including genomic diagnostics, gene-disease discovery, and cohort analytics. In recent years, groups around the world have developed translations of the HPO from English to other languages, and the HPO browser has been internationalized, allowing users to view HPO term labels and in many cases synonyms and definitions in ten languages in addition to English. Since our last report, a total of 2239 new HPO terms and 49235 new HPO annotations were developed, many in collaboration with external groups in the fields of psychiatry, arthrogryposis, immunology and cardiology. The Medical Action Ontology (MAxO) is a new effort to model treatments and other measures taken for clinical management. Finally, the HPO consortium is contributing to efforts to integrate the HPO and the GA4GH Phenopacket Schema into electronic health records (EHRs) with the goal of more standardized and computable integration of rare disease data in EHRs.</p>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-628850618" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
</div>
</div>
</div>
<hr class="hwp-text-button-light-secondary hwp-mt-32"/>
</div>
<div class="hwp-news-footer hwp-py-24 lg:hwp-py-64 hwp-container hwp-container-px">
<div class="page-tags">
</div>
<div class="a2a_kit" data-a2a-title="" data-a2a-url="">
<span class="hwp-mb-16 hwp-block">Share on:</span>
<ul aria-label="Share on:" class="md:hwp-inline-flex md:hwp-flex-wrap">
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_facebook hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-facebook"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Facebook</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_twitter hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-twitter"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Twitter</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_linkedin hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-linkedin"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Linkedin</span>
</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/publications -->
<h1>Publications | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/publications'>avillach-lab.hms.harvard.edu/publications</a></div>
<main aria-label="Publications" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="search-page search-page--no-filters hwp-listing-page hwp-bibcite_reference">
<div class="hwp-page-title hwp-bg-dark-base" lang="en">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Publications
    </h1>
</div>
</div>
<div class="hwp-section-heading" data-component-name="section-heading">
<div class="hwp-section-heading__container hwp-container-px hwp-container">
<div class="hwp-section-heading__description hwp-text-block">
</div>
</div>
</div>
<div class="search-page__columns hwp-container hwp-container-px">
<div class="search-page__results">
<div class="search-page__search-form">
<form accept-charset="UTF-8" action="/publications" class="hwp-search-form-listing" data-drupal-selector="hwp-search-form-listing" id="hwp-search-form-listing" method="post">
<div class="hwp-form-type-search hwp-form-item js-form-item form-item form-type-search js-form-type-search form-item-search js-form-item-search form-no-label">
<label class="visually-hidden" for="edit-search">Search Within Results</label>
<input class="form-search" data-drupal-selector="edit-search" id="edit-search" maxlength="128" name="search" size="60" type="search" value=""/>
</div>
<button class="hwp-button hwp-button--small button js-form-submit form-submit" data-drupal-selector="edit-submit" data-twig-suggestion="header_site_search_submit" id="edit-submit" name="op" type="submit" value="Search Within Results">
<span class="hwp-button__text">Search Within Results</span>
<span aria-hidden="true" class="material-icon">search</span>
</button>
<input autocomplete="off" data-drupal-selector="form-imypagg73lxskojfuiqljy0prljdf1zrgbqht0wzqqg" name="form_build_id" type="hidden" value="form-IMyPAgg73LxsKOjFuiQLJy0PRLJdF1zrgBQht0WzQQg"/>
<input data-drupal-selector="edit-hwp-search-form-listing" name="form_id" type="hidden" value="hwp_search_form_listing"/>
</form>
</div>
<div class="views-element-container">
<div class="js-view-dom-id-ddcb69bd5855ccc9e781df57ec03f7ba1cb5ed7530fbfada62e7b414010061bf">
<div class="search-page__search-header">
<div class="hwp-hidden lg:hwp-block search-page__results-summary">
        100 results 
      </div>
<button aria-expanded="false" class="search-page__filters-trigger" data-filters-trigger="">
<span>Show filters</span>
<span aria-hidden="true" class="material-icon">filter_alt</span>
</button>
<form accept-charset="UTF-8" action="/node/1774776" class="views-exposed-form bef-exposed-form" data-bef-auto-submit="" data-bef-auto-submit-delay="500" data-bef-auto-submit-full-form="" data-bef-auto-submit-minimum-length="3" data-drupal-selector="views-exposed-form-hwp-search-search-page" id="views-exposed-form-hwp-search-search-page" method="get">
<input data-drupal-selector="edit-search" name="search" type="hidden" value=""/>
<div class="js-form-item form-item hwp-form-item js-form-type-select hwp-form-type-select form-item-sort-bef-combine js-form-item-sort-bef-combine">
<label for="edit-sort-bef-combine">Sort by</label>
<select class="form-select" data-drupal-selector="edit-sort-bef-combine" id="edit-sort-bef-combine" name="sort_bef_combine"><option selected="selected" value="hwp_bibcite_reference_year--desc">Year of Publication</option><option value="title--asc">Alphabetical A-Z</option></select>
<span aria-hidden="true" class="material-icon">sort</span>
</div>
<div class="form-actions js-form-wrapper form-wrapper" data-drupal-selector="edit-actions" id="edit-actions"><input class="js-hide button js-form-submit form-submit" data-bef-auto-submit-click="" data-drupal-selector="edit-submit-hwp-search" id="edit-submit-hwp-search" type="submit" value="Apply"/>
</div>
</form>
</div>
<div class="lg:hwp-hidden search-page__results-summary">
    100 results 
  </div>
<div class="search-page__result-items search-page__result-items--">
<div class="hwp-citations-list__downloads">
<div class="button-dropdown" data-component-name="button-dropdown">
<button class="hwp-text-link hwp-text-link--icon-right" id="citations_download">
<span class="hwp-text-link__text">    Download 100 citations  </span>
<span aria-hidden="true" class="material-icon">download</span>
</button>
<ul aria-labelledby="citations_download" class="button-dropdown__items" role="menu">
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=bibtex">BibTeX</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=endnote8">EndNote X3 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=endnote7">EndNote 7 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=tagged">Endnote tagged</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=marc">Marc</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=pubmed_id">PubMedId</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=ris">RIS</a>
</li>
</ul>
</div>
</div>
<div id="">
<div class="hwp-page-list__items">
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">2025</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Alkuraya I, Almansa AS, Eleonu A, Avillach P, Poduri A, Srivastava S. <a href="/publication/use-computational-phenotypes-predicting-genetic-subgroups-cerebral-palsy">Use of Computational Phenotypes for Predicting Genetic Subgroups of Cerebral Palsy.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2025; PMID: 39990554</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Alkuraya I, Almansa AS, Eleonu A, Avillach P, Poduri A, Srivastava S. <a href="/publication/use-computational-phenotypes-predicting-genetic-subgroups-cerebral-palsy">Use of Computational Phenotypes for Predicting Genetic Subgroups of Cerebral Palsy.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2025; PMID: 39990554</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-983783844" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-983783844">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>INTRODUCTION: Emerging evidence suggests that 20-30% of cases of cerebral palsy (CP) may have a genetic cause. Our group previously identified subsets of patients with CP or CP-masquerading conditions who warrant genetic testing, including those with regression or progressive neurological symptoms (CP masqueraders) and those without any known risk factors for CP (cryptogenic CP). Recognition of these subgroups in clinical settings remains challenging.</p><p>METHODS: To address this challenge, we developed and evaluated a computational phenotyping approach using ICD- 9/ICD-10 billing codes to automatically identify patients with unexplained CP or CP-masquerading conditions who may benefit from genetic testing. We applied this computational phenotyping approach to a cohort of 250 participants from the Boston Children's Hospital CP Sequencing Study, aimed at identifying genetic causes in CP and CP-masquerading conditions.</p><p>RESULTS: Manual review served as the gold standard, identifying 8% as CP masqueraders, 42% as cryptogenic CP, and 50% as non-cryptogenic CP. Computational phenotyping based on ICD-9/10 codes achieved a sensitivity of 95%, specificity of 72%, positive predictive value of 77%, and negative predictive value of 94% in identifying cases warranting genetic testing.</p><p>CONCLUSIONS: Our findings demonstrate the feasibility of using computational phenotyping to identify patients with CP or CP- masquerading conditions who warrant genetic testing. Further studies are needed to evaluate the effectiveness and real-world application of this tool in larger healthcare systems. Nonetheless, the computational phenotyping approach holds promise as a possible clinical decision support that could be integrated into electronic health record systems, enhancing clinical workflows and facilitating actionable genetic diagnoses.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yang D, Zhou D, Cai S, Gan Z, Pencina M, Avillach P, Cai T, Hong C. <a href="/publication/robust-automated-harmonization-heterogeneous-data-through-ensemble-machine-learning">Robust Automated Harmonization of Heterogeneous Data Through Ensemble Machine Learning: Algorithm Development and Validation Study.</a>.</div><div class="csl-right-inline">JMIR medical informatics. 2025;13:e54133. PMID: 39844378</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yang D, Zhou D, Cai S, Gan Z, Pencina M, Avillach P, Cai T, Hong C. <a href="/publication/robust-automated-harmonization-heterogeneous-data-through-ensemble-machine-learning">Robust Automated Harmonization of Heterogeneous Data Through Ensemble Machine Learning: Algorithm Development and Validation Study.</a>.</div><div class="csl-right-inline">JMIR medical informatics. 2025;13:e54133. PMID: 39844378</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-1907437921" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-1907437921">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Cohort studies contain rich clinical data across large and diverse patient populations and are a common source of observational data for clinical research. Because large scale cohort studies are both time and resource intensive, one alternative is to harmonize data from existing cohorts through multicohort studies. However, given differences in variable encoding, accurate variable harmonization is difficult.</p><p>OBJECTIVE: We propose SONAR (Semantic and Distribution-Based Harmonization) as a method for harmonizing variables across cohort studies to facilitate multicohort studies.</p><p>METHODS: SONAR used semantic learning from variable descriptions and distribution learning from study participant data. Our method learned an embedding vector for each variable and used pairwise cosine similarity to score the similarity between variables. This approach was built off 3 National Institutes of Health cohorts, including the Cardiovascular Health Study, the Multi-Ethnic Study of Atherosclerosis, and the Women's Health Initiative. We also used gold standard labels to further refine the embeddings in a supervised manner.</p><p>RESULTS: The method was evaluated using manually curated gold standard labels from the 3 National Institutes of Health cohorts. We evaluated both the intracohort and intercohort variable harmonization performance. The supervised SONAR method outperformed existing benchmark methods for almost all intracohort and intercohort comparisons using area under the curve and top-k accuracy metrics. Notably, SONAR was able to significantly improve harmonization of concepts that were difficult for existing semantic methods to harmonize.</p><p>CONCLUSIONS: SONAR achieves accurate variable harmonization within and between cohort studies by harnessing the complementary strengths of semantic learning and variable distribution learning.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Menezes MCS, Hoffmann AF, Tan ALM, Nalbandyan M, Omenn GS, Mazzotti DR, Hernández-Arango A, Visweswaran S, Venkatesh S, Mandl KD, Bourgeois FT, Lee JWK, Makmur A, Hanauer DA, Semanik MG, Kerivan LT, Hill T, Forero J, Restrepo C, Vigna M, Ceriana P, Abu-El-Rub N, Avillach P, Bellazzi R, Callaci T, Gutiérrez-Sacristán A, Malovini A, Mathew JP, Morris M, Murthy VL, Buonocore TM, Parimbelli E, Patel LP, Saez C, Samayamuthu MJ, Thompson JA, Tibollo V, Xia Z, Kohane IS, Records C for CC of C 19 by EH. <a href="/publication/potential-generative-pre-trained-transformer-4-gpt-4-analyse-medical-notes-three">The potential of Generative Pre-trained Transformer 4 (GPT-4) to analyse medical notes in three different languages: a retrospective model-evaluation study.</a>.</div><div class="csl-right-inline">The Lancet. Digital health. 2025;7(1):e35-e43. PMID: 39722251</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Menezes MCS, Hoffmann AF, Tan ALM, Nalbandyan M, Omenn GS, Mazzotti DR, Hernández-Arango A, Visweswaran S, Venkatesh S, Mandl KD, Bourgeois FT, Lee JWK, Makmur A, Hanauer DA, Semanik MG, Kerivan LT, Hill T, Forero J, Restrepo C, Vigna M, Ceriana P, Abu-El-Rub N, Avillach P, Bellazzi R, Callaci T, Gutiérrez-Sacristán A, Malovini A, Mathew JP, Morris M, Murthy VL, Buonocore TM, Parimbelli E, Patel LP, Saez C, Samayamuthu MJ, Thompson JA, Tibollo V, Xia Z, Kohane IS, Records C for CC of C 19 by EH. <a href="/publication/potential-generative-pre-trained-transformer-4-gpt-4-analyse-medical-notes-three">The potential of Generative Pre-trained Transformer 4 (GPT-4) to analyse medical notes in three different languages: a retrospective model-evaluation study.</a>.</div><div class="csl-right-inline">The Lancet. Digital health. 2025;7(1):e35-e43. PMID: 39722251</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-303159569" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-303159569">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Patient notes contain substantial information but are difficult for computers to analyse due to their unstructured format. Large-language models (LLMs), such as Generative Pre-trained Transformer 4 (GPT-4), have changed our ability to process text, but we do not know how effectively they handle medical notes. We aimed to assess the ability of GPT-4 to answer predefined questions after reading medical notes in three different languages.</p><p>METHODS: For this retrospective model-evaluation study, we included eight university hospitals from four countries (ie, the USA, Colombia, Singapore, and Italy). Each site submitted seven de-identified medical notes related to seven separate patients to the coordinating centre between June 1, 2023, and Feb 28, 2024. Medical notes were written between Feb 1, 2020, and June 1, 2023. One site provided medical notes in Spanish, one site provided notes in Italian, and the remaining six sites provided notes in English. We included admission notes, progress notes, and consultation notes. No discharge summaries were included in this study. We advised participating sites to choose medical notes that, at time of hospital admission, were for patients who were male or female, aged 18-65 years, had a diagnosis of obesity, had a diagnosis of COVID-19, and had submitted an admission note. Adherence to these criteria was optional and participating sites randomly chose which medical notes to submit. When entering information into GPT-4, we prepended each medical note with an instruction prompt and a list of 14 questions that had been chosen a priori. Each medical note was individually given to GPT-4 in its original language and in separate sessions; the questions were always given in English. At each site, two physicians independently validated responses by GPT-4 and responded to all 14 questions. Each pair of physicians evaluated responses from GPT-4 to the seven medical notes from their own site only. Physicians were not masked to responses from GPT-4 before providing their own answers, but were masked to responses from the other physician.</p><p>FINDINGS: We collected 56 medical notes, of which 42 (75%) were in English, seven (13%) were in Italian, and seven (13%) were in Spanish. For each medical note, GPT-4 responded to 14 questions, resulting in 784 responses. In 622 (79%, 95% CI 76-82) of 784 responses, both physicians agreed with GPT-4. In 82 (11%, 8-13) responses, only one physician agreed with GPT-4. In the remaining 80 (10%, 8-13) responses, neither physician agreed with GPT-4. Both physicians agreed with GPT-4 more often for medical notes written in Spanish (86 [88%, 95% CI 79-93] of 98 responses) and Italian (82 [84%, 75-90] of 98 responses) than in English (454 [77%, 74-80] of 588 responses).</p><p>INTERPRETATION: The results of our model-evaluation study suggest that GPT-4 is accurate when analysing medical notes in three different languages. In the future, research should explore how LLMs can be integrated into clinical workflows to maximise their use in health care.</p><p>FUNDING: None.</p></div>
</div>
</div>
</div>
</div>
</div>
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">2024</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gutiérrez-Sacristán A, Makwana S, Dionne A, Mahanta S, Dyer KJ, Serrano F, Watrin C, Pages P, Mousavi S, Degala A, Lyons J, Pillion D, Zachariasse JM, Shekerdemian LS, Truong DT, Newburger JW, Avillach P. <a href="/publication/development-and-validation-open-source-pipeline-automatic-population-case-report-forms">Development and validation of an open-source pipeline for automatic population of case report forms from electronic health records: a pediatric multi-center prospective study.</a>.</div><div class="csl-right-inline">EBioMedicine. 2024;108:105337. PMID: 39288532</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gutiérrez-Sacristán A, Makwana S, Dionne A, Mahanta S, Dyer KJ, Serrano F, Watrin C, Pages P, Mousavi S, Degala A, Lyons J, Pillion D, Zachariasse JM, Shekerdemian LS, Truong DT, Newburger JW, Avillach P. <a href="/publication/development-and-validation-open-source-pipeline-automatic-population-case-report-forms">Development and validation of an open-source pipeline for automatic population of case report forms from electronic health records: a pediatric multi-center prospective study.</a>.</div><div class="csl-right-inline">EBioMedicine. 2024;108:105337. PMID: 39288532</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-613921253" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-613921253">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Clinical trials and registry studies are essential for advancing research and developing novel treatments. However, these studies rely on manual entry of thousands of variables for each patient. Repurposing real-world data can significantly simplify the data collection, reduce transcription errors, and make the data entry process more efficient, consistent, and cost-effective.</p><p>METHODS: We developed an open-source computational pipeline to collect laboratory and medication information from the electronic health record (EHR) data and populate case report forms. The pipeline was developed and validated with data from two independent pediatric hospitals in the US as part of the Long-terM OUtcomes after Multisystem Inflammatory Syndrome In Children (MUSIC) study. Our pipeline allowed the completion of two of the most time-consuming forms. We compared automatically extracted results with manually entered values in one hospital and applied the pipeline to a second hospital, where the output served as the primary data source for case report forms.</p><p>FINDINGS: We extracted and populated 51,845 laboratory and 4913 medication values for 159 patients in two hospitals participating in a prospective pediatric study. We evaluated pipeline performance against data for 104 patients manually entered by clinicians in one of the hospitals. The highest concordance was found during patient hospitalization, with 91.59% of the automatically extracted laboratory and medication values corresponding with the manually entered values. In addition to the successfully populated values, we identified an additional 13,396 laboratory and 567 medication values of interest for the study.</p><p>INTERPRETATION: The automatic data entry of laboratory and medication values during admission is feasible and has a high concordance with the manually entered data. By implementing this proof of concept, we demonstrate the quality of automatic data extraction and highlight the potential of secondary use of EHR data to advance medical science by improving data entry efficiency and expediting clinical research.</p><p>FUNDING: NIH Grant 1OT3HL147154-01, U24HL135691, UG1HL135685.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Wack M, Hassen-Khodja C, McDuffie MT, Bliss G, Horn EJ, Kothari C, McLarney B, Davis R, Hanson K, O’Boyle M, Betancur C, Avillach P. <a href="/publication/phenome-wide-profiling-identifies-genotype-phenotype-associations-phelan-mcdermid">Phenome-wide profiling identifies genotype-phenotype associations in Phelan-McDermid syndrome using family-sourced data from an international registry.</a>.</div><div class="csl-right-inline">Molecular autism. 2024;15(1):40. PMID: 39350236</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Wack M, Hassen-Khodja C, McDuffie MT, Bliss G, Horn EJ, Kothari C, McLarney B, Davis R, Hanson K, O’Boyle M, Betancur C, Avillach P. <a href="/publication/phenome-wide-profiling-identifies-genotype-phenotype-associations-phelan-mcdermid">Phenome-wide profiling identifies genotype-phenotype associations in Phelan-McDermid syndrome using family-sourced data from an international registry.</a>.</div><div class="csl-right-inline">Molecular autism. 2024;15(1):40. PMID: 39350236</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-1645799944" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-1645799944">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Phelan-McDermid syndrome (PMS) is a rare neurodevelopmental disorder caused by 22q13 deletions that include the SHANK3 gene or pathogenic sequence variants in SHANK3. It is characterized by global developmental delay, intellectual disability, speech impairment, autism spectrum disorder, and hypotonia; other variable features include epilepsy, brain and renal malformations, and mild dysmorphic features. Here, we conducted genotype-phenotype correlation analyses using the PMS International Registry, a family-driven registry that compiles clinical data in the form of family-reported outcomes and family-sourced genetic test results.</p><p>METHODS: Data from the registry were harmonized and integrated into the i2b2/tranSMART clinical and genomics data warehouse. We gathered information from 401 individuals with 22q13 deletions including SHANK3 (n = 350, ranging in size from 10 kb to 9.1 Mb) or pathogenic or likely pathogenic SHANK3 sequence variants (n = 51), and used regression models with deletion size as a potential predictor of clinical outcomes for 328 phenotypes.</p><p>RESULTS: Our results showed that increased deletion size was significantly associated with delay in gross and fine motor acquisitions, a spectrum of conditions related to poor muscle tone, renal malformations, mild dysmorphic features (e.g., large fleshy hands, sacral dimple, dysplastic toenails, supernumerary teeth), lymphedema, congenital heart defects, and more frequent neuroimaging abnormalities and infections. These findings indicate that genes upstream of SHANK3 also contribute to some of the manifestations of PMS in individuals with larger deletions. We also showed that self-help skills, verbal ability and a range of psychiatric diagnoses (e.g., autism, ADHD, anxiety disorder) were more common among individuals with smaller deletions and SHANK3 variants.</p><p>LIMITATIONS: Some participants were tested with targeted 22q microarrays rather than genome-wide arrays, and karyotypes were unavailable in many cases, thus precluding the analysis of the effect of other copy number variants or chromosomal rearrangements on the phenotype.</p><p>CONCLUSIONS: This is the largest reported case series of individuals with PMS. Overall, we demonstrate the feasibility of using data from a family-sourced registry to conduct genotype-phenotype analyses in rare genetic disorders. We replicate and strengthen previous findings, and reveal novel associations between larger 22q13 deletions and congenital heart defects, neuroimaging abnormalities and recurrent infections.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hernández-Agosto J, Santiago-Quiñones DI, García-Carmona LM, Gutiérrez-Sacristán A. <a href="/publication/substance-use-disorders-and-disparities-hispanic-and-non-hispanic-older-adults">Substance use disorders and disparities in hispanic and non-hispanic older adults.</a>.</div><div class="csl-right-inline">Drug and alcohol dependence. 2024;261:111357. PMID: 38896947</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hernández-Agosto J, Santiago-Quiñones DI, García-Carmona LM, Gutiérrez-Sacristán A. <a href="/publication/substance-use-disorders-and-disparities-hispanic-and-non-hispanic-older-adults">Substance use disorders and disparities in hispanic and non-hispanic older adults.</a>.</div><div class="csl-right-inline">Drug and alcohol dependence. 2024;261:111357. PMID: 38896947</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2110352893" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-2110352893">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: The global longevity revolution increased the older adult population, posing unique health and economic challenges with implications for healthcare, especially substance use disorders (SUD).</p><p>METHODS: This was a retrospective cohort study of United States older adults, Hispanic and non-Hispanic, who got at least one mental and/or behavioral disorder diagnosis between 2017 and 2021 at age 65 or older. SUD prevalence, prescription frequency changes over time, and comorbidities associated with each medication were compared.</p><p>RESULTS: Electronic health records for 356,133 older adults (110,236 Hispanics and 245,897 non-Hispanics) were analyzed. Notably, 79 % of Hispanics fell below the 100 % federal poverty level, compared to 60 % of non-Hispanics (P&lt;.001). Non-Hispanics also had significantly more average encounters (P=.003) and diagnoses (P&lt;.001). Regression analysis on alcohol-related disorders indicated that the odd ratios of being male (OR=2.93, P&lt;.000), and having low income (OR=1.62, P&lt;.000), increase the odds for this SUD, while being Hispanic and primarily speaking Spanish decreases the odds for all SUDs considered in this study.</p><p>CONCLUSIONS: This cohort study revealed significant disparities related to social determinants of health between Hispanic and non-Hispanic older adults and emphasizes the need for continuous surveillance of older adults as with SUDs. Differences in comorbidity patterns imply distinct risk factors within each population, influenced by demographic-specific elements. Recognizing these variations is essential for tailoring culturally sensitive prevention, intervention, and treatment strategies to each population's unique needs.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Gutierrez A, Network UD, Kobren SN, Avillach P. <a href="/publication/varppud-variant-post-prioritization-developed-undiagnosed-genetic-disorders">VarPPUD: Variant post prioritization developed for undiagnosed genetic disorders.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2024; PMID: 38699371</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Gutierrez A, Network UD, Kobren SN, Avillach P. <a href="/publication/varppud-variant-post-prioritization-developed-undiagnosed-genetic-disorders">VarPPUD: Variant post prioritization developed for undiagnosed genetic disorders.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2024; PMID: 38699371</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-551639828" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-551639828">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>Rare and ultra-rare genetic conditions are estimated to impact nearly 1 in 17 people worldwide, yet accurately pinpointing the diagnostic variants underlying each of these conditions remains a formidable challenge. Because comprehensive,  functional assessment of all possible genetic variants is infeasible, clinicians instead consider  variant pathogenicity predictions to distinguish plausibly disease-causing from benign variants across the genome. However, in the most difficult undiagnosed cases, such as those accepted to the Undiagnosed Diseases Network (UDN), existing pathogenicity predictions cannot reliably discern true etiological variant(s) from other deleterious candidate variants that were prioritized through N-of-1 efforts. Pinpointing the disease-causing variant from a pool of plausible candidates remains a largely manual effort requiring extensive clinical workups, functional and experimental assays, and eventual identification of genotype- and phenotype-matched individuals. Here, we introduce VarPPUD, a tool trained on prioritized variants from UDN cases, that leverages gene-, amino acid-, and nucleotide-level features to discern pathogenic variants from other deleterious variants that are unlikely to be confirmed as disease relevant. VarPPUD achieves a cross-validated accuracy of 79.3% and precision of 77.5% on a held-out subset of uniquely challenging UDN cases, respectively representing an average 18.6% and 23.4% improvement over nine traditional pathogenicity prediction approaches on this task. We validate VarPPUD's ability to discriminate likely from unlikely pathogenic variants on synthetic, GAN-generated candidate variants as well. Finally, we show how VarPPUD can be probed to evaluate each input feature's importance and contribution toward prediction-an essential step toward understanding the distinct characteristics of newly-uncovered disease-causing variants.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hutch MR, Son J, Le TT, Hong C, Wang X, Abad ZSH, Morris M, Gutiérrez-Sacristán A, Klann JG, Spiridou A, Batugo A, Bellazzi R, Benoit V, Bonzel CL, Bryant WA, Chiudinelli L, Cho K, Das P, González TG, Hanauer DA, Henderson DW, Ho YL, Loh NHW, Makoudjou A, Makwana S, Malovini A, Moal B, Mowery DL, Neuraz A, Samayamuthu MJ, Vidorreta FJS, Schriver ER, Schubert P, Talbert J, Tan ALM, Tan BWL, Tan BWQ, Tibollo V, Tippman P, Verdy G, Yuan W, Avillach P, Gehlenborg N, Omenn GS, EHR C for CC of C 19 by, Visweswaran S, Cai T, Luo Y, Xia Z. <a href="/publication/neurological-diagnoses-hospitalized-covid-19-patients-associated-adverse-outcomes">Neurological diagnoses in hospitalized COVID-19 patients associated with adverse outcomes: A multinational cohort study.</a>.</div><div class="csl-right-inline">PLOS digital health. 2024;3(4):e0000484. PMID: 38620037</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hutch MR, Son J, Le TT, Hong C, Wang X, Abad ZSH, Morris M, Gutiérrez-Sacristán A, Klann JG, Spiridou A, Batugo A, Bellazzi R, Benoit V, Bonzel CL, Bryant WA, Chiudinelli L, Cho K, Das P, González TG, Hanauer DA, Henderson DW, Ho YL, Loh NHW, Makoudjou A, Makwana S, Malovini A, Moal B, Mowery DL, Neuraz A, Samayamuthu MJ, Vidorreta FJS, Schriver ER, Schubert P, Talbert J, Tan ALM, Tan BWL, Tan BWQ, Tibollo V, Tippman P, Verdy G, Yuan W, Avillach P, Gehlenborg N, Omenn GS, EHR C for CC of C 19 by, Visweswaran S, Cai T, Luo Y, Xia Z. <a href="/publication/neurological-diagnoses-hospitalized-covid-19-patients-associated-adverse-outcomes">Neurological diagnoses in hospitalized COVID-19 patients associated with adverse outcomes: A multinational cohort study.</a>.</div><div class="csl-right-inline">PLOS digital health. 2024;3(4):e0000484. PMID: 38620037</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2100453899" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-2100453899">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>Few studies examining the patient outcomes of concurrent neurological manifestations during acute COVID-19 leveraged multinational cohorts of adults and children or distinguished between central and peripheral nervous system (CNS vs. PNS) involvement. Using a federated multinational network in which local clinicians and informatics experts curated the electronic health records data, we evaluated the risk of prolonged hospitalization and mortality in hospitalized COVID-19 patients from 21 healthcare systems across 7 countries. For adults, we used a federated learning approach whereby we ran Cox proportional hazard models locally at each healthcare system and performed a meta-analysis on the aggregated results to estimate the overall risk of adverse outcomes across our geographically diverse populations. For children, we reported descriptive statistics separately due to their low frequency of neurological involvement and poor outcomes. Among the 106,229 hospitalized COVID-19 patients (104,031 patients ≥18 years; 2,198 patients &lt;18 years, January 2020-October 2021), 15,101 (14%) had at least one CNS diagnosis, while 2,788 (3%) had at least one PNS diagnosis. After controlling for demographics and pre-existing conditions, adults with CNS involvement had longer hospital stay (11 versus 6 days) and greater risk of (Hazard Ratio = 1.78) and faster time to death (12 versus 24 days) than patients with no neurological condition (NNC) during acute COVID-19 hospitalization. Adults with PNS involvement also had longer hospital stay but lower risk of mortality than the NNC group. Although children had a low frequency of neurological involvement during COVID-19 hospitalization, a substantially higher proportion of children with CNS involvement died compared to those with NNC (6% vs 1%). Overall, patients with concurrent CNS manifestation during acute COVID-19 hospitalization faced greater risks for adverse clinical outcomes than patients without any neurological diagnosis. Our global informatics framework using a federated approach (versus a centralized data collection approach) has utility for clinical discovery beyond COVID-19.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Huang M, Gutiérrez-Sacristán A, Janiak E, Young K, Starosta A, Blanton K, Azhir A, Goldfarb CN, Kuperwasser F, Schaefer KM, Stoddard RE, Vatsa R, Merz-Herrala AA, Bartz D. <a href="/publication/contraceptive-content-shared-social-media-analysis-twitter">Contraceptive content shared on social media: an analysis of Twitter.</a>.</div><div class="csl-right-inline">Contraception and reproductive medicine. 2024;9(1):5. PMID: 38321582</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Huang M, Gutiérrez-Sacristán A, Janiak E, Young K, Starosta A, Blanton K, Azhir A, Goldfarb CN, Kuperwasser F, Schaefer KM, Stoddard RE, Vatsa R, Merz-Herrala AA, Bartz D. <a href="/publication/contraceptive-content-shared-social-media-analysis-twitter">Contraceptive content shared on social media: an analysis of Twitter.</a>.</div><div class="csl-right-inline">Contraception and reproductive medicine. 2024;9(1):5. PMID: 38321582</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-1649283050" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-1649283050">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Information on social media may affect peoples' contraceptive decision making. We performed an exploratory analysis of contraceptive content on Twitter (recently renamed X), a popular social media platform.</p><p>METHODS: We selected a random subset of 1% of publicly available, English-language tweets related to reversible, prescription contraceptive methods posted between January 2014 and December 2019. We oversampled tweets for the contraceptive patch to ensure at least 200 tweets per method. To create the codebook, we identified common themes specific to tweet content topics, tweet sources, and tweets soliciting information or providing advice. All posts were coded by two team members, and differences were adjudicated by a third reviewer. Descriptive analyses were reported with accompanying qualitative findings.</p><p>RESULTS: During the study period, 457,369 tweets about reversible contraceptive methods were published, with a random sample of 4,434 tweets used for final analysis. Tweets most frequently discussed contraceptive method decision-making (26.7%) and side effects (20.5%), particularly for long-acting reversible contraceptive methods and the depot medroxyprogesterone acetate shot. Tweets about logistics of use or adherence were common for short-acting reversible contraceptives. Tweets were frequently posted by contraceptive consumers (50.6%). A small proportion of tweets explicitly requested information (6.2%) or provided advice (4.2%).</p><p>CONCLUSIONS: Clinicians should be aware that individuals are exposed to information through Twitter that may affect contraceptive perceptions and decision making, particularly regarding long-acting reversible contraceptives. Social media is a valuable source for studying contraceptive beliefs missing in traditional health research and may be used by professionals to disseminate accurate contraceptive information.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gargano MA, Matentzoglu N, Coleman B, Addo-Lartey EB, Anagnostopoulos A V, Anderton J, Avillach P, Bagley AM, Bakštein E, Balhoff JP, Baynam G, Bello SM, Berk M, Bertram H, Bishop S, Blau H, Bodenstein DF, Botas P, Boztug K, Čady J, Callahan TJ, Cameron R, Carbon SJ, Castellanos F, Caufield H, Chan LE, Chute CG, Cruz-Rojo J, Dahan-Oliel N, Davids JR, de Dieuleveult M, de Souza V, de Vries BBA, de Vries E, DePaulo R, Derfalvi B, Dhombres F, Diaz-Byrd C, Dingemans AJM, Donadille B, Duyzend M, Elfeky R, Essaid S, Fabrizzi C, Fico G, Firth H V, Freudenberg-Hua Y, Fullerton JM, Gabriel DL, Gilmour K, Giordano J, Goes FS, Moses RG, Green I, Griese M, Groza T, Gu W, Guthrie J, Gyori B, Hamosh A, Hanauer M, Hanušová K, He YO, Hegde H, Helbig I, Holasová K, Hoyt CT, Huang S, Hurwitz E, Jacobsen JOB, Jiang X, Joseph L, Keramatian K, King B, Knoflach K, Koolen DA, Kraus ML, Kroll C, Kusters M, Ladewig MS, Lagorce D, Lai MC, Lapunzina P, Laraway B, Lewis-Smith D, Li X, Lucano C, Majd M, Marazita ML, Martinez-Glez V, McHenry TH, McInnis MG, McMurry JA, Mihulová M, Millett CE, Mitchell PB, Moslerová V, Narutomi K, Nematollahi S, Nevado J, Nierenberg AA, Čajbiková NN, Nurnberger JI, Ogishima S, Olson D, Ortiz A, Pachajoa H, de Nanclares GP, Peters A, Putman T, Rapp CK, Rath A, Reese J, Rekerle L, Roberts AM, Roy S, Sanders SJ, Schuetz C, Schulte EC, Schulze TG, Schwarz M, Scott K, Seelow D, Seitz B, Shen Y, Similuk MN, Simon ES, Singh B, Smedley D, Smith CL, Smolinsky JT, Sperry S, Stafford E, Stefancsik R, Steinhaus R, Strawbridge R, Sundaramurthi JC, Talapova P, Castano JAT, Tesner P, Thomas RH, Thurm A, Turnovec M, van Gijn ME, Vasilevsky NA, Vlčková M, Walden A, Wang K, Wapner R, Ware JS, Wiafe AA, Wiafe SA, Wiggins LD, Williams AE, Wu C, Wyrwoll MJ, Xiong H, Yalin N, Yamamoto Y, Yatham LN, Yocum AK, Young AH, Yüksel Z, Zandi PP, Zankl A, Zarante I, Zvolský M, Toro S, Carmody LC, Harris NL, Munoz-Torres MC, Danis D, Mungall CJ, Köhler S, Haendel MA, Robinson PN. <a href="/publication/human-phenotype-ontology-2024-phenotypes-around-world">The Human Phenotype Ontology in 2024: phenotypes around the world.</a>.</div><div class="csl-right-inline">Nucleic acids research. 2024;52(D1):D1333-D1346. PMID: 37953324</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gargano MA, Matentzoglu N, Coleman B, Addo-Lartey EB, Anagnostopoulos A V, Anderton J, Avillach P, Bagley AM, Bakštein E, Balhoff JP, Baynam G, Bello SM, Berk M, Bertram H, Bishop S, Blau H, Bodenstein DF, Botas P, Boztug K, Čady J, Callahan TJ, Cameron R, Carbon SJ, Castellanos F, Caufield H, Chan LE, Chute CG, Cruz-Rojo J, Dahan-Oliel N, Davids JR, de Dieuleveult M, de Souza V, de Vries BBA, de Vries E, DePaulo R, Derfalvi B, Dhombres F, Diaz-Byrd C, Dingemans AJM, Donadille B, Duyzend M, Elfeky R, Essaid S, Fabrizzi C, Fico G, Firth H V, Freudenberg-Hua Y, Fullerton JM, Gabriel DL, Gilmour K, Giordano J, Goes FS, Moses RG, Green I, Griese M, Groza T, Gu W, Guthrie J, Gyori B, Hamosh A, Hanauer M, Hanušová K, He YO, Hegde H, Helbig I, Holasová K, Hoyt CT, Huang S, Hurwitz E, Jacobsen JOB, Jiang X, Joseph L, Keramatian K, King B, Knoflach K, Koolen DA, Kraus ML, Kroll C, Kusters M, Ladewig MS, Lagorce D, Lai MC, Lapunzina P, Laraway B, Lewis-Smith D, Li X, Lucano C, Majd M, Marazita ML, Martinez-Glez V, McHenry TH, McInnis MG, McMurry JA, Mihulová M, Millett CE, Mitchell PB, Moslerová V, Narutomi K, Nematollahi S, Nevado J, Nierenberg AA, Čajbiková NN, Nurnberger JI, Ogishima S, Olson D, Ortiz A, Pachajoa H, de Nanclares GP, Peters A, Putman T, Rapp CK, Rath A, Reese J, Rekerle L, Roberts AM, Roy S, Sanders SJ, Schuetz C, Schulte EC, Schulze TG, Schwarz M, Scott K, Seelow D, Seitz B, Shen Y, Similuk MN, Simon ES, Singh B, Smedley D, Smith CL, Smolinsky JT, Sperry S, Stafford E, Stefancsik R, Steinhaus R, Strawbridge R, Sundaramurthi JC, Talapova P, Castano JAT, Tesner P, Thomas RH, Thurm A, Turnovec M, van Gijn ME, Vasilevsky NA, Vlčková M, Walden A, Wang K, Wapner R, Ware JS, Wiafe AA, Wiafe SA, Wiggins LD, Williams AE, Wu C, Wyrwoll MJ, Xiong H, Yalin N, Yamamoto Y, Yatham LN, Yocum AK, Young AH, Yüksel Z, Zandi PP, Zankl A, Zarante I, Zvolský M, Toro S, Carmody LC, Harris NL, Munoz-Torres MC, Danis D, Mungall CJ, Köhler S, Haendel MA, Robinson PN. <a href="/publication/human-phenotype-ontology-2024-phenotypes-around-world">The Human Phenotype Ontology in 2024: phenotypes around the world.</a>.</div><div class="csl-right-inline">Nucleic acids research. 2024;52(D1):D1333-D1346. PMID: 37953324</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2011850567" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-2011850567">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>The Human Phenotype Ontology (HPO) is a widely used resource that comprehensively organizes and defines the phenotypic features of human disease, enabling computational inference and supporting genomic and phenotypic analyses through semantic similarity and machine learning algorithms. The HPO has widespread applications in clinical diagnostics and translational research, including genomic diagnostics, gene-disease discovery, and cohort analytics. In recent years, groups around the world have developed translations of the HPO from English to other languages, and the HPO browser has been internationalized, allowing users to view HPO term labels and in many cases synonyms and definitions in ten languages in addition to English. Since our last report, a total of 2239 new HPO terms and 49235 new HPO annotations were developed, many in collaboration with external groups in the fields of psychiatry, arthrogryposis, immunology and cardiology. The Medical Action Ontology (MAxO) is a new effort to model treatments and other measures taken for clinical management. Finally, the HPO consortium is contributing to efforts to integrate the HPO and the GA4GH Phenopacket Schema into electronic health records (EHRs) with the goal of more standardized and computable integration of rare disease data in EHRs.</p></div>
</div>
</div>
</div>
</div>
</div>
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">2023</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Sperotto F, Gutiérrez-Sacristán A, Makwana S, Li X, Rofeberg VN, Cai T, Bourgeois FT, Omenn GS, Hanauer DA, Saez C, Bonzel CL, Bucholz E, Dionne A, Elias MD, García-Barrio N, González TG, Issitt RW, Kernan KF, Laird-Gion J, Maidlow SE, Mandl KD, Ahooyi TM, Moraleda C, Morris M, Moshal KL, Pedrera-Jiménez M, Shah MA, South AM, Spiridou A, Taylor DM, Verdy G, Visweswaran S, Wang X, Xia Z, Zachariasse JM, EHR C for CC of C 19 by, Newburger JW, Avillach P. <a href="/publication/clinical-phenotypes-and-outcomes-children-multisystem-inflammatory-syndrome-across-0">Clinical phenotypes and outcomes in children with multisystem inflammatory syndrome across SARS-CoV-2 variant eras: a multinational study from the 4CE consortium.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102212. PMID: 37745025</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Sperotto F, Gutiérrez-Sacristán A, Makwana S, Li X, Rofeberg VN, Cai T, Bourgeois FT, Omenn GS, Hanauer DA, Saez C, Bonzel CL, Bucholz E, Dionne A, Elias MD, García-Barrio N, González TG, Issitt RW, Kernan KF, Laird-Gion J, Maidlow SE, Mandl KD, Ahooyi TM, Moraleda C, Morris M, Moshal KL, Pedrera-Jiménez M, Shah MA, South AM, Spiridou A, Taylor DM, Verdy G, Visweswaran S, Wang X, Xia Z, Zachariasse JM, EHR C for CC of C 19 by, Newburger JW, Avillach P. <a href="/publication/clinical-phenotypes-and-outcomes-children-multisystem-inflammatory-syndrome-across-0">Clinical phenotypes and outcomes in children with multisystem inflammatory syndrome across SARS-CoV-2 variant eras: a multinational study from the 4CE consortium.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102212. PMID: 37745025</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-87103762" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-87103762">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Multisystem inflammatory syndrome in children (MIS-C) is a severe complication of SARS-CoV-2 infection. It remains unclear how MIS-C phenotypes vary across SARS-CoV-2 variants. We aimed to investigate clinical characteristics and outcomes of MIS-C across SARS-CoV-2 eras.</p><p>METHODS: We performed a multicentre observational retrospective study including seven paediatric hospitals in four countries (France, Spain, U.K., and U.S.). All consecutive confirmed patients with MIS-C hospitalised between February 1st, 2020, and May 31st, 2022, were included. Electronic Health Records (EHR) data were used to calculate pooled risk differences (RD) and effect sizes (ES) at site level, using  as reference. Meta-analysis was used to pool data across sites.</p><p>FINDINGS: Of 598 patients with MIS-C (61% male, 39% female; mean age 9.7 years [SD 4.5]), 383 (64%) were admitted in the  era, 111 (19%) in the  era, and 104 (17%) in the  era. Compared with patients admitted in the  era, those admitted in the  era were younger (ES -1.18 years [95% CI -2.05, -0.32]), had fewer respiratory symptoms (RD -0.15 [95% CI -0.33, -0.04]), less frequent non-cardiogenic shock or systemic inflammatory response syndrome (SIRS) (RD -0.35 [95% CI -0.64, -0.07]), lower lymphocyte count (ES -0.16 × 10/uL [95% CI -0.30, -0.01]), lower C-reactive protein (ES -28.5 mg/L [95% CI -46.3, -10.7]), and lower troponin (ES -0.14 ng/mL [95% CI -0.26, -0.03]). Patients admitted in the  versus  eras were younger (ES -1.6 years [95% CI -2.5, -0.8]), had less frequent SIRS (RD -0.18 [95% CI -0.30, -0.05]), lower lymphocyte count (ES -0.39 × 10/uL [95% CI -0.52, -0.25]), lower troponin (ES -0.16 ng/mL [95% CI -0.30, -0.01]) and less frequently received anticoagulation therapy (RD -0.19 [95% CI -0.37, -0.04]). Length of hospitalization was shorter in the  versus  eras (-1.3 days [95% CI -2.3, -0.4]).</p><p>INTERPRETATION: Our study suggested that MIS-C clinical phenotypes varied across SARS-CoV-2 eras, with patients in  and  eras being younger and less sick. EHR data can be effectively leveraged to identify rare complications of pandemic diseases and their variation over time.</p><p>FUNDING: None.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Dagliati A, Strasser ZH, Abad ZSH, Klann JG, Wagholikar KB, Mesa R, Visweswaran S, Morris M, Luo Y, Henderson DW, Samayamuthu MJ, Tan BWQ, Verdy G, Omenn GS, Xia Z, Bellazzi R, EHR C for CC of C 19 by, Murphy SN, Holmes JH, Estiri H, EHR C for CC of C 19 by. <a href="/publication/characterization-long-covid-temporal-sub-phenotypes-distributed-representation-0">Characterization of long COVID temporal sub-phenotypes by distributed representation learning from electronic health record data: a cohort study.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102210. PMID: 37745021</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Dagliati A, Strasser ZH, Abad ZSH, Klann JG, Wagholikar KB, Mesa R, Visweswaran S, Morris M, Luo Y, Henderson DW, Samayamuthu MJ, Tan BWQ, Verdy G, Omenn GS, Xia Z, Bellazzi R, EHR C for CC of C 19 by, Murphy SN, Holmes JH, Estiri H, EHR C for CC of C 19 by. <a href="/publication/characterization-long-covid-temporal-sub-phenotypes-distributed-representation-0">Characterization of long COVID temporal sub-phenotypes by distributed representation learning from electronic health record data: a cohort study.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102210. PMID: 37745021</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-250295374" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-250295374">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Characterizing Post-Acute Sequelae of COVID (SARS-CoV-2 Infection), or  has been challenging due to the multitude of sub-phenotypes, temporal attributes, and definitions. Scalable characterization of PASC sub-phenotypes can enhance screening capacities, disease management, and treatment planning.</p><p>METHODS: We conducted a retrospective multi-centre observational cohort study, leveraging longitudinal electronic health record (EHR) data of 30,422 patients from three healthcare systems in the Consortium for the Clinical Characterization of COVID-19 by EHR (4CE). From the total cohort, we applied a deductive approach on 12,424 individuals with follow-up data and developed a distributed representation learning process for providing augmented definitions for PASC sub-phenotypes.</p><p>FINDINGS: Our framework characterized seven PASC sub-phenotypes. We estimated that on average 15.7% of the hospitalized COVID-19 patients were likely to suffer from at least one PASC symptom and almost 5.98%, on average, had multiple symptoms. Joint pain and dyspnea had the highest prevalence, with an average prevalence of 5.45% and 4.53%, respectively.</p><p>INTERPRETATION: We provided a scalable framework to every participating healthcare system for estimating PASC sub-phenotypes prevalence and temporal attributes, thus developing a unified model that characterizes augmented sub-phenotypes across the different systems.</p><p>FUNDING: Authors are supported by National Institute of Allergy and Infectious Diseases, National Institute on Aging, National Center for Advancing Translational Sciences, National Medical Research Council, National Institute of Neurological Disorders and Stroke, European Union, National Institutes of Health, National Center for Advancing Translational Sciences.</p></div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<nav aria-label="Pagination" class="hwp-pager hwp-pager--theme-light hwp-bg-light-core" role="navigation">
<ul class="hwp-pager__items js-pager__items">
<li>
<div class="hwp-pager__item hwp-pager__item--previous hwp-pager__item--not-active" href="" rel="prev" title="Go to previous page">
<span class="hwp-visually-hidden">Previous page</span>
<span aria-hidden="true" class="material-icon material-icon--chevron-left">chevron_left</span>
</div>
</li>
<li class="hwp-pager__item hwp-pager__item--is-active">
<a aria-current="page" href="?page=0" title="Current page">1</a>
</li>
<li class="hwp-pager__item">
<a href="?page=1" title="Go to page 2">2</a>
</li>
<li class="hwp-pager__item">
<a href="?page=2" title="Go to page 3">3</a>
</li>
<li class="hwp-pager__item hwp-pager__item--last hwp-pager__item--ellipsis">
<a href="?page=8" title="Go to last page">
<span class="hwp-visually-hidden">Last page</span>
<span aria-hidden="true">9</span>
</a>
</li>
<li>
<a class="hwp-pager__item hwp-pager__item--next" href="?page=1" rel="prev" title="Go to next page">
<span class="hwp-visually-hidden">Next page</span>
<span aria-hidden="true" class="material-icon material-icon--chevron-right">chevron_right</span>
</a>
</li>
</ul>
</nav>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/publications -->
<h1>Publications | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/publications'>avillach-lab.hms.harvard.edu/publications</a></div>
<main aria-label="Publications" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="search-page search-page--no-filters hwp-listing-page hwp-bibcite_reference">
<div class="hwp-page-title hwp-bg-dark-base" lang="en">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Publications
    </h1>
</div>
</div>
<div class="hwp-section-heading" data-component-name="section-heading">
<div class="hwp-section-heading__container hwp-container-px hwp-container">
<div class="hwp-section-heading__description hwp-text-block">
</div>
</div>
</div>
<div class="search-page__columns hwp-container hwp-container-px">
<div class="search-page__results">
<div class="search-page__search-form">
<form accept-charset="UTF-8" action="/publications" class="hwp-search-form-listing" data-drupal-selector="hwp-search-form-listing" id="hwp-search-form-listing" method="post">
<div class="hwp-form-type-search hwp-form-item js-form-item form-item form-type-search js-form-type-search form-item-search js-form-item-search form-no-label">
<label class="visually-hidden" for="edit-search">Search Within Results</label>
<input class="form-search" data-drupal-selector="edit-search" id="edit-search" maxlength="128" name="search" size="60" type="search" value=""/>
</div>
<button class="hwp-button hwp-button--small button js-form-submit form-submit" data-drupal-selector="edit-submit" data-twig-suggestion="header_site_search_submit" id="edit-submit" name="op" type="submit" value="Search Within Results">
<span class="hwp-button__text">Search Within Results</span>
<span aria-hidden="true" class="material-icon">search</span>
</button>
<input autocomplete="off" data-drupal-selector="form-imypagg73lxskojfuiqljy0prljdf1zrgbqht0wzqqg" name="form_build_id" type="hidden" value="form-IMyPAgg73LxsKOjFuiQLJy0PRLJdF1zrgBQht0WzQQg"/>
<input data-drupal-selector="edit-hwp-search-form-listing" name="form_id" type="hidden" value="hwp_search_form_listing"/>
</form>
</div>
<div class="views-element-container">
<div class="js-view-dom-id-ddcb69bd5855ccc9e781df57ec03f7ba1cb5ed7530fbfada62e7b414010061bf">
<div class="search-page__search-header">
<div class="hwp-hidden lg:hwp-block search-page__results-summary">
        100 results 
      </div>
<button aria-expanded="false" class="search-page__filters-trigger" data-filters-trigger="">
<span>Show filters</span>
<span aria-hidden="true" class="material-icon">filter_alt</span>
</button>
<form accept-charset="UTF-8" action="/node/1774776" class="views-exposed-form bef-exposed-form" data-bef-auto-submit="" data-bef-auto-submit-delay="500" data-bef-auto-submit-full-form="" data-bef-auto-submit-minimum-length="3" data-drupal-selector="views-exposed-form-hwp-search-search-page" id="views-exposed-form-hwp-search-search-page" method="get">
<input data-drupal-selector="edit-search" name="search" type="hidden" value=""/>
<div class="js-form-item form-item hwp-form-item js-form-type-select hwp-form-type-select form-item-sort-bef-combine js-form-item-sort-bef-combine">
<label for="edit-sort-bef-combine">Sort by</label>
<select class="form-select" data-drupal-selector="edit-sort-bef-combine" id="edit-sort-bef-combine" name="sort_bef_combine"><option selected="selected" value="hwp_bibcite_reference_year--desc">Year of Publication</option><option value="title--asc">Alphabetical A-Z</option></select>
<span aria-hidden="true" class="material-icon">sort</span>
</div>
<div class="form-actions js-form-wrapper form-wrapper" data-drupal-selector="edit-actions" id="edit-actions"><input class="js-hide button js-form-submit form-submit" data-bef-auto-submit-click="" data-drupal-selector="edit-submit-hwp-search" id="edit-submit-hwp-search" type="submit" value="Apply"/>
</div>
</form>
</div>
<div class="lg:hwp-hidden search-page__results-summary">
    100 results 
  </div>
<div class="search-page__result-items search-page__result-items--">
<div class="hwp-citations-list__downloads">
<div class="button-dropdown" data-component-name="button-dropdown">
<button class="hwp-text-link hwp-text-link--icon-right" id="citations_download">
<span class="hwp-text-link__text">    Download 100 citations  </span>
<span aria-hidden="true" class="material-icon">download</span>
</button>
<ul aria-labelledby="citations_download" class="button-dropdown__items" role="menu">
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=bibtex">BibTeX</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=endnote8">EndNote X3 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=endnote7">EndNote 7 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=tagged">Endnote tagged</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=marc">Marc</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=pubmed_id">PubMedId</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/node/1774776/export?format=ris">RIS</a>
</li>
</ul>
</div>
</div>
<div id="">
<div class="hwp-page-list__items">
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">2025</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Alkuraya I, Almansa AS, Eleonu A, Avillach P, Poduri A, Srivastava S. <a href="/publication/use-computational-phenotypes-predicting-genetic-subgroups-cerebral-palsy">Use of Computational Phenotypes for Predicting Genetic Subgroups of Cerebral Palsy.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2025; PMID: 39990554</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Alkuraya I, Almansa AS, Eleonu A, Avillach P, Poduri A, Srivastava S. <a href="/publication/use-computational-phenotypes-predicting-genetic-subgroups-cerebral-palsy">Use of Computational Phenotypes for Predicting Genetic Subgroups of Cerebral Palsy.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2025; PMID: 39990554</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-983783844" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-983783844">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>INTRODUCTION: Emerging evidence suggests that 20-30% of cases of cerebral palsy (CP) may have a genetic cause. Our group previously identified subsets of patients with CP or CP-masquerading conditions who warrant genetic testing, including those with regression or progressive neurological symptoms (CP masqueraders) and those without any known risk factors for CP (cryptogenic CP). Recognition of these subgroups in clinical settings remains challenging.</p><p>METHODS: To address this challenge, we developed and evaluated a computational phenotyping approach using ICD- 9/ICD-10 billing codes to automatically identify patients with unexplained CP or CP-masquerading conditions who may benefit from genetic testing. We applied this computational phenotyping approach to a cohort of 250 participants from the Boston Children's Hospital CP Sequencing Study, aimed at identifying genetic causes in CP and CP-masquerading conditions.</p><p>RESULTS: Manual review served as the gold standard, identifying 8% as CP masqueraders, 42% as cryptogenic CP, and 50% as non-cryptogenic CP. Computational phenotyping based on ICD-9/10 codes achieved a sensitivity of 95%, specificity of 72%, positive predictive value of 77%, and negative predictive value of 94% in identifying cases warranting genetic testing.</p><p>CONCLUSIONS: Our findings demonstrate the feasibility of using computational phenotyping to identify patients with CP or CP- masquerading conditions who warrant genetic testing. Further studies are needed to evaluate the effectiveness and real-world application of this tool in larger healthcare systems. Nonetheless, the computational phenotyping approach holds promise as a possible clinical decision support that could be integrated into electronic health record systems, enhancing clinical workflows and facilitating actionable genetic diagnoses.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yang D, Zhou D, Cai S, Gan Z, Pencina M, Avillach P, Cai T, Hong C. <a href="/publication/robust-automated-harmonization-heterogeneous-data-through-ensemble-machine-learning">Robust Automated Harmonization of Heterogeneous Data Through Ensemble Machine Learning: Algorithm Development and Validation Study.</a>.</div><div class="csl-right-inline">JMIR medical informatics. 2025;13:e54133. PMID: 39844378</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yang D, Zhou D, Cai S, Gan Z, Pencina M, Avillach P, Cai T, Hong C. <a href="/publication/robust-automated-harmonization-heterogeneous-data-through-ensemble-machine-learning">Robust Automated Harmonization of Heterogeneous Data Through Ensemble Machine Learning: Algorithm Development and Validation Study.</a>.</div><div class="csl-right-inline">JMIR medical informatics. 2025;13:e54133. PMID: 39844378</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-1907437921" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-1907437921">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Cohort studies contain rich clinical data across large and diverse patient populations and are a common source of observational data for clinical research. Because large scale cohort studies are both time and resource intensive, one alternative is to harmonize data from existing cohorts through multicohort studies. However, given differences in variable encoding, accurate variable harmonization is difficult.</p><p>OBJECTIVE: We propose SONAR (Semantic and Distribution-Based Harmonization) as a method for harmonizing variables across cohort studies to facilitate multicohort studies.</p><p>METHODS: SONAR used semantic learning from variable descriptions and distribution learning from study participant data. Our method learned an embedding vector for each variable and used pairwise cosine similarity to score the similarity between variables. This approach was built off 3 National Institutes of Health cohorts, including the Cardiovascular Health Study, the Multi-Ethnic Study of Atherosclerosis, and the Women's Health Initiative. We also used gold standard labels to further refine the embeddings in a supervised manner.</p><p>RESULTS: The method was evaluated using manually curated gold standard labels from the 3 National Institutes of Health cohorts. We evaluated both the intracohort and intercohort variable harmonization performance. The supervised SONAR method outperformed existing benchmark methods for almost all intracohort and intercohort comparisons using area under the curve and top-k accuracy metrics. Notably, SONAR was able to significantly improve harmonization of concepts that were difficult for existing semantic methods to harmonize.</p><p>CONCLUSIONS: SONAR achieves accurate variable harmonization within and between cohort studies by harnessing the complementary strengths of semantic learning and variable distribution learning.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Menezes MCS, Hoffmann AF, Tan ALM, Nalbandyan M, Omenn GS, Mazzotti DR, Hernández-Arango A, Visweswaran S, Venkatesh S, Mandl KD, Bourgeois FT, Lee JWK, Makmur A, Hanauer DA, Semanik MG, Kerivan LT, Hill T, Forero J, Restrepo C, Vigna M, Ceriana P, Abu-El-Rub N, Avillach P, Bellazzi R, Callaci T, Gutiérrez-Sacristán A, Malovini A, Mathew JP, Morris M, Murthy VL, Buonocore TM, Parimbelli E, Patel LP, Saez C, Samayamuthu MJ, Thompson JA, Tibollo V, Xia Z, Kohane IS, Records C for CC of C 19 by EH. <a href="/publication/potential-generative-pre-trained-transformer-4-gpt-4-analyse-medical-notes-three">The potential of Generative Pre-trained Transformer 4 (GPT-4) to analyse medical notes in three different languages: a retrospective model-evaluation study.</a>.</div><div class="csl-right-inline">The Lancet. Digital health. 2025;7(1):e35-e43. PMID: 39722251</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Menezes MCS, Hoffmann AF, Tan ALM, Nalbandyan M, Omenn GS, Mazzotti DR, Hernández-Arango A, Visweswaran S, Venkatesh S, Mandl KD, Bourgeois FT, Lee JWK, Makmur A, Hanauer DA, Semanik MG, Kerivan LT, Hill T, Forero J, Restrepo C, Vigna M, Ceriana P, Abu-El-Rub N, Avillach P, Bellazzi R, Callaci T, Gutiérrez-Sacristán A, Malovini A, Mathew JP, Morris M, Murthy VL, Buonocore TM, Parimbelli E, Patel LP, Saez C, Samayamuthu MJ, Thompson JA, Tibollo V, Xia Z, Kohane IS, Records C for CC of C 19 by EH. <a href="/publication/potential-generative-pre-trained-transformer-4-gpt-4-analyse-medical-notes-three">The potential of Generative Pre-trained Transformer 4 (GPT-4) to analyse medical notes in three different languages: a retrospective model-evaluation study.</a>.</div><div class="csl-right-inline">The Lancet. Digital health. 2025;7(1):e35-e43. PMID: 39722251</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-303159569" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-303159569">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Patient notes contain substantial information but are difficult for computers to analyse due to their unstructured format. Large-language models (LLMs), such as Generative Pre-trained Transformer 4 (GPT-4), have changed our ability to process text, but we do not know how effectively they handle medical notes. We aimed to assess the ability of GPT-4 to answer predefined questions after reading medical notes in three different languages.</p><p>METHODS: For this retrospective model-evaluation study, we included eight university hospitals from four countries (ie, the USA, Colombia, Singapore, and Italy). Each site submitted seven de-identified medical notes related to seven separate patients to the coordinating centre between June 1, 2023, and Feb 28, 2024. Medical notes were written between Feb 1, 2020, and June 1, 2023. One site provided medical notes in Spanish, one site provided notes in Italian, and the remaining six sites provided notes in English. We included admission notes, progress notes, and consultation notes. No discharge summaries were included in this study. We advised participating sites to choose medical notes that, at time of hospital admission, were for patients who were male or female, aged 18-65 years, had a diagnosis of obesity, had a diagnosis of COVID-19, and had submitted an admission note. Adherence to these criteria was optional and participating sites randomly chose which medical notes to submit. When entering information into GPT-4, we prepended each medical note with an instruction prompt and a list of 14 questions that had been chosen a priori. Each medical note was individually given to GPT-4 in its original language and in separate sessions; the questions were always given in English. At each site, two physicians independently validated responses by GPT-4 and responded to all 14 questions. Each pair of physicians evaluated responses from GPT-4 to the seven medical notes from their own site only. Physicians were not masked to responses from GPT-4 before providing their own answers, but were masked to responses from the other physician.</p><p>FINDINGS: We collected 56 medical notes, of which 42 (75%) were in English, seven (13%) were in Italian, and seven (13%) were in Spanish. For each medical note, GPT-4 responded to 14 questions, resulting in 784 responses. In 622 (79%, 95% CI 76-82) of 784 responses, both physicians agreed with GPT-4. In 82 (11%, 8-13) responses, only one physician agreed with GPT-4. In the remaining 80 (10%, 8-13) responses, neither physician agreed with GPT-4. Both physicians agreed with GPT-4 more often for medical notes written in Spanish (86 [88%, 95% CI 79-93] of 98 responses) and Italian (82 [84%, 75-90] of 98 responses) than in English (454 [77%, 74-80] of 588 responses).</p><p>INTERPRETATION: The results of our model-evaluation study suggest that GPT-4 is accurate when analysing medical notes in three different languages. In the future, research should explore how LLMs can be integrated into clinical workflows to maximise their use in health care.</p><p>FUNDING: None.</p></div>
</div>
</div>
</div>
</div>
</div>
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">2024</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gutiérrez-Sacristán A, Makwana S, Dionne A, Mahanta S, Dyer KJ, Serrano F, Watrin C, Pages P, Mousavi S, Degala A, Lyons J, Pillion D, Zachariasse JM, Shekerdemian LS, Truong DT, Newburger JW, Avillach P. <a href="/publication/development-and-validation-open-source-pipeline-automatic-population-case-report-forms">Development and validation of an open-source pipeline for automatic population of case report forms from electronic health records: a pediatric multi-center prospective study.</a>.</div><div class="csl-right-inline">EBioMedicine. 2024;108:105337. PMID: 39288532</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gutiérrez-Sacristán A, Makwana S, Dionne A, Mahanta S, Dyer KJ, Serrano F, Watrin C, Pages P, Mousavi S, Degala A, Lyons J, Pillion D, Zachariasse JM, Shekerdemian LS, Truong DT, Newburger JW, Avillach P. <a href="/publication/development-and-validation-open-source-pipeline-automatic-population-case-report-forms">Development and validation of an open-source pipeline for automatic population of case report forms from electronic health records: a pediatric multi-center prospective study.</a>.</div><div class="csl-right-inline">EBioMedicine. 2024;108:105337. PMID: 39288532</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-613921253" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-613921253">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Clinical trials and registry studies are essential for advancing research and developing novel treatments. However, these studies rely on manual entry of thousands of variables for each patient. Repurposing real-world data can significantly simplify the data collection, reduce transcription errors, and make the data entry process more efficient, consistent, and cost-effective.</p><p>METHODS: We developed an open-source computational pipeline to collect laboratory and medication information from the electronic health record (EHR) data and populate case report forms. The pipeline was developed and validated with data from two independent pediatric hospitals in the US as part of the Long-terM OUtcomes after Multisystem Inflammatory Syndrome In Children (MUSIC) study. Our pipeline allowed the completion of two of the most time-consuming forms. We compared automatically extracted results with manually entered values in one hospital and applied the pipeline to a second hospital, where the output served as the primary data source for case report forms.</p><p>FINDINGS: We extracted and populated 51,845 laboratory and 4913 medication values for 159 patients in two hospitals participating in a prospective pediatric study. We evaluated pipeline performance against data for 104 patients manually entered by clinicians in one of the hospitals. The highest concordance was found during patient hospitalization, with 91.59% of the automatically extracted laboratory and medication values corresponding with the manually entered values. In addition to the successfully populated values, we identified an additional 13,396 laboratory and 567 medication values of interest for the study.</p><p>INTERPRETATION: The automatic data entry of laboratory and medication values during admission is feasible and has a high concordance with the manually entered data. By implementing this proof of concept, we demonstrate the quality of automatic data extraction and highlight the potential of secondary use of EHR data to advance medical science by improving data entry efficiency and expediting clinical research.</p><p>FUNDING: NIH Grant 1OT3HL147154-01, U24HL135691, UG1HL135685.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Wack M, Hassen-Khodja C, McDuffie MT, Bliss G, Horn EJ, Kothari C, McLarney B, Davis R, Hanson K, O’Boyle M, Betancur C, Avillach P. <a href="/publication/phenome-wide-profiling-identifies-genotype-phenotype-associations-phelan-mcdermid">Phenome-wide profiling identifies genotype-phenotype associations in Phelan-McDermid syndrome using family-sourced data from an international registry.</a>.</div><div class="csl-right-inline">Molecular autism. 2024;15(1):40. PMID: 39350236</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Wack M, Hassen-Khodja C, McDuffie MT, Bliss G, Horn EJ, Kothari C, McLarney B, Davis R, Hanson K, O’Boyle M, Betancur C, Avillach P. <a href="/publication/phenome-wide-profiling-identifies-genotype-phenotype-associations-phelan-mcdermid">Phenome-wide profiling identifies genotype-phenotype associations in Phelan-McDermid syndrome using family-sourced data from an international registry.</a>.</div><div class="csl-right-inline">Molecular autism. 2024;15(1):40. PMID: 39350236</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-1645799944" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-1645799944">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Phelan-McDermid syndrome (PMS) is a rare neurodevelopmental disorder caused by 22q13 deletions that include the SHANK3 gene or pathogenic sequence variants in SHANK3. It is characterized by global developmental delay, intellectual disability, speech impairment, autism spectrum disorder, and hypotonia; other variable features include epilepsy, brain and renal malformations, and mild dysmorphic features. Here, we conducted genotype-phenotype correlation analyses using the PMS International Registry, a family-driven registry that compiles clinical data in the form of family-reported outcomes and family-sourced genetic test results.</p><p>METHODS: Data from the registry were harmonized and integrated into the i2b2/tranSMART clinical and genomics data warehouse. We gathered information from 401 individuals with 22q13 deletions including SHANK3 (n = 350, ranging in size from 10 kb to 9.1 Mb) or pathogenic or likely pathogenic SHANK3 sequence variants (n = 51), and used regression models with deletion size as a potential predictor of clinical outcomes for 328 phenotypes.</p><p>RESULTS: Our results showed that increased deletion size was significantly associated with delay in gross and fine motor acquisitions, a spectrum of conditions related to poor muscle tone, renal malformations, mild dysmorphic features (e.g., large fleshy hands, sacral dimple, dysplastic toenails, supernumerary teeth), lymphedema, congenital heart defects, and more frequent neuroimaging abnormalities and infections. These findings indicate that genes upstream of SHANK3 also contribute to some of the manifestations of PMS in individuals with larger deletions. We also showed that self-help skills, verbal ability and a range of psychiatric diagnoses (e.g., autism, ADHD, anxiety disorder) were more common among individuals with smaller deletions and SHANK3 variants.</p><p>LIMITATIONS: Some participants were tested with targeted 22q microarrays rather than genome-wide arrays, and karyotypes were unavailable in many cases, thus precluding the analysis of the effect of other copy number variants or chromosomal rearrangements on the phenotype.</p><p>CONCLUSIONS: This is the largest reported case series of individuals with PMS. Overall, we demonstrate the feasibility of using data from a family-sourced registry to conduct genotype-phenotype analyses in rare genetic disorders. We replicate and strengthen previous findings, and reveal novel associations between larger 22q13 deletions and congenital heart defects, neuroimaging abnormalities and recurrent infections.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hernández-Agosto J, Santiago-Quiñones DI, García-Carmona LM, Gutiérrez-Sacristán A. <a href="/publication/substance-use-disorders-and-disparities-hispanic-and-non-hispanic-older-adults">Substance use disorders and disparities in hispanic and non-hispanic older adults.</a>.</div><div class="csl-right-inline">Drug and alcohol dependence. 2024;261:111357. PMID: 38896947</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hernández-Agosto J, Santiago-Quiñones DI, García-Carmona LM, Gutiérrez-Sacristán A. <a href="/publication/substance-use-disorders-and-disparities-hispanic-and-non-hispanic-older-adults">Substance use disorders and disparities in hispanic and non-hispanic older adults.</a>.</div><div class="csl-right-inline">Drug and alcohol dependence. 2024;261:111357. PMID: 38896947</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2110352893" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-2110352893">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: The global longevity revolution increased the older adult population, posing unique health and economic challenges with implications for healthcare, especially substance use disorders (SUD).</p><p>METHODS: This was a retrospective cohort study of United States older adults, Hispanic and non-Hispanic, who got at least one mental and/or behavioral disorder diagnosis between 2017 and 2021 at age 65 or older. SUD prevalence, prescription frequency changes over time, and comorbidities associated with each medication were compared.</p><p>RESULTS: Electronic health records for 356,133 older adults (110,236 Hispanics and 245,897 non-Hispanics) were analyzed. Notably, 79 % of Hispanics fell below the 100 % federal poverty level, compared to 60 % of non-Hispanics (P&lt;.001). Non-Hispanics also had significantly more average encounters (P=.003) and diagnoses (P&lt;.001). Regression analysis on alcohol-related disorders indicated that the odd ratios of being male (OR=2.93, P&lt;.000), and having low income (OR=1.62, P&lt;.000), increase the odds for this SUD, while being Hispanic and primarily speaking Spanish decreases the odds for all SUDs considered in this study.</p><p>CONCLUSIONS: This cohort study revealed significant disparities related to social determinants of health between Hispanic and non-Hispanic older adults and emphasizes the need for continuous surveillance of older adults as with SUDs. Differences in comorbidity patterns imply distinct risk factors within each population, influenced by demographic-specific elements. Recognizing these variations is essential for tailoring culturally sensitive prevention, intervention, and treatment strategies to each population's unique needs.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Gutierrez A, Network UD, Kobren SN, Avillach P. <a href="/publication/varppud-variant-post-prioritization-developed-undiagnosed-genetic-disorders">VarPPUD: Variant post prioritization developed for undiagnosed genetic disorders.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2024; PMID: 38699371</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Gutierrez A, Network UD, Kobren SN, Avillach P. <a href="/publication/varppud-variant-post-prioritization-developed-undiagnosed-genetic-disorders">VarPPUD: Variant post prioritization developed for undiagnosed genetic disorders.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2024; PMID: 38699371</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-551639828" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-551639828">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>Rare and ultra-rare genetic conditions are estimated to impact nearly 1 in 17 people worldwide, yet accurately pinpointing the diagnostic variants underlying each of these conditions remains a formidable challenge. Because comprehensive,  functional assessment of all possible genetic variants is infeasible, clinicians instead consider  variant pathogenicity predictions to distinguish plausibly disease-causing from benign variants across the genome. However, in the most difficult undiagnosed cases, such as those accepted to the Undiagnosed Diseases Network (UDN), existing pathogenicity predictions cannot reliably discern true etiological variant(s) from other deleterious candidate variants that were prioritized through N-of-1 efforts. Pinpointing the disease-causing variant from a pool of plausible candidates remains a largely manual effort requiring extensive clinical workups, functional and experimental assays, and eventual identification of genotype- and phenotype-matched individuals. Here, we introduce VarPPUD, a tool trained on prioritized variants from UDN cases, that leverages gene-, amino acid-, and nucleotide-level features to discern pathogenic variants from other deleterious variants that are unlikely to be confirmed as disease relevant. VarPPUD achieves a cross-validated accuracy of 79.3% and precision of 77.5% on a held-out subset of uniquely challenging UDN cases, respectively representing an average 18.6% and 23.4% improvement over nine traditional pathogenicity prediction approaches on this task. We validate VarPPUD's ability to discriminate likely from unlikely pathogenic variants on synthetic, GAN-generated candidate variants as well. Finally, we show how VarPPUD can be probed to evaluate each input feature's importance and contribution toward prediction-an essential step toward understanding the distinct characteristics of newly-uncovered disease-causing variants.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hutch MR, Son J, Le TT, Hong C, Wang X, Abad ZSH, Morris M, Gutiérrez-Sacristán A, Klann JG, Spiridou A, Batugo A, Bellazzi R, Benoit V, Bonzel CL, Bryant WA, Chiudinelli L, Cho K, Das P, González TG, Hanauer DA, Henderson DW, Ho YL, Loh NHW, Makoudjou A, Makwana S, Malovini A, Moal B, Mowery DL, Neuraz A, Samayamuthu MJ, Vidorreta FJS, Schriver ER, Schubert P, Talbert J, Tan ALM, Tan BWL, Tan BWQ, Tibollo V, Tippman P, Verdy G, Yuan W, Avillach P, Gehlenborg N, Omenn GS, EHR C for CC of C 19 by, Visweswaran S, Cai T, Luo Y, Xia Z. <a href="/publication/neurological-diagnoses-hospitalized-covid-19-patients-associated-adverse-outcomes">Neurological diagnoses in hospitalized COVID-19 patients associated with adverse outcomes: A multinational cohort study.</a>.</div><div class="csl-right-inline">PLOS digital health. 2024;3(4):e0000484. PMID: 38620037</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hutch MR, Son J, Le TT, Hong C, Wang X, Abad ZSH, Morris M, Gutiérrez-Sacristán A, Klann JG, Spiridou A, Batugo A, Bellazzi R, Benoit V, Bonzel CL, Bryant WA, Chiudinelli L, Cho K, Das P, González TG, Hanauer DA, Henderson DW, Ho YL, Loh NHW, Makoudjou A, Makwana S, Malovini A, Moal B, Mowery DL, Neuraz A, Samayamuthu MJ, Vidorreta FJS, Schriver ER, Schubert P, Talbert J, Tan ALM, Tan BWL, Tan BWQ, Tibollo V, Tippman P, Verdy G, Yuan W, Avillach P, Gehlenborg N, Omenn GS, EHR C for CC of C 19 by, Visweswaran S, Cai T, Luo Y, Xia Z. <a href="/publication/neurological-diagnoses-hospitalized-covid-19-patients-associated-adverse-outcomes">Neurological diagnoses in hospitalized COVID-19 patients associated with adverse outcomes: A multinational cohort study.</a>.</div><div class="csl-right-inline">PLOS digital health. 2024;3(4):e0000484. PMID: 38620037</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2100453899" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-2100453899">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>Few studies examining the patient outcomes of concurrent neurological manifestations during acute COVID-19 leveraged multinational cohorts of adults and children or distinguished between central and peripheral nervous system (CNS vs. PNS) involvement. Using a federated multinational network in which local clinicians and informatics experts curated the electronic health records data, we evaluated the risk of prolonged hospitalization and mortality in hospitalized COVID-19 patients from 21 healthcare systems across 7 countries. For adults, we used a federated learning approach whereby we ran Cox proportional hazard models locally at each healthcare system and performed a meta-analysis on the aggregated results to estimate the overall risk of adverse outcomes across our geographically diverse populations. For children, we reported descriptive statistics separately due to their low frequency of neurological involvement and poor outcomes. Among the 106,229 hospitalized COVID-19 patients (104,031 patients ≥18 years; 2,198 patients &lt;18 years, January 2020-October 2021), 15,101 (14%) had at least one CNS diagnosis, while 2,788 (3%) had at least one PNS diagnosis. After controlling for demographics and pre-existing conditions, adults with CNS involvement had longer hospital stay (11 versus 6 days) and greater risk of (Hazard Ratio = 1.78) and faster time to death (12 versus 24 days) than patients with no neurological condition (NNC) during acute COVID-19 hospitalization. Adults with PNS involvement also had longer hospital stay but lower risk of mortality than the NNC group. Although children had a low frequency of neurological involvement during COVID-19 hospitalization, a substantially higher proportion of children with CNS involvement died compared to those with NNC (6% vs 1%). Overall, patients with concurrent CNS manifestation during acute COVID-19 hospitalization faced greater risks for adverse clinical outcomes than patients without any neurological diagnosis. Our global informatics framework using a federated approach (versus a centralized data collection approach) has utility for clinical discovery beyond COVID-19.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Huang M, Gutiérrez-Sacristán A, Janiak E, Young K, Starosta A, Blanton K, Azhir A, Goldfarb CN, Kuperwasser F, Schaefer KM, Stoddard RE, Vatsa R, Merz-Herrala AA, Bartz D. <a href="/publication/contraceptive-content-shared-social-media-analysis-twitter">Contraceptive content shared on social media: an analysis of Twitter.</a>.</div><div class="csl-right-inline">Contraception and reproductive medicine. 2024;9(1):5. PMID: 38321582</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Huang M, Gutiérrez-Sacristán A, Janiak E, Young K, Starosta A, Blanton K, Azhir A, Goldfarb CN, Kuperwasser F, Schaefer KM, Stoddard RE, Vatsa R, Merz-Herrala AA, Bartz D. <a href="/publication/contraceptive-content-shared-social-media-analysis-twitter">Contraceptive content shared on social media: an analysis of Twitter.</a>.</div><div class="csl-right-inline">Contraception and reproductive medicine. 2024;9(1):5. PMID: 38321582</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-1649283050" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-1649283050">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Information on social media may affect peoples' contraceptive decision making. We performed an exploratory analysis of contraceptive content on Twitter (recently renamed X), a popular social media platform.</p><p>METHODS: We selected a random subset of 1% of publicly available, English-language tweets related to reversible, prescription contraceptive methods posted between January 2014 and December 2019. We oversampled tweets for the contraceptive patch to ensure at least 200 tweets per method. To create the codebook, we identified common themes specific to tweet content topics, tweet sources, and tweets soliciting information or providing advice. All posts were coded by two team members, and differences were adjudicated by a third reviewer. Descriptive analyses were reported with accompanying qualitative findings.</p><p>RESULTS: During the study period, 457,369 tweets about reversible contraceptive methods were published, with a random sample of 4,434 tweets used for final analysis. Tweets most frequently discussed contraceptive method decision-making (26.7%) and side effects (20.5%), particularly for long-acting reversible contraceptive methods and the depot medroxyprogesterone acetate shot. Tweets about logistics of use or adherence were common for short-acting reversible contraceptives. Tweets were frequently posted by contraceptive consumers (50.6%). A small proportion of tweets explicitly requested information (6.2%) or provided advice (4.2%).</p><p>CONCLUSIONS: Clinicians should be aware that individuals are exposed to information through Twitter that may affect contraceptive perceptions and decision making, particularly regarding long-acting reversible contraceptives. Social media is a valuable source for studying contraceptive beliefs missing in traditional health research and may be used by professionals to disseminate accurate contraceptive information.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gargano MA, Matentzoglu N, Coleman B, Addo-Lartey EB, Anagnostopoulos A V, Anderton J, Avillach P, Bagley AM, Bakštein E, Balhoff JP, Baynam G, Bello SM, Berk M, Bertram H, Bishop S, Blau H, Bodenstein DF, Botas P, Boztug K, Čady J, Callahan TJ, Cameron R, Carbon SJ, Castellanos F, Caufield H, Chan LE, Chute CG, Cruz-Rojo J, Dahan-Oliel N, Davids JR, de Dieuleveult M, de Souza V, de Vries BBA, de Vries E, DePaulo R, Derfalvi B, Dhombres F, Diaz-Byrd C, Dingemans AJM, Donadille B, Duyzend M, Elfeky R, Essaid S, Fabrizzi C, Fico G, Firth H V, Freudenberg-Hua Y, Fullerton JM, Gabriel DL, Gilmour K, Giordano J, Goes FS, Moses RG, Green I, Griese M, Groza T, Gu W, Guthrie J, Gyori B, Hamosh A, Hanauer M, Hanušová K, He YO, Hegde H, Helbig I, Holasová K, Hoyt CT, Huang S, Hurwitz E, Jacobsen JOB, Jiang X, Joseph L, Keramatian K, King B, Knoflach K, Koolen DA, Kraus ML, Kroll C, Kusters M, Ladewig MS, Lagorce D, Lai MC, Lapunzina P, Laraway B, Lewis-Smith D, Li X, Lucano C, Majd M, Marazita ML, Martinez-Glez V, McHenry TH, McInnis MG, McMurry JA, Mihulová M, Millett CE, Mitchell PB, Moslerová V, Narutomi K, Nematollahi S, Nevado J, Nierenberg AA, Čajbiková NN, Nurnberger JI, Ogishima S, Olson D, Ortiz A, Pachajoa H, de Nanclares GP, Peters A, Putman T, Rapp CK, Rath A, Reese J, Rekerle L, Roberts AM, Roy S, Sanders SJ, Schuetz C, Schulte EC, Schulze TG, Schwarz M, Scott K, Seelow D, Seitz B, Shen Y, Similuk MN, Simon ES, Singh B, Smedley D, Smith CL, Smolinsky JT, Sperry S, Stafford E, Stefancsik R, Steinhaus R, Strawbridge R, Sundaramurthi JC, Talapova P, Castano JAT, Tesner P, Thomas RH, Thurm A, Turnovec M, van Gijn ME, Vasilevsky NA, Vlčková M, Walden A, Wang K, Wapner R, Ware JS, Wiafe AA, Wiafe SA, Wiggins LD, Williams AE, Wu C, Wyrwoll MJ, Xiong H, Yalin N, Yamamoto Y, Yatham LN, Yocum AK, Young AH, Yüksel Z, Zandi PP, Zankl A, Zarante I, Zvolský M, Toro S, Carmody LC, Harris NL, Munoz-Torres MC, Danis D, Mungall CJ, Köhler S, Haendel MA, Robinson PN. <a href="/publication/human-phenotype-ontology-2024-phenotypes-around-world">The Human Phenotype Ontology in 2024: phenotypes around the world.</a>.</div><div class="csl-right-inline">Nucleic acids research. 2024;52(D1):D1333-D1346. PMID: 37953324</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gargano MA, Matentzoglu N, Coleman B, Addo-Lartey EB, Anagnostopoulos A V, Anderton J, Avillach P, Bagley AM, Bakštein E, Balhoff JP, Baynam G, Bello SM, Berk M, Bertram H, Bishop S, Blau H, Bodenstein DF, Botas P, Boztug K, Čady J, Callahan TJ, Cameron R, Carbon SJ, Castellanos F, Caufield H, Chan LE, Chute CG, Cruz-Rojo J, Dahan-Oliel N, Davids JR, de Dieuleveult M, de Souza V, de Vries BBA, de Vries E, DePaulo R, Derfalvi B, Dhombres F, Diaz-Byrd C, Dingemans AJM, Donadille B, Duyzend M, Elfeky R, Essaid S, Fabrizzi C, Fico G, Firth H V, Freudenberg-Hua Y, Fullerton JM, Gabriel DL, Gilmour K, Giordano J, Goes FS, Moses RG, Green I, Griese M, Groza T, Gu W, Guthrie J, Gyori B, Hamosh A, Hanauer M, Hanušová K, He YO, Hegde H, Helbig I, Holasová K, Hoyt CT, Huang S, Hurwitz E, Jacobsen JOB, Jiang X, Joseph L, Keramatian K, King B, Knoflach K, Koolen DA, Kraus ML, Kroll C, Kusters M, Ladewig MS, Lagorce D, Lai MC, Lapunzina P, Laraway B, Lewis-Smith D, Li X, Lucano C, Majd M, Marazita ML, Martinez-Glez V, McHenry TH, McInnis MG, McMurry JA, Mihulová M, Millett CE, Mitchell PB, Moslerová V, Narutomi K, Nematollahi S, Nevado J, Nierenberg AA, Čajbiková NN, Nurnberger JI, Ogishima S, Olson D, Ortiz A, Pachajoa H, de Nanclares GP, Peters A, Putman T, Rapp CK, Rath A, Reese J, Rekerle L, Roberts AM, Roy S, Sanders SJ, Schuetz C, Schulte EC, Schulze TG, Schwarz M, Scott K, Seelow D, Seitz B, Shen Y, Similuk MN, Simon ES, Singh B, Smedley D, Smith CL, Smolinsky JT, Sperry S, Stafford E, Stefancsik R, Steinhaus R, Strawbridge R, Sundaramurthi JC, Talapova P, Castano JAT, Tesner P, Thomas RH, Thurm A, Turnovec M, van Gijn ME, Vasilevsky NA, Vlčková M, Walden A, Wang K, Wapner R, Ware JS, Wiafe AA, Wiafe SA, Wiggins LD, Williams AE, Wu C, Wyrwoll MJ, Xiong H, Yalin N, Yamamoto Y, Yatham LN, Yocum AK, Young AH, Yüksel Z, Zandi PP, Zankl A, Zarante I, Zvolský M, Toro S, Carmody LC, Harris NL, Munoz-Torres MC, Danis D, Mungall CJ, Köhler S, Haendel MA, Robinson PN. <a href="/publication/human-phenotype-ontology-2024-phenotypes-around-world">The Human Phenotype Ontology in 2024: phenotypes around the world.</a>.</div><div class="csl-right-inline">Nucleic acids research. 2024;52(D1):D1333-D1346. PMID: 37953324</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2011850567" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-2011850567">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>The Human Phenotype Ontology (HPO) is a widely used resource that comprehensively organizes and defines the phenotypic features of human disease, enabling computational inference and supporting genomic and phenotypic analyses through semantic similarity and machine learning algorithms. The HPO has widespread applications in clinical diagnostics and translational research, including genomic diagnostics, gene-disease discovery, and cohort analytics. In recent years, groups around the world have developed translations of the HPO from English to other languages, and the HPO browser has been internationalized, allowing users to view HPO term labels and in many cases synonyms and definitions in ten languages in addition to English. Since our last report, a total of 2239 new HPO terms and 49235 new HPO annotations were developed, many in collaboration with external groups in the fields of psychiatry, arthrogryposis, immunology and cardiology. The Medical Action Ontology (MAxO) is a new effort to model treatments and other measures taken for clinical management. Finally, the HPO consortium is contributing to efforts to integrate the HPO and the GA4GH Phenopacket Schema into electronic health records (EHRs) with the goal of more standardized and computable integration of rare disease data in EHRs.</p></div>
</div>
</div>
</div>
</div>
</div>
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">2023</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Sperotto F, Gutiérrez-Sacristán A, Makwana S, Li X, Rofeberg VN, Cai T, Bourgeois FT, Omenn GS, Hanauer DA, Saez C, Bonzel CL, Bucholz E, Dionne A, Elias MD, García-Barrio N, González TG, Issitt RW, Kernan KF, Laird-Gion J, Maidlow SE, Mandl KD, Ahooyi TM, Moraleda C, Morris M, Moshal KL, Pedrera-Jiménez M, Shah MA, South AM, Spiridou A, Taylor DM, Verdy G, Visweswaran S, Wang X, Xia Z, Zachariasse JM, EHR C for CC of C 19 by, Newburger JW, Avillach P. <a href="/publication/clinical-phenotypes-and-outcomes-children-multisystem-inflammatory-syndrome-across-0">Clinical phenotypes and outcomes in children with multisystem inflammatory syndrome across SARS-CoV-2 variant eras: a multinational study from the 4CE consortium.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102212. PMID: 37745025</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Sperotto F, Gutiérrez-Sacristán A, Makwana S, Li X, Rofeberg VN, Cai T, Bourgeois FT, Omenn GS, Hanauer DA, Saez C, Bonzel CL, Bucholz E, Dionne A, Elias MD, García-Barrio N, González TG, Issitt RW, Kernan KF, Laird-Gion J, Maidlow SE, Mandl KD, Ahooyi TM, Moraleda C, Morris M, Moshal KL, Pedrera-Jiménez M, Shah MA, South AM, Spiridou A, Taylor DM, Verdy G, Visweswaran S, Wang X, Xia Z, Zachariasse JM, EHR C for CC of C 19 by, Newburger JW, Avillach P. <a href="/publication/clinical-phenotypes-and-outcomes-children-multisystem-inflammatory-syndrome-across-0">Clinical phenotypes and outcomes in children with multisystem inflammatory syndrome across SARS-CoV-2 variant eras: a multinational study from the 4CE consortium.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102212. PMID: 37745025</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-87103762" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-87103762">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Multisystem inflammatory syndrome in children (MIS-C) is a severe complication of SARS-CoV-2 infection. It remains unclear how MIS-C phenotypes vary across SARS-CoV-2 variants. We aimed to investigate clinical characteristics and outcomes of MIS-C across SARS-CoV-2 eras.</p><p>METHODS: We performed a multicentre observational retrospective study including seven paediatric hospitals in four countries (France, Spain, U.K., and U.S.). All consecutive confirmed patients with MIS-C hospitalised between February 1st, 2020, and May 31st, 2022, were included. Electronic Health Records (EHR) data were used to calculate pooled risk differences (RD) and effect sizes (ES) at site level, using  as reference. Meta-analysis was used to pool data across sites.</p><p>FINDINGS: Of 598 patients with MIS-C (61% male, 39% female; mean age 9.7 years [SD 4.5]), 383 (64%) were admitted in the  era, 111 (19%) in the  era, and 104 (17%) in the  era. Compared with patients admitted in the  era, those admitted in the  era were younger (ES -1.18 years [95% CI -2.05, -0.32]), had fewer respiratory symptoms (RD -0.15 [95% CI -0.33, -0.04]), less frequent non-cardiogenic shock or systemic inflammatory response syndrome (SIRS) (RD -0.35 [95% CI -0.64, -0.07]), lower lymphocyte count (ES -0.16 × 10/uL [95% CI -0.30, -0.01]), lower C-reactive protein (ES -28.5 mg/L [95% CI -46.3, -10.7]), and lower troponin (ES -0.14 ng/mL [95% CI -0.26, -0.03]). Patients admitted in the  versus  eras were younger (ES -1.6 years [95% CI -2.5, -0.8]), had less frequent SIRS (RD -0.18 [95% CI -0.30, -0.05]), lower lymphocyte count (ES -0.39 × 10/uL [95% CI -0.52, -0.25]), lower troponin (ES -0.16 ng/mL [95% CI -0.30, -0.01]) and less frequently received anticoagulation therapy (RD -0.19 [95% CI -0.37, -0.04]). Length of hospitalization was shorter in the  versus  eras (-1.3 days [95% CI -2.3, -0.4]).</p><p>INTERPRETATION: Our study suggested that MIS-C clinical phenotypes varied across SARS-CoV-2 eras, with patients in  and  eras being younger and less sick. EHR data can be effectively leveraged to identify rare complications of pandemic diseases and their variation over time.</p><p>FUNDING: None.</p></div>
</div>
</div>
</div>
<div class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation" lang="en">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Dagliati A, Strasser ZH, Abad ZSH, Klann JG, Wagholikar KB, Mesa R, Visweswaran S, Morris M, Luo Y, Henderson DW, Samayamuthu MJ, Tan BWQ, Verdy G, Omenn GS, Xia Z, Bellazzi R, EHR C for CC of C 19 by, Murphy SN, Holmes JH, Estiri H, EHR C for CC of C 19 by. <a href="/publication/characterization-long-covid-temporal-sub-phenotypes-distributed-representation-0">Characterization of long COVID temporal sub-phenotypes by distributed representation learning from electronic health record data: a cohort study.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102210. PMID: 37745021</div></div></div>
</div>
</div>
<div class="hwp-text-block">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Dagliati A, Strasser ZH, Abad ZSH, Klann JG, Wagholikar KB, Mesa R, Visweswaran S, Morris M, Luo Y, Henderson DW, Samayamuthu MJ, Tan BWQ, Verdy G, Omenn GS, Xia Z, Bellazzi R, EHR C for CC of C 19 by, Murphy SN, Holmes JH, Estiri H, EHR C for CC of C 19 by. <a href="/publication/characterization-long-covid-temporal-sub-phenotypes-distributed-representation-0">Characterization of long COVID temporal sub-phenotypes by distributed representation learning from electronic health record data: a cohort study.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102210. PMID: 37745021</div></div></div>
</div>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-250295374" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
<div aria-hidden="true" class="hwp-citation__full-body" id="hwp-citation-body-content-250295374">
<div class="hwp-text-block">
<div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Characterizing Post-Acute Sequelae of COVID (SARS-CoV-2 Infection), or  has been challenging due to the multitude of sub-phenotypes, temporal attributes, and definitions. Scalable characterization of PASC sub-phenotypes can enhance screening capacities, disease management, and treatment planning.</p><p>METHODS: We conducted a retrospective multi-centre observational cohort study, leveraging longitudinal electronic health record (EHR) data of 30,422 patients from three healthcare systems in the Consortium for the Clinical Characterization of COVID-19 by EHR (4CE). From the total cohort, we applied a deductive approach on 12,424 individuals with follow-up data and developed a distributed representation learning process for providing augmented definitions for PASC sub-phenotypes.</p><p>FINDINGS: Our framework characterized seven PASC sub-phenotypes. We estimated that on average 15.7% of the hospitalized COVID-19 patients were likely to suffer from at least one PASC symptom and almost 5.98%, on average, had multiple symptoms. Joint pain and dyspnea had the highest prevalence, with an average prevalence of 5.45% and 4.53%, respectively.</p><p>INTERPRETATION: We provided a scalable framework to every participating healthcare system for estimating PASC sub-phenotypes prevalence and temporal attributes, thus developing a unified model that characterizes augmented sub-phenotypes across the different systems.</p><p>FUNDING: Authors are supported by National Institute of Allergy and Infectious Diseases, National Institute on Aging, National Center for Advancing Translational Sciences, National Medical Research Council, National Institute of Neurological Disorders and Stroke, European Union, National Institutes of Health, National Center for Advancing Translational Sciences.</p></div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
<nav aria-label="Pagination" class="hwp-pager hwp-pager--theme-light hwp-bg-light-core" role="navigation">
<ul class="hwp-pager__items js-pager__items">
<li>
<div class="hwp-pager__item hwp-pager__item--previous hwp-pager__item--not-active" href="" rel="prev" title="Go to previous page">
<span class="hwp-visually-hidden">Previous page</span>
<span aria-hidden="true" class="material-icon material-icon--chevron-left">chevron_left</span>
</div>
</li>
<li class="hwp-pager__item hwp-pager__item--is-active">
<a aria-current="page" href="?page=0" title="Current page">1</a>
</li>
<li class="hwp-pager__item">
<a href="?page=1" title="Go to page 2">2</a>
</li>
<li class="hwp-pager__item">
<a href="?page=2" title="Go to page 3">3</a>
</li>
<li class="hwp-pager__item hwp-pager__item--last hwp-pager__item--ellipsis">
<a href="?page=8" title="Go to last page">
<span class="hwp-visually-hidden">Last page</span>
<span aria-hidden="true">9</span>
</a>
</li>
<li>
<a class="hwp-pager__item hwp-pager__item--next" href="?page=1" rel="prev" title="Go to next page">
<span class="hwp-visually-hidden">Next page</span>
<span aria-hidden="true" class="material-icon material-icon--chevron-right">chevron_right</span>
</a>
</li>
</ul>
</nav>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/node/1774776/export -->
<!-- Source: avillach-lab.hms.harvard.edu/publication/potential-generative-pre-trained-transformer-4-gpt-4-analyse-medical-notes-three -->
<h1>The potential of Generative Pre-trained Transformer 4 (GPT-4) to analyse medical notes in three different languages: a retrospective model-evaluation study. | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/publication/potential-generative-pre-trained-transformer-4-gpt-4-analyse-medical-notes-three'>avillach-lab.hms.harvard.edu/publication/potential-generative-pre-trained-transformer-4-gpt-4-analyse-medical-notes-three</a></div>
<main aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="biblio-details">
<div class="hwp-page-title bibcite-reference" lang="en">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      The potential of Generative Pre-trained Transformer 4 (GPT-4) to analyse medical notes in three different languages: a retrospective model-evaluation study.
    </h1>
</div>
</div>
<div class="md:hwp-pt-64">
<div class="hwp-container hwp-container-px max-lg:hwp-px-0">
<div class="publication-info" data-component-name="publication-info">
<div class="publication-info__image">
</div>
<div class="publication-info__text">
<h4 class="h4 hwp-mb-8">Publication information:</h4>
<div class="publication-info__body">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Menezes MCS, Hoffmann AF, Tan ALM, Nalbandyan M, Omenn GS, Mazzotti DR, Hernández-Arango A, Visweswaran S, Venkatesh S, Mandl KD, Bourgeois FT, Lee JWK, Makmur A, Hanauer DA, Semanik MG, Kerivan LT, Hill T, Forero J, Restrepo C, Vigna M, Ceriana P, Abu-El-Rub N, Avillach P, Bellazzi R, Callaci T, Gutiérrez-Sacristán A, Malovini A, Mathew JP, Morris M, Murthy VL, Buonocore TM, Parimbelli E, Patel LP, Saez C, Samayamuthu MJ, Thompson JA, Tibollo V, Xia Z, Kohane IS, Records C for CC of C 19 by EH. The potential of Generative Pre-trained Transformer 4 (GPT-4) to analyse medical notes in three different languages: a retrospective model-evaluation study.</div><div class="csl-right-inline">The Lancet. Digital health. 2025;7(1):e35-e43. PMID: 39722251</div></div></div>
</div>
</div>
<ul class="publication-info__actions">
<li>
<div class="button-dropdown" data-component-name="button-dropdown">
<button class="hwp-text-link hwp-text-link--icon-left" id="publication_citation_download">
<span aria-hidden="true" class="material-icon">download_for_offline</span>
<span class="hwp-text-link__text">Download citation</span>
</button>
<ul aria-labelledby="publication_citation_download" class="button-dropdown__items" role="menu">
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773901&amp;format=bibtex">BibTeX</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773901&amp;format=endnote8">EndNote X3 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773901&amp;format=endnote7">EndNote 7 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773901&amp;format=tagged">Endnote tagged</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773901&amp;format=marc">Marc</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773901&amp;format=pubmed_id">PubMedId</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773901&amp;format=ris">RIS</a>
</li>
</ul>
</div>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 hwp-pb-64 md:hwp-py-64">
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">Abstract</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-trim hwp-bg-light-base js-hwp-citation" data-component-name="citation">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<p><b>BACKGROUND: </b>Patient notes contain substantial information but are difficult for computers to analyse due to their unstructured format. Large-language models (LLMs), such as Generative Pre-trained Transformer 4 (GPT-4), have changed our ability to process text, but we do not know how effectively they handle medical notes. We aimed to assess the ability of GPT-4 to answer predefined questions after reading medical notes in three different languages.</p><p><b>METHODS: </b>For this retrospective model-evaluation study, we included eight university hospitals from four countries (ie, the USA, Colombia, Singapore, and Italy). Each site submitted seven de-identified medical notes related to seven separate patients to the coordinating centre between June 1, 2023, and Feb 28, 2024. Medical notes were written between Feb 1, 2020, and June 1, 2023. One site provided medical notes in Spanish, one site provided notes in Italian, and the remaining six sites provided notes in English. We included admission notes, progress notes, and consultation notes. No discharge summaries were included in this study. We advised participating sites to choose medical notes that, at time of hospital admission, were for patients who were male or female, aged 18-65 years, had a diagnosis of obesity, had a diagnosis of COVID-19, and had submitted an admission note. Adherence to these criteria was optional and participating sites randomly chose which medical notes to submit. When entering information into GPT-4, we prepended each medical note with an instruction prompt and a list of 14 questions that had been chosen a priori. Each medical note was individually given to GPT-4 in its original language and in separate sessions; the questions were always given in English. At each site, two physicians independently validated responses by GPT-4 and responded to all 14 questions. Each pair of physicians evaluated responses from GPT-4 to the seven medical notes from their own site only. Physicians were not masked to responses from GPT-4 before providing their own answers, but were masked to responses from the other physician.</p><p><b>FINDINGS: </b>We collected 56 medical notes, of which 42 (75%) were in English, seven (13%) were in Italian, and seven (13%) were in Spanish. For each medical note, GPT-4 responded to 14 questions, resulting in 784 responses. In 622 (79%, 95% CI 76-82) of 784 responses, both physicians agreed with GPT-4. In 82 (11%, 8-13) responses, only one physician agreed with GPT-4. In the remaining 80 (10%, 8-13) responses, neither physician agreed with GPT-4. Both physicians agreed with GPT-4 more often for medical notes written in Spanish (86 [88%, 95% CI 79-93] of 98 responses) and Italian (82 [84%, 75-90] of 98 responses) than in English (454 [77%, 74-80] of 588 responses).</p><p><b>INTERPRETATION: </b>The results of our model-evaluation study suggest that GPT-4 is accurate when analysing medical notes in three different languages. In the future, research should explore how LLMs can be integrated into clinical workflows to maximise their use in health care.</p><p><b>FUNDING: </b>None.</p>
</div>
<div class="hwp-text-block">
<p><b>BACKGROUND: </b>Patient notes contain substantial information but are difficult for computers to analyse due to their unstructured format. Large-language models (LLMs), such as Generative Pre-trained Transformer 4 (GPT-4), have changed our ability to process text, but we do not know how effectively they handle medical notes. We aimed to assess the ability of GPT-4 to answer predefined questions after reading medical notes in three different languages.</p><p><b>METHODS: </b>For this retrospective model-evaluation study, we included eight university hospitals from four countries (ie, the USA, Colombia, Singapore, and Italy). Each site submitted seven de-identified medical notes related to seven separate patients to the coordinating centre between June 1, 2023, and Feb 28, 2024. Medical notes were written between Feb 1, 2020, and June 1, 2023. One site provided medical notes in Spanish, one site provided notes in Italian, and the remaining six sites provided notes in English. We included admission notes, progress notes, and consultation notes. No discharge summaries were included in this study. We advised participating sites to choose medical notes that, at time of hospital admission, were for patients who were male or female, aged 18-65 years, had a diagnosis of obesity, had a diagnosis of COVID-19, and had submitted an admission note. Adherence to these criteria was optional and participating sites randomly chose which medical notes to submit. When entering information into GPT-4, we prepended each medical note with an instruction prompt and a list of 14 questions that had been chosen a priori. Each medical note was individually given to GPT-4 in its original language and in separate sessions; the questions were always given in English. At each site, two physicians independently validated responses by GPT-4 and responded to all 14 questions. Each pair of physicians evaluated responses from GPT-4 to the seven medical notes from their own site only. Physicians were not masked to responses from GPT-4 before providing their own answers, but were masked to responses from the other physician.</p><p><b>FINDINGS: </b>We collected 56 medical notes, of which 42 (75%) were in English, seven (13%) were in Italian, and seven (13%) were in Spanish. For each medical note, GPT-4 responded to 14 questions, resulting in 784 responses. In 622 (79%, 95% CI 76-82) of 784 responses, both physicians agreed with GPT-4. In 82 (11%, 8-13) responses, only one physician agreed with GPT-4. In the remaining 80 (10%, 8-13) responses, neither physician agreed with GPT-4. Both physicians agreed with GPT-4 more often for medical notes written in Spanish (86 [88%, 95% CI 79-93] of 98 responses) and Italian (82 [84%, 75-90] of 98 responses) than in English (454 [77%, 74-80] of 588 responses).</p><p><b>INTERPRETATION: </b>The results of our model-evaluation study suggest that GPT-4 is accurate when analysing medical notes in three different languages. In the future, research should explore how LLMs can be integrated into clinical workflows to maximise their use in health care.</p><p><b>FUNDING: </b>None.</p>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-1356419879" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
</div>
</div>
</div>
<hr class="hwp-text-button-light-secondary hwp-mt-32"/>
</div>
<div class="hwp-news-footer hwp-py-24 lg:hwp-py-64 hwp-container hwp-container-px">
<div class="page-tags">
</div>
<div class="a2a_kit" data-a2a-title="" data-a2a-url="">
<span class="hwp-mb-16 hwp-block">Share on:</span>
<ul aria-label="Share on:" class="md:hwp-inline-flex md:hwp-flex-wrap">
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_facebook hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-facebook"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Facebook</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_twitter hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-twitter"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Twitter</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_linkedin hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-linkedin"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Linkedin</span>
</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/publication/characterization-long-covid-temporal-sub-phenotypes-distributed-representation-0 -->
<h1>Characterization of long COVID temporal sub-phenotypes by distributed representation learning from electronic health record data: a cohort study. | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/publication/characterization-long-covid-temporal-sub-phenotypes-distributed-representation-0'>avillach-lab.hms.harvard.edu/publication/characterization-long-covid-temporal-sub-phenotypes-distributed-representation-0</a></div>
<main aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="biblio-details">
<div class="hwp-page-title bibcite-reference" lang="en">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Characterization of long COVID temporal sub-phenotypes by distributed representation learning from electronic health record data: a cohort study.
    </h1>
</div>
</div>
<div class="md:hwp-pt-64">
<div class="hwp-container hwp-container-px max-lg:hwp-px-0">
<div class="publication-info" data-component-name="publication-info">
<div class="publication-info__image">
</div>
<div class="publication-info__text">
<h4 class="h4 hwp-mb-8">Publication information:</h4>
<div class="publication-info__body">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Dagliati A, Strasser ZH, Abad ZSH, Klann JG, Wagholikar KB, Mesa R, Visweswaran S, Morris M, Luo Y, Henderson DW, Samayamuthu MJ, Tan BWQ, Verdy G, Omenn GS, Xia Z, Bellazzi R, EHR C for CC of C 19 by, Murphy SN, Holmes JH, Estiri H, EHR C for CC of C 19 by. Characterization of long COVID temporal sub-phenotypes by distributed representation learning from electronic health record data: a cohort study.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102210. PMID: 37745021</div></div></div>
</div>
</div>
<ul class="publication-info__actions">
<li>
<div class="button-dropdown" data-component-name="button-dropdown">
<button class="hwp-text-link hwp-text-link--icon-left" id="publication_citation_download">
<span aria-hidden="true" class="material-icon">download_for_offline</span>
<span class="hwp-text-link__text">Download citation</span>
</button>
<ul aria-labelledby="publication_citation_download" class="button-dropdown__items" role="menu">
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773876&amp;format=bibtex">BibTeX</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773876&amp;format=endnote8">EndNote X3 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773876&amp;format=endnote7">EndNote 7 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773876&amp;format=tagged">Endnote tagged</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773876&amp;format=marc">Marc</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773876&amp;format=pubmed_id">PubMedId</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773876&amp;format=ris">RIS</a>
</li>
</ul>
</div>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 hwp-pb-64 md:hwp-py-64">
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">Abstract</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-trim hwp-bg-light-base js-hwp-citation" data-component-name="citation">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<p><b>BACKGROUND: </b>Characterizing Post-Acute Sequelae of COVID (SARS-CoV-2 Infection), or  has been challenging due to the multitude of sub-phenotypes, temporal attributes, and definitions. Scalable characterization of PASC sub-phenotypes can enhance screening capacities, disease management, and treatment planning.</p><p><b>METHODS: </b>We conducted a retrospective multi-centre observational cohort study, leveraging longitudinal electronic health record (EHR) data of 30,422 patients from three healthcare systems in the Consortium for the Clinical Characterization of COVID-19 by EHR (4CE). From the total cohort, we applied a deductive approach on 12,424 individuals with follow-up data and developed a distributed representation learning process for providing augmented definitions for PASC sub-phenotypes.</p><p><b>FINDINGS: </b>Our framework characterized seven PASC sub-phenotypes. We estimated that on average 15.7% of the hospitalized COVID-19 patients were likely to suffer from at least one PASC symptom and almost 5.98%, on average, had multiple symptoms. Joint pain and dyspnea had the highest prevalence, with an average prevalence of 5.45% and 4.53%, respectively.</p><p><b>INTERPRETATION: </b>We provided a scalable framework to every participating healthcare system for estimating PASC sub-phenotypes prevalence and temporal attributes, thus developing a unified model that characterizes augmented sub-phenotypes across the different systems.</p><p><b>FUNDING: </b>Authors are supported by National Institute of Allergy and Infectious Diseases, National Institute on Aging, National Center for Advancing Translational Sciences, National Medical Research Council, National Institute of Neurological Disorders and Stroke, European Union, National Institutes of Health, National Center for Advancing Translational Sciences.</p>
</div>
<div class="hwp-text-block">
<p><b>BACKGROUND: </b>Characterizing Post-Acute Sequelae of COVID (SARS-CoV-2 Infection), or  has been challenging due to the multitude of sub-phenotypes, temporal attributes, and definitions. Scalable characterization of PASC sub-phenotypes can enhance screening capacities, disease management, and treatment planning.</p><p><b>METHODS: </b>We conducted a retrospective multi-centre observational cohort study, leveraging longitudinal electronic health record (EHR) data of 30,422 patients from three healthcare systems in the Consortium for the Clinical Characterization of COVID-19 by EHR (4CE). From the total cohort, we applied a deductive approach on 12,424 individuals with follow-up data and developed a distributed representation learning process for providing augmented definitions for PASC sub-phenotypes.</p><p><b>FINDINGS: </b>Our framework characterized seven PASC sub-phenotypes. We estimated that on average 15.7% of the hospitalized COVID-19 patients were likely to suffer from at least one PASC symptom and almost 5.98%, on average, had multiple symptoms. Joint pain and dyspnea had the highest prevalence, with an average prevalence of 5.45% and 4.53%, respectively.</p><p><b>INTERPRETATION: </b>We provided a scalable framework to every participating healthcare system for estimating PASC sub-phenotypes prevalence and temporal attributes, thus developing a unified model that characterizes augmented sub-phenotypes across the different systems.</p><p><b>FUNDING: </b>Authors are supported by National Institute of Allergy and Infectious Diseases, National Institute on Aging, National Center for Advancing Translational Sciences, National Medical Research Council, National Institute of Neurological Disorders and Stroke, European Union, National Institutes of Health, National Center for Advancing Translational Sciences.</p>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-185763538" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
</div>
</div>
</div>
<hr class="hwp-text-button-light-secondary hwp-mt-32"/>
</div>
<div class="hwp-news-footer hwp-py-24 lg:hwp-py-64 hwp-container hwp-container-px">
<div class="page-tags">
</div>
<div class="a2a_kit" data-a2a-title="" data-a2a-url="">
<span class="hwp-mb-16 hwp-block">Share on:</span>
<ul aria-label="Share on:" class="md:hwp-inline-flex md:hwp-flex-wrap">
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_facebook hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-facebook"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Facebook</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_twitter hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-twitter"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Twitter</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_linkedin hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-linkedin"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Linkedin</span>
</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/publication/use-computational-phenotypes-predicting-genetic-subgroups-cerebral-palsy -->
<h1>Use of Computational Phenotypes for Predicting Genetic Subgroups of Cerebral Palsy. | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/publication/use-computational-phenotypes-predicting-genetic-subgroups-cerebral-palsy'>avillach-lab.hms.harvard.edu/publication/use-computational-phenotypes-predicting-genetic-subgroups-cerebral-palsy</a></div>
<main aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="biblio-details">
<div class="hwp-page-title bibcite-reference" lang="en">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Use of Computational Phenotypes for Predicting Genetic Subgroups of Cerebral Palsy.
    </h1>
</div>
</div>
<div class="md:hwp-pt-64">
<div class="hwp-container hwp-container-px max-lg:hwp-px-0">
<div class="publication-info" data-component-name="publication-info">
<div class="publication-info__image">
</div>
<div class="publication-info__text">
<h4 class="h4 hwp-mb-8">Publication information:</h4>
<div class="publication-info__body">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Alkuraya I, Almansa AS, Eleonu A, Avillach P, Poduri A, Srivastava S. Use of Computational Phenotypes for Predicting Genetic Subgroups of Cerebral Palsy.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2025; PMID: 39990554</div></div></div>
</div>
</div>
<ul class="publication-info__actions">
<li>
<div class="button-dropdown" data-component-name="button-dropdown">
<button class="hwp-text-link hwp-text-link--icon-left" id="publication_citation_download">
<span aria-hidden="true" class="material-icon">download_for_offline</span>
<span class="hwp-text-link__text">Download citation</span>
</button>
<ul aria-labelledby="publication_citation_download" class="button-dropdown__items" role="menu">
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773896&amp;format=bibtex">BibTeX</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773896&amp;format=endnote8">EndNote X3 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773896&amp;format=endnote7">EndNote 7 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773896&amp;format=tagged">Endnote tagged</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773896&amp;format=marc">Marc</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773896&amp;format=pubmed_id">PubMedId</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773896&amp;format=ris">RIS</a>
</li>
</ul>
</div>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 hwp-pb-64 md:hwp-py-64">
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">Abstract</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-trim hwp-bg-light-base js-hwp-citation" data-component-name="citation">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<p><b>INTRODUCTION: </b>Emerging evidence suggests that 20-30% of cases of cerebral palsy (CP) may have a genetic cause. Our group previously identified subsets of patients with CP or CP-masquerading conditions who warrant genetic testing, including those with regression or progressive neurological symptoms (CP masqueraders) and those without any known risk factors for CP (cryptogenic CP). Recognition of these subgroups in clinical settings remains challenging.</p><p><b>METHODS: </b>To address this challenge, we developed and evaluated a computational phenotyping approach using ICD- 9/ICD-10 billing codes to automatically identify patients with unexplained CP or CP-masquerading conditions who may benefit from genetic testing. We applied this computational phenotyping approach to a cohort of 250 participants from the Boston Children's Hospital CP Sequencing Study, aimed at identifying genetic causes in CP and CP-masquerading conditions.</p><p><b>RESULTS: </b>Manual review served as the gold standard, identifying 8% as CP masqueraders, 42% as cryptogenic CP, and 50% as non-cryptogenic CP. Computational phenotyping based on ICD-9/10 codes achieved a sensitivity of 95%, specificity of 72%, positive predictive value of 77%, and negative predictive value of 94% in identifying cases warranting genetic testing.</p><p><b>CONCLUSIONS: </b>Our findings demonstrate the feasibility of using computational phenotyping to identify patients with CP or CP- masquerading conditions who warrant genetic testing. Further studies are needed to evaluate the effectiveness and real-world application of this tool in larger healthcare systems. Nonetheless, the computational phenotyping approach holds promise as a possible clinical decision support that could be integrated into electronic health record systems, enhancing clinical workflows and facilitating actionable genetic diagnoses.</p>
</div>
<div class="hwp-text-block">
<p><b>INTRODUCTION: </b>Emerging evidence suggests that 20-30% of cases of cerebral palsy (CP) may have a genetic cause. Our group previously identified subsets of patients with CP or CP-masquerading conditions who warrant genetic testing, including those with regression or progressive neurological symptoms (CP masqueraders) and those without any known risk factors for CP (cryptogenic CP). Recognition of these subgroups in clinical settings remains challenging.</p><p><b>METHODS: </b>To address this challenge, we developed and evaluated a computational phenotyping approach using ICD- 9/ICD-10 billing codes to automatically identify patients with unexplained CP or CP-masquerading conditions who may benefit from genetic testing. We applied this computational phenotyping approach to a cohort of 250 participants from the Boston Children's Hospital CP Sequencing Study, aimed at identifying genetic causes in CP and CP-masquerading conditions.</p><p><b>RESULTS: </b>Manual review served as the gold standard, identifying 8% as CP masqueraders, 42% as cryptogenic CP, and 50% as non-cryptogenic CP. Computational phenotyping based on ICD-9/10 codes achieved a sensitivity of 95%, specificity of 72%, positive predictive value of 77%, and negative predictive value of 94% in identifying cases warranting genetic testing.</p><p><b>CONCLUSIONS: </b>Our findings demonstrate the feasibility of using computational phenotyping to identify patients with CP or CP- masquerading conditions who warrant genetic testing. Further studies are needed to evaluate the effectiveness and real-world application of this tool in larger healthcare systems. Nonetheless, the computational phenotyping approach holds promise as a possible clinical decision support that could be integrated into electronic health record systems, enhancing clinical workflows and facilitating actionable genetic diagnoses.</p>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2135353342" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
</div>
</div>
</div>
<hr class="hwp-text-button-light-secondary hwp-mt-32"/>
</div>
<div class="hwp-news-footer hwp-py-24 lg:hwp-py-64 hwp-container hwp-container-px">
<div class="page-tags">
</div>
<div class="a2a_kit" data-a2a-title="" data-a2a-url="">
<span class="hwp-mb-16 hwp-block">Share on:</span>
<ul aria-label="Share on:" class="md:hwp-inline-flex md:hwp-flex-wrap">
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_facebook hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-facebook"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Facebook</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_twitter hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-twitter"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Twitter</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_linkedin hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-linkedin"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Linkedin</span>
</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/node/1774776/export -->
<!-- Source: avillach-lab.hms.harvard.edu/publication/neurological-diagnoses-hospitalized-covid-19-patients-associated-adverse-outcomes -->
<h1>Neurological diagnoses in hospitalized COVID-19 patients associated with adverse outcomes: A multinational cohort study. | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/publication/neurological-diagnoses-hospitalized-covid-19-patients-associated-adverse-outcomes'>avillach-lab.hms.harvard.edu/publication/neurological-diagnoses-hospitalized-covid-19-patients-associated-adverse-outcomes</a></div>
<main aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="biblio-details">
<div class="hwp-page-title bibcite-reference" lang="en">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Neurological diagnoses in hospitalized COVID-19 patients associated with adverse outcomes: A multinational cohort study.
    </h1>
</div>
</div>
<div class="md:hwp-pt-64">
<div class="hwp-container hwp-container-px max-lg:hwp-px-0">
<div class="publication-info" data-component-name="publication-info">
<div class="publication-info__image">
</div>
<div class="publication-info__text">
<h4 class="h4 hwp-mb-8">Publication information:</h4>
<div class="publication-info__body">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hutch MR, Son J, Le TT, Hong C, Wang X, Abad ZSH, Morris M, Gutiérrez-Sacristán A, Klann JG, Spiridou A, Batugo A, Bellazzi R, Benoit V, Bonzel CL, Bryant WA, Chiudinelli L, Cho K, Das P, González TG, Hanauer DA, Henderson DW, Ho YL, Loh NHW, Makoudjou A, Makwana S, Malovini A, Moal B, Mowery DL, Neuraz A, Samayamuthu MJ, Vidorreta FJS, Schriver ER, Schubert P, Talbert J, Tan ALM, Tan BWL, Tan BWQ, Tibollo V, Tippman P, Verdy G, Yuan W, Avillach P, Gehlenborg N, Omenn GS, EHR C for CC of C 19 by, Visweswaran S, Cai T, Luo Y, Xia Z. Neurological diagnoses in hospitalized COVID-19 patients associated with adverse outcomes: A multinational cohort study.</div><div class="csl-right-inline">PLOS digital health. 2024;3(4):e0000484. PMID: 38620037</div></div></div>
</div>
</div>
<ul class="publication-info__actions">
<li>
<div class="button-dropdown" data-component-name="button-dropdown">
<button class="hwp-text-link hwp-text-link--icon-left" id="publication_citation_download">
<span aria-hidden="true" class="material-icon">download_for_offline</span>
<span class="hwp-text-link__text">Download citation</span>
</button>
<ul aria-labelledby="publication_citation_download" class="button-dropdown__items" role="menu">
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773921&amp;format=bibtex">BibTeX</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773921&amp;format=endnote8">EndNote X3 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773921&amp;format=endnote7">EndNote 7 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773921&amp;format=tagged">Endnote tagged</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773921&amp;format=marc">Marc</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773921&amp;format=pubmed_id">PubMedId</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773921&amp;format=ris">RIS</a>
</li>
</ul>
</div>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 hwp-pb-64 md:hwp-py-64">
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">Abstract</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-trim hwp-bg-light-base js-hwp-citation" data-component-name="citation">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<p>Few studies examining the patient outcomes of concurrent neurological manifestations during acute COVID-19 leveraged multinational cohorts of adults and children or distinguished between central and peripheral nervous system (CNS vs. PNS) involvement. Using a federated multinational network in which local clinicians and informatics experts curated the electronic health records data, we evaluated the risk of prolonged hospitalization and mortality in hospitalized COVID-19 patients from 21 healthcare systems across 7 countries. For adults, we used a federated learning approach whereby we ran Cox proportional hazard models locally at each healthcare system and performed a meta-analysis on the aggregated results to estimate the overall risk of adverse outcomes across our geographically diverse populations. For children, we reported descriptive statistics separately due to their low frequency of neurological involvement and poor outcomes. Among the 106,229 hospitalized COVID-19 patients (104,031 patients ≥18 years; 2,198 patients &lt;18 years, January 2020-October 2021), 15,101 (14%) had at least one CNS diagnosis, while 2,788 (3%) had at least one PNS diagnosis. After controlling for demographics and pre-existing conditions, adults with CNS involvement had longer hospital stay (11 versus 6 days) and greater risk of (Hazard Ratio = 1.78) and faster time to death (12 versus 24 days) than patients with no neurological condition (NNC) during acute COVID-19 hospitalization. Adults with PNS involvement also had longer hospital stay but lower risk of mortality than the NNC group. Although children had a low frequency of neurological involvement during COVID-19 hospitalization, a substantially higher proportion of children with CNS involvement died compared to those with NNC (6% vs 1%). Overall, patients with concurrent CNS manifestation during acute COVID-19 hospitalization faced greater risks for adverse clinical outcomes than patients without any neurological diagnosis. Our global informatics framework using a federated approach (versus a centralized data collection approach) has utility for clinical discovery beyond COVID-19.</p>
</div>
<div class="hwp-text-block">
<p>Few studies examining the patient outcomes of concurrent neurological manifestations during acute COVID-19 leveraged multinational cohorts of adults and children or distinguished between central and peripheral nervous system (CNS vs. PNS) involvement. Using a federated multinational network in which local clinicians and informatics experts curated the electronic health records data, we evaluated the risk of prolonged hospitalization and mortality in hospitalized COVID-19 patients from 21 healthcare systems across 7 countries. For adults, we used a federated learning approach whereby we ran Cox proportional hazard models locally at each healthcare system and performed a meta-analysis on the aggregated results to estimate the overall risk of adverse outcomes across our geographically diverse populations. For children, we reported descriptive statistics separately due to their low frequency of neurological involvement and poor outcomes. Among the 106,229 hospitalized COVID-19 patients (104,031 patients ≥18 years; 2,198 patients &lt;18 years, January 2020-October 2021), 15,101 (14%) had at least one CNS diagnosis, while 2,788 (3%) had at least one PNS diagnosis. After controlling for demographics and pre-existing conditions, adults with CNS involvement had longer hospital stay (11 versus 6 days) and greater risk of (Hazard Ratio = 1.78) and faster time to death (12 versus 24 days) than patients with no neurological condition (NNC) during acute COVID-19 hospitalization. Adults with PNS involvement also had longer hospital stay but lower risk of mortality than the NNC group. Although children had a low frequency of neurological involvement during COVID-19 hospitalization, a substantially higher proportion of children with CNS involvement died compared to those with NNC (6% vs 1%). Overall, patients with concurrent CNS manifestation during acute COVID-19 hospitalization faced greater risks for adverse clinical outcomes than patients without any neurological diagnosis. Our global informatics framework using a federated approach (versus a centralized data collection approach) has utility for clinical discovery beyond COVID-19.</p>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-354166018" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
</div>
</div>
</div>
<hr class="hwp-text-button-light-secondary hwp-mt-32"/>
</div>
<div class="hwp-news-footer hwp-py-24 lg:hwp-py-64 hwp-container hwp-container-px">
<div class="page-tags">
</div>
<div class="a2a_kit" data-a2a-title="" data-a2a-url="">
<span class="hwp-mb-16 hwp-block">Share on:</span>
<ul aria-label="Share on:" class="md:hwp-inline-flex md:hwp-flex-wrap">
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_facebook hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-facebook"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Facebook</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_twitter hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-twitter"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Twitter</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_linkedin hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-linkedin"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Linkedin</span>
</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/publication/robust-automated-harmonization-heterogeneous-data-through-ensemble-machine-learning -->
<h1>Robust Automated Harmonization of Heterogeneous Data Through Ensemble Machine Learning: Algorithm Development and Validation Study. | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/publication/robust-automated-harmonization-heterogeneous-data-through-ensemble-machine-learning'>avillach-lab.hms.harvard.edu/publication/robust-automated-harmonization-heterogeneous-data-through-ensemble-machine-learning</a></div>
<main aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="biblio-details">
<div class="hwp-page-title bibcite-reference" lang="en">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Robust Automated Harmonization of Heterogeneous Data Through Ensemble Machine Learning: Algorithm Development and Validation Study.
    </h1>
</div>
</div>
<div class="md:hwp-pt-64">
<div class="hwp-container hwp-container-px max-lg:hwp-px-0">
<div class="publication-info" data-component-name="publication-info">
<div class="publication-info__image">
</div>
<div class="publication-info__text">
<h4 class="h4 hwp-mb-8">Publication information:</h4>
<div class="publication-info__body">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yang D, Zhou D, Cai S, Gan Z, Pencina M, Avillach P, Cai T, Hong C. Robust Automated Harmonization of Heterogeneous Data Through Ensemble Machine Learning: Algorithm Development and Validation Study.</div><div class="csl-right-inline">JMIR medical informatics. 2025;13:e54133. PMID: 39844378</div></div></div>
</div>
</div>
<ul class="publication-info__actions">
<li>
<div class="button-dropdown" data-component-name="button-dropdown">
<button class="hwp-text-link hwp-text-link--icon-left" id="publication_citation_download">
<span aria-hidden="true" class="material-icon">download_for_offline</span>
<span class="hwp-text-link__text">Download citation</span>
</button>
<ul aria-labelledby="publication_citation_download" class="button-dropdown__items" role="menu">
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773891&amp;format=bibtex">BibTeX</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773891&amp;format=endnote8">EndNote X3 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773891&amp;format=endnote7">EndNote 7 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773891&amp;format=tagged">Endnote tagged</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773891&amp;format=marc">Marc</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773891&amp;format=pubmed_id">PubMedId</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773891&amp;format=ris">RIS</a>
</li>
</ul>
</div>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 hwp-pb-64 md:hwp-py-64">
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">Abstract</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-trim hwp-bg-light-base js-hwp-citation" data-component-name="citation">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<p><b>BACKGROUND: </b>Cohort studies contain rich clinical data across large and diverse patient populations and are a common source of observational data for clinical research. Because large scale cohort studies are both time and resource intensive, one alternative is to harmonize data from existing cohorts through multicohort studies. However, given differences in variable encoding, accurate variable harmonization is difficult.</p><p><b>OBJECTIVE: </b>We propose SONAR (Semantic and Distribution-Based Harmonization) as a method for harmonizing variables across cohort studies to facilitate multicohort studies.</p><p><b>METHODS: </b>SONAR used semantic learning from variable descriptions and distribution learning from study participant data. Our method learned an embedding vector for each variable and used pairwise cosine similarity to score the similarity between variables. This approach was built off 3 National Institutes of Health cohorts, including the Cardiovascular Health Study, the Multi-Ethnic Study of Atherosclerosis, and the Women's Health Initiative. We also used gold standard labels to further refine the embeddings in a supervised manner.</p><p><b>RESULTS: </b>The method was evaluated using manually curated gold standard labels from the 3 National Institutes of Health cohorts. We evaluated both the intracohort and intercohort variable harmonization performance. The supervised SONAR method outperformed existing benchmark methods for almost all intracohort and intercohort comparisons using area under the curve and top-k accuracy metrics. Notably, SONAR was able to significantly improve harmonization of concepts that were difficult for existing semantic methods to harmonize.</p><p><b>CONCLUSIONS: </b>SONAR achieves accurate variable harmonization within and between cohort studies by harnessing the complementary strengths of semantic learning and variable distribution learning.</p>
</div>
<div class="hwp-text-block">
<p><b>BACKGROUND: </b>Cohort studies contain rich clinical data across large and diverse patient populations and are a common source of observational data for clinical research. Because large scale cohort studies are both time and resource intensive, one alternative is to harmonize data from existing cohorts through multicohort studies. However, given differences in variable encoding, accurate variable harmonization is difficult.</p><p><b>OBJECTIVE: </b>We propose SONAR (Semantic and Distribution-Based Harmonization) as a method for harmonizing variables across cohort studies to facilitate multicohort studies.</p><p><b>METHODS: </b>SONAR used semantic learning from variable descriptions and distribution learning from study participant data. Our method learned an embedding vector for each variable and used pairwise cosine similarity to score the similarity between variables. This approach was built off 3 National Institutes of Health cohorts, including the Cardiovascular Health Study, the Multi-Ethnic Study of Atherosclerosis, and the Women's Health Initiative. We also used gold standard labels to further refine the embeddings in a supervised manner.</p><p><b>RESULTS: </b>The method was evaluated using manually curated gold standard labels from the 3 National Institutes of Health cohorts. We evaluated both the intracohort and intercohort variable harmonization performance. The supervised SONAR method outperformed existing benchmark methods for almost all intracohort and intercohort comparisons using area under the curve and top-k accuracy metrics. Notably, SONAR was able to significantly improve harmonization of concepts that were difficult for existing semantic methods to harmonize.</p><p><b>CONCLUSIONS: </b>SONAR achieves accurate variable harmonization within and between cohort studies by harnessing the complementary strengths of semantic learning and variable distribution learning.</p>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2086817716" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
</div>
</div>
</div>
<hr class="hwp-text-button-light-secondary hwp-mt-32"/>
</div>
<div class="hwp-news-footer hwp-py-24 lg:hwp-py-64 hwp-container hwp-container-px">
<div class="page-tags">
</div>
<div class="a2a_kit" data-a2a-title="" data-a2a-url="">
<span class="hwp-mb-16 hwp-block">Share on:</span>
<ul aria-label="Share on:" class="md:hwp-inline-flex md:hwp-flex-wrap">
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_facebook hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-facebook"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Facebook</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_twitter hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-twitter"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Twitter</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_linkedin hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-linkedin"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Linkedin</span>
</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/node/1774776/export -->
<!-- Source: avillach-lab.hms.harvard.edu/node/1774776/export -->
<!-- Source: avillach-lab.hms.harvard.edu/publication/varppud-variant-post-prioritization-developed-undiagnosed-genetic-disorders -->
<h1>VarPPUD: Variant post prioritization developed for undiagnosed genetic disorders. | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/publication/varppud-variant-post-prioritization-developed-undiagnosed-genetic-disorders'>avillach-lab.hms.harvard.edu/publication/varppud-variant-post-prioritization-developed-undiagnosed-genetic-disorders</a></div>
<main aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="biblio-details">
<div class="hwp-page-title bibcite-reference" lang="en">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      VarPPUD: Variant post prioritization developed for undiagnosed genetic disorders.
    </h1>
</div>
</div>
<div class="md:hwp-pt-64">
<div class="hwp-container hwp-container-px max-lg:hwp-px-0">
<div class="publication-info" data-component-name="publication-info">
<div class="publication-info__image">
</div>
<div class="publication-info__text">
<h4 class="h4 hwp-mb-8">Publication information:</h4>
<div class="publication-info__body">
<div class="bibcite-citation">
<div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Gutierrez A, Network UD, Kobren SN, Avillach P. VarPPUD: Variant post prioritization developed for undiagnosed genetic disorders.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2024; PMID: 38699371</div></div></div>
</div>
</div>
<ul class="publication-info__actions">
<li>
<div class="button-dropdown" data-component-name="button-dropdown">
<button class="hwp-text-link hwp-text-link--icon-left" id="publication_citation_download">
<span aria-hidden="true" class="material-icon">download_for_offline</span>
<span class="hwp-text-link__text">Download citation</span>
</button>
<ul aria-labelledby="publication_citation_download" class="button-dropdown__items" role="menu">
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773916&amp;format=bibtex">BibTeX</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773916&amp;format=endnote8">EndNote X3 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773916&amp;format=endnote7">EndNote 7 XML</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773916&amp;format=tagged">Endnote tagged</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773916&amp;format=marc">Marc</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773916&amp;format=pubmed_id">PubMedId</a>
</li>
<li role="menuitem">
<a class="analytics-cta" href="/bibcite/export?content_filter%5B0%5D=1773916&amp;format=ris">RIS</a>
</li>
</ul>
</div>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 hwp-pb-64 md:hwp-py-64">
<div class="hwp-citations-group">
<h3 class="h3 hwp-mb-24">Abstract</h3>
<div class="hwp-citations-group__inner">
<div class="hwp-citation hwp-citation--text-trim hwp-bg-light-base js-hwp-citation" data-component-name="citation">
<div class="hwp-citation__text">
<div class="hwp-text-block__default-text">
<p>Rare and ultra-rare genetic conditions are estimated to impact nearly 1 in 17 people worldwide, yet accurately pinpointing the diagnostic variants underlying each of these conditions remains a formidable challenge. Because comprehensive,  functional assessment of all possible genetic variants is infeasible, clinicians instead consider  variant pathogenicity predictions to distinguish plausibly disease-causing from benign variants across the genome. However, in the most difficult undiagnosed cases, such as those accepted to the Undiagnosed Diseases Network (UDN), existing pathogenicity predictions cannot reliably discern true etiological variant(s) from other deleterious candidate variants that were prioritized through N-of-1 efforts. Pinpointing the disease-causing variant from a pool of plausible candidates remains a largely manual effort requiring extensive clinical workups, functional and experimental assays, and eventual identification of genotype- and phenotype-matched individuals. Here, we introduce VarPPUD, a tool trained on prioritized variants from UDN cases, that leverages gene-, amino acid-, and nucleotide-level features to discern pathogenic variants from other deleterious variants that are unlikely to be confirmed as disease relevant. VarPPUD achieves a cross-validated accuracy of 79.3% and precision of 77.5% on a held-out subset of uniquely challenging UDN cases, respectively representing an average 18.6% and 23.4% improvement over nine traditional pathogenicity prediction approaches on this task. We validate VarPPUD's ability to discriminate likely from unlikely pathogenic variants on synthetic, GAN-generated candidate variants as well. Finally, we show how VarPPUD can be probed to evaluate each input feature's importance and contribution toward prediction-an essential step toward understanding the distinct characteristics of newly-uncovered disease-causing variants.</p>
</div>
<div class="hwp-text-block">
<p>Rare and ultra-rare genetic conditions are estimated to impact nearly 1 in 17 people worldwide, yet accurately pinpointing the diagnostic variants underlying each of these conditions remains a formidable challenge. Because comprehensive,  functional assessment of all possible genetic variants is infeasible, clinicians instead consider  variant pathogenicity predictions to distinguish plausibly disease-causing from benign variants across the genome. However, in the most difficult undiagnosed cases, such as those accepted to the Undiagnosed Diseases Network (UDN), existing pathogenicity predictions cannot reliably discern true etiological variant(s) from other deleterious candidate variants that were prioritized through N-of-1 efforts. Pinpointing the disease-causing variant from a pool of plausible candidates remains a largely manual effort requiring extensive clinical workups, functional and experimental assays, and eventual identification of genotype- and phenotype-matched individuals. Here, we introduce VarPPUD, a tool trained on prioritized variants from UDN cases, that leverages gene-, amino acid-, and nucleotide-level features to discern pathogenic variants from other deleterious variants that are unlikely to be confirmed as disease relevant. VarPPUD achieves a cross-validated accuracy of 79.3% and precision of 77.5% on a held-out subset of uniquely challenging UDN cases, respectively representing an average 18.6% and 23.4% improvement over nine traditional pathogenicity prediction approaches on this task. We validate VarPPUD's ability to discriminate likely from unlikely pathogenic variants on synthetic, GAN-generated candidate variants as well. Finally, we show how VarPPUD can be probed to evaluate each input feature's importance and contribution toward prediction-an essential step toward understanding the distinct characteristics of newly-uncovered disease-causing variants.</p>
</div>
</div>
<ul class="hwp-citation__ctas">
<li class="expand-button-wrap">
<button aria-controls="hwp-citation-body-content-2000961178" aria-expanded="false" class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand">
<span aria-hidden="true" class="material-icon material-icon--show">add_circle</span>
<span aria-hidden="true" class="material-icon material-icon--hide">do_not_disturb_on</span>
<span class="hwp-text-link__text">Abstract</span>
</button>
</li>
</ul>
</div>
</div>
</div>
<hr class="hwp-text-button-light-secondary hwp-mt-32"/>
</div>
<div class="hwp-news-footer hwp-py-24 lg:hwp-py-64 hwp-container hwp-container-px">
<div class="page-tags">
</div>
<div class="a2a_kit" data-a2a-title="" data-a2a-url="">
<span class="hwp-mb-16 hwp-block">Share on:</span>
<ul aria-label="Share on:" class="md:hwp-inline-flex md:hwp-flex-wrap">
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_facebook hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-facebook"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Facebook</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_twitter hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-twitter"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Twitter</span>
</a>
</li>
<li class="md:hwp-mr-40 last:md:hwp-mr-0 max-md:hwp-mb-16">
<a class="a2a_button_linkedin hwp-icon-link hwp-icon-link--dark-primary" href="#">
<span class="hwp-icon-link__icon">
<svg aria-hidden="true" class="hwp-icon hwp-h-24 hwp-w-24">
<use xlink:href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/dist/icons.svg#hwp-linkedin"></use>
</svg>
</span>
<span class="hwp-icon-link__text">Linkedin</span>
</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/autism-pilot -->
<h1>Autism Pilot | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/autism-pilot'>avillach-lab.hms.harvard.edu/autism-pilot</a></div>
<main aria-label="Autism Pilot" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Autism Pilot</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	BCH internally-funded Gene partnership / Research Connection</p><p>	Co-Investigator ($800k) (PI: Isaac Kohane)</p><p>	Creating a comprehensive autism clinical and genomic database using an i2b2-based research platform: tranSMART</p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/exploring-and-understanding-adverse-drug-reactions-integrative-mining-clinical-records -->
<h1>Exploring and Understanding Adverse Drug Reactions by Integrative Mining of Clinical Records and Biomedical Knowledge | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/exploring-and-understanding-adverse-drug-reactions-integrative-mining-clinical-records'>avillach-lab.hms.harvard.edu/exploring-and-understanding-adverse-drug-reactions-integrative-mining-clinical-records</a></div>
<main aria-label="Exploring and Understanding Adverse Drug Reactions by Integrative Mining of Clinical Records and Biomedical Knowledge" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Exploring and Understanding Adverse Drug Reactions by Integrative Mining of Clinical Records and Biomedical Knowledge</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	FP7 - European Commission grant #215847 - <a href="http://www.euadr-project.org">http://www.euadr-project.org</a></p><p>	Site co-PI and leader of one specific aim (out of 8 in the project) ($266,000) (Total project: $8 million) (PI: Johan Van der Lei)</p><p>	Develop an innovative computerized system to detect adverse drug reactions supplementing spontaneous reporting systems. To achieve this objective, EU-ADR exploited clinical data from electronic healthcare records (EHRs) of over 30 million patients from several European countries. </p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/randomized-control-study-impact-computerized-prescription-entry-drug-prescription -->
<h1>Randomized control study on the impact of computerized prescription entry on drug prescription by undergraduate medical students | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/randomized-control-study-impact-computerized-prescription-entry-drug-prescription'>avillach-lab.hms.harvard.edu/randomized-control-study-impact-computerized-prescription-entry-drug-prescription</a></div>
<main aria-label="Randomized control study on the impact of computerized prescription entry on drug prescription by undergraduate medical students" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Randomized control study on the impact of computerized prescription entry on drug prescription by undergraduate medical students</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	University Paris Descartes innovative grant in pedagogy</p><p>	PI ($7,000)</p><p>	Students in their final year of medical studies learn, in addition to theoretical education faculty, prescription real world "bedside" with supervision by senior physicians. The assumption is that the formalization of an entire prescription with the support of electronic prescribing, under the supervision of a senior to explain the mistakes, is a new educational tool for learning prescritpion in front of the patient bed. We performed a randomized controlled trail to measure the impact on improving the prescribing knowledge.</p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/nihncats-global-rare-diseases-patient-registry-i2b2transmart-data-repository -->
<h1>NIH/NCATS Global Rare Diseases Patient Registry i2b2/tranSMART Data Repository | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/nihncats-global-rare-diseases-patient-registry-i2b2transmart-data-repository'>avillach-lab.hms.harvard.edu/nihncats-global-rare-diseases-patient-registry-i2b2transmart-data-repository</a></div>
<main aria-label="NIH/NCATS Global Rare Diseases Patient Registry i2b2/tranSMART Data Repository" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>NIH/NCATS Global Rare Diseases Patient Registry i2b2/tranSMART Data Repository</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>NIH/NCATS</p><p>Co-PI. Total:$ 2.4 M. PI: Isaac Kohane</p><p>The integration of clinical and biomedical data hosted in multiple distributed repositories is confronted by two significant challenges: i) correctly linking information pertaining to the same patient across repositories, for example, linking lab results data with bedside observations data; and ii) making data available for analysis at different locations across a collaboration network. These problems are exacerbated in the case of rare diseases research, given the very limited availability of data sets and data standards. We propose to develop the NCAT Global Repository for Rare Diseases Research (GRDR) based on BD2K PIC-SURE platform to address these challenges. NCAT GRDR repository will be a scalable, secure, and flexible integration architecture for clinical and biomedical datasets, which by extending the successful i2b2/tranSMART platform will allow data providers to easily share their data with the wider research community without requiring them to subscribe to proprietary vocabulary standards or to develop complex mapping protocols. Using federated data access and querying methods that retrieve relevant data from different locations before combining them, GRDR will make it possible for comparative analysis methods to be executed on the integrated datasets. By assigning generic identifiers (after de-identification) to related data across locations, GRDR will ease the difficulties of linking data while conforming to the requirements of patient data privacy and other security regulations. We aim to progressively integrate up to 17 multidimensional, and heterogeneous patient registries on GRDR and provide encrypted access to this data to authenticated users over a secure connection.</p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/i2b2transmart-pilot-study-cloud-using-redshift-phase-i -->
<h1>i2b2/tranSMART pilot study on the cloud using Redshift - Phase I | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/i2b2transmart-pilot-study-cloud-using-redshift-phase-i'>avillach-lab.hms.harvard.edu/i2b2transmart-pilot-study-cloud-using-redshift-phase-i</a></div>
<main aria-label="i2b2/tranSMART pilot study on the cloud using Redshift - Phase I" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>i2b2/tranSMART pilot study on the cloud using Redshift - Phase I</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	Amazon Web Services Research Grant</p><p>	EDU_R_FY2015_Q2_HarvardMedicalSchool_Avillach-NEW</p><p>	PI $100K</p><p>	To handle increasing data volumes and potential new sources of data, and to promote patient engagement, we intend to conduct a pilot study that involves hosting this infrastructure on a paradigm which provides high storage, computing power and availability, AWS.</p><p>	Our project plan comprises 2 phases:</p><p>	- Phase 1 will involve transforming the open access NDAR (National Database for Autism Research) database to the i2b2/tranSMART model. The transformed NDAR and the existing Oracle databases will be ported to the AWS cloud. The computing power of AWS will be leveraged to identify comorbidities between PMS and autism patients, and correlations between clinical and genomic data.</p><p>	- Phase 2 will involve the adaptation of the i2b2 data model to Redshift Petabyte for the storage and analysis of increasing volumes of patient data. This will entail transferring NDAR phenotypic and annotated genetic data and PMS_DN data to Redshift nodes. </p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/delineating-autism-subtypes-phenotype-wide-scan-across-genome-wide-genotypes-patient -->
<h1>Delineating autism subtypes by phenotype-wide scan across genome-wide genotypes in a patient-centric information commons | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/delineating-autism-subtypes-phenotype-wide-scan-across-genome-wide-genotypes-patient'>avillach-lab.hms.harvard.edu/delineating-autism-subtypes-phenotype-wide-scan-across-genome-wide-genotypes-patient</a></div>
<main aria-label="Delineating autism subtypes by phenotype-wide scan across genome-wide genotypes in a patient-centric information commons" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Delineating autism subtypes by phenotype-wide scan across genome-wide genotypes in a patient-centric information commons</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	Nancy Lurie Marks Foundation</p><p>	PI ($100k)</p><p>	This grant is for a one-year study that would result in a phenome-wide scan across all genotypes measured by SNP array. This will likely generate novel insights about the substructure of the different autisms.</p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/neuropsychiatric-genome-scale-and-rdoc-individualized-domains-n-grid-0 -->
<h1>Neuropsychiatric Genome-Scale and RDoC Individualized Domains (N-GRID) | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/neuropsychiatric-genome-scale-and-rdoc-individualized-domains-n-grid-0'>avillach-lab.hms.harvard.edu/neuropsychiatric-genome-scale-and-rdoc-individualized-domains-n-grid-0</a></div>
<main aria-label="Neuropsychiatric Genome-Scale and RDoC Individualized Domains (N-GRID)" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Neuropsychiatric Genome-Scale and RDoC Individualized Domains (N-GRID)</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	NIH P50 MH106933</p><p>	Co-Investigator (PI: Isaac Kohane)</p><p>	As a result of the accelerated pace of development of technologies for characterizing the human genome, the rate-limiting step for large scale genomic investigation in clinical populations is now phenotyping. This is particularly the case for neuropsychiatric (NP) illness, where phenotypes are complex, biomarkers are lacking, and the primary cell types of interest are difficult to access directly. It has become apparent that both rare and common genetic variation contributes to disease risk and that this risk crosses traditional diagnostic boundaries in psychiatry. Taking advantage of a large, already-established NP biobank could dramatically accelerate progress toward understanding the cross-disorder mechanism of action of disease liability genes. This study proposes novel applications of emerging technologies in informatics and cellular neurobiology to eliminate this phenotyping bottleneck. In doing so, it will accelerate investigation of clinical and cellular phenoytpes for understanding single and multi-locus/polygenic associations.  </p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/evaluating-scalability-packaged-open-source-biomedical-informatics-applications-across-1 -->
<h1>Evaluating the scalability of packaged open source biomedical informatics applications across cloud based providers | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/evaluating-scalability-packaged-open-source-biomedical-informatics-applications-across-1'>avillach-lab.hms.harvard.edu/evaluating-scalability-packaged-open-source-biomedical-informatics-applications-across-1</a></div>
<main aria-label="Evaluating the scalability of packaged open source biomedical informatics applications across cloud based providers" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Evaluating the scalability of packaged open source biomedical informatics applications across cloud based providers</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	NIH Commons Funds</p><p>	PI ($50K)</p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/phelan-mcdermid-syndrome-data-network-pmsdn-phase-i -->
<h1>Phelan-McDermid Syndrome Data Network (PMS_DN) - Phase I | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/phelan-mcdermid-syndrome-data-network-pmsdn-phase-i'>avillach-lab.hms.harvard.edu/phelan-mcdermid-syndrome-data-network-pmsdn-phase-i</a></div>
<main aria-label="Phelan-McDermid Syndrome Data Network (PMS_DN) - Phase I" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Phelan-McDermid Syndrome Data Network (PMS_DN) - Phase I</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	PCORI - PPRN</p><p>	Co-PI ($624K) Total $950K (PI: Megan O'Boyle)</p><p>	To collect all available patient data from Phelan-McDermid Syndrome (PMS) patients to make meaningful, well-annotated clinical data available to researchers and to share insights with members of the PCORI network.</p><p>	TranSMART platform based on i2b2 is being used to integrate Patient Reported Outcomes and Knowledge extracted from Clinical Notes using cTAKES</p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/nihncats-global-rare-diseases-patient-registry-i2b2transmart-data-repository-0 -->
<h1>NIH/NCATS Global Rare Diseases Patient Registry i2b2/tranSMART Data Repository | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/nihncats-global-rare-diseases-patient-registry-i2b2transmart-data-repository-0'>avillach-lab.hms.harvard.edu/nihncats-global-rare-diseases-patient-registry-i2b2transmart-data-repository-0</a></div>
<main aria-label="NIH/NCATS Global Rare Diseases Patient Registry i2b2/tranSMART Data Repository" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>NIH/NCATS Global Rare Diseases Patient Registry i2b2/tranSMART Data Repository</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>NIH/NCATS</p><p>Co-PI  Total $2.4M  (PI: Isaac Kohane)</p><p>The integration of clinical and biomedical data hosted in multiple distributed repositories is confonted by two significant challenges: 1) correctly linking information pertaining to the same patient across repositories, for example, linking lab results data with bedside observations data; and 2) making data available for analysis at different locations across a collaboration network. These problems are exacerbated in the case of rare diseases research, given the very limited availability of data sets and data standards. We propose to develop the NCAT Global Repository for Rare Diseases Research (GRDR) based on BD2K PIC-SURE platform to address these challenges. NCAT GRDR repository will be a scalable, secure, and flexible integration architecture for clinical and biomedical datasets, which by extending the successful i2b2/tranSMART platform will allow data providers to easily share their data with the wider research community without requiring them to subscribe to proprietary vocabulary standards or to develop complex mapping protocols. Using federated data access and querying methods that retrieve relevant data from different locations before combining them, GRDR will make it possible for comparative analysis methods to be executed on the integrated datasets. By assigning generic identifiers (after de-identification) to related data across locations, GRDR will ease the difficulties of linking data while conforming to the requirements of patient data privacy and other security regulations. We aim to progressively integrate up to 17 multidimensional, and heterogeneous patient registries on GRDR and provide encrypted access to this data to authenticated users over a secure connection. </p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/evaluating-scalability-packaged-open-source-biomedical-informatics-applications-across-2 -->
<h1>Evaluating the scalability of packaged open source biomedical informatics applications across cloud based providers - Phase II | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/evaluating-scalability-packaged-open-source-biomedical-informatics-applications-across-2'>avillach-lab.hms.harvard.edu/evaluating-scalability-packaged-open-source-biomedical-informatics-applications-across-2</a></div>
<main aria-label="Evaluating the scalability of packaged open source biomedical informatics applications across cloud based providers - Phase II" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Evaluating the scalability of packaged open source biomedical informatics applications across cloud based providers - Phase II</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	NIH Commons Funds</p><p>	PI ($49K)</p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/2016-2-presentationvideo-nih-bd2k-pic-sure-supplements-1-ncats-grdr-2-pittsburg-ccd-3 -->
<h1>2016-02 PRESENTATION/VIDEO - NIH BD2K PIC-SURE Supplements 1) NCATS GRDR, 2) Pittsburg CCD & 3) Count Everything | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/2016-2-presentationvideo-nih-bd2k-pic-sure-supplements-1-ncats-grdr-2-pittsburg-ccd-3'>avillach-lab.hms.harvard.edu/2016-2-presentationvideo-nih-bd2k-pic-sure-supplements-1-ncats-grdr-2-pittsburg-ccd-3</a></div>
<main aria-label="2016-02 PRESENTATION/VIDEO - NIH BD2K PIC-SURE Supplements 1) NCATS GRDR, 2) Pittsburg CCD &amp; 3) Count Everything" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>2016-02 PRESENTATION/VIDEO - NIH BD2K PIC-SURE Supplements 1) NCATS GRDR, 2) Pittsburg CCD &amp; 3) Count Everything</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>NIH BD2K PIC-SURE Supplements 1) NCATS GRDR, 2) Pittsburg CCD &amp; 3) Count Everything:  <a href="https://vimeo.com/156471254">https://vimeo.com/156471254</a> </p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/i2b2transmart-pilot-study-cloud-using-redshift-phase-ii -->
<h1>i2b2/tranSMART pilot study on the cloud using Redshift - Phase II | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/i2b2transmart-pilot-study-cloud-using-redshift-phase-ii'>avillach-lab.hms.harvard.edu/i2b2transmart-pilot-study-cloud-using-redshift-phase-ii</a></div>
<main aria-label="i2b2/tranSMART pilot study on the cloud using Redshift - Phase II" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>i2b2/tranSMART pilot study on the cloud using Redshift - Phase II</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	Amazon Web Services Research</p><p>	Grant EDU_R_FY2016</p><p>	PI $55K</p><p>	To handle increasing data volumes and potential new sources of data, and to promote patient engagement, we intend to conduct a pilot study that involves hosting this infrastructure on a paradigm which provides high storage, computing power and availability, AWS.</p><p>	Our project plan comprises 2 phases:</p><p>	- Phase 1 will involve transforming the open access NDAR (National Database for Autism Research) database to the i2b2/tranSMART model. The transformed NDAR and the existing Oracle databases will be ported to the AWS cloud. The computing power of AWS will be leveraged to identify comorbidities between PMS and autism patients, and correlations between clinical and genomic data.</p><p>	- Phase 2 will involve the adaptation of the i2b2 data model to Redshift Petabyte for the storage and analysis of increasing volumes of patient data. This will entail transferring NDAR phenotypic and annotated genetic data and PMS_DN data to Redshift nodes. </p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/center-causal-modeling-and-discovery-biomedical-knowledge-big-data-ccd-phase-ii -->
<h1>Center for Causal Modeling and Discovery of Biomedical Knowledge from Big Data (CCD) - Phase II | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/center-causal-modeling-and-discovery-biomedical-knowledge-big-data-ccd-phase-ii'>avillach-lab.hms.harvard.edu/center-causal-modeling-and-discovery-biomedical-knowledge-big-data-ccd-phase-ii</a></div>
<main aria-label="Center for Causal Modeling and Discovery of Biomedical Knowledge from Big Data (CCD) - Phase II" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Center for Causal Modeling and Discovery of Biomedical Knowledge from Big Data (CCD) - Phase II</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	NIH BD2K Admin Supplement 3U54HG008540-03S1</p><p>	Co-investigator (PI: Cooper Greg, University of Pittsburgh)</p><p>	The Center for Causal Modeling and Discovery of Biomedical Knowledge from Big Data (CCD) and the Patient-centered Information Commons (PIC-SURE) will collaborate to develop a system that shares data, and computational accessibility in an externally hosted environment (i.e., cloud). We will start by developing a standardized RESTful Application-Programming-Interface (API) that will enable our organizations to share data and the analysis results. This will serve as providing the base for other BD2K centers and organizations to share data and services in the future. </p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/electronic-health-records-clinical-research-ehr4cr -->
<h1>Electronic Health Records for Clinical Research (EHR4CR) | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/electronic-health-records-clinical-research-ehr4cr'>avillach-lab.hms.harvard.edu/electronic-health-records-clinical-research-ehr4cr</a></div>
<main aria-label="Electronic Health Records for Clinical Research (EHR4CR)" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Electronic Health Records for Clinical Research (EHR4CR)</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	IMI European Commission: Grant No 115189-1 httpL//<a href="http://www.ehr4cr.eu">www.ehr4cr.eu</a></p><p>	Co-Investigator (PI: George de Moore)</p><p>	Support the feasibility, exploration, design, and execution of clinical studies and long-term surveillance of patient populations from multiple European hospitals. Enable trial eligibility and recruitment criteria to be expressed in ways that permit searching for relevant patients across EHR distributed systems, and confidentially initiate participation requests via the patients' authorized clinicians. </p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/center-causal-modeling-and-discovery-biomedical-knowledge-big-data-ccd -->
<h1>Center for Causal Modeling and Discovery of Biomedical Knowledge from Big Data (CCD) | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/center-causal-modeling-and-discovery-biomedical-knowledge-big-data-ccd'>avillach-lab.hms.harvard.edu/center-causal-modeling-and-discovery-biomedical-knowledge-big-data-ccd</a></div>
<main aria-label="Center for Causal Modeling and Discovery of Biomedical Knowledge from Big Data (CCD)" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Center for Causal Modeling and Discovery of Biomedical Knowledge from Big Data (CCD)</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	NIH BD2K Admin Supplement 3U54HG008540-02S1</p><p>	Co-investigator (PI: Cooper Greg, University of Pittsburgh)</p><p>	The Center for Causal Modeling and Discovery of Biomedical Knowledge from Big Data (CCD) and the Patient-centered Information Commons (PIC-SURE) will collaborate to develop a system that shares data, and computational accessibility in an externally hosted environment (i.e., cloud). We will start by developing a standardized RESTful Application-Programming-Interface (API) that will enable our organizations to share data and the analysis results. This will serve as providing the abse for other BD2K centers and organizations to share data and services in the future.</p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/pcori-cdrn-scalable-collaborative-infrastructure-learning-healthcare-system-scilhs -->
<h1>PCORI CDRN: Scalable Collaborative Infrastructure for a Learning Healthcare System (SCILHS) - Phase II | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/pcori-cdrn-scalable-collaborative-infrastructure-learning-healthcare-system-scilhs'>avillach-lab.hms.harvard.edu/pcori-cdrn-scalable-collaborative-infrastructure-learning-healthcare-system-scilhs</a></div>
<main aria-label="PCORI CDRN: Scalable Collaborative Infrastructure for a Learning Healthcare System (SCILHS) - Phase II" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>PCORI CDRN: Scalable Collaborative Infrastructure for a Learning Healthcare System (SCILHS) - Phase II</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>PCORI - CDRN</p><p>Co-Investigator</p><p>PI: Kenneth Mandl</p><p>Leveraging an informatics infrastructure that our investigators have developed over the past 15 years, the Department of Biomedical Informatics (DBMI) at Harvard Medical School will work across 10 healthcare centers throughout the U.S. to develop the Scalable Collaborative Infrastructure for a Learning Health System (SCILHS, pronounced “skills”). An open-source platform, the system will cover more than 8 million patients and enable clinician and patient participation in research</p><p> </p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/bch-precision-link-genomics-research-and-innovation-network-grin -->
<h1>BCH Precision Link - Genomics Research and Innovation Network (GRIN) | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/bch-precision-link-genomics-research-and-innovation-network-grin'>avillach-lab.hms.harvard.edu/bch-precision-link-genomics-research-and-innovation-network-grin</a></div>
<main aria-label="BCH Precision Link - Genomics Research and Innovation Network (GRIN)" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>BCH Precision Link - Genomics Research and Innovation Network (GRIN)</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>Boston Children's Hospital (BCH), Cincinnati Children's Hospital Medical Center (CCHMC), Children's Hospital of Philadelphia (CHOP)</p><p>Co-Investigator BCH  Total: $6M  PI BCH: Ken Mandl</p><p>To accelerate genomic discovery, collaboration, and improved clinical outcomes, research leaders at Boston Children's Hospital (BCH), Cincinnati Children's Hospital Medical Center (CCHMC), and Children's Hospital of Philadelphia (CHOP) have come together to propose the creation of the Genomics Research and Innovation Network (GRIN). </p><p>Four major potential uses of the GRIN resource, including:</p><p>1) Facilitating cohort expansion - especially valuable in the study of rare disease</p><p>2) Creation of a broad knowledgebase for variant interpretation</p><p>3) Matching junior investigators to clinical and research expertise</p><p>4) Matching investigators to expertise and resources for functional studies and animal models</p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/count-everything -->
<h1>Count Everything | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/count-everything'>avillach-lab.hms.harvard.edu/count-everything</a></div>
<main aria-label="Count Everything" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Count Everything</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	NIH BD2K Admin Supplement 3U54HG008540</p><p>	Co-Investigator (PI: Haussler / Kohane / Ohno-Machado)</p><p>	Integrate BD2K RESTful clinical API with GH4GH Genomic RESTful API</p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/data-fusion-self-scaling-open-source-registry-advancing-pediatric-pulmonary-vascular -->
<h1>Data Fusion - A Self-Scaling, Open Source Registry Advancing Pediatric Pulmonary Vascular Disease Research | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/data-fusion-self-scaling-open-source-registry-advancing-pediatric-pulmonary-vascular'>avillach-lab.hms.harvard.edu/data-fusion-self-scaling-open-source-registry-advancing-pediatric-pulmonary-vascular</a></div>
<main aria-label="Data Fusion - A Self-Scaling, Open Source Registry Advancing Pediatric Pulmonary Vascular Disease Research" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Data Fusion - A Self-Scaling, Open Source Registry Advancing Pediatric Pulmonary Vascular Disease Research</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	NHLBI/NIH 3U01HL121518-02S1</p><p>	Co-Investigator (PI: Mandl/Abman)</p><p>	We will enhance a registry of children with pulmonary hypertension by improving the quality of data on medications and adverse events. Our approach leverages uniquely valuable data from pharmacies as well as computerized reading and interpretation of free text clinician notes. </p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/i2b2-clinical-data-warehouse-cdw-implementation-h%C3%B4pital-europ%C3%A9en-georges-pompidou-aphp -->
<h1>I2b2 Clinical Data Warehouse (CDW) implementation at Hôpital Européen Georges Pompidou - APHP, Paris, France | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/i2b2-clinical-data-warehouse-cdw-implementation-h%C3%B4pital-europ%C3%A9en-georges-pompidou-aphp'>avillach-lab.hms.harvard.edu/i2b2-clinical-data-warehouse-cdw-implementation-h%C3%B4pital-europ%C3%A9en-georges-pompidou-aphp</a></div>
<main aria-label="I2b2 Clinical Data Warehouse (CDW) implementation at Hôpital Européen Georges Pompidou - APHP, Paris, France" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>I2b2 Clinical Data Warehouse (CDW) implementation at Hôpital Européen Georges Pompidou - APHP, Paris, France</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	Hospital internally funded</p><p>	Co-PI ($200,000) (PI: Patrice Degoulet)</p><p>	The i2b2 CDW allows reuse of computerized data of patients for achieving cohorts. The health information on 606,000 patients with diagnoses, symptoms, medical procedures, drug therapy, etc. for 13 years were made available for research. </p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/emif-european-medical-information-framework-0 -->
<h1>EMIF: European Medical Information Framework | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/emif-european-medical-information-framework-0'>avillach-lab.hms.harvard.edu/emif-european-medical-information-framework-0</a></div>
<main aria-label="EMIF: European Medical Information Framework" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>EMIF: European Medical Information Framework</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	IMI European Commission: Grant No 115372 <a href="http://www.imi.europa.eu/content/emif">http://www.imi.europa.eu/content/emif</a></p><p>	Site PI and Task Leader ($250K) (total project: $74 million) (PI: Simon Lovestone / Johan van der Lei)</p><p>	Generating a common patient health information framework addressing the two research topics: Obesity and its metabolic complications; and Markers for the development of Alzheimer's disease (AD) and other dementias. Existing human health data sources that are within scope of EMIF can be grouped into three types: 1) General-purpose-collected EHR data - 52 million patients across Europe, 2) Cohort data and 3) Biobanks</p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/patient-centered-information-commons-0 -->
<h1>Patient-Centered Information Commons | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/patient-centered-information-commons-0'>avillach-lab.hms.harvard.edu/patient-centered-information-commons-0</a></div>
<main aria-label="Patient-Centered Information Commons" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Patient-Centered Information Commons</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	NIH/BD2K U54 HG007963</p><p>	Co-Investigator (PI: Isaac Kohane)</p><p>	We propose to create a massively scalable toolkit to enable large, multi-center Patient-centered Informatic Commons (PIC) at loval, regional, and national scale, where the focus is the alignment of all available biomedical data per individual. Such a Commons is a prerequisite for conducting the large-N, Big Data, longitudinal studies essential for understanding causation in the Precision Medicine framework while simultaneously addressing key complexities of Patient Centric Outcome Research studies required under ACA (Affordable Care Act).</p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/nhlbi-cure-sickle-cell-initiative -->
<h1>NHLBI Cure Sickle Cell Initiative | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/nhlbi-cure-sickle-cell-initiative'>avillach-lab.hms.harvard.edu/nhlbi-cure-sickle-cell-initiative</a></div>
<main aria-label="NHLBI Cure Sickle Cell Initiative " aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>NHLBI Cure Sickle Cell Initiative </span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>The Cure Sickle Cell Initiative is a collaborative, patient-focused research eﬀort designed to accelerate promising genetic therapies to cure sickle cell disease. The Initiative considers non-traditional ways to advance research and brings together the sickle cell disease (SCD) community—patients, advocates, caregivers, providers, researchers, industry, and others.</p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/interagency-personnel-agreement-veterans-administration -->
<h1>Interagency Personnel Agreement with Veteran's Administration | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/interagency-personnel-agreement-veterans-administration'>avillach-lab.hms.harvard.edu/interagency-personnel-agreement-veterans-administration</a></div>
<main aria-label="Interagency Personnel Agreement with Veteran's Administration" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Interagency Personnel Agreement with Veteran's Administration</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	Veteran's Administration</p><p>	PI ($20K)</p><p>	To advise the VA administration in the creation of a clinical and genomic data warehouse. </p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/nhlbi-biodatacatalyst -->
<h1>NHLBI BioData Catalyst | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/nhlbi-biodatacatalyst'>avillach-lab.hms.harvard.edu/nhlbi-biodatacatalyst</a></div>
<main aria-label="NHLBI BioData Catalyst" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>NHLBI BioData Catalyst</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>NHLBI BioData Catalyst® (BDC) is a cloud-based platform providing tools, applications, and workflows in secure workspaces. By increasing access to NHLBI datasets and innovative data analysis capabilities, BDC accelerates efficient biomedical research that drives discovery and scientific advancement, leading to novel diagnostic tools, therapeutics, and prevention strategies for heart, lung, blood, and sleep disorders.</p><p>Though the primary goal of the BDC project is to build a data science ecosystem, at its core, this is a people-centric endeavor. BDC is also building a community of practice working collaboratively to solve technical and scientific challenges.</p><p><a href="https://biodatacatalyst.nhlbi.nih.gov/">https://biodatacatalyst.nhlbi.nih.gov/</a></p><p>Explore hosted data on <em>BDC Powered by PIC-SURE</em>: <a href="https://picsure.biodatacatalyst.nhlbi.nih.gov/">https://picsure.biodatacatalyst.nhlbi.nih.gov/</a> </p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/access-data/restricted-data-access">
              Restricted Data Access
            </a>
</li>
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/current">
              Current
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/scalable-collaborative-infrastructure-learning-healthcare-system-scilhs-phase-ii -->
<h1>Scalable Collaborative Infrastructure for a Learning Healthcare System (SCILHS) - Phase II | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/scalable-collaborative-infrastructure-learning-healthcare-system-scilhs-phase-ii'>avillach-lab.hms.harvard.edu/scalable-collaborative-infrastructure-learning-healthcare-system-scilhs-phase-ii</a></div>
<main aria-label="Scalable Collaborative Infrastructure for a Learning Healthcare System (SCILHS) - Phase II" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Scalable Collaborative Infrastructure for a Learning Healthcare System (SCILHS) - Phase II</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	PCOR - CDRN</p><p>	Co-Investigator (PI: Ken Mandl)</p><p>	Leveraging an informatics infrastructure that our investigators have developed over the past 15 years, the Department of Biomedical Informatics (DBMI) at Harvard Medical School will work across 10 healthcare centers throughout the U.S. to develop the Scalable Collaborative Infrastructure for a Learning Health System (SCILHS, pronounced "skills"). An open-source platform, the system will cover more than 8 million patients and enable clinician and patient participation in research. </p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/pcori-pprn-phelan-mcdermid-syndrome-data-network-pmsdn-phase-ii -->
<h1>PCORI PPRN - Phelan McDermid Syndrome Data Network (PMS_DN) Phase II | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/pcori-pprn-phelan-mcdermid-syndrome-data-network-pmsdn-phase-ii'>avillach-lab.hms.harvard.edu/pcori-pprn-phelan-mcdermid-syndrome-data-network-pmsdn-phase-ii</a></div>
<main aria-label="PCORI PPRN - Phelan McDermid Syndrome Data Network (PMS_DN) Phase II" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>PCORI PPRN - Phelan McDermid Syndrome Data Network (PMS_DN) Phase II</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>Co-PI ($1.2M) Total $1.6M (PI: Megan O'Boyle from <a href="http://22q13.org/j15/">PMSF</a>)</p><p><a href="http://www.pcori.org/research-results/2013/phelan-mcdermid-syndrome-data-network">Phelan McDermid Syndrome Data Network (PMS_DN): A PCORI Patient Powered Research Network</a></p><p>The PMS_DN mission is to advance knowledge, care, and treatment of Phelan-McDermid Syndrome (PMS), and related conditions by integrating diverse, complex datasets from multiple sources into a richly populated, high quality, and accessible centralized database to facilitate patient-centered research. The network promotes a culture of transparency and authentic engagement and leadership of families in network vision, governance, and generation of research priorities. The organizations participating in the PMS_DN include: The Phelan McDermid Syndrome Foundation (PMSF), the Department of Biomedical Informatics at Harvard Medical School (HMS), and Boston Children's Hospital</p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/nhlbinih-data-fusion-self-scaling-open-source-registry-advancing-pediatric-pulmonary-vascular -->
<h1>NHLBI/NIH Data Fusion - A Self-Scaling, Open Source Registry Advancing Pediatric Pulmonary Vascular Disease Research | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/nhlbinih-data-fusion-self-scaling-open-source-registry-advancing-pediatric-pulmonary-vascular'>avillach-lab.hms.harvard.edu/nhlbinih-data-fusion-self-scaling-open-source-registry-advancing-pediatric-pulmonary-vascular</a></div>
<main aria-label="NHLBI/NIH Data Fusion - A Self-Scaling, Open Source Registry Advancing Pediatric Pulmonary Vascular Disease Research" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>NHLBI/NIH Data Fusion - A Self-Scaling, Open Source Registry Advancing Pediatric Pulmonary Vascular Disease Research</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>3U01HL121518-02S1</p><p>Co-Investigator (PI: Mandl/Abman)</p><p>We will enhance a registry of children with pulmonary hypertension by improving the quality of data on medications and adverse events. Our approach leverages uniquely valuable data from pharmacies as well as computerized reading and interpretation of free text clinician notes. </p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/nihncats-global-rare-diseases-patient-registry-data-repository-grdr%C2%AE -->
<h1>NIH/NCATS Global Rare Diseases Patient Registry Data Repository (GRDR®) | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/nihncats-global-rare-diseases-patient-registry-data-repository-grdr%C2%AE'>avillach-lab.hms.harvard.edu/nihncats-global-rare-diseases-patient-registry-data-repository-grdr%C2%AE</a></div>
<main aria-label="NIH/NCATS Global Rare Diseases Patient Registry Data Repository (GRDR®)" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>NIH/NCATS Global Rare Diseases Patient Registry Data Repository (GRDR®)</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>This is a web-based resource that aggregates, secures, and stores deidentified patient information from 12 different registries for rare diseases (5,277 patients with 178 different rare dieseases) all in one place. This platform allows for the wider dissemination of data collected by individual rare disease registries as well as the increased accessibility of data for researchers conducting intra-registry and cross-registry queries in the genesis of hypotheses for potential study. It uses the BD2K PIC-SURE RESTful API international ontologies for semantic interoperability, mappings to UMLS Concept Unique Identifiers, and mappings to 55 different GRDR Common Data Elements.</p><p> </p><p dir="ltr"><strong>10 Minute Demo Video: </strong><a href="https://vimeo.com/151171529">https://vimeo.com/151171529</a> </p><p dir="ltr"> </p></div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/access-data/restricted-data-access">
              Restricted Data Access
            </a>
</li>
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/pcori-pprn-phelan-mcdermid-syndrome-data-network -->
<h1>PCORI PPRN - Phelan-McDermid Syndrome Data Network | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/pcori-pprn-phelan-mcdermid-syndrome-data-network'>avillach-lab.hms.harvard.edu/pcori-pprn-phelan-mcdermid-syndrome-data-network</a></div>
<main aria-label="PCORI PPRN - Phelan-McDermid Syndrome Data Network" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>PCORI PPRN - Phelan-McDermid Syndrome Data Network</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>Co-PI ($624K)  Total $950K   (PI  Megan O'Boyle from <a href="http://22q13.org/j15/">PMSF</a> )<br/>PCORI - PPRN <a href="http://www.pcori.org/research-results/2013/phelan-mcdermid-syndrome-data-network">http://www.pcori.org/research-results/2013/phelan-mcdermid-syndrome-data-network</a></p><p><span> The PMS_DN mission is to advance knowledge, care, and treatment of Phelan-McDermid Syndrome (PMS) and related conditions by integrating diverse, complex data sources into a richly populated, high quality, and accessible centralized database to facilitate patient-centered research. The network promotes a culture of transparency and authentic engagement and leadership of families in network vision, governance, and generation of research priorities. The organizations participating in the PMS_DN include: The Phelan-McDermid Syndrome Foundation, the Harvard Medical School (HMS) Department of Biomedical Informatics, and Boston Children’s Hospital.</span><br/> </p><p><span>PMS, a rare genetic condition caused by errors on the end of chromosome 22 (22q13), is associated with autism/autism spectrum disorders, intellectual disabilities, sleep disorders, and seizures. There is no treatment for PMS. There are ~1,100 </span><em><span>diagnosed cases</span></em><span> worldwide, with at least 520 cases confirmed in the US. However, a conservative estimate, regardless if known/diagnosed, is 8,898 US cases. Since PMS_DN’s launch in July 2014, 281 families have consented to participate in the network and PCORnet studies (~50% of known US diagnosed cases). The PMS_DN’s enrollment target for Phase II is 400 US cases (~75% of known US diagnosed cases). PCORnet offers a vital opportunity to advance knowledge and care for PMS, capitalizing on the consortium to enroll families in observational studies and pragmatic clinical trials, while identifying undiagnosed individuals and connecting families to needed services. The PMS_DN brings unique assets to PCORnet: an established data portal for collecting patient-reported outcomes (PROs), the PMS International Registry (PMSIR), and i2b2/tranSMART, a platform that allows researchers to analyze complex data types harmonized to the PCORnet Common Data Model (CDM) v2 and enables integration of PROs with validated knowledge extracted from EHR clinical notes through Natural Language Processing (NLP).</span></p><p><span>The PMS_DN integrates PROs with data from the application of NLP to EHR clinical notes submitted by families. The technical platform to achieve this integration and harmonize data domains with the current version of the PCORnet CDM is i2b2/tranSMART. The PMS_DN completed a mapping file for PRO data that has been validated with real patient data and production of an audit trail of the clinical data integration pipeline in tranSMART. An advantage of i2b2/tranSMART is that an investigator can quickly validate outcomes of interest from the output of the NLP with the i2b2/tranSMART interface, viewing all anonymized sentences from which the concept was generated. The network uses cTAKES including a new temporal module that extracts time and event concepts from clinical notes, enabling a complete longitudinal view of a patient. The PMS_DN maps to several ontologies/terminologies: SNOMED CT, ICD9, ICD10, Human Phenotype Ontology, Gene Ontology, and MeSH.</span></p><div class="hwp-media hwp-media--full-width">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <img alt="capture_decran_2014-05-06_21.04.15.png" height="664" loading="lazy" src="/sites/g/files/omnuum4001/files/avillach_lab/files/capture_decran_2014-05-06_21.04.15_0.png" width="1000"/>
</div>
</div>
</div>
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="page-tags">
<span class="hwp-mb-16 hwp-block">See also:</span>
<ul class="hwp-news-footer__tags hwp-flex hwp-flex-wrap hwp-gap-16 hwp-mb-24">
<li class="max-sm:hwp-w-full">
<a class="hwp-button-tag hwp-button-tag--dark" href="/research/past">
              Past
            </a>
</li>
</ul>
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/people/pi -->
<h1>Principal Investigator | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/people/pi'>avillach-lab.hms.harvard.edu/people/pi</a></div>
<main lang="en" role="main">
<a id="main-content" tabindex="-1"></a>
<div class="layout-content">
<div class="taxonomy-page">
<div class="hwp-bg-dark-base" lang="en" role="main">
<div class="hwp-container hwp-container-px hwp-pt-32">
<div class="hwp-section-heading" data-component-name="section-heading">
<div class="hwp-section-heading__container hwp-container">
<h1 class="hwp-section-heading__title h1">
      Principal Investigator
    </h1>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32">
<div class="taxonomy-page__results">
<div class="views-element-container">
<div class="js-view-dom-id-502345edca19d9bf4a976c8eb6d0db4fc550177412c056fb440404655b97b012">
<h2 class="hwp-section-heading__title h2">Content tagged with Principal Investigator</h2>
<div class="taxonomy-page__taxonomy-header">
<div class="hwp-hidden lg:hwp-block taxonomy-page__results-summary">
        Not finding what you're looking for? Try using <a href="/search?search=Principal Investigator">Advanced Search</a>.
      </div>
<form accept-charset="UTF-8" action="/people/pi" class="views-exposed-form bef-exposed-form" data-bef-auto-submit="" data-bef-auto-submit-delay="500" data-bef-auto-submit-full-form="" data-bef-auto-submit-minimum-length="3" data-drupal-selector="views-exposed-form-hwp-taxonomy-term-term-page-block" id="views-exposed-form-hwp-taxonomy-term-term-page-block" method="get">
<div class="js-form-item form-item hwp-form-item js-form-type-select hwp-form-type-select form-item-sort-bef-combine js-form-item-sort-bef-combine">
<label for="edit-sort-bef-combine">Sort by:</label>
<select class="form-select" data-drupal-selector="edit-sort-bef-combine" id="edit-sort-bef-combine" name="sort_bef_combine"><option selected="selected" value="created_DESC">Date descending</option><option value="created_ASC">Date ascending</option><option value="title_ASC">Alphabetical A-Z</option><option value="title_DESC">Alphabetical Z-A</option></select>
<span aria-hidden="true" class="material-icon">sort</span>
</div>
<div class="form-actions js-form-wrapper form-wrapper" data-drupal-selector="edit-actions" id="edit-actions"><input class="js-hide button js-form-submit form-submit" data-bef-auto-submit-click="" data-drupal-selector="edit-submit-hwp-taxonomy-term" id="edit-submit-hwp-taxonomy-term" type="submit" value="Apply"/>
</div>
</form>
</div>
<div class="lg:hwp-hidden taxonomy-page__results-summary">
    Not finding what you're looking for? Try using <a href="/search?search=Principal Investigator">Advanced Search</a>.
  </div>
<div class="taxonomy-page__result-items taxonomy-page__result-items--">
<div class="views-row">
<article class="page-card page-card--vertical page-card--hwp-person page-card--hwp-search-result hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/paul-avillach-md-phd"> <h3 class="page-card__heading hwp-order-2"><span>Paul Avillach, MD, PhD</span>
</h3>
</a>
<div class="page-card__eyebrow hwp-order-1">Person</div>
<div class="page-card__description hwp-order-3 hwp-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"> <p>Dr. Avillach is Assistant Professor in Pediatrics &amp; Biomedical Informatics, Harvard Medical School and Boston Children’s Hospital. He received his MD in Public Health from University of Bordeaux, France and PhD in Biomedical Informatics from University of...</p>
</div>
</div>
</div>
</article>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/people -->
<h1>People | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/people'>avillach-lab.hms.harvard.edu/people</a></div>
<main aria-label="People" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="search-page search-page--no-filters hwp-listing-page hwp-hwp_person">
<div class="hwp-page-title hwp-bg-dark-base" lang="en">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      People
    </h1>
</div>
</div>
<div class="hwp-section-heading" data-component-name="section-heading">
<div class="hwp-section-heading__container hwp-container-px hwp-container">
<div class="hwp-section-heading__description hwp-text-block">
</div>
</div>
</div>
<div class="search-page__columns hwp-container hwp-container-px">
<div class="search-page__results">
<div class="search-page__search-form">
<form accept-charset="UTF-8" action="/people" class="hwp-search-form-listing" data-drupal-selector="hwp-search-form-listing" id="hwp-search-form-listing" method="post">
<div class="hwp-form-type-search hwp-form-item js-form-item form-item form-type-search js-form-type-search form-item-search js-form-item-search form-no-label">
<label class="visually-hidden" for="edit-search">Search Within Results</label>
<input class="form-search" data-drupal-selector="edit-search" id="edit-search" maxlength="128" name="search" size="60" type="search" value=""/>
</div>
<button class="hwp-button hwp-button--small button js-form-submit form-submit" data-drupal-selector="edit-submit" data-twig-suggestion="header_site_search_submit" id="edit-submit" name="op" type="submit" value="Search Within Results">
<span class="hwp-button__text">Search Within Results</span>
<span aria-hidden="true" class="material-icon">search</span>
</button>
<input autocomplete="off" data-drupal-selector="form-abkaq5baekb6tdbbbcdykv6tnrzar4vba4igsx0mywy" name="form_build_id" type="hidden" value="form-ABKaQ5BaEKB6TdbbBCDyKV6tNrZAR4Vba4iGSX0MYwY"/>
<input data-drupal-selector="edit-hwp-search-form-listing" name="form_id" type="hidden" value="hwp_search_form_listing"/>
</form>
</div>
<div class="views-element-container">
<div class="js-view-dom-id-7831d88ec955941044d8b8e247b05e7655844b18928bd0066f3e13dd295747b1">
<div class="search-page__search-header">
<div class="hwp-hidden lg:hwp-block search-page__results-summary">
        81 results 
      </div>
<button aria-expanded="false" class="search-page__filters-trigger" data-filters-trigger="">
<span>Show filters</span>
<span aria-hidden="true" class="material-icon">filter_alt</span>
</button>
<form accept-charset="UTF-8" action="/node/1774796" class="views-exposed-form bef-exposed-form" data-bef-auto-submit="" data-bef-auto-submit-delay="500" data-bef-auto-submit-full-form="" data-bef-auto-submit-minimum-length="3" data-drupal-selector="views-exposed-form-hwp-search-search-page" id="views-exposed-form-hwp-search-search-page" method="get">
<input data-drupal-selector="edit-search" name="search" type="hidden" value=""/>
<div class="js-form-item form-item hwp-form-item js-form-type-select hwp-form-type-select form-item-sort-bef-combine js-form-item-sort-bef-combine">
<label for="edit-sort-bef-combine">Sort by</label>
<select class="form-select" data-drupal-selector="edit-sort-bef-combine" id="edit-sort-bef-combine" name="sort_bef_combine"><option selected="selected" value="field_hwp_person_name.family--asc">Last Name A-Z</option><option value="field_hwp_person_name.family--desc">Last Name Z-A</option></select>
<span aria-hidden="true" class="material-icon">sort</span>
</div>
<div id="edit-list-type"><div class="hwp-form-type-radio hwp-form-item js-form-item form-item form-type-radio js-form-type-radio form-item-list-type js-form-item-list-type">
<input checked="checked" class="form-radio" data-drupal-selector="edit-list-type-list" id="edit-list-type-list" name="list_type" type="radio" value="list"/>
<label aria-label="List" class="option" for="edit-list-type-list">
<span aria-hidden="true" class="material-icon">list</span>
</label>
</div>
<div class="hwp-form-type-radio hwp-form-item js-form-item form-item form-type-radio js-form-type-radio form-item-list-type js-form-item-list-type">
<input class="form-radio" data-drupal-selector="edit-list-type-grid" id="edit-list-type-grid" name="list_type" type="radio" value="grid"/>
<label aria-label="Grid" class="option" for="edit-list-type-grid">
<span aria-hidden="true" class="material-icon">grid_view</span>
</label>
</div>
</div>
<div class="form-actions js-form-wrapper form-wrapper" data-drupal-selector="edit-actions" id="edit-actions"><input class="js-hide button js-form-submit form-submit" data-bef-auto-submit-click="" data-drupal-selector="edit-submit-hwp-search" id="edit-submit-hwp-search" type="submit" value="Apply"/>
</div>
</form>
</div>
<div class="lg:hwp-hidden search-page__results-summary">
    81 results 
  </div>
<div class="search-page__result-items search-page__result-items--list">
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/andre-rosa"> <h3 class="page-card__heading hwp-order-2">Andre Rosa</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:andre_rosa@hms.harvard.edu">andre_rosa@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior Java Web Developer</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/headshot-cleaned.jpg?itok=2PjFEe9k 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/headshot-cleaned.jpg?itok=dgS0x1JY 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/headshot-cleaned.jpg?itok=vaCHFmDU 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/headshot-cleaned.jpg?itok=8Je78izZ 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/headshot-cleaned.jpg?itok=2PjFEe9k 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/headshot-cleaned.jpg?itok=dgS0x1JY 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/headshot-cleaned.jpg?itok=vaCHFmDU 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/headshot-cleaned.jpg?itok=8Je78izZ 2x" type="image/jpeg" width="480"/>
<img alt="Andre Rosa" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/headshot-cleaned.jpg?itok=2PjFEe9k" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/ambia-ahmed"> <h3 class="page-card__heading hwp-order-2">Ambia Ahmed</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior Quality Assurance Engineer</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/1690403079698.jpg?itok=TiJBwf0a 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/1690403079698.jpg?itok=kaONz3Vc 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/1690403079698.jpg?itok=QlGUEd2y 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/1690403079698.jpg?itok=p0UPH6Nn 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/1690403079698.jpg?itok=TiJBwf0a 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/1690403079698.jpg?itok=kaONz3Vc 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/1690403079698.jpg?itok=QlGUEd2y 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/1690403079698.jpg?itok=p0UPH6Nn 2x" type="image/jpeg" width="480"/>
<img alt="Ambia Ahmed" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/1690403079698.jpg?itok=TiJBwf0a" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/yuri-ahuja"> <h3 class="page-card__heading hwp-order-2">Yuri Ahuja</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:yuri_ahuja@hms.harvard.edu">yuri_ahuja@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>MD/PhD Candidate at Harvard Medical School</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/pic.jpg?itok=MCgZEcNO 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/pic.jpg?itok=EP1ZbJrl 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/pic.jpg?itok=ILjdT9dp 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/pic.jpg?itok=UCV3BFBB 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/pic.jpg?itok=MCgZEcNO 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/pic.jpg?itok=EP1ZbJrl 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/pic.jpg?itok=ILjdT9dp 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/pic.jpg?itok=UCV3BFBB 2x" type="image/jpeg" width="480"/>
<img alt="pic.jpg" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/pic.jpg?itok=MCgZEcNO" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/jumanah-alshenaifi-md"> <h3 class="page-card__heading hwp-order-2">Jumanah Alshenaifi, MD</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Student - MBI Biomedical Informatics, DBMI, Harvard Medical School</div>
</div>
</div>
<div class="page-card__description hwp-order-3 hwp-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden">  MD working on genomic data analysis of large scale genotype-phenotype assocations for autism spectrum disorders

</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/photo_03.jpg?itok=uSVPMzEt 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/photo_03.jpg?itok=L_G8LCwB 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/photo_03.jpg?itok=FViT4WGE 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/photo_03.jpg?itok=Xf6_o8gV 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/photo_03.jpg?itok=uSVPMzEt 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/photo_03.jpg?itok=L_G8LCwB 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/photo_03.jpg?itok=FViT4WGE 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/photo_03.jpg?itok=Xf6_o8gV 2x" type="image/jpeg" width="480"/>
<img alt="Jumanah" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/photo_03.jpg?itok=uSVPMzEt" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/ryan-amari"> <h3 class="page-card__heading hwp-order-2">Ryan Amari</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior Full Stack Software Engineer</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/screen_shot_2021-09-13_at_3.12.45_pm.png?itok=72YrFmhk 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/screen_shot_2021-09-13_at_3.12.45_pm.png?itok=Np1nK8v9 2x" type="image/png" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/screen_shot_2021-09-13_at_3.12.45_pm.png?itok=XzRQcUMD 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/screen_shot_2021-09-13_at_3.12.45_pm.png?itok=bLz3IIXd 2x" type="image/png" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/screen_shot_2021-09-13_at_3.12.45_pm.png?itok=72YrFmhk 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/screen_shot_2021-09-13_at_3.12.45_pm.png?itok=Np1nK8v9 2x" type="image/png" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/screen_shot_2021-09-13_at_3.12.45_pm.png?itok=XzRQcUMD 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/screen_shot_2021-09-13_at_3.12.45_pm.png?itok=bLz3IIXd 2x" type="image/png" width="480"/>
<img alt="ryan amari" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/screen_shot_2021-09-13_at_3.12.45_pm.png?itok=72YrFmhk" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/norah-anthony-0"> <h3 class="page-card__heading hwp-order-2">Norah Anthony</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:norah_anthony@hms.harvard.edu">norah_anthony@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Visiting Research Associate</div>
</div>
</div>
<div class="page-card__description hwp-order-3 hwp-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden">  MD Resident in Public Health, MSc in methodology and statistics in biomedical research. Working on the FAIR (findable, accessible, interoperable, reusable) principles

</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/photo_02.jpg?itok=kmFWzbkB 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/photo_02.jpg?itok=DoyQTf5h 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/photo_02.jpg?itok=BM1lfnqN 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/photo_02.jpg?itok=QzFXBKac 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/photo_02.jpg?itok=kmFWzbkB 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/photo_02.jpg?itok=DoyQTf5h 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/photo_02.jpg?itok=BM1lfnqN 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/photo_02.jpg?itok=QzFXBKac 2x" type="image/jpeg" width="480"/>
<img alt="na" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/photo_02.jpg?itok=kmFWzbkB" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/paul-avillach-md-phd"> <h3 class="page-card__heading hwp-order-2">Paul Avillach, MD, PhD</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:paul_avillach@hms.harvard.edu">paul_avillach@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Assistant Professor at Harvard Medical School</div>
</div>
</div>
<div class="page-card__description hwp-order-3 hwp-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden">  Dr. Avillach is Assistant Professor in Pediatrics &amp; Biomedical Informatics, Harvard Medical School and Boston Children’s Hospital. He received his MD in Public Health from University of Bordeaux, France and PhD in Biomedical Informatics from University of...

</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/paul_avillach_h.jpg?itok=q5VtKch5 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/paul_avillach_h.jpg?itok=ZU8ONmSp 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/paul_avillach_h.jpg?itok=DT9lYtlP 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/paul_avillach_h.jpg?itok=ggQZ85Au 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/paul_avillach_h.jpg?itok=q5VtKch5 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/paul_avillach_h.jpg?itok=ZU8ONmSp 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/paul_avillach_h.jpg?itok=DT9lYtlP 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/paul_avillach_h.jpg?itok=ggQZ85Au 2x" type="image/jpeg" width="480"/>
<img alt="Dr. Paul Avillach" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/paul_avillach_h.jpg?itok=q5VtKch5" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/burt-beckwith"> <h3 class="page-card__heading hwp-order-2">Burt Beckwith</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior Full Stack Web Developer</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/burt.beckwith.jpg?itok=S2zrKVIY 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/burt.beckwith.jpg?itok=aQQezQCs 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/burt.beckwith.jpg?itok=FkggTWre 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/burt.beckwith.jpg?itok=zZtLDdvU 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/burt.beckwith.jpg?itok=S2zrKVIY 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/burt.beckwith.jpg?itok=aQQezQCs 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/burt.beckwith.jpg?itok=FkggTWre 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/burt.beckwith.jpg?itok=zZtLDdvU 2x" type="image/jpeg" width="480"/>
<img alt="burt_beckwith" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/burt.beckwith.jpg?itok=S2zrKVIY" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/nick-benik"> <h3 class="page-card__heading hwp-order-2">Nick  Benik</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:nick_benik@hms.harvard.edu">nick_benik@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior Software Engineer</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/nickb.jpeg?itok=ubjuBxoM 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/nickb.jpeg?itok=xYGNngpg 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/nickb.jpeg?itok=yeTO8iUU 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/nickb.jpeg?itok=7gryldff 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/nickb.jpeg?itok=ubjuBxoM 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/nickb.jpeg?itok=xYGNngpg 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/nickb.jpeg?itok=yeTO8iUU 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/nickb.jpeg?itok=7gryldff 2x" type="image/jpeg" width="480"/>
<img alt="Nick Benik" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/nickb.jpeg?itok=ubjuBxoM" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/arabdha-biswas"> <h3 class="page-card__heading hwp-order-2">Arabdha Biswas</h3>
</a>
<div class="page-card__subheading hwp-order-2"></div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/dsc0334_1.jpg?itok=osH2gWk0 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/dsc0334_1.jpg?itok=a5h7kPDv 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/dsc0334_1.jpg?itok=I0QevyAJ 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/dsc0334_1.jpg?itok=u9wpq2BX 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/dsc0334_1.jpg?itok=osH2gWk0 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/dsc0334_1.jpg?itok=a5h7kPDv 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/dsc0334_1.jpg?itok=I0QevyAJ 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/dsc0334_1.jpg?itok=u9wpq2BX 2x" type="image/jpeg" width="480"/>
<img alt="Arabdha Biswas" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/dsc0334_1.jpg?itok=osH2gWk0" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/arabdha-biswas-0"> <h3 class="page-card__heading hwp-order-2">Arabdha Biswas</h3>
</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Summer Institute in Biomedical Informatics Intern</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/dsc0334_1_01.jpg?itok=ZHKGImFH 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/dsc0334_1_01.jpg?itok=lEcitwV1 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/dsc0334_1_01.jpg?itok=CZje8kIN 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/dsc0334_1_01.jpg?itok=HWdrN0ju 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/dsc0334_1_01.jpg?itok=ZHKGImFH 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/dsc0334_1_01.jpg?itok=lEcitwV1 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/dsc0334_1_01.jpg?itok=CZje8kIN 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/dsc0334_1_01.jpg?itok=HWdrN0ju 2x" type="image/jpeg" width="480"/>
<img alt="Arabdha Biswas" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/dsc0334_1_01.jpg?itok=ZHKGImFH" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
<article class="page-card page-card--horizontal page-card--hwp-person page-card--teaser hwp-bg-light-base" data-component-name="page-card" lang="en">
<div class="page-card__text hwp-order-1">
<a class="page-card__link hwp-order-2" href="/people/jessica-carpenter-lyons"> <h3 class="page-card__heading hwp-order-2">Jessica  Carpenter Lyons</h3>
</a>
<a class="page-card__email hwp-order-3" href="mailto:jessica_lyons@hms.harvard.edu">jessica_lyons@hms.harvard.edu</a>
<div class="page-card__subheading hwp-order-2">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior Project Manager</div>
</div>
</div>
</div>
<div class="page-card__image page-card__image--4-5">
<div class="page-card__thumb">
<div class="hwp-media hwp-media--4-5-person-list-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="865" media="all and (min-width: 1024px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=xv1SCPBa 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=pKzi9Bp- 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 768px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=37BK6FiK 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=2CGF93KE 2x" type="image/jpeg" width="480"/>
<source height="865" media="all and (min-width: 480px)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=xv1SCPBa 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__1380x1730/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=pKzi9Bp- 2x" type="image/jpeg" width="690"/>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=37BK6FiK 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=2CGF93KE 2x" type="image/jpeg" width="480"/>
<img alt="Jessica Lyons" height="865" loading="eager" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__690x865/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=xv1SCPBa" width="690"/>
</picture>
</div>
</div>
</div>
</div>
</article>
</div>
<nav aria-label="Pagination" class="hwp-pager hwp-pager--theme-light hwp-bg-light-core" role="navigation">
<ul class="hwp-pager__items js-pager__items">
<li>
<div class="hwp-pager__item hwp-pager__item--previous hwp-pager__item--not-active" href="" rel="prev" title="Go to previous page">
<span class="hwp-visually-hidden">Previous page</span>
<span aria-hidden="true" class="material-icon material-icon--chevron-left">chevron_left</span>
</div>
</li>
<li class="hwp-pager__item hwp-pager__item--is-active">
<a aria-current="page" href="?page=0" title="Current page">1</a>
</li>
<li class="hwp-pager__item">
<a href="?page=1" title="Go to page 2">2</a>
</li>
<li class="hwp-pager__item">
<a href="?page=2" title="Go to page 3">3</a>
</li>
<li class="hwp-pager__item hwp-pager__item--last hwp-pager__item--ellipsis">
<a href="?page=6" title="Go to last page">
<span class="hwp-visually-hidden">Last page</span>
<span aria-hidden="true">7</span>
</a>
</li>
<li>
<a class="hwp-pager__item hwp-pager__item--next" href="?page=1" rel="prev" title="Go to next page">
<span class="hwp-visually-hidden">Next page</span>
<span aria-hidden="true" class="material-icon material-icon--chevron-right">chevron_right</span>
</a>
</li>
</ul>
</nav>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/numbers-polypharmacy-outpatient-autism-care-pandemic-behaviors -->
<h1>By the Numbers: Polypharmacy, outpatient autism care, pandemic behaviors | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/numbers-polypharmacy-outpatient-autism-care-pandemic-behaviors'>avillach-lab.hms.harvard.edu/numbers-polypharmacy-outpatient-autism-care-pandemic-behaviors</a></div>
<main aria-label="By the Numbers: Polypharmacy, outpatient autism care, pandemic behaviors" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>By the Numbers: Polypharmacy, outpatient autism care, pandemic behaviors</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><a href="https://www.spectrumnews.org/news/by-the-numbers-polypharmacy-outpatient-autism-care-pandemic-behaviors/">https://www.spectrumnews.org/news/by-the-numbers-polypharmacy-outpatien…</a></div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/creating-clinical-and-omics-information-commons-using-i2b2-and-transmart -->
<h1>2013-12 KEYNOTE - Institut Curie - Paris France - Creating clinical and 'omics information commons using i2b2 and tranSMART | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/creating-clinical-and-omics-information-commons-using-i2b2-and-transmart'>avillach-lab.hms.harvard.edu/creating-clinical-and-omics-information-commons-using-i2b2-and-transmart</a></div>
<main aria-label="2013-12 KEYNOTE - Institut Curie - Paris France - Creating clinical and 'omics information commons using i2b2 and tranSMART " aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>2013-12 KEYNOTE - Institut Curie - Paris France - Creating clinical and 'omics information commons using i2b2 and tranSMART </span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="hwp-attachments" data-component-name="attachments">
<hr class="hwp-text-button-light-secondary hwp-my-32"/>
<span>Attachments</span>
<ul aria-label="Attachments" class="lg:hwp-grid hwp-attachments__items">
<li>
<a class="hwp-icon-link hwp-icon-link--x-large hwp-icon-link--has-bg hwp-icon-link--dark-primary analytics-cta" href="/sites/g/files/omnuum4001/files/avillach_lab/files/avillach_i2b2_transmart_dec_5th_2013_curie.pdf">
<span class="hwp-icon-link__icon">
<span aria-hidden="true" class="material-icon">picture_as_pdf</span>
</span>
<span class="hwp-icon-link__text">avillach_i2b2_transmart_dec_5th_2013_curie.pdf</span>
</a>
</li>
</ul>
<hr class="hwp-text-button-light-secondary hwp-my-24"/>
</div>
<div class="page-tags">
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/toward-precision-medicine-building-patient-centric-information-commons-common-and-rare-0 -->
<h1>2015-03 PRESENTATION - "Toward Precision Medicine: Building a Patient -centric Information Commons on Common and Rare Diseases with i2b2/tranSMART” Application to Autism and Phelan McDermid Syndrome | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/toward-precision-medicine-building-patient-centric-information-commons-common-and-rare-0'>avillach-lab.hms.harvard.edu/toward-precision-medicine-building-patient-centric-information-commons-common-and-rare-0</a></div>
<main aria-label='2015-03 PRESENTATION - "Toward Precision Medicine: Building a Patient -centric Information Commons on Common and Rare Diseases with i2b2/tranSMART” Application to Autism and Phelan McDermid Syndrome' aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>2015-03 PRESENTATION - "Toward Precision Medicine: Building a Patient -centric Information Commons on Common and Rare Diseases with i2b2/tranSMART” Application to Autism and Phelan McDermid Syndrome</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>Paul Avillach, MD, PhD will present at the HMS Catalyst symposium: ReSourcing Big Data March 23rd 2015</p><p><a href="http://catalyst.harvard.edu/programs/reactor/">http://catalyst.harvard.edu/programs/reactor/</a><br/><a href="http://catalyst.harvard.edu/pdf/reactor/Big%20Data%20Event_List%20of%20confirmed%20Speakers.pdf">http://catalyst.harvard.edu/pdf/reactor/Big%20Data%20Event_List%20of%20confirmed%20Speakers.pdf</a></p></div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/about-30-people-autism-take-multiple-medications-manage-comorbid-conditions-0 -->
<h1>About 30% Of People With Autism Take Multiple Medications To Manage Comorbid Conditions | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/about-30-people-autism-take-multiple-medications-manage-comorbid-conditions-0'>avillach-lab.hms.harvard.edu/about-30-people-autism-take-multiple-medications-manage-comorbid-conditions-0</a></div>
<main aria-label="About 30% Of People With Autism Take Multiple Medications To Manage Comorbid Conditions" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>About 30% Of People With Autism Take Multiple Medications To Manage Comorbid Conditions</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><a href="https://openminds.com/market-intelligence/news/about-30-of-people-with-autism-take-multiple-medications-to-manage-comorbid-conditions/">https://openminds.com/market-intelligence/news/about-30-of-people-with-…</a></div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/more-precise-approach-fighting-cancer -->
<h1>A More Precise Approach to Fighting Cancer | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/more-precise-approach-fighting-cancer'>avillach-lab.hms.harvard.edu/more-precise-approach-fighting-cancer</a></div>
<main aria-label="A More Precise Approach to Fighting Cancer" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>A More Precise Approach to Fighting Cancer</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><a href="https://www.health.harvard.edu/cancer/a-more-precise-approach-to-fighting-cancer">https://www.health.harvard.edu/cancer/a-more-precise-approach-to-fighti…</a></div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/press-release-transmart-informatics-platform-be-used-1m-phelan-mcdermid-syndrome -->
<h1>2014-07 PRESS RELEASE - TranSmart Informatics Platform to be Used in $1M Phelan-McDermid Syndrome Research Project | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/press-release-transmart-informatics-platform-be-used-1m-phelan-mcdermid-syndrome'>avillach-lab.hms.harvard.edu/press-release-transmart-informatics-platform-be-used-1m-phelan-mcdermid-syndrome</a></div>
<main aria-label="2014-07 PRESS RELEASE - TranSmart Informatics Platform to be Used in $1M Phelan-McDermid Syndrome Research Project" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>2014-07 PRESS RELEASE - TranSmart Informatics Platform to be Used in $1M Phelan-McDermid Syndrome Research Project</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>NEW YORK (GenomeWeb) – The TranSmart Foundation said today that its data-sharing and -analysis platform will be used by the Phelan-McDermid Syndrome Foundation and the Center of Biomedical Informatics at Harvard Medical School as part of a nearly $1 million contract research project.<br/><br/></p><p><a href="https://www.genomeweb.com/informatics/transmart-informatics-platform-be-used-1m-phelan-mcdermid-syndrome-research-proj">https://www.genomeweb.com/informatics/transmart-informatics-platform-be-used-1m-phelan-mcdermid-syndrome-research-proj</a></p></div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/phenome-wide-association-studies-quantitative-trait-application-tpmt-enzymatic-activity -->
<h1>2014-04 PRESENTATION - Phenome-wide association studies on a quantitative trait: Application to TPMT enzymatic activity | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/phenome-wide-association-studies-quantitative-trait-application-tpmt-enzymatic-activity'>avillach-lab.hms.harvard.edu/phenome-wide-association-studies-quantitative-trait-application-tpmt-enzymatic-activity</a></div>
<main aria-label="2014-04 PRESENTATION - Phenome-wide association studies on a quantitative trait: Application to TPMT enzymatic activity " aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>2014-04 PRESENTATION - Phenome-wide association studies on a quantitative trait: Application to TPMT enzymatic activity </span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-news-footer hwp-py-24 hwp-pt-0 hwp-container hwp-container-px">
<div class="hwp-attachments" data-component-name="attachments">
<hr class="hwp-text-button-light-secondary hwp-my-32"/>
<span>Attachments</span>
<ul aria-label="Attachments" class="lg:hwp-grid hwp-attachments__items">
<li>
<a class="hwp-icon-link hwp-icon-link--x-large hwp-icon-link--has-bg hwp-icon-link--dark-primary analytics-cta" href="/sites/g/files/omnuum4001/files/avillach_lab/files/avillach_phewas_amia_tbi_april_7th.pdf">
<span class="hwp-icon-link__icon">
<span aria-hidden="true" class="material-icon">picture_as_pdf</span>
</span>
<span class="hwp-icon-link__text">avillach_phewas_amia_tbi_april_7th.pdf</span>
</a>
</li>
</ul>
<hr class="hwp-text-button-light-secondary hwp-my-24"/>
</div>
<div class="page-tags">
</div>
</div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/2015-11-keynote-address -->
<h1>2015-11 KEYNOTE - tranSMART Foundation Annual Meeting - Creating Trust in Translational Research Platforms | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/2015-11-keynote-address'>avillach-lab.hms.harvard.edu/2015-11-keynote-address</a></div>
<main aria-label="2015-11 KEYNOTE - tranSMART Foundation Annual Meeting -  Creating Trust in Translational Research Platforms" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>2015-11 KEYNOTE - tranSMART Foundation Annual Meeting -  Creating Trust in Translational Research Platforms</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>Paul Avillach's presentation at tranSMART Foundation Annual Meeting, Amsterdam, Netherlands, November 2015</p><p>YouTube: <a href="https://www.youtube.com/watch?v=Y8Eqsr571C8">https://www.youtube.com/watch?v=Y8Eqsr571C8</a></p><p>Content: <a href="http://lanyrd.com/2015/transmart-foundation-annual-meeting/sdrzqg/">http://lanyrd.com/2015/transmart-foundation-annual-meeting/sdrzqg/</a></p><p></p></div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/phelan-mcdermid-syndrome-foundation-team-approved-funding-award-become-part-new -->
<h1>2014-02 PRESS RELEASE - Phelan-McDermid Syndrome Foundation team approved for funding award to become part of new National Clinical Research Network | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/phelan-mcdermid-syndrome-foundation-team-approved-funding-award-become-part-new'>avillach-lab.hms.harvard.edu/phelan-mcdermid-syndrome-foundation-team-approved-funding-award-become-part-new</a></div>
<main aria-label="2014-02 PRESS RELEASE - Phelan-McDermid Syndrome Foundation team approved for funding award to become part of new National Clinical Research Network" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>2014-02 PRESS RELEASE - Phelan-McDermid Syndrome Foundation team approved for funding award to become part of new National Clinical Research Network</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p><a href="http://22q13.org/j15/index.php?option=com_content&amp;view=article&amp;id=299%3A2014-grant">http://22q13.org/j15/index.php?option=com_content&amp;view=article&amp;id=299%3…</a></p></div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/presentation-creating-patient-centric-information-commons-autism-research -->
<h1>2015-05 PRESENTATION - INSAR - Creating a patient centric information commons for autism research | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/presentation-creating-patient-centric-information-commons-autism-research'>avillach-lab.hms.harvard.edu/presentation-creating-patient-centric-information-commons-autism-research</a></div>
<main aria-label="2015-05 PRESENTATION - INSAR - Creating a patient centric information commons for autism research" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>2015-05 PRESENTATION - INSAR - Creating a patient centric information commons for autism research</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>Megan O’Boyle and Geraldine Bliss - patients advocates from the <a href="http://22q13.org">Phelan McDermid Syndrome fondation</a> - with  Isaac Kohane and Paul Avillach from CBMI are presenting a Special Interest Group on <strong>Creating a patient centric information commons for autism research</strong> at the International Society for Autism Research (INSAR) annual meeting May 13-16  Salt Lake City, Utah, USA</p><p><a href="http://www.autism-insar.org/imfar-annual-meeting/imfar-2015">http://www.autism-insar.org/imfar-annual-meeting/imfar-2015</a></p></div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/2015-09-press-release-dbmi-and-ncats-collaborate-global-rare-diseases-data-repository -->
<h1>2015-09 PRESS RELEASE - DBMI and NCATS to Collaborate on Global Rare Diseases Data Repository | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/2015-09-press-release-dbmi-and-ncats-collaborate-global-rare-diseases-data-repository'>avillach-lab.hms.harvard.edu/2015-09-press-release-dbmi-and-ncats-collaborate-global-rare-diseases-data-repository</a></div>
<main aria-label="2015-09 PRESS RELEASE - DBMI and NCATS to Collaborate on Global Rare Diseases Data Repository" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>2015-09 PRESS RELEASE - DBMI and NCATS to Collaborate on Global Rare Diseases Data Repository</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>In August 2015, the National Center for Advancing Translational Sciences (NCATS) awarded DBMI a supplemental grant of $2.4M to further develop the Global Rare Diseases Patient Registry Data Repository/GRDR program. </p><p></p><p><a href="http://dbmi.hms.harvard.edu/news/dbmi-and-ncats-collaborate-global-rare-diseases-data-repository">http://dbmi.hms.harvard.edu/news/dbmi-and-ncats-collaborate-global-rare-diseases-data-repository</a></p><p><a href="http://ncats.nih.gov/grdr">http://ncats.nih.gov/grdr</a></p><p></p><p><br/><br/></p><p></p></div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/toward-precision-medicine-building-patient-centric-information-commons-common-and-rare -->
<h1>2014-11 KEYNOTE - VIDEO - Toward Precision Medicine: Building a patient centric information commons on common and rare diseases with tranSMART | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/toward-precision-medicine-building-patient-centric-information-commons-common-and-rare'>avillach-lab.hms.harvard.edu/toward-precision-medicine-building-patient-centric-information-commons-common-and-rare</a></div>
<main aria-label="2014-11 KEYNOTE - VIDEO - Toward Precision Medicine: Building a patient centric information commons on common and rare diseases with tranSMART" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>2014-11 KEYNOTE - VIDEO - Toward Precision Medicine: Building a patient centric information commons on common and rare diseases with tranSMART</span>
</h1>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/datathonhackathon-september-14-15-2017 -->
<h1>Datathon/Hackathon September 14 - 15, 2017 | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/datathonhackathon-september-14-15-2017'>avillach-lab.hms.harvard.edu/datathonhackathon-september-14-15-2017</a></div>
<main aria-label="Datathon/Hackathon September 14 - 15, 2017" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Datathon/Hackathon September 14 - 15, 2017</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	Register for the Datathon/Hackathon organized jointly by the Harvard Medical School Department of Biomedical Informatics, the Dana Farber Cancer Institute, and Boston Children's Hospital</p><p>	 <a href="https://dbmi.hms.harvard.edu/news/datathon-hackathon-sep-14-15">https://dbmi.hms.harvard.edu/news/datathon-hackathon-sep-14-15</a></p></div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/presentation-technological-tools-open-science-and-collaboration-neuroscience -->
<h1>2015-05 PRESENTATION - Technological tools for Open Science and Collaboration in neuroscience | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/presentation-technological-tools-open-science-and-collaboration-neuroscience'>avillach-lab.hms.harvard.edu/presentation-technological-tools-open-science-and-collaboration-neuroscience</a></div>
<main aria-label="2015-05 PRESENTATION - Technological tools for Open Science and Collaboration in neuroscience" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>2015-05 PRESENTATION - Technological tools for Open Science and Collaboration in neuroscience</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>Paul Avillach, MD, PhD  to present at the 4th  One MIND for research annual Summit « Technological tools for Open Science and Collaboration in neuroscience »</p><p>May 27-29 th, Arlington, VA</p><p><a href="http://onemind.org/">http://onemind.org</a></p></div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/polypharmacy-shifting-prescriptions-common-autism-comorbidities -->
<h1>Polypharmacy, shifting prescriptions common for autism comorbidities | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/polypharmacy-shifting-prescriptions-common-autism-comorbidities'>avillach-lab.hms.harvard.edu/polypharmacy-shifting-prescriptions-common-autism-comorbidities</a></div>
<main aria-label="Polypharmacy, shifting prescriptions common for autism comorbidities" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>Polypharmacy, shifting prescriptions common for autism comorbidities</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><a href="https://www.spectrumnews.org/news/polypharmacy-shifting-prescriptions-common-for-autism-comorbidities/">https://www.spectrumnews.org/news/polypharmacy-shifting-prescriptions-c…</a></div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/presentation-functional-genomics-predictive-medicine -->
<h1>2015-05 PRESENTATION - Functional Genomics & Predictive Medicine | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/presentation-functional-genomics-predictive-medicine'>avillach-lab.hms.harvard.edu/presentation-functional-genomics-predictive-medicine</a></div>
<main aria-label="2015-05 PRESENTATION - Functional Genomics &amp; Predictive Medicine" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>2015-05 PRESENTATION - Functional Genomics &amp; Predictive Medicine</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>Paul Avillach, MD, PhD will present at the Functional genomics &amp; predictive medicine symposium<br/>May 21st 2015</p><p><br/><a href="https://www.gtcbio.com/conferences/functional-genomics-predictive-medicine-speakers">https://www.gtcbio.com/conferences/functional-genomics-predictive-medic…</a></p></div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/2016-10-splunk-revolution-awards -->
<h1>2016-10 Splunk Revolution Awards | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/2016-10-splunk-revolution-awards'>avillach-lab.hms.harvard.edu/2016-10-splunk-revolution-awards</a></div>
<main aria-label="2016-10 Splunk Revolution Awards" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<article class="node node--type-hwp-page node--view-mode-full" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large" id="page-title">
<span>2016-10 Splunk Revolution Awards</span>
</h1>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-w-full hwp-py-32 lg:hwp-py-64">
<div class="hwp-flex hwp-flex-col hwp-flex-1 hwp-overflow-hidden">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>Paul Avillach, MD, PhD - Splunk Innovation Award Winner 2016</p><p>The Splunk Innovation Award recognizes customers who have used the Splunk platform to solve a unique problem within their organizations, uncovered new business opportunities and revenue streams, or developed or integrated a new app into a cross-platform solution.</p><p><a href="http://blogs.splunk.com/2016/10/28/meet-the-2016-splunk-revolution-award-winners/">http://blogs.splunk.com/2016/10/28/meet-the-2016-splunk-revolution-award-winners/</a></p></div>
</div>
</div>
</article>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/people/samantha-pliatt -->
<h1>Samantha Piatt | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/people/samantha-pliatt'>avillach-lab.hms.harvard.edu/people/samantha-pliatt</a></div>
<main aria-label="Samantha Piatt" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="page-content-top" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Samantha Piatt
    </h1>
<div class="hwp-page-header__subtitle">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Research Software Developer II</div>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 lg:hwp-pt-64 lg:hwp-flex lg:hwp-gap-24">
<div class="lg:hwp-w-[19.375rem]">
<div class="hwp-person-card" data-component-name="person-card">
<div class="hwp-person-card__image">
<div class="field field--name-field-hwp-person-headshot field--type-entity-reference field--label-hidden"><div class="hwp-media hwp-media--4-5-small-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/samanthapiatt.jpg?itok=s6_tkhLW 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/samanthapiatt.jpg?itok=HHFU-p4k 2x" type="image/jpeg" width="480"/>
<img alt="Samantha Piatt" height="600" loading="lazy" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/samanthapiatt.jpg?itok=s6_tkhLW" width="480"/>
</picture>
</div>
</div>
</div>
</div>
<div class="hwp-person-card__information hwp-bg-light-base hwp-flex-1"></div>
</div>
</div>
<div class="hwp-flex-1 lg:hwp-px-32">
<div class="hwp-person-text-block">
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px">
</div>
<div class="hwp-taxonomy-group hwp-bg-light-core">
<div class="hwp-container hwp-container-px">
<ul class="hwp-taxonomy-group__grid">
<li class="hwp-taxonomy-group-card hwp-bg-light-base">
<h2 class="hwp-taxonomy-group-card__title text-x-small">People</h2>
<a href="/people/software-engineers" hreflang="en">Software Engineers and Programmers</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/people/luke-sikina -->
<h1>Luke Sikina | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/people/luke-sikina'>avillach-lab.hms.harvard.edu/people/luke-sikina</a></div>
<main aria-label="Luke Sikina" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="page-content-top" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Luke Sikina
    </h1>
<div class="hwp-page-header__subtitle">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior Full Stack Software Engineer </div>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 lg:hwp-pt-64 lg:hwp-flex lg:hwp-gap-24">
<div class="lg:hwp-w-[19.375rem]">
<div class="hwp-person-card" data-component-name="person-card">
<div class="hwp-person-card__image">
<div class="field field--name-field-hwp-person-headshot field--type-entity-reference field--label-hidden"><div class="hwp-media hwp-media--4-5-small-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/2025-03/Option%209.jpg?itok=Mk8TT1F9 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/2025-03/Option%209.jpg?itok=mNbkYRmW 2x" type="image/jpeg" width="480"/>
<img alt="Luke Sikina" height="600" loading="lazy" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/2025-03/Option%209.jpg?itok=Mk8TT1F9" width="480"/>
</picture>
</div>
</div>
</div>
</div>
<div class="hwp-person-card__information hwp-bg-light-base hwp-flex-1"></div>
</div>
</div>
<div class="hwp-flex-1 lg:hwp-px-32">
<div class="hwp-person-text-block">
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px">
</div>
<div class="hwp-taxonomy-group hwp-bg-light-core">
<div class="hwp-container hwp-container-px">
<ul class="hwp-taxonomy-group__grid">
<li class="hwp-taxonomy-group-card hwp-bg-light-base">
<h2 class="hwp-taxonomy-group-card__title text-x-small">People</h2>
<a href="/people/software-engineers" hreflang="en">Software Engineers and Programmers</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/people/andrew-j-guidetti -->
<h1>Andrew J Guidetti | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/people/andrew-j-guidetti'>avillach-lab.hms.harvard.edu/people/andrew-j-guidetti</a></div>
<main aria-label="Andrew J Guidetti" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="page-content-top" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Andrew J Guidetti
    </h1>
<div class="hwp-page-header__subtitle">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior DevOps Engineer</div>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 lg:hwp-pt-64 lg:hwp-flex lg:hwp-gap-24">
<div class="lg:hwp-w-[19.375rem]">
<div class="hwp-person-card" data-component-name="person-card">
<div class="hwp-person-card__image">
<div class="field field--name-field-hwp-person-headshot field--type-entity-reference field--label-hidden"><div class="hwp-media hwp-media--4-5-small-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/2016-02-04_23.53.59.jpg?itok=SJ3smFdZ 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/2016-02-04_23.53.59.jpg?itok=bhxb6rLd 2x" type="image/jpeg" width="480"/>
<img alt="Andrew J. Guidetti" height="600" loading="lazy" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/2016-02-04_23.53.59.jpg?itok=SJ3smFdZ" width="480"/>
</picture>
</div>
</div>
</div>
</div>
<div class="hwp-person-card__information hwp-bg-light-base hwp-flex-1"> <div class="hwp-flex hwp-gap-8 hwp-items-center">
<span aria-hidden="true" class="material-icon">email</span>
<a class="hwp-break-all" href="mailto:andrew_guidetti@hms.harvard.edu">andrew_guidetti@hms.harvard.edu</a>
</div>
</div>
</div>
</div>
<div class="hwp-flex-1 lg:hwp-px-32">
<div class="hwp-person-text-block">
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px">
</div>
<div class="hwp-taxonomy-group hwp-bg-light-core">
<div class="hwp-container hwp-container-px">
<ul class="hwp-taxonomy-group__grid">
<li class="hwp-taxonomy-group-card hwp-bg-light-base">
<h2 class="hwp-taxonomy-group-card__title text-x-small">People</h2>
<a href="/people/former-members" hreflang="en">Alumni</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/people/danielle-pillion -->
<h1>Danielle Pillion | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/people/danielle-pillion'>avillach-lab.hms.harvard.edu/people/danielle-pillion</a></div>
<main aria-label="Danielle Pillion" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="page-content-top" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Danielle Pillion
    </h1>
<div class="hwp-page-header__subtitle">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Software Engineering Manager</div>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 lg:hwp-pt-64 lg:hwp-flex lg:hwp-gap-24">
<div class="lg:hwp-w-[19.375rem]">
<div class="hwp-person-card" data-component-name="person-card">
<div class="hwp-person-card__image">
<div class="field field--name-field-hwp-person-headshot field--type-entity-reference field--label-hidden"><div class="hwp-media hwp-media--4-5-small-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/dpillion_profile_image.jpeg?itok=-FK2Dnv0 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/dpillion_profile_image.jpeg?itok=I3Tuserr 2x" type="image/jpeg" width="480"/>
<img alt="DaniellePillion" height="600" loading="lazy" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/dpillion_profile_image.jpeg?itok=-FK2Dnv0" width="480"/>
</picture>
</div>
</div>
</div>
</div>
<div class="hwp-person-card__information hwp-bg-light-base hwp-flex-1"></div>
</div>
</div>
<div class="hwp-flex-1 lg:hwp-px-32">
<div class="hwp-person-text-block">
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px">
</div>
<div class="hwp-taxonomy-group hwp-bg-light-core">
<div class="hwp-container hwp-container-px">
<ul class="hwp-taxonomy-group__grid">
<li class="hwp-taxonomy-group-card hwp-bg-light-base">
<h2 class="hwp-taxonomy-group-card__title text-x-small">People</h2>
<a href="/people/software-engineers" hreflang="en">Software Engineers and Programmers</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/people/emily-hughes -->
<h1>Emily Hughes, MS | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/people/emily-hughes'>avillach-lab.hms.harvard.edu/people/emily-hughes</a></div>
<main aria-label="Emily Hughes, MS" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="page-content-top" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Emily Hughes, MS
    </h1>
<div class="hwp-page-header__subtitle">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Bioinformatics Systems Product Manager</div>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 lg:hwp-pt-64 lg:hwp-flex lg:hwp-gap-24">
<div class="lg:hwp-w-[19.375rem]">
<div class="hwp-person-card" data-component-name="person-card">
<div class="hwp-person-card__image">
<div class="field field--name-field-hwp-person-headshot field--type-entity-reference field--label-hidden"><div class="hwp-media hwp-media--4-5-small-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/2025-03/0083_Emily_Hughes_001_tiny.jpeg?itok=UGXT2lnD 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/2025-03/0083_Emily_Hughes_001_tiny.jpeg?itok=olmXQqAl 2x" type="image/jpeg" width="480"/>
<img alt="Emily Hughes" height="600" loading="lazy" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/2025-03/0083_Emily_Hughes_001_tiny.jpeg?itok=UGXT2lnD" width="480"/>
</picture>
</div>
</div>
</div>
</div>
<div class="hwp-person-card__information hwp-bg-light-base hwp-flex-1"></div>
</div>
</div>
<div class="hwp-flex-1 lg:hwp-px-32">
<div class="hwp-person-text-block">
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px">
</div>
<div class="hwp-taxonomy-group hwp-bg-light-core">
<div class="hwp-container hwp-container-px">
<ul class="hwp-taxonomy-group__grid">
<li class="hwp-taxonomy-group-card hwp-bg-light-base">
<h2 class="hwp-taxonomy-group-card__title text-x-small">People</h2>
<a href="/people/product-team" hreflang="en">Product &amp; Researcher Engagement Team</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/people/carlos-s%C3%A1ez -->
<h1>Carlos Sáez | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/people/carlos-s%C3%A1ez'>avillach-lab.hms.harvard.edu/people/carlos-s%C3%A1ez</a></div>
<main aria-label="Carlos Sáez" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="page-content-top" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Carlos Sáez
    </h1>
<div class="hwp-page-header__subtitle">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Visiting Research Fellow at Harvard Medical School</div>
<div>Postdoctoral Researcher at Universitat Politècnica de València, Spain</div>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 lg:hwp-pt-64 lg:hwp-flex lg:hwp-gap-24">
<div class="lg:hwp-w-[19.375rem]">
<div class="hwp-person-card" data-component-name="person-card">
<div class="hwp-person-card__image">
<div class="field field--name-field-hwp-person-headshot field--type-entity-reference field--label-hidden"><div class="hwp-media hwp-media--4-5-small-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/img_0275_s2.jpg?itok=kYAk_ZrW 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/img_0275_s2.jpg?itok=xysADbTM 2x" type="image/jpeg" width="480"/>
<img alt="Carlos Sáez" height="600" loading="lazy" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/img_0275_s2.jpg?itok=kYAk_ZrW" width="480"/>
</picture>
</div>
</div>
</div>
</div>
<div class="hwp-person-card__information hwp-bg-light-base hwp-flex-1"> <div class="hwp-flex hwp-gap-8 hwp-items-center">
<span aria-hidden="true" class="material-icon">email</span>
<a class="hwp-break-all" href="mailto:carlos_saez@hms.harvard.edu">carlos_saez@hms.harvard.edu</a>
</div>
<div class="hwp-mb-24 hwp-flex hwp-flex-col hwp-gap-20">
<div class="hwp-flex hwp-gap-8 hwp-items-center">
<span aria-hidden="true" class="material-icon">laptop_windows</span>
<a href="https://scholar.google.es/citations?user=OkBVoyoAAAAJ&amp;sortby=pubdate">Publications in Google Scholar</a>
</div>
</div>
</div>
</div>
</div>
<div class="hwp-flex-1 lg:hwp-px-32">
<div class="hwp-person-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>	PhD in Technologies for Health and Well-Being, MsC in Artificial Intelligence, Pattern Recognition and Digital Imaging. He is researcher at the Biomedical Data Science Lab (ITACA Institute) of the Universitat Politècnica de València (UPV) since 2006. His current research focuses on the characterization of biomedical data variability and in the developemnt of novel biomedical data quality assessment methods. He has participated in several European, National and private projects, has been PI of two National projects. Two of his publications were selected as best of published worldwide by the International Medical Informatics Association in the fields of Secondary Use of Patient Data (2016) and Health Information Systems (2013). At UPV he collaborates teaching Data Quality and Interoperability, Health Information Systems and Bioinformatics.</p><p>	 </p></div>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px">
</div>
<div class="hwp-taxonomy-group hwp-bg-light-core">
<div class="hwp-container hwp-container-px">
<ul class="hwp-taxonomy-group__grid">
<li class="hwp-taxonomy-group-card hwp-bg-light-base">
<h2 class="hwp-taxonomy-group-card__title text-x-small">People</h2>
<a href="/people/former-members" hreflang="en">Alumni</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/people/michael-corace -->
<h1>Michael Corace | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/people/michael-corace'>avillach-lab.hms.harvard.edu/people/michael-corace</a></div>
<main aria-label="Michael Corace" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="page-content-top" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Michael Corace
    </h1>
<div class="hwp-page-header__subtitle">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Bioinformatics Data Scientist</div>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 lg:hwp-pt-64 lg:hwp-flex lg:hwp-gap-24">
<div class="lg:hwp-w-[19.375rem]">
<div class="hwp-person-card" data-component-name="person-card">
<div class="hwp-person-card__image">
<div class="field field--name-field-hwp-person-headshot field--type-entity-reference field--label-hidden"><div class="hwp-media hwp-media--4-5-small-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/image_6.png?itok=o2sjKVN_ 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/image_6.png?itok=SEIUsgBN 2x" type="image/png" width="480"/>
<img alt="Michael Corace" height="600" loading="lazy" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/image_6.png?itok=o2sjKVN_" width="480"/>
</picture>
</div>
</div>
</div>
</div>
<div class="hwp-person-card__information hwp-bg-light-base hwp-flex-1"></div>
</div>
</div>
<div class="hwp-flex-1 lg:hwp-px-32">
<div class="hwp-person-text-block">
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px">
</div>
<div class="hwp-taxonomy-group hwp-bg-light-core">
<div class="hwp-container hwp-container-px">
<ul class="hwp-taxonomy-group__grid">
<li class="hwp-taxonomy-group-card hwp-bg-light-base">
<h2 class="hwp-taxonomy-group-card__title text-x-small">People</h2>
<a href="/people/software-engineers" hreflang="en">Software Engineers and Programmers</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/people/aleksander-nikitin -->
<h1>Aleksandr Nikitin | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/people/aleksander-nikitin'>avillach-lab.hms.harvard.edu/people/aleksander-nikitin</a></div>
<main aria-label="Aleksandr Nikitin" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="page-content-top" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Aleksandr Nikitin
    </h1>
<div class="hwp-page-header__subtitle">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Application Development Specialist</div>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 lg:hwp-pt-64 lg:hwp-flex lg:hwp-gap-24">
<div class="lg:hwp-w-[19.375rem]">
<div class="hwp-person-card" data-component-name="person-card">
<div class="hwp-person-card__image">
<div class="field field--name-field-hwp-person-headshot field--type-entity-reference field--label-hidden"><div class="hwp-media hwp-media--4-5-small-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/20160617_163651_0.jpg?itok=aU-L7w4d 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/20160617_163651_0.jpg?itok=nhp7T8Bg 2x" type="image/jpeg" width="480"/>
<img alt="Aleksandr Nikitin" height="600" loading="lazy" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/20160617_163651_0.jpg?itok=aU-L7w4d" width="480"/>
</picture>
</div>
</div>
</div>
</div>
<div class="hwp-person-card__information hwp-bg-light-base hwp-flex-1"> <div class="hwp-flex hwp-gap-8 hwp-items-start">
<span aria-hidden="true" class="material-icon">location_on</span>
          Boston Children’s Hospital Information Services Department
1295 Boylston Street Boston, MA 02215
        </div>
<div class="hwp-flex hwp-gap-8 hwp-items-center">
<span aria-hidden="true" class="material-icon">email</span>
<a class="hwp-break-all" href="mailto:aleksandr.nikitin@childrens.harvard.edu">aleksandr.nikitin@childrens.harvard.edu</a>
</div>
</div>
</div>
</div>
<div class="hwp-flex-1 lg:hwp-px-32">
<div class="hwp-person-text-block">
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px">
</div>
<div class="hwp-taxonomy-group hwp-bg-light-core">
<div class="hwp-container hwp-container-px">
<ul class="hwp-taxonomy-group__grid">
<li class="hwp-taxonomy-group-card hwp-bg-light-base">
<h2 class="hwp-taxonomy-group-card__title text-x-small">People</h2>
<a href="/people/former-members" hreflang="en">Alumni</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/people/jessica-carpenter-lyons -->
<h1>Jessica Carpenter Lyons | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/people/jessica-carpenter-lyons'>avillach-lab.hms.harvard.edu/people/jessica-carpenter-lyons</a></div>
<main aria-label="Jessica  Carpenter Lyons" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="page-content-top" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Jessica  Carpenter Lyons
    </h1>
<div class="hwp-page-header__subtitle">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior Project Manager</div>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 lg:hwp-pt-64 lg:hwp-flex lg:hwp-gap-24">
<div class="lg:hwp-w-[19.375rem]">
<div class="hwp-person-card" data-component-name="person-card">
<div class="hwp-person-card__image">
<div class="field field--name-field-hwp-person-headshot field--type-entity-reference field--label-hidden"><div class="hwp-media hwp-media--4-5-small-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=37BK6FiK 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=2CGF93KE 2x" type="image/jpeg" width="480"/>
<img alt="Jessica Lyons" height="600" loading="lazy" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/jlyons_headshot_002_0.jpg?itok=37BK6FiK" width="480"/>
</picture>
</div>
</div>
</div>
</div>
<div class="hwp-person-card__information hwp-bg-light-base hwp-flex-1"> <div class="hwp-flex hwp-gap-8 hwp-items-center">
<span aria-hidden="true" class="material-icon">email</span>
<a class="hwp-break-all" href="mailto:jessica_lyons@hms.harvard.edu">jessica_lyons@hms.harvard.edu</a>
</div>
</div>
</div>
</div>
<div class="hwp-flex-1 lg:hwp-px-32">
<div class="hwp-person-text-block">
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px">
</div>
<div class="hwp-taxonomy-group hwp-bg-light-core">
<div class="hwp-container hwp-container-px">
<ul class="hwp-taxonomy-group__grid">
<li class="hwp-taxonomy-group-card hwp-bg-light-base">
<h2 class="hwp-taxonomy-group-card__title text-x-small">People</h2>
<a href="/people/project-manager" hreflang="en">Project Manager</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/people/george-colon -->
<h1>George Colon | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/people/george-colon'>avillach-lab.hms.harvard.edu/people/george-colon</a></div>
<main aria-label="George Colon" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="page-content-top" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      George Colon
    </h1>
<div class="hwp-page-header__subtitle">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Full Stack Software Engineer </div>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 lg:hwp-pt-64 lg:hwp-flex lg:hwp-gap-24">
<div class="lg:hwp-w-[19.375rem]">
<div class="hwp-person-card" data-component-name="person-card">
<div class="hwp-person-card__image">
<div class="field field--name-field-hwp-person-headshot field--type-entity-reference field--label-hidden"><div class="hwp-media hwp-media--4-5-small-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/george_colon_headshot.jpg?itok=fvwG64cS 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/george_colon_headshot.jpg?itok=r1KNsJ1y 2x" type="image/jpeg" width="480"/>
<img alt="George Colon" height="600" loading="lazy" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/george_colon_headshot.jpg?itok=fvwG64cS" width="480"/>
</picture>
</div>
</div>
</div>
</div>
<div class="hwp-person-card__information hwp-bg-light-base hwp-flex-1"></div>
</div>
</div>
<div class="hwp-flex-1 lg:hwp-px-32">
<div class="hwp-person-text-block">
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px">
</div>
<div class="hwp-taxonomy-group hwp-bg-light-core">
<div class="hwp-container hwp-container-px">
<ul class="hwp-taxonomy-group__grid">
<li class="hwp-taxonomy-group-card hwp-bg-light-base">
<h2 class="hwp-taxonomy-group-card__title text-x-small">People</h2>
<a href="/people/software-engineers" hreflang="en">Software Engineers and Programmers</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/people/simran-makwana -->
<h1>Simran Makwana, MS | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/people/simran-makwana'>avillach-lab.hms.harvard.edu/people/simran-makwana</a></div>
<main aria-label="Simran Makwana, MS" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="page-content-top" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Simran Makwana, MS
    </h1>
<div class="hwp-page-header__subtitle">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Bioinformatics Research Engagement Manager</div>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 lg:hwp-pt-64 lg:hwp-flex lg:hwp-gap-24">
<div class="lg:hwp-w-[19.375rem]">
<div class="hwp-person-card" data-component-name="person-card">
<div class="hwp-person-card__image">
<div class="field field--name-field-hwp-person-headshot field--type-entity-reference field--label-hidden"><div class="hwp-media hwp-media--4-5-small-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/simran_headshot.jpg?itok=PfshZHXt 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/simran_headshot.jpg?itok=1o2Dt3Wi 2x" type="image/jpeg" width="480"/>
<img alt="makwana" height="600" loading="lazy" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/simran_headshot.jpg?itok=PfshZHXt" width="480"/>
</picture>
</div>
</div>
</div>
</div>
<div class="hwp-person-card__information hwp-bg-light-base hwp-flex-1"></div>
</div>
</div>
<div class="hwp-flex-1 lg:hwp-px-32">
<div class="hwp-person-text-block">
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px">
</div>
<div class="hwp-taxonomy-group hwp-bg-light-core">
<div class="hwp-container hwp-container-px">
<ul class="hwp-taxonomy-group__grid">
<li class="hwp-taxonomy-group-card hwp-bg-light-base">
<h2 class="hwp-taxonomy-group-card__title text-x-small">People</h2>
<a href="/people/former-members" hreflang="en">Alumni</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/people/alba-gutierrez-sacristan -->
<h1>Alba Gutiérrez-Sacristán, PhD | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/people/alba-gutierrez-sacristan'>avillach-lab.hms.harvard.edu/people/alba-gutierrez-sacristan</a></div>
<main aria-label="Alba Gutiérrez-Sacristán, PhD" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="page-content-top" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Alba Gutiérrez-Sacristán, PhD
    </h1>
<div class="hwp-page-header__subtitle">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Instructor in Biomedical Informatics</div>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 lg:hwp-pt-64 lg:hwp-flex lg:hwp-gap-24">
<div class="lg:hwp-w-[19.375rem]">
<div class="hwp-person-card" data-component-name="person-card">
<div class="hwp-person-card__image">
<div class="field field--name-field-hwp-person-headshot field--type-entity-reference field--label-hidden"><div class="hwp-media hwp-media--4-5-small-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/foto2.png?itok=IjflggkL 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/foto2.png?itok=SHyvXr-I 2x" type="image/png" width="480"/>
<img alt="Alba Gutierrez, MSc" height="600" loading="lazy" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/foto2.png?itok=IjflggkL" width="480"/>
</picture>
</div>
</div>
</div>
</div>
<div class="hwp-person-card__information hwp-bg-light-base hwp-flex-1"> <div class="hwp-flex hwp-gap-8 hwp-items-center">
<span aria-hidden="true" class="material-icon">email</span>
<a class="hwp-break-all" href="mailto:alba_gutierrez@hms.harvard.edu">alba_gutierrez@hms.harvard.edu</a>
</div>
</div>
</div>
</div>
<div class="hwp-flex-1 lg:hwp-px-32">
<div class="hwp-person-text-block">
<div class="hwp-text-block field field--name-field-hwp-body field--type-text-long field--label-hidden"><p>Ph.D. in Biomedicine. Working on ASD.</p></div>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px">
</div>
<div class="hwp-taxonomy-group hwp-bg-light-core">
<div class="hwp-container hwp-container-px">
<ul class="hwp-taxonomy-group__grid">
<li class="hwp-taxonomy-group-card hwp-bg-light-base">
<h2 class="hwp-taxonomy-group-card__title text-x-small">People</h2>
<a href="/people/instructor" hreflang="en">Instructor</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</main><hr>
<!-- Source: avillach-lab.hms.harvard.edu/people/burt-beckwith -->
<h1>Burt Beckwith | Avillach Lab</h1>
<div class='source-url'><a href='https://avillach-lab.hms.harvard.edu/people/burt-beckwith'>avillach-lab.hms.harvard.edu/people/burt-beckwith</a></div>
<main aria-label="Burt Beckwith" aria-labelledby="page-title" id="main-content" lang="en" role="main" tabindex="-1">
<div class="layout-content">
<div>
<div class="hidden" data-drupal-messages-fallback=""></div>
<div class="page-content-top" lang="en">
<div class="hwp-page-title hwp-bg-dark-base">
<div class="hwp-container hwp-container-px hwp-py-32">
<h1 class="text-large">
      Burt Beckwith
    </h1>
<div class="hwp-page-header__subtitle">
<div class="field field--name-field-hwp-person-prof-title field--type-string field--label-hidden">
<div>Senior Full Stack Web Developer</div>
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px hwp-pt-32 lg:hwp-pt-64 lg:hwp-flex lg:hwp-gap-24">
<div class="lg:hwp-w-[19.375rem]">
<div class="hwp-person-card" data-component-name="person-card">
<div class="hwp-person-card__image">
<div class="field field--name-field-hwp-person-headshot field--type-entity-reference field--label-hidden"><div class="hwp-media hwp-media--4-5-small-no-caption">
<div class="field field--name-field-media-image field--type-image field--label-hidden"> <picture>
<source height="600" media="all and (min-width: 0)" srcset="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/burt.beckwith.jpg?itok=FkggTWre 1x, /sites/g/files/omnuum4001/files/styles/hwp_4_5__960x1200/public/avillach_lab/files/burt.beckwith.jpg?itok=zZtLDdvU 2x" type="image/jpeg" width="480"/>
<img alt="burt_beckwith" height="600" loading="lazy" src="/sites/g/files/omnuum4001/files/styles/hwp_4_5__480x600/public/avillach_lab/files/burt.beckwith.jpg?itok=FkggTWre" width="480"/>
</picture>
</div>
</div>
</div>
</div>
<div class="hwp-person-card__information hwp-bg-light-base hwp-flex-1"></div>
</div>
</div>
<div class="hwp-flex-1 lg:hwp-px-32">
<div class="hwp-person-text-block">
</div>
</div>
</div>
</div>
<div class="hwp-container hwp-container-px">
</div>
<div class="hwp-taxonomy-group hwp-bg-light-core">
<div class="hwp-container hwp-container-px">
<ul class="hwp-taxonomy-group__grid">
<li class="hwp-taxonomy-group-card hwp-bg-light-base">
<h2 class="hwp-taxonomy-group-card__title text-x-small">People</h2>
<a href="/people/former-members" hreflang="en">Alumni</a>
</li>
</ul>
</div>
</div>
</div>
</div>
</main><hr>
</body></html>