%0 Journal Article
%A Imen Alkuraya
%A Alexandra Almansa
%A Azubuike Eleonu
%A Paul Avillach
%A Annapurna Poduri
%A Siddharth Srivastava
%B medRxiv : the preprint server for health sciences
%D 2025
%G eng
%R 10.1101/2025.02.12.25322169
%T Use of Computational Phenotypes for Predicting Genetic Subgroups of Cerebral Palsy.
%1 https://www.ncbi.nlm.nih.gov/pubmed/39990554
%8 02/2025
%X <p><b>INTRODUCTION: </b>Emerging evidence suggests that 20-30% of cases of cerebral palsy (CP) may have a genetic cause. Our group previously identified subsets of patients with CP or CP-masquerading conditions who warrant genetic testing, including those with regression or progressive neurological symptoms (CP masqueraders) and those without any known risk factors for CP (cryptogenic CP). Recognition of these subgroups in clinical settings remains challenging.</p><p><b>METHODS: </b>To address this challenge, we developed and evaluated a computational phenotyping approach using ICD- 9/ICD-10 billing codes to automatically identify patients with unexplained CP or CP-masquerading conditions who may benefit from genetic testing. We applied this computational phenotyping approach to a cohort of 250 participants from the Boston Children's Hospital CP Sequencing Study, aimed at identifying genetic causes in CP and CP-masquerading conditions.</p><p><b>RESULTS: </b>Manual review served as the gold standard, identifying 8% as CP masqueraders, 42% as cryptogenic CP, and 50% as non-cryptogenic CP. Computational phenotyping based on ICD-9/10 codes achieved a sensitivity of 95%, specificity of 72%, positive predictive value of 77%, and negative predictive value of 94% in identifying cases warranting genetic testing.</p><p><b>CONCLUSIONS: </b>Our findings demonstrate the feasibility of using computational phenotyping to identify patients with CP or CP- masquerading conditions who warrant genetic testing. Further studies are needed to evaluate the effectiveness and real-world application of this tool in larger healthcare systems. Nonetheless, the computational phenotyping approach holds promise as a possible clinical decision support that could be integrated into electronic health record systems, enhancing clinical workflows and facilitating actionable genetic diagnoses.</p>

%0 Journal Article
%K cardiovascular health study
%K distribution learning
%K ensemble learning
%K gold standard labels
%K intercohort comparison
%K intracohort comparison
%K machine learning
%K semantic learning
%K variable harmonization
%A Doris Yang
%A Doudou Zhou
%A Steven Cai
%A Ziming Gan
%A Michael Pencina
%A Paul Avillach
%A Tianxi Cai
%A Chuan Hong
%B JMIR medical informatics
%D 2025
%G eng
%P e54133
%R 10.2196/54133
%T Robust Automated Harmonization of Heterogeneous Data Through Ensemble Machine Learning: Algorithm Development and Validation Study.
%V 13
%1 https://www.ncbi.nlm.nih.gov/pubmed/39844378
%8 01/2025
%X <p><b>BACKGROUND: </b>Cohort studies contain rich clinical data across large and diverse patient populations and are a common source of observational data for clinical research. Because large scale cohort studies are both time and resource intensive, one alternative is to harmonize data from existing cohorts through multicohort studies. However, given differences in variable encoding, accurate variable harmonization is difficult.</p><p><b>OBJECTIVE: </b>We propose SONAR (Semantic and Distribution-Based Harmonization) as a method for harmonizing variables across cohort studies to facilitate multicohort studies.</p><p><b>METHODS: </b>SONAR used semantic learning from variable descriptions and distribution learning from study participant data. Our method learned an embedding vector for each variable and used pairwise cosine similarity to score the similarity between variables. This approach was built off 3 National Institutes of Health cohorts, including the Cardiovascular Health Study, the Multi-Ethnic Study of Atherosclerosis, and the Women's Health Initiative. We also used gold standard labels to further refine the embeddings in a supervised manner.</p><p><b>RESULTS: </b>The method was evaluated using manually curated gold standard labels from the 3 National Institutes of Health cohorts. We evaluated both the intracohort and intercohort variable harmonization performance. The supervised SONAR method outperformed existing benchmark methods for almost all intracohort and intercohort comparisons using area under the curve and top-k accuracy metrics. Notably, SONAR was able to significantly improve harmonization of concepts that were difficult for existing semantic methods to harmonize.</p><p><b>CONCLUSIONS: </b>SONAR achieves accurate variable harmonization within and between cohort studies by harnessing the complementary strengths of semantic learning and variable distribution learning.</p>

%0 Journal Article
%A Maria Menezes
%A Alexander Hoffmann
%A Amelia Tan
%A Mariné Nalbandyan
%A Gilbert Omenn
%A Diego Mazzotti
%A Alejandro Hernández-Arango
%A Shyam Visweswaran
%A Shruthi Venkatesh
%A Kenneth Mandl
%A Florence Bourgeois
%A James Lee
%A Andrew Makmur
%A David Hanauer
%A Michael Semanik
%A Lauren Kerivan
%A Terra Hill
%A Julian Forero
%A Carlos Restrepo
%A Matteo Vigna
%A Piero Ceriana
%A Noor Abu-El-Rub
%A Paul Avillach
%A Riccardo Bellazzi
%A Thomas Callaci
%A Alba Gutiérrez-Sacristán
%A Alberto Malovini
%A Jomol Mathew
%A Michele Morris
%A Venkatesh Murthy
%A Tommaso Buonocore
%A Enea Parimbelli
%A Lav Patel
%A Carlos Saez
%A Malarkodi Samayamuthu
%A Jeffrey Thompson
%A Valentina Tibollo
%A Zongqi Xia
%A Isaac Kohane
%A Consortium Records
%B The Lancet. Digital health
%D 2025
%G eng
%N 1
%P e35-e43
%R 10.1016/S2589-7500(24)00246-2
%T The potential of Generative Pre-trained Transformer 4 (GPT-4) to analyse medical notes in three different languages: a retrospective model-evaluation study.
%V 7
%1 https://www.ncbi.nlm.nih.gov/pubmed/39722251
%8 01/2025
%X <p><b>BACKGROUND: </b>Patient notes contain substantial information but are difficult for computers to analyse due to their unstructured format. Large-language models (LLMs), such as Generative Pre-trained Transformer 4 (GPT-4), have changed our ability to process text, but we do not know how effectively they handle medical notes. We aimed to assess the ability of GPT-4 to answer predefined questions after reading medical notes in three different languages.</p><p><b>METHODS: </b>For this retrospective model-evaluation study, we included eight university hospitals from four countries (ie, the USA, Colombia, Singapore, and Italy). Each site submitted seven de-identified medical notes related to seven separate patients to the coordinating centre between June 1, 2023, and Feb 28, 2024. Medical notes were written between Feb 1, 2020, and June 1, 2023. One site provided medical notes in Spanish, one site provided notes in Italian, and the remaining six sites provided notes in English. We included admission notes, progress notes, and consultation notes. No discharge summaries were included in this study. We advised participating sites to choose medical notes that, at time of hospital admission, were for patients who were male or female, aged 18-65 years, had a diagnosis of obesity, had a diagnosis of COVID-19, and had submitted an admission note. Adherence to these criteria was optional and participating sites randomly chose which medical notes to submit. When entering information into GPT-4, we prepended each medical note with an instruction prompt and a list of 14 questions that had been chosen a priori. Each medical note was individually given to GPT-4 in its original language and in separate sessions; the questions were always given in English. At each site, two physicians independently validated responses by GPT-4 and responded to all 14 questions. Each pair of physicians evaluated responses from GPT-4 to the seven medical notes from their own site only. Physicians were not masked to responses from GPT-4 before providing their own answers, but were masked to responses from the other physician.</p><p><b>FINDINGS: </b>We collected 56 medical notes, of which 42 (75%) were in English, seven (13%) were in Italian, and seven (13%) were in Spanish. For each medical note, GPT-4 responded to 14 questions, resulting in 784 responses. In 622 (79%, 95% CI 76-82) of 784 responses, both physicians agreed with GPT-4. In 82 (11%, 8-13) responses, only one physician agreed with GPT-4. In the remaining 80 (10%, 8-13) responses, neither physician agreed with GPT-4. Both physicians agreed with GPT-4 more often for medical notes written in Spanish (86 [88%, 95% CI 79-93] of 98 responses) and Italian (82 [84%, 75-90] of 98 responses) than in English (454 [77%, 74-80] of 588 responses).</p><p><b>INTERPRETATION: </b>The results of our model-evaluation study suggest that GPT-4 is accurate when analysing medical notes in three different languages. In the future, research should explore how LLMs can be integrated into clinical workflows to maximise their use in health care.</p><p><b>FUNDING: </b>None.</p>

%0 Journal Article
%K Data collection
%K Electronic Health Records
%K Multicenter study
%K Prospective Studies
%A Alba Gutiérrez-Sacristán
%A Simran Makwana
%A Audrey Dionne
%A Simran Mahanta
%A Karla Dyer
%A Faridis Serrano
%A Carmen Watrin
%A Pierre Pages
%A Sajad Mousavi
%A Anil Degala
%A Jessica Lyons
%A Danielle Pillion
%A Joany Zachariasse
%A Lara Shekerdemian
%A Dongngan Truong
%A Jane Newburger
%A Paul Avillach
%B EBioMedicine
%D 2024
%G eng
%P 105337
%R 10.1016/j.ebiom.2024.105337
%T Development and validation of an open-source pipeline for automatic population of case report forms from electronic health records: a pediatric multi-center prospective study.
%V 108
%1 https://www.ncbi.nlm.nih.gov/pubmed/39288532
%8 10/2024
%X <p><b>BACKGROUND: </b>Clinical trials and registry studies are essential for advancing research and developing novel treatments. However, these studies rely on manual entry of thousands of variables for each patient. Repurposing real-world data can significantly simplify the data collection, reduce transcription errors, and make the data entry process more efficient, consistent, and cost-effective.</p><p><b>METHODS: </b>We developed an open-source computational pipeline to collect laboratory and medication information from the electronic health record (EHR) data and populate case report forms. The pipeline was developed and validated with data from two independent pediatric hospitals in the US as part of the Long-terM OUtcomes after Multisystem Inflammatory Syndrome In Children (MUSIC) study. Our pipeline allowed the completion of two of the most time-consuming forms. We compared automatically extracted results with manually entered values in one hospital and applied the pipeline to a second hospital, where the output served as the primary data source for case report forms.</p><p><b>FINDINGS: </b>We extracted and populated 51,845 laboratory and 4913 medication values for 159 patients in two hospitals participating in a prospective pediatric study. We evaluated pipeline performance against data for 104 patients manually entered by clinicians in one of the hospitals. The highest concordance was found during patient hospitalization, with 91.59% of the automatically extracted laboratory and medication values corresponding with the manually entered values. In addition to the successfully populated values, we identified an additional 13,396 laboratory and 567 medication values of interest for the study.</p><p><b>INTERPRETATION: </b>The automatic data entry of laboratory and medication values during admission is feasible and has a high concordance with the manually entered data. By implementing this proof of concept, we demonstrate the quality of automatic data extraction and highlight the potential of secondary use of EHR data to advance medical science by improving data entry efficiency and expediting clinical research.</p><p><b>FUNDING: </b>NIH Grant 1OT3HL147154-01, U24HL135691, UG1HL135685.</p>

%0 Journal Article
%K SHANK3
%K 22q13.3 deletion
%K Phelan-McDermid syndrome
%K Phenome-wide association study
%A Rui Yin
%A Maxime Wack
%A Claire Hassen-Khodja
%A Michael McDuffie
%A Geraldine Bliss
%A Elizabeth Horn
%A Cartik Kothari
%A Brittany McLarney
%A Rebecca Davis
%A Kristen Hanson
%A Megan O'Boyle
%A Catalina Betancur
%A Paul Avillach
%B Molecular autism
%D 2024
%G eng
%N 1
%P 40
%R 10.1186/s13229-024-00619-z
%T Phenome-wide profiling identifies genotype-phenotype associations in Phelan-McDermid syndrome using family-sourced data from an international registry.
%V 15
%1 https://www.ncbi.nlm.nih.gov/pubmed/39350236
%8 09/2024
%X <p><b>BACKGROUND: </b>Phelan-McDermid syndrome (PMS) is a rare neurodevelopmental disorder caused by 22q13 deletions that include the SHANK3 gene or pathogenic sequence variants in SHANK3. It is characterized by global developmental delay, intellectual disability, speech impairment, autism spectrum disorder, and hypotonia; other variable features include epilepsy, brain and renal malformations, and mild dysmorphic features. Here, we conducted genotype-phenotype correlation analyses using the PMS International Registry, a family-driven registry that compiles clinical data in the form of family-reported outcomes and family-sourced genetic test results.</p><p><b>METHODS: </b>Data from the registry were harmonized and integrated into the i2b2/tranSMART clinical and genomics data warehouse. We gathered information from 401 individuals with 22q13 deletions including SHANK3 (n = 350, ranging in size from 10 kb to 9.1 Mb) or pathogenic or likely pathogenic SHANK3 sequence variants (n = 51), and used regression models with deletion size as a potential predictor of clinical outcomes for 328 phenotypes.</p><p><b>RESULTS: </b>Our results showed that increased deletion size was significantly associated with delay in gross and fine motor acquisitions, a spectrum of conditions related to poor muscle tone, renal malformations, mild dysmorphic features (e.g., large fleshy hands, sacral dimple, dysplastic toenails, supernumerary teeth), lymphedema, congenital heart defects, and more frequent neuroimaging abnormalities and infections. These findings indicate that genes upstream of SHANK3 also contribute to some of the manifestations of PMS in individuals with larger deletions. We also showed that self-help skills, verbal ability and a range of psychiatric diagnoses (e.g., autism, ADHD, anxiety disorder) were more common among individuals with smaller deletions and SHANK3 variants.</p><p><b>LIMITATIONS: </b>Some participants were tested with targeted 22q microarrays rather than genome-wide arrays, and karyotypes were unavailable in many cases, thus precluding the analysis of the effect of other copy number variants or chromosomal rearrangements on the phenotype.</p><p><b>CONCLUSIONS: </b>This is the largest reported case series of individuals with PMS. Overall, we demonstrate the feasibility of using data from a family-sourced registry to conduct genotype-phenotype analyses in rare genetic disorders. We replicate and strengthen previous findings, and reveal novel associations between larger 22q13 deletions and congenital heart defects, neuroimaging abnormalities and recurrent infections.</p>

%0 Journal Article
%K Disparities
%K Hispanics
%K Older adults
%K Substance use disorders
%A Jonathan Hernández-Agosto
%A Darlene Santiago-Quiñones
%A Lucía García-Carmona
%A Alba Gutiérrez-Sacristán
%B Drug and alcohol dependence
%D 2024
%G eng
%P 111357
%R 10.1016/j.drugalcdep.2024.111357
%T Substance use disorders and disparities in hispanic and non-hispanic older adults.
%V 261
%1 https://www.ncbi.nlm.nih.gov/pubmed/38896947
%8 08/2024
%X <p><b>BACKGROUND: </b>The global longevity revolution increased the older adult population, posing unique health and economic challenges with implications for healthcare, especially substance use disorders (SUD).</p><p><b>METHODS: </b>This was a retrospective cohort study of United States older adults, Hispanic and non-Hispanic, who got at least one mental and/or behavioral disorder diagnosis between 2017 and 2021 at age 65 or older. SUD prevalence, prescription frequency changes over time, and comorbidities associated with each medication were compared.</p><p><b>RESULTS: </b>Electronic health records for 356,133 older adults (110,236 Hispanics and 245,897 non-Hispanics) were analyzed. Notably, 79 % of Hispanics fell below the 100 % federal poverty level, compared to 60 % of non-Hispanics (P<.001). Non-Hispanics also had significantly more average encounters (P=.003) and diagnoses (P<.001). Regression analysis on alcohol-related disorders indicated that the odd ratios of being male (OR=2.93, P<.000), and having low income (OR=1.62, P<.000), increase the odds for this SUD, while being Hispanic and primarily speaking Spanish decreases the odds for all SUDs considered in this study.</p><p><b>CONCLUSIONS: </b>This cohort study revealed significant disparities related to social determinants of health between Hispanic and non-Hispanic older adults and emphasizes the need for continuous surveillance of older adults as with SUDs. Differences in comorbidity patterns imply distinct risk factors within each population, influenced by demographic-specific elements. Recognizing these variations is essential for tailoring culturally sensitive prevention, intervention, and treatment strategies to each population's unique needs.</p>

%0 Journal Article
%K Undiagnosed Diseases Network
%K diagnostic variant prediction
%K ensemble predictor
%K generative adversarial network
%K model interpretability
%K variants of uncertain significance
%A Rui Yin
%A Alba Gutierrez
%A Undiagnosed Network
%A Shilpa Kobren
%A Paul Avillach
%B medRxiv : the preprint server for health sciences
%D 2024
%G eng
%R 10.1101/2024.04.15.24305876
%T VarPPUD: Variant post prioritization developed for undiagnosed genetic disorders.
%1 https://www.ncbi.nlm.nih.gov/pubmed/38699371
%8 04/2024
%X <p>Rare and ultra-rare genetic conditions are estimated to impact nearly 1 in 17 people worldwide, yet accurately pinpointing the diagnostic variants underlying each of these conditions remains a formidable challenge. Because comprehensive,  functional assessment of all possible genetic variants is infeasible, clinicians instead consider  variant pathogenicity predictions to distinguish plausibly disease-causing from benign variants across the genome. However, in the most difficult undiagnosed cases, such as those accepted to the Undiagnosed Diseases Network (UDN), existing pathogenicity predictions cannot reliably discern true etiological variant(s) from other deleterious candidate variants that were prioritized through N-of-1 efforts. Pinpointing the disease-causing variant from a pool of plausible candidates remains a largely manual effort requiring extensive clinical workups, functional and experimental assays, and eventual identification of genotype- and phenotype-matched individuals. Here, we introduce VarPPUD, a tool trained on prioritized variants from UDN cases, that leverages gene-, amino acid-, and nucleotide-level features to discern pathogenic variants from other deleterious variants that are unlikely to be confirmed as disease relevant. VarPPUD achieves a cross-validated accuracy of 79.3% and precision of 77.5% on a held-out subset of uniquely challenging UDN cases, respectively representing an average 18.6% and 23.4% improvement over nine traditional pathogenicity prediction approaches on this task. We validate VarPPUD's ability to discriminate likely from unlikely pathogenic variants on synthetic, GAN-generated candidate variants as well. Finally, we show how VarPPUD can be probed to evaluate each input feature's importance and contribution toward prediction-an essential step toward understanding the distinct characteristics of newly-uncovered disease-causing variants.</p>

%0 Journal Article
%A Meghan Hutch
%A Jiyeon Son
%A Trang Le
%A Chuan Hong
%A Xuan Wang
%A Zahra Abad
%A Michele Morris
%A Alba Gutiérrez-Sacristán
%A Jeffrey Klann
%A Anastasia Spiridou
%A Ashley Batugo
%A Riccardo Bellazzi
%A Vincent Benoit
%A Clara-Lea Bonzel
%A William Bryant
%A Lorenzo Chiudinelli
%A Kelly Cho
%A Priyam Das
%A Tomás González
%A David Hanauer
%A Darren Henderson
%A Yuk-Lam Ho
%A Ne Loh
%A Adeline Makoudjou
%A Simran Makwana
%A Alberto Malovini
%A Bertrand Moal
%A Danielle Mowery
%A Antoine Neuraz
%A Malarkodi Samayamuthu
%A Fernando Vidorreta
%A Emily Schriver
%A Petra Schubert
%A Jeffery Talbert
%A Amelia Tan
%A Byorn Tan
%A Bryce Tan
%A Valentina Tibollo
%A Patric Tippman
%A Guillaume Verdy
%A William Yuan
%A Paul Avillach
%A Nils Gehlenborg
%A Gilbert Omenn
%A Consortium EHR
%A Shyam Visweswaran
%A Tianxi Cai
%A Yuan Luo
%A Zongqi Xia
%B PLOS digital health
%D 2024
%G eng
%N 4
%P e0000484
%R 10.1371/journal.pdig.0000484
%T Neurological diagnoses in hospitalized COVID-19 patients associated with adverse outcomes: A multinational cohort study.
%V 3
%1 https://www.ncbi.nlm.nih.gov/pubmed/38620037
%8 04/2024
%X <p>Few studies examining the patient outcomes of concurrent neurological manifestations during acute COVID-19 leveraged multinational cohorts of adults and children or distinguished between central and peripheral nervous system (CNS vs. PNS) involvement. Using a federated multinational network in which local clinicians and informatics experts curated the electronic health records data, we evaluated the risk of prolonged hospitalization and mortality in hospitalized COVID-19 patients from 21 healthcare systems across 7 countries. For adults, we used a federated learning approach whereby we ran Cox proportional hazard models locally at each healthcare system and performed a meta-analysis on the aggregated results to estimate the overall risk of adverse outcomes across our geographically diverse populations. For children, we reported descriptive statistics separately due to their low frequency of neurological involvement and poor outcomes. Among the 106,229 hospitalized COVID-19 patients (104,031 patients ≥18 years; 2,198 patients <18 years, January 2020-October 2021), 15,101 (14%) had at least one CNS diagnosis, while 2,788 (3%) had at least one PNS diagnosis. After controlling for demographics and pre-existing conditions, adults with CNS involvement had longer hospital stay (11 versus 6 days) and greater risk of (Hazard Ratio = 1.78) and faster time to death (12 versus 24 days) than patients with no neurological condition (NNC) during acute COVID-19 hospitalization. Adults with PNS involvement also had longer hospital stay but lower risk of mortality than the NNC group. Although children had a low frequency of neurological involvement during COVID-19 hospitalization, a substantially higher proportion of children with CNS involvement died compared to those with NNC (6% vs 1%). Overall, patients with concurrent CNS manifestation during acute COVID-19 hospitalization faced greater risks for adverse clinical outcomes than patients without any neurological diagnosis. Our global informatics framework using a federated approach (versus a centralized data collection approach) has utility for clinical discovery beyond COVID-19.</p>

%0 Journal Article
%K Birth control pill
%K Contraceptive decision making
%K Contraceptive side effects
%K LARC
%K SARC
%K Social networks
%K Tweets
%A Melody Huang
%A Alba Gutiérrez-Sacristán
%A Elizabeth Janiak
%A Katherine Young
%A Anabel Starosta
%A Katherine Blanton
%A Alaleh Azhir
%A Caroline Goldfarb
%A Felícita Kuperwasser
%A Kimberly Schaefer
%A Rachel Stoddard
%A Rajet Vatsa
%A Allison Merz-Herrala
%A Deborah Bartz
%B Contraception and reproductive medicine
%D 2024
%G eng
%N 1
%P 5
%R 10.1186/s40834-024-00262-2
%T Contraceptive content shared on social media: an analysis of Twitter.
%V 9
%1 https://www.ncbi.nlm.nih.gov/pubmed/38321582
%8 02/2024
%X <p><b>BACKGROUND: </b>Information on social media may affect peoples' contraceptive decision making. We performed an exploratory analysis of contraceptive content on Twitter (recently renamed X), a popular social media platform.</p><p><b>METHODS: </b>We selected a random subset of 1% of publicly available, English-language tweets related to reversible, prescription contraceptive methods posted between January 2014 and December 2019. We oversampled tweets for the contraceptive patch to ensure at least 200 tweets per method. To create the codebook, we identified common themes specific to tweet content topics, tweet sources, and tweets soliciting information or providing advice. All posts were coded by two team members, and differences were adjudicated by a third reviewer. Descriptive analyses were reported with accompanying qualitative findings.</p><p><b>RESULTS: </b>During the study period, 457,369 tweets about reversible contraceptive methods were published, with a random sample of 4,434 tweets used for final analysis. Tweets most frequently discussed contraceptive method decision-making (26.7%) and side effects (20.5%), particularly for long-acting reversible contraceptive methods and the depot medroxyprogesterone acetate shot. Tweets about logistics of use or adherence were common for short-acting reversible contraceptives. Tweets were frequently posted by contraceptive consumers (50.6%). A small proportion of tweets explicitly requested information (6.2%) or provided advice (4.2%).</p><p><b>CONCLUSIONS: </b>Clinicians should be aware that individuals are exposed to information through Twitter that may affect contraceptive perceptions and decision making, particularly regarding long-acting reversible contraceptives. Social media is a valuable source for studying contraceptive beliefs missing in traditional health research and may be used by professionals to disseminate accurate contraceptive information.</p>

%0 Journal Article
%A Michael Gargano
%A Nicolas Matentzoglu
%A Ben Coleman
%A Eunice Addo-Lartey
%A Anna Anagnostopoulos V
%A Joel Anderton
%A Paul Avillach
%A Anita Bagley
%A Eduard Bakštein
%A James Balhoff
%A Gareth Baynam
%A Susan Bello
%A Michael Berk
%A Holli Bertram
%A Somer Bishop
%A Hannah Blau
%A David Bodenstein
%A Pablo Botas
%A Kaan Boztug
%A Jolana Čady
%A Tiffany Callahan
%A Rhiannon Cameron
%A Seth Carbon
%A Francisco Castellanos
%A Harry Caufield
%A Lauren Chan
%A Christopher Chute
%A Jaime Cruz-Rojo
%A Noémi Dahan-Oliel
%A Jon Davids
%A Maud de Dieuleveult
%A Vinicius de Souza
%A Bert de Vries
%A Esther de Vries
%A Raymond DePaulo
%A Beata Derfalvi
%A Ferdinand Dhombres
%A Claudia Diaz-Byrd
%A Alexander Dingemans
%A Bruno Donadille
%A Michael Duyzend
%A Reem Elfeky
%A Shahim Essaid
%A Carolina Fabrizzi
%A Giovanna Fico
%A Helen Firth V
%A Yun Freudenberg-Hua
%A Janice Fullerton
%A Davera Gabriel
%A Kimberly Gilmour
%A Jessica Giordano
%A Fernando Goes
%A Rachel Moses
%A Ian Green
%A Matthias Griese
%A Tudor Groza
%A Weihong Gu
%A Julia Guthrie
%A Benjamin Gyori
%A Ada Hamosh
%A Marc Hanauer
%A Kateřina Hanušová
%A Yongqun He
%A Harshad Hegde
%A Ingo Helbig
%A Kateřina Holasová
%A Charles Hoyt
%A Shangzhi Huang
%A Eric Hurwitz
%A Julius Jacobsen
%A Xiaofeng Jiang
%A Lisa Joseph
%A Kamyar Keramatian
%A Bryan King
%A Katrin Knoflach
%A David Koolen
%A Megan Kraus
%A Carlo Kroll
%A Maaike Kusters
%A Markus Ladewig
%A David Lagorce
%A Meng-Chuan Lai
%A Pablo Lapunzina
%A Bryan Laraway
%A David Lewis-Smith
%A Xiarong Li
%A Caterina Lucano
%A Marzieh Majd
%A Mary Marazita
%A Victor Martinez-Glez
%A Toby McHenry
%A Melvin McInnis
%A Julie McMurry
%A Michaela Mihulová
%A Caitlin Millett
%A Philip Mitchell
%A Veronika Moslerová
%A Kenji Narutomi
%A Shahrzad Nematollahi
%A Julian Nevado
%A Andrew Nierenberg
%A Nikola Čajbiková
%A John Nurnberger
%A Soichi Ogishima
%A Daniel Olson
%A Abigail Ortiz
%A Harry Pachajoa
%A Guiomar de Nanclares
%A Amy Peters
%A Tim Putman
%A Christina Rapp
%A Ana Rath
%A Justin Reese
%A Lauren Rekerle
%A Angharad Roberts
%A Suzy Roy
%A Stephan Sanders
%A Catharina Schuetz
%A Eva Schulte
%A Thomas Schulze
%A Martin Schwarz
%A Katie Scott
%A Dominik Seelow
%A Berthold Seitz
%A Yiping Shen
%A Morgan Similuk
%A Eric Simon
%A Balwinder Singh
%A Damian Smedley
%A Cynthia Smith
%A Jake Smolinsky
%A Sarah Sperry
%A Elizabeth Stafford
%A Ray Stefancsik
%A Robin Steinhaus
%A Rebecca Strawbridge
%A Jagadish Sundaramurthi
%A Polina Talapova
%A Jair Castano
%A Pavel Tesner
%A Rhys Thomas
%A Audrey Thurm
%A Marek Turnovec
%A Marielle van Gijn
%A Nicole Vasilevsky
%A Markéta Vlčková
%A Anita Walden
%A Kai Wang
%A Ron Wapner
%A James Ware
%A Addo Wiafe
%A Samuel Wiafe
%A Lisa Wiggins
%A Andrew Williams
%A Chen Wu
%A Margot Wyrwoll
%A Hui Xiong
%A Nefize Yalin
%A Yasunori Yamamoto
%A Lakshmi Yatham
%A Anastasia Yocum
%A Allan Young
%A Zafer Yüksel
%A Peter Zandi
%A Andreas Zankl
%A Ignacio Zarante
%A Miroslav Zvolský
%A Sabrina Toro
%A Leigh Carmody
%A Nomi Harris
%A Monica Munoz-Torres
%A Daniel Danis
%A Christopher Mungall
%A Sebastian Köhler
%A Melissa Haendel
%A Peter Robinson
%B Nucleic acids research
%D 2024
%G eng
%N D1
%P D1333-D1346
%R 10.1093/nar/gkad1005
%T The Human Phenotype Ontology in 2024: phenotypes around the world.
%V 52
%1 https://www.ncbi.nlm.nih.gov/pubmed/37953324
%8 01/2024
%X <p>The Human Phenotype Ontology (HPO) is a widely used resource that comprehensively organizes and defines the phenotypic features of human disease, enabling computational inference and supporting genomic and phenotypic analyses through semantic similarity and machine learning algorithms. The HPO has widespread applications in clinical diagnostics and translational research, including genomic diagnostics, gene-disease discovery, and cohort analytics. In recent years, groups around the world have developed translations of the HPO from English to other languages, and the HPO browser has been internationalized, allowing users to view HPO term labels and in many cases synonyms and definitions in ten languages in addition to English. Since our last report, a total of 2239 new HPO terms and 49235 new HPO annotations were developed, many in collaboration with external groups in the fields of psychiatry, arthrogryposis, immunology and cardiology. The Medical Action Ontology (MAxO) is a new effort to model treatments and other measures taken for clinical management. Finally, the HPO consortium is contributing to efforts to integrate the HPO and the GA4GH Phenopacket Schema into electronic health records (EHRs) with the goal of more standardized and computable integration of rare disease data in EHRs.</p>

%0 Journal Article
%K COVID-19
%K Clinical phenotypes
%K Multisystem inflammatory syndrome
%K Outcomes
%K Paediatric inflammatory multisystem syndrome
%K Pediatrics
%K SARS-CoV-2
%K Variants
%A Francesca Sperotto
%A Alba Gutiérrez-Sacristán
%A Simran Makwana
%A Xiudi Li
%A Valerie Rofeberg
%A Tianxi Cai
%A Florence Bourgeois
%A Gilbert Omenn
%A David Hanauer
%A Carlos Saez
%A Clara-Lea Bonzel
%A Emily Bucholz
%A Audrey Dionne
%A Matthew Elias
%A Noelia García-Barrio
%A Tomás González
%A Richard Issitt
%A Kate Kernan
%A Jessica Laird-Gion
%A Sarah Maidlow
%A Kenneth Mandl
%A Taha Ahooyi
%A Cinta Moraleda
%A Michele Morris
%A Karyn Moshal
%A Miguel Pedrera-Jiménez
%A Mohsin Shah
%A Andrew South
%A Anastasia Spiridou
%A Deanne Taylor
%A Guillaume Verdy
%A Shyam Visweswaran
%A Xuan Wang
%A Zongqi Xia
%A Joany Zachariasse
%A Consortium EHR
%A Jane Newburger
%A Paul Avillach
%B EClinicalMedicine
%D 2023
%G eng
%P 102212
%R 10.1016/j.eclinm.2023.102212
%T Clinical phenotypes and outcomes in children with multisystem inflammatory syndrome across SARS-CoV-2 variant eras: a multinational study from the 4CE consortium.
%V 64
%1 https://www.ncbi.nlm.nih.gov/pubmed/37745025
%8 10/2023
%X <p><b>BACKGROUND: </b>Multisystem inflammatory syndrome in children (MIS-C) is a severe complication of SARS-CoV-2 infection. It remains unclear how MIS-C phenotypes vary across SARS-CoV-2 variants. We aimed to investigate clinical characteristics and outcomes of MIS-C across SARS-CoV-2 eras.</p><p><b>METHODS: </b>We performed a multicentre observational retrospective study including seven paediatric hospitals in four countries (France, Spain, U.K., and U.S.). All consecutive confirmed patients with MIS-C hospitalised between February 1st, 2020, and May 31st, 2022, were included. Electronic Health Records (EHR) data were used to calculate pooled risk differences (RD) and effect sizes (ES) at site level, using  as reference. Meta-analysis was used to pool data across sites.</p><p><b>FINDINGS: </b>Of 598 patients with MIS-C (61% male, 39% female; mean age 9.7 years [SD 4.5]), 383 (64%) were admitted in the  era, 111 (19%) in the  era, and 104 (17%) in the  era. Compared with patients admitted in the  era, those admitted in the  era were younger (ES -1.18 years [95% CI -2.05, -0.32]), had fewer respiratory symptoms (RD -0.15 [95% CI -0.33, -0.04]), less frequent non-cardiogenic shock or systemic inflammatory response syndrome (SIRS) (RD -0.35 [95% CI -0.64, -0.07]), lower lymphocyte count (ES -0.16 × 10/uL [95% CI -0.30, -0.01]), lower C-reactive protein (ES -28.5 mg/L [95% CI -46.3, -10.7]), and lower troponin (ES -0.14 ng/mL [95% CI -0.26, -0.03]). Patients admitted in the  versus  eras were younger (ES -1.6 years [95% CI -2.5, -0.8]), had less frequent SIRS (RD -0.18 [95% CI -0.30, -0.05]), lower lymphocyte count (ES -0.39 × 10/uL [95% CI -0.52, -0.25]), lower troponin (ES -0.16 ng/mL [95% CI -0.30, -0.01]) and less frequently received anticoagulation therapy (RD -0.19 [95% CI -0.37, -0.04]). Length of hospitalization was shorter in the  versus  eras (-1.3 days [95% CI -2.3, -0.4]).</p><p><b>INTERPRETATION: </b>Our study suggested that MIS-C clinical phenotypes varied across SARS-CoV-2 eras, with patients in  and  eras being younger and less sick. EHR data can be effectively leveraged to identify rare complications of pandemic diseases and their variation over time.</p><p><b>FUNDING: </b>None.</p>

%0 Journal Article
%K COVID-19
%K Electronic Health Records
%K PASC
%K Post-acute sequelae of SARS-CoV-2
%K SARS-CoV-2
%A Arianna Dagliati
%A Zachary Strasser
%A Zahra Abad
%A Jeffrey Klann
%A Kavishwar Wagholikar
%A Rebecca Mesa
%A Shyam Visweswaran
%A Michele Morris
%A Yuan Luo
%A Darren Henderson
%A Malarkodi Samayamuthu
%A Bryce Tan
%A Guillame Verdy
%A Gilbert Omenn
%A Zongqi Xia
%A Riccardo Bellazzi
%A Consortium EHR
%A Shawn Murphy
%A John Holmes
%A Hossein Estiri
%A Consortium EHR
%B EClinicalMedicine
%D 2023
%G eng
%P 102210
%R 10.1016/j.eclinm.2023.102210
%T Characterization of long COVID temporal sub-phenotypes by distributed representation learning from electronic health record data: a cohort study.
%V 64
%1 https://www.ncbi.nlm.nih.gov/pubmed/37745021
%8 10/2023
%X <p><b>BACKGROUND: </b>Characterizing Post-Acute Sequelae of COVID (SARS-CoV-2 Infection), or  has been challenging due to the multitude of sub-phenotypes, temporal attributes, and definitions. Scalable characterization of PASC sub-phenotypes can enhance screening capacities, disease management, and treatment planning.</p><p><b>METHODS: </b>We conducted a retrospective multi-centre observational cohort study, leveraging longitudinal electronic health record (EHR) data of 30,422 patients from three healthcare systems in the Consortium for the Clinical Characterization of COVID-19 by EHR (4CE). From the total cohort, we applied a deductive approach on 12,424 individuals with follow-up data and developed a distributed representation learning process for providing augmented definitions for PASC sub-phenotypes.</p><p><b>FINDINGS: </b>Our framework characterized seven PASC sub-phenotypes. We estimated that on average 15.7% of the hospitalized COVID-19 patients were likely to suffer from at least one PASC symptom and almost 5.98%, on average, had multiple symptoms. Joint pain and dyspnea had the highest prevalence, with an average prevalence of 5.45% and 4.53%, respectively.</p><p><b>INTERPRETATION: </b>We provided a scalable framework to every participating healthcare system for estimating PASC sub-phenotypes prevalence and temporal attributes, thus developing a unified model that characterizes augmented sub-phenotypes across the different systems.</p><p><b>FUNDING: </b>Authors are supported by National Institute of Allergy and Infectious Diseases, National Institute on Aging, National Center for Advancing Translational Sciences, National Medical Research Council, National Institute of Neurological Disorders and Stroke, European Union, National Institutes of Health, National Center for Advancing Translational Sciences.</p>

%0 Journal Article
%K cloud computing
%K data analysis
%K data-driven science
%K Reproducibility of Results
%K team science
%A Stan Ahalt
%A Paul Avillach
%A Rebecca Boyles
%A Kira Bradford
%A Steven Cox
%A Brandi Davis-Dusenbery
%A Robert Grossman
%A Ashok Krishnamurthy
%A Alisa Manning
%A Benedict Paten
%A Anthony Philippakis
%A Ingrid Borecki
%A Shu Chen
%A Jon Kaltman
%A Sweta Ladwa
%A Chip Schwartz
%A Alastair Thomson
%A Sarah Davis
%A Alison Leaf
%A Jessica Lyons
%A Elizabeth Sheets
%A Joshua Bis
%A Matthew Conomos
%A Alessandro Culotti
%A Thomas Desain
%A Jack Digiovanna
%A Milan Domazet
%A Stephanie Gogarten
%A Alba Gutiérrez-Sacristán
%A Tim Harris
%A Ben Heavner
%A Deepti Jain
%A Brian O'Connor
%A Kevin Osborn
%A Danielle Pillion
%A Jacob Pleiness
%A Ken Rice
%A Garrett Rupp
%A Arnaud Serret-Larmande
%A Albert Smith
%A Jason Stedman
%A Adrienne Stilp
%A Teresa Barsanti
%A John Cheadle
%A Christopher Erdmann
%A Brandy Farlow
%A Allie Gartland-Gray
%A Julie Hayes
%A Hannah Hiles
%A Paul Kerr
%A Chris Lenhardt
%A Tom Madden
%A Joanna Mieczkowska
%A Amanda Miller
%A Patrick Patton
%A Marcie Rathbun
%A Stephanie Suber
%A Joe Asare
%B Journal of the American Medical Informatics Association : JAMIA
%D 2023
%G eng
%N 7
%P 1293-1300
%R 10.1093/jamia/ocad048
%T Building a collaborative cloud platform to accelerate heart, lung, blood, and sleep research.
%V 30
%1 https://www.ncbi.nlm.nih.gov/pubmed/37192819
%8 06/2023
%X <p>Research increasingly relies on interrogating large-scale data resources. The NIH National Heart, Lung, and Blood Institute developed the NHLBI BioData CatalystⓇ (BDC), a community-driven ecosystem where researchers, including bench and clinical scientists, statisticians, and algorithm developers, find, access, share, store, and compute on large-scale datasets. This ecosystem provides secure, cloud-based workspaces, user authentication and authorization, search, tools and workflows, applications, and new innovative features to address community needs, including exploratory data analysis, genomic and imaging tools, tools for reproducibility, and improved interoperability with other NIH data science platforms. BDC offers straightforward access to large-scale datasets and computational resources that support precision medicine for heart, lung, blood, and sleep conditions, leveraging separately developed and managed platforms to maximize flexibility based on researcher needs, expertise, and backgrounds. Through the NHLBI BioData Catalyst Fellows Program, BDC facilitates scientific discoveries and technological advances. BDC also facilitated accelerated research on the coronavirus disease-2019 (COVID-19) pandemic.</p>

%0 Journal Article
%A Harrison Zhang
%A Jacqueline Honerlaw
%A Monika Maripuri
%A Malarkodi Samayamuthu
%A Brendin Beaulieu-Jones
%A Huma Baig
%A Sehi L'Yi
%A Yuk-Lam Ho
%A Michele Morris
%A Vidul Panickan
%A Xuan Wang
%A Griffin Weber
%A Katherine Liao
%A Shyam Visweswaran
%A Bryce Tan
%A William Yuan
%A Nils Gehlenborg
%A Sumitra Muralidhar
%A Rachel Ramoni
%A Consortium EHR
%A Isaac Kohane
%A Zongqi Xia
%A Kelly Cho
%A Tianxi Cai
%A Gabriel Brat
%B Nature medicine
%D 2023
%G eng
%N 5
%P 1040-1043
%R 10.1038/s41591-023-02274-y
%T Potential pitfalls in the use of real-world data for studying long COVID.
%V 29
%1 https://www.ncbi.nlm.nih.gov/pubmed/37055567
%8 05/2023

%0 Journal Article
%K COVID-19
%K Electronic Health Records
%K Laboratory tests
%K Missing data
%K Multi-site health data
%A Amelia Tan
%A Emily Getzen
%A Meghan Hutch
%A Zachary Strasser
%A Alba Gutiérrez-Sacristán
%A Trang Le
%A Arianna Dagliati
%A Michele Morris
%A David Hanauer
%A Bertrand Moal
%A Clara-Lea Bonzel
%A William Yuan
%A Lorenzo Chiudinelli
%A Priam Das
%A Harrison Zhang
%A Bruce Aronow
%A Paul Avillach
%A Gabriel Brat
%A Tianxi Cai
%A Chuan Hong
%A William La Cava
%A He Loh
%A Yuan Luo
%A Shawn Murphy
%A Kee Hgiam
%A Gilbert Omenn
%A Lav Patel
%A Malarkodi Samayamuthu
%A Emily Shriver
%A Zahra Abad
%A Byorn Tan
%A Shyam Visweswaran
%A Xuan Wang
%A Griffin Weber
%A Zongqi Xia
%A Bertrand Verdy
%A COVID-19 EHR
%A Qi Long
%A Danielle Mowery
%A John Holmes
%B Journal of biomedical informatics
%D 2023
%G eng
%P 104306
%R 10.1016/j.jbi.2023.104306
%T Informative missingness: What can we learn from patterns in missing laboratory data in the electronic health record?
%V 139
%1 https://www.ncbi.nlm.nih.gov/pubmed/36738870
%8 03/2023
%X <p><b>BACKGROUND: </b>In electronic health records, patterns of missing laboratory test results could capture patients' course of disease as well as ​​reflect clinician's concerns or worries for possible conditions. These patterns are often understudied and overlooked. This study aims to identify informative patterns of missingness among laboratory data collected across 15 healthcare system sites in three countries for COVID-19 inpatients.</p><p><b>METHODS: </b>We collected and analyzed demographic, diagnosis, and laboratory data for 69,939 patients with positive COVID-19 PCR tests across three countries from 1 January 2020 through 30 September 2021. We analyzed missing laboratory measurements across sites, missingness stratification by demographic variables, temporal trends of missingness, correlations between labs based on missingness indicators over time, and clustering of groups of labs based on their missingness/ordering pattern.</p><p><b>RESULTS: </b>With these analyses, we identified mapping issues faced in seven out of 15 sites. We also identified nuances in data collection and variable definition for the various sites. Temporal trend analyses may support the use of laboratory test result missingness patterns in identifying severe COVID-19 patients. Lastly, using missingness patterns, we determined relationships between various labs that reflect clinical behaviors.</p><p><b>CONCLUSION: </b>In this work, we use computational approaches to relate missingness patterns to hospital treatment capacity and highlight the heterogeneity of looking at COVID-19 over time and at multiple sites, where there might be different phases, policies, etc. Changes in missingness could suggest a change in a patient's condition, and patterns of missingness among laboratory measurements could potentially identify clinical outcomes. This allows sites to consider missing data as informative to analyses and help researchers identify which sites are better poised to study particular questions.</p>

%0 Journal Article
%K Acute kidney injury
%K COVID-19
%K Chronic kidney disease
%K Electronic Health Records
%K SARS-CoV-2
%A Byorn Tan
%A Bryce Tan
%A Amelia Tan
%A Emily Schriver
%A Alba Gutiérrez-Sacristán
%A Priyam Das
%A William Yuan
%A Meghan Hutch
%A Noelia Barrio
%A Miguel Jimenez
%A Noor Abu-El-Rub
%A Michele Morris
%A Bertrand Moal
%A Guillaume Verdy
%A Kelly Cho
%A Yuk-Lam Ho
%A Lav Patel
%A Arianna Dagliati
%A Antoine Neuraz
%A Jeffrey Klann
%A Andrew South
%A Shyam Visweswaran
%A David Hanauer
%A Sarah Maidlow
%A Mei Liu
%A Danielle Mowery
%A Ashley Batugo
%A Adeline Makoudjou
%A Patric Tippmann
%A Daniela Zöller
%A Gabriel Brat
%A Yuan Luo
%A Paul Avillach
%A Riccardo Bellazzi
%A Luca Chiovato
%A Alberto Malovini
%A Valentina Tibollo
%A Malarkodi Samayamuthu
%A Pablo Balazote
%A Zongqi Xia
%A Ne Loh
%A Lorenzo Chiudinelli
%A Clara-Lea Bonzel
%A Chuan Hong
%A Harrison Zhang
%A Griffin Weber
%A Isaac Kohane
%A Tianxi Cai
%A Gilbert Omenn
%A John Holmes
%A Kee Ngiam
%A Consortium EHR
%B EClinicalMedicine
%D 2023
%G eng
%P 101724
%R 10.1016/j.eclinm.2022.101724
%T Long-term kidney function recovery and mortality after COVID-19-associated acute kidney injury: An international multi-centre observational cohort study.
%V 55
%1 https://www.ncbi.nlm.nih.gov/pubmed/36381999
%8 01/2023
%X <p><b>BACKGROUND: </b>While acute kidney injury (AKI) is a common complication in COVID-19, data on post-AKI kidney function recovery and the clinical factors associated with poor kidney function recovery is lacking.</p><p><b>METHODS: </b>A retrospective multi-centre observational cohort study comprising 12,891 hospitalized patients aged 18 years or older with a diagnosis of SARS-CoV-2 infection confirmed by polymerase chain reaction from 1 January 2020 to 10 September 2020, and with at least one serum creatinine value 1-365 days prior to admission. Mortality and serum creatinine values were obtained up to 10 September 2021.</p><p><b>FINDINGS: </b>Advanced age (HR 2.77, 95%CI 2.53-3.04, p < 0.0001), severe COVID-19 (HR 2.91, 95%CI 2.03-4.17, p < 0.0001), severe AKI (KDIGO stage 3: HR 4.22, 95%CI 3.55-5.00, p < 0.0001), and ischemic heart disease (HR 1.26, 95%CI 1.14-1.39, p < 0.0001) were associated with worse mortality outcomes. AKI severity (KDIGO stage 3: HR 0.41, 95%CI 0.37-0.46, p < 0.0001) was associated with worse kidney function recovery, whereas remdesivir use (HR 1.34, 95%CI 1.17-1.54, p < 0.0001) was associated with better kidney function recovery. In a subset of patients without chronic kidney disease, advanced age (HR 1.38, 95%CI 1.20-1.58, p < 0.0001), male sex (HR 1.67, 95%CI 1.45-1.93, p < 0.0001), severe AKI (KDIGO stage 3: HR 11.68, 95%CI 9.80-13.91, p < 0.0001), and hypertension (HR 1.22, 95%CI 1.10-1.36, p = 0.0002) were associated with post-AKI kidney function impairment. Furthermore, patients with COVID-19-associated AKI had significant and persistent elevations of baseline serum creatinine 125% or more at 180 days (RR 1.49, 95%CI 1.32-1.67) and 365 days (RR 1.54, 95%CI 1.21-1.96) compared to COVID-19 patients with no AKI.</p><p><b>INTERPRETATION: </b>COVID-19-associated AKI was associated with higher mortality, and severe COVID-19-associated AKI was associated with worse long-term post-AKI kidney function recovery.</p><p><b>FUNDING: </b>Authors are supported by various funders, with full details stated in the acknowledgement section.</p>

%0 Journal Article
%A Bertrand Moal
%A Arthur Orieux
%A Thomas Ferté
%A Antoine Neuraz
%A Gabriel Brat
%A Paul Avillach
%A Clara-Lea Bonzel
%A Tianxi Cai
%A Kelly Cho
%A Sébastien Cossin
%A Romain Griffier
%A David Hanauer
%A Christian Haverkamp
%A Yuk-Lam Ho
%A Chuan Hong
%A Meghan Hutch
%A Jeffrey Klann
%A Trang Le
%A Ne Loh
%A Yuan Luo
%A Adeline Makoudjou
%A Michele Morris
%A Danielle Mowery
%A Karen Olson
%A Lav Patel
%A Malarkodi Samayamuthu
%A Fernando Vidorreta
%A Emily Schriver
%A Petra Schubert
%A Guillaume Verdy
%A Shyam Visweswaran
%A Xuan Wang
%A Griffin Weber
%A Zongqi Xia
%A William Yuan
%A Harrison Zhang
%A Daniela Zöller
%A Isaac Kohane
%A Consortium EHR
%A Alexandre Boyer
%A Vianney Jouhet
%B PloS one
%D 2023
%G eng
%N 1
%P e0266985
%R 10.1371/journal.pone.0266985
%T Acute respiratory distress syndrome after SARS-CoV-2 infection on young adult population: International observational federated study based on electronic health records through the 4CE consortium.
%V 18
%1 https://www.ncbi.nlm.nih.gov/pubmed/36598895
%8 01/2023
%X <p><b>PURPOSE: </b>In young adults (18 to 49 years old), investigation of the acute respiratory distress syndrome (ARDS) after severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection has been limited. We evaluated the risk factors and outcomes of ARDS following infection with SARS-CoV-2 in a young adult population.</p><p><b>METHODS: </b>A retrospective cohort study was conducted between January 1st, 2020 and February 28th, 2021 using patient-level electronic health records (EHR), across 241 United States hospitals and 43 European hospitals participating in the Consortium for Clinical Characterization of COVID-19 by EHR (4CE). To identify the risk factors associated with ARDS, we compared young patients with and without ARDS through a federated analysis. We further compared the outcomes between young and old patients with ARDS.</p><p><b>RESULTS: </b>Among the 75,377 hospitalized patients with positive SARS-CoV-2 PCR, 1001 young adults presented with ARDS (7.8% of young hospitalized adults). Their mortality rate at 90 days was 16.2% and they presented with a similar complication rate for infection than older adults with ARDS. Peptic ulcer disease, paralysis, obesity, congestive heart failure, valvular disease, diabetes, chronic pulmonary disease and liver disease were associated with a higher risk of ARDS. We described a high prevalence of obesity (53%), hypertension (38%- although not significantly associated with ARDS), and diabetes (32%).</p><p><b>CONCLUSION: </b>Trough an innovative method, a large international cohort study of young adults developing ARDS after SARS-CoV-2 infection has been gather. It demonstrated the poor outcomes of this population and associated risk factor.</p>

%0 Journal Article
%A Bertrand Moal
%A Arthur Orieux
%A Thomas Ferté
%A Antoine Neuraz
%A Gabriel Brat
%A Paul Avillach
%A Clara-Lea Bonzel
%A Tianxi Cai
%A Kelly Cho
%A Sébastien Cossin
%A Romain Griffier
%A David Hanauer
%A Christian Haverkamp
%A Yuk-Lam Ho
%A Chuan Hong
%A Meghan Hutch
%A Jeffrey Klann
%A Trang Le
%A Ne Loh
%A Yuan Luo
%A Adeline Makoudjou
%A Michele Morris
%A Danielle Mowery
%A Karen Olson
%A Lav Patel
%A Malarkodi Samayamuthu
%A Fernando Vidorreta
%A Emily Schriver
%A Petra Schubert
%A Guillaume Verdy
%A Shyam Visweswaran
%A Xuan Wang
%A Griffin Weber
%A Zongqi Xia
%A William Yuan
%A Harrison Zhang
%A Daniela Zöller
%A Isaac Kohane
%A Consortium EHR
%A Alexandre Boyer
%A Vianney Jouhet
%B PloS one
%D 2023
%G eng
%N 1
%P e0266985
%R 10.1371/journal.pone.0266985
%T Acute respiratory distress syndrome after SARS-CoV-2 infection on young adult population: International observational federated study based on electronic health records through the 4CE consortium.
%V 18
%1 https://www.ncbi.nlm.nih.gov/pubmed/36598895
%8 01/2023
%X <p><b>PURPOSE: </b>In young adults (18 to 49 years old), investigation of the acute respiratory distress syndrome (ARDS) after severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection has been limited. We evaluated the risk factors and outcomes of ARDS following infection with SARS-CoV-2 in a young adult population.</p><p><b>METHODS: </b>A retrospective cohort study was conducted between January 1st, 2020 and February 28th, 2021 using patient-level electronic health records (EHR), across 241 United States hospitals and 43 European hospitals participating in the Consortium for Clinical Characterization of COVID-19 by EHR (4CE). To identify the risk factors associated with ARDS, we compared young patients with and without ARDS through a federated analysis. We further compared the outcomes between young and old patients with ARDS.</p><p><b>RESULTS: </b>Among the 75,377 hospitalized patients with positive SARS-CoV-2 PCR, 1001 young adults presented with ARDS (7.8% of young hospitalized adults). Their mortality rate at 90 days was 16.2% and they presented with a similar complication rate for infection than older adults with ARDS. Peptic ulcer disease, paralysis, obesity, congestive heart failure, valvular disease, diabetes, chronic pulmonary disease and liver disease were associated with a higher risk of ARDS. We described a high prevalence of obesity (53%), hypertension (38%- although not significantly associated with ARDS), and diabetes (32%).</p><p><b>CONCLUSION: </b>Trough an innovative method, a large international cohort study of young adults developing ARDS after SARS-CoV-2 infection has been gather. It demonstrated the poor outcomes of this population and associated risk factor.</p>

%0 Journal Article
%A Alba Gutiérrez-Sacristán
%A Arnaud Serret-Larmande
%A Meghan Hutch
%A Carlos Saez
%A Bruce Aronow
%A Surbhi Bhatnagar
%A Clara-Lea Bonzel
%A Tianxi Cai
%A Batsal Devkota
%A David Hanauer
%A Ne Loh
%A Yuan Luo
%A Bertrand Moal
%A Taha Ahooyi
%A Wanjiku Njoroge
%A Gilbert Omenn
%A Nelson Sanchez-Pinto
%A Andrew South
%A Francesca Sperotto
%A Amelia Tan
%A Deanne Taylor
%A Guillaume Verdy
%A Shyam Visweswaran
%A Zongqi Xia
%A Janet Zahner
%A Paul Avillach
%A Florence Bourgeois
%A Consortium EHR
%B JAMA network open
%D 2022
%G eng
%N 12
%P e2246548
%R 10.1001/jamanetworkopen.2022.46548
%T Hospitalizations Associated With Mental Health Conditions Among Adolescents in the US and France During the COVID-19 Pandemic.
%V 5
%1 https://www.ncbi.nlm.nih.gov/pubmed/36512353
%8 12/2022
%X <p><b>IMPORTANCE: </b>The COVID-19 pandemic has been associated with an increase in mental health diagnoses among adolescents, though the extent of the increase, particularly for severe cases requiring hospitalization, has not been well characterized. Large-scale federated informatics approaches provide the ability to efficiently and securely query health care data sets to assess and monitor hospitalization patterns for mental health conditions among adolescents.</p><p><b>OBJECTIVE: </b>To estimate changes in the proportion of hospitalizations associated with mental health conditions among adolescents following onset of the COVID-19 pandemic.</p><p><b>DESIGN, SETTING, AND PARTICIPANTS: </b>This retrospective, multisite cohort study of adolescents 11 to 17 years of age who were hospitalized with at least 1 mental health condition diagnosis between February 1, 2019, and April 30, 2021, used patient-level data from electronic health records of 8 children's hospitals in the US and France.</p><p><b>MAIN OUTCOMES AND MEASURES: </b>Change in the monthly proportion of mental health condition-associated hospitalizations between the prepandemic (February 1, 2019, to March 31, 2020) and pandemic (April 1, 2020, to April 30, 2021) periods using interrupted time series analysis.</p><p><b>RESULTS: </b>There were 9696 adolescents hospitalized with a mental health condition during the prepandemic period (5966 [61.5%] female) and 11 101 during the pandemic period (7603 [68.5%] female). The mean (SD) age in the prepandemic cohort was 14.6 (1.9) years and in the pandemic cohort, 14.7 (1.8) years. The most prevalent diagnoses during the pandemic were anxiety (6066 [57.4%]), depression (5065 [48.0%]), and suicidality or self-injury (4673 [44.2%]). There was an increase in the proportions of monthly hospitalizations during the pandemic for anxiety (0.55%; 95% CI, 0.26%-0.84%), depression (0.50%; 95% CI, 0.19%-0.79%), and suicidality or self-injury (0.38%; 95% CI, 0.08%-0.68%). There was an estimated 0.60% increase (95% CI, 0.31%-0.89%) overall in the monthly proportion of mental health-associated hospitalizations following onset of the pandemic compared with the prepandemic period.</p><p><b>CONCLUSIONS AND RELEVANCE: </b>In this cohort study, onset of the COVID-19 pandemic was associated with increased hospitalizations with mental health diagnoses among adolescents. These findings support the need for greater resources within children's hospitals to care for adolescents with mental health conditions during the pandemic and beyond.</p>

%0 Journal Article
%A Xuan Wang
%A Harrison Zhang
%A Xin Xiong
%A Chuan Hong
%A Griffin Weber
%A Gabriel Brat
%A Clara-Lea Bonzel
%A Yuan Luo
%A Rui Duan
%A Nathan Palmer
%A Meghan Hutch
%A Alba Gutiérrez-Sacristán
%A Riccardo Bellazzi
%A Luca Chiovato
%A Kelly Cho
%A Arianna Dagliati
%A Hossein Estiri
%A Noelia García-Barrio
%A Romain Griffier
%A David Hanauer
%A Yuk-Lam Ho
%A John Holmes
%A Mark Keller
%A Jeffrey MEng
%A Sehi L'Yi
%A Sara Lozano-Zahonero
%A Sarah Maidlow
%A Adeline Makoudjou
%A Alberto Malovini
%A Bertrand Moal
%A Jason Moore
%A Michele Morris
%A Danielle Mowery
%A Shawn Murphy
%A Antoine Neuraz
%A Kee Ngiam
%A Gilbert Omenn
%A Lav Patel
%A Miguel Pedrera-Jiménez
%A Andrea Prunotto
%A Malarkodi Samayamuthu
%A Fernando Vidorreta
%A Emily Schriver
%A Petra Schubert
%A Pablo Serrano-Balazote
%A Andrew South
%A Amelia Tan
%A Byorn Tan
%A Valentina Tibollo
%A Patric Tippmann
%A Shyam Visweswaran
%A Zongqi Xia
%A William Yuan
%A Daniela Zöller
%A Isaac Kohane
%A Paul Avillach
%A Zijian Guo
%A Tianxi Cai
%A Consortium 4CE
%B Journal of biomedical informatics
%D 2022
%G eng
%P 104176
%R 10.1016/j.jbi.2022.104176
%T SurvMaximin: Robust federated approach to transporting survival risk prediction models.
%V 134
%1 https://www.ncbi.nlm.nih.gov/pubmed/36007785
%8 10/2022
%X <p><b>OBJECTIVE: </b>For multi-center heterogeneous Real-World Data (RWD) with time-to-event outcomes and high-dimensional features, we propose the SurvMaximin algorithm to estimate Cox model feature coefficients for a target population by borrowing summary information from a set of health care centers without sharing patient-level information.</p><p><b>MATERIALS AND METHODS: </b>For each of the centers from which we want to borrow information to improve the prediction performance for the target population, a penalized Cox model is fitted to estimate feature coefficients for the center. Using estimated feature coefficients and the covariance matrix of the target population, we then obtain a SurvMaximin estimated set of feature coefficients for the target population. The target population can be an entire cohort comprised of all centers, corresponding to federated learning, or a single center, corresponding to transfer learning.</p><p><b>RESULTS: </b>Simulation studies and a real-world international electronic health records application study, with 15 participating health care centers across three countries (France, Germany, and the U.S.), show that the proposed SurvMaximin algorithm achieves comparable or higher accuracy compared with the estimator using only the information of the target site and other existing methods. The SurvMaximin estimator is robust to variations in sample sizes and estimated feature coefficients between centers, which amounts to significantly improved estimates for target sites with fewer observations.</p><p><b>CONCLUSIONS: </b>The SurvMaximin method is well suited for both federated and transfer learning in the high-dimensional survival analysis setting. SurvMaximin only requires a one-time summary information exchange from participating centers. Estimated regression vectors can be very heterogeneous. SurvMaximin provides robust Cox feature coefficient estimates without outcome information in the target population and is privacy-preserving.</p>

%0 Journal Article
%A Arnaud Serret-Larmande
%A Jonathan Kaltman
%A Paul Avillach
%B JAMIA Open
%D 2022
%G eng
%N 1
%T Streamlining statistical reproducibility: NHLBI ORCHID clinical trial results reproduction
%U https://doi.org/10.1093/jamiaopen/ooac001
%V 5
%X Reproducibility in medical research has been a long-standing issue. More recently, the COVID-19 pandemic has publicly underlined this fact as the retraction of several studies reached out to general media audiences. A significant number of these retractions occurred after in-depth scrutiny of the methodology and results by the scientific community. Consequently, these retractions have undermined confidence in the peer-review process, which is not considered sufficiently reliable to generate trust in the published results. This partly stems from opacity in published results, the practical implementation of the statistical analysis often remaining undisclosed. We present a workflow that uses a combination of informatics tools to foster statistical reproducibility: an open-source programming language, Jupyter Notebook, cloud-based data repository, and an application programming interface can streamline an analysis and help to kick-start new analyses. We illustrate this principle by (1) reproducing the results of the ORCHID clinical trial, which evaluated the efficacy of hydroxychloroquine in COVID-19 patients, and (2) expanding on the analyses conducted in the original trial by investigating the association of premedication with biological laboratory results. Such workflows will be encouraged for future publications from National Heart, Lung, and Blood Institute-funded studies.

%0 Journal Article
%A FT Bourgeois
%A Gutiérrez-Sacristán
%A MS Keller
%A Liu
%A Hong
%A CL Bonzel
%A ALM Tan
%A BJ Aronow
%A Boeker
%A Booth
%A Rojo Cruz
%A Devkota
%A García Barrio
%A Geva
%A DA Hanauer
%A MR Hutch
%A RW Issitt
%A JG Klann
%A Luo
%A KD Mandl
%A Mao
%A Moal
%A KL Moshal
%A SN Murphy
%A Neuraz
%A KY Ngiam
%A GS Omenn
%A LP Patel
%A MP Jiménez
%A NJ Sebire
%A PS Balazote
%A Serret-Larmande
%A AM South
%A Spiridou
%A Taylor
%A Tippmann
%A Visweswaran
%A GM Weber
%A IS Kohane
%A Cai
%A Avillach
%B JAMA Network Open
%D 2021
%G eng
%T International Analysis of Electronic Health Records of Children and Youth Hospitalized With COVID-19 Infection in 6 Countries
%U https://pubmed.ncbi.nlm.nih.gov/34115127/
%X <div class="abstract" id="abstract" style='box-sizing:inherit;margin:2.4rem0px;word-break:break-word;position:relative;color:rgb(33,33,33);BlinkMacSystemFont,-apple-system,"SegoeUI",Roboto,Oxygen,Ubuntu,Cantarell,"FiraSans","DroidSans","HelveticaNeue",sans-serif;16px;font-style:normal;font-variant-ligatures:normal;font-variant-caps:normal;font-weight:400;letter-spacing:normal;orphans:2;text-align:start;text-indent:0px;text-transform:none;white-space:normal;widows:2;word-spacing:0px;-webkit-text-stroke-width:0px;background-color:rgb(255,255,255);text-decoration-thickness:initial;text-decoration-style:initial;text-decoration-color:initial;'>	<h2 class="title" style='box-sizing:inherit;clear:both;Merriweather,Georgia,Cambria,"TimesNewRoman",Times,serif;line-height:2.4rem;margin-bottom:1.2rem;margin-top:3.6rem;2rem;font-weight:700;'>		Abstract	</h2>	<div class="abstract-content selected" id="enc-abstract" style="box-sizing:inherit;clear:left;display:block;">		<p style="box-sizing:inherit;line-height:1.5;margin:1.2rem0px;">			<strong class="sub-title" style="box-sizing:inherit;font-weight:bold;">Importance:<span> </span></strong>Additional sources of pediatric epidemiological and clinical data are needed to efficiently study COVID-19 in children and youth and inform infection prevention and clinical treatment of pediatric patients.		</p>		<p style="box-sizing:inherit;line-height:1.5;margin:1.2rem0px;">			<strong class="sub-title" style="box-sizing:inherit;font-weight:bold;">Objective:<span> </span></strong>To describe international hospitalization trends and key epidemiological and clinical features of children and youth with COVID-19.		</p>		<p style="box-sizing:inherit;line-height:1.5;margin:1.2rem0px;">			<strong class="sub-title" style="box-sizing:inherit;font-weight:bold;">Design, setting, and participants:<span> </span></strong>This retrospective cohort study included pediatric patients hospitalized between February 2 and October 10, 2020. Patient-level electronic health record (EHR) data were collected across 27 hospitals in France, Germany, Spain, Singapore, the UK, and the US. Patients younger than 21 years who tested positive for COVID-19 and were hospitalized at an institution participating in the Consortium for Clinical Characterization of COVID-19 by EHR were included in the study.		</p>		<p style="box-sizing:inherit;line-height:1.5;margin:1.2rem0px;">			<strong class="sub-title" style="box-sizing:inherit;font-weight:bold;">Main outcomes and measures:<span> </span></strong>Patient characteristics, clinical features, and medication use.		</p>		<p style="box-sizing:inherit;line-height:1.5;margin:1.2rem0px;">			<strong class="sub-title" style="box-sizing:inherit;font-weight:bold;">Results:<span> </span></strong>There were 347 males (52%; 95% CI, 48.5-55.3) and 324 females (48%; 95% CI, 44.4-51.3) in this study's cohort. There was a bimodal age distribution, with the greatest proportion of patients in the 0- to 2-year (199 patients [30%]) and 12- to 17-year (170 patients [25%]) age range. Trends in hospitalizations for 671 children and youth found discrete surges with variable timing across 6 countries. Data from this cohort mirrored national-level pediatric hospitalization trends for most countries with available data, with peaks in hospitalizations during the initial spring surge occurring within 23 days in the national-level and 4CE data. A total of 27 364 laboratory values for 16 laboratory tests were analyzed, with mean values indicating elevations in markers of inflammation (C-reactive protein, 83 mg/L; 95% CI, 53-112 mg/L; ferritin, 417 ng/mL; 95% CI, 228-607 ng/mL; and procalcitonin, 1.45 ng/mL; 95% CI, 0.13-2.77 ng/mL). Abnormalities in coagulation were also evident (D-dimer, 0.78 ug/mL; 95% CI, 0.35-1.21 ug/mL; and fibrinogen, 477 mg/dL; 95% CI, 385-569 mg/dL). Cardiac troponin, when checked (n = 59), was elevated (0.032 ng/mL; 95% CI, 0.000-0.080 ng/mL). Common complications included cardiac arrhythmias (15.0%; 95% CI, 8.1%-21.7%), viral pneumonia (13.3%; 95% CI, 6.5%-20.1%), and respiratory failure (10.5%; 95% CI, 5.8%-15.3%). Few children were treated with COVID-19-directed medications.		</p>		<p style="box-sizing:inherit;line-height:1.5;margin:1.2rem0px;">			<strong class="sub-title" style="box-sizing:inherit;font-weight:bold;">Conclusions and relevance:<span> </span></strong>This study of EHRs of children and youth hospitalized for COVID-19 in 6 countries demonstrated variability in hospitalization trends across countries and identified common complications and laboratory abnormalities in children and youth with COVID-19 infection. Large-scale informatics-based approaches to integrate and analyze data across health care systems complement methods of disease surveillance and advance understanding of epidemiological and clinical features associated with COVID-19 in children and youth.		</p>	</div></div><div class="conflict-of-interest" id="conflict-of-interest" style='box-sizing:inherit;margin:2.4rem0px;word-break:break-word;position:relative;overflow-wrap:break-word;color:rgb(33,33,33);BlinkMacSystemFont,-apple-system,"SegoeUI",Roboto,Oxygen,Ubuntu,Cantarell,"FiraSans","DroidSans","HelveticaNeue",sans-serif;16px;font-style:normal;font-variant-ligatures:normal;font-variant-caps:normal;font-weight:400;letter-spacing:normal;orphans:2;text-align:start;text-indent:0px;text-transform:none;white-space:normal;widows:2;word-spacing:0px;-webkit-text-stroke-width:0px;background-color:rgb(255,255,255);text-decoration-thickness:initial;text-decoration-style:initial;text-decoration-color:initial;'>	 </div>

%0 Journal Article
%A AG Feroe
%A Uppal
%A Gutiérrez-Sacristán
%A Mousavi
%A Greenspun
%A Surati
%A IS Kohane
%A Avillach
%B JAMA Pediatrics
%D 2021
%G eng
%T Medication Use in the Management of Comorbidities Among Individuals With Autism Spectrum Disorder From a Large Nationwide Insurance Database.
%U https://pubmed.ncbi.nlm.nih.gov/34097007/
%X <div class="abstract" id="abstract">	<h2 class="title">		Abstract	</h2>	<div class="abstract-content selected" id="enc-abstract">		<p>			<strong class="sub-title">Importance: </strong> Although there is no pharmacological treatment for autism spectrum disorder (ASD) itself, behavioral and pharmacological therapies have been used to address its symptoms and common comorbidities. A better understanding of the medications used to manage comorbid conditions in this growing population is critical; however, most previous efforts have been limited in size, duration, and lack of broad representation.		</p>		<p>			<strong class="sub-title">Objective: </strong> To use a nationally representative database to uncover trends in the prevalence of co-occurring conditions and medication use in the management of symptoms and comorbidities over time among US individuals with ASD.		</p>		<p>			<strong class="sub-title">Design, setting, and participants: </strong> This retrospective, population-based cohort study mined a nationwide, managed health plan claims database containing more than 86 million unique members. Data from January 1, 2014, to December 31, 2019, were used to analyze prescription frequency and diagnoses of comorbidities. A total of 26 722 individuals with ASD who had been prescribed at least 1 of 24 medications most commonly prescribed to treat ASD symptoms or comorbidities during the 6-year study period were included in the analysis.		</p>		<p>			<strong class="sub-title">Exposures: </strong> Diagnosis codes for ASD based on International Classification of Diseases, Ninth Revision, and International Statistical Classification of Diseases and Related Health Problems, Tenth Revision.		</p>		<p>			<strong class="sub-title">Main outcomes and measures: </strong> Quantitative estimates of prescription frequency for the 24 most commonly prescribed medications among the study cohort and the most common comorbidities associated with each medication in this population.		</p>		<p>			<strong class="sub-title">Results: </strong> Among the 26 722 individuals with ASD included in the analysis (77.7% male; mean [SD] age, 14.45 [9.40] years), polypharmacy was common, ranging from 28.6% to 31.5%. Individuals' prescription regimens changed frequently within medication classes, rather than between classes. The prescription frequency of a specific medication varied considerably, depending on the coexisting diagnosis of a given comorbidity. Of the 24 medications assessed, 15 were associated with at least a 15% prevalence of a mood disorder, and 11 were associated with at least a 15% prevalence of attention-deficit/hyperactivity disorder. For patients taking antipsychotics, the 2 most common comorbidities were combined type attention-deficit/hyperactivity disorder (11.6%-17.8%) and anxiety disorder (13.1%-30.1%).		</p>		<p>			<strong class="sub-title">Conclusions and relevance: </strong> This study demonstrated considerable variability and transiency in the use of prescription medications by US clinicians to manage symptoms and comorbidities associated with ASD. These findings support the importance of early and ongoing surveillance of patients with ASD and co-occurring conditions and offer clinicians insight on the targeted therapies most commonly used to manage co-occurring conditions. Future research and policy efforts are critical to assess the extent to which pharmacological management of comorbidities affects quality of life and functioning in patients with ASD while continuing to optimize clinical guidelines, to ensure effective care for this growing population.		</p>	</div></div>

%0 Journal Article
%K FAIR data
%K common data elements
%K data standards
%K ontology
%K open science
%K rare disease patients
%A Yaffa Rubinstein
%A Peter Robinson
%A William Gahl
%A Paul Avillach
%A Gareth Baynam
%A Helene Cederroth
%A Rebecca Goodwin
%A Stephen Groft
%A Mats Hansson
%A Nomi Harris
%A Vojtech Huser
%A Deborah Mascalzoni
%A Julie McMurry
%A Matthew Might
%A Christoffer Nellaker
%A Barend Mons
%A Dina Paltoo
%A Jonathan Pevsner
%A Manuel Posada
%A Alison Rockett-Frase
%A Marco Roos
%A Tamar Rubinstein
%A Domenica Taruscio
%A Esther van Enckevort
%A Melissa Haendel
%B JAMIA open
%D 2020
%G eng
%N 3
%P 472-486
%R 10.1093/jamiaopen/ooaa030
%T The case for open science: rare diseases.
%V 3
%1 https://www.ncbi.nlm.nih.gov/pubmed/33426479
%8 10/2020
%X <p>The premise of Open Science is that research and medical management will progress faster if data and knowledge are openly shared. The value of Open Science is nowhere more important and appreciated than in the rare disease (RD) community. Research into RDs has been limited by insufficient patient data and resources, a paucity of trained disease experts, and lack of therapeutics, leading to long delays in diagnosis and treatment. These issues can be ameliorated by following the principles and practices of sharing that are intrinsic to Open Science. Here, we describe how the RD community has adopted the core pillars of Open Science, adding new initiatives to promote care and research for RD patients and, ultimately, for all of medicine. We also present recommendations that can advance Open Science more globally.</p>

%0 Journal Article
%A Alba Gutiérrez-Sacristán
%A Carlos De Niz
%A Cartik Kothari
%A Sek Won Kong
%A Kenneth Mandl
%A Paul Avillach
%B Brief Bioinform
%D 2020
%G eng
%R 10.1093/bib/bbaa033
%T GenoPheno: cataloging large-scale phenotypic and next-generation sequencing data within human datasets
%1 http://www.ncbi.nlm.nih.gov/pubmed/32249310?dopt=Abstract
%8 04/2020
%X Precision medicine promises to revolutionize treatment, shifting therapeutic approaches from the classical one-size-fits-all to those more tailored to the patient's individual genomic profile, lifestyle and environmental exposures. Yet, to advance precision medicine's main objective-ensuring the optimum diagnosis, treatment and prognosis for each individual-investigators need access to large-scale clinical and genomic data repositories. Despite the vast proliferation of these datasets, locating and obtaining access to many remains a challenge. We sought to provide an overview of available patient-level datasets that contain both genotypic data, obtained by next-generation sequencing, and phenotypic data-and to create a dynamic, online catalog for consultation, contribution and revision by the research community. Datasets included in this review conform to six specific inclusion parameters that are: (i) contain data from more than 500 human subjects; (ii) contain both genotypic and phenotypic data from the same subjects; (iii) include whole genome sequencing or whole exome sequencing data; (iv) include at least 100 recorded phenotypic variables per subject; (v) accessible through a website or collaboration with investigators and (vi) make access information available in English. Using these criteria, we identified 30 datasets, reviewed them and provided results in the release version of a catalog, which is publicly available through a dynamic Web application and on GitHub. Users can review as well as contribute new datasets for inclusion (Web: https://avillachlab.shinyapps.io/genophenocatalog/; GitHub: https://github.com/hms-dbmi/GenoPheno-CatalogShiny).

%0 Journal Article
%A Danielle Newby
%A Daniel Prieto-Alhambra
%A Talita Duarte-Salles
%A David Ansell
%A Lars Pedersen
%A van der Johan Lei
%A Mees Mosseveld
%A Peter Rijnbeek
%A Glen James
%A Myriam Alexander
%A Peter Egger
%A Jana Podhorna
%A Robert Stewart
%A Gayan Perera
%A Paul Avillach
%A Solène Grosdidier
%A Simon Lovestone
%A Alejo Nevado-Holgado
%B Alzheimers Res Ther
%D 2020
%G eng
%N 1
%P 38
%R 10.1186/s13195-020-00606-5
%T Methotrexate and relative risk of dementia amongst patients with rheumatoid arthritis: a multi-national multi-database case-control study
%V 12
%1 http://www.ncbi.nlm.nih.gov/pubmed/32252806?dopt=Abstract
%8 04/2020
%X BACKGROUND: Inflammatory processes have been shown to play a role in dementia. To understand this role, we selected two anti-inflammatory drugs (methotrexate and sulfasalazine) to study their association with dementia risk.METHODS: A retrospective matched case-control study of patients over 50 with rheumatoid arthritis (486 dementia cases and 641 controls) who were identified from electronic health records in the UK, Spain, Denmark and the Netherlands. Conditional logistic regression models were fitted to estimate the risk of dementia.RESULTS: Prior methotrexate use was associated with a lower risk of dementia (OR 0.71, 95% CI 0.52-0.98). Furthermore, methotrexate use with therapy longer than 4 years had the lowest risk of dementia (odds ratio 0.37, 95% CI 0.17-0.79). Sulfasalazine use was not associated with dementia (odds ratio 0.88, 95% CI 0.57-1.37).CONCLUSIONS: Further studies are still required to clarify the relationship between prior methotrexate use and duration as well as biological treatments with dementia risk.

%0 Journal Article
%A Grégoire Versmée
%A Laura Versmée
%A Mikaël Dusenne
%A Niloofar Jalali
%A Paul Avillach
%B Bioinformatics
%D 2020
%G eng
%N 4
%P 1305-1306
%R 10.1093/bioinformatics/btz680
%T dbgap2x: an R package to explore and extract data from the database of Genotypes and Phenotypes (dbGaP)
%V 36
%1 http://www.ncbi.nlm.nih.gov/pubmed/31504194?dopt=Abstract
%8 02/2020
%X SUMMARY: Based on the Genomic Data Sharing Policy issued in August 2007, the National Institutes of Health (NIH) has supported several repositories such as the database of Genotypes and Phenotypes (dbGaP). dbGaP is an online repository that provides access to large-scale genetic and phenotypic datasets with more than 1000 studies. However, navigating the website and understanding the relationship between the studies are not easy tasks. Moreover, the decryption of the files is a complex procedure. In this study we propose the dbgap2x R package that covers a broad range of functions for searching dbGaP studies, exploring the characteristics of a study and easily decrypting the files from dbGaP.AVAILABILITY AND IMPLEMENTATION: dbgap2x is an R package with the code available at https://github.com/gversmee/dbgap2x. A containerized version including the package, a Jupyter server and with a Notebook example is available at https://hub.docker.com/r/gversmee/dbgap2x.SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online.

%0 Journal Article
%A Yuan Luo
%A Alal Eran
%A Nathan Palmer
%A Paul Avillach
%A Ami Levy-Moonshine
%A Peter Szolovitis
%A Isaac Kohane
%B Nat Med
%D 2020
%G eng
%T A multidimensional precision medicine approach identifies an autism subtype characterized by dyslipidemia.
%U https://pubmed.ncbi.nlm.nih.gov/32778826/
%X <h2>	Abstract</h2><p>	The promise of precision medicine lies in data diversity. More than the sheer size of biomedical data, it is the layering of multiple data modalities, offering complementary perspectives, that is thought to enable the identification of patient subgroups with shared pathophysiology. In the present study, we use autism to test this notion. By combining healthcare claims, electronic health records, familial whole-exome sequences and neurodevelopmental gene expression patterns, we identified a subgroup of patients with dyslipidemia-associated autism.</p>

%0 Journal Article
%A Timothy Miller
%A Paul Avillach
%A Kenneth Mandl
%B JAMIA Open
%D 2020
%G eng
%N 2
%P 185-189
%T Experiences implementing scalable, containerized, cloud-based NLP for extracting biobank participant phenotypes at scale.
%U https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7382623/
%V 3
%X <h2>	Abstract</h2><p>	<strong>Objective: </strong>To develop scalable natural language processing (NLP) infrastructure for processing the free text in electronic health records (EHRs).</p><p>	<strong>Materials and methods: </strong>We extend the open-source Apache cTAKES NLP software with several standard technologies for scalability. We remove processing bottlenecks by monitoring component queue size. We process EHR free text for patients in the PrecisionLink Biobank at Boston Children's Hospital. The extracted concepts are made searchable via a web-based portal.</p><p>	<strong>Results: </strong>We processed over 1.2 million notes for over 8000 patients, extracting 154 million concepts. Our largest tested configuration processes over 1 million notes per day.</p><p>	<strong>Discussion: </strong>The unique information represented by extracted NLP concepts has great potential to provide a more complete picture of patient status.</p><p>	<strong>Conclusion: </strong>NLP large EHR document collections can be done efficiently, in service of high throughput phenotyping.</p>

%0 Journal Article
%A Carlos Saez
%A Alba Gutiérrez-Sacristán
%A Isaac Kohane
%A Juan Garcia-Gomez
%A Paul Avillach
%B Gigascience
%D 2020
%G eng
%T EHRtemporalVariability: delineating temporal data-set shifts in electronic health records.
%U https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7391413/
%X <h2>	Abstract</h2><p>	<strong>Background: </strong>Temporal variability in health-care processes or protocols is intrinsic to medicine. Such variability can potentially introduce dataset shifts, a data quality issue when reusing electronic health records (EHRs) for secondary purposes. Temporal data-set shifts can present as trends, as well as abrupt or seasonal changes in the statistical distributions of data over time. The latter are particularly complicated to address in multimodal and highly coded data. These changes, if not delineated, can harm population and data-driven research, such as machine learning. Given that biomedical research repositories are increasingly being populated with large sets of historical data from EHRs, there is a need for specific software methods to help delineate temporal data-set shifts to ensure reliable data reuse.</p><p>	<strong>Results: </strong>EHRtemporalVariability is an open-source R package and Shiny app designed to explore and identify temporal data-set shifts. EHRtemporalVariability estimates the statistical distributions of coded and numerical data over time; projects their temporal evolution through non-parametric information geometric temporal plots; and enables the exploration of changes in variables through data temporal heat maps. We demonstrate the capability of EHRtemporalVariability to delineate data-set shifts in three impact case studies, one of which is available for reproducibility.</p><p>	<strong>Conclusions: </strong>EHRtemporalVariability enables the exploration and identification of data-set shifts, contributing to the broad examination and repurposing of large, longitudinal data sets. Our goal is to help ensure reliable data reuse for a wide range of biomedical data users. EHRtemporalVariability is designed for technical users who are programmatically utilizing the R package, as well as users who are not familiar with programming via the Shiny user interface.Availability: https://github.com/hms-dbmi/EHRtemporalVariability/Reproducible vignette: https://cran.r-project.org/web/packages/EHRtemporalVariability/vignettes/EHRtemporalVariability.htmlOnline demo: http://ehrtemporalvariability.upv.es/.</p><p>	<strong>Keywords: </strong>R package; claims data; data quality; data-set shifts; electronic health records; information geometry; research repositories; scientific data sets; temporal variability; visual analytics.</p>

%0 Journal Article
%A Ines Krissaane
%A Carlos De Niz
%A Alba Gutiérrez-Sacristán
%A Gabor Korodi
%A Nneka Ede
%A Ranjay Kumar
%A Jessica Lyons
%A Arjun Manrai
%A Chirag Patel
%A Isaac Kohane
%A Paul Avillach
%B JAMIA
%D 2020
%G eng
%T Scalability and cost-effectiveness analysis of whole genome-wide association studies on Google Cloud Platform and Amazon Web Services.
%U https://academic.oup.com/jamia/advance-article/doi/10.1093/jamia/ocaa068/5876972
%X <h2>	Abstract</h2><p>	<strong>Objective: </strong>Advancements in human genomics have generated a surge of available data, fueling the growth and accessibility of databases for more comprehensive, in-depth genetic studies.</p><p>	<strong>Methods: </strong>We provide a straightforward and innovative methodology to optimize cloud configuration in order to conduct genome-wide association studies. We utilized Spark clusters on both Google Cloud Platform and Amazon Web Services, as well as Hail (http://doi.org/10.5281/zenodo.2646680) for analysis and exploration of genomic variants dataset.</p><p>	<strong>Results: </strong>Comparative evaluation of numerous cloud-based cluster configurations demonstrate a successful and unprecedented compromise between speed and cost for performing genome-wide association studies on 4 distinct whole-genome sequencing datasets. Results are consistent across the 2 cloud providers and could be highly useful for accelerating research in genetics.</p><p>	<strong>Conclusions: </strong>We present a timely piece for one of the most frequently asked questions when moving to the cloud: what is the trade-off between speed and cost?</p><p>	<strong>Keywords: </strong>cloud computing; distributed systems; genome-wide association study; whole genome.</p>

%0 Journal Article
%A Joany Zachariasse
%A Daan Nieboer
%A Ian Maconochie
%A Frank Smit
%A Claudio Alves
%A Susanne Greber-Platzer
%A Maria Tsolia
%A Ewout Steyerberg
%A Paul Avillach
%A van der Johan Lei
%A Henriette Moll
%B Lancet Child Adolesc Health
%D 2020
%G eng
%T Development and validation of a Paediatric Early Warning Score for use in the emergency department: a multicentre study.
%U https://www.sciencedirect.com/science/article/abs/pii/S2352464220301395?via%3Dihub
%X <p>	<strong>Background: </strong>Paediatric Early Warning Scores (PEWSs) are being used increasingly in hospital wards to identify children at risk of clinical deterioration, but few scores exist that were designed for use in emergency care settings. To improve the prioritisation of children in the emergency department (ED), we developed and validated an ED-PEWS.</p><p>	<strong>Methods: </strong>The TrIAGE project is a prospective European observational study based on electronic health record data collected between Jan 1, 2012, and Nov 1, 2015, from five diverse EDs in four European countries (Netherlands, the UK, Austria, and Portugal). This study included data from all consecutive ED visits of children under age 16 years. The main outcome measure was a three-category reference standard (high, intermediate, low urgency) that was developed as part of the TrIAGE project as a proxy for true patient urgency. The ED-PEWS was developed based on an ordinal logistic regression model, with cross-validation by setting. After completing the study, we fully externally validated the ED-PEWS in an independent cohort of febrile children from a different ED (Greece).</p><p>	<strong>Findings: </strong>Of 119 209 children, 2007 (1·7%) were of high urgency and 29 127 (24·4%) of intermediate urgency, according to our reference standard. We developed an ED-PEWS consisting of age and the predictors heart rate, respiratory rate, oxygen saturation, consciousness, capillary refill time, and work of breathing. The ED-PEWS showed a cross-validated c-statistic of 0·86 (95% prediction interval 0·82-0·90) for high-urgency patients and 0·67 (0·61-0·73) for high-urgency or intermediate-urgency patients. A cutoff of score of at least 15 was useful for identifying high-urgency patients with a specificity of 0·90 (95% CI 0·87-0·92) while a cutoff score of less than 6 was useful for identifying low-urgency patients with a sensitivity of 0·83 (0·81-0·85).</p><p>	<strong>Interpretation: </strong>The proposed ED-PEWS can assist in identifying high-urgency and low-urgency patients in the ED, and improves prioritisation compared with existing PEWSs.</p><p>	<strong>Funding: </strong>Stichting de Drie Lichten, Stichting Sophia Kinderziekenhuis Fonds, and the European Union's Horizon 2020 research and innovation programme.</p>

%0 Journal Article
%A Emmanuelle Sylvestre
%A Guillaume Bouzille
%A Michael McDuffie
%A Emmanuel Chazard
%A Paul Avillach
%A Marc Cuggia
%B Stud Health Technol Inform
%D 2020
%G eng
%T A Semi-Automated Approach for Multilingual Terminology Matching: Mapping the French Version of the ICD-10 to the ICD-10 CM
%U https://pubmed.ncbi.nlm.nih.gov/32570338/
%X <p>	The aim of this study was to develop a simple method to map the French International Statistical Classification of Diseases and Related Health Problems, 10th revision (ICD-10) with the International Classification of Diseases, 10th Revision, Clinical Modification (ICD-10 CM). We sought to map these terminologies forward (ICD-10 to ICD-10 CM) and backward (ICD-10 CM to ICD-10) and to assess the accuracy of these two mappings. We used several terminology resources such as the Unified Medical Language System (UMLS) Metathesaurus, Bioportal, the latest version available of the French ICD-10 and several official mapping files between different versions of the ICD-10. We first retrieved existing partial mapping between the ICD-10 and the ICD-10 CM. Then, we automatically matched the ICD-10 with the ICD-10-CM, using our different reference mapping files. Finally, we used manual review and natural language processing (NLP) to match labels between the two terminologies. We assessed the accuracy of both methods with a manual review of a random dataset from the results files. The overall matching was between 94.2 and 100%. The backward mapping was better than the forward one, especially regarding exact matches. In both cases, the NLP step was highly accurate. When there are no available experts from the ontology or NLP fields for multi-lingual ontology matching, this simple approach enables secondary reuse of Electronic Health Records (EHR) and billing data for research purposes in an international context.</p><p>	<strong>Keywords: </strong>Clinical terminologies; ICD-10; Interoperability; Multilingual matching.</p>

%0 Journal Article
%A Glen James
%A Estelle Collin
%A Marcus Lawrance
%A Achim Mueller
%A Jana Podhorna
%A Liliana Zaremba-Pechmann
%A Peter Rijnbeek
%A van der Johan Lei
%A Paul Avillach
%A Lars Pederson
%A David Ansell
%A Alessandro Pasqua
%A Mees Mosseveld
%A Solene Grosdidier
%A Usha Gungabissoon
%A Peter Egger
%A Robert Stewart
%A Carlos Celis-Morales
%A Myriam Alexander
%A Gerald Novak
%A Mark Forest Gordon
%B Soc Psychiatry Psychiatry Epidemiol. 
%D 2020
%G eng
%T Treatment pathway analysis of newly diagnosed dementia patients in four electronic health record databases in Europe
%U https://link.springer.com/article/10.1007%2Fs00127-020-01872-2
%X <p>	<strong>Purpose: </strong>Real-world studies to describe the use of first, second and third line therapies for the management and symptomatic treatment of dementia are lacking. This retrospective cohort study describes the first-, second- and third-line therapies used for the management and symptomatic treatment of dementia, and in particular Alzheimer's Disease.</p><p>	<strong>Methods: </strong>Medical records of patients with newly diagnosed dementia between 1997 and 2017 were collected using four databases from the UK, Denmark, Italy and the Netherlands.</p><p>	<strong>Results: </strong>We identified 191,933 newly diagnosed dementia patients in the four databases between 1997 and 2017 with 39,836 (IPCI (NL): 3281, HSD (IT): 1601, AUH (DK): 4474, THIN (UK): 30,480) fulfilling the inclusion criteria, and of these, 21,131 had received a specific diagnosis of Alzheimer's disease. The most common first line therapy initiated within a year (± 365 days) of diagnosis were Acetylcholinesterase inhibitors, namely rivastigmine in IPCI, donepezil in HSD and the THIN and the N-methyl-D-aspartate blocker memantine in AUH.</p><p>	<strong>Conclusion: </strong>We provide a real-world insight into the heterogeneous management and treatment pathways of newly diagnosed dementia patients and a subset of Alzheimer's Disease patients from across Europe.</p><p>	<strong>Keywords: </strong>Alzheimer’s disease; Dementia; Epidemiology; Real-world data.</p>

%0 Journal Article
%A Michael Kessler
%A Douglas Loesch
%A James Perry
%A Nancy Heard-Costa
%A Daniel Taliun
%A Brian Cade
%A Heming Wang
%A Michelle Daya
%A John Ziniti
%A Soma Datta
%A Juan Celedon
%A Manuel Soto-Quiros
%A Lydiana Avila
%A Scott Weiss
%A Kathleen Barnes
%A Susan Redline
%A Ramachandran Vasan
%A Andrew Johnson
%A Rasika Mathias
%A Ryan Hernandez
%A James Wilson
%A Deborah Nickerson
%A Goncolo Abecasis
%A Sharon Browning
%A Sebastian Zollner
%A Jeffrey O'Connell
%A Braxton Mitchell
%A for NHLBI Trans-Omics Precision Medicine Consortium
%A TOPMed Population Genetics Working Group
%A Timothy O'Connor
%B Proc Natl Acad Sci U S A
%D 2020
%G eng
%N 5
%P 2560-2569
%T De novo mutations across 1,465 diverse genomes reveal mutational insights and reductions in the Amish founder populations
%U https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7007577/
%V 117
%X <p>	De novo mutations (DNMs), or mutations that appear in an individual despite not being seen in their parents, are an important source of genetic variation whose impact is relevant to studies of human evolution, genetics, and disease. Utilizing high-coverage whole-genome sequencing data as part of the Trans-Omics for Precision Medicine (TOPMed) Program, we called 93,325 single-nucleotide DNMs across 1,465 trios from an array of diverse human populations, and used them to directly estimate and analyze DNM counts, rates, and spectra. We find a significant positive correlation between local recombination rate and local DNM rate, and that DNM rate explains a substantial portion (8.98 to 34.92%, depending on the model) of the genome-wide variation in population-level genetic variation from 41K unrelated TOPMed samples. Genome-wide heterozygosity does correlate with DNM rate, but only explains &lt;1% of variation. While we are underpowered to see small differences, we do not find significant differences in DNM rate between individuals of European, African, and Latino ancestry, nor across ancestrally distinct segments within admixed individuals. However, we did find significantly fewer DNMs in Amish individuals, even when compared with other Europeans, and even after accounting for parental age and sequencing center. Specifically, we found significant reductions in the number of C→A and T→C mutations in the Amish, which seem to underpin their overall reduction in DNMs. Finally, we calculated near-zero estimates of narrow sense heritability (<em>h</em> <sup>2</sup>), which suggest that variation in DNM rate is significantly shaped by nonadditive genetic effects and the environment.</p><p>	<strong>Keywords: </strong>Amish; de novo mutations; diversity; mutation rate; recombination.</p>

%0 Journal Article
%A Daniel Clarke
%A Lily Wang
%A Alex Jones
%A Megan Wojciechowicz
%A Denis Torre
%A Kathleen Jagodnik
%A Sherry Jenkins
%A Peter McQuilton
%A Zachary Flamholz
%A Moshe Silverstein
%A Brian Schilder
%A Kimberly Robasky
%A Claris Castillo
%A Ray Idaszak
%A Stanley Ahalt
%A Jason Williams
%A Stephan Schurer
%A Daniel Cooper
%A de Ricardo Miranda Azevedo
%A Juergen Klenk
%A Melissa Haendel
%A Jared Nedzel
%A Paul Avillach
%A Mary Shimoyama
%A Rayna Harris
%A Meredith Gamble
%A Rudy Poten
%A Amanda Charbonneau
%A Jennie Larkin
%A Titus Brown
%A Vivien Bonazzi
%A Michel Dumontier
%A Susanna-Assunta Sansone
%A Avi Ma'ayan
%B Cell Syst
%D 2019
%G eng
%N 5
%P 417-421
%R 10.1016/j.cels.2019.09.011
%T FAIRshake: Toolkit to Evaluate the FAIRness of Research Digital Resources
%V 9
%1 http://www.ncbi.nlm.nih.gov/pubmed/31677972?dopt=Abstract
%8 11/2019
%X As more digital resources are produced by the research community, it is becoming increasingly important to harmonize and organize them for synergistic utilization. The findable, accessible, interoperable, and reusable (FAIR) guiding principles have prompted many stakeholders to consider strategies for tackling this challenge. The FAIRshake toolkit was developed to enable the establishment of community-driven FAIR metrics and rubrics paired with manual and automated FAIR assessments. FAIR assessments are visualized as an insignia that can be embedded within digital-resources-hosting websites. Using FAIRshake, a variety of biomedical digital resources were manually and automatically evaluated for their level of FAIRness.

%0 Journal Article
%A Myriam Alexander
%A Katrina Loomis
%A van der Johan Lei
%A Talita Duarte-Salles
%A Daniel Prieto-Alhambra
%A David Ansell
%A Alessandro Pasqua
%A Francesco Lapi
%A Peter Rijnbeek
%A Mees Mosseveld
%A Paul Avillach
%A Peter Egger
%A Nafeesa Dhalwani
%A Stuart Kendrick
%A Carlos Celis-Morales
%A Dawn Waterworth
%A William Alazawi
%A Naveed Sattar
%B BMJ
%D 2019
%G eng
%P l5367
%R 10.1136/bmj.l5367
%T Non-alcoholic fatty liver disease and risk of incident acute myocardial infarction and stroke: findings from matched cohort study of 18 million European adults
%V 367
%1 http://www.ncbi.nlm.nih.gov/pubmed/31594780?dopt=Abstract
%8 10/2019
%X OBJECTIVE: To estimate the risk of acute myocardial infarction (AMI) or stroke in adults with non-alcoholic fatty liver disease (NAFLD) or non-alcoholic steatohepatitis (NASH).DESIGN: Matched cohort study.SETTING: Population based, electronic primary healthcare databases before 31 December 2015 from four European countries: Italy (n=1 542 672), Netherlands (n=2 225 925), Spain (n=5 488 397), and UK (n=12 695 046).PARTICIPANTS: 120 795 adults with a recorded diagnosis of NAFLD or NASH and no other liver diseases, matched at time of NAFLD diagnosis (index date) by age, sex, practice site, and visit, recorded at six months before or after the date of diagnosis, with up to 100 patients without NAFLD or NASH in the same database.MAIN OUTCOME MEASURES: Primary outcome was incident fatal or non-fatal AMI and ischaemic or unspecified stroke. Hazard ratios were estimated using Cox models and pooled across databases by random effect meta-analyses.RESULTS: 120 795 patients with recorded NAFLD or NASH diagnoses were identified with mean follow-up 2.1-5.5 years. After adjustment for age and smoking the pooled hazard ratio for AMI was 1.17 (95% confidence interval 1.05 to 1.30; 1035 events in participants with NAFLD or NASH, 67 823 in matched controls). In a group with more complete data on risk factors (86 098 NAFLD and 4 664 988 matched controls), the hazard ratio for AMI after adjustment for systolic blood pressure, type 2 diabetes, total cholesterol level, statin use, and hypertension was 1.01 (0.91 to 1.12; 747 events in participants with NAFLD or NASH, 37 462 in matched controls). After adjustment for age and smoking status the pooled hazard ratio for stroke was 1.18 (1.11 to 1.24; 2187 events in participants with NAFLD or NASH, 134 001 in matched controls). In the group with more complete data on risk factors, the hazard ratio for stroke was 1.04 (0.99 to 1.09; 1666 events in participants with NAFLD, 83 882 in matched controls) after further adjustment for type 2 diabetes, systolic blood pressure, total cholesterol level, statin use, and hypertension.CONCLUSIONS: The diagnosis of NAFLD in current routine care of 17.7 million patient appears not to be associated with AMI or stroke risk after adjustment for established cardiovascular risk factors. Cardiovascular risk assessment in adults with a diagnosis of NAFLD is important but should be done in the same way as for the general population.

%0 Journal Article
%A Stephan Sanders
%A Mustafa Sahin
%A Joseph Hostyk
%A Audrey Thurm
%A Sebastien Jacquemont
%A Paul Avillach
%A Elise Douard
%A Christa Martin
%A Meera Modi
%A Andres Moreno-De-Luca
%A Armin Raznahan
%A Alan Anticevic
%A Ricardo Dolmetsch
%A Guoping Feng
%A Daniel Geschwind
%A David Glahn
%A David Goldstein
%A David Ledbetter
%A Jennifer Mulle
%A Sergiu Pasca
%A Rodney Samaco
%A Jonathan Sebat
%A Anne Pariser
%A Thomas Lehner
%A Raquel Gur
%A Carrie Bearden
%B Nat Med
%D 2019
%G eng
%N 10
%P 1477-1487
%R 10.1038/s41591-019-0581-5
%T A framework for the investigation of rare genetic disorders in neuropsychiatry
%V 25
%1 http://www.ncbi.nlm.nih.gov/pubmed/31548702?dopt=Abstract
%8 10/2019
%X De novo and inherited rare genetic disorders (RGDs) are a major cause of human morbidity, frequently involving neuropsychiatric symptoms. Recent advances in genomic technologies and data sharing have revolutionized the identification and diagnosis of RGDs, presenting an opportunity to elucidate the mechanisms underlying neuropsychiatric disorders by investigating the pathophysiology of high-penetrance genetic risk factors. Here we seek out the best path forward for achieving these goals. We think future research will require consistent approaches across multiple RGDs and developmental stages, involving both the characterization of shared neuropsychiatric dimensions in humans and the identification of neurobiological commonalities in model systems. A coordinated and concerted effort across patients, families, researchers, clinicians and institutions, including rapid and broad sharing of data, is now needed to translate these discoveries into urgently needed therapies.

%0 Journal Article
%A Sulev Reisberg
%A Nicholas Galwey
%A Paul Avillach
%A Anna-Stina Sahlqvist
%A Liis Kolberg
%A Reedik Mägi
%A Tõnu Esko
%A Jaak Vilo
%A Glen James
%B Int J Immunogenet
%D 2019
%G eng
%N 2
%P 49-58
%R 10.1111/iji.12413
%T Comparison of variation in frequency for SNPs associated with asthma or liver disease between Estonia, HapMap populations and the 1000 genome project populations
%V 46
%1 http://www.ncbi.nlm.nih.gov/pubmed/30659741?dopt=Abstract
%8 04/2019
%X Allele-specific analyses to understand frequency differences across populations, particularly populations not well studied, are important to help identify variants that may have a functional effect on disease mechanisms and phenotypic predisposition, facilitating new Genome-Wide Association Studies (GWAS). We aimed to compare the allele frequency of 11 asthma-associated and 16 liver disease-associated single nucleotide polymorphisms (SNPs) between the Estonian, HapMap and 1000 genome project populations. When comparing EGCUT with HapMap populations, the largest difference in allele frequencies was observed with the Maasai population in Kinyawa, Kenya, with 12 SNP variants reporting statistical significance. Similarly, when comparing EGCUT with 1000 genomes project populations, the largest difference in allele frequencies was observed with pooled African populations with 22 SNP variants reporting statistical significance. For 11 asthma-associated and 16 liver disease-associated SNPs, Estonians are genetically similar to other European populations but significantly different from African populations. Understanding differences in genetic architecture between ethnic populations is important to facilitate new GWAS targeted at underserved ethnic groups to enable novel genetic findings to aid the development of new therapies to reduce morbidity and mortality.

%0 Journal Article
%A Qiu-Yue Zhong
%A Bizu Gelaye
%A Elizabeth Karlson
%A Paul Avillach
%A Jordan Smoller
%A Tianxi Cai
%A Michelle Williams
%B Paediatr Perinat Epidemiol
%D 2019
%G eng
%N 2
%P 137-144
%R 10.1111/ppe.12535
%T Associations of antepartum suicidal behaviour with adverse infant and obstetric outcomes
%V 33
%1 http://www.ncbi.nlm.nih.gov/pubmed/30790331?dopt=Abstract
%8 03/2019
%X BACKGROUND: Relatively little is known about antepartum suicidal behaviour and pregnancy outcomes. We examined associations of antepartum suicidal behaviour, alone and in combination with psychiatric disorders, with adverse infant and obstetric outcomes.METHODS: We included 188 925 singleton livebirths from a retrospective cohort (1996-2016). Suicidal behaviour, psychiatric disorders, and outcomes were derived from electronic medical records. We performed multivariable logistic regressions with generalised estimating equations to estimate adjusted odds ratios (aOR) with 95% confidence intervals (95%CI).RESULTS: The prevalence of antepartum suicidal behaviour was 152.44 per 100 000 singleton livebirths. Nearly two-thirds (64.24%) of women with suicidal behaviour also had psychiatric disorders. Compared to women without psychiatric disorders and suicidal behaviour, women with psychiatric disorders alone had 1.3-fold to 1.4-fold increased odds of delivering low birthweight or preterm infants and 1.2-fold increased odds of experiencing obstetric complications. Women with suicidal behaviour alone had increased odds of preterm labour (aOR 2.05, 95% CI 1.16, 3.62). Women with both suicidal behaviour and psychiatric disorders had &gt; twofold increased odds of delivering low birthweight (aOR 2.52, 95% CI 1.40, 4.54), preterm birth (aOR 2.44, 95% CI 1.63, 3.66), and low birthweight/preterm birth (aOR 2.30, 95% CI 1.54, 3.44) infants; the odds of preterm labour (aOR 1.62, 95% CI 1.06, 2.47), placental abruption (aOR 2.33, 95% CI 1.20, 4.51), preterm rupture of membranes (aOR 1.63, 95% CI 1.08, 2.46), and postpartum haemorrhage (aOR 1.93, 95%CI 1.09, 3.40) were elevated.CONCLUSIONS: Antepartum suicidal behaviour, when co-occurring with psychiatric disorders, is associated with increased odds of adverse infant and obstetric outcomes. Future studies are warranted to understand the causal roles of suicidal behaviour and psychiatric disorders in pregnancy.

%0 Journal Article
%A Qiu-Yue Zhong
%A Leena Mittal
%A Margo Nathan
%A Kara Brown
%A Deborah Knudson González
%A Tianrun Cai
%A Sean Finan
%A Bizu Gelaye
%A Paul Avillach
%A Jordan Smoller
%A Elizabeth Karlson
%A Tianxi Cai
%A Michelle Williams
%B Eur J Epidemiol
%D 2019
%G eng
%N 2
%P 153-162
%R 10.1007/s10654-018-0470-0
%T Use of natural language processing in electronic medical records to identify pregnant women with suicidal behavior: towards a solution to the complex classification problem
%V 34
%1 http://www.ncbi.nlm.nih.gov/pubmed/30535584?dopt=Abstract
%8 02/2019
%X We developed algorithms to identify pregnant women with suicidal behavior using information extracted from clinical notes by natural language processing (NLP) in electronic medical records. Using both codified data and NLP applied to unstructured clinical notes, we first screened pregnant women in Partners HealthCare for suicidal behavior. Psychiatrists manually reviewed clinical charts to identify relevant features for suicidal behavior and to obtain gold-standard labels. Using the adaptive elastic net, we developed algorithms to classify suicidal behavior. We then validated algorithms in an independent validation dataset. From 275,843 women with codes related to pregnancy or delivery, 9331 women screened positive for suicidal behavior by either codified data (N = 196) or NLP (N = 9,145). Using expert-curated features, our algorithm achieved an area under the curve of 0.83. By setting a positive predictive value comparable to that of diagnostic codes related to suicidal behavior (0.71), we obtained a sensitivity of 0.34, specificity of 0.96, and negative predictive value of 0.83. The algorithm identified 1423 pregnant women with suicidal behavior among 9331 women screened positive. Mining unstructured clinical notes using NLP resulted in a 11-fold increase in the number of pregnant women identified with suicidal behavior, as compared to solely reliance on diagnostic codes.

%0 Journal Article
%A Peter Brown
%A RELISH Consortium
%A Yaoqi Zhou
%B Database : the journal of biological databases and curation
%D 2019
%G eng
%R 10.1093/database/baz085
%T Large expert-curated database for benchmarking document similarity detection in biomedical literature search.
%V 2019
%1 https://www.ncbi.nlm.nih.gov/pubmed/33326193
%8 01/2019
%X <p>Document recommendation systems for locating relevant literature have mostly relied on methods developed a decade ago. This is largely due to the lack of a large offline gold-standard benchmark of relevant documents that cover a variety of research fields such that newly developed literature search techniques can be compared, improved and translated into practice. To overcome this bottleneck, we have established the RElevant LIterature SearcH consortium consisting of more than 1500 scientists from 84 countries, who have collectively annotated the relevance of over 180 000 PubMed-listed articles with regard to their respective seed (input) article/s. The majority of annotations were contributed by highly experienced, original authors of the seed articles. The collected data cover 76% of all unique PubMed Medical Subject Headings descriptors. No systematic biases were observed across different experience levels, research fields or time spent on annotations. More importantly, annotations of the same document pairs contributed by different scientists were highly concordant. We further show that the three representative baseline methods used to generate recommended articles for evaluation (Okapi Best Matching 25, Term Frequency-Inverse Document Frequency and PubMed Related Articles) had similar overall performances. Additionally, we found that these methods each tend to produce distinct collections of recommended articles, suggesting that a hybrid method may be required to completely capture all relevant articles. The established database server located at https://relishdb.ict.griffith.edu.au is freely available for the downloading of annotation data and the blind testing of new methods. We expect that this benchmark will be useful for stimulating the development of new powerful techniques for title and title/abstract-based search engines for relevant articles in biomedical research.</p>

%0 Journal Article
%A Glen James
%A Sulev Reisberg
%A Kaido Lepik
%A Nicholas Galwey
%A Paul Avillach
%A Liis Kolberg
%A Reedik Mägi
%A Tõnu Esko
%A Myriam Alexander
%A Dawn Waterworth
%A Katrina Loomis
%A Jaak Vilo
%B PLoS One
%D 2019
%G eng
%N 4
%P e0215026
%R 10.1371/journal.pone.0215026
%T An exploratory phenome wide association study linking asthma and liver disease genetic variants to electronic health records from the Estonian Biobank
%V 14
%1 http://www.ncbi.nlm.nih.gov/pubmed/30978214?dopt=Abstract
%X The Estonian Biobank, governed by the Institute of Genomics at the University of Tartu (Biobank), has stored genetic material/DNA and continuously collected data since 2002 on a total of 52,274 individuals representing ~5% of the Estonian adult population and is increasing. To explore the utility of data available in the Biobank, we conducted a phenome-wide association study (PheWAS) in two areas of interest to healthcare researchers; asthma and liver disease. We used 11 asthma and 13 liver disease-associated single nucleotide polymorphisms (SNPs), identified from published genome-wide association studies, to test our ability to detect established associations. We confirmed 2 asthma and 5 liver disease associated variants at nominal significance and directionally consistent with published results. We found 2 associations that were opposite to what was published before (rs4374383:AA increases risk of NASH/NAFLD, rs11597086 increases ALT level). Three SNP-diagnosis pairs passed the phenome-wide significance threshold: rs9273349 and E06 (thyroiditis, p = 5.50x10-8); rs9273349 and E10 (type-1 diabetes, p = 2.60x10-7); and rs2281135 and K76 (non-alcoholic liver diseases, including NAFLD, p = 4.10x10-7). We have validated our approach and confirmed the quality of the data for these conditions. Importantly, we demonstrate that the extensive amount of genetic and medical information from the Estonian Biobank can be successfully utilized for scientific research.

%0 Journal Article
%A Myriam Alexander
%A Katrina Loomis
%A Jolyon Fairburn-Beech
%A van der Johan Lei
%A Talita Duarte-Salles
%A Daniel Prieto-Alhambra
%A David Ansell
%A Alessandro Pasqua
%A Francesco Lapi
%A Peter Rijnbeek
%A Mees Mosseveld
%A Paul Avillach
%A Peter Egger
%A Stuart Kendrick
%A Dawn Waterworth
%A Naveed Sattar
%A William Alazawi
%B BMC Med
%D 2018
%G eng
%N 1
%P 130
%R 10.1186/s12916-018-1103-x
%T Real-world data reveal a diagnostic gap in non-alcoholic fatty liver disease
%V 16
%1 http://www.ncbi.nlm.nih.gov/pubmed/30099968?dopt=Abstract
%8 08/2018
%X BACKGROUND: Non-alcoholic fatty liver disease (NAFLD) is the most common cause of liver disease worldwide. It affects an estimated 20% of the general population, based on cohort studies of varying size and heterogeneous selection. However, the prevalence and incidence of recorded NAFLD diagnoses in unselected real-world health-care records is unknown. We harmonised health records from four major European territories and assessed age- and sex-specific point prevalence and incidence of NAFLD over the past decade.METHODS: Data were extracted from The Health Improvement Network (UK), Health Search Database (Italy), Information System for Research in Primary Care (Spain) and Integrated Primary Care Information (Netherlands). Each database uses a different coding system. Prevalence and incidence estimates were pooled across databases by random-effects meta-analysis after a log-transformation.RESULTS: Data were available for 17,669,973 adults, of which 176,114 had a recorded diagnosis of NAFLD. Pooled prevalence trebled from 0.60% in 2007 (95% confidence interval: 0.41-0.79) to 1.85% (0.91-2.79) in 2014. Incidence doubled from 1.32 (0.83-1.82) to 2.35 (1.29-3.40) per 1000 person-years. The FIB-4 non-invasive estimate of liver fibrosis could be calculated in 40.6% of patients, of whom 29.6-35.7% had indeterminate or high-risk scores.CONCLUSIONS: In the largest primary-care record study of its kind to date, rates of recorded NAFLD are much lower than expected suggesting under-diagnosis and under-recording. Despite this, we have identified rising incidence and prevalence of the diagnosis. Improved recognition of NAFLD may identify people who will benefit from risk factor modification or emerging therapies to prevent progression to cardiometabolic and hepatic complications.

%0 Journal Article
%A Qiu-Yue Zhong
%A Elizabeth Karlson
%A Bizu Gelaye
%A Sean Finan
%A Paul Avillach
%A Jordan Smoller
%A Tianxi Cai
%A Michelle Williams
%B BMC Med Inform Decis Mak
%D 2018
%G eng
%N 1
%P 30
%R 10.1186/s12911-018-0617-7
%T Screening pregnant women for suicidal behavior in electronic medical records: diagnostic codes vs. clinical notes processed by natural language processing
%V 18
%1 http://www.ncbi.nlm.nih.gov/pubmed/29843698?dopt=Abstract
%8 05/2018
%X BACKGROUND: We examined the comparative performance of structured, diagnostic codes vs. natural language processing (NLP) of unstructured text for screening suicidal behavior among pregnant women in electronic medical records (EMRs).METHODS: Women aged 10-64 years with at least one diagnostic code related to pregnancy or delivery (N = 275,843) from Partners HealthCare were included as our "datamart." Diagnostic codes related to suicidal behavior were applied to the datamart to screen women for suicidal behavior. Among women without any diagnostic codes related to suicidal behavior (n = 273,410), 5880 women were randomly sampled, of whom 1120 had at least one mention of terms related to suicidal behavior in clinical notes. NLP was then used to process clinical notes for the 1120 women. Chart reviews were performed for subsamples of women.RESULTS: Using diagnostic codes, 196 pregnant women were screened positive for suicidal behavior, among whom 149 (76%) had confirmed suicidal behavior by chart review. Using NLP among those without diagnostic codes, 486 pregnant women were screened positive for suicidal behavior, among whom 146 (30%) had confirmed suicidal behavior by chart review.CONCLUSIONS: The use of NLP substantially improves the sensitivity of screening suicidal behavior in EMRs. However, the prevalence of confirmed suicidal behavior was lower among women who did not have diagnostic codes for suicidal behavior but screened positive by NLP. NLP should be used together with diagnostic codes for future EMR-based phenotyping studies for suicidal behavior.

%0 Journal Article
%A Gayan Perera
%A Lars Pedersen
%A David Ansel
%A Myriam Alexander
%A Michael Arrighi
%A Paul Avillach
%A Nadia Foskett
%A Rosa Gini
%A Mark Gordon
%A Usha Gungabissoon
%A Miguel-Angel Mayer
%A Gerald Novak
%A Peter Rijnbeek
%A Gianluca Trifirò
%A van der Johan Lei
%A Pieter Visser
%A Robert Stewart
%B Alzheimers Dement
%D 2018
%G eng
%N 2
%P 130-139
%R 10.1016/j.jalz.2017.06.2270
%T Dementia prevalence and incidence in a federation of European Electronic Health Record databases: The European Medical Informatics Framework resource
%V 14
%1 http://www.ncbi.nlm.nih.gov/pubmed/28734783?dopt=Abstract
%8 02/2018
%X INTRODUCTION: The European Medical Information Framework consortium has assembled electronic health record (EHR) databases for dementia research. We calculated dementia prevalence and incidence in 25 million persons from 2004 to 2012.METHODS: Six EHR databases (three primary care and three secondary care) from five countries were interrogated. Dementia was ascertained by consensus harmonization of clinical/diagnostic codes. Annual period prevalences and incidences by age and gender were calculated and meta-analyzed.RESULTS: The six databases contained 138,625 dementia cases. Age-specific prevalences were around 30% of published estimates from community samples and incidences were around 50%. Pooled prevalences had increased from 2004 to 2012 in all age groups but pooled incidences only after age 75 years. Associations with age and gender were stable over time.DISCUSSION: The European Medical Information Framework initiative supports EHR data on unprecedented number of people with dementia. Age-specific prevalences and incidences mirror estimates from community samples in pattern at levels that are lower but increasing over time.

%0 Journal Article
%A Qiu-Yue Zhong
%A Bizu Gelaye
%A Jordan Smoller
%A Paul Avillach
%A Tianxi Cai
%A Michelle Williams
%B PLoS One
%D 2018
%G eng
%N 2
%P e0192943
%R 10.1371/journal.pone.0192943
%T Adverse obstetric outcomes during delivery hospitalizations complicated by suicidal behavior among US pregnant women
%V 13
%1 http://www.ncbi.nlm.nih.gov/pubmed/29447245?dopt=Abstract
%X OBJECTIVE: The effects of suicidal behavior on obstetric outcomes remain dangerously unquantified. We sought to report on the risk of adverse obstetric outcomes for US women with suicidal behavior at the time of delivery.METHODS: We performed a cross-sectional analysis of delivery hospitalizations from 2007-2012 National (Nationwide) Inpatient Sample. From the same hospitalization record, International Classification of Diseases codes were used to identify suicidal behavior and adverse obstetric outcomes. Adjusted odds ratios (aOR) and 95% confidence intervals (CI) were obtained using logistic regression.RESULTS: Of the 23,507,597 delivery hospitalizations, 2,180 were complicated by suicidal behavior. Women with suicidal behavior were at a heightened risk for outcomes including antepartum hemorrhage (aOR = 2.34; 95% CI: 1.47-3.74), placental abruption (aOR = 2.07; 95% CI: 1.17-3.66), postpartum hemorrhage (aOR = 2.33; 95% CI: 1.61-3.37), premature delivery (aOR = 3.08; 95% CI: 2.43-3.90), stillbirth (aOR = 10.73; 95% CI: 7.41-15.56), poor fetal growth (aOR = 1.70; 95% CI: 1.10-2.62), and fetal anomalies (aOR = 3.72; 95% CI: 2.57-5.40). No significant association was observed for maternal suicidal behavior with cesarean delivery, induction of labor, premature rupture of membranes, excessive fetal growth, and fetal distress. The mean length of stay was longer for women with suicidal behavior.CONCLUSION: During delivery hospitalization, women with suicidal behavior are at increased risk for many adverse obstetric outcomes, highlighting the importance of screening for and providing appropriate clinical care for women with suicidal behavior during pregnancy.

%0 Journal Article
%A QY Zhong
%A Gelaye
%A GL Fricchione
%A Avillach
%A EW Karlson
%A MA Williams
%B BMC Pregnancy Childbirth
%D 2018
%G eng
%N 1
%T Adverse obstetric and neonatal outcomes complicated by psychosis among pregnant women in the United States
%U https://bmcpregnancychildbirth.biomedcentral.com/articles/10.1186/s12884-018-1750-0
%V 18
%X Adverse obstetric and neonatal outcomes among women with psychosis, particularly affective psychosis, has rarely been studied at the population level. We aimed to assess the risk of adverse obstetric and neonatal outcomes among women with psychosis (schizophrenia, affective psychosis, and other psychoses). 

%0 Journal Article
%A Alba Gutiérrez-Sacristán
%A Romain Guedj
%A Gabur Korodi
%A Jason Stedman
%A Laura Furlong
%A Chirag Patel
%A Isaac Kohane
%A Paul Avillach
%B Bioinformatics
%D 2018
%G eng
%T Rcupcake: an R package for querying and analyzing biomedical data through BD2K Pic-Sure restful API
%U https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5905576/
%X <p>	<strong>Motivation: </strong>In the era of big data and precision medicine, the number of databases containing clinical, environmental, self-reported and biochemical variables is increasing exponentially. Enabling the experts to focus on their research questions rather than on computational data management, access and analysis is one of the most significant challenges nowadays.</p><p>	<strong>Results: </strong>We present Rcupcake, an R package that contains a variety of functions for leveraging different databases through the BD2K PIC-SURE RESTful API and facilitating its query, analysis and interpretation. The package offers a variety of analysis and visualization tools, including the study of the phenotype co-occurrence and prevalence, according to multiple layers of data, such as phenome, exposome or genome.</p><p>	<strong>Availability and implementation: </strong>The package is implemented in R and is available under Mozilla v2 license from GitHub (https://github.com/hms-dbmi/Rcupcake). Two reproducible case studies are also available (https://github.com/hms-dbmi/Rcupcake-case-studies/blob/master/SSCcaseStudy_v01.ipynb, https://github.com/hms-dbmi/Rcupcake-case-studies/blob/master/NHANEScaseStudy_v01.ipynb).</p><p>	<strong>Contact: </strong>paul_avillach@hms.harvard.edu.</p><p>	<strong>Supplementary information: </strong>Supplementary data are available at Bioinformatics online.</p>

%0 Journal Article
%A Florence Bourgeois
%A Paul Avillach
%A Sek Won Kong
%A Michelle Heinz
%A Tram Tran
%A Ramkrishna Chakrabarty
%A Jonathan Bickel
%A Piotr Sliz
%A Erin Borglund
%A Susan Kornetsky
%A Kenneth Mandl
%B J Pers Med
%D 2017
%G eng
%N 4
%R 10.3390/jpm7040021
%T Development of the Precision Link Biobank at Boston Children's Hospital: Challenges and Opportunities
%V 7
%1 http://www.ncbi.nlm.nih.gov/pubmed/29244735?dopt=Abstract
%8 12/2017
%X Increasingly, biobanks are being developed to support organized collections of biological specimens and associated clinical information on broadly consented, diverse patient populations. We describe the implementation of a pediatric biobank, comprised of a fully-informed patient cohort linking specimens to phenotypic data derived from electronic health records (EHR). The Biobank was launched after multiple stakeholders' input and implemented initially in a pilot phase before hospital-wide expansion in 2016. In-person informed consent is obtained from all participants enrolling in the Biobank and provides permission to: (1) access EHR data for research; (2) collect and use residual specimens produced as by-products of routine care; and (3) share de-identified data and specimens outside of the institution. Participants are recruited throughout the hospital, across diverse clinical settings. We have enrolled 4900 patients to date, and 41% of these have an associated blood sample for DNA processing. Current efforts are focused on aligning the Biobank with other ongoing research efforts at our institution and extending our electronic consenting system to support remote enrollment. A number of pediatric-specific challenges and opportunities is reviewed, including the need to re-consent patients when they reach 18 years of age, the ability to enroll family members accompanying patients and alignment with disease-specific research efforts at our institution and other pediatric centers to increase cohort sizes, particularly for rare diseases.

%0 Journal Article
%A Cartik Kothari
%A Maxime Wack
%A Claire Hassen-Khodja
%A Sean Finan
%A Guergana Savova
%A Megan O'Boyle
%A Geraldine Bliss
%A Andria Cornell
%A Elizabeth Horn
%A Rebecca Davis
%A Jacquelyn Jacobs
%A Isaac Kohane
%A Paul Avillach
%B American Journal of Medical Genetics Part B: Neuropsychiatric Genetics
%D 2017
%G eng
%T Phelan-McDermid syndrome data network: Integrating patient reported outcomes with clinical notes and curated genetic reports
%U http://onlinelibrary.wiley.com/doi/10.1002/ajmg.b.32579/full
%X The heterogeneity of patient phenotype data are an impediment to the research into the origins and progression of neuropsychiatric disorders. This difficulty is compounded in the case of rare disorders such as Phelan-McDermid Syndrome (PMS) by the paucity of patient clinical data. PMS is a rare syndromic genetic cause of autism and intellectual deficiency. In this paper, we describe the Phelan-McDermid Syndrome Data Network (PMS_DN), a platform that facilitates research into phenotype–genotype correlation and progression of PMS by: a) integrating knowledge of patient phenotypes extracted from Patient Reported Outcomes (PRO) data and clinical notes—two heterogeneous, underutilized sources of knowledge about patient phenotypes—with curated genetic information from the same patient cohort and b) making this integrated knowledge, along with a suite of statistical tools, available free of charge to authorized investigators on a Web portal <a href="https://pmsdn.hms.harvard.edu/" target="_blank" title="Link to external resource: https://pmsdn.hms.harvard.edu">https://pmsdn.hms.harvard.edu</a>. PMS_DN is a Patient Centric Outcomes Research Initiative (PCORI) where patients and their families are involved in all aspects of the management of patient data in driving research into PMS. To foster collaborative research, PMS_DN also makes patient aggregates from this knowledge available to authorized investigators using distributed research networks such as the PCORnet PopMedNet. PMS_DN is hosted on a scalable cloud based environment and complies with all patient data privacy regulations. As of October 31, 2016, PMS_DN integrates high-quality knowledge extracted from the clinical notes of 112 patients and curated genetic reports of 176 patients with preprocessed PRO data from 415 patients.

%0 Journal Article
%A Benedikt F.H. Becker
%A Paul Avillach
%A Silvana Romio
%A van Erik M. Mulligan
%A Daniel Weibel
%A Miriam C.J.M. Sturkenboom
%A Jan A. Kors
%B Pharmacoepidemiology & Drug Safety
%D 2017
%G eng
%N 8
%P 990-1005
%T CodeMapper: semiautomatic coding of case definitions. A contribution from the ADVANCE project
%V 26
%X <section id="pds4245-sec-0001">	<p>		Assessment of drug and vaccine effects by combining information from different healthcare databases in the European Union requires extensive efforts in the harmonization of codes as different vocabularies are being used across countries. In this paper, we present a web application called CodeMapper, which assists in the mapping of case definitions to codes from different vocabularies, while keeping a transparent record of the complete mapping process.	</p></section><section id="pds4245-sec-0002">	<p>		CodeMapper builds upon coding vocabularies contained in the Metathesaurus of the Unified Medical Language System. The mapping approach consists of three phases. First, medical concepts are automatically identified in a free-text case definition. Second, the user revises the set of medical concepts by adding or removing concepts, or expanding them to related concepts that are more general or more specific. Finally, the selected concepts are projected to codes from the targeted coding vocabularies. We evaluated the application by comparing codes that were automatically generated from case definitions by applying CodeMapper's concept identification and successive concept expansion, with reference codes that were manually created in a previous epidemiological study.	</p></section><section id="pds4245-sec-0003">	<p>		Automated concept identification alone had a sensitivity of 0.246 and positive predictive value (PPV) of 0.420 for reproducing the reference codes. Three successive steps of concept expansion increased sensitivity to 0.953 and PPV to 0.616.	</p></section><section id="pds4245-sec-0004">	<p>		Automatic concept identification in the case definition alone was insufficient to reproduce the reference codes, but CodeMapper's operations for concept expansion provide an effective, efficient, and transparent way for reproducing the reference codes.	</p></section>

%0 Journal Article
%A Anne-Sophie Jannot
%A Eric Zapletal
%A Paul Avillach
%A Marie-France Mamzer
%A Anita Burgan
%A Patrice Degoulet
%B International Journal of Medical Informatics
%D 2017
%G eng
%P 21-28
%T The Georges Pompidou University Hospital Clinical Data Warehouse: A 8-years follow-up experience
%V 102
%X <h3 id="sect0015">	Background</h3><p id="spar0020">	When developed jointly with <a href="http://www.sciencedirect.com/topics/medicine-and-dentistry/clinical-information-system" title="Learn more about Clinical Information System">clinical information systems</a>, clinical data warehouses (CDWs) facilitate the reuse of healthcare data and leverage clinical research.</p><h3 id="sect0020">	Objective</h3><p id="spar0025">	To describe both data access and use for clinical research, epidemiology and <a href="http://www.sciencedirect.com/topics/medicine-and-dentistry/health-services" title="Learn more about Health Services">health service</a>research of the “Hôpital Européen Georges Pompidou” (HEGP) CDW.</p><h3 id="sect0025">	Methods</h3><p id="spar0030">	The CDW has been developed since 2008 using an i2b2 platform. It was made available to health professionals and researchers in October 2010. Procedures to access data have been implemented and different access levels have been distinguished according to the nature of queries.</p><h3 id="sect0030">	Results</h3><p id="spar0035">	As of July 2016, the CDW contained the consolidated data of over 860,000 patients followed since the opening of the HEGP hospital in July 2000. These data correspond to more than 122 million clinical item values, 124 million biological item values, and 3.7 million free text reports. The ethics committee of the hospital evaluates all CDW projects that generate <a href="http://www.sciencedirect.com/topics/medicine-and-dentistry/secondary-data" title="Learn more about Secondary data">secondary data</a> marts. Characteristics of the 74 research projects validated between January 2011 and December 2015 are described.</p><h3 id="sect0035">	Conclusion</h3><p id="spar0040">	The use of HEGP CDWs is a key facilitator for clinical research studies. It required however important methodological and organizational support efforts from a biomedical informatics department.</p>

%0 Journal Article
%A Shawn Murphy
%A Paul Avillach
%A Riccardo Bellazzi
%A Lori Phillips
%A Matteo Gabetta
%A Alal Eran
%A Michael McDuffie
%A Isaac Kohane
%B PLoS One
%D 2017
%G eng
%T Combining clinical and genomics queries using i2b2 - Three methods
%X We are fortunate to be living in an era of twin biomedical data surges: a burgeoning representation of human phenotypes in the medical records of our healthcare systems, and high-throughput sequencing making rapid technological advances. The difficulty representing genomic data and its annotations has almost by itself led to the recognition of a biomedical “Big Data” challenge, and the complexity of healthcare data only compounds the problem to the point that coherent representation of both systems on the same platform seems insuperably difficult. We investigated the capability for complex, integrative genomic and clinical queries to be supported in the Informatics for Integrating Biology and the Bedside (i2b2) translational software package. Three different data integration approaches were developed: The first is based on Sequence Ontology, the second is based on the tranSMART engine, and the third on CouchDB. These novel methods for representing and querying complex genomic and clinical data on the i2b2 platform are available today for advancing precision medicine.

%0 Journal Article
%A Tran
%A Tran
%A Geghre
%A Darmon
%A Rampal M
%A Brandone D
%A Gozzo JM
%A Haas H
%A Rebouillat-Savy K
%A Caci H
%A Avillach P
%B PLoS One
%D 2017
%G eng
%N 11
%T Health assessment of French university students and risk factors associated with mental health disorders
%V 12
%X <h4>	The first year of university is a particularly stressful period and can impact academic performance and students' health. The aim of this study was to evaluate the health and lifestyle of undergraduates and assess risk factors associated with psychiatric symptoms.</h4><h4>	Between September 2012 and June 2013, we included all undergraduate students who underwent compulsory a medical visit at the university medical service in Nice (France) during which they were screened for potential diseases during a diagnostic interview. Data were collected prospectively in the CALCIUM database (Consultations Assistés par Logiciel pour les Centres Inter-Universitaire de Médecine) and included information about the students' lifestyle (living conditions, dietary behavior, physical activity, use of recreational drugs). The prevalence of psychiatric symptoms related to depression, anxiety and panic attacks was assessed and risk factors for these symptoms were analyzed using logistic regression.</h4><p>	A total of 4,184 undergraduates were included. Prevalence for depression, anxiety and panic attacks were 12.6%, 7.6% and 1.0%, respectively. During the 30 days preceding the evaluation, 0.6% of the students regularly drank alcohol, 6.3% were frequent-to-heavy tobacco smokers, and 10.0% smoked marijuana. Dealing with financial difficulties and having learning disabilities were associated with psychiatric symptoms. Students who were dissatisfied with their living conditions and those with poor dietary behavior were at risk of depression. Being a woman and living alone were associated with anxiety. Students who screened positively for any psychiatric disorder assessed were at a higher risk of having another psychiatric disorder concomitantly.</p><p>	The prevalence of psychiatric disorders in undergraduate students is low but the rate of students at risk of developing chronic disease is far from being negligible. Understanding predictors for these symptoms may improve students' health by implementing targeted prevention campaigns. Further research in other French universities is necessary to confirm our results.</p>

%0 Journal Article
%A Chirag Patel
%A Nam Pho
%A Michael McDuffie
%A Jeremy Easton-Marks
%A Cartik Kothari
%A Isaac Kohane
%A Paul Avillach
%B Sci Data
%D 2016
%G ENG
%P 160096
%R 10.1038/sdata.2016.96
%T A database of human exposomes and phenomes from the US National Health and Nutrition Examination Survey.
%V 3
%1 http://www.ncbi.nlm.nih.gov/pubmed/27779619?dopt=Abstract
%8 10/2016
%X The National Health and Nutrition Examination Survey (NHANES) is a population survey implemented by the Centers for Disease Control and Prevention (CDC) to monitor the health of the United States whose data is publicly available in hundreds of files. This Data Descriptor describes a single unified and universally accessible data file, merging across 255 separate files and stitching data across 4 surveys, encompassing 41,474 individuals and 1,191 variables. The variables consist of phenotype and environmental exposure information on each individual, specifically (1) demographic information, physical exam results (e.g., height, body mass index), laboratory results (e.g., cholesterol, glucose, and environmental exposures), and (4) questionnaire items. Second, the data descriptor describes a dictionary to enable analysts find variables by category and human-readable description. The datasets are available on DataDryad and a hands-on analytics tutorial is available on GitHub. Through a new big data platform, BD2K Patient Centered Information Commons (http://pic-sure.org), we provide a new way to browse the dataset via a web browser (https://nhanes.hms.harvard.edu) and provide application programming interface for programmatic access.

%0 Journal Article
%A Jessica Tenenbaum
%A Paul Avillach
%A Marge Benham-Hutchins
%A Matthew Breitenstein
%A Erin Crowgey
%A Mark Hoffman
%A Xia Jiang
%A Subha Madhavan
%A John Mattison
%A Radhakrishnan Nagarajan
%A Bisakha Ray
%A Dmitriy Shin
%A Shyam Visweswaran
%A Zhongming Zhao
%A Robert Freimuth
%B J Am Med Inform Assoc
%D 2016
%G eng
%N 4
%P 791-5
%R 10.1093/jamia/ocv213
%T An informatics research agenda to support precision medicine: seven key areas.
%V 23
%1 http://www.ncbi.nlm.nih.gov/pubmed/27107452?dopt=Abstract
%8 07/2016
%X The recent announcement of the Precision Medicine Initiative by President Obama has brought precision medicine (PM) to the forefront for healthcare providers, researchers, regulators, innovators, and funders alike. As technologies continue to evolve and datasets grow in magnitude, a strong computational infrastructure will be essential to realize PM's vision of improved healthcare derived from personal data. In addition, informatics research and innovation affords a tremendous opportunity to drive the science underlying PM. The informatics community must lead the development of technologies and methodologies that will increase the discovery and application of biomedical knowledge through close collaboration between researchers, clinicians, and patients. This perspective highlights seven key areas that are in need of further informatics research and innovation to support the realization of PM.

%0 Journal Article
%A Pedro Augusto Gondim Teixeira
%A Jacques De Verbizier
%A Sabine Aptel
%A Maxime Wack
%A François Dap
%A Gilles Dautel
%A Alain Blum
%B AJR Am J Roentgenol
%D 2016
%G eng
%N 1
%P 144-50
%R 10.2214/AJR.15.14606
%T Posterior Radioscaphoid Angle as a Predictor of Wrist Degenerative Joint Disease in Patients With Scapholunate Ligament Tears.
%V 206
%1 http://www.ncbi.nlm.nih.gov/pubmed/26700346?dopt=Abstract
%8 01/2016
%X OBJECTIVE: The purpose of this study is to determine whether the posterior radioscaphoid angle, a marker of posterior displacement of the scaphoid, is associated with degenerative joint disease in patients with scapholunate ligament tears.MATERIALS AND METHODS: Images from 150 patients with wrist pain who underwent CT arthrography and radiography were retrospectively evaluated. Patients with and without scapholunate ligament ruptures were divided into two groups according to CT arthrography findings. The presence of degenerative changes (scapholunate advanced collapse [SLAC] wrist) was evaluated and graded on conventional radiographs. Images were evaluated by two readers independently, and an adjudicator analyzed the discordant cases. Posterior radioscaphoid angle values were correlated with CT arthrography and radiographic findings. The association between posterior radioscaphoid angle and degenerative joint disease was evaluated. Scapholunate and radiolunate angles were considered in the analysis.RESULTS: The posterior radioscaphoid angle was measurable in all patients, with substantial interobserver agreement (intraclass correlation coefficient, 0.75). The posterior radioscaphoid angle performed better than did the scapholunate and radiolunate angles in the differentiation of patients with and without SLAC wrist (p  0.02). Posterior radioscaphoid angles greater than 114° presented an 80.0% sensitivity and 89.7% specificity for the detection of SLAC wrist.CONCLUSION: Posterior radioscaphoid angles were strongly associated with degenerative wrist disease, with potential prognostic implications in patients with wrist trauma and scapholunate ligament ruptures.

%0 Journal Article
%A Giuseppe Roberto
%A Ingrid Leal
%A Naveed Sattar
%A Katrina Loomis
%A Paul Avillach
%A Peter Egger
%A van Rients Wijngaarden
%A David Ansell
%A Sulev Reisberg
%A Mari-Liis Tammesoo
%A Helene Alavere
%A Alessandro Pasqua
%A Lars Pedersen
%A James Cunningham
%A Lara Tramontan
%A Miguel Mayer
%A Ron Herings
%A Preciosa Coloma
%A Francesco Lapi
%A Miriam Sturkenboom
%A van der Johan Lei
%A Martijn Schuemie
%A Peter Rijnbeek
%A Rosa Gini
%B PLoS One
%D 2016
%G eng
%N 8
%P e0160648
%R 10.1371/journal.pone.0160648
%T Identifying Cases of Type 2 Diabetes in Heterogeneous Data Sources: Strategy from the EMIF Project.
%V 11
%1 http://www.ncbi.nlm.nih.gov/pubmed/27580049?dopt=Abstract
%X Due to the heterogeneity of existing European sources of observational healthcare data, data source-tailored choices are needed to execute multi-data source, multi-national epidemiological studies. This makes transparent documentation paramount. In this proof-of-concept study, a novel standard data derivation procedure was tested in a set of heterogeneous data sources. Identification of subjects with type 2 diabetes (T2DM) was the test case. We included three primary care data sources (PCDs), three record linkage of administrative and/or registry data sources (RLDs), one hospital and one biobank. Overall, data from 12 million subjects from six European countries were extracted. Based on a shared event definition, sixteeen standard algorithms (components) useful to identify T2DM cases were generated through a top-down/bottom-up iterative approach. Each component was based on one single data domain among diagnoses, drugs, diagnostic test utilization and laboratory results. Diagnoses-based components were subclassified considering the healthcare setting (primary, secondary, inpatient care). The Unified Medical Language System was used for semantic harmonization within data domains. Individual components were extracted and proportion of population identified was compared across data sources. Drug-based components performed similarly in RLDs and PCDs, unlike diagnoses-based components. Using components as building blocks, logical combinations with AND, OR, AND NOT were tested and local experts recommended their preferred data source-tailored combination. The population identified per data sources by resulting algorithms varied from 3.5% to 15.7%, however, age-specific results were fairly comparable. The impact of individual components was assessed: diagnoses-based components identified the majority of cases in PCDs (93-100%), while drug-based components were the main contributors in RLDs (81-100%). The proposed data derivation procedure allowed the generation of data source-tailored case-finding algorithms in a standardized fashion, facilitated transparent documentation of the process and benchmarking of data sources, and provided bases for interpretation of possible inter-data source inconsistency of findings in future studies.

%0 Journal Article
%A Rosa Gini
%A Martijn Schuemie
%A Jeffrey Brown
%A Patrick Ryan
%A Edoardo Vacchi
%A Massimo Coppola
%A Walter Cazzola
%A Preciosa Coloma
%A Roberto Berni
%A Gayo Diallo
%A José Luis Oliveira
%A Paul Avillach
%A Gianluca Trifirò
%A Peter Rijnbeek
%A Mariadonata Bellentani
%A van der Johan Lei
%A Niek Klazinga
%A Miriam Sturkenboom
%B EGEMS (Wash DC)
%D 2016
%G eng
%N 1
%P 1189
%R 10.13063/2327-9214.1189
%T Data Extraction and Management in Networks of Observational Health Care Databases for Scientific Research: A Comparison of EU-ADR, OMOP, Mini-Sentinel and MATRICE Strategies.
%V 4
%1 http://www.ncbi.nlm.nih.gov/pubmed/27014709?dopt=Abstract
%X INTRODUCTION: We see increased use of existing observational data in order to achieve fast and transparent production of empirical evidence in health care research. Multiple databases are often used to increase power, to assess rare exposures or outcomes, or to study diverse populations. For privacy and sociological reasons, original data on individual subjects can't be shared, requiring a distributed network approach where data processing is performed prior to data sharing.CASE DESCRIPTIONS AND VARIATION AMONG SITES: We created a conceptual framework distinguishing three steps in local data processing: (1) data reorganization into a data structure common across the network; (2) derivation of study variables not present in original data; and (3) application of study design to transform longitudinal data into aggregated data sets for statistical analysis. We applied this framework to four case studies to identify similarities and differences in the United States and Europe: Exploring and Understanding Adverse Drug Reactions by Integrative Mining of Clinical Records and Biomedical Knowledge (EU-ADR), Observational Medical Outcomes Partnership (OMOP), the Food and Drug Administration's (FDA's) Mini-Sentinel, and the Italian network-the Integration of Content Management Information on the Territory of Patients with Complex Diseases or with Chronic Conditions (MATRICE).FINDINGS: National networks (OMOP, Mini-Sentinel, MATRICE) all adopted shared procedures for local data reorganization. The multinational EU-ADR network needed locally defined procedures to reorganize its heterogeneous data into a common structure. Derivation of new data elements was centrally defined in all networks but the procedure was not shared in EU-ADR. Application of study design was a common and shared procedure in all the case studies. Computer procedures were embodied in different programming languages, including SAS, R, SQL, Java, and C++.CONCLUSION: Using our conceptual framework we found several areas that would benefit from research to identify optimal standards for production of empirical knowledge from existing databases.an opportunity to advance evidence-based care management. In addition, formalized CM outcomes assessment methodologies will enable us to compare CM effectiveness across health delivery settings.

%0 Journal Article
%A Leslie Grammatico-Guillon
%A Guillaume Gras
%A Claire Hassen-Khodja
%A Zoha Maakaroun
%A Frédéric Bastides
%A Francis Barin
%A Louis Bernard
%B J Antimicrob Chemother
%D 2015
%G ENG
%R 10.1093/jac/dkv242
%T Comment on: Persistence and adherence to single-tablet regimens in HIV treatment: a cohort study from the French National Healthcare Insurance Database.
%1 http://www.ncbi.nlm.nih.gov/pubmed/26253090?dopt=Abstract
%8 08/2015

%0 Journal Article
%A Antoine Neuraz
%A Claude Guérin
%A Cécile Payet
%A Stéphanie Polazzi
%A Frédéric Aubrun
%A Frédéric Dailler
%A Jean-Jacques Lehot
%A Vincent Piriou
%A Jean Neidecker
%A Thomas Rimmelé
%A Anne-Marie Schott
%A Antoine Duclos
%B Crit Care Med
%D 2015
%G eng
%N 8
%P 1587-94
%R 10.1097/CCM.0000000000001015
%T Patient Mortality Is Associated With Staff Resources and Workload in the ICU: A Multicenter Observational Study.
%V 43
%1 http://www.ncbi.nlm.nih.gov/pubmed/25867907?dopt=Abstract
%8 08/2015
%X OBJECTIVE: Matching healthcare staff resources to patient needs in the ICU is a key factor for quality of care. We aimed to assess the impact of the staffing-to-patient ratio and workload on ICU mortality.DESIGN: We performed a multicenter longitudinal study using routinely collected hospital data.SETTING: Information pertaining to every patient in eight ICUs from four university hospitals from January to December 2013 was analyzed.PATIENTS: A total of 5,718 inpatient stays were included.INTERVENTIONS: None.MEASUREMENTS AND MAIN RESULTS: We used a shift-by-shift varying measure of the patient-to-caregiver ratio in combination with workload to establish their relationships with ICU mortality over time, excluding patients with decision to forego life-sustaining therapy. Using a multilevel Poisson regression, we quantified ICU mortality-relative risk, adjusted for patient turnover, severity, and staffing levels. The risk of death was increased by 3.5 (95% CI, 1.3-9.1) when the patient-to-nurse ratio was greater than 2.5, and it was increased by 2.0 (95% CI, 1.3-3.2) when the patient-to-physician ratio exceeded 14. The highest ratios occurred more frequently during the weekend for nurse staffing and during the night for physicians (p  0.001). High patient turnover (adjusted relative risk, 5.6 [2.0-15.0]) and the volume of life-sustaining procedures performed by staff (adjusted relative risk, 5.9 [4.3-7.9]) were also associated with increased mortality.CONCLUSIONS: This study proposes evidence-based thresholds for patient-to-caregiver ratios, above which patient safety may be endangered in the ICU. Real-time monitoring of staffing levels and workload is feasible for adjusting caregivers' resources to patients' needs.

%0 Journal Article
%A Nagham Khanafer
%A Antoine Neuraz
%A Thomas Bénet
%A Martin Cour
%A Florence Persat
%A Hélène Labussière
%A Laurent Argaud
%A Mauricette Michallet
%A Philippe Vanhems
%B Anaerobe
%D 2015
%G eng
%P 98-100
%R 10.1016/j.anaerobe.2015.02.007
%T Acute graft-versus-host disease, invasive aspergillosis and Clostridium difficile colitis after peripheral blood stem cell transplantation: A complex network of causalities and a challenge for prevention.
%V 33
%1 http://www.ncbi.nlm.nih.gov/pubmed/25749258?dopt=Abstract
%8 06/2015
%X Graft-versus-host disease (GVHD) is a known risk factor for invasive aspergillosis (IA), but remains poorly studied in relation to Clostridium difficile infection (CDI). We report a case of a 58-years-old patient who developed an IA within a protected room, CDI and GVHD after allogeneic allogeneic peripheral blood stem cell transplantation (PBSCT). Factors associated with this complex condition in patients receiving allogeneic PBSCT need to be identified.

%0 Journal Article
%A Aude Fleurier
%A Cecile Pelatan
%A Stephanie Willot
%A Jean-Louis Ginies
%A Estelle Breton
%A Laure Bridoux
%A Jean-Francois Segura
%A Emilie Chaillou
%A Agathe Jobert
%A Estelle Darviot
%A Benoit Cagnard
%A Nadege Delaperriere
%A Isabelle Grimal
%A Emilie Carre
%A Anne-Claire Wagner
%A Emmanuelle Sylvestre
%A Alain Dabadie
%B Dig Liver Dis
%D 2015
%G eng
%N 6
%P 460-4
%R 10.1016/j.dld.2015.02.009
%T Vaccination coverage of children with inflammatory bowel disease after an awareness campaign on the risk of infection.
%V 47
%1 http://www.ncbi.nlm.nih.gov/pubmed/25770456?dopt=Abstract
%8 06/2015
%X BACKGROUND: Children with inflammatory bowel disease are at risk of vaccine-preventable diseases mostly due to immunosuppressive drugs.AIM: To evaluate coverage after an awareness campaign informing patients, their parents and general practitioner about the vaccination schedule.METHODS: Vaccination coverage was firstly evaluated and followed by an awareness campaign on the risk of infection via postal mail. The trial is a case-control study on the same patients before and after the awareness campaign. Overall, 92 children were included. A questionnaire was then completed during a routine appointment to collect data including age at diagnosis, age at data collection, treatment history, and vaccination status.RESULTS: Vaccination rates significantly increased for vaccines against diphtheria-tetanus-poliomyelitis (92% vs. 100%), Haemophilus influenzae (88% vs. 98%), hepatitis B (52% vs. 71%), pneumococcus (36% vs. 57%), and meningococcus C (17% vs. 41%) (p0.05). Children who were older at diagnosis were 1.26 times more likely to be up-to-date with a minimum vaccination schedule (diphtheria-tetanus-poliomyelitis, pertussis, H. influenzae, measles-mumps-rubella, tuberculosis) (p=0.002).CONCLUSION: Informing inflammatory bowel disease patients, their parents and general practitioner about the vaccination schedule via postal mail is easy, inexpensive, reproducible, and increases vaccination coverage. This method reinforces information on the risk of infection during routine visits.

%0 Journal Article
%A Faivre
%A Agopiantz
%A Loeb
%A Cassinari
%A Wack
%A Catoire
%A Braun
%A Thilly
%A Coudane
%B Rev Med Interne
%D 2015
%G FRE
%R 10.1016/j.revmed.2015.02.009
%T [Evaluation of the theoretical teaching of postgraduate medical students in France].
%1 http://www.ncbi.nlm.nih.gov/pubmed/25980929?dopt=Abstract
%8 05/2015
%X OBJECTIVES: In France, medical students regularly complain about the shortcomings of their theoretical training and the necessity of its adaptation to better fit the needs of students. The goal was to evaluate the theoretical teaching practices in postgraduate medical studies by: 1) collecting data from medical students in different medical faculties in France; 2) comparing this data with expected practices when it is possible; 3) and proposing several lines of improvement.METHODS: A survey of theoretical practices in the 3rd cycle of medical studies was conducted by self-administered questionnaires which were free of charge, anonymous, and administered electronically from July 3 to October 31, 2013 to all medical students in France.RESULTS: National, inter-regional, regional and field internship educational content was absent in respectively 50.5%, 42.8%, 26.0% and 30.2% of cases. Medical students follow complementary training due to insufficient DES and/or DESC 2 training in 43.7% of cases or as part of a professional project in 54.9% of cases. The knowledge sought by medical students concerns the following crosscutting topics: career development (58.9%), practice management (50.7%), medical English (50.4%) and their specialty organization (49.9%). Fifty-four point one percent would like to be evaluated on their theoretical training on an annual basis.CONCLUSION: The results of this first national survey give insights into the theoretical teaching conditions in postgraduate medical education in France and the aspirations of medical students.

%0 Journal Article
%A Vincent Canuel
%A Bastien Rance
%A Paul Avillach
%A Patrice Degoulet
%A Anita Burgun
%B Brief Bioinform
%D 2015
%G eng
%N 2
%P 280-90
%R 10.1093/bib/bbu006
%T Translational research platforms integrating clinical and omics data: a review of publicly available solutions.
%V 16
%1 http://www.ncbi.nlm.nih.gov/pubmed/24608524?dopt=Abstract
%8 03/2015
%X The rise of personalized medicine and the availability of high-throughput molecular analyses in the context of clinical care have increased the need for adequate tools for translational researchers to manage and explore these data. We reviewed the biomedical literature for translational platforms allowing the management and exploration of clinical and omics data, and identified seven platforms: BRISK, caTRIP, cBio Cancer Portal, G-DOC, iCOD, iDASH and tranSMART. We analyzed these platforms along seven major axes. (1) The community axis regrouped information regarding initiators and funders of the project, as well as availability status and references. (2) We regrouped under the information content axis the nature of the clinical and omics data handled by each system. (3) The privacy management environment axis encompassed functionalities allowing control over data privacy. (4) In the analysis support axis, we detailed the analytical and statistical tools provided by the platforms. We also explored (5) interoperability support and (6) system requirements. The final axis (7) platform support listed the availability of documentation and installation procedures. A large heterogeneity was observed in regard to the capability to manage phenotype information in addition to omics data, their security and interoperability features. The analytical and visualization features strongly depend on the considered platform. Similarly, the availability of the systems is variable. This review aims at providing the reader with the background to choose the platform best suited to their needs. To conclude, we discuss the desiderata for optimal translational research platforms, in terms of privacy, interoperability and technical features.

%0 Journal Article
%A Kelly Regan
%A Satyajeet Raje
%A Cartik Saravanamuthu
%A Philip Payne
%B Stud Health Technol Inform
%D 2015
%G eng
%P 663-7
%T Conceptual Knowledge Discovery in Databases for Drug Combinations Predictions in Malignant Melanoma
%V 216
%1 http://www.ncbi.nlm.nih.gov/pubmed/26262134?dopt=Abstract
%X The worldwide incidence of melanoma is rising faster than any other cancer, and prognosis for patients with metastatic disease is poor. Current targeted therapies are limited in their durability and/or effect size in certain patient populations due to acquired mechanisms of resistance. Thus, the development of synergistic combinatorial treatment regimens holds great promise to improve patient outcomes. We have previously shown that a model for in-silico knowledge discovery, Translational Ontology-anchored Knowledge Discovery Engine (TOKEn), is able to generate valid relationships between bimolecular and clinical phenotypes. In this study, we have aggregated observational and canonical knowledge consisting of melanoma-related biomolecular entities and targeted therapeutics in a computationally tractable model. We demonstrate here that the explicit linkage of therapeutic modalities with biomolecular underpinnings of melanoma utilizing the TOKEn pipeline yield a set of informed relationships that have the potential to generate combination therapy strategies.

%0 Journal Article
%A Doshi-Velez
%A Avillach
%A Palmer
%A Bousvaros
%A Ge
%A Fox
%A Steinberg
%A Spettell
%A Juster
%A Kohane
%B Inflammatory Bowel Disease
%D 2015
%G eng
%T Prevalence of Inflammatory Bowel Disease Among Patience with Autism Spectrum Disorders
%U http://www.ncbi.nlm.nih.gov/pubmed/26218138
%X <p id="">Background: The objective of this study was to measure the prevalence of inflammatory bowel disease (IBD) among patients with autism spectrum disorders (ASD), which has not been well described previously.</p><p id="">Methods: The rates of IBD among patients with and without ASD were measured in 4 study populations with distinct modes of ascertainment: a health care benefits company, 2 pediatric tertiary care centers, and a national ASD repository. The rates of IBD (established through International Classification of Diseases, Ninth Revision, Clinical Modification [ICD-9-CM] codes) were compared with respective controls and combined using a Stouffer meta-analysis. Clinical charts were also reviewed for IBD among patients with ICD-9-CM codes for both IBD and ASD at one of the pediatric tertiary care centers. This expert-verified rate was compared with the rate in the repository study population (where IBD diagnoses were established by expert review) and in nationally reported rates for pediatric IBD.</p><p id="">Results: In all of case-control study populations, the rates of IBD-related ICD-9-CM codes for patients with ASD were significantly higher than that of their respective controls (Stouffer meta-analysis, P &lt; 0.001). Expert-verified rates of IBD among patients with ASD were 7 of 2728 patients in one study population and 16 of 7201 in a second study population. The age-adjusted prevalence of IBD among patients with ASD was higher than their respective controls and nationally reported rates of pediatric IBD.</p><p id="">Conclusions: Across each population with different kinds of ascertainment, there was a consistent and statistically significant increased prevalance of IBD in patients with ASD than their respective controls and nationally reported rates for pediatric IBD.</p>

%0 Journal Article
%A Girardeau
%A Trivin
%A Durieux
%A Le Beller
%A LL Louet Agnes
%A Neuraz
%A Degoulet
%A Avillach
%B Drug Safety
%D 2015
%G eng
%T Detection of Drug-Drug Interactions Inducing Acute Kidney Injury by Electronic Health Records Mining
%X <div><h4>BACKGROUND AND OBJECTIVE:</h4><p>While risk of acute kidney injury (AKI) is a well documented adverse effect of some drugs, few studies have assessed the relationship between drug-drug interactions (DDIs) and AKI. Our objective was to develop an algorithm capable of detecting potential signals on this relationship by retrospectively mining data from electronic health records.</p><h4>MATERIAL AND METHODS:</h4><p>Data were extracted from the clinical data warehouse (CDW) of the Hôpital Européen Georges Pompidou (HEGP). AKI was defined as the first level of the RIFLE criteria, that is, an increase ≥50 % of creatinine basis. Algorithm accuracy was tested on 20 single drugs, 10 nephrotoxic and 10 non-nephrotoxic. We then tested 45 pairs of non-nephrotoxic drugs, among the most prescribed at our hospital and representing distinct pharmacological classes for DDIs.</p><h4>RESULTS:</h4><p>Sensitivity and specificity were 50 % [95 % confidence interval (CI) 23.66-76.34] and 90 % (95 % CI 59.58-98.21), respectively, for single drugs. Our algorithm confirmed a previously identified signal concerning clarithromycin and calcium-channel blockers (unadjusted odds ratio (ORu) 2.92; 95 % CI 1.11-7.69, p = 0.04). Among the 45 drug pairs investigated, we identified a signal concerning 55 patients in association with bromazepam and hydroxyzine (ORu 1.66; 95 % CI 1.23-2.23). This signal was not confirmed after a chart review. Even so, AKI and co-prescription were confirmed for 96 % (95 % CI 88-99) and 88 % (95 % CI 76-94) of these patients, respectively.</p><h4>CONCLUSION:</h4><p>Data mining techniques on CDW can foster the detection of adverse drug reactions when drugs are used alone or in combination.</p></div>

%0 Journal Article
%A Cartik Kothari
%A Philip Payne
%B Stud Health Technol Inform
%D 2015
%G ENG
%P 1071
%T A Metadata based Knowledge Discovery Methodology for Seeding Translational Research.
%V 216
%1 http://www.ncbi.nlm.nih.gov/pubmed/26262370?dopt=Abstract
%X In this paper, we present a semantic, metadata based knowledge discovery methodology for identifying teams of researchers from diverse backgrounds who can collaborate on interdisciplinary research projects: projects in areas that have been identified as high-impact areas at The Ohio State University. This methodology involves the semantic annotation of keywords and the postulation of semantic metrics to improve the efficiency of the path exploration algorithm as well as to rank the results. Results indicate that our methodology can discover groups of experts from diverse areas who can collaborate on translational research projects.

%0 Journal Article
%A Guillaume Bouzillé
%A Emmanuelle Sylvestre
%A Boris Campillo-Gimenez
%A Eric Renault
%A Thibault Ledieu
%A Denis Delamarre
%A Marc Cuggia
%B Stud Health Technol Inform
%D 2015
%G ENG
%P 913
%T An Integrated Workflow For Secondary Use of Patient Data for Clinical Research.
%V 216
%1 http://www.ncbi.nlm.nih.gov/pubmed/26262215?dopt=Abstract
%X This work proposes an integrated workflow for secondary use of medical data to serve feasibility studies, and the prescreening and monitoring of research studies. All research issues are initially addressed by the Clinical Research Office through a research portal and subsequently redirected to relevant experts in the determined field of concentration. For secondary use of data, the workflow is then based on the clinical data warehouse of the hospital. A datamart with potentially eligible research candidates is constructed. Datamarts can either produce aggregated data, de-identified data, or identified data, according to the kind of study being treated. In conclusion, integrating the secondary use of data process into a general research workflow allows visibility of information technologies and improves the accessability of clinical data.

%0 Journal Article
%A Maxime Wack
%A Etienne Puymirat
%A Brigitte Ranque
%A Sophie Georgin-Lavialle
%A Isabelle Pierre
%A Aurelia Tanguy
%A Felix Ackermann
%A Celine Mallet
%A Juliette Pavie
%A Hakima Boultache
%A Pierre Durieux
%A Paul Avillach
%B PLoS One
%D 2015
%G ENG
%N 9
%P e0138094
%R 10.1371/journal.pone.0138094
%T Evaluating the Impact of Computerized Provider Order Entry on Medical Students Training at Bedside: A Randomized Controlled Trial.
%V 10
%1 http://www.ncbi.nlm.nih.gov/pubmed/26367388?dopt=Abstract
%X OBJECTIVE: To evaluate the impact of computerized provider order entry (CPOE) at the bedside on medical students training.MATERIALS AND METHODS: We conducted a randomized cross-controlled educational trial on medical students during two clerkship rotations in three departments, assessing the impact of the use of CPOE on their ability to place adequate monitoring and therapeutic orders using a written test before and after each rotation. Students' satisfaction with their practice and the order placement system was surveyed. A multivariate mixed model was used to take individual students and chief resident (CR) effects into account. Factorial analysis was applied on the satisfaction questionnaire to identify dimensions, and scores were compared on these dimensions.RESULTS: Thirty-six students show no better progress (beginning and final test means = 69.87 and 80.98 points out of 176 for the control group, 64.60 and 78.11 for the CPOE group, p = 0.556) during their rotation in either group, even after adjusting for each student and CR, but show a better satisfaction with patient care and greater involvement in the medical team in the CPOE group (p = 0.035*). Both groups have a favorable opinion regarding CPOE as an educational tool, especially because of the order reviewing by the supervisor.CONCLUSION: This is the first randomized controlled trial assessing the performance of CPOE in both the progress in prescriptions ability and satisfaction of the students. The absence of effect on the medical skills must be weighted by the small time scale and low sample size. However, students are more satisfied when using CPOE rather than usual training.

%0 Journal Article
%A Avillach
%A Kerdelhué
%A Devos
%A Maisonneuve
%A Darmoni
%B Rev Epidemiol Sante Publique
%D 2014
%G fre
%N 6
%P 361-5
%R 10.1016/j.respe.2014.09.008
%T Limiting a Medline/PubMed query to the "best" articles using the JCR relative impact factor.
%V 62
%1 http://www.ncbi.nlm.nih.gov/pubmed/25454750?dopt=Abstract
%8 12/2014
%X <p>BACKGROUND: Medline/PubMed is the most frequently used medical bibliographic research database. The aim of this study was to propose a new generic method to limit any Medline/PubMed query based on the relative impact factor and the A &amp; B categories of the SIGAPS score. MATERIAL AND METHODS: The entire PubMed corpus was used for the feasibility study, then ten frequent diseases in terms of PubMed indexing and the citations of four Nobel prize winners. The relative impact factor (RIF) was calculated by medical specialty defined in Journal Citation Reports. The two queries, which included all the journals in category A (or A OR B), were added to any Medline/PubMed query as a central point of the feasibility study. RESULTS: Limitation using the SIGAPS category A was larger than the when using the Core Clinical Journals (CCJ): 15.65% of PubMed corpus vs 8.64% for CCJ. The response time of this limit applied to the entire PubMed corpus was less than two seconds. For five diseases out of ten, limiting the citations with the RIF was more effective than with the CCJ. For the four Nobel prize winners, limiting the citations with the RIF was more effective than the CCJ. CONCLUSION: The feasibility study to apply a new filter based on the relative impact factor on any Medline/PubMed query was positive.</p>

%0 Journal Article
%A Hassen-Khodja
%A Gras
%A Grammatico-Guillon
%A Dupuy
%A Gomez
%A Freslon
%A Dailloux
%A Soufflet
%A Bernard
%B Med Mal Infect
%D 2014
%G eng
%N 9
%P 423-8
%R 10.1016/j.medmal.2014.08.004
%T Hospital and ambulatory management, and compliance to treatment in HIV infection: regional health insurance agency analysis.
%V 44
%1 http://www.ncbi.nlm.nih.gov/pubmed/25234379?dopt=Abstract
%8 09/2014
%X OBJECTIVE: We had for objective to study HIV management (hospital, ambulatory, and mixed) and assess compliance with health insurance database.METHOD: We conducted a retrospective study using the French Social Security (CPAM) database. The inclusion criteria were: age&gt;18years of age, at least 2 prescriptions of antiretroviral therapy.RESULTS: Five hundred and seventy-five patients were included: extra-hospital (12), hospital (162), mixed (401). The prescriptions were exclusively hospital issued for 76.2% of the patients. Among the mixed group patients, 91% of treatments were delivered at least once in the community, and 45.6% of biological tests were performed in private laboratories at least once. The sex ratio (2.1 vs. 1.3), the number of patients having switched antiretroviral therapy (36.7% vs. 27.8%), and the frequency of biological tests (3.1 vs. 2.6) were significantly higher in the mixed group compared to the hospital group. The mean compliance was 90% in the hospital group and 91.8% in the mixed group. The compliance was80% for 104 patients (21.8%). Patients with≥80% compliance were older (46.1years of age vs. 42.7years of age), with more frequent biological tests (3 per year vs. 2.5 per year), and more frequent switches in treatment (35.4% vs. 26.0%).CONCLUSION: Prescriptions of ARV were almost exclusively hospital issued. Their dispensation and biological tests were split between hospital and extra-hospital settings. Most patients demonstrated an optimal compliance. The CPAM database allows describing HIV management and assessing compliance.

%0 Journal Article
%A Virginie Planche
%A Sophie Georgin-Lavialle
%A Paul Avillach
%A Brigitte Ranque
%A Juliette Pavie
%A Thibaut Caruba
%A Luc Darnige
%A Jacques Pouchot
%B Am J Hematol
%D 2014
%G eng
%N 6
%P 665-6
%R 10.1002/ajh.23718
%T Etiologies and diagnostic work-up of extreme macrocytosis defined by an erythrocyte mean corpuscular volume over 130°fL: A study of 109 patients.
%V 89
%1 http://www.ncbi.nlm.nih.gov/pubmed/24668797?dopt=Abstract
%8 06/2014

%0 Journal Article
%A Nicolas Pallet
%A Sophie Chauvet
%A Jean-François Chassé
%A Marc Vincent
%A Paul Avillach
%A Charlene Levi
%A Vannary Meas-Yedid
%A Jean-Christophe Olivo-Marin
%A Diane Nga-Matsogo
%A Philippe Beaune
%A Eric Thervet
%A Alexandre Karras
%B PLoS One
%D 2014
%G eng
%N 1
%P e84708
%R 10.1371/journal.pone.0084708
%T Urinary retinol binding protein is a marker of the extent of interstitial kidney fibrosis.
%V 9
%1 http://www.ncbi.nlm.nih.gov/pubmed/24416268?dopt=Abstract
%X Currently, a non-invasive method to estimate the degree of interstitial fibrosis (IF) in chronic kidney disease is not available in routine. The aim of our study was to evaluate the diagnostic performance of the measurement of urinary low molecular weight (LMW) protein concentrations as a method to determine the extent of IF. The urines specimen from 162 consecutive patients who underwent renal biopsy were used in the analysis. Numerical quantification software based on the colorimetric analysis of fibrous areas was used to assess the percentage IF. Total proteinuria, albuminuria, and the urinary levels of retinol binding protein (RBP), alpha1-microglobulin (α1MG), beta 2-microglobulin (β2MG), transferrin, and IgG immunoglobulins were measured. There was a significant correlation between the degree of IF and the RBP/creatinine (creat) ratio (R2: 0.11, p0.0001). IF was associated to a lesser extent with urinary β2MG and α1MG; however, there was no association with total proteinuria or high molecular weight (HMW) proteinuria. The correlation between IF and RBP/creat remained significant after adjustment to the estimated glomerular filtration rate, age, body mass index, α1MG, and β2MG. The specificity of the test for diagnosing a fibrosis score of &gt;25% of the parenchyma was 95% when using a threshold of 20 mg/g creat. In conclusion, RBP appears to be a quantitative and non-invasive marker for the independent prediction of the extent of kidney IF. Because methods for the measurement of urinary RBP are available in most clinical chemistry departments, RBP measurement is appealing for implementation in the routine care of patients with chronic kidney disease.

%0 Journal Article
%A Eloi Marijon
%A Wulfran Bougouin
%A David Celermajer
%A Marie-Cécile Périer
%A Florence Dumas
%A Nordine Benameur
%A Nicole Karam
%A Lionel Lamhaut
%A Muriel Tafflet
%A Hazrije Mustafic
%A de Natalia Machado Deus
%A Jean-Yves Le Heuzey
%A Michel Desnos
%A Paul Avillach
%A Christian Spaulding
%A Alain Cariou
%A Christof Prugger
%A Jean-Philippe Empana
%A Xavier Jouven
%B Circ Arrhythm Electrophysiol
%D 2013
%G eng
%N 6
%P 1185-91
%R 10.1161/CIRCEP.113.000651
%T Characteristics and outcomes of sudden cardiac arrest during sports in women.
%V 6
%1 http://www.ncbi.nlm.nih.gov/pubmed/24190898?dopt=Abstract
%8 12/2013
%X BACKGROUND: No specific data are available on characteristics and outcome of sudden cardiac death (SCD) during sport activities among women in the general population.METHODS AND RESULTS: From a prospective 5-year national survey, involving 820 subjects 10 to 75 years old who presented with SCD (resuscitated or not) during competitive or recreational sport activities, 43 (5.2%) such events occurred in women, principally during jogging, cycling, and swimming. The level of activity at the time of SCD was moderate to vigorous in 35 cases (81.4%). The overall incidence of sport-related SCD, among 15- to 75-year-old women, was estimated as 0.59 (95% confidence interval [CI], 0.39-0.79) to 2.17 (95% CI, 1.38-2.96) per year per million female sports participants for the 80th and 20th percentiles of reporting districts, respectively. Compared with men, the incidence of SCDs in women was dramatically lower, particularly in the 45- to 54-year range (relative risk, 0.033; 95% CI, 0.015-0.075). Despite similar circumstances of occurrence, survival at hospital admission (46.5%; 95% CI, 31.0-60.0) was significantly higher than that for men (30.0%; 95% CI, 26.8-33.2; P=0.02), although this did not reach statistical significance for hospital discharge. Favorable neurological outcomes were similar (80%). Cause of death seemed less likely to be associated with structural heart disease in women compared with men (58.3% versus 95.8%; P=0.003).CONCLUSIONS: Sports-related SCDs in women participants seems dramatically less common (up to 30-fold less frequent) compared with men. Our results also suggest a higher likelihood of successful resuscitation as well as less frequency of structural heart disease in women compared with men.

%0 Journal Article
%A Eloi Marijon
%A Wulfran Bougouin
%A David Celermajer
%A Marie-Cécile Perier
%A Nordine Benameur
%A Lionel Lamhaut
%A Nicole Karam
%A Florence Dumas
%A Muriel Tafflet
%A Christof Prugger
%A Hazrije Mustafic
%A Jean-Pierre Rifler
%A Michel Desnos
%A Jean-Yves Le Heuzey
%A Christian Spaulding
%A Paul Avillach
%A Alain Cariou
%A Jean-Philippe Empana
%A Xavier Jouven
%B Eur Heart J
%D 2013
%G eng
%N 47
%P 3632-40
%R 10.1093/eurheartj/eht282
%T Major regional disparities in outcomes after sudden cardiac arrest during sports.
%V 34
%1 http://www.ncbi.nlm.nih.gov/pubmed/23918760?dopt=Abstract
%8 12/2013
%X AIMS: Characteristics of sudden cardiac arrest (SCA) during sports offers a novel (and unexplored) setting to assess factors associated with disparities in outcomes across regions.METHODS AND RESULTS: From a prospective 5-year community-based French registry concerning SCA during sports in 10-75 year-olds, we evaluated whether outcomes differed significantly between geographic regions. We then determined the extent to which variations in community-related early interventions were associated with regional variations in survival. Among 820 SCA cases studied, overall survival at hospital discharge was 15.7% (95% confidence interval, 13.2-18.2%), with considerable regional disparities (from 3.4 to 42.6%, P  0.001). Major differences were noted regarding bystander initiation of cardiopulmonary resuscitation (15.3-80.9%, P  0.001) and presence of initial shockable rhythm (28.6-79.1%, P  0.001), with higher values of these being associated with better survival rates. The proportion of survivors with favourable neurological outcome at discharge was fairly uniform among survival groups (CPC-1/2, varying from 77.4 to 90.0%, P = 0.83). No difference was observed regarding subjects' characteristics and circumstances of SCA occurrence, including delays in resuscitation (collapse-to-call period). With a comparable in-hospital mortality (P = 0.44), survival at hospital discharge was highly correlated with that at hospital admission (regional variations from 7.4 to 75.0%, P  0.001).CONCLUSION: Major regional disparities exist in survival rates (up to 10-fold) after SCA during sports. SCA cases from regions with the highest levels of bystander resuscitation had the best survival rates to hospital admission and discharge.

%0 Journal Article
%A José Luis Oliveira
%A Pedro Lopes
%A Tiago Nunes
%A David Campos
%A Scott Boyer
%A Ernst Ahlberg
%A van Erik Mulligen
%A Jan Kors
%A Bharat Singh
%A Laura Furlong
%A Ferran Sanz
%A Anna Bauer-Mehren
%A Maria Carrascosa
%A Jordi Mestres
%A Paul Avillach
%A Gayo Diallo
%A Carlos Díaz Acedo
%A van der Johan Lei
%B Pharmacoepidemiol Drug Saf
%D 2013
%G eng
%N 5
%P 459-67
%R 10.1002/pds.3375
%T The EU-ADR Web Platform: delivering advanced pharmacovigilance tools.
%V 22
%1 http://www.ncbi.nlm.nih.gov/pubmed/23208789?dopt=Abstract
%8 05/2013
%X PURPOSE: Pharmacovigilance methods have advanced greatly during the last decades, making post-market drug assessment an essential drug evaluation component. These methods mainly rely on the use of spontaneous reporting systems and health information databases to collect expertise from huge amounts of real-world reports. The EU-ADR Web Platform was built to further facilitate accessing, monitoring and exploring these data, enabling an in-depth analysis of adverse drug reactions risks.METHODS: The EU-ADR Web Platform exploits the wealth of data collected within a large-scale European initiative, the EU-ADR project. Millions of electronic health records, provided by national health agencies, are mined for specific drug events, which are correlated with literature, protein and pathway data, resulting in a rich drug-event dataset. Next, advanced distributed computing methods are tailored to coordinate the execution of data-mining and statistical analysis tasks. This permits obtaining a ranked drug-event list, removing spurious entries and highlighting relationships with high risk potential.RESULTS: The EU-ADR Web Platform is an open workspace for the integrated analysis of pharmacovigilance datasets. Using this software, researchers can access a variety of tools provided by distinct partners in a single centralized environment. Besides performing standalone drug-event assessments, they can also control the pipeline for an improved batch analysis of custom datasets. Drug-event pairs can be substantiated and statistically analysed within the platform's innovative working environment.CONCLUSIONS: A pioneering workspace that helps in explaining the biological path of adverse drug reactions was developed within the EU-ADR project consortium. This tool, targeted at the pharmacovigilance community, is available online at https://bioinformatics.ua.pt/euadr/.

%0 Journal Article
%A Paul Avillach
%A Jean-Charles Dufour
%A Gayo Diallo
%A Francesco Salvo
%A Michel Joubert
%A Frantz Thiessard
%A Fleur Mougin
%A Gianluca Trifirò
%A Annie Fourrier-Réglat
%A Antoine Pariente
%A Marius Fieschi
%B J Am Med Inform Assoc
%D 2013
%G eng
%N 3
%P 446-52
%R 10.1136/amiajnl-2012-001083
%T Design and validation of an automated method to detect known adverse drug reactions in MEDLINE: a contribution from the EU-ADR project.
%V 20
%1 http://www.ncbi.nlm.nih.gov/pubmed/23195749?dopt=Abstract
%8 05/2013
%X OBJECTIVES: The aim of this research was to automate the search of publications concerning adverse drug reactions (ADR) by defining the queries used to search MEDLINE and by determining the required threshold for the number of extracted publications to confirm the drug/event association in the literature.METHODS: We defined an approach based on the medical subject headings (MeSH) 'descriptor records' and 'supplementary concept records' thesaurus, using the subheadings 'chemically induced' and 'adverse effects' with the 'pharmacological action' knowledge. An expert-built validation set of true positive and true negative drug/adverse event associations (n=61) was used to validate our method.RESULTS: Using a threshold of three of more extracted publications, the automated search method presented a sensitivity of 90% and a specificity of 100%. For nine different drug/event pairs selected, the recall of the automated search ranged from 24% to 64% and the precision from 93% to 48%.CONCLUSIONS: This work presents a method to find previously established relationships between drugs and adverse events in the literature. Using MEDLINE, following a MeSH approach to filter the signals, is a valid option. Our contribution is available as a web service that will be integrated in the final European EU-ADR project (Exploring and Understanding Adverse Drug Reactions by integrative mining of clinical records and biomedical knowledge) automated system.

%0 Journal Article
%A Preciosa Coloma
%A Paul Avillach
%A Francesco Salvo
%A Martijn Schuemie
%A Carmen Ferrajolo
%A Antoine Pariente
%A Annie Fourrier-Réglat
%A Mariam Molokhia
%A Vaishali Patadia
%A van der Johan Lei
%A Miriam Sturkenboom
%A Gianluca Trifirò
%B Drug Saf
%D 2013
%G eng
%N 1
%P 13-23
%R 10.1007/s40264-012-0002-x
%T A reference standard for evaluation of methods for drug safety signal detection using electronic healthcare record databases.
%V 36
%1 http://www.ncbi.nlm.nih.gov/pubmed/23315292?dopt=Abstract
%8 01/2013
%X BACKGROUND: The growing interest in using electronic healthcare record (EHR) databases for drug safety surveillance has spurred development of new methodologies for signal detection. Although several drugs have been withdrawn postmarketing by regulatory authorities after scientific evaluation of harms and benefits, there is no definitive list of confirmed signals (i.e. list of all known adverse reactions and which drugs can cause them). As there is no true gold standard, prospective evaluation of signal detection methods remains a challenge.OBJECTIVE: Within the context of methods development and evaluation in the EU-ADR Project (Exploring and Understanding Adverse Drug Reactions by integrative mining of clinical records and biomedical knowledge), we propose a surrogate reference standard of drug-adverse event associations based on existing scientific literature and expert opinion.METHODS: The reference standard was constructed for ten top-ranked events judged as important in pharmacovigilance. A stepwise approach was employed to identify which, among a list of drug-event associations, are well recognized (known positive associations) or highly unlikely ('negative controls') based on MEDLINE-indexed publications, drug product labels, spontaneous reports made to the WHO's pharmacovigilance database, and expert opinion. Only drugs with adequate exposure in the EU-ADR database network (comprising ≈60 million person-years of healthcare data) to allow detection of an association were considered. Manual verification of positive associations and negative controls was independently performed by two experts proficient in clinical medicine, pharmacoepidemiology and pharmacovigilance. A third expert adjudicated equivocal cases and arbitrated any disagreement between evaluators.RESULTS: Overall, 94 drug-event associations comprised the reference standard, which included 44 positive associations and 50 negative controls for the ten events of interest: bullous eruptions; acute renal failure; anaphylactic shock; acute myocardial infarction; rhabdomyolysis; aplastic anaemia/pancytopenia; neutropenia/agranulocytosis; cardiac valve fibrosis; acute liver injury; and upper gastrointestinal bleeding. For cardiac valve fibrosis, there was no drug with adequate exposure in the database network that satisfied the criteria for a positive association.CONCLUSION: A strategy for the construction of a reference standard to evaluate signal detection methods that use EHR has been proposed. The resulting reference standard is by no means definitive, however, and should be seen as dynamic. As knowledge on drug safety evolves over time and new issues in drug safety arise, this reference standard can be re-evaluated.

%0 Journal Article
%A Antoine Neuraz
%A Laurent Chouchana
%A Georgia Malamut
%A Christine Le Beller
%A Denis Roche
%A Philippe Beaune
%A Patrice Degoulet
%A Anita Burgun
%A Marie-Anne Loriot
%A Paul Avillach
%B PLoS Comput Biol
%D 2013
%G eng
%N 12
%P e1003405
%R 10.1371/journal.pcbi.1003405
%T Phenome-wide association studies on a quantitative trait: application to TPMT enzyme activity and thiopurine therapy in pharmacogenomics.
%V 9
%1 http://www.ncbi.nlm.nih.gov/pubmed/24385893?dopt=Abstract
%X Phenome-Wide Association Studies (PheWAS) investigate whether genetic polymorphisms associated with a phenotype are also associated with other diagnoses. In this study, we have developed new methods to perform a PheWAS based on ICD-10 codes and biological test results, and to use a quantitative trait as the selection criterion. We tested our approach on thiopurine S-methyltransferase (TPMT) activity in patients treated by thiopurine drugs. We developed 2 aggregation methods for the ICD-10 codes: an ICD-10 hierarchy and a mapping to existing ICD-9-CM based PheWAS codes. Eleven biological test results were also analyzed using discretization algorithms. We applied these methods in patients having a TPMT activity assessment from the clinical data warehouse of a French academic hospital between January 2000 and July 2013. Data after initiation of thiopurine treatment were analyzed and patient groups were compared according to their TPMT activity level. A total of 442 patient records were analyzed representing 10,252 ICD-10 codes and 72,711 biological test results. The results from the ICD-9-CM based PheWAS codes and ICD-10 hierarchy codes were concordant. Cross-validation with the biological test results allowed us to validate the ICD phenotypes. Iron-deficiency anemia and diabetes mellitus were associated with a very high TPMT activity (p = 0.0004 and p = 0.0015, respectively). We describe here an original method to perform PheWAS on a quantitative trait using both ICD-10 diagnosis codes and biological test results to identify associated phenotypes. In the field of pharmacogenomics, PheWAS allow for the identification of new subgroups of patients who require personalized clinical and therapeutic management.

%0 Journal Article
%A Pedro Lopes
%A Tiago Nunes
%A David Campos
%A Laura Ines Furlong
%A Anna Bauer-Mehren
%A Ferran Sanz
%A Maria Carmen Carrascosa
%A Jordi Mestres
%A Jan Kors
%A Bharat Singh
%A van Erik Mulligen
%A van der Johan Lei
%A Gayo Diallo
%A Paul Avillach
%A Ernst Ahlberg
%A Scott Boyer
%A Carlos Diaz
%A José Luís Oliveira
%B PLoS One
%D 2013
%G eng
%N 12
%P e83016
%R 10.1371/journal.pone.0083016
%T Gathering and exploring scientific knowledge in pharmacovigilance.
%V 8
%1 http://www.ncbi.nlm.nih.gov/pubmed/24349421?dopt=Abstract
%X Pharmacovigilance plays a key role in the healthcare domain through the assessment, monitoring and discovery of interactions amongst drugs and their effects in the human organism. However, technological advances in this field have been slowing down over the last decade due to miscellaneous legal, ethical and methodological constraints. Pharmaceutical companies started to realize that collaborative and integrative approaches boost current drug research and development processes. Hence, new strategies are required to connect researchers, datasets, biomedical knowledge and analysis algorithms, allowing them to fully exploit the true value behind state-of-the-art pharmacovigilance efforts. This manuscript introduces a new platform directed towards pharmacovigilance knowledge providers. This system, based on a service-oriented architecture, adopts a plugin-based approach to solve fundamental pharmacovigilance software challenges. With the wealth of collected clinical and pharmaceutical data, it is now possible to connect knowledge providers' analysis and exploration algorithms with real data. As a result, new strategies allow a faster identification of high-risk interactions between marketed drugs and adverse events, and enable the automated uncovering of scientific evidence behind them. With this architecture, the pharmacovigilance field has a new platform to coordinate large-scale drug evaluation efforts in a unique ecosystem, publicly available at http://bioinformatics.ua.pt/euadr/.

%0 Journal Article
%A Sahut D'Izarn
%A Caumont Prim
%A Planquette
%A Revel
%A Avillach
%A Chatellier
%A Sanchez
%A Meyer
%B J Thromb Haemost
%D 2012
%G eng
%N 10
%P 2032-8
%R 10.1111/j.1538-7836.2012.04868.x
%T Risk factors and clinical outcome of unsuspected pulmonary embolism in cancer patients: a case-control study.
%V 10
%1 http://www.ncbi.nlm.nih.gov/pubmed/22845852?dopt=Abstract
%8 10/2012
%X BACKGROUND: Little is known about the risk factors and outcome of unsuspected pulmonary embolism (UPE) in cancer patients.OBJECTIVES: To assess the risk factors and outcome of UPE in cancer patients.METHODS: The charts of 66 patients diagnosed with UPE were reviewed. Two control groups were selected: 132 cancer patients without pulmonary embolism (PE) and 65 cancer patients with clinically suspected PE. Variables associated with UPE were identified by multivariable analysis. Six-month survival and recurrent venous thromboembolism were compared by use of Cox proportional analysis.RESULTS: Twenty-seven (40.9%) patients with UPE had symptoms suggesting PE. Adenocarcinoma (odds ratio [OR] 4.45; 95% confidence interval [CI] 1.98-9.97), advanced age (OR 1.18; 95% CI 1.02-1.38), recent chemotherapy (OR 4.62; 95% CI 2.26-9.44), performance status &gt; 2 (OR 7.31; 95% CI 1.90-28.15) and previous venous thromboembolism (OR 4.47; 95% CI 1.16-17.13) were associated with UPE. When adjusted for tumor stage and performance status, 6-month mortality did not differ between patients with UPE and patients without PE (hazard ratio 1.40; 95% CI 0.53-3.66; P = 0.50). Patients with UPE were more likely to have central venous catheters and chemotherapy and less likely to have proximal clots than patients with clinically suspected PE. Recurrent venous thromboembolism occurred in 6.1% and 7.7% of patients with UPE and symptomatic PE, respectively.CONCLUSION: UPE is not associated with an increased risk of death. Patients with clinically suspected PE and those with UPE have similar risks of recurrent venous thromboembolism.

%0 Journal Article
%A Anna Bauer-Mehren
%A van Erik Mullingen
%A Paul Avillach
%A María Del Carmen Carrascosa
%A Ricard Garcia-Serna
%A Janet Piñero
%A Bharat Singh
%A Pedro Lopes
%A José Oliveira
%A Gayo Diallo
%A Ernst Ahlberg Helgee
%A Scott Boyer
%A Jordi Mestres
%A Ferran Sanz
%A Jan Kors
%A Laura Furlong
%B PLoS Comput Biol
%D 2012
%G eng
%N 4
%P e1002457
%R 10.1371/journal.pcbi.1002457
%T Automatic filtering and substantiation of drug safety signals.
%V 8
%1 http://www.ncbi.nlm.nih.gov/pubmed/22496632?dopt=Abstract
%X Drug safety issues pose serious health threats to the population and constitute a major cause of mortality worldwide. Due to the prominent implications to both public health and the pharmaceutical industry, it is of great importance to unravel the molecular mechanisms by which an adverse drug reaction can be potentially elicited. These mechanisms can be investigated by placing the pharmaco-epidemiologically detected adverse drug reaction in an information-rich context and by exploiting all currently available biomedical knowledge to substantiate it. We present a computational framework for the biological annotation of potential adverse drug reactions. First, the proposed framework investigates previous evidences on the drug-event association in the context of biomedical literature (signal filtering). Then, it seeks to provide a biological explanation (signal substantiation) by exploring mechanistic connections that might explain why a drug produces a specific adverse reaction. The mechanistic connections include the activity of the drug, related compounds and drug metabolites on protein targets, the association of protein targets to clinical events, and the annotation of proteins (both protein targets and proteins associated with clinical events) to biological pathways. Hence, the workflows for signal filtering and substantiation integrate modules for literature and database mining, in silico drug-target profiling, and analyses based on gene-disease networks and biological pathways. Application examples of these workflows carried out on selected cases of drug safety signals are discussed. The methodology and workflows presented offer a novel approach to explore the molecular mechanisms underlying adverse drug reactions.

%0 Journal Article
%A Huoi
%A Casalegno
%A Bénet
%A Neuraz
%A Billaud
%A Eibach
%A Mekki
%A Rudigoz
%A Massardier
%A Huissoud
%A Massoud
%A Gaucherand
%A Claris
%A Gillet
%A Floret
%A Lina
%A Vanhems
%B Euro Surveill
%D 2012
%G eng
%N 36
%P 20264
%T A report on the large measles outbreak in Lyon, France, 2010 to 2011.
%V 17
%1 http://www.ncbi.nlm.nih.gov/pubmed/22971330?dopt=Abstract
%X In 2010 and 2011, the city of Lyon, located in the Rhône-Alpes region (France), has experienced one of the highest incidences of measles in Europe. We describe a measles outbreak in the Lyon area, where cases were diagnosed at Lyon University hospitals (LUH) between 2010 and mid-2011. Data were collected from the mandatory notification system of the regional public health agency, and from the virology department of the LUH. All patients and healthcare workers who had contracted measles were included. Overall, 407 cases were diagnosed, with children of less than one year of age accounting for the highest proportion (n=129, 32%), followed by individuals between 17 and 29 years-old (n=126, 31%). Of the total cases, 72 (18%) had complications. The proportions of patients and healthcare workers who were not immune to measles were higher among those aged up to 30 years. Consequently, women of childbearing age constituted a specific population at high risk to contract measles and during this outbreak, 13 cases of measles, seven under 30 years-old, were identified among pregnant women. This study highlights the importance of being vaccinated with two doses of measles vaccine, the only measure which could prevent and allow elimination of the disease.

%0 Journal Article
%A Gianluca Trifirò
%A Vaishali Patadia
%A Martijn Schuemie
%A Preciosa Coloma
%A Rosa Gini
%A Ron Herings
%A Julia Hippisley-Cox
%A Giampiero Mazzaglia
%A Carlo Giaquinto
%A Lorenza Scotti
%A Lars Pedersen
%A Paul Avillach
%A Miriam Sturkenboom
%A van der Johan Lei
%A van der Johan Lei
%B Stud Health Technol Inform
%D 2011
%G eng
%P 25-30
%T EU-ADR healthcare database network vs. spontaneous reporting system database: preliminary comparison of signal detection.
%V 166
%1 http://www.ncbi.nlm.nih.gov/pubmed/21685607?dopt=Abstract
%X The EU-ADR project aims to exploit different European electronic healthcare records (EHR) databases for drug safety signal detection. In this paper we report the preliminary results concerning the comparison of signal detection between EU-ADR network and two spontaneous reporting databases, the Food and Drug Administration and World Health Organization databases. EU-ADR data sources consist of eight databases in four countries (Denmark, Italy, Netherlands, and United Kingdom) that are virtually linked through distributed data network. A custom-built software (Jerboa©) elaborates harmonized input data that are produced locally and generates aggregated data which are then stored in a central repository. Those data are subsequently analyzed through different statistics (i.e. Longitudinal Gamma Poisson Shrinker). As potential signals, all the drugs that are associated to six events of interest (bullous eruptions - BE, acute renal failure - ARF, acute myocardial infarction - AMI, anaphylactic shock - AS, rhabdomyolysis - RHABD, and upper gastrointestinal bleeding - UGIB) have been detected via different data mining techniques in the two systems. Subsequently a comparison concerning the number of drugs that could be investigated and the potential signals detected for each event in the spontaneous reporting systems (SRSs) and EU-ADR network was made. SRSs could explore, as potential signals, a larger number of drugs for the six events, in comparison to EU-ADR (range: 630-3,393 vs. 87-856), particularly for those events commonly thought to be potentially drug-induced (i.e. BE: 3,393 vs. 228). The highest proportion of signals detected in SRSs was found for BE, ARF and AS, while for ARF, and UGIB in EU-ADR. In conclusion, it seems that EU-ADR longitudinal database network may complement traditional spontaneous reporting system for signal detection, especially for those adverse events that are frequent in general population and are not commonly thought to be drug-induced. The methodology for signal detection in EU-ADR is still under development and testing phase.

%0 Journal Article
%A Antoine Pariente
%A Marie Didailler
%A Paul Avillach
%A Ghada Miremont-Salamé
%A Annie Fourrier-Reglat
%A Françoise Haramburu
%A Nicholas Moore
%B Pharmacoepidemiol Drug Saf
%D 2010
%G eng
%N 11
%P 1166-71
%R 10.1002/pds.2022
%T A potential competition bias in the detection of safety signals from spontaneous reporting databases.
%V 19
%1 http://www.ncbi.nlm.nih.gov/pubmed/20848561?dopt=Abstract
%8 11/2010
%X PURPOSE: To study whether reports related to known drug-event associations could hinder the detection of new signals by increasing the detection thresholds when using disporportionality analyses in spontaneous reporting (SR) databases.METHODS: The French SR database (2005-2006 data) was used to test this hypothesis for the following events: bleeding, headache, hepatitis, myalgia, myocardial infarction, stroke, and toxic epidermal necrolysis (TEN). For each of these, using the Proportional Reporting Ratio (PRR) and the Reporting Odds Ratio (ROR), the number of cases needed to trigger a signal out of 50, 100, and 200 reports for a hypothetical newly introduced drug were computed before and after removing from the database reports involving drugs known to be associated with the event.RESULTS: For bleeding and stroke, removing potentially competitive data resulted in a decrease of the number of cases needed to trigger a signal for a newly introduced drug for both PRR and ROR (e.g., from 9 to 4, and 5 to 3 cases out of 50 reports for bleeding and stroke, respectively using the PRR). They were not or only slightly modified for the other studied events.CONCLUSIONS: Removing reports related to known drug-event associations could increase the sensitivity of signal detection in SR databases. This should be considered when using SR databases for signal detection as it could result in earlier identification of new drug-event associations.

%0 Journal Article
%A Paul Avillach
%A Michel Joubert
%A Frantz Thiessard
%A Gianluca Trifirò
%A Jean-Charles Dufour
%A Antoine Pariente
%A Fleur Mougin
%A Giovanni Polimeni
%A Maria Antonietta Catania
%A Carlo Giaquinto
%A Giampiero Mazzaglia
%A Carla Fornari
%A Ron Herings
%A Rosa Gini
%A Julia Hippisley-Cox
%A Mariam Molokhia
%A Lars Pedersen
%A Annie Fourrier-Réglat
%A Miriam Sturkenboom
%A Marius Fieschi
%B Stud Health Technol Inform
%D 2010
%G eng
%N Pt 2
%P 1085-9
%T Design and evaluation of a semantic approach for the homogeneous identification of events in eight patient databases: a contribution to the European EU-ADR project.
%V 160
%1 http://www.ncbi.nlm.nih.gov/pubmed/20841851?dopt=Abstract
%X The overall objective of the EU-ADR project is the design, development, and validation of a computerised system that exploits data from electronic health records and biomedical databases for the early detection of adverse drug reactions. Eight different databases, containing health records of more than 30 million European citizens, are involved in the project. Unique queries cannot be performed across different databases because of their heterogeneity: Medical record and Claims databases, four different terminologies for coding diagnoses, and two languages for the information described in free text. The aim of our study was to provide database owners with a common basis for the construction of their queries. Using the UMLS, we provided a list of medical concepts, with their corresponding terms and codes in the four terminologies, which should be considered to retrieve the relevant information for the events of interest from the databases.

%0 Journal Article
%A James Balhoff
%A Wasila Dahdul
%A Cartik Kothari
%A Hilmar Lapp
%A John Lundberg
%A Paula Mabee
%A Peter Midford
%A Monte Westerfield
%A Todd Vision
%B PLoS One
%D 2010
%G eng
%N 5
%P e10500
%R 10.1371/journal.pone.0010500
%T Phenex: ontological annotation of phenotypic diversity.
%V 5
%1 http://www.ncbi.nlm.nih.gov/pubmed/20463926?dopt=Abstract
%X BACKGROUND: Phenotypic differences among species have long been systematically itemized and described by biologists in the process of investigating phylogenetic relationships and trait evolution. Traditionally, these descriptions have been expressed in natural language within the context of individual journal publications or monographs. As such, this rich store of phenotype data has been largely unavailable for statistical and computational comparisons across studies or integration with other biological knowledge.METHODOLOGY/PRINCIPAL FINDINGS: Here we describe Phenex, a platform-independent desktop application designed to facilitate efficient and consistent annotation of phenotypic similarities and differences using Entity-Quality syntax, drawing on terms from community ontologies for anatomical entities, phenotypic qualities, and taxonomic names. Phenex can be configured to load only those ontologies pertinent to a taxonomic group of interest. The graphical user interface was optimized for evolutionary biologists accustomed to working with lists of taxa, characters, character states, and character-by-taxon matrices.CONCLUSIONS/SIGNIFICANCE: Annotation of phenotypic data using ontologies and globally unique taxonomic identifiers will allow biologists to integrate phenotypic data from different organisms and studies, leveraging decades of work in systematics and comparative morphology.

%0 Journal Article
%A Wasila Dahdul
%A James Balhoff
%A Jeffrey Engeman
%A Terry Grande
%A Eric Hilton
%A Cartik Kothari
%A Hilmar Lapp
%A John Lundberg
%A Peter Midford
%A Todd Vision
%A Monte Westerfield
%A Paula Mabee
%B PLoS One
%D 2010
%G eng
%N 5
%P e10708
%R 10.1371/journal.pone.0010708
%T Evolutionary characters, phenotypes and ontologies: curating data from the systematic biology literature.
%V 5
%1 http://www.ncbi.nlm.nih.gov/pubmed/20505755?dopt=Abstract
%X BACKGROUND: The wealth of phenotypic descriptions documented in the published articles, monographs, and dissertations of phylogenetic systematics is traditionally reported in a free-text format, and it is therefore largely inaccessible for linkage to biological databases for genetics, development, and phenotypes, and difficult to manage for large-scale integrative work. The Phenoscape project aims to represent these complex and detailed descriptions with rich and formal semantics that are amenable to computation and integration with phenotype data from other fields of biology. This entails reconceptualizing the traditional free-text characters into the computable Entity-Quality (EQ) formalism using ontologies.METHODOLOGY/PRINCIPAL FINDINGS: We used ontologies and the EQ formalism to curate a collection of 47 phylogenetic studies on ostariophysan fishes (including catfishes, characins, minnows, knifefishes) and their relatives with the goal of integrating these complex phenotype descriptions with information from an existing model organism database (zebrafish, http://zfin.org). We developed a curation workflow for the collection of character, taxonomic and specimen data from these publications. A total of 4,617 phenotypic characters (10,512 states) for 3,449 taxa, primarily species, were curated into EQ formalism (for a total of 12,861 EQ statements) using anatomical and taxonomic terms from teleost-specific ontologies (Teleost Anatomy Ontology and Teleost Taxonomy Ontology) in combination with terms from a quality ontology (Phenotype and Trait Ontology). Standards and guidelines for consistently and accurately representing phenotypes were developed in response to the challenges that were evident from two annotation experiments and from feedback from curators.CONCLUSIONS/SIGNIFICANCE: The challenges we encountered and many of the curation standards and methods for improving consistency that we developed are generally applicable to any effort to represent phenotypes using ontologies. This is because an ontological representation of the detailed variations in phenotype, whether between mutant or wildtype, among individual humans, or across the diversity of species, requires a process by which a precise combination of terms from domain ontologies are selected and organized according to logical relations. The efficiencies that we have developed in this process will be useful for any attempt to annotate complex phenotypic descriptions using ontologies. We also discuss some ramifications of EQ representation for the domain of systematics.

%0 Journal Article
%A Paul Avillach
%A Fleur Mougin
%A Michel Joubert
%A Frantz Thiessard
%A Antoine Pariente
%A Jean-Charles Dufour
%A Gianluca Trifirò
%A Giovanni Polimeni
%A Maria Antonietta Catania
%A Carlo Giaquinto
%A Giampiero Mazzaglia
%A Gianluca Baio
%A Ron Herings
%A Rosa Gini
%A Julia Hippisley-Cox
%A Mariam Molokhia
%A Lars Pedersen
%A Annie Fourrier-Réglat
%A Miriam Sturkenboom
%A Marius Fieschi
%B Stud Health Technol Inform
%D 2009
%G eng
%P 190-4
%T A semantic approach for the homogeneous identification of events in eight patient databases: a contribution to the European eu-ADR project.
%V 150
%1 http://www.ncbi.nlm.nih.gov/pubmed/19745295?dopt=Abstract
%X The overall objective of the eu-ADR project is the design, development, and validation of a computerised system that exploits data from electronic health records and biomedical databases for the early detection of adverse drug reactions. Eight different databases, containing health records of more than 30 million European citizens, are involved in the project. Unique queries cannot be performed across different databases because of their heterogeneity: Medical record and Claims databases, four different terminologies for coding diagnoses, and two languages for the information described in free text. The aim of our study was to provide database owners with a common basis for the construction of their queries. Using the UMLS, we provided a list of medical concepts, with their corresponding terms and codes in the four terminologies, which should be considered to retrieve the relevant information for the events of interest from the databases.

%0 Journal Article
%A Catherine Quantin
%A Béatrice Gouyon
%A Paul Avillach
%A Cyril Ferdynus
%A Paul Sagot
%A Jean-Bernard Gouyon
%B Int J Telemed Appl
%D 2009
%G eng
%P 181842
%R 10.1155/2009/181842
%T Using discharge abstracts to evaluate a regional perinatal network: assessment of the linkage procedure of anonymous data.
%V 2009
%1 http://www.ncbi.nlm.nih.gov/pubmed/19125196?dopt=Abstract
%X To assess the Burgundy perinatal network (18 obstetrical units; 18 500 births per year), discharge abstracts and additional data were collected for all mothers and newborns. In accordance with French law, data were rendered anonymous before statistical analysis, and were linked to patients using a specific procedure. This procedure allowed data concerning each mother to be linked to those for her newborn(s). This study showed that all mothers and newborns were included in the regional database; the data for all mothers were linked to those for their infant(s) in all cases. Additional data (gestational age) were obtained for 99.9% of newborns.

%0 Journal Article
%A Paul Avillach
%A Michel Joubert
%A Marius Fieschi
%B Health Care Manag Sci
%D 2008
%G eng
%N 2
%P 147-51
%T Improving the quality of the coding of primary diagnosis in standardized discharge summaries.
%V 11
%1 http://www.ncbi.nlm.nih.gov/pubmed/18581821?dopt=Abstract
%8 06/2008
%X We propose to design and test an information-processing model to participate in appraising the quality and the consistency of the coding, for billing, of Standardized Discharge Summaries (SDSs). We designed a model using both symbolic knowledge extracted from the NLM's UMLS and statistical knowledge. The aim is to retrieve from the ICD-10 terms recorded in a SDS the Principal Diagnosis (PD) at the time of coding. In 90% of cases the PD was retrieved 1st or 2nd in SDS including three ICD-10 codes or more. This model could contribute as part of an automated quality control process in a hospital information system by checking consistency in coded SDSs and improve the income of the hospital.

%0 Journal Article
%A Michel Joubert
%A Stefan Darmoni
%A Paul Avillach
%A Badisse Dahamna
%A Marius Fieschi
%B Stud Health Technol Inform
%D 2008
%G eng
%P 205-10
%T Using knowledge for indexing health web resources in a quality-controlled gateway.
%V 136
%1 http://www.ncbi.nlm.nih.gov/pubmed/18487732?dopt=Abstract
%X OBJECTIVES: The aim of this study is to provide to indexers MeSH terms to be considered as major ones in a list of terms automatically extracted from a document.MATERIAL AND METHODS: We propose a method combining symbolic knowledge - the UMLS Metathesaurus and Semantic Network - and statistical knowledge drawn from co-occurrences of terms in the CISMeF database (a French-language quality-controlled health gateway) using data mining measures. The method was tested on CISMeF corpus of 293 resources.RESULTS: There was a proportion of 0.37+/-0.26 major terms in the processed records. The method produced lists of terms with a proportion of terms initially pointed out as major of 0.54+/-0.31.DISCUSSION: The method we propose reduces the number of terms, which seem not useful for content description of resources, such as "check tags", but retains the most descriptive ones. Discarding these terms is accounted for by: 1) the removal by using semantic knowledge of associations of concepts bearing no real medical significance, 2) the removal by using statistical knowledge of nonstatistically significant associations of terms.CONCLUSION: This method can assist effectively indexers in their daily work and will be soon applied in the CISMeF system.

%0 Journal Article
%A Catherine Quantin
%A François-André Allaert
%A Paul Avillach
%A Maniane Fassa
%A Benoît Riandey
%A Gilles Trouessin
%A Olivier Cohen
%B Int J Telemed Appl
%D 2008
%G eng
%P 678302
%R 10.1155/2008/678302
%T Building application-related patient identifiers: what solution for a European country?
%1 http://www.ncbi.nlm.nih.gov/pubmed/18401447?dopt=Abstract
%X We propose a method utilizing a derived social security number with the same reliability as the social security number. We show the anonymity techniques classically based on unidirectional hash functions (such as the secure hash algorithm (SHA-2) function that can guarantee the security, quality, and reliability of information if these techniques are applied to the Social Security Number). Hashing produces a strictly anonymous code that is always the same for a given individual, and thus enables patient data to be linked. Different solutions are developed and proposed in this article. Hashing the social security number will make it possible to link the information in the personal medical file to other national health information sources with the aim of completing or validating the personal medical record or conducting epidemiological and clinical research. This data linkage would meet the anonymous data requirements of the European directive on data protection.

%0 Journal Article
%A Paul Avillach
%A Michel Joubert
%A Marius Fieschi
%B AMIA Annu Symp Proc
%D 2007
%G eng
%P 31-5
%T A model for indexing medical documents combining statistical and symbolic knowledge.
%1 http://www.ncbi.nlm.nih.gov/pubmed/18693792?dopt=Abstract
%X OBJECTIVES: To develop and evaluate an information processing method based on terminologies, in order to index medical documents in any given documentary context.METHODS: We designed a model using both symbolic general knowledge extracted from the Unified Medical Language System (UMLS) and statistical knowledge extracted from a domain of application. Using statistical knowledge allowed us to contextualize the general knowledge for every particular situation. For each document studied, the extracted terms are ranked to highlight the most significant ones. The model was tested on a set of 17,079 French standardized discharge summaries (SDSs).RESULTS: The most important ICD-10 term of each SDS was ranked 1st or 2nd by the method in nearly 90% of the cases.CONCLUSIONS: The use of several terminologies leads to more precise indexing. The improvement achieved in the models implementation performances as a result of using semantic relationships is encouraging.

%0 Journal Article
%A Quantin
%A Allaert
%A Fassa
%A Avillach
%A Fieschi
%A Cohen
%B Conf Proc IEEE Eng Med Biol Soc
%D 2007
%G eng
%P 6161
%R 10.1109/IEMBS.2007.4353760
%T Interoperability issues regarding patient identification in Europe.
%V 2007
%1 http://www.ncbi.nlm.nih.gov/pubmed/18003426?dopt=Abstract

%0 Journal Article
%A Catherine Quantin
%A François-André Allaert
%A Paul Avillach
%A Benoît Riandey
%A Marius Fieschi
%A Maniane Fassa
%A Olivier Cohen
%B Stud Health Technol Inform
%D 2007
%G eng
%N Pt 1
%P 503-7
%T Proposal of a French health identification number interoperable at the European level.
%V 129
%1 http://www.ncbi.nlm.nih.gov/pubmed/17911768?dopt=Abstract
%X The French ministry of Health is setting up the Personal Medical Record (PMR). This innovative tool has long been expected by French Health Authorities, Associations of Patients, other Health's associations, those defending Individual Liberties and the French National Data Protection Authority. The PMR will lead to improvements in many areas such as Diagnosis (Research and monitoring) Healthcare (Management of emergencies, urgent situations, Temporal health monitoring and evaluation), Therapy (Cohorts of patients for Clinical trials and epidemiological studies). The PMR will foster safe healthcare management, clinical research and epidemiological studies. Nevertheless, it raises many important questions regarding duplicates and the quality, precision and coherence of the linkage with other health data coming from different sources. The currently planned identifying process raises many questions with regard to its ability to deal with potential duplicates and to perform data linkage with other health data sources. Through this article, using the electronic health records, we develop and propose an identification process to improve the French PMR. Our proposed unique patient identifier will guarantee the security, confidentiality and privacy of the personal data, and will prove to be particularly useful for health planning, health policies and research as well as clinical and epidemiological studies. Finally, it will certainly be interoperable with other European health information systems. We propose here an alternative identification procedure that would allow France to broaden the scope of its PMR project by making it possible to contribute to public health research and policy while increasing interoperability with European health information systems and preserving the confidentiality of the data.

%0 Journal Article
%A Catherine Quantin
%A François André Allaert
%A Maniane Fassa
%A Benoît Riandey
%A Paul Avillach
%A Olivier Cohen
%B Stud Health Technol Inform
%D 2007
%G eng
%P 246-55
%T How to manage secure direct access of European patients to their computerized medical record and personal medical record.
%V 127
%1 http://www.ncbi.nlm.nih.gov/pubmed/17901616?dopt=Abstract
%X The multiplication of the requests of the patients for a direct access to their Medical Record (MR), the development of Personal Medical Record (PMR) supervised by the patients themselves, the increasing development of the patients' electronic medical records (EMRs) and the world wide internet utilization will lead to envisage an access by using technical automatic and scientific way. It will require the addition of different conditions: a unique patient identifier which could base on a familial component in order to get access to the right record anywhere in Europe, very strict identity checks using cryptographic techniques such as those for the electronic signature, which will ensure the authentication of the requests sender and the integrity of the file but also the protection of the confidentiality and the access follow up. The electronic medical record must also be electronically signed by the practitioner in order to get evidence that he has given his agreement and taken the liability for that. This electronic signature also avoids any kind of post-transmission falsification. This will become extremely important, especially in France where patients will have the possibility to mask information that, they do not want to appear in their personal medical record. Currently, the idea of every citizen having electronic signatures available appears positively Utopian. But this is yet the case in eGovernment, eHealth and eShopping, world-wide. The same was thought about smart cards before they became generally available and useful when banks issued them.
