<!DOCTYPE html>
<html lang="en" dir="ltr" prefix="og: https://ogp.me/ns#">
  <head>
    <meta charset="utf-8" />
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-KDQLXZMJ');
</script>
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
    j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
    })(window,document,'script','dataLayer','GTM-52MDQDLC');
</script>
<link rel="canonical" href="https://avillach-lab.hms.harvard.edu/publications" />
<meta name="Generator" content="Drupal 10 (https://www.drupal.org)" />
<meta name="MobileOptimized" content="width" />
<meta name="HandheldFriendly" content="true" />
<meta name="viewport" content="width=device-width, initial-scale=1.0" />
<link rel="stylesheet" id="font-merriweather-css" href="//fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,400;0,700;1,400;1,700&amp;display=swap" type="text/css" media="all" />
<link rel="stylesheet" id="font-roboto-css" href="//fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&amp;display=swap" type="text/css" media="all" />
<link rel="stylesheet" id="font-material-icons-css" href="//fonts.googleapis.com/icon?family=Material+Icons" type="text/css" media="all" />
<link rel="stylesheet" id="font-material-icons-outlined-css" href="//fonts.googleapis.com/icon?family=Material+Icons+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" type="text/css" media="all" />
<link rel="stylesheet" id="font-postgrotesk-css" href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/fonts/postgrotesk.css" type="text/css" media="all" />
<link rel="stylesheet" id="font-brandontext-css" href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/fonts/brandontext.css" type="text/css" media="all" />
<link rel="stylesheet" id="font-arial-css" href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/fonts/arial.css" type="text/css" media="all" />
<link rel="stylesheet" id="font-treehouse-css" href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/fonts/treehouse.css" type="text/css" media="all" />
<link rel="stylesheet" id="font-sanomat-css" href="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/fonts/sanomat.css" type="text/css" media="all" />
<link rel="stylesheet" id="font-libre-franklin-css" href="//fonts.googleapis.com/css2?family=Libre+Franklin:ital,wght@0,100..900;1,100..900&amp;display=swap" type="text/css" media="all" />
<link rel="icon" href="/profiles/hwp_dstan/themes/hwp_hyard/favicon.ico" type="image/vnd.microsoft.icon" />

    <title lang="en">Publications | Avillach Lab</title>
    <link rel="stylesheet" media="all" href="/sites/g/files/omnuum4001/files/css/css_7Mkh_8OmRBjeLNhiXb-uK0ZUP5P9x2ehwIU6cf_0Z2A.css?delta=0&amp;language=en&amp;theme=hwp_hyard&amp;include=eJxdjEsOwyAQQy-EgnIiNBQ3GXX4iIEm3L40yqLqxpKf_OzRGqrDWbIiuCfLrGo3JFQSg7MJp5cNtReS5a5mP4rbB9VgEbjlyrSu44dukv20I80zJnH8yEmtJ8XU8c9z_74iGB3aEK-deTMOtVcuMYcu-ACq_UMo" />
<link rel="stylesheet" media="all" href="//fonts.googleapis.com/icon?family=Material+Icons" />
<link rel="stylesheet" media="all" href="//fonts.googleapis.com/icon?family=Material+Icons+Outlined" />
<link rel="stylesheet" media="all" href="/sites/g/files/omnuum4001/files/css/css_5hZOGLBmmKL-W7Ne88Z4F-Ul9VITfKpMQ5GKtgo_tzQ.css?delta=3&amp;language=en&amp;theme=hwp_hyard&amp;include=eJxdjEsOwyAQQy-EgnIiNBQ3GXX4iIEm3L40yqLqxpKf_OzRGqrDWbIiuCfLrGo3JFQSg7MJp5cNtReS5a5mP4rbB9VgEbjlyrSu44dukv20I80zJnH8yEmtJ8XU8c9z_74iGB3aEK-deTMOtVcuMYcu-ACq_UMo" />

    
  
                              <script>!function(e){var n="https://s.go-mpulse.net/boomerang/";if("False"=="True")e.BOOMR_config=e.BOOMR_config||{},e.BOOMR_config.PageParams=e.BOOMR_config.PageParams||{},e.BOOMR_config.PageParams.pci=!0,n="https://s2.go-mpulse.net/boomerang/";if(window.BOOMR_API_key="GBUU5-S9S78-P7UW2-BSS9F-LR483",function(){function e(){if(!o){var e=document.createElement("script");e.id="boomr-scr-as",e.src=window.BOOMR.url,e.async=!0,i.parentNode.appendChild(e),o=!0}}function t(e){o=!0;var n,t,a,r,d=document,O=window;if(window.BOOMR.snippetMethod=e?"if":"i",t=function(e,n){var t=d.createElement("script");t.id=n||"boomr-if-as",t.src=window.BOOMR.url,BOOMR_lstart=(new Date).getTime(),e=e||d.body,e.appendChild(t)},!window.addEventListener&&window.attachEvent&&navigator.userAgent.match(/MSIE [67]\./))return window.BOOMR.snippetMethod="s",void t(i.parentNode,"boomr-async");a=document.createElement("IFRAME"),a.src="about:blank",a.title="",a.role="presentation",a.loading="eager",r=(a.frameElement||a).style,r.width=0,r.height=0,r.border=0,r.display="none",i.parentNode.appendChild(a);try{O=a.contentWindow,d=O.document.open()}catch(_){n=document.domain,a.src="javascript:var d=document.open();d.domain='"+n+"';void(0);",O=a.contentWindow,d=O.document.open()}if(n)d._boomrl=function(){this.domain=n,t()},d.write("<bo"+"dy onload='document._boomrl();'>");else if(O._boomrl=function(){t()},O.addEventListener)O.addEventListener("load",O._boomrl,!1);else if(O.attachEvent)O.attachEvent("onload",O._boomrl);d.close()}function a(e){window.BOOMR_onload=e&&e.timeStamp||(new Date).getTime()}if(!window.BOOMR||!window.BOOMR.version&&!window.BOOMR.snippetExecuted){window.BOOMR=window.BOOMR||{},window.BOOMR.snippetStart=(new Date).getTime(),window.BOOMR.snippetExecuted=!0,window.BOOMR.snippetVersion=12,window.BOOMR.url=n+"GBUU5-S9S78-P7UW2-BSS9F-LR483";var i=document.currentScript||document.getElementsByTagName("script")[0],o=!1,r=document.createElement("link");if(r.relList&&"function"==typeof r.relList.supports&&r.relList.supports("preload")&&"as"in r)window.BOOMR.snippetMethod="p",r.href=window.BOOMR.url,r.rel="preload",r.as="script",r.addEventListener("load",e),r.addEventListener("error",function(){t(!0)}),setTimeout(function(){if(!o)t(!0)},3e3),BOOMR_lstart=(new Date).getTime(),i.parentNode.appendChild(r);else t(!1);if(window.addEventListener)window.addEventListener("load",a,!1);else if(window.attachEvent)window.attachEvent("onload",a)}}(),"".length>0)if(e&&"performance"in e&&e.performance&&"function"==typeof e.performance.setResourceTimingBufferSize)e.performance.setResourceTimingBufferSize();!function(){if(BOOMR=e.BOOMR||{},BOOMR.plugins=BOOMR.plugins||{},!BOOMR.plugins.AK){var n=""=="true"?1:0,t="",a="tmqygbyxfte4y2g5qx2a-f-4a44a4252-clientnsv4-s.akamaihd.net",i="false"=="true"?2:1,o={"ak.v":"39","ak.cp":"1597782","ak.ai":parseInt("841065",10),"ak.ol":"0","ak.cr":9,"ak.ipv":4,"ak.proto":"http/1.1","ak.rid":"227de049","ak.r":46720,"ak.a2":n,"ak.m":"b","ak.n":"ff","ak.bpcip":"155.33.131.0","ak.cport":43621,"ak.gh":"23.44.110.140","ak.quicv":"","ak.tlsv":"tls1.3","ak.0rtt":"","ak.0rtt.ed":"","ak.csrc":"-","ak.acc":"bbr","ak.t":"1759348212","ak.ak":"hOBiQwZUYzCg5VSAfCLimQ==WNA1fki4FIpLWzVlO6snpKJC0k0OW+3yWwlGRL7snvNMWGFdaNnvNhrT5Rhc7GNkGPk2uNoNFb8OrGruOJ5aEdRIuYBawykrRauNAz0qgyxNmT0yzDBHLtM1pQlVRCmTmBHQULOXe9NdhfTtqtCgB4v7SfsS2Hb8DXJhu80nLrSC7ZmDntkKx5tYWigOY9xAwTa/Fb/EqXLyYZEGnKDOPMtAiao5vXcaIOSU86CvGYQtANMtdUoEqnLO85yduDcZ81p/XJdPcm90oEkxSL82oD7j7UWav+wQl0pMFmGdJBmbKJLbH2N+u0tiaflfCcnklhY7dTbIZN7gOdUp2T1ruQIea/lIvAWho3AE3DK9QW/mXqviUVBQRiynz8HALyyQGOxf64+CH4Nmld70vgHxojyqy4ctGhtPozATxMl1Ihk=","ak.pv":"372","ak.dpoabenc":"","ak.tf":i};if(""!==t)o["ak.ruds"]=t;var r={i:!1,av:function(n){var t="http.initiator";if(n&&(!n[t]||"spa_hard"===n[t]))o["ak.feo"]=void 0!==e.aFeoApplied?1:0,BOOMR.addVar(o)},rv:function(){var e=["ak.bpcip","ak.cport","ak.cr","ak.csrc","ak.gh","ak.ipv","ak.m","ak.n","ak.ol","ak.proto","ak.quicv","ak.tlsv","ak.0rtt","ak.0rtt.ed","ak.r","ak.acc","ak.t","ak.tf"];BOOMR.removeVar(e)}};BOOMR.plugins.AK={akVars:o,akDNSPreFetchDomain:a,init:function(){if(!r.i){var e=BOOMR.subscribe;e("before_beacon",r.av,null,null),e("onbeacon",r.rv,null,null),r.i=!0}return this},is_complete:function(){return!0}}}}()}(window);</script></head>
  <body class="tex2jax_ignore hwp-font-group-libre-franklin hwp-icons-filled hwp-style-hms">
        <a href="#main-content" class="hwp-skip-to-content visually-hidden">
      <div class="hwp-relative hwp-button hwp-button--large hwp-bg-button-light-primary">
        <span class="hwp-button__text">Skip to main content</span>
        <span  class="material-icon material-icon--arrow-circle-down tex2jax_ignore hwp-font-group-libre-franklin hwp-icons-filled hwp-style-hms" aria-hidden="true">arrow_circle_down</span>
      </div>
    </a>
    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-KDQLXZMJ" height="0" width="0" style="display:none;visibility:hidden;"></iframe>
</noscript><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-52MDQDLC" height="0" width="0" style="display:none;visibility:hidden;"></iframe>
</noscript>
      <div class="dialog-off-canvas-main-canvas" data-off-canvas-main-canvas>
    <div class="layout-container">

  <header role="banner">
      <div>
    <div id="block-hwp-announcement-banners">
  
    
      



  </div>
<div id="block-hwp-globalheader">
  
    
      

<div  class="header-global header-global--stacked hwp-bg-dark-core">
  <div class="hwp-container hwp-container-px">
          <a class="header-global__logo"
         href="https://hms.harvard.edu/">
        <img alt="Harvard Medical School" style="height: 20px;" src="/sites/g/files/omnuum4001/files/hwp_hyard/header_logo/HMS-wordmark-white-Fixed_global%20header%20%283%29.svg" />

      </a>
          </div>
</div>

  </div>
<div id="block-hwp-siteheader">
  
    
      




<div  class="header-site header-site--stacked header-site--light-core">
  <div class="hwp-bg-light-core">
    <div class="header-site__main hwp-container">
      <div class="header-site__brand">
          <a href="/">
                <div class="header-site__title">Avillach Lab</div>
              <div class="header-site__subtitle">Department of Biomedical Informatics</div>
            </a>

      </div>

      <button class="header-site__menu-trigger js-header-site__menu-trigger" aria-expanded="false">
        <span class="material-icon material-icon--menu" aria-hidden="true">menu</span>
        <span class="material-icon material-icon--close" aria-hidden="true">close</span>
        <span class="header-site__menu-trigger-label">Menu</span>
      </button>
      <div class="header-site__utils hwp-shrink-0">
        <search role="search" class="header-site__search">
                    <form class="hwp-search-form-header" data-drupal-selector="hwp-search-form-header" action="/publications" method="post" id="hwp-search-form-header" accept-charset="UTF-8">
  <div class="hwp-form-type-search hwp-form-item js-form-item form-item form-type-search js-form-type-search form-item-search js-form-item-search form-no-label">
      <label for="edit-search" class="visually-hidden">Search</label>
        <input data-drupal-selector="edit-search" type="search" id="edit-search" name="search" value="" size="60" maxlength="128" class="form-search" />

        </div>

  


<button  data-twig-suggestion="header_site_search_submit" data-drupal-selector="edit-submit" type="submit" id="edit-submit" name="op" value="Search" class="hwp-button hwp-button--small button js-form-submit form-submit" >
      <span class="hwp-button__text">Search</span>
        <span class="material-icon" aria-hidden="true">search</span>
  </button>
<input autocomplete="off" data-drupal-selector="form-mfnvcgjgbem-npr2miify2scq84juzndo6zh1apyrio" type="hidden" name="form_build_id" value="form-MFNVcGJGbem-Npr2mIify2scQ84juzndo6zH1ApYrio" />
<input data-drupal-selector="edit-hwp-search-form-header" type="hidden" name="form_id" value="hwp_search_form_header" />

</form>

              </search>
                  <div class="header-site__utility-menu hwp-mt-16">
                <nav aria-label="utility">
  


    
                                        
    <ul  class="hwp-menu hwp-menu--utility">
                

<li  class="hwp-menu__item">
        <a href="/" class="menu-heading hwp-menu__link" lang="en" data-drupal-link-system-path="&lt;front&gt;">Archive</a>
    </li>
          </ul>
  
</nav>

            </div>
              </div>
    </div>
  </div>
  <div class="hwp-bg-light-core md:hwp-bg-light-core">
    <div class="header-site__navigation">
      <div class="header-site__brand">
          <a href="/">
                <div class="header-site__title">Avillach Lab</div>
              <div class="header-site__subtitle">Department of Biomedical Informatics</div>
            </a>

      </div>
          

<nav  class="hwp-main-menu" aria-label="Main">
  <div class="hwp-main-menu__container">
                      <ul class="hwp-main-menu__list" >
                      
        
        
        
        
        <li class="hwp-main-menu__item">
                      <a href="/pic-sure" lang="en" class="hwp-main-menu__link" id="menu_link_content:dae0d2b8-b536-460c-9a71-43c5eed0beeb" data-drupal-link-system-path="node/1610081">PIC-SURE</a>
                                      </li>
                      
        
                          
        
        
        <li class="hwp-main-menu__item hwp-main-menu__item--has-submenu">
                                                        <div data-level=0 class="hwp-expanded-item--wrapper">
                <a href="/research" lang="en" class="hwp-main-menu__link" data-drupal-link-system-path="node/278441">Research</a>
                <button class="js-hwp-main-menu__submenu-trigger hwp-main-menu__submenu-trigger-icon" aria-expanded="false">
                  <span class="hwp-visually-hidden">Research</span>
                  <span class="material-icon !hwp-block" aria-hidden="true">expand_more</span>
                </button>
              </div>
                                <ul class="hwp-main-menu__list-submenu" tabindex="-1">
                      
        
        
        
        
        <li class="hwp-main-menu__item">
                      <a href="/publications" class="hwp-main-menu__link is-active" id="menu_link_content:6bbcc4f9-2df4-4cf1-8587-686b487a3696" data-drupal-link-system-path="node/1774776" aria-current="page">Publications</a>
                                      </li>
                      
        
        
        
        
        <li class="hwp-main-menu__item">
                      <a href="https://avillach-lab.hms.harvard.edu/posters" class="hwp-main-menu__link" id="menu_link_content:0e7f47f5-8471-4442-a91d-124c395cead3">Posters</a>
                                      </li>
          </ul>
        
                  </li>
                      
        
        
        
        
        <li class="hwp-main-menu__item">
                      <a href="/courses-0" lang="en" class="hwp-main-menu__link" id="menu_link_content:3d05200b-0435-4c5d-9191-b4db5262d494" data-drupal-link-system-path="node/1150356">Courses</a>
                                      </li>
                      
        
        
        
        
        <li class="hwp-main-menu__item">
                      <a href="/people-display" lang="en" class="hwp-main-menu__link" id="menu_link_content:74e57527-be73-4fc6-b175-b80bbeca2937" data-drupal-link-system-path="node/589736">People</a>
                                      </li>
                      
        
        
        
        
        <li class="hwp-main-menu__item">
                      <a href="/news-events" title="" lang="en" class="hwp-main-menu__link" id="menu_link_content:dfabb64f-c029-40ea-8271-9bc6cb40f0a4" data-drupal-link-system-path="node/278436">News</a>
                                      </li>
                      
        
        
        
        
        <li class="hwp-main-menu__item">
                      <a href="/about" title="" lang="en" class="hwp-main-menu__link" id="menu_link_content:9d26eedf-3b89-4346-b244-7e99a5b019d5" data-drupal-link-system-path="node/278426">Contact us</a>
                                      </li>
          </ul>
                                                                                                      
    </div>
</nav>

      </div>
  </div>
</div>

  </div>
<div id="block-hwp-breadcrumbs">
  
    
      
  <nav  class="hwp-breadcrumbs hwp-bg-light-core" aria-labelledby="system-breadcrumb">
    <div class="hwp-container hwp-container-px">
      <p id="system-breadcrumb" hidden>Breadcrumbs</p>
      <ol>
                  <li>
                          <a href="/">Home</a>
                                      <span  class="material-icon material-icon--chevron-right" aria-hidden="true">chevron_right</span>
                      </li>
                  <li>
                          <a href="/research">Research</a>
                                      <span  class="material-icon material-icon--chevron-right" aria-hidden="true">chevron_right</span>
                      </li>
                  <li>
                          <span>Publications</span>
                                  </li>
              </ol>
    </div>
  </nav>

  </div>
<style>:root {
    --hwp-font-light-text: #FFFFFF;
    --hwp-font-light-link: #F3F4F4;
    --hwp-font-dark-text: #1E1E1E;
    --hwp-font-dark-link: #1E1E1E;
    --hwp-bg-light-base: #FFFFFF;
    --hwp-bg-light-alternative: #F3F4F4;
    --hwp-bg-light-core: #FFFFFF;
    --hwp-bg-dark-base: #A51C30;
    --hwp-bg-dark-alternative: #1E1E1E;
    --hwp-bg-dark-core: #1E1E1E;
    --hwp-button-light-primary: #FFFFFF;
    --hwp-button-light-secondary: #F3F4F4;
    --hwp-button-dark-primary: #1E1E1E;
    --hwp-button-dark-secondary: #A51C30;
    --hwp-button-strong-primary: #A51C30;
    --hwp-button-strong-secondary: #A51C30;
    --hwp-accent-light-regular: #FFFFFF;
    --hwp-accent-light-vibrant: #F3F4F4;
    --hwp-accent-dark-regular: #A51C30;
    --hwp-accent-dark-vibrant: #282828;
  }</style>

  </div>

  </header>

  
  

  

  

  

  <main lang="en" id="main-content" role="main" aria-label="Publications" aria-labelledby="page-title" tabindex="-1">
    <div class="layout-content">
        <div>
    <div data-drupal-messages-fallback class="hidden"></div>


<div class="search-page search-page--no-filters hwp-listing-page hwp-bibcite_reference">

      <div  lang="en" class="hwp-page-title hwp-bg-dark-base">
  <div class="hwp-container hwp-container-px hwp-py-32">
    
    <h1 class="text-large" >
      Publications
    </h1>
    
     </div>
</div>
  
      
        
    
<div class="hwp-section-heading" data-component-name="section-heading">
  <div class="hwp-section-heading__container hwp-container-px hwp-container">
    
        
              <div class="hwp-section-heading__description hwp-text-block ">
      
    </div>
    </div>
</div>
  
  <div class="search-page__columns hwp-container hwp-container-px">
    
    <div class="search-page__results">
    <div class="search-page__search-form">
          <form class="hwp-search-form-listing" data-drupal-selector="hwp-search-form-listing" action="/publications" method="post" id="hwp-search-form-listing" accept-charset="UTF-8">
  <div class="hwp-form-type-search hwp-form-item js-form-item form-item form-type-search js-form-type-search form-item-search js-form-item-search form-no-label">
      <label for="edit-search" class="visually-hidden">Search Within Results</label>
        <input data-drupal-selector="edit-search" type="search" id="edit-search" name="search" value="" size="60" maxlength="128" class="form-search" />

        </div>

  


<button  data-twig-suggestion="header_site_search_submit" data-drupal-selector="edit-submit" type="submit" id="edit-submit" name="op" value="Search Within Results" class="hwp-button hwp-button--small button js-form-submit form-submit" >
      <span class="hwp-button__text">Search Within Results</span>
        <span class="material-icon" aria-hidden="true">search</span>
  </button>
<input autocomplete="off" data-drupal-selector="form-imypagg73lxskojfuiqljy0prljdf1zrgbqht0wzqqg" type="hidden" name="form_build_id" value="form-IMyPAgg73LxsKOjFuiQLJy0PRLJdF1zrgBQht0WzQQg" />
<input data-drupal-selector="edit-hwp-search-form-listing" type="hidden" name="form_id" value="hwp_search_form_listing" />

</form>

      </div>

        <div class="views-element-container">
<div class="js-view-dom-id-ddcb69bd5855ccc9e781df57ec03f7ba1cb5ed7530fbfada62e7b414010061bf">
  
  
  

  <div class="search-page__search-header">
          <div class="hwp-hidden lg:hwp-block search-page__results-summary">
        100 results 
      </div>
    
    <button data-filters-trigger aria-expanded="false" class="search-page__filters-trigger">
      <span>Show filters</span>
      <span class="material-icon" aria-hidden="true">filter_alt</span>
    </button>
    <form class="views-exposed-form bef-exposed-form" data-bef-auto-submit-full-form="" data-bef-auto-submit="" data-bef-auto-submit-delay="500" data-bef-auto-submit-minimum-length="3" data-drupal-selector="views-exposed-form-hwp-search-search-page" action="/node/1774776" method="get" id="views-exposed-form-hwp-search-search-page" accept-charset="UTF-8">
  <input data-drupal-selector="edit-search" type="hidden" name="search" value="" />
<div class="js-form-item form-item hwp-form-item js-form-type-select hwp-form-type-select form-item-sort-bef-combine js-form-item-sort-bef-combine">
      <label for="edit-sort-bef-combine">Sort by</label>
        <select data-drupal-selector="edit-sort-bef-combine" id="edit-sort-bef-combine" name="sort_bef_combine" class="form-select"><option value="hwp_bibcite_reference_year--desc" selected="selected">Year of Publication</option><option value="title--asc">Alphabetical A-Z</option></select>
        
  <span class="material-icon" aria-hidden="true">sort</span>
</div>
<div data-drupal-selector="edit-actions" class="form-actions js-form-wrapper form-wrapper" id="edit-actions"><input data-bef-auto-submit-click="" class="js-hide button js-form-submit form-submit" data-drupal-selector="edit-submit-hwp-search" type="submit" id="edit-submit-hwp-search" value="Apply" />
</div>


</form>

  </div>

  

  <div class="lg:hwp-hidden search-page__results-summary">
    100 results 
  </div>

  <div class="search-page__result-items search-page__result-items-- ">
    

    
  <div class="hwp-citations-list__downloads">
    <div class="button-dropdown" data-component-name="button-dropdown">
  




<button
   class="hwp-text-link hwp-text-link--icon-right"
    id="citations_download">
<span class="hwp-text-link__text">    Download 100 citations  </span>
  <span class="material-icon" aria-hidden="true">download</span>
</button>

  
  <ul class="button-dropdown__items" role="menu" aria-labelledby="citations_download">
          <li role="menuitem">
        <a class="analytics-cta" href="/node/1774776/export?format=bibtex">BibTeX</a>
      </li>
          <li role="menuitem">
        <a class="analytics-cta" href="/node/1774776/export?format=endnote8">EndNote X3 XML</a>
      </li>
          <li role="menuitem">
        <a class="analytics-cta" href="/node/1774776/export?format=endnote7">EndNote 7 XML</a>
      </li>
          <li role="menuitem">
        <a class="analytics-cta" href="/node/1774776/export?format=tagged">Endnote tagged</a>
      </li>
          <li role="menuitem">
        <a class="analytics-cta" href="/node/1774776/export?format=marc">Marc</a>
      </li>
          <li role="menuitem">
        <a class="analytics-cta" href="/node/1774776/export?format=pubmed_id">PubMedId</a>
      </li>
          <li role="menuitem">
        <a class="analytics-cta" href="/node/1774776/export?format=ris">RIS</a>
      </li>
      </ul>
</div>
  </div>

<div id=>
      <div class="hwp-page-list__items" >
              <div  class="hwp-citations-group">
      <h3 class="h3 hwp-mb-24">2025</h3>
  
  <div class="hwp-citations-group__inner">
                              




<div  lang="en" class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation">
  <div class="hwp-citation__text">
    <div class="hwp-text-block__default-text">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Alkuraya I, Almansa AS, Eleonu A, Avillach P, Poduri A, Srivastava S. <a href="/publication/use-computational-phenotypes-predicting-genetic-subgroups-cerebral-palsy">Use of Computational Phenotypes for Predicting Genetic Subgroups of Cerebral Palsy.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2025; PMID: 39990554</div></div></div>
  </div>

    </div>

    <div class="hwp-text-block">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Alkuraya I, Almansa AS, Eleonu A, Avillach P, Poduri A, Srivastava S. <a href="/publication/use-computational-phenotypes-predicting-genetic-subgroups-cerebral-palsy">Use of Computational Phenotypes for Predicting Genetic Subgroups of Cerebral Palsy.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2025; PMID: 39990554</div></div></div>
  </div>

    </div>
  </div>

      <ul class="hwp-citation__ctas">
              <li class="expand-button-wrap">
          <button
            class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand"
            aria-expanded="false"
            aria-controls="hwp-citation-body-content-983783844"
          >
            <span class="material-icon material-icon--show" aria-hidden="true">add_circle</span>

            <span class="material-icon material-icon--hide" aria-hidden="true">do_not_disturb_on</span>
            <span class="hwp-text-link__text">Abstract</span>
          </button>
        </li>
      
          </ul>
  
            <div class="hwp-citation__full-body" aria-hidden="true" id="hwp-citation-body-content-983783844">
        <div class="hwp-text-block">
          
            <div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>INTRODUCTION: Emerging evidence suggests that 20-30% of cases of cerebral palsy (CP) may have a genetic cause. Our group previously identified subsets of patients with CP or CP-masquerading conditions who warrant genetic testing, including those with regression or progressive neurological symptoms (CP masqueraders) and those without any known risk factors for CP (cryptogenic CP). Recognition of these subgroups in clinical settings remains challenging.</p><p>METHODS: To address this challenge, we developed and evaluated a computational phenotyping approach using ICD- 9/ICD-10 billing codes to automatically identify patients with unexplained CP or CP-masquerading conditions who may benefit from genetic testing. We applied this computational phenotyping approach to a cohort of 250 participants from the Boston Children's Hospital CP Sequencing Study, aimed at identifying genetic causes in CP and CP-masquerading conditions.</p><p>RESULTS: Manual review served as the gold standard, identifying 8% as CP masqueraders, 42% as cryptogenic CP, and 50% as non-cryptogenic CP. Computational phenotyping based on ICD-9/10 codes achieved a sensitivity of 95%, specificity of 72%, positive predictive value of 77%, and negative predictive value of 94% in identifying cases warranting genetic testing.</p><p>CONCLUSIONS: Our findings demonstrate the feasibility of using computational phenotyping to identify patients with CP or CP- masquerading conditions who warrant genetic testing. Further studies are needed to evaluate the effectiveness and real-world application of this tool in larger healthcare systems. Nonetheless, the computational phenotyping approach holds promise as a possible clinical decision support that could be integrated into electronic health record systems, enhancing clinical workflows and facilitating actionable genetic diagnoses.</p></div>
      
        </div>
      </div>
    
      </div>

                          




<div  lang="en" class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation">
  <div class="hwp-citation__text">
    <div class="hwp-text-block__default-text">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yang D, Zhou D, Cai S, Gan Z, Pencina M, Avillach P, Cai T, Hong C. <a href="/publication/robust-automated-harmonization-heterogeneous-data-through-ensemble-machine-learning">Robust Automated Harmonization of Heterogeneous Data Through Ensemble Machine Learning: Algorithm Development and Validation Study.</a>.</div><div class="csl-right-inline">JMIR medical informatics. 2025;13:e54133. PMID: 39844378</div></div></div>
  </div>

    </div>

    <div class="hwp-text-block">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yang D, Zhou D, Cai S, Gan Z, Pencina M, Avillach P, Cai T, Hong C. <a href="/publication/robust-automated-harmonization-heterogeneous-data-through-ensemble-machine-learning">Robust Automated Harmonization of Heterogeneous Data Through Ensemble Machine Learning: Algorithm Development and Validation Study.</a>.</div><div class="csl-right-inline">JMIR medical informatics. 2025;13:e54133. PMID: 39844378</div></div></div>
  </div>

    </div>
  </div>

      <ul class="hwp-citation__ctas">
              <li class="expand-button-wrap">
          <button
            class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand"
            aria-expanded="false"
            aria-controls="hwp-citation-body-content-1907437921"
          >
            <span class="material-icon material-icon--show" aria-hidden="true">add_circle</span>

            <span class="material-icon material-icon--hide" aria-hidden="true">do_not_disturb_on</span>
            <span class="hwp-text-link__text">Abstract</span>
          </button>
        </li>
      
          </ul>
  
            <div class="hwp-citation__full-body" aria-hidden="true" id="hwp-citation-body-content-1907437921">
        <div class="hwp-text-block">
          
            <div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Cohort studies contain rich clinical data across large and diverse patient populations and are a common source of observational data for clinical research. Because large scale cohort studies are both time and resource intensive, one alternative is to harmonize data from existing cohorts through multicohort studies. However, given differences in variable encoding, accurate variable harmonization is difficult.</p><p>OBJECTIVE: We propose SONAR (Semantic and Distribution-Based Harmonization) as a method for harmonizing variables across cohort studies to facilitate multicohort studies.</p><p>METHODS: SONAR used semantic learning from variable descriptions and distribution learning from study participant data. Our method learned an embedding vector for each variable and used pairwise cosine similarity to score the similarity between variables. This approach was built off 3 National Institutes of Health cohorts, including the Cardiovascular Health Study, the Multi-Ethnic Study of Atherosclerosis, and the Women's Health Initiative. We also used gold standard labels to further refine the embeddings in a supervised manner.</p><p>RESULTS: The method was evaluated using manually curated gold standard labels from the 3 National Institutes of Health cohorts. We evaluated both the intracohort and intercohort variable harmonization performance. The supervised SONAR method outperformed existing benchmark methods for almost all intracohort and intercohort comparisons using area under the curve and top-k accuracy metrics. Notably, SONAR was able to significantly improve harmonization of concepts that were difficult for existing semantic methods to harmonize.</p><p>CONCLUSIONS: SONAR achieves accurate variable harmonization within and between cohort studies by harnessing the complementary strengths of semantic learning and variable distribution learning.</p></div>
      
        </div>
      </div>
    
      </div>

                          




<div  lang="en" class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation">
  <div class="hwp-citation__text">
    <div class="hwp-text-block__default-text">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Menezes MCS, Hoffmann AF, Tan ALM, Nalbandyan M, Omenn GS, Mazzotti DR, Hernández-Arango A, Visweswaran S, Venkatesh S, Mandl KD, Bourgeois FT, Lee JWK, Makmur A, Hanauer DA, Semanik MG, Kerivan LT, Hill T, Forero J, Restrepo C, Vigna M, Ceriana P, Abu-El-Rub N, Avillach P, Bellazzi R, Callaci T, Gutiérrez-Sacristán A, Malovini A, Mathew JP, Morris M, Murthy VL, Buonocore TM, Parimbelli E, Patel LP, Saez C, Samayamuthu MJ, Thompson JA, Tibollo V, Xia Z, Kohane IS, Records C for CC of C 19 by EH. <a href="/publication/potential-generative-pre-trained-transformer-4-gpt-4-analyse-medical-notes-three">The potential of Generative Pre-trained Transformer 4 (GPT-4) to analyse medical notes in three different languages: a retrospective model-evaluation study.</a>.</div><div class="csl-right-inline">The Lancet. Digital health. 2025;7(1):e35-e43. PMID: 39722251</div></div></div>
  </div>

    </div>

    <div class="hwp-text-block">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Menezes MCS, Hoffmann AF, Tan ALM, Nalbandyan M, Omenn GS, Mazzotti DR, Hernández-Arango A, Visweswaran S, Venkatesh S, Mandl KD, Bourgeois FT, Lee JWK, Makmur A, Hanauer DA, Semanik MG, Kerivan LT, Hill T, Forero J, Restrepo C, Vigna M, Ceriana P, Abu-El-Rub N, Avillach P, Bellazzi R, Callaci T, Gutiérrez-Sacristán A, Malovini A, Mathew JP, Morris M, Murthy VL, Buonocore TM, Parimbelli E, Patel LP, Saez C, Samayamuthu MJ, Thompson JA, Tibollo V, Xia Z, Kohane IS, Records C for CC of C 19 by EH. <a href="/publication/potential-generative-pre-trained-transformer-4-gpt-4-analyse-medical-notes-three">The potential of Generative Pre-trained Transformer 4 (GPT-4) to analyse medical notes in three different languages: a retrospective model-evaluation study.</a>.</div><div class="csl-right-inline">The Lancet. Digital health. 2025;7(1):e35-e43. PMID: 39722251</div></div></div>
  </div>

    </div>
  </div>

      <ul class="hwp-citation__ctas">
              <li class="expand-button-wrap">
          <button
            class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand"
            aria-expanded="false"
            aria-controls="hwp-citation-body-content-303159569"
          >
            <span class="material-icon material-icon--show" aria-hidden="true">add_circle</span>

            <span class="material-icon material-icon--hide" aria-hidden="true">do_not_disturb_on</span>
            <span class="hwp-text-link__text">Abstract</span>
          </button>
        </li>
      
          </ul>
  
            <div class="hwp-citation__full-body" aria-hidden="true" id="hwp-citation-body-content-303159569">
        <div class="hwp-text-block">
          
            <div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Patient notes contain substantial information but are difficult for computers to analyse due to their unstructured format. Large-language models (LLMs), such as Generative Pre-trained Transformer 4 (GPT-4), have changed our ability to process text, but we do not know how effectively they handle medical notes. We aimed to assess the ability of GPT-4 to answer predefined questions after reading medical notes in three different languages.</p><p>METHODS: For this retrospective model-evaluation study, we included eight university hospitals from four countries (ie, the USA, Colombia, Singapore, and Italy). Each site submitted seven de-identified medical notes related to seven separate patients to the coordinating centre between June 1, 2023, and Feb 28, 2024. Medical notes were written between Feb 1, 2020, and June 1, 2023. One site provided medical notes in Spanish, one site provided notes in Italian, and the remaining six sites provided notes in English. We included admission notes, progress notes, and consultation notes. No discharge summaries were included in this study. We advised participating sites to choose medical notes that, at time of hospital admission, were for patients who were male or female, aged 18-65 years, had a diagnosis of obesity, had a diagnosis of COVID-19, and had submitted an admission note. Adherence to these criteria was optional and participating sites randomly chose which medical notes to submit. When entering information into GPT-4, we prepended each medical note with an instruction prompt and a list of 14 questions that had been chosen a priori. Each medical note was individually given to GPT-4 in its original language and in separate sessions; the questions were always given in English. At each site, two physicians independently validated responses by GPT-4 and responded to all 14 questions. Each pair of physicians evaluated responses from GPT-4 to the seven medical notes from their own site only. Physicians were not masked to responses from GPT-4 before providing their own answers, but were masked to responses from the other physician.</p><p>FINDINGS: We collected 56 medical notes, of which 42 (75%) were in English, seven (13%) were in Italian, and seven (13%) were in Spanish. For each medical note, GPT-4 responded to 14 questions, resulting in 784 responses. In 622 (79%, 95% CI 76-82) of 784 responses, both physicians agreed with GPT-4. In 82 (11%, 8-13) responses, only one physician agreed with GPT-4. In the remaining 80 (10%, 8-13) responses, neither physician agreed with GPT-4. Both physicians agreed with GPT-4 more often for medical notes written in Spanish (86 [88%, 95% CI 79-93] of 98 responses) and Italian (82 [84%, 75-90] of 98 responses) than in English (454 [77%, 74-80] of 588 responses).</p><p>INTERPRETATION: The results of our model-evaluation study suggest that GPT-4 is accurate when analysing medical notes in three different languages. In the future, research should explore how LLMs can be integrated into clinical workflows to maximise their use in health care.</p><p>FUNDING: None.</p></div>
      
        </div>
      </div>
    
      </div>

                        </div>
</div>
              <div  class="hwp-citations-group">
      <h3 class="h3 hwp-mb-24">2024</h3>
  
  <div class="hwp-citations-group__inner">
                              




<div  lang="en" class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation">
  <div class="hwp-citation__text">
    <div class="hwp-text-block__default-text">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gutiérrez-Sacristán A, Makwana S, Dionne A, Mahanta S, Dyer KJ, Serrano F, Watrin C, Pages P, Mousavi S, Degala A, Lyons J, Pillion D, Zachariasse JM, Shekerdemian LS, Truong DT, Newburger JW, Avillach P. <a href="/publication/development-and-validation-open-source-pipeline-automatic-population-case-report-forms">Development and validation of an open-source pipeline for automatic population of case report forms from electronic health records: a pediatric multi-center prospective study.</a>.</div><div class="csl-right-inline">EBioMedicine. 2024;108:105337. PMID: 39288532</div></div></div>
  </div>

    </div>

    <div class="hwp-text-block">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gutiérrez-Sacristán A, Makwana S, Dionne A, Mahanta S, Dyer KJ, Serrano F, Watrin C, Pages P, Mousavi S, Degala A, Lyons J, Pillion D, Zachariasse JM, Shekerdemian LS, Truong DT, Newburger JW, Avillach P. <a href="/publication/development-and-validation-open-source-pipeline-automatic-population-case-report-forms">Development and validation of an open-source pipeline for automatic population of case report forms from electronic health records: a pediatric multi-center prospective study.</a>.</div><div class="csl-right-inline">EBioMedicine. 2024;108:105337. PMID: 39288532</div></div></div>
  </div>

    </div>
  </div>

      <ul class="hwp-citation__ctas">
              <li class="expand-button-wrap">
          <button
            class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand"
            aria-expanded="false"
            aria-controls="hwp-citation-body-content-613921253"
          >
            <span class="material-icon material-icon--show" aria-hidden="true">add_circle</span>

            <span class="material-icon material-icon--hide" aria-hidden="true">do_not_disturb_on</span>
            <span class="hwp-text-link__text">Abstract</span>
          </button>
        </li>
      
          </ul>
  
            <div class="hwp-citation__full-body" aria-hidden="true" id="hwp-citation-body-content-613921253">
        <div class="hwp-text-block">
          
            <div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Clinical trials and registry studies are essential for advancing research and developing novel treatments. However, these studies rely on manual entry of thousands of variables for each patient. Repurposing real-world data can significantly simplify the data collection, reduce transcription errors, and make the data entry process more efficient, consistent, and cost-effective.</p><p>METHODS: We developed an open-source computational pipeline to collect laboratory and medication information from the electronic health record (EHR) data and populate case report forms. The pipeline was developed and validated with data from two independent pediatric hospitals in the US as part of the Long-terM OUtcomes after Multisystem Inflammatory Syndrome In Children (MUSIC) study. Our pipeline allowed the completion of two of the most time-consuming forms. We compared automatically extracted results with manually entered values in one hospital and applied the pipeline to a second hospital, where the output served as the primary data source for case report forms.</p><p>FINDINGS: We extracted and populated 51,845 laboratory and 4913 medication values for 159 patients in two hospitals participating in a prospective pediatric study. We evaluated pipeline performance against data for 104 patients manually entered by clinicians in one of the hospitals. The highest concordance was found during patient hospitalization, with 91.59% of the automatically extracted laboratory and medication values corresponding with the manually entered values. In addition to the successfully populated values, we identified an additional 13,396 laboratory and 567 medication values of interest for the study.</p><p>INTERPRETATION: The automatic data entry of laboratory and medication values during admission is feasible and has a high concordance with the manually entered data. By implementing this proof of concept, we demonstrate the quality of automatic data extraction and highlight the potential of secondary use of EHR data to advance medical science by improving data entry efficiency and expediting clinical research.</p><p>FUNDING: NIH Grant 1OT3HL147154-01, U24HL135691, UG1HL135685.</p></div>
      
        </div>
      </div>
    
      </div>

                          




<div  lang="en" class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation">
  <div class="hwp-citation__text">
    <div class="hwp-text-block__default-text">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Wack M, Hassen-Khodja C, McDuffie MT, Bliss G, Horn EJ, Kothari C, McLarney B, Davis R, Hanson K, O’Boyle M, Betancur C, Avillach P. <a href="/publication/phenome-wide-profiling-identifies-genotype-phenotype-associations-phelan-mcdermid">Phenome-wide profiling identifies genotype-phenotype associations in Phelan-McDermid syndrome using family-sourced data from an international registry.</a>.</div><div class="csl-right-inline">Molecular autism. 2024;15(1):40. PMID: 39350236</div></div></div>
  </div>

    </div>

    <div class="hwp-text-block">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Wack M, Hassen-Khodja C, McDuffie MT, Bliss G, Horn EJ, Kothari C, McLarney B, Davis R, Hanson K, O’Boyle M, Betancur C, Avillach P. <a href="/publication/phenome-wide-profiling-identifies-genotype-phenotype-associations-phelan-mcdermid">Phenome-wide profiling identifies genotype-phenotype associations in Phelan-McDermid syndrome using family-sourced data from an international registry.</a>.</div><div class="csl-right-inline">Molecular autism. 2024;15(1):40. PMID: 39350236</div></div></div>
  </div>

    </div>
  </div>

      <ul class="hwp-citation__ctas">
              <li class="expand-button-wrap">
          <button
            class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand"
            aria-expanded="false"
            aria-controls="hwp-citation-body-content-1645799944"
          >
            <span class="material-icon material-icon--show" aria-hidden="true">add_circle</span>

            <span class="material-icon material-icon--hide" aria-hidden="true">do_not_disturb_on</span>
            <span class="hwp-text-link__text">Abstract</span>
          </button>
        </li>
      
          </ul>
  
            <div class="hwp-citation__full-body" aria-hidden="true" id="hwp-citation-body-content-1645799944">
        <div class="hwp-text-block">
          
            <div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Phelan-McDermid syndrome (PMS) is a rare neurodevelopmental disorder caused by 22q13 deletions that include the SHANK3 gene or pathogenic sequence&nbsp;variants in SHANK3. It is characterized by global developmental delay, intellectual disability, speech impairment, autism spectrum disorder, and hypotonia; other variable features include epilepsy, brain and renal malformations, and mild dysmorphic features. Here, we conducted genotype-phenotype correlation analyses using the PMS International Registry, a family-driven registry that compiles clinical data in the form of family-reported outcomes and family-sourced genetic test results.</p><p>METHODS: Data from the registry were harmonized and integrated into the i2b2/tranSMART clinical and genomics data warehouse. We gathered information from 401 individuals with 22q13 deletions including SHANK3 (n = 350, ranging in size from 10&nbsp;kb to 9.1&nbsp;Mb) or pathogenic or likely pathogenic SHANK3 sequence variants (n = 51), and used regression models with deletion size as a potential predictor of clinical outcomes for 328 phenotypes.</p><p>RESULTS: Our results showed that increased deletion size was significantly associated with delay in gross and fine motor acquisitions, a spectrum of conditions related to poor muscle tone, renal malformations, mild dysmorphic features (e.g., large fleshy hands, sacral dimple, dysplastic toenails, supernumerary teeth), lymphedema, congenital heart defects, and more frequent neuroimaging abnormalities and infections. These findings indicate that genes upstream of SHANK3 also contribute to some of the manifestations of PMS in individuals with larger deletions. We also showed that self-help skills, verbal ability and a range of psychiatric diagnoses (e.g., autism, ADHD, anxiety disorder) were more common among individuals with smaller deletions and SHANK3 variants.</p><p>LIMITATIONS: Some participants were tested with targeted 22q microarrays rather than genome-wide arrays, and karyotypes were unavailable in many cases, thus precluding the analysis of the effect of other copy number variants or chromosomal rearrangements on the phenotype.</p><p>CONCLUSIONS: This is the largest reported case series of individuals with PMS. Overall, we demonstrate the feasibility of using data from a family-sourced registry to conduct genotype-phenotype analyses in rare genetic disorders. We replicate and strengthen previous findings, and reveal novel associations between larger 22q13 deletions and congenital heart defects, neuroimaging abnormalities and recurrent infections.</p></div>
      
        </div>
      </div>
    
      </div>

                          




<div  lang="en" class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation">
  <div class="hwp-citation__text">
    <div class="hwp-text-block__default-text">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hernández-Agosto J, Santiago-Quiñones DI, García-Carmona LM, Gutiérrez-Sacristán A. <a href="/publication/substance-use-disorders-and-disparities-hispanic-and-non-hispanic-older-adults">Substance use disorders and disparities in hispanic and non-hispanic older adults.</a>.</div><div class="csl-right-inline">Drug and alcohol dependence. 2024;261:111357. PMID: 38896947</div></div></div>
  </div>

    </div>

    <div class="hwp-text-block">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hernández-Agosto J, Santiago-Quiñones DI, García-Carmona LM, Gutiérrez-Sacristán A. <a href="/publication/substance-use-disorders-and-disparities-hispanic-and-non-hispanic-older-adults">Substance use disorders and disparities in hispanic and non-hispanic older adults.</a>.</div><div class="csl-right-inline">Drug and alcohol dependence. 2024;261:111357. PMID: 38896947</div></div></div>
  </div>

    </div>
  </div>

      <ul class="hwp-citation__ctas">
              <li class="expand-button-wrap">
          <button
            class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand"
            aria-expanded="false"
            aria-controls="hwp-citation-body-content-2110352893"
          >
            <span class="material-icon material-icon--show" aria-hidden="true">add_circle</span>

            <span class="material-icon material-icon--hide" aria-hidden="true">do_not_disturb_on</span>
            <span class="hwp-text-link__text">Abstract</span>
          </button>
        </li>
      
          </ul>
  
            <div class="hwp-citation__full-body" aria-hidden="true" id="hwp-citation-body-content-2110352893">
        <div class="hwp-text-block">
          
            <div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: The global longevity revolution increased the older adult population, posing unique health and economic challenges with implications for healthcare, especially substance use disorders (SUD).</p><p>METHODS: This was a retrospective cohort study of United States older adults, Hispanic and non-Hispanic, who got at least one mental and/or behavioral disorder diagnosis between 2017 and 2021 at age 65 or older. SUD prevalence, prescription frequency changes over time, and comorbidities associated with each medication were compared.</p><p>RESULTS: Electronic health records for 356,133 older adults (110,236 Hispanics and 245,897 non-Hispanics) were analyzed. Notably, 79 % of Hispanics fell below the 100 % federal poverty level, compared to 60 % of non-Hispanics (P&lt;.001). Non-Hispanics also had significantly more average encounters (P=.003) and diagnoses (P&lt;.001). Regression analysis on alcohol-related disorders indicated that the odd ratios of being male (OR=2.93, P&lt;.000), and having low income (OR=1.62, P&lt;.000), increase the odds for this SUD, while being Hispanic and primarily speaking Spanish decreases the odds for all SUDs considered in this study.</p><p>CONCLUSIONS: This cohort study revealed significant disparities related to social determinants of health between Hispanic and non-Hispanic older adults and emphasizes the need for continuous surveillance of older adults as with SUDs. Differences in comorbidity patterns imply distinct risk factors within each population, influenced by demographic-specific elements. Recognizing these variations is essential for tailoring culturally sensitive prevention, intervention, and treatment strategies to each population's unique needs.</p></div>
      
        </div>
      </div>
    
      </div>

                          




<div  lang="en" class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation">
  <div class="hwp-citation__text">
    <div class="hwp-text-block__default-text">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Gutierrez A, Network UD, Kobren SN, Avillach P. <a href="/publication/varppud-variant-post-prioritization-developed-undiagnosed-genetic-disorders">VarPPUD: Variant post prioritization developed for undiagnosed genetic disorders.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2024; PMID: 38699371</div></div></div>
  </div>

    </div>

    <div class="hwp-text-block">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Yin R, Gutierrez A, Network UD, Kobren SN, Avillach P. <a href="/publication/varppud-variant-post-prioritization-developed-undiagnosed-genetic-disorders">VarPPUD: Variant post prioritization developed for undiagnosed genetic disorders.</a>.</div><div class="csl-right-inline">medRxiv : the preprint server for health sciences. 2024; PMID: 38699371</div></div></div>
  </div>

    </div>
  </div>

      <ul class="hwp-citation__ctas">
              <li class="expand-button-wrap">
          <button
            class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand"
            aria-expanded="false"
            aria-controls="hwp-citation-body-content-551639828"
          >
            <span class="material-icon material-icon--show" aria-hidden="true">add_circle</span>

            <span class="material-icon material-icon--hide" aria-hidden="true">do_not_disturb_on</span>
            <span class="hwp-text-link__text">Abstract</span>
          </button>
        </li>
      
          </ul>
  
            <div class="hwp-citation__full-body" aria-hidden="true" id="hwp-citation-body-content-551639828">
        <div class="hwp-text-block">
          
            <div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>Rare and ultra-rare genetic conditions are estimated to impact nearly 1 in 17 people worldwide, yet accurately pinpointing the diagnostic variants underlying each of these conditions remains a formidable challenge. Because comprehensive,  functional assessment of all possible genetic variants is infeasible, clinicians instead consider  variant pathogenicity predictions to distinguish plausibly disease-causing from benign variants across the genome. However, in the most difficult undiagnosed cases, such as those accepted to the Undiagnosed Diseases Network (UDN), existing pathogenicity predictions cannot reliably discern true etiological variant(s) from other deleterious candidate variants that were prioritized through N-of-1 efforts. Pinpointing the disease-causing variant from a pool of plausible candidates remains a largely manual effort requiring extensive clinical workups, functional and experimental assays, and eventual identification of genotype- and phenotype-matched individuals. Here, we introduce VarPPUD, a tool trained on prioritized variants from UDN cases, that leverages gene-, amino acid-, and nucleotide-level features to discern pathogenic variants from other deleterious variants that are unlikely to be confirmed as disease relevant. VarPPUD achieves a cross-validated accuracy of 79.3% and precision of 77.5% on a held-out subset of uniquely challenging UDN cases, respectively representing an average 18.6% and 23.4% improvement over nine traditional pathogenicity prediction approaches on this task. We validate VarPPUD's ability to discriminate likely from unlikely pathogenic variants on synthetic, GAN-generated candidate variants as well. Finally, we show how VarPPUD can be probed to evaluate each input feature's importance and contribution toward prediction-an essential step toward understanding the distinct characteristics of newly-uncovered disease-causing variants.</p></div>
      
        </div>
      </div>
    
      </div>

                          




<div  lang="en" class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation">
  <div class="hwp-citation__text">
    <div class="hwp-text-block__default-text">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hutch MR, Son J, Le TT, Hong C, Wang X, Abad ZSH, Morris M, Gutiérrez-Sacristán A, Klann JG, Spiridou A, Batugo A, Bellazzi R, Benoit V, Bonzel CL, Bryant WA, Chiudinelli L, Cho K, Das P, González TG, Hanauer DA, Henderson DW, Ho YL, Loh NHW, Makoudjou A, Makwana S, Malovini A, Moal B, Mowery DL, Neuraz A, Samayamuthu MJ, Vidorreta FJS, Schriver ER, Schubert P, Talbert J, Tan ALM, Tan BWL, Tan BWQ, Tibollo V, Tippman P, Verdy G, Yuan W, Avillach P, Gehlenborg N, Omenn GS, EHR C for CC of C 19 by, Visweswaran S, Cai T, Luo Y, Xia Z. <a href="/publication/neurological-diagnoses-hospitalized-covid-19-patients-associated-adverse-outcomes">Neurological diagnoses in hospitalized COVID-19 patients associated with adverse outcomes: A multinational cohort study.</a>.</div><div class="csl-right-inline">PLOS digital health. 2024;3(4):e0000484. PMID: 38620037</div></div></div>
  </div>

    </div>

    <div class="hwp-text-block">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Hutch MR, Son J, Le TT, Hong C, Wang X, Abad ZSH, Morris M, Gutiérrez-Sacristán A, Klann JG, Spiridou A, Batugo A, Bellazzi R, Benoit V, Bonzel CL, Bryant WA, Chiudinelli L, Cho K, Das P, González TG, Hanauer DA, Henderson DW, Ho YL, Loh NHW, Makoudjou A, Makwana S, Malovini A, Moal B, Mowery DL, Neuraz A, Samayamuthu MJ, Vidorreta FJS, Schriver ER, Schubert P, Talbert J, Tan ALM, Tan BWL, Tan BWQ, Tibollo V, Tippman P, Verdy G, Yuan W, Avillach P, Gehlenborg N, Omenn GS, EHR C for CC of C 19 by, Visweswaran S, Cai T, Luo Y, Xia Z. <a href="/publication/neurological-diagnoses-hospitalized-covid-19-patients-associated-adverse-outcomes">Neurological diagnoses in hospitalized COVID-19 patients associated with adverse outcomes: A multinational cohort study.</a>.</div><div class="csl-right-inline">PLOS digital health. 2024;3(4):e0000484. PMID: 38620037</div></div></div>
  </div>

    </div>
  </div>

      <ul class="hwp-citation__ctas">
              <li class="expand-button-wrap">
          <button
            class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand"
            aria-expanded="false"
            aria-controls="hwp-citation-body-content-2100453899"
          >
            <span class="material-icon material-icon--show" aria-hidden="true">add_circle</span>

            <span class="material-icon material-icon--hide" aria-hidden="true">do_not_disturb_on</span>
            <span class="hwp-text-link__text">Abstract</span>
          </button>
        </li>
      
          </ul>
  
            <div class="hwp-citation__full-body" aria-hidden="true" id="hwp-citation-body-content-2100453899">
        <div class="hwp-text-block">
          
            <div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>Few studies examining the patient outcomes of concurrent neurological manifestations during acute COVID-19 leveraged multinational cohorts of adults and children or distinguished between central and peripheral nervous system (CNS vs. PNS) involvement. Using a federated multinational network in which local clinicians and informatics experts curated the electronic health records data, we evaluated the risk of prolonged hospitalization and mortality in hospitalized COVID-19 patients from 21 healthcare systems across 7 countries. For adults, we used a federated learning approach whereby we ran Cox proportional hazard models locally at each healthcare system and performed a meta-analysis on the aggregated results to estimate the overall risk of adverse outcomes across our geographically diverse populations. For children, we reported descriptive statistics separately due to their low frequency of neurological involvement and poor outcomes. Among the 106,229 hospitalized COVID-19 patients (104,031 patients ≥18 years; 2,198 patients &lt;18 years, January 2020-October 2021), 15,101 (14%) had at least one CNS diagnosis, while 2,788 (3%) had at least one PNS diagnosis. After controlling for demographics and pre-existing conditions, adults with CNS involvement had longer hospital stay (11 versus 6 days) and greater risk of (Hazard Ratio = 1.78) and faster time to death (12 versus 24 days) than patients with no neurological condition (NNC) during acute COVID-19 hospitalization. Adults with PNS involvement also had longer hospital stay but lower risk of mortality than the NNC group. Although children had a low frequency of neurological involvement during COVID-19 hospitalization, a substantially higher proportion of children with CNS involvement died compared to those with NNC (6% vs 1%). Overall, patients with concurrent CNS manifestation during acute COVID-19 hospitalization faced greater risks for adverse clinical outcomes than patients without any neurological diagnosis. Our global informatics framework using a federated approach (versus a centralized data collection approach) has utility for clinical discovery beyond COVID-19.</p></div>
      
        </div>
      </div>
    
      </div>

                          




<div  lang="en" class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation">
  <div class="hwp-citation__text">
    <div class="hwp-text-block__default-text">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Huang M, Gutiérrez-Sacristán A, Janiak E, Young K, Starosta A, Blanton K, Azhir A, Goldfarb CN, Kuperwasser F, Schaefer KM, Stoddard RE, Vatsa R, Merz-Herrala AA, Bartz D. <a href="/publication/contraceptive-content-shared-social-media-analysis-twitter">Contraceptive content shared on social media: an analysis of Twitter.</a>.</div><div class="csl-right-inline">Contraception and reproductive medicine. 2024;9(1):5. PMID: 38321582</div></div></div>
  </div>

    </div>

    <div class="hwp-text-block">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Huang M, Gutiérrez-Sacristán A, Janiak E, Young K, Starosta A, Blanton K, Azhir A, Goldfarb CN, Kuperwasser F, Schaefer KM, Stoddard RE, Vatsa R, Merz-Herrala AA, Bartz D. <a href="/publication/contraceptive-content-shared-social-media-analysis-twitter">Contraceptive content shared on social media: an analysis of Twitter.</a>.</div><div class="csl-right-inline">Contraception and reproductive medicine. 2024;9(1):5. PMID: 38321582</div></div></div>
  </div>

    </div>
  </div>

      <ul class="hwp-citation__ctas">
              <li class="expand-button-wrap">
          <button
            class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand"
            aria-expanded="false"
            aria-controls="hwp-citation-body-content-1649283050"
          >
            <span class="material-icon material-icon--show" aria-hidden="true">add_circle</span>

            <span class="material-icon material-icon--hide" aria-hidden="true">do_not_disturb_on</span>
            <span class="hwp-text-link__text">Abstract</span>
          </button>
        </li>
      
          </ul>
  
            <div class="hwp-citation__full-body" aria-hidden="true" id="hwp-citation-body-content-1649283050">
        <div class="hwp-text-block">
          
            <div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Information on social media may affect peoples' contraceptive decision making. We performed an exploratory analysis of contraceptive content on Twitter (recently renamed X), a popular social media platform.</p><p>METHODS: We selected a random subset of 1% of publicly available, English-language tweets related to reversible, prescription contraceptive methods posted between January 2014 and December 2019. We oversampled tweets for the contraceptive patch to ensure at least 200 tweets per method. To create the codebook, we identified common themes specific to tweet content topics, tweet sources, and tweets soliciting information or providing advice. All posts were coded by two team members, and differences were adjudicated by a third reviewer. Descriptive analyses were reported with accompanying qualitative findings.</p><p>RESULTS: During the study period, 457,369 tweets about reversible contraceptive methods were published, with a random sample of 4,434 tweets used for final analysis. Tweets most frequently discussed contraceptive method decision-making (26.7%) and side effects (20.5%), particularly for long-acting reversible contraceptive methods and the depot medroxyprogesterone acetate shot. Tweets about logistics of use or adherence were common for short-acting reversible contraceptives. Tweets were frequently posted by contraceptive consumers (50.6%). A small proportion of tweets explicitly requested information (6.2%) or provided advice (4.2%).</p><p>CONCLUSIONS: Clinicians should be aware that individuals are exposed to information through Twitter that may affect contraceptive perceptions and decision making, particularly regarding long-acting reversible contraceptives. Social media is a valuable source for studying contraceptive beliefs missing in traditional health research and may be used by professionals to disseminate accurate contraceptive information.</p></div>
      
        </div>
      </div>
    
      </div>

                          




<div  lang="en" class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation">
  <div class="hwp-citation__text">
    <div class="hwp-text-block__default-text">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gargano MA, Matentzoglu N, Coleman B, Addo-Lartey EB, Anagnostopoulos A V, Anderton J, Avillach P, Bagley AM, Bakštein E, Balhoff JP, Baynam G, Bello SM, Berk M, Bertram H, Bishop S, Blau H, Bodenstein DF, Botas P, Boztug K, Čady J, Callahan TJ, Cameron R, Carbon SJ, Castellanos F, Caufield H, Chan LE, Chute CG, Cruz-Rojo J, Dahan-Oliel N, Davids JR, de Dieuleveult M, de Souza V, de Vries BBA, de Vries E, DePaulo R, Derfalvi B, Dhombres F, Diaz-Byrd C, Dingemans AJM, Donadille B, Duyzend M, Elfeky R, Essaid S, Fabrizzi C, Fico G, Firth H V, Freudenberg-Hua Y, Fullerton JM, Gabriel DL, Gilmour K, Giordano J, Goes FS, Moses RG, Green I, Griese M, Groza T, Gu W, Guthrie J, Gyori B, Hamosh A, Hanauer M, Hanušová K, He YO, Hegde H, Helbig I, Holasová K, Hoyt CT, Huang S, Hurwitz E, Jacobsen JOB, Jiang X, Joseph L, Keramatian K, King B, Knoflach K, Koolen DA, Kraus ML, Kroll C, Kusters M, Ladewig MS, Lagorce D, Lai MC, Lapunzina P, Laraway B, Lewis-Smith D, Li X, Lucano C, Majd M, Marazita ML, Martinez-Glez V, McHenry TH, McInnis MG, McMurry JA, Mihulová M, Millett CE, Mitchell PB, Moslerová V, Narutomi K, Nematollahi S, Nevado J, Nierenberg AA, Čajbiková NN, Nurnberger JI, Ogishima S, Olson D, Ortiz A, Pachajoa H, de Nanclares GP, Peters A, Putman T, Rapp CK, Rath A, Reese J, Rekerle L, Roberts AM, Roy S, Sanders SJ, Schuetz C, Schulte EC, Schulze TG, Schwarz M, Scott K, Seelow D, Seitz B, Shen Y, Similuk MN, Simon ES, Singh B, Smedley D, Smith CL, Smolinsky JT, Sperry S, Stafford E, Stefancsik R, Steinhaus R, Strawbridge R, Sundaramurthi JC, Talapova P, Castano JAT, Tesner P, Thomas RH, Thurm A, Turnovec M, van Gijn ME, Vasilevsky NA, Vlčková M, Walden A, Wang K, Wapner R, Ware JS, Wiafe AA, Wiafe SA, Wiggins LD, Williams AE, Wu C, Wyrwoll MJ, Xiong H, Yalin N, Yamamoto Y, Yatham LN, Yocum AK, Young AH, Yüksel Z, Zandi PP, Zankl A, Zarante I, Zvolský M, Toro S, Carmody LC, Harris NL, Munoz-Torres MC, Danis D, Mungall CJ, Köhler S, Haendel MA, Robinson PN. <a href="/publication/human-phenotype-ontology-2024-phenotypes-around-world">The Human Phenotype Ontology in 2024: phenotypes around the world.</a>.</div><div class="csl-right-inline">Nucleic acids research. 2024;52(D1):D1333-D1346. PMID: 37953324</div></div></div>
  </div>

    </div>

    <div class="hwp-text-block">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Gargano MA, Matentzoglu N, Coleman B, Addo-Lartey EB, Anagnostopoulos A V, Anderton J, Avillach P, Bagley AM, Bakštein E, Balhoff JP, Baynam G, Bello SM, Berk M, Bertram H, Bishop S, Blau H, Bodenstein DF, Botas P, Boztug K, Čady J, Callahan TJ, Cameron R, Carbon SJ, Castellanos F, Caufield H, Chan LE, Chute CG, Cruz-Rojo J, Dahan-Oliel N, Davids JR, de Dieuleveult M, de Souza V, de Vries BBA, de Vries E, DePaulo R, Derfalvi B, Dhombres F, Diaz-Byrd C, Dingemans AJM, Donadille B, Duyzend M, Elfeky R, Essaid S, Fabrizzi C, Fico G, Firth H V, Freudenberg-Hua Y, Fullerton JM, Gabriel DL, Gilmour K, Giordano J, Goes FS, Moses RG, Green I, Griese M, Groza T, Gu W, Guthrie J, Gyori B, Hamosh A, Hanauer M, Hanušová K, He YO, Hegde H, Helbig I, Holasová K, Hoyt CT, Huang S, Hurwitz E, Jacobsen JOB, Jiang X, Joseph L, Keramatian K, King B, Knoflach K, Koolen DA, Kraus ML, Kroll C, Kusters M, Ladewig MS, Lagorce D, Lai MC, Lapunzina P, Laraway B, Lewis-Smith D, Li X, Lucano C, Majd M, Marazita ML, Martinez-Glez V, McHenry TH, McInnis MG, McMurry JA, Mihulová M, Millett CE, Mitchell PB, Moslerová V, Narutomi K, Nematollahi S, Nevado J, Nierenberg AA, Čajbiková NN, Nurnberger JI, Ogishima S, Olson D, Ortiz A, Pachajoa H, de Nanclares GP, Peters A, Putman T, Rapp CK, Rath A, Reese J, Rekerle L, Roberts AM, Roy S, Sanders SJ, Schuetz C, Schulte EC, Schulze TG, Schwarz M, Scott K, Seelow D, Seitz B, Shen Y, Similuk MN, Simon ES, Singh B, Smedley D, Smith CL, Smolinsky JT, Sperry S, Stafford E, Stefancsik R, Steinhaus R, Strawbridge R, Sundaramurthi JC, Talapova P, Castano JAT, Tesner P, Thomas RH, Thurm A, Turnovec M, van Gijn ME, Vasilevsky NA, Vlčková M, Walden A, Wang K, Wapner R, Ware JS, Wiafe AA, Wiafe SA, Wiggins LD, Williams AE, Wu C, Wyrwoll MJ, Xiong H, Yalin N, Yamamoto Y, Yatham LN, Yocum AK, Young AH, Yüksel Z, Zandi PP, Zankl A, Zarante I, Zvolský M, Toro S, Carmody LC, Harris NL, Munoz-Torres MC, Danis D, Mungall CJ, Köhler S, Haendel MA, Robinson PN. <a href="/publication/human-phenotype-ontology-2024-phenotypes-around-world">The Human Phenotype Ontology in 2024: phenotypes around the world.</a>.</div><div class="csl-right-inline">Nucleic acids research. 2024;52(D1):D1333-D1346. PMID: 37953324</div></div></div>
  </div>

    </div>
  </div>

      <ul class="hwp-citation__ctas">
              <li class="expand-button-wrap">
          <button
            class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand"
            aria-expanded="false"
            aria-controls="hwp-citation-body-content-2011850567"
          >
            <span class="material-icon material-icon--show" aria-hidden="true">add_circle</span>

            <span class="material-icon material-icon--hide" aria-hidden="true">do_not_disturb_on</span>
            <span class="hwp-text-link__text">Abstract</span>
          </button>
        </li>
      
          </ul>
  
            <div class="hwp-citation__full-body" aria-hidden="true" id="hwp-citation-body-content-2011850567">
        <div class="hwp-text-block">
          
            <div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>The Human Phenotype Ontology (HPO) is a widely used resource that comprehensively organizes and defines the phenotypic features of human disease, enabling computational inference and supporting genomic and phenotypic analyses through semantic similarity and machine learning algorithms. The HPO has widespread applications in clinical diagnostics and translational research, including genomic diagnostics, gene-disease discovery, and cohort analytics. In recent years, groups around the world have developed translations of the HPO from English to other languages, and the HPO browser has been internationalized, allowing users to view HPO term labels and in many cases synonyms and definitions in ten languages in addition to English. Since our last report, a total of 2239 new HPO terms and 49235 new HPO annotations were developed, many in collaboration with external groups in the fields of psychiatry, arthrogryposis, immunology&nbsp;and cardiology. The Medical Action Ontology (MAxO) is a new effort to model treatments and other measures taken for clinical management. Finally, the HPO consortium is contributing to efforts to integrate the HPO and the GA4GH Phenopacket Schema into electronic health records (EHRs) with the goal of more standardized and computable integration of rare disease data in EHRs.</p></div>
      
        </div>
      </div>
    
      </div>

                        </div>
</div>
              <div  class="hwp-citations-group">
      <h3 class="h3 hwp-mb-24">2023</h3>
  
  <div class="hwp-citations-group__inner">
                              




<div  lang="en" class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation">
  <div class="hwp-citation__text">
    <div class="hwp-text-block__default-text">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Sperotto F, Gutiérrez-Sacristán A, Makwana S, Li X, Rofeberg VN, Cai T, Bourgeois FT, Omenn GS, Hanauer DA, Saez C, Bonzel CL, Bucholz E, Dionne A, Elias MD, García-Barrio N, González TG, Issitt RW, Kernan KF, Laird-Gion J, Maidlow SE, Mandl KD, Ahooyi TM, Moraleda C, Morris M, Moshal KL, Pedrera-Jiménez M, Shah MA, South AM, Spiridou A, Taylor DM, Verdy G, Visweswaran S, Wang X, Xia Z, Zachariasse JM, EHR C for CC of C 19 by, Newburger JW, Avillach P. <a href="/publication/clinical-phenotypes-and-outcomes-children-multisystem-inflammatory-syndrome-across-0">Clinical phenotypes and outcomes in children with multisystem inflammatory syndrome across SARS-CoV-2 variant eras: a multinational study from the 4CE consortium.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102212. PMID: 37745025</div></div></div>
  </div>

    </div>

    <div class="hwp-text-block">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Sperotto F, Gutiérrez-Sacristán A, Makwana S, Li X, Rofeberg VN, Cai T, Bourgeois FT, Omenn GS, Hanauer DA, Saez C, Bonzel CL, Bucholz E, Dionne A, Elias MD, García-Barrio N, González TG, Issitt RW, Kernan KF, Laird-Gion J, Maidlow SE, Mandl KD, Ahooyi TM, Moraleda C, Morris M, Moshal KL, Pedrera-Jiménez M, Shah MA, South AM, Spiridou A, Taylor DM, Verdy G, Visweswaran S, Wang X, Xia Z, Zachariasse JM, EHR C for CC of C 19 by, Newburger JW, Avillach P. <a href="/publication/clinical-phenotypes-and-outcomes-children-multisystem-inflammatory-syndrome-across-0">Clinical phenotypes and outcomes in children with multisystem inflammatory syndrome across SARS-CoV-2 variant eras: a multinational study from the 4CE consortium.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102212. PMID: 37745025</div></div></div>
  </div>

    </div>
  </div>

      <ul class="hwp-citation__ctas">
              <li class="expand-button-wrap">
          <button
            class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand"
            aria-expanded="false"
            aria-controls="hwp-citation-body-content-87103762"
          >
            <span class="material-icon material-icon--show" aria-hidden="true">add_circle</span>

            <span class="material-icon material-icon--hide" aria-hidden="true">do_not_disturb_on</span>
            <span class="hwp-text-link__text">Abstract</span>
          </button>
        </li>
      
          </ul>
  
            <div class="hwp-citation__full-body" aria-hidden="true" id="hwp-citation-body-content-87103762">
        <div class="hwp-text-block">
          
            <div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Multisystem inflammatory syndrome in children (MIS-C) is a severe complication of SARS-CoV-2 infection. It remains unclear how MIS-C phenotypes vary across SARS-CoV-2 variants. We aimed to investigate clinical characteristics and outcomes of MIS-C across SARS-CoV-2 eras.</p><p>METHODS: We performed a multicentre observational retrospective study including seven paediatric hospitals in four countries (France, Spain, U.K., and U.S.). All consecutive confirmed patients with MIS-C hospitalised between February 1st, 2020, and May 31st, 2022, were included. Electronic Health Records (EHR) data were used to calculate pooled risk differences (RD) and effect sizes (ES) at site level, using  as reference. Meta-analysis was used to pool data across sites.</p><p>FINDINGS: Of 598 patients with MIS-C (61% male, 39% female; mean age 9.7 years [SD 4.5]), 383 (64%) were admitted in the  era, 111 (19%) in the  era, and 104 (17%) in the  era. Compared with patients admitted in the  era, those admitted in the  era were younger (ES&nbsp;-1.18 years [95% CI&nbsp;-2.05,&nbsp;-0.32]), had fewer respiratory symptoms (RD&nbsp;-0.15 [95% CI&nbsp;-0.33,&nbsp;-0.04]), less frequent non-cardiogenic shock or systemic inflammatory response syndrome (SIRS) (RD&nbsp;-0.35 [95% CI&nbsp;-0.64,&nbsp;-0.07]), lower lymphocyte count (ES&nbsp;-0.16&nbsp;×&nbsp;10/uL [95% CI&nbsp;-0.30,&nbsp;-0.01]), lower C-reactive protein (ES&nbsp;-28.5&nbsp;mg/L [95% CI&nbsp;-46.3,&nbsp;-10.7]), and lower troponin (ES&nbsp;-0.14&nbsp;ng/mL [95% CI&nbsp;-0.26,&nbsp;-0.03]). Patients admitted in the  versus  eras were younger (ES&nbsp;-1.6 years [95% CI&nbsp;-2.5,&nbsp;-0.8]), had less frequent SIRS (RD&nbsp;-0.18 [95% CI&nbsp;-0.30,&nbsp;-0.05]), lower lymphocyte count (ES&nbsp;-0.39&nbsp;×&nbsp;10/uL [95% CI&nbsp;-0.52,&nbsp;-0.25]), lower troponin (ES&nbsp;-0.16&nbsp;ng/mL [95% CI&nbsp;-0.30,&nbsp;-0.01]) and less frequently received anticoagulation therapy (RD&nbsp;-0.19 [95% CI&nbsp;-0.37,&nbsp;-0.04]). Length of hospitalization was shorter in the  versus  eras (-1.3 days [95% CI&nbsp;-2.3,&nbsp;-0.4]).</p><p>INTERPRETATION: Our study suggested that MIS-C clinical phenotypes varied across SARS-CoV-2 eras, with patients in  and  eras being younger and less sick. EHR data can be effectively leveraged to identify rare complications of pandemic diseases and their variation over time.</p><p>FUNDING: None.</p></div>
      
        </div>
      </div>
    
      </div>

                          




<div  lang="en" class="hwp-citation hwp-citation--text-additional hwp-bg-light-base js-hwp-citation bibcite-reference" data-component-name="citation">
  <div class="hwp-citation__text">
    <div class="hwp-text-block__default-text">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Dagliati A, Strasser ZH, Abad ZSH, Klann JG, Wagholikar KB, Mesa R, Visweswaran S, Morris M, Luo Y, Henderson DW, Samayamuthu MJ, Tan BWQ, Verdy G, Omenn GS, Xia Z, Bellazzi R, EHR C for CC of C 19 by, Murphy SN, Holmes JH, Estiri H, EHR C for CC of C 19 by. <a href="/publication/characterization-long-covid-temporal-sub-phenotypes-distributed-representation-0">Characterization of long COVID temporal sub-phenotypes by distributed representation learning from electronic health record data: a cohort study.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102210. PMID: 37745021</div></div></div>
  </div>

    </div>

    <div class="hwp-text-block">
      <div class="bibcite-citation">
      <div class="csl-bib-body"><div class="csl-entry"><div class="csl-left-margin">Dagliati A, Strasser ZH, Abad ZSH, Klann JG, Wagholikar KB, Mesa R, Visweswaran S, Morris M, Luo Y, Henderson DW, Samayamuthu MJ, Tan BWQ, Verdy G, Omenn GS, Xia Z, Bellazzi R, EHR C for CC of C 19 by, Murphy SN, Holmes JH, Estiri H, EHR C for CC of C 19 by. <a href="/publication/characterization-long-covid-temporal-sub-phenotypes-distributed-representation-0">Characterization of long COVID temporal sub-phenotypes by distributed representation learning from electronic health record data: a cohort study.</a>.</div><div class="csl-right-inline">EClinicalMedicine. 2023;64:102210. PMID: 37745021</div></div></div>
  </div>

    </div>
  </div>

      <ul class="hwp-citation__ctas">
              <li class="expand-button-wrap">
          <button
            class="hwp-text-link hwp-text-link--icon-left hwp-citation__full-body-expand js-hwp-citation__full-body-expand"
            aria-expanded="false"
            aria-controls="hwp-citation-body-content-250295374"
          >
            <span class="material-icon material-icon--show" aria-hidden="true">add_circle</span>

            <span class="material-icon material-icon--hide" aria-hidden="true">do_not_disturb_on</span>
            <span class="hwp-text-link__text">Abstract</span>
          </button>
        </li>
      
          </ul>
  
            <div class="hwp-citation__full-body" aria-hidden="true" id="hwp-citation-body-content-250295374">
        <div class="hwp-text-block">
          
            <div class="hwp-text-block field field--name-bibcite-abst-e field--type-text-long field--label-hidden"><p>BACKGROUND: Characterizing Post-Acute Sequelae of COVID (SARS-CoV-2 Infection), or  has been challenging due to the multitude of sub-phenotypes, temporal attributes, and definitions. Scalable characterization of PASC sub-phenotypes can enhance screening capacities, disease management, and treatment planning.</p><p>METHODS: We conducted a retrospective multi-centre observational cohort study, leveraging longitudinal electronic health record (EHR) data of 30,422 patients from three healthcare systems in the Consortium for the Clinical Characterization of COVID-19 by EHR (4CE). From the total cohort, we applied a deductive approach on 12,424 individuals with follow-up data and developed a distributed representation learning process for providing augmented definitions for PASC sub-phenotypes.</p><p>FINDINGS: Our framework characterized seven PASC sub-phenotypes. We estimated that on average 15.7% of the hospitalized COVID-19 patients were likely to suffer from at least one PASC symptom and almost 5.98%, on average, had multiple symptoms. Joint pain and dyspnea had the highest prevalence, with an average prevalence of 5.45% and 4.53%, respectively.</p><p>INTERPRETATION: We provided a scalable framework to every participating healthcare system for estimating PASC sub-phenotypes prevalence and temporal attributes, thus developing a unified model that characterizes augmented sub-phenotypes across the different systems.</p><p>FUNDING: Authors are supported by National Institute of Allergy and Infectious Diseases, National Institute on Aging, National Center for Advancing Translational Sciences, National Medical Research Council, National Institute of Neurological Disorders and Stroke, European Union, National Institutes of Health, National Center for Advancing Translational Sciences.</p></div>
      
        </div>
      </div>
    
      </div>

                        </div>
</div>
          </div>
        
</div>


      </div>

  
            <nav class="hwp-pager hwp-pager--theme-light hwp-bg-light-core" role="navigation" aria-label="Pagination">
    <ul class="hwp-pager__items js-pager__items">
      <li>
                <div href="" title="Go to previous page"
        class="hwp-pager__item hwp-pager__item--previous hwp-pager__item--not-active"
        rel="prev">
          <span class="hwp-visually-hidden">Previous page</span>
          <span  class="material-icon material-icon--chevron-left" aria-hidden="true">chevron_left</span>
        </div>
      </li>

                          <li class="hwp-pager__item hwp-pager__item--is-active">
                                          <a href="?page=0" title="Current page" aria-current="page">1</a>
        </li>
              <li class="hwp-pager__item">
                                          <a href="?page=1" title="Go to page 2">2</a>
        </li>
              <li class="hwp-pager__item">
                                          <a href="?page=2" title="Go to page 3">3</a>
        </li>
                    <li class="hwp-pager__item hwp-pager__item--last hwp-pager__item--ellipsis">
          <a href="?page=8"
             title="Go to last page">
            <span class="hwp-visually-hidden">Last page</span>
            <span aria-hidden="true">9</span>
          </a>
        </li>
      
      <li>
                <a href="?page=1" title="Go to next page"
          class="hwp-pager__item hwp-pager__item--next "
          rel="prev">
          <span class="hwp-visually-hidden">Next page</span>
          <span  class="material-icon material-icon--chevron-right" aria-hidden="true">chevron_right</span>
        </a>
      </li>
    </ul>
  </nav>


  
  

  
  
</div>
</div>

      </div>
  </div>
</div>


  </div>

    </div>
    
      </main>

      <footer role="contentinfo">
        <div>
    <div id="block-hwp-footer-content">
  
    
      

      
<div  class="footer-site footer-site--columns hwp-bg-light-core hwp-text-14 hwp-leading-150">
  <div class="hwp-container hwp-container-px hwp-pt-24 hwp-pb-32 lg:hwp-pt-40">
    <div class="hwp-mb-24 lg:hwp-mb-32 hwp-footer__logo-container ">
                        <a href="/" class="footer-site__logo hwp-inline-flex">
            
<img
   class="image"
      src="/sites/g/files/omnuum4001/files/hwp_hyard/logo/HMS_footer%20logo%20%283%29.svg"
      alt="Harvard University"
            height="50"
        style="height: 50px;"
  />
          </a>
                  </div>
    <div class="lg:hwp-gap-40 hwp-footer__left-aligned lg:hwp-flex">
      <div class="lg:hwp-w-1/3 xl:hwp-w-1/4">
        <div class="hwp-flex hwp-flex-col hwp-gap-24 ">
                      <div></div>
                                <div class="hwp-flex hwp-gap-20 ">
              <span class="material-icon material-icon--footer-contact" aria-hidden="true">location_on</span>
              <div class="hwp-flex-1 "><p>Avillach Lab<br>Department of Biomedical Informatics<br>10 Shattuck St, 4th Floor<br>Boston, MA 02115</p></div>
            </div>
                    <div class="hwp-gap-24 hwp-grid ">
                                      <div class="hwp-flex hwp-gap-20 hwp-items-center">
                <span class="material-icon material-icon--footer-contact" aria-hidden="true">email</span>
                <a href="mailto:paul_avillach@hms.harvard.edu">paul_avillach@hms.harvard.edu</a>
              </div>
                                              </div>
        </div>
      </div>
      <div class="hwp-mt-32 lg:hwp-w-2/3 xl:hwp-w-3/4 lg:hwp-mt-0 lg:hwp-flex lg:hwp-flex-col">
            
                    <div class="lg:hwp-text-right lg:hwp-mt-auto">
            <a href="/login" class="hwp-py-8 hwp-mt-16 hwp-inline-flex">Admin login</a>
          </div>
              </div>
    </div>
  </div>
      

<div  class="hwp-global-footer hwp-bg-dark-core hwp-py-12 md:hwp-pb-20">
  <div
    class="hwp-container hwp-container-px hwp-text-14 hwp-text-white xl:hwp-flex
     xl:hwp-gap-x-24 hwp-flex-wrap xl:hwp-items-center xl:hwp-justify-between">
    <div class="hwp-global-footer__logo">
      <a href="https://www.harvard.edu" class="hwp-inline-flex hwp-py-12">
        
<img
   class="image"
      src="/profiles/hwp_dstan/themes/hwp_hyard/hwp-goldin/images/svg/footer-logo.svg"
      alt="Harvard University"
            height="50"
    />
      </a>
    </div>
    <div class="hwp-global-footer__copyright hwp-py-20 md:hwp-py-0">
      Copyright © 2025 The President and Fellows of Harvard College
    </div>
    <div class="hwp-global-footer__menu hwp-flex hwp-flex-col md:hwp-flex-row md:hwp-gap-x-32 xl:hwp-ml-auto">
                        <a href="https://accessibility.harvard.edu/" class="hwp-py-12">Accessibility</a>
                  <a href="https://accessibility.huit.harvard.edu/digital-accessibility-policy" class="hwp-py-12">Digital Accessibility</a>
                  <a href="https://www.harvard.edu/copyright-issue/" class="hwp-py-12">Report Copyright Infringement</a>
                  <a href="https://hwp.harvard.edu/harvardsites-platform-information-gathering-statement" class="hwp-py-12">Info Practices</a>
                  </div>
  </div>
</div>
  </div>

  </div>

  </div>

    </footer>
  
</div>
  </div>

    <script>window.dataLayer = window.dataLayer || []; window.dataLayer.push({"drupalLanguage":"en","drupalCountry":null,"siteName":"Avillach Lab","entityCreated":"1730986605","entityType":"node","content_type":"hwp_listing_page","entityId":"1774776","content_title":"Publications","primaryCategory":"publications","userUid":0,"harvard_affiliation_name":"harvard medical school","harvard_affiliation_short_name":"HMS","site_type":"lab_project","site_section":"external public facing","platform_name":"harvardsites_drupal","site_page_count_level":"251-500"});</script>

    <script type="application/json" data-drupal-selector="drupal-settings-json">{"path":{"baseUrl":"\/","pathPrefix":"","currentPath":"node\/1774776","currentPathIsAdmin":false,"isFront":false,"currentLanguage":"en"},"pluralDelimiter":"\u0003","suppressDeprecationErrors":true,"dataLayer":{"defaultLang":"en","languages":{"en":{"id":"en","name":"English","direction":"ltr","weight":0}}},"mathjax":{"config_type":1,"config":{"tex2jax":{"inlineMath":[["{$","$}"],["\\(","\\)"]],"processEscapes":"true","processClass":"tex2jax_process","ignoreClass":"tex2jax_ignore","displayMath":[["$$","$$"],["\\[","\\]"]]},"showProcessingMessages":"false","messageStyle":"none"}},"data":{"extlink":{"extTarget":false,"extTargetAppendNewWindowLabel":"(opens in a new window)","extTargetNoOverride":true,"extNofollow":true,"extTitleNoOverride":false,"extNoreferrer":true,"extFollowNoOverride":true,"extClass":"0","extLabel":"(link is external)","extImgClass":false,"extSubdomains":true,"extExclude":"","extInclude":"","extCssExclude":"","extCssInclude":"","extCssExplicit":"","extAlert":false,"extAlertText":"This link will take you to an external web site. We are not responsible for their content.","extHideIcons":false,"mailtoClass":"0","telClass":"","mailtoLabel":"(link sends email)","telLabel":"(link is a phone number)","extUseFontAwesome":true,"extIconPlacement":"append","extPreventOrphan":false,"extFaLinkClasses":"open-in-new","extFaMailtoClasses":"fa fa-envelope-o","extAdditionalLinkClasses":"","extAdditionalMailtoClasses":"","extAdditionalTelClasses":"","extFaTelClasses":"fa fa-phone","whitelistedDomains":[],"extExcludeNoreferrer":""}},"better_exposed_filters":{"autosubmit_exclude_textfield":false},"ajaxTrustedUrl":{"\/node\/1774776":true,"form_action_p_pvdeGsVG5zNF_XLGPTvYSKCf43t8qZYSwcfZl2uzM":true},"nodeBundle":"hwp_listing_page","nodeTitle":"Publications","baseFullUrl":"https:\/\/avillach-lab.hms.harvard.edu\/publications","user":{"uid":0,"permissionsHash":"6ac57c7fdce09b025fe5537ae9f2374e4194e6706e19f24430d351dec9cbd269"}}</script>
<script src="/sites/g/files/omnuum4001/files/js/js_afxJuUP7EbgrXGMPxB1TeHlYyl9f6QsHsOzlmDsDYVE.js?scope=footer&amp;delta=0&amp;language=en&amp;theme=hwp_hyard&amp;include=eJxtjFFuxCAMRC8UxJGQCdOFFmxkTDZ7-43ysaJqf0aa90YTYQYNOLsMpPBV6lWHp2kSxoyt2Bb_nzzAUKpbIqNKL6iPyHQU0bHlZw9pGHHYReE_k4ADbH88TquFf26cX6TJE7NM3tGuuYtXgy42TjNhl1R6kicvZi9GVmRFjQq762cubBSDy6D06_ampXWVAysWNYoV7s6tkeVvOv2Azf4GlEt-Og"></script>
<script src="//siteimproveanalytics.com/js/siteanalyze_2992.js"></script>
<script src="/sites/g/files/omnuum4001/files/js/js_ev-stpXjQqYn62qXzRBVBpdDNJtZjZWsWlki907-mIY.js?scope=footer&amp;delta=2&amp;language=en&amp;theme=hwp_hyard&amp;include=eJxtjFFuxCAMRC8UxJGQCdOFFmxkTDZ7-43ysaJqf0aa90YTYQYNOLsMpPBV6lWHp2kSxoyt2Bb_nzzAUKpbIqNKL6iPyHQU0bHlZw9pGHHYReE_k4ADbH88TquFf26cX6TJE7NM3tGuuYtXgy42TjNhl1R6kicvZi9GVmRFjQq762cubBSDy6D06_ampXWVAysWNYoV7s6tkeVvOv2Azf4GlEt-Og"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML&amp;config=default"></script>
<script src="/sites/g/files/omnuum4001/files/js/js_l4CJFfNETcd4ZwSHvsieZVLAKXrbPpS2FLTNe0yIhJg.js?scope=footer&amp;delta=4&amp;language=en&amp;theme=hwp_hyard&amp;include=eJxtjFFuxCAMRC8UxJGQCdOFFmxkTDZ7-43ysaJqf0aa90YTYQYNOLsMpPBV6lWHp2kSxoyt2Bb_nzzAUKpbIqNKL6iPyHQU0bHlZw9pGHHYReE_k4ADbH88TquFf26cX6TJE7NM3tGuuYtXgy42TjNhl1R6kicvZi9GVmRFjQq762cubBSDy6D06_ampXWVAysWNYoV7s6tkeVvOv2Azf4GlEt-Og"></script>

  </body>
</html>
